
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Mendelian_randomization_README/">
      
      
        <link rel="next" href="../Tools_Polygenic_risk_scores_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.9">
    
    
      
        <title>Meta_and_Multi_triat - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.4af4bdda.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Meta_and_Multi_triat
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Single_Cell_README/" class="md-tabs__link">
          
  
  Single_Cell

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Phenotype_Phenotype_README/" class="md-tabs__link">
          
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
      
        
        
      
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Functional_prediction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Functional_prediction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      Meta-analysis
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Meta-analysis">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Multi-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asset" class="md-nav__link">
    <span class="md-ellipsis">
      ASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#galesloot" class="md-nav__link">
    <span class="md-ellipsis">
      Galesloot
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipo" class="md-nav__link">
    <span class="md-ellipsis">
      HIPO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jass" class="md-nav__link">
    <span class="md-ellipsis">
      JASS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lcp-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      LCP-GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manova" class="md-nav__link">
    <span class="md-ellipsis">
      MANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mostest" class="md-nav__link">
    <span class="md-ellipsis">
      MOSTest
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mv-plink-mqfam" class="md-nav__link">
    <span class="md-ellipsis">
      MV-PLINK (MQFAM)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiphen" class="md-nav__link">
    <span class="md-ellipsis">
      MultiPhen
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pchat" class="md-nav__link">
    <span class="md-ellipsis">
      PCHAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porter" class="md-nav__link">
    <span class="md-ellipsis">
      Porter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salinas" class="md-nav__link">
    <span class="md-ellipsis">
      Salinas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stephens" class="md-nav__link">
    <span class="md-ellipsis">
      Stephens
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tates" class="md-nav__link">
    <span class="md-ellipsis">
      TATES
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang" class="md-nav__link">
    <span class="md-ellipsis">
      Yang
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amat" class="md-nav__link">
    <span class="md-ellipsis">
      aMAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastasset" class="md-nav__link">
    <span class="md-ellipsis">
      fastASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metacca" class="md-nav__link">
    <span class="md-ellipsis">
      metaCCA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metausatmetamanova" class="md-nav__link">
    <span class="md-ellipsis">
      metaUSAT/metaMANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Rare-variant
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Rare-variant">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Genealogy_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Single_Cell
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Single_Cell
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Single_Cell
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Framework_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Framework
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Harmonization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Harmonization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Network_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Network
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_QC_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    QC
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Trajectory_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Trajectory
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Reference
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Phenotype_Phenotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Gene_Gene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Protein_Protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cell_scRNA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    scRNA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial_Spatial_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      Meta-analysis
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Meta-analysis">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Multi-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asset" class="md-nav__link">
    <span class="md-ellipsis">
      ASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#galesloot" class="md-nav__link">
    <span class="md-ellipsis">
      Galesloot
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipo" class="md-nav__link">
    <span class="md-ellipsis">
      HIPO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jass" class="md-nav__link">
    <span class="md-ellipsis">
      JASS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lcp-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      LCP-GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manova" class="md-nav__link">
    <span class="md-ellipsis">
      MANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mostest" class="md-nav__link">
    <span class="md-ellipsis">
      MOSTest
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mv-plink-mqfam" class="md-nav__link">
    <span class="md-ellipsis">
      MV-PLINK (MQFAM)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiphen" class="md-nav__link">
    <span class="md-ellipsis">
      MultiPhen
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pchat" class="md-nav__link">
    <span class="md-ellipsis">
      PCHAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porter" class="md-nav__link">
    <span class="md-ellipsis">
      Porter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salinas" class="md-nav__link">
    <span class="md-ellipsis">
      Salinas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stephens" class="md-nav__link">
    <span class="md-ellipsis">
      Stephens
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tates" class="md-nav__link">
    <span class="md-ellipsis">
      TATES
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang" class="md-nav__link">
    <span class="md-ellipsis">
      Yang
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amat" class="md-nav__link">
    <span class="md-ellipsis">
      aMAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastasset" class="md-nav__link">
    <span class="md-ellipsis">
      fastASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metacca" class="md-nav__link">
    <span class="md-ellipsis">
      metaCCA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metausatmetamanova" class="md-nav__link">
    <span class="md-ellipsis">
      metaUSAT/metaMANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Rare-variant
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Rare-variant">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>Meta_and_Multi_triat</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#gwama">GWAMA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Mägi R, Morris AP. (2010) GWAMA: software for genome-wide association meta-analysis BMC Bioinformatics, 11 (1) 288. doi:10.1186/1471-2105-11-288. PMID 20509871</td>
<td style="text-align: left;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mantra">MANTRA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Morris AP. (2011) Transethnic meta-analysis of genomewide association studies Genet. Epidemiol., 35 (8) 809-822. doi:10.1002/gepi.20630. PMID 22125221</td>
<td style="text-align: left;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metal">METAL</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Willer CJ, Li Y, Abecasis GR. (2010) METAL: fast and efficient meta-analysis of genomewide association scans Bioinformatics, 26 (17) 2190-2191. doi:10.1093/bioinformatics/btq340. PMID 20616382</td>
<td style="text-align: left;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mr-mega">MR-MEGA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Mägi R, Horikoshi M, Sofer T, Mahajan A, ...&amp;, Morris AP. (2017) Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution Hum. Mol. Genet., 26 (18) 3639-3650. doi:10.1093/hmg/ddx280. PMID 28911207</td>
<td style="text-align: left;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#asset">ASSET</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Bhattacharjee S, Rajaraman P, Jacobs KB, Wheeler WA, ...&amp;, Chatterjee N. (2012) A subset-based approach improves power and interpretation for the combined analysis of genetic association studies of heterogeneous traits Am. J. Hum. Genet., 90 (5) 821-835. doi:10.1016/j.ajhg.2012.03.015. PMID 22560090</td>
<td style="text-align: left;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#factorgo">FactorGO</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Zhang Z, Jung J, Kim A, Suboc N, ...&amp;, Mancuso N. (2023) A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics Am. J. Hum. Genet.,  () . doi:10.1016/j.ajhg.2023.09.015. PMID 37879338</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#galesloot">Galesloot</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Galesloot TE, van Steen K, Kiemeney LA, Janss LL, ...&amp;, Vermeulen SH. (2014) A comparison of multivariate genome-wide association methods PLoS One, 9 (4) e95923. doi:10.1371/journal.pone.0095923. PMID 24763738</td>
<td style="text-align: left;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#genomic-sem">Genomic-SEM</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Grotzinger AD, Rhemtulla M, de Vlaming R, Ritchie SJ, ...&amp;, Tucker-Drob EM. (2019) Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits Nat Hum Behav, 3 (5) 513-525. doi:10.1038/s41562-019-0566-x. PMID 30962613</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hipo">HIPO</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Qi G, Chatterjee N. (2018) Heritability informed power optimization (HIPO) leads to enhanced detection of genetic associations across multiple traits PLoS Genet., 14 (10) e1007549. doi:10.1371/journal.pgen.1007549. PMID 30289880</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#jass">JASS</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Julienne H, Lechat P, Guillemot V, Lasry C, ...&amp;, Aschard H. (2020) JASS: command line and web interface for the joint analysis of GWAS results NAR Genom. Bioinform., 2 (1) lqaa003. doi:10.1093/nargab/lqaa003. PMID 32002517</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lcp-gwas">LCP-GWAS</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Ruotsalainen SE, Partanen JJ, Cichonska A, Lin J, ...&amp;, Koskela J. (2021) An expanded analysis framework for multivariate GWAS connects inflammatory biomarkers to functional variants and disease Eur. J. Hum. Genet., 29 (2) 309-324. doi:10.1038/s41431-020-00730-8. PMID 33110245</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#manova">MANOVA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Pillai, K. C. S. Some new test criteria in multivariate analysis. Ann. Math. Stat. 26, 117–121 (1955).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mostest">MOSTest</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">van der Meer D, Frei O, Kaufmann T, Shadrin AA, ...&amp;, Dale AM. (2020) Understanding the genetic determinants of the brain with MOSTest Nat. Commun., 11 (1) 3512. doi:10.1038/s41467-020-17368-1. PMID 32665545</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mtag">MTAG</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Turley P, Walters RK, Maghzian O, Okbay A, ...&amp;, Pitts SJ. (2018) Multi-trait analysis of genome-wide association summary statistics using MTAG Nat. Genet., 50 (2) 229-237. doi:10.1038/s41588-017-0009-4. PMID 29292387</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mv-plink-mqfam">MV-PLINK (MQFAM)</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Ferreira MA, Purcell SM. (2009) A multivariate test of association Bioinformatics, 25 (1) 132-133. doi:10.1093/bioinformatics/btn563. PMID 19019849</td>
<td style="text-align: left;">2009</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multiphen">MultiPhen</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">O'Reilly PF, Hoggart CJ, Pomyen Y, Calboli FC, ...&amp;, Coin LJ. (2012) MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS PLoS One, 7 (5) e34861. doi:10.1371/journal.pone.0034861. PMID 22567092</td>
<td style="text-align: left;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pchat">PCHAT</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Klei L, Luca D, Devlin B, Roeder K. (2008) Pleiotropy and principal components of heritability combine to increase power for association analysis Genet. Epidemiol., 32 (1) 9-19. doi:10.1002/gepi.20257. PMID 17922480</td>
<td style="text-align: left;">2008</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#porter">Porter</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Porter HF, O'Reilly PF. (2017) Multivariate simulation framework reveals performance of multi-trait GWAS methods Sci. Rep., 7 (1) 38837. doi:10.1038/srep38837. PMID 28287610</td>
<td style="text-align: left;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#salinas">Salinas</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Salinas YD, Wang Z, DeWan AT. (2018) Statistical analysis of multiple phenotypes in genetic epidemiologic studies: From cross-phenotype associations to pleiotropy Am. J. Epidemiol., 187 (4) 855-863. doi:10.1093/aje/kwx296. PMID 29020254</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#stephens">Stephens</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Stephens M. (2013) A unified framework for association analysis with multiple related phenotypes PLoS One, 8 (7) e65245. doi:10.1371/journal.pone.0065245. PMID 23861737</td>
<td style="text-align: left;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tates">TATES</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">van der Sluis S, Posthuma D, Dolan CV. (2013) TATES: efficient multivariate genotype-phenotype analysis for genome-wide association studies PLoS Genet., 9 (1) e1003235. doi:10.1371/journal.pgen.1003235. PMID 23359524</td>
<td style="text-align: left;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#yang">Yang</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Yang Q, Wang Y. (2012) Methods for analyzing multivariate phenotypes in genetic association studies J. Probab. Stat., 2012 () 652569. doi:10.1155/2012/652569. PMID 24748889</td>
<td style="text-align: left;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#amat">aMAT</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Wu C. (2020) Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank Genetics, 215 (4) 947-958. doi:10.1534/genetics.120.303242. PMID 32540950</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastasset">fastASSET</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Qi G, Chhetri SB, Ray D, Dutta D, ...&amp;, Chatterjee N. (2024) Genome-wide large-scale multi-trait analysis characterizes global patterns of pleiotropy and unique trait-specific variants Nat. Commun., 15 (1) 6985. doi:10.1038/s41467-024-51075-5. PMID 39143063</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metacca">metaCCA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Cichonska A, Rousu J, Marttinen P, Kangas AJ, ...&amp;, Pirinen M. (2016) metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis Bioinformatics, 32 (13) 1981-1989. doi:10.1093/bioinformatics/btw052. PMID 27153689</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metausatmetamanova">metaUSAT/metaMANOVA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Ray D, Boehnke M. (2018) Methods for meta-analysis of multiple traits using GWAS summary statistics Genet. Epidemiol., 42 (2) 134-145. doi:10.1002/gepi.22105. PMID 29226385</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mvgwama">mvGWAMA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Jansen IE, Savage JE, Watanabe K, Bryois J, ...&amp;, Posthuma D. (2019) Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk Nat. Genet., 51 (3) 404-413. doi:10.1038/s41588-018-0311-9. PMID 30617256</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metaskat">MetaSKAT</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Lee S, Teslovich TM, Boehnke M, Lin X. (2013) General framework for meta-analysis of rare variants in sequencing association studies Am. J. Hum. Genet., 93 (1) 42-53. doi:10.1016/j.ajhg.2013.05.010. PMID 23768515</td>
<td style="text-align: left;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metastaar">MetaSTAAR</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Li X, Quick C, Zhou H, Gaynor SM, ...&amp;, Lin X. (2023) Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies Nat. Genet., 55 (1) 154-164. doi:10.1038/s41588-022-01225-6. PMID 36564505</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#raremetal">RareMETAL</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Feng S, Liu D, Zhan X, Wing MK, ...&amp;, Abecasis GR. (2014) RAREMETAL: fast and powerful meta-analysis for rare variants Bioinformatics, 30 (19) 2828-2829. doi:10.1093/bioinformatics/btu367. PMID 24894501</td>
<td style="text-align: left;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#smmat">SMMAT</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Chen H, Huffman JE, Brody JA, Wang C, ...&amp;, Lin X. (2019) Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies Am. J. Hum. Genet., 104 (2) 260-274. doi:10.1016/j.ajhg.2018.12.012. PMID 30639324</td>
<td style="text-align: left;">2019</td>
</tr>
</tbody>
</table>
<h2 id="meta-analysis">Meta-analysis</h2>
<h3 id="gwama">GWAMA</h3>
<ul>
<li><strong>NAME</strong> : GWAMA </li>
<li><strong>SHORT NAME</strong> : GWAMA </li>
<li><strong>FULL NAME</strong> : Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : Software tool for meta analysis of whole genome association data </li>
<li><strong>URL</strong> : <a href="https://genomics.ut.ee/en/tools">https://genomics.ut.ee/en/tools</a> </li>
<li><strong>TITLE</strong> : GWAMA: software for genome-wide association meta-analysis </li>
<li><strong>DOI</strong> : 10.1186/1471-2105-11-288 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: Despite the recent success of genome-wide association studies in identifying novel loci contributing effects to complex human traits, such as type 2 diabetes and obesity, much of the genetic component of variation in these phenotypes remains unexplained. One way to improving power to detect further novel loci is through meta-analysis of studies from the same population, increasing the sample size over any individual study. Although statistical software analysis packages incorporate routines for meta-analysis, they are ill equipped to meet the challenges of the scale and complexity of data generated in genome-wide association studies. RESULTS: We have developed flexible, open-source software for the meta-analysis of genome-wide association studies. The software incorporates a variety of error trapping facilities, and provides a range of meta-analysis summary statistics. The software is distributed with scripts that allow simple formatting of files containing the results of each association study and generate graphical summaries of genome-wide meta-analysis results. CONCLUSIONS: The GWAMA (Genome-Wide Association Meta-Analysis) software has been developed to perform meta-analysis of summary statistics generated from genome-wide association studies of dichotomous phenotypes or quantitative traits. Software with source files, documentation and example data files are freely available online at http://www.well.ox.ac.uk/GWAMA. </li>
<li><strong>CITATION</strong> : Mägi R, Morris AP. (2010) GWAMA: software for genome-wide association meta-analysis BMC Bioinformatics, 11 (1) 288. doi:10.1186/1471-2105-11-288. PMID 20509871 </li>
<li><strong>JOURNAL_INFO</strong> : BMC bioinformatics ; BMC Bioinformatics ; 2010 ; 11 ; 1 ; 288 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20509871">20509871</a> </li>
</ul>
<h3 id="mantra">MANTRA</h3>
<ul>
<li><strong>NAME</strong> : MANTRA </li>
<li><strong>SHORT NAME</strong> : MANTRA </li>
<li><strong>FULL NAME</strong> : Meta-ANalysis of Transethnic Association studies </li>
<li><strong>KEYWORDS</strong> : cross-population </li>
<li><strong>TITLE</strong> : Transethnic meta-analysis of genomewide association studies </li>
<li><strong>DOI</strong> : 10.1002/gepi.20630 </li>
<li><strong>ABSTRACT</strong> : The detection of loci contributing effects to complex human traits, and their subsequent fine-mapping for the location of causal variants, remains a considerable challenge for the genetics research community. Meta-analyses of genomewide association studies, primarily ascertained from European-descent populations, have made considerable advances in our understanding of complex trait genetics, although much of their heritability is still unexplained. With the increasing availability of genomewide association data from diverse populations, transethnic meta-analysis may offer an exciting opportunity to increase the power to detect novel complex trait loci and to improve the resolution of fine-mapping of causal variants by leveraging differences in local linkage disequilibrium structure between ethnic groups. However, we might also expect there to be substantial genetic heterogeneity between diverse populations, both in terms of the spectrum of causal variants and their allelic effects, which cannot easily be accommodated through traditional approaches to meta-analysis. In order to address this challenge, I propose novel transethnic meta-analysis methodology that takes account of the expected similarity in allelic effects between the most closely related populations, while allowing for heterogeneity between more diverse ethnic groups. This approach yields substantial improvements in performance, compared to fixed-effects meta-analysis, both in terms of power to detect association, and localization of the causal variant, over a range of models of heterogeneity between ethnic groups. Furthermore, when the similarity in allelic effects between populations is well captured by their relatedness, this approach has increased power and mapping resolution over random-effects meta-analysis. </li>
<li><strong>CITATION</strong> : Morris AP. (2011) Transethnic meta-analysis of genomewide association studies Genet. Epidemiol., 35 (8) 809-822. doi:10.1002/gepi.20630. PMID 22125221 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet. Epidemiol. ; 2011 ; 35 ; 8 ; 809-822 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22125221">22125221</a> </li>
</ul>
<h3 id="metal">METAL</h3>
<ul>
<li><strong>NAME</strong> : METAL </li>
<li><strong>SHORT NAME</strong> : METAL </li>
<li><strong>FULL NAME</strong> : METAL </li>
<li><strong>DESCRIPTION</strong> : METAL is a tool for meta-analysis genomewide association scans. METAL can combine either (a) test statistics and standard errors or (b) p-values across studies (taking sample size and direction of effect into account). METAL analysis is a convenient alternative to a direct analysis of merged data from multiple studies. It is especially appropriate when data from the individual studies cannot be analyzed together because of differences in ethnicity, phenotype distribution, gender or constraints in sharing of individual level data imposed. Meta-analysis results in little or no loss of efficiency compared to analysis of a combined dataset including data from all individual studies. </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/METAL_Documentation">https://genome.sph.umich.edu/wiki/METAL_Documentation</a> </li>
<li><strong>TITLE</strong> : METAL: fast and efficient meta-analysis of genomewide association scans </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btq340 </li>
<li><strong>ABSTRACT</strong> : SUMMARY: METAL provides a computationally efficient tool for meta-analysis of genome-wide association scans, which is a commonly used approach for improving power complex traits gene mapping studies. METAL provides a rich scripting interface and implements efficient memory management to allow analyses of very large data sets and to support a variety of input file formats. AVAILABILITY AND IMPLEMENTATION: METAL, including source code, documentation, examples, and executables, is available at http://www.sph.umich.edu/csg/abecasis/metal/. </li>
<li><strong>CITATION</strong> : Willer CJ, Li Y, Abecasis GR. (2010) METAL: fast and efficient meta-analysis of genomewide association scans Bioinformatics, 26 (17) 2190-2191. doi:10.1093/bioinformatics/btq340. PMID 20616382 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics  ; Bioinformatics ; 2010 ; 26 ; 17 ; 2190-2191 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20616382">20616382</a> </li>
</ul>
<h3 id="mr-mega">MR-MEGA</h3>
<ul>
<li><strong>NAME</strong> : MR-MEGA </li>
<li><strong>SHORT NAME</strong> : MR-MEGA </li>
<li><strong>FULL NAME</strong> : Meta-Regression of Multi-AncEstry Genetic Association </li>
<li><strong>DESCRIPTION</strong> : MR-MEGA (Meta-Regression of Multi-AncEstry Genetic Association) is a tool to detect and fine-map complex trait association signals via multi-ancestry meta-regression.  This approach uses genome-wide metrics of diversity between populations to derive axes of genetic variation via multi-dimensional scaling [Purcell 2007].  Allelic effects of a variant across GWAS, weighted by their corresponding standard errors, can then be modelled in a linear regression framework, including the axes of genetic variation as covariates.  The flexibility of this model enables partitioning of the heterogeneity into components due to ancestry and residual variation, which would be expected to improve fine-mapping resolution. </li>
<li><strong>URL</strong> : <a href="https://genomics.ut.ee/en/tools">https://genomics.ut.ee/en/tools</a> </li>
<li><strong>KEYWORDS</strong> : cross-population, Meta-Regression </li>
<li><strong>TITLE</strong> : Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution </li>
<li><strong>DOI</strong> : 10.1093/hmg/ddx280 </li>
<li><strong>ABSTRACT</strong> : Trans-ethnic meta-analysis of genome-wide association studies (GWAS) across diverse populations can increase power to detect complex trait loci when the underlying causal variants are shared between ancestry groups. However, heterogeneity in allelic effects between GWAS at these loci can occur that is correlated with ancestry. Here, a novel approach is presented to detect SNP association and quantify the extent of heterogeneity in allelic effects that is correlated with ancestry. We employ trans-ethnic meta-regression to model allelic effects as a function of axes of genetic variation, derived from a matrix of mean pairwise allele frequency differences between GWAS, and implemented in the MR-MEGA software. Through detailed simulations, we demonstrate increased power to detect association for MR-MEGA over fixed- and random-effects meta-analysis across a range of scenarios of heterogeneity in allelic effects between ethnic groups. We also demonstrate improved fine-mapping resolution, in loci containing a single causal variant, compared to these meta-analysis approaches and PAINTOR, and equivalent performance to MANTRA at reduced computational cost. Application of MR-MEGA to trans-ethnic GWAS of kidney function in 71,461 individuals indicates stronger signals of association than fixed-effects meta-analysis when heterogeneity in allelic effects is correlated with ancestry. Application of MR-MEGA to fine-mapping four type 2 diabetes susceptibility loci in 22,086 cases and 42,539 controls highlights: (i) strong evidence for heterogeneity in allelic effects that is correlated with ancestry only at the index SNP for the association signal at the CDKAL1 locus; and (ii) 99% credible sets with six or fewer variants for five distinct association signals. </li>
<li><strong>CITATION</strong> : Mägi R, Horikoshi M, Sofer T, Mahajan A, ...&amp;, Morris AP. (2017) Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution Hum. Mol. Genet., 26 (18) 3639-3650. doi:10.1093/hmg/ddx280. PMID 28911207 </li>
<li><strong>JOURNAL_INFO</strong> : Human molecular genetics ; Hum. Mol. Genet. ; 2017 ; 26 ; 18 ; 3639-3650 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28911207">28911207</a> </li>
</ul>
<h2 id="multi-trait">Multi-trait</h2>
<h3 id="asset">ASSET</h3>
<ul>
<li><strong>NAME</strong> : ASSET </li>
<li><strong>SHORT NAME</strong> : ASSET </li>
<li><strong>FULL NAME</strong> : association analysis based on subsets </li>
<li><strong>URL</strong> : <a href="https://github.com/sbstatgen/ASSET">https://github.com/sbstatgen/ASSET</a> </li>
<li><strong>TITLE</strong> : A subset-based approach improves power and interpretation for the combined analysis of genetic association studies of heterogeneous traits </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2012.03.015 </li>
<li><strong>ABSTRACT</strong> : Pooling genome-wide association studies (GWASs) increases power but also poses methodological challenges because studies are often heterogeneous. For example, combining GWASs of related but distinct traits can provide promising directions for the discovery of loci with small but common pleiotropic effects. Classical approaches for meta-analysis or pooled analysis, however, might not be suitable for such analysis because individual variants are likely to be associated with only a subset of the traits or might demonstrate effects in different directions. We propose a method that exhaustively explores subsets of studies for the presence of true association signals that are in either the same direction or possibly opposite directions. An efficient approximation is used for rapid evaluation of p values. We present two illustrative applications, one for a meta-analysis of separate case-control studies of six distinct cancers and another for pooled analysis of a case-control study of glioma, a class of brain tumors that contains heterogeneous subtypes. Both the applications and additional simulation studies demonstrate that the proposed methods offer improved power and more interpretable results when compared to traditional methods for the analysis of heterogeneous traits. The proposed framework has applications beyond genetic association studies. </li>
<li><strong>COPYRIGHT</strong> : https://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Bhattacharjee S, Rajaraman P, Jacobs KB, Wheeler WA, ...&amp;, Chatterjee N. (2012) A subset-based approach improves power and interpretation for the combined analysis of genetic association studies of heterogeneous traits Am. J. Hum. Genet., 90 (5) 821-835. doi:10.1016/j.ajhg.2012.03.015. PMID 22560090 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2012 ; 90 ; 5 ; 821-835 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22560090">22560090</a> </li>
</ul>
<h3 id="factorgo">FactorGO</h3>
<ul>
<li><strong>NAME</strong> : FactorGO </li>
<li><strong>SHORT NAME</strong> : FactorGO </li>
<li><strong>FULL NAME</strong> : Factor analysis model in Genetic assOciation </li>
<li><strong>DESCRIPTION</strong> : FactorGo is a scalable variational factor analysis model that learns pleiotropic factors using GWAS summary statistics. </li>
<li><strong>URL</strong> : <a href="https://github.com/mancusolab/FactorGo">https://github.com/mancusolab/FactorGo</a> </li>
<li><strong>KEYWORDS</strong> : pleiotropy, factor analysis </li>
<li><strong>TITLE</strong> : A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2023.09.015 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWASs) across thousands of traits have revealed the pervasive pleiotropy of trait-associated genetic variants. While methods have been proposed to characterize pleiotropic components across groups of phenotypes, scaling these approaches to ultra-large-scale biobanks has been challenging. Here, we propose FactorGo, a scalable variational factor analysis model to identify and characterize pleiotropic components using biobank GWAS summary data. In extensive simulations, we observe that FactorGo outperforms the state-of-the-art (model-free) approach tSVD in capturing latent pleiotropic factors across phenotypes while maintaining a similar computational cost. We apply FactorGo to estimate 100 latent pleiotropic factors from GWAS summary data of 2,483 phenotypes measured in European-ancestry Pan-UK BioBank individuals (N = 420,531). Next, we find that factors from FactorGo are more enriched with relevant tissue-specific annotations than those identified by tSVD (p = 2.58E-10) and validate our approach by recapitulating brain-specific enrichment for BMI and the height-related connection between reproductive system and muscular-skeletal growth. Finally, our analyses suggest shared etiologies between rheumatoid arthritis and periodontal condition in addition to alkaline phosphatase as a candidate prognostic biomarker for prostate cancer. Overall, FactorGo improves our biological understanding of shared etiologies across thousands of GWASs. </li>
<li><strong>CITATION</strong> : Zhang Z, Jung J, Kim A, Suboc N, ...&amp;, Mancuso N. (2023) A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics Am. J. Hum. Genet.,  () . doi:10.1016/j.ajhg.2023.09.015. PMID 37879338 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2023 ; <NA> ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37879338">37879338</a> </li>
</ul>
<h3 id="galesloot">Galesloot</h3>
<ul>
<li><strong>NAME</strong> : Galesloot </li>
<li><strong>TITLE</strong> : A comparison of multivariate genome-wide association methods </li>
<li><strong>DOI</strong> : 10.1371/journal.pone.0095923 </li>
<li><strong>ABSTRACT</strong> : Joint association analysis of multiple traits in a genome-wide association study (GWAS), i.e. a multivariate GWAS, offers several advantages over analyzing each trait in a separate GWAS. In this study we directly compared a number of multivariate GWAS methods using simulated data. We focused on six methods that are implemented in the software packages PLINK, SNPTEST, MultiPhen, BIMBAM, PCHAT and TATES, and also compared them to standard univariate GWAS, analysis of the first principal component of the traits, and meta-analysis of univariate results. We simulated data (N = 1000) for three quantitative traits and one bi-allelic quantitative trait locus (QTL), and varied the number of traits associated with the QTL (explained variance 0.1%), minor allele frequency of the QTL, residual correlation between the traits, and the sign of the correlation induced by the QTL relative to the residual correlation. We compared the power of the methods using empirically fixed significance thresholds (α = 0.05). Our results showed that the multivariate methods implemented in PLINK, SNPTEST, MultiPhen and BIMBAM performed best for the majority of the tested scenarios, with a notable increase in power for scenarios with an opposite sign of genetic and residual correlation. All multivariate analyses resulted in a higher power than univariate analyses, even when only one of the traits was associated with the QTL. Hence, use of multivariate GWAS methods can be recommended, even when genetic correlations between traits are weak. </li>
<li><strong>CITATION</strong> : Galesloot TE, van Steen K, Kiemeney LA, Janss LL, ...&amp;, Vermeulen SH. (2014) A comparison of multivariate genome-wide association methods PLoS One, 9 (4) e95923. doi:10.1371/journal.pone.0095923. PMID 24763738 </li>
<li><strong>JOURNAL_INFO</strong> : PloS one ; PLoS One ; 2014 ; 9 ; 4 ; e95923 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24763738">24763738</a> </li>
</ul>
<h3 id="genomic-sem">Genomic-SEM</h3>
<ul>
<li><strong>NAME</strong> : Genomic-SEM </li>
<li><strong>SHORT NAME</strong> : Genomic-SEM </li>
<li><strong>FULL NAME</strong> : genomic structural equation modelling </li>
<li><strong>DESCRIPTION</strong> : R-package which allows the user to fit structural equation models based on the summary statistics obtained from genome wide association studies (GWAS). </li>
<li><strong>URL</strong> : <a href="https://github.com/GenomicSEM/GenomicSEM">https://github.com/GenomicSEM/GenomicSEM</a> </li>
<li><strong>KEYWORDS</strong> : SEM </li>
<li><strong>TITLE</strong> : Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits </li>
<li><strong>DOI</strong> : 10.1038/s41562-019-0566-x </li>
<li><strong>ABSTRACT</strong> : Genetic correlations estimated from genome-wide association studies (GWASs) reveal pervasive pleiotropy across a wide variety of phenotypes. We introduce genomic structural equation modelling (genomic SEM): a multivariate method for analysing the joint genetic architecture of complex traits. Genomic SEM synthesizes genetic correlations and single-nucleotide polymorphism heritabilities inferred from GWAS summary statistics of individual traits from samples with varying and unknown degrees of overlap. Genomic SEM can be used to model multivariate genetic associations among phenotypes, identify variants with effects on general dimensions of cross-trait liability, calculate more predictive polygenic scores and identify loci that cause divergence between traits. We demonstrate several applications of genomic SEM, including a joint analysis of summary statistics from five psychiatric traits. We identify 27 independent single-nucleotide polymorphisms not previously identified in the contributing univariate GWASs. Polygenic scores from genomic SEM consistently outperform those from univariate GWASs. Genomic SEM is flexible and open ended, and allows for continuous innovation in multivariate genetic analysis. </li>
<li><strong>CITATION</strong> : Grotzinger AD, Rhemtulla M, de Vlaming R, Ritchie SJ, ...&amp;, Tucker-Drob EM. (2019) Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits Nat Hum Behav, 3 (5) 513-525. doi:10.1038/s41562-019-0566-x. PMID 30962613 </li>
<li><strong>JOURNAL_INFO</strong> : Nature human behaviour ; Nat Hum Behav ; 2019 ; 3 ; 5 ; 513-525 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30962613">30962613</a> </li>
</ul>
<h3 id="hipo">HIPO</h3>
<ul>
<li><strong>NAME</strong> : HIPO </li>
<li><strong>SHORT NAME</strong> : HIPO </li>
<li><strong>FULL NAME</strong> : heritability informed power optimization </li>
<li><strong>DESCRIPTION</strong> : hipo is an R package that performs heritability informed power optimization (HIPO) for conducting multi-trait association analysis on summary level data. </li>
<li><strong>URL</strong> : <a href="https://github.com/gqi/hipo">https://github.com/gqi/hipo</a> </li>
<li><strong>TITLE</strong> : Heritability informed power optimization (HIPO) leads to enhanced detection of genetic associations across multiple traits </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1007549 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies have shown that pleiotropy is a common phenomenon that can potentially be exploited for enhanced detection of susceptibility loci. We propose heritability informed power optimization (HIPO) for conducting powerful pleiotropic analysis using summary-level association statistics. We find optimal linear combinations of association coefficients across traits that are expected to maximize non-centrality parameter for the underlying test statistics, taking into account estimates of heritability, sample size variations and overlaps across the traits. Simulation studies show that the proposed method has correct type I error, robust to population stratification and leads to desired genome-wide enrichment of association signals. Application of the proposed method to publicly available data for three groups of genetically related traits, lipids (N = 188,577), psychiatric diseases (Ncase = 33,332, Ncontrol = 27,888) and social science traits (N ranging between 161,460 to 298,420 across individual traits) increased the number of genome-wide significant loci by 12%, 200% and 50%, respectively, compared to those found by analysis of individual traits. Evidence of replication is present for many of these loci in subsequent larger studies for individual traits. HIPO can potentially be extended to high-dimensional phenotypes as a way of dimension reduction to maximize power for subsequent genetic association testing. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Qi G, Chatterjee N. (2018) Heritability informed power optimization (HIPO) leads to enhanced detection of genetic associations across multiple traits PLoS Genet., 14 (10) e1007549. doi:10.1371/journal.pgen.1007549. PMID 30289880 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2018 ; 14 ; 10 ; e1007549 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30289880">30289880</a> </li>
</ul>
<h3 id="jass">JASS</h3>
<ul>
<li><strong>NAME</strong> : JASS </li>
<li><strong>SHORT NAME</strong> : JASS </li>
<li><strong>FULL NAME</strong> : Joint Analysis of Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : JASS is a python package that handles the computation of the joint statistics over sets of selected GWAS results, and the interactive exploration of the results through a web interface. The generation of joint statistics over a set of selected studies, and the generation of static plots to display the results, is easily performed using the command line interface. These functionalities can also be accessed through a web application embedded in the python package, which also enables the exploration of the results through a dynamic Javascript interface. The JASS analysis module handles the data processing, going from the import of the data up to the computation of the joint statistics and the generation of the various static plots to illustrate the results. However, we also briefly describe in the next section the pre-processing of raw GWAS data which can be performed through a companion script provided on behalf of the JASS package. </li>
<li><strong>URL</strong> : <a href="https://gitlab.pasteur.fr/statistical-genetics/jass">https://gitlab.pasteur.fr/statistical-genetics/jass</a> </li>
<li><strong>TITLE</strong> : JASS: command line and web interface for the joint analysis of GWAS results </li>
<li><strong>DOI</strong> : 10.1093/nargab/lqaa003 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association study (GWAS) has been the driving force for identifying association between genetic variants and human phenotypes. Thousands of GWAS summary statistics covering a broad range of human traits and diseases are now publicly available. These GWAS have proven their utility for a range of secondary analyses, including in particular the joint analysis of multiple phenotypes to identify new associated genetic variants. However, although several methods have been proposed, there are very few large-scale applications published so far because of challenges in implementing these methods on real data. Here, we present JASS (Joint Analysis of Summary Statistics), a polyvalent Python package that addresses this need. Our package incorporates recently developed joint tests such as the omnibus approach and various weighted sum of Z-score tests while solving all practical and computational barriers for large-scale multivariate analysis of GWAS summary statistics. This includes data cleaning and harmonization tools, an efficient algorithm for fast derivation of joint statistics, an optimized data management process and a web interface for exploration purposes. Both benchmark analyses and real data applications demonstrated the robustness and strong potential of JASS for the detection of new associated genetic variants. Our package is freely available at https://gitlab.pasteur.fr/statistical-genetics/jass. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Julienne H, Lechat P, Guillemot V, Lasry C, ...&amp;, Aschard H. (2020) JASS: command line and web interface for the joint analysis of GWAS results NAR Genom. Bioinform., 2 (1) lqaa003. doi:10.1093/nargab/lqaa003. PMID 32002517 </li>
<li><strong>JOURNAL_INFO</strong> : NAR genomics and bioinformatics ; NAR Genom. Bioinform. ; 2020 ; 2 ; 1 ; lqaa003 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32002517">32002517</a> </li>
</ul>
<h3 id="lcp-gwas">LCP-GWAS</h3>
<ul>
<li><strong>NAME</strong> : LCP-GWAS </li>
<li><strong>SHORT NAME</strong> : LCP-GWAS </li>
<li><strong>FULL NAME</strong> : Linear Combination Phenotype GWAS </li>
<li><strong>KEYWORDS</strong> : multivariate GWAS follow-up analyses </li>
<li><strong>TITLE</strong> : An expanded analysis framework for multivariate GWAS connects inflammatory biomarkers to functional variants and disease </li>
<li><strong>DOI</strong> : 10.1038/s41431-020-00730-8 </li>
<li><strong>ABSTRACT</strong> : Multivariate methods are known to increase the statistical power to detect associations in the case of shared genetic basis between phenotypes. They have, however, lacked essential analytic tools to follow-up and understand the biology underlying these associations. We developed a novel computational workflow for multivariate GWAS follow-up analyses, including fine-mapping and identification of the subset of traits driving associations (driver traits). Many follow-up tools require univariate regression coefficients which are lacking from multivariate results. Our method overcomes this problem by using Canonical Correlation Analysis to turn each multivariate association into its optimal univariate Linear Combination Phenotype (LCP). This enables an LCP-GWAS, which in turn generates the statistics required for follow-up analyses. We implemented our method on 12 highly correlated inflammatory biomarkers in a Finnish population-based study. Altogether, we identified 11 associations, four of which (F5, ABO, C1orf140 and PDGFRB) were not detected by biomarker-specific analyses. Fine-mapping identified 19 signals within the 11 loci and driver trait analysis determined the traits contributing to the associations. A phenome-wide association study on the 19 representative variants from the signals in 176,899 individuals from the FinnGen study revealed 53 disease associations (p &lt; 1 × 10-4). Several reported pQTLs in the 11 loci provided orthogonal evidence for the biologically relevant functions of the representative variants. Our novel multivariate analysis workflow provides a powerful addition to standard univariate GWAS analyses by enabling multivariate GWAS follow-up and thus promoting the advancement of powerful multivariate methods in genomics. </li>
<li><strong>CITATION</strong> : Ruotsalainen SE, Partanen JJ, Cichonska A, Lin J, ...&amp;, Koskela J. (2021) An expanded analysis framework for multivariate GWAS connects inflammatory biomarkers to functional variants and disease Eur. J. Hum. Genet., 29 (2) 309-324. doi:10.1038/s41431-020-00730-8. PMID 33110245 </li>
<li><strong>JOURNAL_INFO</strong> : European journal of human genetics: EJHG ; Eur. J. Hum. Genet. ; 2021 ; 29 ; 2 ; 309-324 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33110245">33110245</a> </li>
</ul>
<h3 id="manova">MANOVA</h3>
<ul>
<li><strong>NAME</strong> : MANOVA </li>
<li><strong>SHORT NAME</strong> : MANOVA </li>
<li><strong>FULL NAME</strong> : multivariate analysis of variance </li>
<li><strong>CITATION</strong> : Pillai, K. C. S. Some new test criteria in multivariate analysis. Ann. Math. Stat. 26, 117–121 (1955). </li>
</ul>
<h3 id="mostest">MOSTest</h3>
<ul>
<li><strong>NAME</strong> : MOSTest </li>
<li><strong>SHORT NAME</strong> : MOSTest </li>
<li><strong>FULL NAME</strong> : Multivariate Omnibus Statistical Test </li>
<li><strong>DESCRIPTION</strong> : MOSTest is a tool for join genetical analysis of multiple traits, using multivariate analysis to boost the power of discovering associated loci. </li>
<li><strong>URL</strong> : <a href="https://github.com/precimed/mostest">https://github.com/precimed/mostest</a> </li>
<li><strong>TITLE</strong> : Understanding the genetic determinants of the brain with MOSTest </li>
<li><strong>DOI</strong> : 10.1038/s41467-020-17368-1 </li>
<li><strong>ABSTRACT</strong> : Regional brain morphology has a complex genetic architecture, consisting of many common polymorphisms with small individual effects. This has proven challenging for genome-wide association studies (GWAS). Due to the distributed nature of genetic signal across brain regions, multivariate analysis of regional measures may enhance discovery of genetic variants. Current multivariate approaches to GWAS are ill-suited for complex, large-scale data of this kind. Here, we introduce the Multivariate Omnibus Statistical Test (MOSTest), with an efficient computational design enabling rapid and reliable inference, and apply it to 171 regional brain morphology measures from 26,502 UK Biobank participants. At the conventional genome-wide significance threshold of α = 5 × 10-8, MOSTest identifies 347 genomic loci associated with regional brain morphology, more than any previous study, improving upon the discovery of established GWAS approaches more than threefold. Our findings implicate more than 5% of all protein-coding genes and provide evidence for gene sets involved in neuron development and differentiation. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : van der Meer D, Frei O, Kaufmann T, Shadrin AA, ...&amp;, Dale AM. (2020) Understanding the genetic determinants of the brain with MOSTest Nat. Commun., 11 (1) 3512. doi:10.1038/s41467-020-17368-1. PMID 32665545 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2020 ; 11 ; 1 ; 3512 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32665545">32665545</a> </li>
</ul>
<h3 id="mtag">MTAG</h3>
<ul>
<li><strong>NAME</strong> : MTAG </li>
<li><strong>SHORT NAME</strong> : MTAG </li>
<li><strong>FULL NAME</strong> : Multi-Trait Analysis of GWAS </li>
<li><strong>DESCRIPTION</strong> : mtag is a Python-based command line tool for jointly analyzing multiple sets of GWAS summary statistics as described by Turley et. al. (2018). It can also be used as a tool to meta-analyze GWAS results. </li>
<li><strong>URL</strong> : <a href="https://github.com/JonJala/mtag">https://github.com/JonJala/mtag</a> </li>
<li><strong>KEYWORDS</strong> : Multi-trait </li>
<li><strong>TITLE</strong> : Multi-trait analysis of genome-wide association summary statistics using MTAG </li>
<li><strong>DOI</strong> : 10.1038/s41588-017-0009-4 </li>
<li><strong>ABSTRACT</strong> : We introduce multi-trait analysis of GWAS (MTAG), a method for joint analysis of summary statistics from genome-wide association studies (GWAS) of different traits, possibly from overlapping samples. We apply MTAG to summary statistics for depressive symptoms (N eff = 354,862), neuroticism (N = 168,105), and subjective well-being (N = 388,538). As compared to the 32, 9, and 13 genome-wide significant loci identified in the single-trait GWAS (most of which are themselves novel), MTAG increases the number of associated loci to 64, 37, and 49, respectively. Moreover, association statistics from MTAG yield more informative bioinformatics analyses and increase the variance explained by polygenic scores by approximately 25%, matching theoretical expectations. </li>
<li><strong>CITATION</strong> : Turley P, Walters RK, Maghzian O, Okbay A, ...&amp;, Pitts SJ. (2018) Multi-trait analysis of genome-wide association summary statistics using MTAG Nat. Genet., 50 (2) 229-237. doi:10.1038/s41588-017-0009-4. PMID 29292387 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2018 ; 50 ; 2 ; 229-237 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29292387">29292387</a> </li>
</ul>
<h3 id="mv-plink-mqfam">MV-PLINK (MQFAM)</h3>
<ul>
<li><strong>NAME</strong> : MV-PLINK (MQFAM) </li>
<li><strong>SHORT NAME</strong> : MV-PLINK (MQFAM) </li>
<li><strong>TITLE</strong> : A multivariate test of association </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btn563 </li>
<li><strong>ABSTRACT</strong> : UNLABELLED: Although genetic association studies often test multiple, related phenotypes, few formal multivariate tests of association are available. We describe a test of association that can be efficiently applied to large population-based designs. AVAILABILITY: A C++ implementation can be obtained from the authors. </li>
<li><strong>CITATION</strong> : Ferreira MA, Purcell SM. (2009) A multivariate test of association Bioinformatics, 25 (1) 132-133. doi:10.1093/bioinformatics/btn563. PMID 19019849 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2009 ; 25 ; 1 ; 132-133 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19019849">19019849</a> </li>
</ul>
<h3 id="multiphen">MultiPhen</h3>
<ul>
<li><strong>NAME</strong> : MultiPhen </li>
<li><strong>SHORT NAME</strong> : MultiPhen </li>
<li><strong>DESCRIPTION</strong> : Performs genetic association tests between SNPs (one-at-a-time) and multiple phenotypes (separately or in joint model). </li>
<li><strong>URL</strong> : <a href="https://cran.r-project.org/web/packages/MultiPhen/index.html">https://cran.r-project.org/web/packages/MultiPhen/index.html</a> </li>
<li><strong>TITLE</strong> : MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS </li>
<li><strong>DOI</strong> : 10.1371/journal.pone.0034861 </li>
<li><strong>ABSTRACT</strong> : The genome-wide association study (GWAS) approach has discovered hundreds of genetic variants associated with diseases and quantitative traits. However, despite clinical overlap and statistical correlation between many phenotypes, GWAS are generally performed one-phenotype-at-a-time. Here we compare the performance of modelling multiple phenotypes jointly with that of the standard univariate approach. We introduce a new method and software, MultiPhen, that models multiple phenotypes simultaneously in a fast and interpretable way. By performing ordinal regression, MultiPhen tests the linear combination of phenotypes most associated with the genotypes at each SNP, and thus potentially captures effects hidden to single phenotype GWAS. We demonstrate via simulation that this approach provides a dramatic increase in power in many scenarios. There is a boost in power for variants that affect multiple phenotypes and for those that affect only one phenotype. While other multivariate methods have similar power gains, we describe several benefits of MultiPhen over these. In particular, we demonstrate that other multivariate methods that assume the genotypes are normally distributed, such as canonical correlation analysis (CCA) and MANOVA, can have highly inflated type-1 error rates when testing case-control or non-normal continuous phenotypes, while MultiPhen produces no such inflation. To test the performance of MultiPhen on real data we applied it to lipid traits in the Northern Finland Birth Cohort 1966 (NFBC1966). In these data MultiPhen discovers 21% more independent SNPs with known associations than the standard univariate GWAS approach, while applying MultiPhen in addition to the standard approach provides 37% increased discovery. The most associated linear combinations of the lipids estimated by MultiPhen at the leading SNPs accurately reflect the Friedewald Formula, suggesting that MultiPhen could be used to refine the definition of existing phenotypes or uncover novel heritable phenotypes. </li>
<li><strong>CITATION</strong> : O'Reilly PF, Hoggart CJ, Pomyen Y, Calboli FC, ...&amp;, Coin LJ. (2012) MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS PLoS One, 7 (5) e34861. doi:10.1371/journal.pone.0034861. PMID 22567092 </li>
<li><strong>JOURNAL_INFO</strong> : PloS one ; PLoS One ; 2012 ; 7 ; 5 ; e34861 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22567092">22567092</a> </li>
</ul>
<h3 id="pchat">PCHAT</h3>
<ul>
<li><strong>NAME</strong> : PCHAT </li>
<li><strong>SHORT NAME</strong> : PCHAT </li>
<li><strong>FULL NAME</strong> : principal component of heritability association test </li>
<li><strong>TITLE</strong> : Pleiotropy and principal components of heritability combine to increase power for association analysis </li>
<li><strong>DOI</strong> : 10.1002/gepi.20257 </li>
<li><strong>ABSTRACT</strong> : When many correlated traits are measured the potential exists to discover the coordinated control of these traits via genotyped polymorphisms. A common statistical approach to this problem involves assessing the relationship between each phenotype and each single nucleotide polymorphism (SNP) individually (PHN); and taking a Bonferroni correction for the effective number of independent tests conducted. Alternatively, one can apply a dimension reduction technique, such as estimation of principal components, and test for an association with the principal components of the phenotypes (PCP) rather than the individual phenotypes. Building on the work of Lange and colleagues we develop an alternative method based on the principal component of heritability (PCH). For each SNP the PCH approach reduces the phenotypes to a single trait that has a higher heritability than any other linear combination of the phenotypes. As a result, the association between a SNP and derived trait is often easier to detect than an association with any of the individual phenotypes or the PCP. When applied to unrelated subjects, PCH has a drawback. For each SNP it is necessary to estimate the vector of loadings that maximize the heritability over all phenotypes. We develop a method of iterated sample splitting that uses one portion of the data for training and the remainder for testing. This cross-validation approach maintains the type I error control and yet utilizes the data efficiently, resulting in a powerful test for association. </li>
<li><strong>COPYRIGHT</strong> : http://onlinelibrary.wiley.com/termsAndConditions#vor </li>
<li><strong>CITATION</strong> : Klei L, Luca D, Devlin B, Roeder K. (2008) Pleiotropy and principal components of heritability combine to increase power for association analysis Genet. Epidemiol., 32 (1) 9-19. doi:10.1002/gepi.20257. PMID 17922480 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet. Epidemiol. ; 2008 ; 32 ; 1 ; 9-19 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17922480">17922480</a> </li>
</ul>
<h3 id="porter">Porter</h3>
<ul>
<li><strong>NAME</strong> : Porter </li>
<li><strong>TITLE</strong> : Multivariate simulation framework reveals performance of multi-trait GWAS methods </li>
<li><strong>DOI</strong> : 10.1038/srep38837 </li>
<li><strong>ABSTRACT</strong> : Burgeoning availability of genome-wide association study (GWAS) results and national biobank data has led to growing interest in performing multi-trait genetic analyses. Numerous multi-trait GWAS methods that exploit either summary statistics or individual-level data have been developed, but their relative performance is unclear. Here we develop a simulation framework to model the complex networks underlying multivariate genetic epidemiology, enabling the vast model space of genetic effects on multiple correlated traits to be explored systematically. We perform a comprehensive comparison of the leading multi-trait GWAS methods, finding: (1) method performance is highly sensitive to the specific combination of genetic effects and phenotypic correlations, (2) most of the current multivariate methods have remarkably similar statistical power, and (3) multivariate methods may offer a substantial increase in the discovery of genetic variants over the standard univariate approach. We believe our findings offer the clearest picture to date of the relative performance of multi-trait GWAS methods and act as a guide for method selection. We provide a web application and open-source software program implementing our simulation framework, for: (i) further benchmarking of multivariate GWAS methods, (ii) power calculations for multivariate genetic studies, and (iii) generating data for testing any multivariate method in genetic epidemiology. </li>
<li><strong>CITATION</strong> : Porter HF, O'Reilly PF. (2017) Multivariate simulation framework reveals performance of multi-trait GWAS methods Sci. Rep., 7 (1) 38837. doi:10.1038/srep38837. PMID 28287610 </li>
<li><strong>JOURNAL_INFO</strong> : Scientific reports ; Sci. Rep. ; 2017 ; 7 ; 1 ; 38837 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28287610">28287610</a> </li>
</ul>
<h3 id="salinas">Salinas</h3>
<ul>
<li><strong>NAME</strong> : Salinas </li>
<li><strong>TITLE</strong> : Statistical analysis of multiple phenotypes in genetic epidemiologic studies: From cross-phenotype associations to pleiotropy </li>
<li><strong>DOI</strong> : 10.1093/aje/kwx296 </li>
<li><strong>ABSTRACT</strong> : In the context of genetics, pleiotropy refers to the phenomenon in which a single genetic locus affects more than 1 trait or disease. Genetic epidemiologic studies have identified loci associated with multiple phenotypes, and these cross-phenotype associations are often incorrectly interpreted as examples of pleiotropy. Pleiotropy is only one possible explanation for cross-phenotype associations. Cross-phenotype associations may also arise due to issues related to study design, confounder bias, or nongenetic causal links between the phenotypes under analysis. Therefore, it is necessary to dissect cross-phenotype associations carefully to uncover true pleiotropic loci. In this review, we describe statistical methods that can be used to identify robust statistical evidence of pleiotropy. First, we provide an overview of univariate and multivariate methods for discovery of cross-phenotype associations and highlight important considerations for choosing among available methods. Then, we describe how to dissect cross-phenotype associations by using mediation analysis. Pleiotropic loci provide insights into the mechanistic underpinnings of disease comorbidity, and they may serve as novel targets for interventions that simultaneously treat multiple diseases. Discerning between different types of cross-phenotype associations is necessary to realize the public health potential of pleiotropic loci. </li>
<li><strong>CITATION</strong> : Salinas YD, Wang Z, DeWan AT. (2018) Statistical analysis of multiple phenotypes in genetic epidemiologic studies: From cross-phenotype associations to pleiotropy Am. J. Epidemiol., 187 (4) 855-863. doi:10.1093/aje/kwx296. PMID 29020254 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of epidemiology ; Am. J. Epidemiol. ; 2018 ; 187 ; 4 ; 855-863 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29020254">29020254</a> </li>
</ul>
<h3 id="stephens">Stephens</h3>
<ul>
<li><strong>NAME</strong> : Stephens </li>
<li><strong>TITLE</strong> : A unified framework for association analysis with multiple related phenotypes </li>
<li><strong>DOI</strong> : 10.1371/journal.pone.0065245 </li>
<li><strong>ABSTRACT</strong> : We consider the problem of assessing associations between multiple related outcome variables, and a single explanatory variable of interest. This problem arises in many settings, including genetic association studies, where the explanatory variable is genotype at a genetic variant. We outline a framework for conducting this type of analysis, based on Bayesian model comparison and model averaging for multivariate regressions. This framework unifies several common approaches to this problem, and includes both standard univariate and standard multivariate association tests as special cases. The framework also unifies the problems of testing for associations and explaining associations - that is, identifying which outcome variables are associated with genotype. This provides an alternative to the usual, but conceptually unsatisfying, approach of resorting to univariate tests when explaining and interpreting significant multivariate findings. The method is computationally tractable genome-wide for modest numbers of phenotypes (e.g. 5-10), and can be applied to summary data, without access to raw genotype and phenotype data. We illustrate the methods on both simulated examples, and to a genome-wide association study of blood lipid traits where we identify 18 potential novel genetic associations that were not identified by univariate analyses of the same data. </li>
<li><strong>CITATION</strong> : Stephens M. (2013) A unified framework for association analysis with multiple related phenotypes PLoS One, 8 (7) e65245. doi:10.1371/journal.pone.0065245. PMID 23861737 </li>
<li><strong>JOURNAL_INFO</strong> : PloS one ; PLoS One ; 2013 ; 8 ; 7 ; e65245 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23861737">23861737</a> </li>
</ul>
<h3 id="tates">TATES</h3>
<ul>
<li><strong>NAME</strong> : TATES </li>
<li><strong>SHORT NAME</strong> : TATES </li>
<li><strong>FULL NAME</strong> : Trait-based Association Test that uses Extended Simes procedure </li>
<li><strong>TITLE</strong> : TATES: efficient multivariate genotype-phenotype analysis for genome-wide association studies </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1003235 </li>
<li><strong>ABSTRACT</strong> : To date, the genome-wide association study (GWAS) is the primary tool to identify genetic variants that cause phenotypic variation. As GWAS analyses are generally univariate in nature, multivariate phenotypic information is usually reduced to a single composite score. This practice often results in loss of statistical power to detect causal variants. Multivariate genotype-phenotype methods do exist but attain maximal power only in special circumstances. Here, we present a new multivariate method that we refer to as TATES (Trait-based Association Test that uses Extended Simes procedure), inspired by the GATES procedure proposed by Li et al (2011). For each component of a multivariate trait, TATES combines p-values obtained in standard univariate GWAS to acquire one trait-based p-value, while correcting for correlations between components. Extensive simulations, probing a wide variety of genotype-phenotype models, show that TATES's false positive rate is correct, and that TATES's statistical power to detect causal variants explaining 0.5% of the variance can be 2.5-9 times higher than the power of univariate tests based on composite scores and 1.5-2 times higher than the power of the standard MANOVA. Unlike other multivariate methods, TATES detects both genetic variants that are common to multiple phenotypes and genetic variants that are specific to a single phenotype, i.e. TATES provides a more complete view of the genetic architecture of complex traits. As the actual causal genotype-phenotype model is usually unknown and probably phenotypically and genetically complex, TATES, available as an open source program, constitutes a powerful new multivariate strategy that allows researchers to identify novel causal variants, while the complexity of traits is no longer a limiting factor. </li>
<li><strong>CITATION</strong> : van der Sluis S, Posthuma D, Dolan CV. (2013) TATES: efficient multivariate genotype-phenotype analysis for genome-wide association studies PLoS Genet., 9 (1) e1003235. doi:10.1371/journal.pgen.1003235. PMID 23359524 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2013 ; 9 ; 1 ; e1003235 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23359524">23359524</a> </li>
</ul>
<h3 id="yang">Yang</h3>
<ul>
<li><strong>NAME</strong> : Yang </li>
<li><strong>TITLE</strong> : Methods for analyzing multivariate phenotypes in genetic association studies </li>
<li><strong>DOI</strong> : 10.1155/2012/652569 </li>
<li><strong>ABSTRACT</strong> : This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. Multivariate phenotypes are frequently encountered in genetic association studies. The purpose of analyzing multivariate phenotypes usually includes discovery of novel genetic variants of pleiotropy effects, that is, affecting multiple phenotypes, and the ultimate goal of uncovering the underlying genetic mechanism. In recent years, there have been new method development and application of existing statistical methods to such phenotypes. In this paper, we provide a review of the available methods for analyzing association between a single marker and a multivariate phenotype consisting of the same type of components (e.g., all continuous or all categorical) or different types of components (e.g., some are continuous and others are categorical). We also reviewed causal inference methods designed to test whether the detected association with the multivariate phenotype is truly pleiotropy or the genetic marker exerts its effects on some phenotypes through affecting the others. </li>
<li><strong>CITATION</strong> : Yang Q, Wang Y. (2012) Methods for analyzing multivariate phenotypes in genetic association studies J. Probab. Stat., 2012 () 652569. doi:10.1155/2012/652569. PMID 24748889 </li>
<li><strong>JOURNAL_INFO</strong> : Journal of probability and statistics ; J. Probab. Stat. ; 2012 ; 2012 ; <NA> ; 652569 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24748889">24748889</a> </li>
</ul>
<h3 id="amat">aMAT</h3>
<ul>
<li><strong>NAME</strong> : aMAT </li>
<li><strong>SHORT NAME</strong> : aMAT </li>
<li><strong>FULL NAME</strong> : adaptive multi-trait association test </li>
<li><strong>TITLE</strong> : Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank </li>
<li><strong>DOI</strong> : 10.1534/genetics.120.303242 </li>
<li><strong>ABSTRACT</strong> : Many genetic variants identified in genome-wide association studies (GWAS) are associated with multiple, sometimes seemingly unrelated, traits. This motivates multi-trait association analyses, which have successfully identified novel associated loci for many complex diseases. While appealing, most existing methods focus on analyzing a relatively small number of traits, and may yield inflated Type 1 error rates when a large number of traits need to be analyzed jointly. As deep phenotyping data are becoming rapidly available, we develop a novel method, referred to as aMAT (adaptive multi-trait association test), for multi-trait analysis of any number of traits. We applied aMAT to GWAS summary statistics for a set of 58 volumetric imaging derived phenotypes from the UK Biobank. aMAT had a genomic inflation factor of 1.04, indicating the Type 1 error rate was well controlled. More important, aMAT identified 24 distinct risk loci, 13 of which were ignored by standard GWAS. In comparison, the competing methods either had a suspicious genomic inflation factor or identified much fewer risk loci. Finally, four additional sets of traits have been analyzed and provided similar conclusions. </li>
<li><strong>COPYRIGHT</strong> : https://academic.oup.com/journals/pages/open_access/funder_policies/chorus/standard_publication_model </li>
<li><strong>CITATION</strong> : Wu C. (2020) Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank Genetics, 215 (4) 947-958. doi:10.1534/genetics.120.303242. PMID 32540950 </li>
<li><strong>JOURNAL_INFO</strong> : Genetics ; Genetics ; 2020 ; 215 ; 4 ; 947-958 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32540950">32540950</a> </li>
</ul>
<h3 id="fastasset">fastASSET</h3>
<ul>
<li><strong>NAME</strong> : fastASSET </li>
<li><strong>SHORT NAME</strong> : fastASSET </li>
<li><strong>URL</strong> : <a href="https://github.com/gqi/fastASSET">https://github.com/gqi/fastASSET</a> </li>
<li><strong>TITLE</strong> : Genome-wide large-scale multi-trait analysis characterizes global patterns of pleiotropy and unique trait-specific variants </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-51075-5 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) have found widespread evidence of pleiotropy, but characterization of global patterns of pleiotropy remain highly incomplete due to insufficient power of current approaches. We develop fastASSET, a method that allows efficient detection of variant-level pleiotropic association across many traits. We analyze GWAS summary statistics of 116 complex traits of diverse types collected from the GRASP repository and large GWAS Consortia. We identify 2293 independent loci and find that the lead variants in nearly all these loci (~99%) to be associated with ≥ 2 traits (median = 6). We observe that degree of pleiotropy estimated from our study predicts that observed in the UK Biobank for a much larger number of traits (K = 4114) (correlation = 0.43, p-value &lt; 2.2 × 10 - 16 ). Follow-up analyzes of 21 trait-specific variants indicate their link to the expression in trait-related tissues for a small number of genes involved in relevant biological processes. Our findings provide deeper insight into the nature of pleiotropy and leads to identification of highly trait-specific susceptibility variants. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by-nc-nd/4.0 </li>
<li><strong>CITATION</strong> : Qi G, Chhetri SB, Ray D, Dutta D, ...&amp;, Chatterjee N. (2024) Genome-wide large-scale multi-trait analysis characterizes global patterns of pleiotropy and unique trait-specific variants Nat. Commun., 15 (1) 6985. doi:10.1038/s41467-024-51075-5. PMID 39143063 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2024 ; 15 ; 1 ; 6985 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39143063">39143063</a> </li>
</ul>
<h3 id="metacca">metaCCA</h3>
<ul>
<li><strong>NAME</strong> : metaCCA </li>
<li><strong>SHORT NAME</strong> : metaCCA </li>
<li><strong>FULL NAME</strong> : meta canonical
correlation analysis </li>
<li><strong>DESCRIPTION</strong> : metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness. </li>
<li><strong>URL</strong> : <a href="https://github.com/aalto-ics-kepaco/metaCCA-matlab">https://github.com/aalto-ics-kepaco/metaCCA-matlab</a> </li>
<li><strong>TITLE</strong> : metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btw052 </li>
<li><strong>ABSTRACT</strong> : MOTIVATION: A dominant approach to genetic association studies is to perform univariate tests between genotype-phenotype pairs. However, analyzing related traits together increases statistical power, and certain complex associations become detectable only when several variants are tested jointly. Currently, modest sample sizes of individual cohorts, and restricted availability of individual-level genotype-phenotype data across the cohorts limit conducting multivariate tests. RESULTS: We introduce metaCCA, a computational framework for summary statistics-based analysis of a single or multiple studies that allows multivariate representation of both genotype and phenotype. It extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness.Multivariate meta-analysis of two Finnish studies of nuclear magnetic resonance metabolomics by metaCCA, using standard univariate output from the program SNPTEST, shows an excellent agreement with the pooled individual-level analysis of original data. Motivated by strong multivariate signals in the lipid genes tested, we envision that multivariate association testing using metaCCA has a great potential to provide novel insights from already published summary statistics from high-throughput phenotyping technologies. AVAILABILITY AND IMPLEMENTATION: Code is available at https://github.com/aalto-ics-kepaco CONTACTS: anna.cichonska@helsinki.fi or matti.pirinen@helsinki.fi SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>CITATION</strong> : Cichonska A, Rousu J, Marttinen P, Kangas AJ, ...&amp;, Pirinen M. (2016) metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis Bioinformatics, 32 (13) 1981-1989. doi:10.1093/bioinformatics/btw052. PMID 27153689 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2016 ; 32 ; 13 ; 1981-1989 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27153689">27153689</a> </li>
</ul>
<h3 id="metausatmetamanova">metaUSAT/metaMANOVA</h3>
<ul>
<li><strong>NAME</strong> : metaUSAT/metaMANOVA </li>
<li><strong>SHORT NAME</strong> : metaUSAT/metaMANOVA </li>
<li><strong>FULL NAME</strong> : unified score-based association test </li>
<li><strong>DESCRIPTION</strong> : metaUSAT is a data-adaptive statistical approach for testing genetic associations of multiple traits from single/multiple studies using univariate GWAS summary statistics. This multivariate meta-analysis method can appropriately account for overlapping samples (if any) and can potentially test binary and/or continuous traits. </li>
<li><strong>URL</strong> : <a href="https://github.com/RayDebashree/metaUSAT">https://github.com/RayDebashree/metaUSAT</a> </li>
<li><strong>TITLE</strong> : Methods for meta-analysis of multiple traits using GWAS summary statistics </li>
<li><strong>DOI</strong> : 10.1002/gepi.22105 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) for complex diseases have focused primarily on single-trait analyses for disease status and disease-related quantitative traits. For example, GWAS on risk factors for coronary artery disease analyze genetic associations of plasma lipids such as total cholesterol, LDL-cholesterol, HDL-cholesterol, and triglycerides (TGs) separately. However, traits are often correlated and a joint analysis may yield increased statistical power for association over multiple univariate analyses. Recently several multivariate methods have been proposed that require individual-level data. Here, we develop metaUSAT (where USAT is unified score-based association test), a novel unified association test of a single genetic variant with multiple traits that uses only summary statistics from existing GWAS. Although the existing methods either perform well when most correlated traits are affected by the genetic variant in the same direction or are powerful when only a few of the correlated traits are associated, metaUSAT is designed to be robust to the association structure of correlated traits. metaUSAT does not require individual-level data and can test genetic associations of categorical and/or continuous traits. One can also use metaUSAT to analyze a single trait over multiple studies, appropriately accounting for overlapping samples, if any. metaUSAT provides an approximate asymptotic P-value for association and is computationally efficient for implementation at a genome-wide level. Simulation experiments show that metaUSAT maintains proper type-I error at low error levels. It has similar and sometimes greater power to detect association across a wide array of scenarios compared to existing methods, which are usually powerful for some specific association scenarios only. When applied to plasma lipids summary data from the METSIM and the T2D-GENES studies, metaUSAT detected genome-wide significant loci beyond the ones identified by univariate analyses. Evidence from larger studies suggest that the variants additionally detected by our test are, indeed, associated with lipid levels in humans. In summary, metaUSAT can provide novel insights into the genetic architecture of a common disease or traits. </li>
<li><strong>CITATION</strong> : Ray D, Boehnke M. (2018) Methods for meta-analysis of multiple traits using GWAS summary statistics Genet. Epidemiol., 42 (2) 134-145. doi:10.1002/gepi.22105. PMID 29226385 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet. Epidemiol. ; 2018 ; 42 ; 2 ; 134-145 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29226385">29226385</a> </li>
</ul>
<h3 id="mvgwama">mvGWAMA</h3>
<ul>
<li><strong>NAME</strong> : mvGWAMA </li>
<li><strong>SHORT NAME</strong> : mvGWAMA </li>
<li><strong>FULL NAME</strong> : Multivariate Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : mvGWAMA is a python script to perform a GWAS meta-analysis when there are sample overlap. </li>
<li><strong>URL</strong> : <a href="https://github.com/Kyoko-wtnb/mvGWAMA">https://github.com/Kyoko-wtnb/mvGWAMA</a> </li>
<li><strong>TITLE</strong> : Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0311-9 </li>
<li><strong>ABSTRACT</strong> : Alzheimer's disease (AD) is highly heritable and recent studies have identified over 20 disease-associated genomic loci. Yet these only explain a small proportion of the genetic variance, indicating that undiscovered loci remain. Here, we performed a large genome-wide association study of clinically diagnosed AD and AD-by-proxy (71,880 cases, 383,378 controls). AD-by-proxy, based on parental diagnoses, showed strong genetic correlation with AD (rg = 0.81). Meta-analysis identified 29 risk loci, implicating 215 potential causative genes. Associated genes are strongly expressed in immune-related tissues and cell types (spleen, liver, and microglia). Gene-set analyses indicate biological mechanisms involved in lipid-related processes and degradation of amyloid precursor proteins. We show strong genetic correlations with multiple health-related outcomes, and Mendelian randomization results suggest a protective effect of cognitive ability on AD risk. These results are a step forward in identifying the genetic factors that contribute to AD risk and add novel insights into the neurobiology of AD. </li>
<li><strong>CITATION</strong> : Jansen IE, Savage JE, Watanabe K, Bryois J, ...&amp;, Posthuma D. (2019) Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk Nat. Genet., 51 (3) 404-413. doi:10.1038/s41588-018-0311-9. PMID 30617256 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2019 ; 51 ; 3 ; 404-413 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30617256">30617256</a> </li>
</ul>
<h2 id="rare-variant">Rare-variant</h2>
<h3 id="metaskat">MetaSKAT</h3>
<ul>
<li><strong>NAME</strong> : MetaSKAT </li>
<li><strong>SHORT NAME</strong> : MetaSKAT </li>
<li><strong>FULL NAME</strong> : MetaSKAT </li>
<li><strong>DESCRIPTION</strong> : MetaSKAT is a R package for multiple marker meta-analysis. It can carry out meta-analysis of SKAT, SKAT-O and burden tests with individual level genotype data or gene level summary statistics. </li>
<li><strong>URL</strong> : <a href="https://www.hsph.harvard.edu/skat/metaskat/">https://www.hsph.harvard.edu/skat/metaskat/</a> </li>
<li><strong>TITLE</strong> : General framework for meta-analysis of rare variants in sequencing association studies </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2013.05.010 </li>
<li><strong>ABSTRACT</strong> : We propose a general statistical framework for meta-analysis of gene- or region-based multimarker rare variant association tests in sequencing association studies. In genome-wide association studies, single-marker meta-analysis has been widely used to increase statistical power by combining results via regression coefficients and standard errors from different studies. In analysis of rare variants in sequencing studies, region-based multimarker tests are often used to increase power. We propose meta-analysis methods for commonly used gene- or region-based rare variants tests, such as burden tests and variance component tests. Because estimation of regression coefficients of individual rare variants is often unstable or not feasible, the proposed method avoids this difficulty by calculating score statistics instead that only require fitting the null model for each study and then aggregating these score statistics across studies. Our proposed meta-analysis rare variant association tests are conducted based on study-specific summary statistics, specifically score statistics for each variant and between-variant covariance-type (linkage disequilibrium) relationship statistics for each gene or region. The proposed methods are able to incorporate different levels of heterogeneity of genetic effects across studies and are applicable to meta-analysis of multiple ancestry groups. We show that the proposed methods are essentially as powerful as joint analysis by directly pooling individual level genotype data. We conduct extensive simulations to evaluate the performance of our methods by varying levels of heterogeneity across studies, and we apply the proposed methods to meta-analysis of rare variant effects in a multicohort study of the genetics of blood lipid levels. </li>
<li><strong>CITATION</strong> : Lee S, Teslovich TM, Boehnke M, Lin X. (2013) General framework for meta-analysis of rare variants in sequencing association studies Am. J. Hum. Genet., 93 (1) 42-53. doi:10.1016/j.ajhg.2013.05.010. PMID 23768515 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2013 ; 93 ; 1 ; 42-53 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23768515">23768515</a> </li>
</ul>
<h3 id="metastaar">MetaSTAAR</h3>
<ul>
<li><strong>NAME</strong> : MetaSTAAR </li>
<li><strong>SHORT NAME</strong> : MetaSTAAR </li>
<li><strong>FULL NAME</strong> : MetaSTAAR </li>
<li><strong>DESCRIPTION</strong> : MetaSTAAR is an R package for performing Meta-analysis of variant-Set Test for Association using Annotation infoRmation (MetaSTAAR) procedure in whole-genome sequencing (WGS) studies. MetaSTAAR enables functionally-informed rare variant meta-analysis of large WGS studies using an efficient, sparse matrix approach for storing summary statistic, while protecting data privacy of study participants and avoiding sharing subject-level data. MetaSTAAR accounts for relatedness and population structure of continuous and dichotomous traits, and boosts the power of rare variant meta-analysis by incorporating multiple variant functional annotations. </li>
<li><strong>URL</strong> : <a href="https://github.com/xihaoli/MetaSTAAR">https://github.com/xihaoli/MetaSTAAR</a> </li>
<li><strong>TITLE</strong> : Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies </li>
<li><strong>DOI</strong> : 10.1038/s41588-022-01225-6 </li>
<li><strong>ABSTRACT</strong> : Meta-analysis of whole genome sequencing/whole exome sequencing (WGS/WES) studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes. Existing rare variant meta-analysis approaches are not scalable to biobank-scale WGS data. Here we present MetaSTAAR, a powerful and resource-efficient rare variant meta-analysis framework for large-scale WGS/WES studies. MetaSTAAR accounts for relatedness and population structure, can analyze both quantitative and dichotomous traits and boosts the power of rare variant tests by incorporating multiple variant functional annotations. Through meta-analysis of four lipid traits in 30,138 ancestrally diverse samples from 14 studies of the Trans Omics for Precision Medicine (TOPMed) Program, we show that MetaSTAAR performs rare variant meta-analysis at scale and produces results comparable to using pooled data. Additionally, we identified several conditionally significant rare variant associations with lipid traits. We further demonstrate that MetaSTAAR is scalable to biobank-scale cohorts through meta-analysis of TOPMed WGS data and UK Biobank WES data of ~200,000 samples. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Li X, Quick C, Zhou H, Gaynor SM, ...&amp;, Lin X. (2023) Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies Nat. Genet., 55 (1) 154-164. doi:10.1038/s41588-022-01225-6. PMID 36564505 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2023 ; 55 ; 1 ; 154-164 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36564505">36564505</a> </li>
</ul>
<h3 id="raremetal">RareMETAL</h3>
<ul>
<li><strong>NAME</strong> : RareMETAL </li>
<li><strong>SHORT NAME</strong> : RareMETAL </li>
<li><strong>FULL NAME</strong> : RareMETAL </li>
<li><strong>DESCRIPTION</strong> : RAREMETAL is a program that facilitates the meta-analysis of rare variants from genotype arrays or sequencing (manuscript in preparation). </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/RAREMETAL">https://genome.sph.umich.edu/wiki/RAREMETAL</a> </li>
<li><strong>KEYWORDS</strong> : rare variants </li>
<li><strong>TITLE</strong> : RAREMETAL: fast and powerful meta-analysis for rare variants </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btu367 </li>
<li><strong>ABSTRACT</strong> : SUMMARY: RAREMETAL is a computationally efficient tool for meta-analysis of rare variants genotyped using sequencing or arrays. RAREMETAL facilitates analyses of individual studies, accommodates a variety of input file formats, handles related and unrelated individuals, executes both single variant and burden tests and performs conditional association analyses. AVAILABILITY AND IMPLEMENTATION: http://genome.sph.umich.edu/wiki/RAREMETAL for executables, source code, documentation and tutorial. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/3.0/ </li>
<li><strong>CITATION</strong> : Feng S, Liu D, Zhan X, Wing MK, ...&amp;, Abecasis GR. (2014) RAREMETAL: fast and powerful meta-analysis for rare variants Bioinformatics, 30 (19) 2828-2829. doi:10.1093/bioinformatics/btu367. PMID 24894501 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2014 ; 30 ; 19 ; 2828-2829 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24894501">24894501</a> </li>
</ul>
<h3 id="smmat">SMMAT</h3>
<ul>
<li><strong>NAME</strong> : SMMAT </li>
<li><strong>SHORT NAME</strong> : SMMAT </li>
<li><strong>FULL NAME</strong> : variant set mixed model association tests </li>
<li><strong>DESCRIPTION</strong> : For rare variant analysis from sequencing association studies, GMMAT performs the variant Set Mixed Model Association Tests (SMMAT) as proposed in Chen et al. (2019), including the burden test, the sequence kernel association test (SKAT), SKAT-O and an efficient hybrid test of the burden test and SKAT, based on user-defined variant sets. </li>
<li><strong>URL</strong> : <a href="https://github.com/hanchenphd/GMMAT">https://github.com/hanchenphd/GMMAT</a> </li>
<li><strong>TITLE</strong> : Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2018.12.012 </li>
<li><strong>ABSTRACT</strong> : With advances in whole-genome sequencing (WGS) technology, more advanced statistical methods for testing genetic association with rare variants are being developed. Methods in which variants are grouped for analysis are also known as variant-set, gene-based, and aggregate unit tests. The burden test and sequence kernel association test (SKAT) are two widely used variant-set tests, which were originally developed for samples of unrelated individuals and later have been extended to family data with known pedigree structures. However, computationally efficient and powerful variant-set tests are needed to make analyses tractable in large-scale WGS studies with complex study samples. In this paper, we propose the variant-set mixed model association tests (SMMAT) for continuous and binary traits using the generalized linear mixed model framework. These tests can be applied to large-scale WGS studies involving samples with population structure and relatedness, such as in the National Heart, Lung, and Blood Institute's Trans-Omics for Precision Medicine (TOPMed) program. SMMATs share the same null model for different variant sets, and a virtue of this null model, which includes covariates only, is that it needs to be fit only once for all tests in each genome-wide analysis. Simulation studies show that all the proposed SMMATs correctly control type I error rates for both continuous and binary traits in the presence of population structure and relatedness. We also illustrate our tests in a real data example of analysis of plasma fibrinogen levels in the TOPMed program (n = 23,763), using the Analysis Commons, a cloud-based computing platform. </li>
<li><strong>CITATION</strong> : Chen H, Huffman JE, Brody JA, Wang C, ...&amp;, Lin X. (2019) Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies Am. J. Hum. Genet., 104 (2) 260-274. doi:10.1016/j.ajhg.2018.12.012. PMID 30639324 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2019 ; 104 ; 2 ; 260-274 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30639324">30639324</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f8cc74c7.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.c8b220af.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>