
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Mendelian_randomization_README/">
      
      
        <link rel="next" href="../Tools_Polygenic_risk_scores_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>Meta_and_Multi_triat - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Meta_and_Multi_triat
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Reference_data_README/" class="md-tabs__link">
          
  
  Reference_data

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          <label class="md-nav__link" for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Reference_data
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Allele_Frequency_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Allele_Frequency
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Epigenomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Expression_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Expression
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Gene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            Gene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_Ensembl_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Ensembl
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_GENCODE_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GENCODE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_HGNC_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HGNC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_NCBI_refSeq_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    NCBI_refSeq
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5_6" >
        
          
          <label class="md-nav__link" for="__nav_4_5_6" id="__nav_4_5_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_5_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5_6">
            <span class="md-nav__icon md-icon"></span>
            UCSC_refGene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ucsc_genes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_knownGenes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    knownGenes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_6" >
        
          
          <label class="md-nav__link" for="__nav_4_6" id="__nav_4_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Genome
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_6">
            <span class="md-nav__icon md-icon"></span>
            Genome
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_hg38_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    hg38
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Phenotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_xQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    xQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Venn_diagram_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Venn_diagram
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Workflow_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Workflow
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_IBD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Meta_and_Multi_triat</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: right;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#factorgo">FactorGO</a></td>
<td style="text-align: left;">Zhang Z, ...&amp;, Mancuso N. (2023) A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics. Am J Hum Genet, 110 (11) 1863-1874. doi:10.1016/j.ajhg.2023.09.015. PMID 37879338</td>
<td style="text-align: right;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gwama">GWAMA</a></td>
<td style="text-align: left;">Mägi R, Morris AP. (2010) GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics, 11  288. doi:10.1186/1471-2105-11-288. PMID 20509871</td>
<td style="text-align: right;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#genomic-sem">Genomic-SEM</a></td>
<td style="text-align: left;">Grotzinger AD, ...&amp;, Tucker-Drob EM. (2019) Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nat Hum Behav, 3 (5) 513-525. doi:10.1038/s41562-019-0566-x. PMID 30962613</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mantra">MANTRA</a></td>
<td style="text-align: left;">Morris AP. (2011) Transethnic meta-analysis of genomewide association studies. Genet Epidemiol, 35 (8) 809-22. doi:10.1002/gepi.20630. PMID 22125221</td>
<td style="text-align: right;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metal">METAL</a></td>
<td style="text-align: left;">Willer CJ, ...&amp;, Abecasis GR. (2010) METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26 (17) 2190-1. doi:10.1093/bioinformatics/btq340. PMID 20616382</td>
<td style="text-align: right;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mr-mega">MR-MEGA</a></td>
<td style="text-align: left;">Mägi R, ...&amp;, Morris AP. (2017) Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Hum Mol Genet, 26 (18) 3639-3650. doi:10.1093/hmg/ddx280. PMID 28911207</td>
<td style="text-align: right;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mtag">MTAG</a></td>
<td style="text-align: left;">Turley P, ...&amp;, Benjamin DJ. (2018) Multi-trait analysis of genome-wide association summary statistics using MTAG. Nat Genet, 50 (2) 229-237. doi:10.1038/s41588-017-0009-4. PMID 29292387</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metaskat">MetaSKAT</a></td>
<td style="text-align: left;">Lee S, ...&amp;, Lin X. (2013) General framework for meta-analysis of rare variants in sequencing association studies. Am J Hum Genet, 93 (1) 42-53. doi:10.1016/j.ajhg.2013.05.010. PMID 23768515</td>
<td style="text-align: right;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metastaar">MetaSTAAR</a></td>
<td style="text-align: left;">Li X, ...&amp;, Lin X. (2023) Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. Nat Genet, 55 (1) 154-164. doi:10.1038/s41588-022-01225-6. PMID 36564505</td>
<td style="text-align: right;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#raremetal">RareMETAL</a></td>
<td style="text-align: left;">Feng S, ...&amp;, Abecasis GR. (2014) RAREMETAL: fast and powerful meta-analysis for rare variants. Bioinformatics, 30 (19) 2828-9. doi:10.1093/bioinformatics/btu367. PMID 24894501</td>
<td style="text-align: right;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#smmat">SMMAT</a></td>
<td style="text-align: left;">Chen H, ...&amp;, Lin X. (2019) Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies. Am J Hum Genet, 104 (2) 260-274. doi:10.1016/j.ajhg.2018.12.012. PMID 30639324</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mvgwama">mvGWAMA</a></td>
<td style="text-align: left;">Jansen IE, ...&amp;, Posthuma D. (2019) Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. Nat Genet, 51 (3) 404-413. doi:10.1038/s41588-018-0311-9. PMID 30617256</td>
<td style="text-align: right;">2019</td>
</tr>
</tbody>
</table>
<h2 id="factorgo">FactorGO</h2>
<ul>
<li><strong>NAME</strong> : FactorGO </li>
<li><strong>SHORT NAME</strong> : FactorGO </li>
<li><strong>FULL NAME</strong> : Factor analysis model in Genetic assOciation </li>
<li><strong>DESCRIPTION</strong> : FactorGo is a scalable variational factor analysis model that learns pleiotropic factors using GWAS summary statistics. </li>
<li><strong>URL</strong> : https://github.com/mancusolab/FactorGo </li>
<li><strong>KEYWORDS</strong> : pleiotropy, factor analysis </li>
<li><strong>TITLE</strong> : A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics. </li>
<li><strong>CITATION</strong> : Zhang Z, ...&amp;, Mancuso N. (2023) A scalable approach to characterize pleiotropy across thousands of human diseases and complex traits using GWAS summary statistics. Am J Hum Genet, 110 (11) 1863-1874. doi:10.1016/j.ajhg.2023.09.015. PMID 37879338 </li>
<li><strong>MESH_MAJOR</strong> : Arthritis, Rheumatoid, Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Brain, Genetic Pleiotropy, Humans, Male, Multifactorial Inheritance, Phenotype, Polymorphism, Single Nucleotide </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWASs) across thousands of traits have revealed the pervasive pleiotropy of trait-associated genetic variants. While methods have been proposed to characterize pleiotropic components across groups of phenotypes, scaling these approaches to ultra-large-scale biobanks has been challenging. Here, we propose FactorGo, a scalable variational factor analysis model to identify and characterize pleiotropic components using biobank GWAS summary data. In extensive simulations, we observe that FactorGo outperforms the state-of-the-art (model-free) approach tSVD in capturing latent pleiotropic factors across phenotypes while maintaining a similar computational cost. We apply FactorGo to estimate 100 latent pleiotropic factors from GWAS summary data of 2,483 phenotypes measured in European-ancestry Pan-UK BioBank individuals (N = 420,531). Next, we find that factors from FactorGo are more enriched with relevant tissue-specific annotations than those identified by tSVD (p = 2.58E-10) and validate our approach by recapitulating brain-specific enrichment for BMI and the height-related connection between reproductive system and muscular-skeletal growth. Finally, our analyses suggest shared etiologies between rheumatoid arthritis and periodontal condition in addition to alkaline phosphatase as a candidate prognostic biomarker for prostate cancer. Overall, FactorGo improves our biological understanding of shared etiologies across thousands of GWASs. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2023 American Society of Human Genetics. Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2023.09.015 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am J Hum Genet ; 2023 ; 110 ; 11 ; 1863-1874 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37879338">37879338</a> </li>
</ul>
<h2 id="gwama">GWAMA</h2>
<ul>
<li><strong>NAME</strong> : GWAMA </li>
<li><strong>SHORT NAME</strong> : GWAMA </li>
<li><strong>FULL NAME</strong> : Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : Software tool for meta analysis of whole genome association data </li>
<li><strong>URL</strong> : https://genomics.ut.ee/en/tools </li>
<li><strong>TITLE</strong> : GWAMA: software for genome-wide association meta-analysis. </li>
<li><strong>CITATION</strong> : Mägi R, Morris AP. (2010) GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics, 11  288. doi:10.1186/1471-2105-11-288. PMID 20509871 </li>
<li><strong>MESH_MAJOR</strong> : Genome, Software </li>
<li><strong>MESH_MINOR</strong> : Databases, Genetic, Genome-Wide Association Study, Genotype, Phenotype </li>
<li><strong>ABSTRACT</strong> : Despite the recent success of genome-wide association studies in identifying novel loci contributing effects to complex human traits, such as type 2 diabetes and obesity, much of the genetic component of variation in these phenotypes remains unexplained. One way to improving power to detect further novel loci is through meta-analysis of studies from the same population, increasing the sample size over any individual study. Although statistical software analysis packages incorporate routines for meta-analysis, they are ill equipped to meet the challenges of the scale and complexity of data generated in genome-wide association studies.|We have developed flexible, open-source software for the meta-analysis of genome-wide association studies. The software incorporates a variety of error trapping facilities, and provides a range of meta-analysis summary statistics. The software is distributed with scripts that allow simple formatting of files containing the results of each association study and generate graphical summaries of genome-wide meta-analysis results.|The GWAMA (Genome-Wide Association Meta-Analysis) software has been developed to perform meta-analysis of summary statistics generated from genome-wide association studies of dichotomous phenotypes or quantitative traits. Software with source files, documentation and example data files are freely available online at http://www.well.ox.ac.uk/GWAMA. </li>
<li><strong>DOI</strong> : 10.1186/1471-2105-11-288 </li>
<li><strong>JOURNAL_INFO</strong> : BMC bioinformatics ; BMC Bioinformatics ; 2010 ; 11 ; <NA> ; 288 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20509871">20509871</a> </li>
</ul>
<h2 id="genomic-sem">Genomic-SEM</h2>
<ul>
<li><strong>NAME</strong> : Genomic-SEM </li>
<li><strong>SHORT NAME</strong> : Genomic-SEM </li>
<li><strong>FULL NAME</strong> : genomic structural equation modelling </li>
<li><strong>DESCRIPTION</strong> : R-package which allows the user to fit structural equation models based on the summary statistics obtained from genome wide association studies (GWAS). </li>
<li><strong>URL</strong> : https://github.com/GenomicSEM/GenomicSEM </li>
<li><strong>KEYWORDS</strong> : SEM </li>
<li><strong>TITLE</strong> : Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. </li>
<li><strong>CITATION</strong> : Grotzinger AD, ...&amp;, Tucker-Drob EM. (2019) Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nat Hum Behav, 3 (5) 513-525. doi:10.1038/s41562-019-0566-x. PMID 30962613 </li>
<li><strong>MESH_MAJOR</strong> : Latent Class Analysis </li>
<li><strong>MESH_MINOR</strong> : Factor Analysis, Statistical, Genome-Wide Association Study, Genomics, Humans, Mental Disorders, Multifactorial Inheritance, Multivariate Analysis, Polymorphism, Single Nucleotide </li>
<li><strong>ABSTRACT</strong> : Genetic correlations estimated from genome-wide association studies (GWASs) reveal pervasive pleiotropy across a wide variety of phenotypes. We introduce genomic structural equation modelling (genomic SEM): a multivariate method for analysing the joint genetic architecture of complex traits. Genomic SEM synthesizes genetic correlations and single-nucleotide polymorphism heritabilities inferred from GWAS summary statistics of individual traits from samples with varying and unknown degrees of overlap. Genomic SEM can be used to model multivariate genetic associations among phenotypes, identify variants with effects on general dimensions of cross-trait liability, calculate more predictive polygenic scores and identify loci that cause divergence between traits. We demonstrate several applications of genomic SEM, including a joint analysis of summary statistics from five psychiatric traits. We identify 27 independent single-nucleotide polymorphisms not previously identified in the contributing univariate GWASs. Polygenic scores from genomic SEM consistently outperform those from univariate GWASs. Genomic SEM is flexible and open ended, and allows for continuous innovation in multivariate genetic analysis. </li>
<li><strong>DOI</strong> : 10.1038/s41562-019-0566-x </li>
<li><strong>JOURNAL_INFO</strong> : Nature human behaviour ; Nat Hum Behav ; 2019 ; 3 ; 5 ; 513-525 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30962613">30962613</a> </li>
</ul>
<h2 id="mantra">MANTRA</h2>
<ul>
<li><strong>NAME</strong> : MANTRA </li>
<li><strong>SHORT NAME</strong> : MANTRA </li>
<li><strong>FULL NAME</strong> : Meta-ANalysis of Transethnic Association studies </li>
<li><strong>KEYWORDS</strong> : cross-population </li>
<li><strong>TITLE</strong> : Transethnic meta-analysis of genomewide association studies. </li>
<li><strong>CITATION</strong> : Morris AP. (2011) Transethnic meta-analysis of genomewide association studies. Genet Epidemiol, 35 (8) 809-22. doi:10.1002/gepi.20630. PMID 22125221 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Software </li>
<li><strong>MESH_MINOR</strong> : Alleles, Bayes Theorem, Cyclin-Dependent Kinase 5, Diabetes Mellitus, Type 2, Ethnicity, Genetic Heterogeneity, Humans, Sample Size, tRNA Methyltransferases </li>
<li><strong>ABSTRACT</strong> : The detection of loci contributing effects to complex human traits, and their subsequent fine-mapping for the location of causal variants, remains a considerable challenge for the genetics research community. Meta-analyses of genomewide association studies, primarily ascertained from European-descent populations, have made considerable advances in our understanding of complex trait genetics, although much of their heritability is still unexplained. With the increasing availability of genomewide association data from diverse populations, transethnic meta-analysis may offer an exciting opportunity to increase the power to detect novel complex trait loci and to improve the resolution of fine-mapping of causal variants by leveraging differences in local linkage disequilibrium structure between ethnic groups. However, we might also expect there to be substantial genetic heterogeneity between diverse populations, both in terms of the spectrum of causal variants and their allelic effects, which cannot easily be accommodated through traditional approaches to meta-analysis. In order to address this challenge, I propose novel transethnic meta-analysis methodology that takes account of the expected similarity in allelic effects between the most closely related populations, while allowing for heterogeneity between more diverse ethnic groups. This approach yields substantial improvements in performance, compared to fixed-effects meta-analysis, both in terms of power to detect association, and localization of the causal variant, over a range of models of heterogeneity between ethnic groups. Furthermore, when the similarity in allelic effects between populations is well captured by their relatedness, this approach has increased power and mapping resolution over random-effects meta-analysis. </li>
<li><strong>COPYRIGHT</strong> : © 2011 Wiley Periodicals, Inc. </li>
<li><strong>DOI</strong> : 10.1002/gepi.20630 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet Epidemiol ; 2011 ; 35 ; 8 ; 809-22 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22125221">22125221</a> </li>
</ul>
<h2 id="metal">METAL</h2>
<ul>
<li><strong>NAME</strong> : METAL </li>
<li><strong>SHORT NAME</strong> : METAL </li>
<li><strong>FULL NAME</strong> : METAL </li>
<li><strong>DESCRIPTION</strong> : METAL is a tool for meta-analysis genomewide association scans. METAL can combine either (a) test statistics and standard errors or (b) p-values across studies (taking sample size and direction of effect into account). METAL analysis is a convenient alternative to a direct analysis of merged data from multiple studies. It is especially appropriate when data from the individual studies cannot be analyzed together because of differences in ethnicity, phenotype distribution, gender or constraints in sharing of individual level data imposed. Meta-analysis results in little or no loss of efficiency compared to analysis of a combined dataset including data from all individual studies. </li>
<li><strong>URL</strong> : https://genome.sph.umich.edu/wiki/METAL_Documentation </li>
<li><strong>TITLE</strong> : METAL: fast and efficient meta-analysis of genomewide association scans. </li>
<li><strong>CITATION</strong> : Willer CJ, ...&amp;, Abecasis GR. (2010) METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26 (17) 2190-1. doi:10.1093/bioinformatics/btq340. PMID 20616382 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Software </li>
<li><strong>MESH_MINOR</strong> : Chromosome Mapping, Computational Biology </li>
<li><strong>ABSTRACT</strong> : METAL provides a computationally efficient tool for meta-analysis of genome-wide association scans, which is a commonly used approach for improving power complex traits gene mapping studies. METAL provides a rich scripting interface and implements efficient memory management to allow analyses of very large data sets and to support a variety of input file formats.|METAL, including source code, documentation, examples, and executables, is available at http://www.sph.umich.edu/csg/abecasis/metal/. </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btq340 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2010 ; 26 ; 17 ; 2190-1 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20616382">20616382</a> </li>
</ul>
<h2 id="mr-mega">MR-MEGA</h2>
<ul>
<li><strong>NAME</strong> : MR-MEGA </li>
<li><strong>SHORT NAME</strong> : MR-MEGA </li>
<li><strong>FULL NAME</strong> : Meta-Regression of Multi-AncEstry Genetic Association </li>
<li><strong>DESCRIPTION</strong> : MR-MEGA (Meta-Regression of Multi-AncEstry Genetic Association) is a tool to detect and fine-map complex trait association signals via multi-ancestry meta-regression.  This approach uses genome-wide metrics of diversity between populations to derive axes of genetic variation via multi-dimensional scaling [Purcell 2007].  Allelic effects of a variant across GWAS, weighted by their corresponding standard errors, can then be modelled in a linear regression framework, including the axes of genetic variation as covariates.  The flexibility of this model enables partitioning of the heterogeneity into components due to ancestry and residual variation, which would be expected to improve fine-mapping resolution. </li>
<li><strong>URL</strong> : https://genomics.ut.ee/en/tools </li>
<li><strong>KEYWORDS</strong> : cross-population, Meta-Regression </li>
<li><strong>TITLE</strong> : Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. </li>
<li><strong>CITATION</strong> : Mägi R, ...&amp;, Morris AP. (2017) Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Hum Mol Genet, 26 (18) 3639-3650. doi:10.1093/hmg/ddx280. PMID 28911207 </li>
<li><strong>MESH_MAJOR</strong> :  </li>
<li><strong>MESH_MINOR</strong> : Alleles, Diabetes Mellitus, Type 2, Ethnicity, Gene Frequency, Genome-Wide Association Study, Humans, Linkage Disequilibrium, Polymorphism, Single Nucleotide, Sequence Analysis, DNA, Software, White People </li>
<li><strong>ABSTRACT</strong> : Trans-ethnic meta-analysis of genome-wide association studies (GWAS) across diverse populations can increase power to detect complex trait loci when the underlying causal variants are shared between ancestry groups. However, heterogeneity in allelic effects between GWAS at these loci can occur that is correlated with ancestry. Here, a novel approach is presented to detect SNP association and quantify the extent of heterogeneity in allelic effects that is correlated with ancestry. We employ trans-ethnic meta-regression to model allelic effects as a function of axes of genetic variation, derived from a matrix of mean pairwise allele frequency differences between GWAS, and implemented in the MR-MEGA software. Through detailed simulations, we demonstrate increased power to detect association for MR-MEGA over fixed- and random-effects meta-analysis across a range of scenarios of heterogeneity in allelic effects between ethnic groups. We also demonstrate improved fine-mapping resolution, in loci containing a single causal variant, compared to these meta-analysis approaches and PAINTOR, and equivalent performance to MANTRA at reduced computational cost. Application of MR-MEGA to trans-ethnic GWAS of kidney function in 71,461 individuals indicates stronger signals of association than fixed-effects meta-analysis when heterogeneity in allelic effects is correlated with ancestry. Application of MR-MEGA to fine-mapping four type 2 diabetes susceptibility loci in 22,086 cases and 42,539 controls highlights: (i) strong evidence for heterogeneity in allelic effects that is correlated with ancestry only at the index SNP for the association signal at the CDKAL1 locus; and (ii) 99% credible sets with six or fewer variants for five distinct association signals. </li>
<li><strong>COPYRIGHT</strong> : © The Author 2017. Published by Oxford University Press. </li>
<li><strong>DOI</strong> : 10.1093/hmg/ddx280 </li>
<li><strong>JOURNAL_INFO</strong> : Human molecular genetics ; Hum Mol Genet ; 2017 ; 26 ; 18 ; 3639-3650 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28911207">28911207</a> </li>
</ul>
<h2 id="mtag">MTAG</h2>
<ul>
<li><strong>NAME</strong> : MTAG </li>
<li><strong>SHORT NAME</strong> : MTAG </li>
<li><strong>FULL NAME</strong> : Multi-Trait Analysis of GWAS </li>
<li><strong>DESCRIPTION</strong> : mtag is a Python-based command line tool for jointly analyzing multiple sets of GWAS summary statistics as described by Turley et. al. (2018). It can also be used as a tool to meta-analyze GWAS results. </li>
<li><strong>URL</strong> : https://github.com/JonJala/mtag </li>
<li><strong>KEYWORDS</strong> : Multi-trait </li>
<li><strong>TITLE</strong> : Multi-trait analysis of genome-wide association summary statistics using MTAG. </li>
<li><strong>CITATION</strong> : Turley P, ...&amp;, Benjamin DJ. (2018) Multi-trait analysis of genome-wide association summary statistics using MTAG. Nat Genet, 50 (2) 229-237. doi:10.1038/s41588-017-0009-4. PMID 29292387 </li>
<li><strong>MESH_MAJOR</strong> : Data Interpretation, Statistical, Multifactorial Inheritance, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Algorithms, Datasets as Topic, Depression, Diagnostic Self Evaluation, Genetic Association Studies, Genome-Wide Association Study, Health, Humans, Meta-Analysis as Topic, Neuroticism, Phenotype, Polymorphism, Single Nucleotide </li>
<li><strong>ABSTRACT</strong> : We introduce multi-trait analysis of GWAS (MTAG), a method for joint analysis of summary statistics from genome-wide association studies (GWAS) of different traits, possibly from overlapping samples. We apply MTAG to summary statistics for depressive symptoms (N <sub>eff</sub> = 354,862), neuroticism (N = 168,105), and subjective well-being (N = 388,538). As compared to the 32, 9, and 13 genome-wide significant loci identified in the single-trait GWAS (most of which are themselves novel), MTAG increases the number of associated loci to 64, 37, and 49, respectively. Moreover, association statistics from MTAG yield more informative bioinformatics analyses and increase the variance explained by polygenic scores by approximately 25%, matching theoretical expectations. </li>
<li><strong>DOI</strong> : 10.1038/s41588-017-0009-4 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2018 ; 50 ; 2 ; 229-237 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29292387">29292387</a> </li>
</ul>
<h2 id="metaskat">MetaSKAT</h2>
<ul>
<li><strong>NAME</strong> : MetaSKAT </li>
<li><strong>SHORT NAME</strong> : MetaSKAT </li>
<li><strong>FULL NAME</strong> : MetaSKAT </li>
<li><strong>DESCRIPTION</strong> : MetaSKAT is a R package for multiple marker meta-analysis. It can carry out meta-analysis of SKAT, SKAT-O and burden tests with individual level genotype data or gene level summary statistics. </li>
<li><strong>URL</strong> : https://www.hsph.harvard.edu/skat/metaskat/ </li>
<li><strong>TITLE</strong> : General framework for meta-analysis of rare variants in sequencing association studies. </li>
<li><strong>CITATION</strong> : Lee S, ...&amp;, Lin X. (2013) General framework for meta-analysis of rare variants in sequencing association studies. Am J Hum Genet, 93 (1) 42-53. doi:10.1016/j.ajhg.2013.05.010. PMID 23768515 </li>
<li><strong>MESH_MAJOR</strong> : Meta-Analysis as Topic </li>
<li><strong>MESH_MINOR</strong> : Computer Simulation, Genetic Association Studies, Genetic Diseases, Inborn, Genetic Predisposition to Disease, Genetics, Population, Humans, Lipids, Logistic Models, Polymorphism, Single Nucleotide, Reproducibility of Results, White People </li>
<li><strong>ABSTRACT</strong> : We propose a general statistical framework for meta-analysis of gene- or region-based multimarker rare variant association tests in sequencing association studies. In genome-wide association studies, single-marker meta-analysis has been widely used to increase statistical power by combining results via regression coefficients and standard errors from different studies. In analysis of rare variants in sequencing studies, region-based multimarker tests are often used to increase power. We propose meta-analysis methods for commonly used gene- or region-based rare variants tests, such as burden tests and variance component tests. Because estimation of regression coefficients of individual rare variants is often unstable or not feasible, the proposed method avoids this difficulty by calculating score statistics instead that only require fitting the null model for each study and then aggregating these score statistics across studies. Our proposed meta-analysis rare variant association tests are conducted based on study-specific summary statistics, specifically score statistics for each variant and between-variant covariance-type (linkage disequilibrium) relationship statistics for each gene or region. The proposed methods are able to incorporate different levels of heterogeneity of genetic effects across studies and are applicable to meta-analysis of multiple ancestry groups. We show that the proposed methods are essentially as powerful as joint analysis by directly pooling individual level genotype data. We conduct extensive simulations to evaluate the performance of our methods by varying levels of heterogeneity across studies, and we apply the proposed methods to meta-analysis of rare variant effects in a multicohort study of the genetics of blood lipid levels. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2013 The American Society of Human Genetics. Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2013.05.010 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am J Hum Genet ; 2013 ; 93 ; 1 ; 42-53 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23768515">23768515</a> </li>
</ul>
<h2 id="metastaar">MetaSTAAR</h2>
<ul>
<li><strong>NAME</strong> : MetaSTAAR </li>
<li><strong>SHORT NAME</strong> : MetaSTAAR </li>
<li><strong>FULL NAME</strong> : MetaSTAAR </li>
<li><strong>DESCRIPTION</strong> : MetaSTAAR is an R package for performing Meta-analysis of variant-Set Test for Association using Annotation infoRmation (MetaSTAAR) procedure in whole-genome sequencing (WGS) studies. MetaSTAAR enables functionally-informed rare variant meta-analysis of large WGS studies using an efficient, sparse matrix approach for storing summary statistic, while protecting data privacy of study participants and avoiding sharing subject-level data. MetaSTAAR accounts for relatedness and population structure of continuous and dichotomous traits, and boosts the power of rare variant meta-analysis by incorporating multiple variant functional annotations. </li>
<li><strong>URL</strong> : https://github.com/xihaoli/MetaSTAAR </li>
<li><strong>TITLE</strong> : Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. </li>
<li><strong>CITATION</strong> : Li X, ...&amp;, Lin X. (2023) Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. Nat Genet, 55 (1) 154-164. doi:10.1038/s41588-022-01225-6. PMID 36564505 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Lipids </li>
<li><strong>MESH_MINOR</strong> : Exome Sequencing, Phenotype, Whole Genome Sequencing </li>
<li><strong>ABSTRACT</strong> : Meta-analysis of whole genome sequencing/whole exome sequencing (WGS/WES) studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes. Existing rare variant meta-analysis approaches are not scalable to biobank-scale WGS data. Here we present MetaSTAAR, a powerful and resource-efficient rare variant meta-analysis framework for large-scale WGS/WES studies. MetaSTAAR accounts for relatedness and population structure, can analyze both quantitative and dichotomous traits and boosts the power of rare variant tests by incorporating multiple variant functional annotations. Through meta-analysis of four lipid traits in 30,138 ancestrally diverse samples from 14 studies of the Trans Omics for Precision Medicine (TOPMed) Program, we show that MetaSTAAR performs rare variant meta-analysis at scale and produces results comparable to using pooled data. Additionally, we identified several conditionally significant rare variant associations with lipid traits. We further demonstrate that MetaSTAAR is scalable to biobank-scale cohorts through meta-analysis of TOPMed WGS data and UK Biobank WES data of ~200,000 samples. </li>
<li><strong>COPYRIGHT</strong> : © 2022. The Author(s), under exclusive licence to Springer Nature America, Inc. </li>
<li><strong>DOI</strong> : 10.1038/s41588-022-01225-6 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2023 ; 55 ; 1 ; 154-164 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36564505">36564505</a> </li>
</ul>
<h2 id="raremetal">RareMETAL</h2>
<ul>
<li><strong>NAME</strong> : RareMETAL </li>
<li><strong>SHORT NAME</strong> : RareMETAL </li>
<li><strong>FULL NAME</strong> : RareMETAL </li>
<li><strong>DESCRIPTION</strong> : RAREMETAL is a program that facilitates the meta-analysis of rare variants from genotype arrays or sequencing (manuscript in preparation). </li>
<li><strong>URL</strong> : https://genome.sph.umich.edu/wiki/RAREMETAL </li>
<li><strong>KEYWORDS</strong> : rare variants </li>
<li><strong>TITLE</strong> : RAREMETAL: fast and powerful meta-analysis for rare variants. </li>
<li><strong>CITATION</strong> : Feng S, ...&amp;, Abecasis GR. (2014) RAREMETAL: fast and powerful meta-analysis for rare variants. Bioinformatics, 30 (19) 2828-9. doi:10.1093/bioinformatics/btu367. PMID 24894501 </li>
<li><strong>MESH_MAJOR</strong> : Software </li>
<li><strong>MESH_MINOR</strong> : Algorithms, Computational Biology, Data Interpretation, Statistical, Genome-Wide Association Study, Genotype, Humans, Linkage Disequilibrium, Models, Statistical, Programming Languages, Quantitative Trait Loci, Reproducibility of Results </li>
<li><strong>ABSTRACT</strong> : RAREMETAL is a computationally efficient tool for meta-analysis of rare variants genotyped using sequencing or arrays. RAREMETAL facilitates analyses of individual studies, accommodates a variety of input file formats, handles related and unrelated individuals, executes both single variant and burden tests and performs conditional association analyses.|http://genome.sph.umich.edu/wiki/RAREMETAL for executables, source code, documentation and tutorial. </li>
<li><strong>COPYRIGHT</strong> : © The Author 2014. Published by Oxford University Press. </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btu367 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2014 ; 30 ; 19 ; 2828-9 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24894501">24894501</a> </li>
</ul>
<h2 id="smmat">SMMAT</h2>
<ul>
<li><strong>NAME</strong> : SMMAT </li>
<li><strong>SHORT NAME</strong> : SMMAT </li>
<li><strong>FULL NAME</strong> : variant set mixed model association tests </li>
<li><strong>DESCRIPTION</strong> : For rare variant analysis from sequencing association studies, GMMAT performs the variant Set Mixed Model Association Tests (SMMAT) as proposed in Chen et al. (2019), including the burden test, the sequence kernel association test (SKAT), SKAT-O and an efficient hybrid test of the burden test and SKAT, based on user-defined variant sets. </li>
<li><strong>URL</strong> : https://github.com/hanchenphd/GMMAT </li>
<li><strong>TITLE</strong> : Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies. </li>
<li><strong>CITATION</strong> : Chen H, ...&amp;, Lin X. (2019) Efficient Variant Set Mixed Model Association Tests for Continuous and Binary Traits in Large-Scale Whole-Genome Sequencing Studies. Am J Hum Genet, 104 (2) 260-274. doi:10.1016/j.ajhg.2018.12.012. PMID 30639324 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Association Studies, Models, Genetic, Whole Genome Sequencing </li>
<li><strong>MESH_MINOR</strong> : Chromosomes, Human, Pair 4, Cloud Computing, Female, Fibrinogen, Genetics, Population, Humans, Male, National Heart, Lung, and Blood Institute (U.S.), Precision Medicine, Research Design, Time Factors, United States </li>
<li><strong>ABSTRACT</strong> : With advances in whole-genome sequencing (WGS) technology, more advanced statistical methods for testing genetic association with rare variants are being developed. Methods in which variants are grouped for analysis are also known as variant-set, gene-based, and aggregate unit tests. The burden test and sequence kernel association test (SKAT) are two widely used variant-set tests, which were originally developed for samples of unrelated individuals and later have been extended to family data with known pedigree structures. However, computationally efficient and powerful variant-set tests are needed to make analyses tractable in large-scale WGS studies with complex study samples. In this paper, we propose the variant-set mixed model association tests (SMMAT) for continuous and binary traits using the generalized linear mixed model framework. These tests can be applied to large-scale WGS studies involving samples with population structure and relatedness, such as in the National Heart, Lung, and Blood Institute's Trans-Omics for Precision Medicine (TOPMed) program. SMMATs share the same null model for different variant sets, and a virtue of this null model, which includes covariates only, is that it needs to be fit only once for all tests in each genome-wide analysis. Simulation studies show that all the proposed SMMATs correctly control type I error rates for both continuous and binary traits in the presence of population structure and relatedness. We also illustrate our tests in a real data example of analysis of plasma fibrinogen levels in the TOPMed program (n = 23,763), using the Analysis Commons, a cloud-based computing platform. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2018 American Society of Human Genetics. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2018.12.012 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am J Hum Genet ; 2019 ; 104 ; 2 ; 260-274 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30639324">30639324</a> </li>
</ul>
<h2 id="mvgwama">mvGWAMA</h2>
<ul>
<li><strong>NAME</strong> : mvGWAMA </li>
<li><strong>SHORT NAME</strong> : mvGWAMA </li>
<li><strong>FULL NAME</strong> : Multivariate Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : mvGWAMA is a python script to perform a GWAS meta-analysis when there are sample overlap. </li>
<li><strong>URL</strong> : https://github.com/Kyoko-wtnb/mvGWAMA </li>
<li><strong>TITLE</strong> : Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. </li>
<li><strong>CITATION</strong> : Jansen IE, ...&amp;, Posthuma D. (2019) Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. Nat Genet, 51 (3) 404-413. doi:10.1038/s41588-018-0311-9. PMID 30617256 </li>
<li><strong>MESH_MAJOR</strong> :  </li>
<li><strong>MESH_MINOR</strong> : Adult, Alzheimer Disease, Case-Control Studies, Female, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Male, Middle Aged, Polymorphism, Single Nucleotide, Quantitative Trait Loci, Risk, Young Adult </li>
<li><strong>ABSTRACT</strong> : Alzheimer's disease (AD) is highly heritable and recent studies have identified over 20 disease-associated genomic loci. Yet these only explain a small proportion of the genetic variance, indicating that undiscovered loci remain. Here, we performed a large genome-wide association study of clinically diagnosed AD and AD-by-proxy (71,880 cases, 383,378 controls). AD-by-proxy, based on parental diagnoses, showed strong genetic correlation with AD (r<sub>g</sub> = 0.81). Meta-analysis identified 29 risk loci, implicating 215 potential causative genes. Associated genes are strongly expressed in immune-related tissues and cell types (spleen, liver, and microglia). Gene-set analyses indicate biological mechanisms involved in lipid-related processes and degradation of amyloid precursor proteins. We show strong genetic correlations with multiple health-related outcomes, and Mendelian randomization results suggest a protective effect of cognitive ability on AD risk. These results are a step forward in identifying the genetic factors that contribute to AD risk and add novel insights into the neurobiology of AD. </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0311-9 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2019 ; 51 ; 3 ; 404-413 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30617256">30617256</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>