
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Mendelian_randomization_README/">
      
      
        <link rel="next" href="../Tools_Polygenic_risk_scores_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Meta_and_Multi_triat - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Meta_and_Multi_triat
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      Meta-analysis
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Meta-analysis">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Multi-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asset" class="md-nav__link">
    <span class="md-ellipsis">
      ASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#galesloot" class="md-nav__link">
    <span class="md-ellipsis">
      Galesloot
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipo" class="md-nav__link">
    <span class="md-ellipsis">
      HIPO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jass" class="md-nav__link">
    <span class="md-ellipsis">
      JASS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lcp-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      LCP-GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manova" class="md-nav__link">
    <span class="md-ellipsis">
      MANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mostest" class="md-nav__link">
    <span class="md-ellipsis">
      MOSTest
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mv-plink-mqfam" class="md-nav__link">
    <span class="md-ellipsis">
      MV-PLINK (MQFAM)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiphen" class="md-nav__link">
    <span class="md-ellipsis">
      MultiPhen
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pchat" class="md-nav__link">
    <span class="md-ellipsis">
      PCHAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porter" class="md-nav__link">
    <span class="md-ellipsis">
      Porter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salinas" class="md-nav__link">
    <span class="md-ellipsis">
      Salinas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stephens" class="md-nav__link">
    <span class="md-ellipsis">
      Stephens
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tates" class="md-nav__link">
    <span class="md-ellipsis">
      TATES
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang" class="md-nav__link">
    <span class="md-ellipsis">
      Yang
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amat" class="md-nav__link">
    <span class="md-ellipsis">
      aMAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastasset" class="md-nav__link">
    <span class="md-ellipsis">
      fastASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metacca" class="md-nav__link">
    <span class="md-ellipsis">
      metaCCA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metausatmetamanova" class="md-nav__link">
    <span class="md-ellipsis">
      metaUSAT/metaMANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Rare-variant
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Rare-variant">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      Meta-analysis
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Meta-analysis">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwama" class="md-nav__link">
    <span class="md-ellipsis">
      GWAMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mantra" class="md-nav__link">
    <span class="md-ellipsis">
      MANTRA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metal" class="md-nav__link">
    <span class="md-ellipsis">
      METAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Multi-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asset" class="md-nav__link">
    <span class="md-ellipsis">
      ASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#factorgo" class="md-nav__link">
    <span class="md-ellipsis">
      FactorGO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#galesloot" class="md-nav__link">
    <span class="md-ellipsis">
      Galesloot
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomic-sem" class="md-nav__link">
    <span class="md-ellipsis">
      Genomic-SEM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hipo" class="md-nav__link">
    <span class="md-ellipsis">
      HIPO
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jass" class="md-nav__link">
    <span class="md-ellipsis">
      JASS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lcp-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      LCP-GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manova" class="md-nav__link">
    <span class="md-ellipsis">
      MANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mostest" class="md-nav__link">
    <span class="md-ellipsis">
      MOSTest
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mtag" class="md-nav__link">
    <span class="md-ellipsis">
      MTAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mv-plink-mqfam" class="md-nav__link">
    <span class="md-ellipsis">
      MV-PLINK (MQFAM)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiphen" class="md-nav__link">
    <span class="md-ellipsis">
      MultiPhen
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pchat" class="md-nav__link">
    <span class="md-ellipsis">
      PCHAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#porter" class="md-nav__link">
    <span class="md-ellipsis">
      Porter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#salinas" class="md-nav__link">
    <span class="md-ellipsis">
      Salinas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#stephens" class="md-nav__link">
    <span class="md-ellipsis">
      Stephens
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tates" class="md-nav__link">
    <span class="md-ellipsis">
      TATES
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang" class="md-nav__link">
    <span class="md-ellipsis">
      Yang
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#amat" class="md-nav__link">
    <span class="md-ellipsis">
      aMAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastasset" class="md-nav__link">
    <span class="md-ellipsis">
      fastASSET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metacca" class="md-nav__link">
    <span class="md-ellipsis">
      metaCCA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metausatmetamanova" class="md-nav__link">
    <span class="md-ellipsis">
      metaUSAT/metaMANOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mvgwama" class="md-nav__link">
    <span class="md-ellipsis">
      mvGWAMA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Rare-variant
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Rare-variant">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#metaskat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metastaar" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSTAAR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#raremetal" class="md-nav__link">
    <span class="md-ellipsis">
      RareMETAL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smmat" class="md-nav__link">
    <span class="md-ellipsis">
      SMMAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Meta_and_Multi_triat</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#gwama">GWAMA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Mägi, R., &amp; Morris, A. P. (2010). GWAMA: software for genome-wide association meta-analysis. BMC bioinformatics, 11(1), 1-6.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mantra">MANTRA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Morris, A. P. (2011). Transethnic meta‐analysis of genomewide association studies. Genetic epidemiology, 35(8), 809-822.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metal">METAL</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Willer, C. J., Li, Y., &amp; Abecasis, G. R. (2010). METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26(17), 2190-2191.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mr-mega">MR-MEGA</a></td>
<td style="text-align: left;">Meta-analysis</td>
<td style="text-align: left;">Mägi, R., Horikoshi, M., Sofer, T., Mahajan, A., Kitajima, H., Franceschini, N., ... &amp; Morris, A. P. (2017). Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Human molecular genetics, 26(18), 3639-3650.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#asset">ASSET</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#factorgo">FactorGO</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Zhang, Z., Jung, J., Kim, A., Suboc, N., Gazal, S., &amp; Mancuso, N. (2023). A scalable variational approach to characterize pleiotropic components across thousands of human diseases and complex traits using GWAS summary statistics. medRxiv, 2023-03.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#galesloot">Galesloot</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Galesloot, T. E., van Steen, K., Kiemeney, L. A. L. M., Janss, L. L. &amp; Vermeulen, S. H. A comparison of multivariate genome-wide association methods. PLoS One 9, e95923 (2014).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#genomic-sem">Genomic-SEM</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Grotzinger, A. D., Rhemtulla, M., de Vlaming, R., Ritchie, S. J., Mallard, T. T., Hill, W. D., ... &amp; Tucker-Drob, E. M. (2019). Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nature human behaviour, 3(5), 513-525.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hipo">HIPO</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Qi, G. &amp; Chatterjee, N. Heritability informed power optimization (HIPO) leads to enhanced detection of genetic associations across multiple traits. PLoS Genet. 14, e1007549 (2018).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#jass">JASS</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lcp-gwas">LCP-GWAS</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Ruotsalainen, S. E. et al. An expanded analysis framework for multivariate GWAS connects inflammatory biomarkers to functional variants and disease. Eur. J. Hum. Genet. 29, 309–324 (2021).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#manova">MANOVA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Pillai, K. C. S. Some new test criteria in multivariate analysis. Ann. Math. Stat. 26, 117–121 (1955).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mostest">MOSTest</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">van der Meer, D. et al. Understanding the genetic determinants of the brain with MOSTest. Nat. Commun. 11, 3512 (2020).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mtag">MTAG</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Turley, P., Walters, R. K., Maghzian, O., Okbay, A., Lee, J. J., Fontana, M. A., ... &amp; Benjamin, D. J. (2018). Multi-trait analysis of genome-wide association summary statistics using MTAG. Nature genetics, 50(2), 229-237.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mv-plink-mqfam">MV-PLINK (MQFAM)</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Ferreira, M. A. R. &amp; Purcell, S. M. A multivariate test of association. Bioinformatics 25, 132–133 (2009).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multiphen">MultiPhen</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>O’Reilly, P. F. et al. MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS. PLoS One 7, e34861 (2012).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pchat">PCHAT</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Klei, L., Luca, D., Devlin, B. &amp; Roeder, K. Pleiotropy and principal components of heritability combine to increase power for association analysis. Genet. Epidemiol. 32, 9–19 (2008).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#porter">Porter</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Porter, H. F. &amp; O’Reilly, P. F. Multivariate simulation framework reveals performance of multi-trait GWAS methods. Sci. Rep. 7, 38837 (2017).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#salinas">Salinas</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Salinas, Y. D., Wang, Z. &amp; DeWan, A. T. Statistical analysis of multiple phenotypes in genetic epidemiologic studies: From cross-phenotype associations to pleiotropy. Am. J. Epidemiol. 187, 855–863 (2018).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#stephens">Stephens</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Stephens, M. A unified framework for association analysis with multiple related phenotypes. PLoS One 8, e65245 (2013).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tates">TATES</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">van der Sluis, S., Posthuma, D. &amp; Dolan, C. V. TATES: efficient multivariate genotype-phenotype analysis for genome-wide association studies. PLoS Genet. 9, e1003235 (2013).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#yang">Yang</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Yang, Q. &amp; Wang, Y. Methods for analyzing multivariate phenotypes in genetic association studies. J. Probab. Stat. 2012, 652569 (2012).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#amat">aMAT</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Wu, C. Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank. Genetics 215, 947–958 (2020).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastasset">fastASSET</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Qi, G. et al. Genome-wide large-scale multi-trait analysis characterizes global patterns of pleiotropy and unique trait-specific variants. Nat. Commun. 15, 6985 (2024).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metacca">metaCCA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Cichonska, A. et al. metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis. Bioinformatics 32, 1981–1989 (2016).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metausatmetamanova">metaUSAT/metaMANOVA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;"><br><br>Ray, D. &amp; Boehnke, M. Methods for meta-analysis of multiple traits using GWAS summary statistics. Genet. Epidemiol. 42, 134–145 (2018).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mvgwama">mvGWAMA</a></td>
<td style="text-align: left;">Multi-trait</td>
<td style="text-align: left;">Jansen, Iris E., et al. "Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer’s disease risk." Nature genetics 51.3 (2019): 404-413.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metaskat">MetaSKAT</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Lee, S., Teslovich, T. M., Boehnke, M., &amp; Lin, X. (2013). General framework for meta-analysis of rare variants in sequencing association studies. The American Journal of Human Genetics, 93(1), 42-53.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metastaar">MetaSTAAR</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Li, X., Quick, C., Zhou, H. et al. Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. Nat Genet (2022). https://doi.org/10.1038/s41588-022-01225-6</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#raremetal">RareMETAL</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Feng, S., Liu, D., Zhan, X., Wing, M. K., &amp; Abecasis, G. R. (2014). RAREMETAL: fast and powerful meta-analysis for rare variants. Bioinformatics, 30(19), 2828-2829.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#smmat">SMMAT</a></td>
<td style="text-align: left;">Rare-variant</td>
<td style="text-align: left;">Chen, H., Huffman, J. E., Brody, J. A., Wang, C., Lee, S., Li, Z., ... &amp; Hemostasis Working Group. (2019). Efficient variant set mixed model association tests for continuous and binary traits in large-scale whole-genome sequencing studies. The American Journal of Human Genetics, 104(2), 260-274.</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="meta-analysis">Meta-analysis</h2>
<h3 id="gwama">GWAMA</h3>
<ul>
<li><strong>NAME</strong> : GWAMA </li>
<li><strong>SHORT NAME</strong> : GWAMA </li>
<li><strong>FULL NAME</strong> : Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : Software tool for meta analysis of whole genome association data </li>
<li><strong>URL</strong> : <a href="https://genomics.ut.ee/en/tools">https://genomics.ut.ee/en/tools</a> </li>
<li><strong>CITATION</strong> : Mägi, R., &amp; Morris, A. P. (2010). GWAMA: software for genome-wide association meta-analysis. BMC bioinformatics, 11(1), 1-6. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20509871">20509871</a> </li>
</ul>
<h3 id="mantra">MANTRA</h3>
<ul>
<li><strong>NAME</strong> : MANTRA </li>
<li><strong>SHORT NAME</strong> : MANTRA </li>
<li><strong>FULL NAME</strong> : Meta-ANalysis of Transethnic Association studies </li>
<li><strong>KEYWORDS</strong> : cross-population </li>
<li><strong>CITATION</strong> : Morris, A. P. (2011). Transethnic meta‐analysis of genomewide association studies. Genetic epidemiology, 35(8), 809-822. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22125221">22125221</a> </li>
</ul>
<h3 id="metal">METAL</h3>
<ul>
<li><strong>NAME</strong> : METAL </li>
<li><strong>SHORT NAME</strong> : METAL </li>
<li><strong>FULL NAME</strong> : METAL </li>
<li><strong>DESCRIPTION</strong> : METAL is a tool for meta-analysis genomewide association scans. METAL can combine either (a) test statistics and standard errors or (b) p-values across studies (taking sample size and direction of effect into account). METAL analysis is a convenient alternative to a direct analysis of merged data from multiple studies. It is especially appropriate when data from the individual studies cannot be analyzed together because of differences in ethnicity, phenotype distribution, gender or constraints in sharing of individual level data imposed. Meta-analysis results in little or no loss of efficiency compared to analysis of a combined dataset including data from all individual studies. </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/METAL_Documentation">https://genome.sph.umich.edu/wiki/METAL_Documentation</a> </li>
<li><strong>CITATION</strong> : Willer, C. J., Li, Y., &amp; Abecasis, G. R. (2010). METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26(17), 2190-2191. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20616382">20616382</a> </li>
</ul>
<h3 id="mr-mega">MR-MEGA</h3>
<ul>
<li><strong>NAME</strong> : MR-MEGA </li>
<li><strong>SHORT NAME</strong> : MR-MEGA </li>
<li><strong>FULL NAME</strong> : Meta-Regression of Multi-AncEstry Genetic Association </li>
<li><strong>DESCRIPTION</strong> : MR-MEGA (Meta-Regression of Multi-AncEstry Genetic Association) is a tool to detect and fine-map complex trait association signals via multi-ancestry meta-regression.  This approach uses genome-wide metrics of diversity between populations to derive axes of genetic variation via multi-dimensional scaling [Purcell 2007].  Allelic effects of a variant across GWAS, weighted by their corresponding standard errors, can then be modelled in a linear regression framework, including the axes of genetic variation as covariates.  The flexibility of this model enables partitioning of the heterogeneity into components due to ancestry and residual variation, which would be expected to improve fine-mapping resolution. </li>
<li><strong>URL</strong> : <a href="https://genomics.ut.ee/en/tools">https://genomics.ut.ee/en/tools</a> </li>
<li><strong>KEYWORDS</strong> : cross-population, Meta-Regression </li>
<li><strong>CITATION</strong> : Mägi, R., Horikoshi, M., Sofer, T., Mahajan, A., Kitajima, H., Franceschini, N., ... &amp; Morris, A. P. (2017). Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Human molecular genetics, 26(18), 3639-3650. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28911207">28911207</a> </li>
</ul>
<h2 id="multi-trait">Multi-trait</h2>
<h3 id="asset">ASSET</h3>
<ul>
<li><strong>NAME</strong> : ASSET </li>
<li><strong>SHORT NAME</strong> : ASSET </li>
<li><strong>FULL NAME</strong> : association analysis based on subsets </li>
<li><strong>URL</strong> : <a href="https://github.com/sbstatgen/ASSET">https://github.com/sbstatgen/ASSET</a> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22560090">22560090</a> </li>
</ul>
<h3 id="factorgo">FactorGO</h3>
<ul>
<li><strong>NAME</strong> : FactorGO </li>
<li><strong>SHORT NAME</strong> : FactorGO </li>
<li><strong>FULL NAME</strong> : Factor analysis model in Genetic assOciation </li>
<li><strong>DESCRIPTION</strong> : FactorGo is a scalable variational factor analysis model that learns pleiotropic factors using GWAS summary statistics. </li>
<li><strong>URL</strong> : <a href="https://github.com/mancusolab/FactorGo">https://github.com/mancusolab/FactorGo</a> </li>
<li><strong>KEYWORDS</strong> : pleiotropy, factor analysis </li>
<li><strong>CITATION</strong> : Zhang, Z., Jung, J., Kim, A., Suboc, N., Gazal, S., &amp; Mancuso, N. (2023). A scalable variational approach to characterize pleiotropic components across thousands of human diseases and complex traits using GWAS summary statistics. medRxiv, 2023-03. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37879338">37879338</a> </li>
</ul>
<h3 id="galesloot">Galesloot</h3>
<ul>
<li><strong>NAME</strong> : Galesloot </li>
<li><strong>CITATION</strong> : Galesloot, T. E., van Steen, K., Kiemeney, L. A. L. M., Janss, L. L. &amp; Vermeulen, S. H. A comparison of multivariate genome-wide association methods. PLoS One 9, e95923 (2014). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24763738">24763738</a> </li>
</ul>
<h3 id="genomic-sem">Genomic-SEM</h3>
<ul>
<li><strong>NAME</strong> : Genomic-SEM </li>
<li><strong>SHORT NAME</strong> : Genomic-SEM </li>
<li><strong>FULL NAME</strong> : genomic structural equation modelling </li>
<li><strong>DESCRIPTION</strong> : R-package which allows the user to fit structural equation models based on the summary statistics obtained from genome wide association studies (GWAS). </li>
<li><strong>URL</strong> : <a href="https://github.com/GenomicSEM/GenomicSEM">https://github.com/GenomicSEM/GenomicSEM</a> </li>
<li><strong>KEYWORDS</strong> : SEM </li>
<li><strong>CITATION</strong> : Grotzinger, A. D., Rhemtulla, M., de Vlaming, R., Ritchie, S. J., Mallard, T. T., Hill, W. D., ... &amp; Tucker-Drob, E. M. (2019). Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nature human behaviour, 3(5), 513-525. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30962613">30962613</a> </li>
</ul>
<h3 id="hipo">HIPO</h3>
<ul>
<li><strong>NAME</strong> : HIPO </li>
<li><strong>SHORT NAME</strong> : HIPO </li>
<li><strong>FULL NAME</strong> : heritability informed power optimization </li>
<li><strong>DESCRIPTION</strong> : hipo is an R package that performs heritability informed power optimization (HIPO) for conducting multi-trait association analysis on summary level data. </li>
<li><strong>URL</strong> : <a href="https://github.com/gqi/hipo">https://github.com/gqi/hipo</a> </li>
<li><strong>CITATION</strong> : <br><br>Qi, G. &amp; Chatterjee, N. Heritability informed power optimization (HIPO) leads to enhanced detection of genetic associations across multiple traits. PLoS Genet. 14, e1007549 (2018).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30289880">30289880</a> </li>
</ul>
<h3 id="jass">JASS</h3>
<ul>
<li><strong>NAME</strong> : JASS </li>
<li><strong>SHORT NAME</strong> : JASS </li>
<li><strong>FULL NAME</strong> : Joint Analysis of Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : JASS is a python package that handles the computation of the joint statistics over sets of selected GWAS results, and the interactive exploration of the results through a web interface. The generation of joint statistics over a set of selected studies, and the generation of static plots to display the results, is easily performed using the command line interface. These functionalities can also be accessed through a web application embedded in the python package, which also enables the exploration of the results through a dynamic Javascript interface. The JASS analysis module handles the data processing, going from the import of the data up to the computation of the joint statistics and the generation of the various static plots to illustrate the results. However, we also briefly describe in the next section the pre-processing of raw GWAS data which can be performed through a companion script provided on behalf of the JASS package. </li>
<li><strong>URL</strong> : <a href="https://gitlab.pasteur.fr/statistical-genetics/jass">https://gitlab.pasteur.fr/statistical-genetics/jass</a> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32002517">32002517</a> </li>
</ul>
<h3 id="lcp-gwas">LCP-GWAS</h3>
<ul>
<li><strong>NAME</strong> : LCP-GWAS </li>
<li><strong>SHORT NAME</strong> : LCP-GWAS </li>
<li><strong>FULL NAME</strong> : Linear Combination Phenotype GWAS </li>
<li><strong>KEYWORDS</strong> : multivariate GWAS follow-up analyses </li>
<li><strong>CITATION</strong> : <br><br>Ruotsalainen, S. E. et al. An expanded analysis framework for multivariate GWAS connects inflammatory biomarkers to functional variants and disease. Eur. J. Hum. Genet. 29, 309–324 (2021).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33110245">33110245</a> </li>
</ul>
<h3 id="manova">MANOVA</h3>
<ul>
<li><strong>NAME</strong> : MANOVA </li>
<li><strong>SHORT NAME</strong> : MANOVA </li>
<li><strong>FULL NAME</strong> : multivariate analysis of variance </li>
<li><strong>CITATION</strong> : Pillai, K. C. S. Some new test criteria in multivariate analysis. Ann. Math. Stat. 26, 117–121 (1955). </li>
</ul>
<h3 id="mostest">MOSTest</h3>
<ul>
<li><strong>NAME</strong> : MOSTest </li>
<li><strong>SHORT NAME</strong> : MOSTest </li>
<li><strong>FULL NAME</strong> : Multivariate Omnibus Statistical Test </li>
<li><strong>DESCRIPTION</strong> : MOSTest is a tool for join genetical analysis of multiple traits, using multivariate analysis to boost the power of discovering associated loci. </li>
<li><strong>URL</strong> : <a href="https://github.com/precimed/mostest">https://github.com/precimed/mostest</a> </li>
<li><strong>CITATION</strong> : van der Meer, D. et al. Understanding the genetic determinants of the brain with MOSTest. Nat. Commun. 11, 3512 (2020). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32665545">32665545</a> </li>
</ul>
<h3 id="mtag">MTAG</h3>
<ul>
<li><strong>NAME</strong> : MTAG </li>
<li><strong>SHORT NAME</strong> : MTAG </li>
<li><strong>FULL NAME</strong> : Multi-Trait Analysis of GWAS </li>
<li><strong>DESCRIPTION</strong> : mtag is a Python-based command line tool for jointly analyzing multiple sets of GWAS summary statistics as described by Turley et. al. (2018). It can also be used as a tool to meta-analyze GWAS results. </li>
<li><strong>URL</strong> : <a href="https://github.com/JonJala/mtag">https://github.com/JonJala/mtag</a> </li>
<li><strong>KEYWORDS</strong> : Multi-trait </li>
<li><strong>CITATION</strong> : Turley, P., Walters, R. K., Maghzian, O., Okbay, A., Lee, J. J., Fontana, M. A., ... &amp; Benjamin, D. J. (2018). Multi-trait analysis of genome-wide association summary statistics using MTAG. Nature genetics, 50(2), 229-237. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29292387">29292387</a> </li>
</ul>
<h3 id="mv-plink-mqfam">MV-PLINK (MQFAM)</h3>
<ul>
<li><strong>NAME</strong> : MV-PLINK (MQFAM) </li>
<li><strong>SHORT NAME</strong> : MV-PLINK (MQFAM) </li>
<li><strong>CITATION</strong> : Ferreira, M. A. R. &amp; Purcell, S. M. A multivariate test of association. Bioinformatics 25, 132–133 (2009). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19019849">19019849</a> </li>
</ul>
<h3 id="multiphen">MultiPhen</h3>
<ul>
<li><strong>NAME</strong> : MultiPhen </li>
<li><strong>SHORT NAME</strong> : MultiPhen </li>
<li><strong>DESCRIPTION</strong> : Performs genetic association tests between SNPs (one-at-a-time) and multiple phenotypes (separately or in joint model). </li>
<li><strong>URL</strong> : <a href="https://cran.r-project.org/web/packages/MultiPhen/index.html">https://cran.r-project.org/web/packages/MultiPhen/index.html</a> </li>
<li><strong>CITATION</strong> : <br><br>O’Reilly, P. F. et al. MultiPhen: joint model of multiple phenotypes can increase discovery in GWAS. PLoS One 7, e34861 (2012).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22567092">22567092</a> </li>
</ul>
<h3 id="pchat">PCHAT</h3>
<ul>
<li><strong>NAME</strong> : PCHAT </li>
<li><strong>SHORT NAME</strong> : PCHAT </li>
<li><strong>FULL NAME</strong> : principal component of heritability association test </li>
<li><strong>CITATION</strong> : <br><br>Klei, L., Luca, D., Devlin, B. &amp; Roeder, K. Pleiotropy and principal components of heritability combine to increase power for association analysis. Genet. Epidemiol. 32, 9–19 (2008).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17922480">17922480</a> </li>
</ul>
<h3 id="porter">Porter</h3>
<ul>
<li><strong>NAME</strong> : Porter </li>
<li><strong>CITATION</strong> : Porter, H. F. &amp; O’Reilly, P. F. Multivariate simulation framework reveals performance of multi-trait GWAS methods. Sci. Rep. 7, 38837 (2017). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28287610">28287610</a> </li>
</ul>
<h3 id="salinas">Salinas</h3>
<ul>
<li><strong>NAME</strong> : Salinas </li>
<li><strong>CITATION</strong> : <br><br>Salinas, Y. D., Wang, Z. &amp; DeWan, A. T. Statistical analysis of multiple phenotypes in genetic epidemiologic studies: From cross-phenotype associations to pleiotropy. Am. J. Epidemiol. 187, 855–863 (2018).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29020254">29020254</a> </li>
</ul>
<h3 id="stephens">Stephens</h3>
<ul>
<li><strong>NAME</strong> : Stephens </li>
<li><strong>CITATION</strong> : Stephens, M. A unified framework for association analysis with multiple related phenotypes. PLoS One 8, e65245 (2013). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23861737">23861737</a> </li>
</ul>
<h3 id="tates">TATES</h3>
<ul>
<li><strong>NAME</strong> : TATES </li>
<li><strong>SHORT NAME</strong> : TATES </li>
<li><strong>FULL NAME</strong> : Trait-based Association Test that uses Extended Simes procedure </li>
<li><strong>CITATION</strong> : van der Sluis, S., Posthuma, D. &amp; Dolan, C. V. TATES: efficient multivariate genotype-phenotype analysis for genome-wide association studies. PLoS Genet. 9, e1003235 (2013). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23359524">23359524</a> </li>
</ul>
<h3 id="yang">Yang</h3>
<ul>
<li><strong>NAME</strong> : Yang </li>
<li><strong>CITATION</strong> : Yang, Q. &amp; Wang, Y. Methods for analyzing multivariate phenotypes in genetic association studies. J. Probab. Stat. 2012, 652569 (2012).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24748889">24748889</a> </li>
</ul>
<h3 id="amat">aMAT</h3>
<ul>
<li><strong>NAME</strong> : aMAT </li>
<li><strong>SHORT NAME</strong> : aMAT </li>
<li><strong>FULL NAME</strong> : adaptive multi-trait association test </li>
<li><strong>CITATION</strong> : Wu, C. Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank. Genetics 215, 947–958 (2020). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32540950">32540950</a> </li>
</ul>
<h3 id="fastasset">fastASSET</h3>
<ul>
<li><strong>NAME</strong> : fastASSET </li>
<li><strong>SHORT NAME</strong> : fastASSET </li>
<li><strong>URL</strong> : <a href="https://github.com/gqi/fastASSET">https://github.com/gqi/fastASSET</a> </li>
<li><strong>CITATION</strong> : <br><br>Qi, G. et al. Genome-wide large-scale multi-trait analysis characterizes global patterns of pleiotropy and unique trait-specific variants. Nat. Commun. 15, 6985 (2024).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39143063">39143063</a> </li>
</ul>
<h3 id="metacca">metaCCA</h3>
<ul>
<li><strong>NAME</strong> : metaCCA </li>
<li><strong>SHORT NAME</strong> : metaCCA </li>
<li><strong>FULL NAME</strong> : meta canonical
correlation analysis </li>
<li><strong>DESCRIPTION</strong> : metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness. </li>
<li><strong>URL</strong> : <a href="https://github.com/aalto-ics-kepaco/metaCCA-matlab">https://github.com/aalto-ics-kepaco/metaCCA-matlab</a> </li>
<li><strong>CITATION</strong> : <br><br>Cichonska, A. et al. metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis. Bioinformatics 32, 1981–1989 (2016).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27153689">27153689</a> </li>
</ul>
<h3 id="metausatmetamanova">metaUSAT/metaMANOVA</h3>
<ul>
<li><strong>NAME</strong> : metaUSAT/metaMANOVA </li>
<li><strong>SHORT NAME</strong> : metaUSAT/metaMANOVA </li>
<li><strong>FULL NAME</strong> : unified score-based association test </li>
<li><strong>DESCRIPTION</strong> : metaUSAT is a data-adaptive statistical approach for testing genetic associations of multiple traits from single/multiple studies using univariate GWAS summary statistics. This multivariate meta-analysis method can appropriately account for overlapping samples (if any) and can potentially test binary and/or continuous traits. </li>
<li><strong>URL</strong> : <a href="https://github.com/RayDebashree/metaUSAT">https://github.com/RayDebashree/metaUSAT</a> </li>
<li><strong>CITATION</strong> : <br><br>Ray, D. &amp; Boehnke, M. Methods for meta-analysis of multiple traits using GWAS summary statistics. Genet. Epidemiol. 42, 134–145 (2018).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29226385">29226385</a> </li>
</ul>
<h3 id="mvgwama">mvGWAMA</h3>
<ul>
<li><strong>NAME</strong> : mvGWAMA </li>
<li><strong>SHORT NAME</strong> : mvGWAMA </li>
<li><strong>FULL NAME</strong> : Multivariate Genome-Wide Association Meta-Analysis </li>
<li><strong>DESCRIPTION</strong> : mvGWAMA is a python script to perform a GWAS meta-analysis when there are sample overlap. </li>
<li><strong>URL</strong> : <a href="https://github.com/Kyoko-wtnb/mvGWAMA">https://github.com/Kyoko-wtnb/mvGWAMA</a> </li>
<li><strong>CITATION</strong> : Jansen, Iris E., et al. "Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer’s disease risk." Nature genetics 51.3 (2019): 404-413. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30617256">30617256</a> </li>
</ul>
<h2 id="rare-variant">Rare-variant</h2>
<h3 id="metaskat">MetaSKAT</h3>
<ul>
<li><strong>NAME</strong> : MetaSKAT </li>
<li><strong>SHORT NAME</strong> : MetaSKAT </li>
<li><strong>FULL NAME</strong> : MetaSKAT </li>
<li><strong>DESCRIPTION</strong> : MetaSKAT is a R package for multiple marker meta-analysis. It can carry out meta-analysis of SKAT, SKAT-O and burden tests with individual level genotype data or gene level summary statistics. </li>
<li><strong>URL</strong> : <a href="https://www.hsph.harvard.edu/skat/metaskat/">https://www.hsph.harvard.edu/skat/metaskat/</a> </li>
<li><strong>CITATION</strong> : Lee, S., Teslovich, T. M., Boehnke, M., &amp; Lin, X. (2013). General framework for meta-analysis of rare variants in sequencing association studies. The American Journal of Human Genetics, 93(1), 42-53. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23768515">23768515</a> </li>
</ul>
<h3 id="metastaar">MetaSTAAR</h3>
<ul>
<li><strong>NAME</strong> : MetaSTAAR </li>
<li><strong>SHORT NAME</strong> : MetaSTAAR </li>
<li><strong>FULL NAME</strong> : MetaSTAAR </li>
<li><strong>DESCRIPTION</strong> : MetaSTAAR is an R package for performing Meta-analysis of variant-Set Test for Association using Annotation infoRmation (MetaSTAAR) procedure in whole-genome sequencing (WGS) studies. MetaSTAAR enables functionally-informed rare variant meta-analysis of large WGS studies using an efficient, sparse matrix approach for storing summary statistic, while protecting data privacy of study participants and avoiding sharing subject-level data. MetaSTAAR accounts for relatedness and population structure of continuous and dichotomous traits, and boosts the power of rare variant meta-analysis by incorporating multiple variant functional annotations. </li>
<li><strong>URL</strong> : <a href="https://github.com/xihaoli/MetaSTAAR">https://github.com/xihaoli/MetaSTAAR</a> </li>
<li><strong>CITATION</strong> : Li, X., Quick, C., Zhou, H. et al. Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies. Nat Genet (2022). https://doi.org/10.1038/s41588-022-01225-6 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36564505">36564505</a> </li>
</ul>
<h3 id="raremetal">RareMETAL</h3>
<ul>
<li><strong>NAME</strong> : RareMETAL </li>
<li><strong>SHORT NAME</strong> : RareMETAL </li>
<li><strong>FULL NAME</strong> : RareMETAL </li>
<li><strong>DESCRIPTION</strong> : RAREMETAL is a program that facilitates the meta-analysis of rare variants from genotype arrays or sequencing (manuscript in preparation). </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/RAREMETAL">https://genome.sph.umich.edu/wiki/RAREMETAL</a> </li>
<li><strong>KEYWORDS</strong> : rare variants </li>
<li><strong>CITATION</strong> : Feng, S., Liu, D., Zhan, X., Wing, M. K., &amp; Abecasis, G. R. (2014). RAREMETAL: fast and powerful meta-analysis for rare variants. Bioinformatics, 30(19), 2828-2829. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24894501">24894501</a> </li>
</ul>
<h3 id="smmat">SMMAT</h3>
<ul>
<li><strong>NAME</strong> : SMMAT </li>
<li><strong>SHORT NAME</strong> : SMMAT </li>
<li><strong>FULL NAME</strong> : variant set mixed model association tests </li>
<li><strong>DESCRIPTION</strong> : For rare variant analysis from sequencing association studies, GMMAT performs the variant Set Mixed Model Association Tests (SMMAT) as proposed in Chen et al. (2019), including the burden test, the sequence kernel association test (SKAT), SKAT-O and an efficient hybrid test of the burden test and SKAT, based on user-defined variant sets. </li>
<li><strong>URL</strong> : <a href="https://github.com/hanchenphd/GMMAT">https://github.com/hanchenphd/GMMAT</a> </li>
<li><strong>CITATION</strong> : Chen, H., Huffman, J. E., Brody, J. A., Wang, C., Lee, S., Li, Z., ... &amp; Hemostasis Working Group. (2019). Efficient variant set mixed model association tests for continuous and binary traits in large-scale whole-genome sequencing studies. The American Journal of Human Genetics, 104(2), 260-274. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30639324">30639324</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>