
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.2.15">
    
    
      
        <title>Biobanks_Cohorts - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.c382b1dc.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cc9b2e1e.min.css">
        
          
          
          <meta name="theme-color" content="#4051b5">
        
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Biobanks_Cohorts
            
          </span>
        </div>
      </div>
    </div>
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-tabs__inner md-grid">
    <ul class="md-tabs__list">
      
        
  
  


  <li class="md-tabs__item">
    <a href=".." class="md-tabs__link">
      Home
    </a>
  </li>

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../Tools_README/" class="md-tabs__link">
        Tools
      </a>
    </li>
  

      
        
  
  
    
  


  
  
  
    <li class="md-tabs__item">
      <a href="../Sumstats_README/" class="md-tabs__link md-tabs__link--active">
        Sumstats
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../Reference_data_README/" class="md-tabs__link">
        Reference_data
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../Visualization_README/" class="md-tabs__link">
        Visualization
      </a>
    </li>
  

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../Population_Genetics_README/" class="md-tabs__link">
        Population_Genetics
      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Tools
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Tools" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Tools
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        Tools
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        Annotation
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_3" type="checkbox" id="__nav_2_3" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2_3">
          Association_tests
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Association_tests" data-md-level="2">
        <label class="md-nav__title" for="__nav_2_3">
          <span class="md-nav__icon md-icon"></span>
          Association_tests
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        Association_tests
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        RWAS
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        TWAS
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        eQTL
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        sQTL
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        Colocalization
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        Data_processing
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        Drug_discovery
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        Fine_mapping
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        Gene_prioritization
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        Gene_set_pathway_analysis
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        GxE_interactions
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        HLA
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        Heritability_and_genetic_correlation
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        Imputation
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        Mendelian_randomization
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        Meta_and_Multi_triat
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        Polygenic_risk_scores
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        Proteomics
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        Simulation
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        Tissue_and_single_cell
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_3">
          Sumstats
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Sumstats" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Sumstats
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_README/" class="md-nav__link">
        Sumstats
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3_2" type="checkbox" id="__nav_3_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_3_2">
          Biobanks_Cohorts
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Biobanks_Cohorts" data-md-level="2">
        <label class="md-nav__title" for="__nav_3_2">
          <span class="md-nav__icon md-icon"></span>
          Biobanks_Cohorts
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        Biobanks_Cohorts
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
    
  
  
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3_3" type="checkbox" id="__nav_3_3" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_3_3">
          Biobanks_Cohorts
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Biobanks_Cohorts" data-md-level="2">
        <label class="md-nav__title" for="__nav_3_3">
          <span class="md-nav__icon md-icon"></span>
          Biobanks_Cohorts
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Biobanks_Cohorts
      </a>
      
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-nav__link">
        GWAS_with_metabolomics
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_proteomics_README/" class="md-nav__link">
        GWAS_with_proteomics
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4" type="checkbox" id="__nav_4" >
      
      
      
      
        <label class="md-nav__link" for="__nav_4">
          Reference_data
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Reference_data" data-md-level="1">
        <label class="md-nav__title" for="__nav_4">
          <span class="md-nav__icon md-icon"></span>
          Reference_data
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_README/" class="md-nav__link">
        Reference_data
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Allele_Frequency_README/" class="md-nav__link">
        Allele_Frequency
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Epigenomics_README/" class="md-nav__link">
        Epigenomics
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Expression_README/" class="md-nav__link">
        Expression
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4_5" type="checkbox" id="__nav_4_5" >
      
      
      
      
        <label class="md-nav__link" for="__nav_4_5">
          Gene
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Gene" data-md-level="2">
        <label class="md-nav__title" for="__nav_4_5">
          <span class="md-nav__icon md-icon"></span>
          Gene
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_README/" class="md-nav__link">
        Gene
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_Ensembl_README/" class="md-nav__link">
        Ensembl
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_GENCODE_README/" class="md-nav__link">
        GENCODE
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_HGNC_README/" class="md-nav__link">
        HGNC
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_NCBI_refSeq_README/" class="md-nav__link">
        NCBI_refSeq
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4_5_6" type="checkbox" id="__nav_4_5_6" >
      
      
      
      
        <label class="md-nav__link" for="__nav_4_5_6">
          UCSC_refGene
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="UCSC_refGene" data-md-level="3">
        <label class="md-nav__title" for="__nav_4_5_6">
          <span class="md-nav__icon md-icon"></span>
          UCSC_refGene
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_README/" class="md-nav__link">
        UCSC_refGene
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/" class="md-nav__link">
        ucsc_genes
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_knownGenes_README/" class="md-nav__link">
        knownGenes
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4_6" type="checkbox" id="__nav_4_6" >
      
      
      
      
        <label class="md-nav__link" for="__nav_4_6">
          Genome
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Genome" data-md-level="2">
        <label class="md-nav__title" for="__nav_4_6">
          <span class="md-nav__icon md-icon"></span>
          Genome
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_README/" class="md-nav__link">
        Genome
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_hg38_README/" class="md-nav__link">
        hg38
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Phenotype_README/" class="md-nav__link">
        Phenotype
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Protein_README/" class="md-nav__link">
        Protein
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Variant_README/" class="md-nav__link">
        Variant
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_xQTL_README/" class="md-nav__link">
        xQTL
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5" type="checkbox" id="__nav_5" >
      
      
      
      
        <label class="md-nav__link" for="__nav_5">
          Visualization
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Visualization" data-md-level="1">
        <label class="md-nav__title" for="__nav_5">
          <span class="md-nav__icon md-icon"></span>
          Visualization
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        Visualization
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        Heatmap
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        Variants_on_protein
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Venn_diagram_README/" class="md-nav__link">
        Venn_diagram
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Workflow_README/" class="md-nav__link">
        Workflow
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_6" type="checkbox" id="__nav_6" >
      
      
      
      
        <label class="md-nav__link" for="__nav_6">
          Population_Genetics
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Population_Genetics" data-md-level="1">
        <label class="md-nav__title" for="__nav_6">
          <span class="md-nav__icon md-icon"></span>
          Population_Genetics
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        Population_Genetics
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        Admixture
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        Phylogenetic_tree
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                


  <h1>Biobanks_Cohorts</h1>

<table>
<thead>
<tr>
<th align="left">BIOBANK&amp;COHORT</th>
<th align="left">CONTINENT</th>
<th align="left">LOCATION</th>
<th align="left">POPULATION</th>
<th align="left">SAMPLE SIZE</th>
<th align="left">GENOTYPED SAMPLE SIZE</th>
<th align="left">CITATION</th>
<th align="left">URL</th>
<th align="left">DESCRIPTION</th>
<th align="left">NOTE</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left">UK Biobank (UKB)</td>
<td align="left">EUROPE</td>
<td align="left">U.K.</td>
<td align="left"><NA></td>
<td align="left">~500k</td>
<td align="left"><NA></td>
<td align="left">Bycroft, C., Freeman, C., Petkova, D., Band, G., Elliott, L. T., Sharp, K., ... &amp; Marchini, J. (2018). The UK Biobank resource with deep phenotyping and genomic data. Nature, 562(7726), 203-209.</td>
<td align="left">https://www.ukbiobank.ac.uk/</td>
<td align="left">UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">FinnGen</td>
<td align="left">EUROPE</td>
<td align="left">Finland</td>
<td align="left"><NA></td>
<td align="left">~343k</td>
<td align="left"><NA></td>
<td align="left">Kurki, M. I., Karjalainen, J., Palta, P., Sipilä, T. P., Kristiansson, K., Donner, K., ... &amp; Nelis, M. (2022). FinnGen: Unique genetic insights from combining isolated population and national health register data. medRxiv.</td>
<td align="left">https://www.finngen.fi/en</td>
<td align="left">FinnGen study launched in Finland in the autumn of 2017 is a unique study that combines genome information with digital health care data. The FinnGen study is an unprecedented global research project representing one of the largest studies of this type. Project aims to improve human health through genetic research, and ultimately identify new therapeutic targets and diagnostics for treating numerous diseases. The collaborative nature of the project is exceptional compare to many ongoing studies, and all the partners are working closely together to ensure appropriate transparency, data security and ownership.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Estonian Biobank</td>
<td align="left">EUROPE</td>
<td align="left">Estonia</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Leitsalu, L., Haller, T., Esko, T., Tammesoo, M. L., Alavere, H., Snieder, H., ... &amp; Metspalu, A. (2015). Cohort profile: Estonian biobank of the Estonian genome center, university of Tartu. International journal of epidemiology, 44(4), 1137-1147.</td>
<td align="left">https://genomics.ut.ee/en/content/estonian-biobank</td>
<td align="left">The Estonian Biobank has established a population-based biobank of Estonia with a current cohort size of more than 200,000 individuals (genotyped with genome-wide arrays), reflecting the age, sex and geographical distribution of the adult Estonian population. Considering the fact that about 20% of Estonia's adult population has joined the programme, it is indeed a database that is very important for the development of medical science both domestically and internationally.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Lifelines</td>
<td align="left">EUROPE</td>
<td align="left">Netherlands</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Scholtens, S., Smidt, N., Swertz, M. A., Bakker, S. J., Dotinga, A., Vonk, J. M., ... &amp; Stolk, R. P. (2015). Cohort Profile: LifeLines, a three-generation cohort study and biobank. International journal of epidemiology, 44(4), 1172-1180.</td>
<td align="left">https://www.lifelines.nl/researcher</td>
<td align="left">Lifelines is a large, multigenerational cohort study that includes over 167,000 participants (10%) from the northern population of the Netherlands. We included participants from three generations, who are followed for at least 30 years, to obtain insight into healthy ageing. The aim of Lifelines is to be a resource for the national and international scientific community.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">HUNT</td>
<td align="left">EUROPE</td>
<td align="left">Norway</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Brumpton, B. M., Graham, S., Surakka, I., Skogholt, A. H., Løset, M., Fritsche, L. G., ... &amp; Willer, C. J. (2022). The HUNT Study: a population-based cohort for genetic research. Cell Genomics, 2(10), 100193.</td>
<td align="left">https://www.ntnu.edu/hunt/hunt-biobank</td>
<td align="left">HUNT Biobank is an established and modern research biobank with high-technology equipment for storage, analysis, sample handling and delivery of samples. Our samples satisfy high quality standards and are stored in accordance with the Data Inspectorates laws and regulations. HUNT Biobank engages in sample handling from The Nord-Trøndelag Health Study (HUNT), Cohort of Norway (CONOR), and can receive samples from other researchers and research projects for storage, analysis and processing of DNA. We do not store samples from private individuals.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Generation Scotland</td>
<td align="left">EUROPE</td>
<td align="left">Scotland</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Smith, B. H., Campbell, A., Linksted, P., Fitzpatrick, B., Jackson, C., Kerr, S. M., ... &amp; Morris, A. D. (2013). Cohort Profile: Generation Scotland: Scottish Family Health Study (GS: SFHS). The study, its participants and their potential for genetic research on health and illness. International journal of epidemiology, 42(3), 689-700.</td>
<td align="left">https://www.ed.ac.uk/generation-scotland</td>
<td align="left">Generation Scotland is a research study looking at the health and well-being of volunteers and their families. Generation Scotland combines responses to questionnaires of health and well-being from birth through life. We combine this with NHS health records and innovative laboratory science to understand health trajectories. We work closely with researchers and our volunteers to create a rich evidence base for understanding health. Through this rigorous, ethical and safe approach to research, we seek to enable meaningful change in public health.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">East London Genes &amp; Health</td>
<td align="left">EUROPE</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Finer, S., Martin, H. C., Khan, A., Hunt, K. A., MacLaughlin, B., Ahmed, Z., ... &amp; van Heel, D. A. (2020). Cohort Profile: East London Genes &amp; Health (ELGH), a community-based population genomics and health study in British Bangladeshi and British Pakistani people. International journal of epidemiology, 49(1), 20-21i.</td>
<td align="left">https://www.genesandhealth.org/</td>
<td align="left">Genes &amp; Health is a huge long-term study of 100,000 people of Bangladeshi and Pakistani origin. We will link genes with health records, to study disease and treatments. Some volunteers may be invited for further studies. We are inviting volunteers to take part in two regions of the UK: East London (East London Genes &amp; Health) and Bradford (Bradford Genes &amp; Health).</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">deCODE Genetics</td>
<td align="left">EUROPE</td>
<td align="left">Iceland</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://www.decode.com/</td>
<td align="left">deCODE leads the world in the discovery of genetic risk factors for common diseases. Our gene discovery engine is driven by our unique approach and resources, including detailed genetic and medical information on some 500,000 individuals from around the globe taking part in our discovery work and proprietary statistical algorithms and informatics tools for gathering, analyzing, visualizing and storing large amounts of data.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">The International Agency for Research on Cancer (IARC) Biobank (IBB)</td>
<td align="left">EUROPE</td>
<td align="left">France</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://ibb.iarc.fr/</td>
<td align="left">The IARC BioBank (IBB) is one of the largest, most varied and richest International collections of samples in the world. The Biobank is publicly funded, (approximately 60% of its budget is provided by IARC Participating States through the regular budget and the remainder is from research grants) and hosts over 50 different studies, led or coordinated by IARC scientists. The IBB contains both population-based collections from research projects focusing on gene-environment interactions (as in the European Prospective Investigation into Cancer and Nutrition (EPIC) study) and disease-based collections which focus on biomarkers (as in the International Head and Neck Cancer Epidemiology (INHANCE)). Study designs include case-series, prevalence studies, case-control and cohort studies, etc. The IBB contains 5.1 million biological samples from 562,000 individuals. 4 million of the samples are from the EPIC study (over 370,000 individuals) and about one million samples from other collections (close to 200,000 individuals). Most of the samples are body fluids, including plasma, serum and urine as well as extracted DNA samples.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Biobank Graz</td>
<td align="left">EUROPE</td>
<td align="left">Austria</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Huppertz, B., Bayer, M., Macheiner, T., &amp; Sargsyan, K. (2016). Biobank Graz: the hub for innovative biomedical research. Open journal of bioresources, 3(1).</td>
<td align="left">https://biobank.medunigraz.at/en/?link=http%3A%2F%2F169.254.169.254%2Flatest%2Fmeta-data%2F&amp;cHash=3b3a94b34935e2b8509a838b4a34b0eb</td>
<td align="left">Biobank Graz is one of the largest and most well-known clinical biobanks in the world. Around 20 million individual specimens of body fluids and human tissue are stored here. Biobank Graz allows access to these specimens and associated data for scientific research purposes. The common goal is to develop approaches to diagnosing and treating disease.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">INTERVAL Study</td>
<td align="left">EUROPE</td>
<td align="left">U.K.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Moore, C., Bolton, T., Walker, M., Kaptoge, S., Allen, D., Daynes, M., ... &amp; Thompson, S. G. (2016). Recruitment and representativeness of blood donors in the INTERVAL randomised trial assessing varying inter-donation intervals. Trials, 17(1), 1-12.</td>
<td align="left">https://www.intervalstudy.org.uk/</td>
<td align="left">Between June 2012 and June 2014, the INTERVAL study recruited about 25,000 men and about 25,000 women at NHS Blood and Transplant (NSHBT) blood donation centres across England. During the study participants are asked to give blood either at usual donation intervals or more frequently. Men donate every 12, 10 or 8 weeks and women every 16, 14 or 12 weeks.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Fenland Study</td>
<td align="left">EUROPE</td>
<td align="left">U.K.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">MRC Epidemiology Unit, University of Cambridge. Fenland Study. [Internet]. Cambridge (UK): MRC Epidemiology Unit; 2017; [cited 2017 July 8]. Available from: http://www.mrc-epid.cam.ac.uk/research/studies/fenland/.</td>
<td align="left">https://www.mrc-epid.cam.ac.uk/research/studies/fenland/</td>
<td align="left">The Fenland Study investigates the interaction between environmental and genetic factors in determining obesity, type 2 diabetes, and related metabolic disorders. These conditions are a considerable public health concern, but their causes and factors that predict who will be affected by them are not completely understood. What makes the Fenland Study unique is the level of detail it collects about the health and lifestyle of participants, and the objective measurement techniques used in the screening. The first phase of the Fenland Study is now complete, and we are now inviting participants who attended an initial Fenland Study visit between 2005 and 2015 to return for a second visit in Phase 2.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">China Kadoorie Biobank (CKB)</td>
<td align="left">ASIA</td>
<td align="left">U.K. and China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Chen, Z., Chen, J., Collins, R., Guo, Y., Peto, R., Wu, F., &amp; Li, L. (2011). China Kadoorie Biobank of 0.5 million people: survey methods, baseline characteristics and long-term follow-up. International journal of epidemiology, 40(6), 1652-1666.</td>
<td align="left">https://www.ckbiobank.org/</td>
<td align="left">The China Kadoorie Biobank is one of the world’s largest prospective cohort studies.  A long-term collaboration between the UK and China, it aims to generate reliable evidence about the lifestyle, environmental and genetic determinants of a wide range of common diseases that can inform disease prevention, risk prediction and treatment worldwide.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Westlake BioBank for Chinese (WBBC)</td>
<td align="left">ASIA</td>
<td align="left">China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Cong, P. K., Bai, W. Y., Li, J. C., Yang, M. Y., Khederzadeh, S., Gai, S. R., ... &amp; Zheng, H. F. (2022). Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project. Nature Communications, 13(1), 1-15.</td>
<td align="left">https://wbbc.westlake.edu.cn/</td>
<td align="left">The Westlake BioBank for Chinese (WBBC) cohort is a population-based prospective study with its major purpose to better understand the effect of genetic and environmental factors on growth and development from youngster to elderly. The dataset comprises a wide range of demographics and anthropometric measures, serological tests, physical activity, sleep quality, age at menarche and bone mineral density. WBBC is designed as a prospective cohort study and will recruit at least 100,000 Chinese samples. The pilot project of WBBC has recruited a total of 14,726 participants (4,751 males and 9,975 females) and the baseline survey was carried out from 2017 to 2019.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left">Zhu, X. W., Liu, K. Q., Wang, P. Y., Liu, J. Q., Chen, J. Y., Xu, X. J., ... &amp; Zheng, H. F. (2021). Cohort profile: the Westlake BioBank for Chinese (WBBC) pilot project. BMJ open, 11(6), e045564.</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr>
<td align="left">Han Chinese Genome Initiative (the Han100K Project, PGG.Han)</td>
<td align="left">ASIA</td>
<td align="left">China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Gao, Y., Zhang, C., Yuan, L., Ling, Y., Wang, X., Liu, C., ... &amp; Xu, S. (2020). PGG. Han: the Han Chinese genome database and analysis platform. Nucleic acids research, 48(D1), D971-D976.</td>
<td align="left">https://www.hanchinesegenomes.org/</td>
<td align="left">a reference panel of 114 783 Han Chinese individuals (the Han100K), with whole-genome deep-sequenced or high-density genome-wide single-nucleotide variants (SNVs) genotyped or imputed.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">The China Metabolic Analytics Project (ChinaMAP)</td>
<td align="left">ASIA</td>
<td align="left">China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Cao, Y., Li, L., Xu, M., Feng, Z., Sun, X., Lu, J., ... &amp; Wang, W. (2020). The ChinaMAP analytics of deep whole genome sequences in 10,588 individuals. Cell research, 30(9), 717-731.</td>
<td align="left">http://www.mbiobank.com/</td>
<td align="left">The ChinaMAP is based on three large-scale cohorts: The China Noncommunicable Disease Surveillance 2010, a nationally representative study with 150,000 participants; the Risk Evaluation of cAncers in Chinese diabeTic Individuals: a lONgitudinal (REACTION) study with 250,000 participants15 and the Community-based Cardiovascular Risk During Urbanization in Shanghai with 50,000 participants.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Chinese Millionome Database (CMDB)</td>
<td align="left">ASIA</td>
<td align="left">China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Li, Z., Jiang, X., Fang, M., Bai, Y., Liu, S., Huang, S., &amp; Jin, X. (2022). CMDB: the comprehensive population genome variation database of China. Nucleic Acids Research.</td>
<td align="left">https://db.cngb.org/cmdb/</td>
<td align="left">the largest and the most representative Chinese genome variation database to date. The CMDB database contains 9.04 million single nucleotide variants (SNVs) and the allele frequency information from low-coverage (0.06×–0.1×) WGS data of 141 431 unrelated healthy Chinese individuals.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left">Liu, S., Huang, S., Chen, F., Zhao, L., Yuan, Y., Francis, S. S., ... &amp; Xu, X. (2018). Genomic analyses from non-invasive prenatal testing reveal genetic associations, patterns of viral infections, and Chinese population history. Cell, 175(2), 347-359.</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr>
<td align="left">NyuWa genome resource</td>
<td align="left">ASIA</td>
<td align="left">China</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Zhang, P., Luo, H., Li, Y., Wang, Y., Wang, J., Zheng, Y., ... &amp; Han100K Initiative. (2021). NyuWa Genome resource: a deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Reports, 37(7), 110017.</td>
<td align="left">http://bigdata.ibp.ac.cn/NyuWa/</td>
<td align="left">NyuWa, or NüWa, is the mother goddess who was the creator of the human population in Chinese mythology. Here we presented the NyuWa genome resource based on high depth (median 26X) WGS of 2,999 Chinese individuals from 23 out of 34 administrative divisions in China. NyuWa Genome Resource present in this website mainly contains two parts as NyuWa Chinese Population Variant Database and NyuWa reference panel server.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Taiwan Biobank (TWB)</td>
<td align="left">ASIA</td>
<td align="left">China, Taiwan</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Feng, Y. C. A., Chen, C. Y., Chen, T. T., Kuo, P. H., Hsu, Y. H., Yang, H. I., ... &amp; Lin, Y. F. (2021). Taiwan Biobank: a rich biomedical research database of the Taiwanese population. medRxiv.</td>
<td align="left">https://www.twbiobank.org.tw/</td>
<td align="left">The Taiwan Biobank (TWB) is an ongoing prospective study of over 150,000 individuals aged 30-70 recruited from across Taiwan beginning in 2012. A comprehensive list of phenotypes was collected for each consented participant at recruitment and follow-up visits through structured interviews and physical measurements. Biomarkers and genetic data were also generated for all participants from blood and urine samples.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left">Feng, Y. C. A., Chen, C. Y., Chen, T. T., Kuo, P. H., Hsu, Y. H., Yang, H. I., ... &amp; Lin, Y. F. (2022). Taiwan Biobank: a rich biomedical research database of the Taiwanese population. Cell Genomics, 100197.</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr>
<td align="left">SG10K_Health</td>
<td align="left">ASIA</td>
<td align="left">Singapore</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Chan, Sock Hoai, et al. "Analysis of clinically relevant variants from ancestrally diverse Asian genomes." Nature communications 13.1 (2022): 1-15.</td>
<td align="left">https://www.npm.sg/collaborate/partners/sg10k/</td>
<td align="left">SG10K_Health is the headline project of the Singapore National Precision Medicine programme (NPM Phase I). Comprising 10,000 whole-genome sequences from healthy Chinese, Indian, and Malay consented volunteers. SG10K_Health involved a research collaboration across multiple institutions in Singapore, enabling the country to develop the necessary infrastructure and deep capabilities to process, store, and analyse genetic data at the population scale in a safe, secure, and rapid manner. SG10K_Health provides near complete assessment of common genetic variants in Singapore’s three major ethnic groups, which can be used by clinicians to better manage Asian patients with genetic disease and as a control data set to compare against disease studies. Work is ongoing to link the SG10K_Health genomic data to research traits (e.g., height, weight, blood pressure) and clinical records.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">BioBank Japan (BBJ)</td>
<td align="left">ASIA</td>
<td align="left">Japan</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Nagai, A., Hirata, M., Kamatani, Y., Muto, K., Matsuda, K., Kiyohara, Y., ... &amp; Kubo, M. (2017). Overview of the BioBank Japan Project: study design and profile. Journal of epidemiology, 27(Supplement_III), S2-S8.</td>
<td align="left">https://biobankjp.org/</td>
<td align="left">In 2003, BioBank Japan (BBJ) started developing one of the world’s largest disease biobanks, creating a foundation for research aimed at achieving medical care tailored to the individual traits of each patient. From a total of 260,000 patients representing 440,000 cases of 51 primarily multifactorial (common) diseases, BBJ has collected DNA, serum, medical records (clinical information), etc. with their consent. No less than 5,800 items of screened information are available for research, including the patients’ survival information, with 95% of the patients tracked over an average of 10 years. In addition to large-scale genomic analyses, omics analyses including whole genome sequencing and metabolome/proteome analyses have been performed on the DNA, serum and other biological samples collected, producing significant research findings. The genomic information acquired through the analyses continues to be used as data. The biological samples and data are widely distributed and used by researchers.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Tohoku Medical Megabank (TMM)</td>
<td align="left">ASIA</td>
<td align="left">Japan</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Kuriyama, S., Yaegashi, N., Nagami, F., Arai, T., Kawaguchi, Y., Osumi, N., ... &amp; Tohoku Medical Megabank Project Study Group. (2016). The Tohoku medical megabank project: design and mission. Journal of epidemiology, 26(9), 493-511.</td>
<td align="left">https://www.megabank.tohoku.ac.jp/english/</td>
<td align="left">Tohoku University Tohoku Medical Megabank Organization was founded to establish an advanced medical system to foster the reconstruction from the Great East Japan Earthquake. The organization has been developing a biobank that combines medical and genome information during the process of rebuilding the community medical system and supporting health and welfare in the Tohoku area. The information from the brand-new biobank will create a new medical system, and, based on the findings of its analysis, the organization aims to attract more medical practitioners from all over the country to the area, promote industry-academic partnerships, create employment in related fields, and restore the medical system in Tohoku.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">National Center Biobank Network</td>
<td align="left">ASIA</td>
<td align="left">Japan</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Omae, Yosuke, Yu-ichi Goto, and Katsushi Tokunaga. "National Center Biobank Network." Human Genome Variation 9.1 (2022): 1-6.</td>
<td align="left">https://ncbiobank.org/en/home.php</td>
<td align="left">Six National Centers in Japan conduct specialized medical research under the coordination of the National Center Biobank Network (NCBN) and develop therapeutics to improve and protect national health. They actively collaborate to establish a shared biobank and are developing a structure to facilitate industry-academia-government cooperation regarding bioresources through broad joint research. NCBN strives to promote the success of the National Centers and to create bright future for health and human life.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">National Biobank of Korea</td>
<td align="left">ASIA</td>
<td align="left">Korea</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Cho, S. Y., Hong, E. J., Nam, J. M., Han, B., Chu, C., &amp; Park, O. (2012). Opening of the national biobank of Korea as the infrastructure of future biomedical science in Korea. Osong public health and research perspectives, 3(3), 177-184.</td>
<td align="left">https://nih.go.kr/NIH/cms/content/eng/14/65714_view.html</td>
<td align="left">The NBK is the national control center for the collection, management, and utilization of human bioresources in Korea. And NBK manages KBN, it contributes to the development of policies related to human bioresources, standardization of human bioresource management, and advancement of domestic biobanks through developing and providing support for human bioresource technologies. For guaranteeing the fairness in bioresource distribution and development of an efficient distribution system, the NBK also serves as the human bioresource supply hub that supports national healthcare and medical R&amp;D.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left">Nam, K., Kim, J., &amp; Lee, S. (2022). Genome-wide study on 72,298 individuals in Korean biobank data for 76 traits. Cell Genomics, 100189.</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr>
<td align="left">Qatar Biobank</td>
<td align="left">ASIA</td>
<td align="left">Qatar</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Al Kuwari, H., Al Thani, A., Al Marri, A., Al Kaabi, A., Abderrahim, H., Afifi, N., ... &amp; Elliott, P. (2015). The Qatar Biobank: background and methods. BMC public health, 15(1), 1-9.</td>
<td align="left">: https://www.qatarbiobank.org.qa/</td>
<td align="left">KoGES, part of the National Biobank of Korea, is a prospective cohort study with a comprehensive range of phenotypic measures and biological samples, such as DNA, serum, plasma, and urine, collected on approximately 210,000 individuals. KoGES includes the community-based Ansan and Ansung study, the urban community-based health examinee study, and the rural community-based cardiovascular disease association study.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">The Malaysian Cohort (TMC)</td>
<td align="left">ASIA</td>
<td align="left">Malaysia</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Jamal, R., Syed Zakaria, S. Z., Kamaruddin, M. A., Abd Jalal, N., Ismail, N., Mohd Kamil, N., ... &amp; Malaysian Cohort Study Group. (2015). Cohort profile: The Malaysian Cohort (TMC) project: a prospective study of non-communicable diseases in a multi-ethnic population. International journal of epidemiology, 44(2), 423-431.</td>
<td align="left">https://www.ukm.my/mycohort/ms/</td>
<td align="left">Qatar Biobank, a center within Qatar Foundation, was created in collaboration with Hamad Medical Corporation and the Ministry of Public Health to enable local scientists to conduct medical research on prevalent health issues in Qatar.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">IndiGenomes</td>
<td align="left">ASIA</td>
<td align="left">India</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Jain, A., Bhoyar, R. C., Pandhare, K., Mishra, A., Sharma, D., Imran, M., ... &amp; Sivasubbu, S. (2021). IndiGenomes: a comprehensive resource of genetic variants from over 1000 Indian genomes. Nucleic acids research, 49(D1), D1225-D1232.</td>
<td align="left">http://clingen.igib.res.in/indigen/</td>
<td align="left">The Malaysian Cohort study was initiated in 2005 by the Malaysian government. The top-down approach to this population-based cohort study ensured the allocation of sufficient funding for the project which aimed to recruit 100 000 individuals aged 35–70 years. Participants were recruited from rural and urban areas as well as from various socioeconomic groups. The main objectives of the study were to identify risk factors, to study gene-environment interaction and to discover biomarkers for the early detection of cancers and other diseases.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Uganda Genome Resource</td>
<td align="left">AFRICA</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Gurdasani, D., Carstensen, T., Fatumo, S., Chen, G., Franklin, C. S., Prado-Martinez, J., ... &amp; Sandhu, M. S. (2019). Uganda genome resource enables insights into population history and genomic discovery in Africa. Cell, 179(4), 984-1002.</td>
<td align="left">https://ega-archive.org/studies/EGAS00001000545</td>
<td align="left">The IndiGenomes resource encompasses the genomic data from over 1000 whole genome sequences sequenced from across India as part of the IndiGen programme and represents diverse geographies and ethnicities. The resource provides access to over 55 million genetic variants comprising of single nucleotide variants and indels. The variants are systematically annotated according to the recent Genome Reference Consortium Human Build 38 (GRCh38). Clinically relevant annotations as well as allele frequencies from global populations have also been integrated.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Nigerian 100K Genome Project (coming soon)</td>
<td align="left">AFRICA</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Fatumo, S., Yakubu, A., Oyedele, O., Popoola, J., Attipoe, D. A., Eze-Echesi, G., ... &amp; Ene-Obong, A. (2022). Promoting the genomic revolution in Africa through the Nigerian 100K Genome Project. Nature Genetics, 54(5), 531-536.</td>
<td align="left">https://allofus.nih.gov/ , https://www.researchallofus.org/register/</td>
<td align="left">Genomic studies in African populations provide unique opportunities to understand disease aetiology, human genetic diversity and population history in a regional and a global context. To leverage the relative benefits of different strategies, we undertook a combined approach of genotyping and whole-genome sequencing (WGS) in a population-based study of 6,400 individuals from a geographically defined rural community in South-West Uganda. We present data from 4,778 individuals with genotypes for ~2.2 million SNPs from the Uganda GWAS resource (UGWAS), and sequence data on up to 1,978 individuals spanning 41.5M SNPs and 4.5M indels (UG2G); 343 individuals overlap between the two datasets. We highlight the value of the largest sequence panel from Africa to date as a global resource for variant discovery, imputation and understanding the mutational spectrum and its clinical relevance in African populations. Alongside phenotype data, we provide a rich new genomic resource for researchers in Africa and globally</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">All of Us (AoU)</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Kathiresan, N., Cho, S. M. J., Bhattacharya, R., Truong, B., Hornsby, W., &amp; Natarajan, P. (2022). Representation of Race and Ethnicity in a Contemporary US Health Cohort: The All of Us Research Program. medRxiv.</td>
<td align="left">https://precisionhealth.umich.edu/our-research/michigangenomics/</td>
<td align="left">The All of Us Research Program is a historic effort to collect and study data from one million or more people living in the United States. The goal of the program is better health for all of us. The program began national enrollment in 2018 and is expected to last at least 10 years.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Michigan Genomics Initiative</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Zawistowski, M., Fritsche, L. G., Pandit, A., Vanderwerff, B., Patil, S., Scmidt, E. M., ... &amp; Zoellner, S. (2021). The Michigan Genomics Initiative: a biobank linking genotypes and electronic clinical records in Michigan Medicine patients. medRxiv.</td>
<td align="left">https://pmbb.med.upenn.edu/</td>
<td align="left">The Michigan Genomics Initiative (MGI) is a collaborative research effort among physicians, researchers, and patients at the University of Michigan (U-M) with the goal of combining patient electronic health record (EHR) data with corresponding genetic data to gain novel biomedical insights. There are currently ~84K consented participants through the MGI and partner studies and the addition of ~10K new participants per year is anticipated. Currently, all MGI participants with available genetic data have received care at the University of Michigan Health System.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Penn Medicine Biobank</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://www.uclahealth.org/precision-health/programs/ucla-atlas-community-health-initiative/ucla-atlas-precision-health-biobank</td>
<td align="left">The Penn Medicine BioBank (PMBB) is a research program created to study the causes and treatments of many diseases. Any Penn Medicine patient (age 18 and up) can sign up. The PMBB is a collection of biological samples, such as blood or tissue, that are donated by patient volunteers. These samples are then connected to clinical information, such as diseases or lab measures. These data are then used by researchers to discover new ways to detect, treat, and maybe even prevent or cure disease. Some of these studies may be about how genes affect health and disease. Other studies look at how genes affect response to medicines.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">UCLA Precision Health Biobank</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Johnson, R. D., Ding, Y., Bhattacharya, A., Chiu, A., Lajonchere, C., Geschwind, D. H., &amp; Pasaniuc, B. (2022). The UCLA ATLAS Community Health Initiative: promoting precision health research in a diverse biobank. medRxiv.</td>
<td align="left">https://icahn.mssm.edu/research/ipm/programs/biome-biobank</td>
<td align="left">The UCLA ATLAS Precision Health Biobank, under the supervision of the Translational Pathology Core Laboratory (TCPL), collects biological samples from patients who have consented to participate in the UCLA ATLAS Community Health Initiative. As a collaborator with UCLA ATLAS Community Health Initiative, the UCLA ATLAS Precision Health Biobank manages the collection and distribution of biological samples by removing the personally identifiable information.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">BioMe</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Roden, D. M., Pulley, J. M., Basford, M. A., Bernard, G. R., Clayton, E. W., Balser, J. R., &amp; Masys, D. R. (2008). Development of a large‐scale de‐identified DNA biobank to enable personalized medicine. Clinical Pharmacology &amp; Therapeutics, 84(3), 362-369.</td>
<td align="left">https://www.vumc.org/dbmi/biovu</td>
<td align="left">The Institute for Personalized Medicine at the Icahn School of Medicine at Mount Sinai is leading the movement toward diagnosis and classification of disease according to the patient’s molecular profile. This approach accommodates differences at all possible levels of exposure (genome, environment, and lifestyle) and at all stages of the process, from prevention to post-treatment follow-up. At the center of this effort is BioMe, an electronic medical record-linked biobank that enables researchers to rapidly and efficiently conduct genetic, epidemiologic, molecular, and genomic studies on large collections of research specimens linked with medical information.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">BioVU</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://bbofa.org/</td>
<td align="left">Planning for BioVU began in mid-2004 and the first samples were collected in February 2007. Prior to collecting DNA samples, all aspects of the BioVU project were extensively tested. BioVU now accrues 500-1000 samples per week, totaling more than 275,000 DNA samples as of January 2022. Vanderbilt clinic patients may sign the BioVU Consent Form if they wish to donate their excess blood samples, or not sign the form if they do not wish to participate.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Biobank of the Americas</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://www.galatea.bio/#main-biobank</td>
<td align="left">Biobank consented samples with associated clinical data from diverse populations from throughout the United States and Latin America via healthcare and biopharma partnerships.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Colorado Center for Personalized Medicine</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://medschool.cuanschutz.edu/cobiobank</td>
<td align="left">Established in 2014 as a partnership between UCHealth and University of Colorado Anschutz Medical Campus, the Colorado Center for Personalized Medicine (CCPM) brings together multiple disciplines and institutions to uncover advancements in genomics that can improve diagnosis and treatment of disease, and identify more tailored approaches to population health management.To facilitate discoveries in personalized medicine, CCPM has created a Biobank that aims to be one of the largest academic medicine biospecimen repositories in the mountain and midwest regions of the U.S. The CCPM Biobank is able to link biospecimens and genotype information with patient health information from electronic medical records in an enterprise data warehouse (Health Data Compass) to support a broad range of research, operational, and clinical quality improvement agendas.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">CanPath - Ontario Health Study</td>
<td align="left">AMERICA</td>
<td align="left">Canada</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Kirsh, V. A., Skead, K., McDonald, K., Kreiger, N., Little, J., Menard, K., ... &amp; Awadalla, P. (2022). Cohort Profile: The Ontario Health Study (OHS). International Journal of Epidemiology.</td>
<td align="left">https://canpath.ca/cohort/ontario-health-study/</td>
<td align="left">The Ontario Health Study (OHS) is a resource for investigating the ways in which lifestyle, the environment and genetics affect people’s health. It is one of the regional cohorts that collectively form the Canadian Partnership for Tomorrow’s Health (CanPath)—a pan-Canadian cohort with &gt;330 000 participants. The linking of Canada’s rich collection of administrative health data with the cohort’s data represents a powerful means to disseminate high-quality, timely data.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Massachusetts General Brigham Biobank</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Boutin, N. T., Schecter, S. B., Perez, E. F., Tchamitchian, N. S., Cerretani, X. R., Gainer, V. S., ... &amp; Smoller, J. W. (2022). The Evolution of a Large Biobank at Mass General Brigham. Journal of Personalized Medicine, 12(8), 1323.</td>
<td align="left">https://www.massgeneralbrigham.org/en/research-and-innovation/participate-in-research/biobank</td>
<td align="left">The Mass General Brigham Biobank is a large research program designed to help researchers understand how people’s health is affected by their genes, lifestyle, and environment. By participating in the Mass General Brigham Biobank, you can help us better understand, treat, and even prevent the diseases that might affect your health and the health of future generations.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left">Castro, V. M., Gainer, V., Wattanasin, N., Benoit, B., Cagan, A., Ghosh, B., ... &amp; Murphy, S. N. (2022). The Mass General Brigham Biobank Portal: an i2b2-based data repository linking disparate and high-dimensional patient data to support multimodal analytics. Journal of the American Medical Informatics Association, 29(4), 643-651.</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr>
<td align="left">Million Veteran Program (MVP)</td>
<td align="left">AMERICA</td>
<td align="left">U.S.</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Gaziano, J. M., Concato, J., Brophy, M., Fiore, L., Pyarajan, S., Breeling, J., ... &amp; O'Leary, T. J. (2016). Million Veteran Program: A mega-biobank to study genetic influences on health and disease. Journal of clinical epidemiology, 70, 214-223.</td>
<td align="left">https://www.mvp.va.gov/pwa/</td>
<td align="left">The Million Veteran Program (MVP) is a national research program to learn how genes, lifestyle, and military exposures affect health and illness. Since launching in 2011, over 900,000 Veteran partners have joined one of the world's largest programs on genetics and health.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">Mexico City Prospective Study</td>
<td align="left">AMERICA</td>
<td align="left">Mexico</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">Ziyatdinov, A., Torres, J., Alegre-Diaz, J., Backman, J., Mbatchou, J., Turner, M., ... &amp; Tapia-Conyer, R. (2022). Genotyping, sequencing and analysis of 140,000 adults from the Mexico City Prospective Study. bioRxiv.</td>
<td align="left">https://www.ctsu.ox.ac.uk/research/prospective-blood-based-study-of-150-000-individuals-in-mexico</td>
<td align="left">Between 1998 and 2004, CTSU, in collaboration with the Mexican Ministry of Health, established a study in Mexico City, in which over 150,000 middle-aged adults (including 100,000 women and 50,000 men) provided information about their lifestyle and disease history, had physical measurements recorded (including weight, waist and hip circumference, blood pressure) and had a blood sample taken.</td>
<td align="left"><NA></td>
</tr>
<tr>
<td align="left">QIMR Berghofer - QIMR Biobank (QSkin and GenEpi)</td>
<td align="left">OCIENIA</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">~17k</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
<td align="left">https://genepi.qimr.edu.au/</td>
<td align="left"><NA></td>
<td align="left"><NA></td>
</tr>
</tbody>
</table>

              
            </article>
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    <nav class="md-footer__inner md-grid" aria-label="Footer">
      
        
        <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Biobanks_Cohorts" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Biobanks_Cohorts
            </div>
          </div>
        </a>
      
      
        
        <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-footer__link md-footer__link--next" aria-label="Next: GWAS_with_metabolomics" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              GWAS_with_metabolomics
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs"], "search": "../assets/javascripts/workers/search.2a1c317c.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.a6c66575.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
    
  </body>
</html>