
## OmicsPred portal (INTERVAL study)
- URL: https://www.omicspred.org/ 
- URL: https://www.omicspred.org/Scores/Olink/INTERVAL
- URL: https://www.omicspred.org/Scores/Somalogic/INTERVAL
- DESCRIPTION: OMICSPRED is a resource for predicting multi-omics data (proteomics, metabolomics, transcriptomics etc.) directly from genotypes. To do this, we use extensive multi-omics data to train genetic scores using machine learning. Here, you can explore and download the genetic scores for a wide range of biomolecular traits in human blood as well as the summary statistics of their associations with key traits and diseases in the UK Biobank.
- CITATION: Xu, Y., Ritchie, S. C., Liang, Y., Timmers, P. R., Pietzner, M., Lannelongue, L., ... & Inouye, M. (2023). An atlas of genetic scores to predict multi-omic traits. Nature, 1-9.

## Proteome PheWAS browser

- URL: http://www.epigraphdb.org/pqtl
- DESCRIPTION: This browser currently contains the Mendelian Randomization and sensitivity analyses results for 989 proteins and 225 traits, i.e. diseases and risk factors. To start using this browser, simply type a protein or trait name into the "search" field, for example, ADAM19 or Lung cancer. The full list of proteins can be found by following the link on top of the "search" field.
- CITATION: Zheng, J., Haberland, V., Baird, D., Walker, V., Haycock, P. C., Hurle, M. R., ... & Gaunt, T. R. (2020). Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases. Nature genetics, 52(10), 1122-1131.

## SCALLOP 

- URL: http://www.scallop-consortium.com/
- DESCRIPTION: The SCALLOP consortium (Systematic and Combined AnaLysis of Olink Proteins) is a collaborative framework for discovery and follow-up of genetic associations with proteins on the Olink Proteomics platform. To date, 35 PIs from 28 research institutions have joined the effort, which now comprises summary level data for more than 70,000 patients and controls from 45 cohort studies. SCALLOP welcomes new members.

## A Table of all published GWAS with proteomics

- URL : [http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/](http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/)
- CITATION: Suhre, K., Arnold, M., Bhagwat, A. M., Cotton, R. J., Engelke, R., Raffler, J., ... & Graumann, J. (2017). Connecting genetic risk to disease end points through the human blood plasma proteome. Nature communications, 8(1), 1-14.
- CITATION: Suhre, K., McCarthy, M. I., & Schwenk, J. M. (2021). Genetics meets proteomics: perspectives for large population-based studies. Nature Reviews Genetics, 22(1), 19-37.

## pGWAS server
- URL: https://metabolomics.helmholtz-muenchen.de/pgwas/
- DESCRIPTION: In our study, we performed a genome-wide association study with protein levels (pGWAS). Using a highly multiplexed, aptamer-based, affinity proteomics platform (SOMAscanâ„¢), we quantified levels of 1,124 proteins in blood plasma samples from 1,000 German individuals (KORA cohort) and 338 Arab or Asian individuals (QMDiab cohort). We identified 539 independent, genome-wide significant SNP-to-protein associations, which can be investigated using this webserver.
- CITATION: Suhre, K., Arnold, M., Bhagwat, A. M., Cotton, R. J., Engelke, R., Raffler, J., ... & Graumann, J. (2017). Connecting genetic risk to disease end points through the human blood plasma proteome. Nature communications, 8(1), 1-14.
