
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Sumstats_Biobanks_Cohorts_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Sumstats - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Sumstats
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="./" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#biobanks_cohorts" class="md-nav__link">
    <span class="md-ellipsis">
      Biobanks_Cohorts
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Biobanks_Cohorts">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biobank-japan-bbj-jenger" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Japan (BBJ) JENGER
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biobank-japan-bbj-phewebjp" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Japan (BBJ) Phewebjp
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biobank-russia" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Russia
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cartagene-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      CARTaGENE PheWeb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#china-kadoorie-biobank-ckb" class="md-nav__link">
    <span class="md-ellipsis">
      China Kadoorie Biobank (CKB)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finmetseq" class="md-nav__link">
    <span class="md-ellipsis">
      FinMetSeq
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-ukbb-meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      FinnGen-UKBB meta-analysis
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-10-december-18-2023" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 10 (December 18 2023)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-11-june-24-2024" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 11 (June 24 2024)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-4-november-30-2020" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 4 (November 30 2020)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-5-may-11-2021" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 5 (May 11 2021)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-6-january-24-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 6 (January 24 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-7-june-1-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 7 (June 1 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-8-dec-1-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 8 (Dec 1 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-9-may-11-2023" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 9 (May 11 2023)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#generation-scotland" class="md-nav__link">
    <span class="md-ellipsis">
      Generation Scotland
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#global-biobank" class="md-nav__link">
    <span class="md-ellipsis">
      Global Biobank
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koges-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      KoGES Pheweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koreanchip" class="md-nav__link">
    <span class="md-ellipsis">
      KoreanChip
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-1" class="md-nav__link">
    <span class="md-ellipsis">
      MGI 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-2" class="md-nav__link">
    <span class="md-ellipsis">
      MGI 2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-biouv" class="md-nav__link">
    <span class="md-ellipsis">
      MGI BioUV
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pan-ukb" class="md-nav__link">
    <span class="md-ellipsis">
      Pan-UKB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taiwan-biobank-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      Taiwan BioBank Pheweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tohoku-medical-megabank-tmm-jmorp" class="md-nav__link">
    <span class="md-ellipsis">
      Tohoku Medical Megabank (TMM) Jmorp
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-neale" class="md-nav__link">
    <span class="md-ellipsis">
      UKB Neale
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-topmed" class="md-nav__link">
    <span class="md-ellipsis">
      UKB TOPMed
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-exome" class="md-nav__link">
    <span class="md-ellipsis">
      UKB exome
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-fastgwa" class="md-nav__link">
    <span class="md-ellipsis">
      UKB fastgwa
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-fastgwa-glmm" class="md-nav__link">
    <span class="md-ellipsis">
      UKB fastgwa-glmm
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-gene-based-genebass" class="md-nav__link">
    <span class="md-ellipsis">
      UKB gene-based (Genebass)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-saige" class="md-nav__link">
    <span class="md-ellipsis">
      UKB saige
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortiums" class="md-nav__link">
    <span class="md-ellipsis">
      Consortiums
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortiums">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#diagram" class="md-nav__link">
    <span class="md-ellipsis">
      DIAGRAM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giant-genetic-investigation-of-anthropometric-traits" class="md-nav__link">
    <span class="md-ellipsis">
      GIANT (Genetic Investigation of ANthropometric Traits)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glgc-global-lipids-genetics-consortium" class="md-nav__link">
    <span class="md-ellipsis">
      GLGC (Global Lipids Genetics Consortium)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megastroke" class="md-nav__link">
    <span class="md-ellipsis">
      Megastroke
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgc-psychiatric-genomics-consortium" class="md-nav__link">
    <span class="md-ellipsis">
      PGC (Psychiatric Genomics Consortium)
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#institution" class="md-nav__link">
    <span class="md-ellipsis">
      Institution
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Institution">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cncr-ctglab" class="md-nav__link">
    <span class="md-ellipsis">
      CNCR CTGLAB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cnsgenomics" class="md-nav__link">
    <span class="md-ellipsis">
      CNSGENOMICS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cardiovascular-disease-knowledge-portal" class="md-nav__link">
    <span class="md-ellipsis">
      Cardiovascular Disease Knowledge Portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gwas-catalog" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS catalog
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opengwas" class="md-nav__link">
    <span class="md-ellipsis">
      OpenGWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#biobanks_cohorts" class="md-nav__link">
    <span class="md-ellipsis">
      Biobanks_Cohorts
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Biobanks_Cohorts">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biobank-japan-bbj-jenger" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Japan (BBJ) JENGER
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biobank-japan-bbj-phewebjp" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Japan (BBJ) Phewebjp
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#biobank-russia" class="md-nav__link">
    <span class="md-ellipsis">
      Biobank Russia
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cartagene-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      CARTaGENE PheWeb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#china-kadoorie-biobank-ckb" class="md-nav__link">
    <span class="md-ellipsis">
      China Kadoorie Biobank (CKB)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finmetseq" class="md-nav__link">
    <span class="md-ellipsis">
      FinMetSeq
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-ukbb-meta-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      FinnGen-UKBB meta-analysis
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-10-december-18-2023" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 10 (December 18 2023)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-11-june-24-2024" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 11 (June 24 2024)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-4-november-30-2020" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 4 (November 30 2020)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-5-may-11-2021" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 5 (May 11 2021)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-6-january-24-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 6 (January 24 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-7-june-1-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 7 (June 1 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-8-dec-1-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 8 (Dec 1 2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#finngen-9-may-11-2023" class="md-nav__link">
    <span class="md-ellipsis">
      Finngen 9 (May 11 2023)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#generation-scotland" class="md-nav__link">
    <span class="md-ellipsis">
      Generation Scotland
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#global-biobank" class="md-nav__link">
    <span class="md-ellipsis">
      Global Biobank
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koges-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      KoGES Pheweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koreanchip" class="md-nav__link">
    <span class="md-ellipsis">
      KoreanChip
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-1" class="md-nav__link">
    <span class="md-ellipsis">
      MGI 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-2" class="md-nav__link">
    <span class="md-ellipsis">
      MGI 2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mgi-biouv" class="md-nav__link">
    <span class="md-ellipsis">
      MGI BioUV
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pan-ukb" class="md-nav__link">
    <span class="md-ellipsis">
      Pan-UKB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taiwan-biobank-pheweb" class="md-nav__link">
    <span class="md-ellipsis">
      Taiwan BioBank Pheweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tohoku-medical-megabank-tmm-jmorp" class="md-nav__link">
    <span class="md-ellipsis">
      Tohoku Medical Megabank (TMM) Jmorp
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-neale" class="md-nav__link">
    <span class="md-ellipsis">
      UKB Neale
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-topmed" class="md-nav__link">
    <span class="md-ellipsis">
      UKB TOPMed
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-exome" class="md-nav__link">
    <span class="md-ellipsis">
      UKB exome
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-fastgwa" class="md-nav__link">
    <span class="md-ellipsis">
      UKB fastgwa
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-fastgwa-glmm" class="md-nav__link">
    <span class="md-ellipsis">
      UKB fastgwa-glmm
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-gene-based-genebass" class="md-nav__link">
    <span class="md-ellipsis">
      UKB gene-based (Genebass)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ukb-saige" class="md-nav__link">
    <span class="md-ellipsis">
      UKB saige
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortiums" class="md-nav__link">
    <span class="md-ellipsis">
      Consortiums
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortiums">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#diagram" class="md-nav__link">
    <span class="md-ellipsis">
      DIAGRAM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#giant-genetic-investigation-of-anthropometric-traits" class="md-nav__link">
    <span class="md-ellipsis">
      GIANT (Genetic Investigation of ANthropometric Traits)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glgc-global-lipids-genetics-consortium" class="md-nav__link">
    <span class="md-ellipsis">
      GLGC (Global Lipids Genetics Consortium)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megastroke" class="md-nav__link">
    <span class="md-ellipsis">
      Megastroke
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgc-psychiatric-genomics-consortium" class="md-nav__link">
    <span class="md-ellipsis">
      PGC (Psychiatric Genomics Consortium)
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#institution" class="md-nav__link">
    <span class="md-ellipsis">
      Institution
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Institution">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cncr-ctglab" class="md-nav__link">
    <span class="md-ellipsis">
      CNCR CTGLAB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cnsgenomics" class="md-nav__link">
    <span class="md-ellipsis">
      CNSGENOMICS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cardiovascular-disease-knowledge-portal" class="md-nav__link">
    <span class="md-ellipsis">
      Cardiovascular Disease Knowledge Portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gwas-catalog" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS catalog
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#opengwas" class="md-nav__link">
    <span class="md-ellipsis">
      OpenGWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Sumstats</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">MAIN_ANCESTRY</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#biobank-japan-bbj-jenger">Biobank Japan (BBJ) JENGER</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#biobank-japan-bbj-phewebjp">Biobank Japan (BBJ) Phewebjp</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#biobank-russia">Biobank Russia</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cartagene-pheweb">CARTaGENE PheWeb</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#china-kadoorie-biobank-ckb-">China Kadoorie Biobank (CKB)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finmetseq">FinMetSeq</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-ukbb-meta-analysis">FinnGen-UKBB meta-analysis</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-10-december-18-2023-">Finngen 10 (December 18 2023)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-11-june-24-2024-">Finngen 11 (June 24 2024)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-4-november-30-2020-">Finngen 4 (November 30 2020)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-5-may-11-2021-">Finngen 5 (May 11 2021)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-6-january-24-2022-">Finngen 6 (January 24 2022)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-7-june-1-2022-">Finngen 7 (June 1 2022)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-8-dec-1-2022-">Finngen 8 (Dec 1 2022)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finngen-9-may-11-2023-">Finngen 9 (May 11 2023)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#generation-scotland">Generation Scotland</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#global-biobank">Global Biobank</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">ALL</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#koges-pheweb">KoGES Pheweb</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#koreanchip">KoreanChip</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mgi-1">MGI 1</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mgi-2">MGI 2</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mgi-biouv">MGI BioUV</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pan-ukb">Pan-UKB</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#taiwan-biobank-pheweb">Taiwan BioBank Pheweb</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tohoku-medical-megabank-tmm-jmorp">Tohoku Medical Megabank (TMM) Jmorp</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EAS</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-neale">UKB Neale</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-topmed">UKB TOPMed</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-exome">UKB exome</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-fastgwa-glmm">UKB fastgwa-glmm</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-fastgwa">UKB fastgwa</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-gene-based-genebass-">UKB gene-based (Genebass)</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ukb-saige">UKB saige</a></td>
<td style="text-align: left;">Biobanks_Cohorts</td>
<td style="text-align: left;">EUR</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#diagram">DIAGRAM</a></td>
<td style="text-align: left;">Consortiums</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#giant-genetic-investigation-of-anthropometric-traits-">GIANT (Genetic Investigation of ANthropometric Traits)</a></td>
<td style="text-align: left;">Consortiums</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#glgc-global-lipids-genetics-consortium-">GLGC (Global Lipids Genetics Consortium)</a></td>
<td style="text-align: left;">Consortiums</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#megastroke">Megastroke</a></td>
<td style="text-align: left;">Consortiums</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgc-psychiatric-genomics-consortium-">PGC (Psychiatric Genomics Consortium)</a></td>
<td style="text-align: left;">Consortiums</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cncr-ctglab">CNCR CTGLAB</a></td>
<td style="text-align: left;">Institution</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cnsgenomics">CNSGENOMICS</a></td>
<td style="text-align: left;">Institution</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cardiovascular-disease-knowledge-portal">Cardiovascular Disease Knowledge Portal</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gwas-catalog">GWAS catalog</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#opengwas">OpenGWAS</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="biobanks_cohorts">Biobanks_Cohorts</h2>
<h3 id="biobank-japan-bbj-jenger">Biobank Japan (BBJ) JENGER</h3>
<ul>
<li><strong>NAME</strong> : Biobank Japan (BBJ) JENGER </li>
<li><strong>URL</strong> : <a href="http://jenger.riken.jp/result"> http://jenger.riken.jp/result</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#biobank-japan">biobank japan</a> </li>
</ul>
<h3 id="biobank-japan-bbj-phewebjp">Biobank Japan (BBJ) Phewebjp</h3>
<ul>
<li><strong>NAME</strong> : Biobank Japan (BBJ) Phewebjp </li>
<li><strong>URL</strong> : <a href="https://pheweb.jp/"> https://pheweb.jp/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#biobank-japan">biobank japan</a> </li>
</ul>
<h3 id="biobank-russia">Biobank Russia</h3>
<ul>
<li><strong>NAME</strong> : Biobank Russia </li>
<li><strong>URL</strong> : <a href="https://biobank.almazovcentre.ru/#">https://biobank.almazovcentre.ru/#</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Complex trait susceptibilities and population diversity in a sample of 4,145 Russians. </li>
<li><strong>CITATION</strong> : Usoltsev D, ...&amp;, Artomov M. (2024) Complex trait susceptibilities and population diversity in a sample of 4,145 Russians. Nat Commun, 15 (1) 6212. doi:10.1038/s41467-024-50304-1. PMID 39043636 </li>
<li><strong>MESH_MAJOR</strong> : Gene Frequency, Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Asian People, Cohort Studies, Eastern European People, Ethnicity, Female, Finland, Genetic Predisposition to Disease, Genetic Variation, Genetics, Population, Humans, Male, Multifactorial Inheritance, Phenotype, Polymorphism, Single Nucleotide, Russia, White People </li>
<li><strong>ABSTRACT</strong> : The population of Russia consists of more than 150 local ethnicities. The ethnic diversity and geographic origins, which extend from eastern Europe to Asia, make the population uniquely positioned to investigate the shared properties of inherited disease risks between European and Asian ancestries. We present the analysis of genetic and phenotypic data from a cohort of 4,145 individuals collected in three metro areas in western Russia. We show the presence of multiple admixed genetic ancestry clusters spanning from primarily European to Asian and high identity-by-descent sharing with the Finnish population. As a result, there was notable enrichment of Finnish-specific variants in Russia. We illustrate the utility of Russian-descent cohorts for discovery of novel population-specific genetic associations, as well as replication of previously identified associations that were thought to be population-specific in other cohorts. Finally, we provide access to a database of allele frequencies and GWAS results for 464 phenotypes. </li>
<li><strong>COPYRIGHT</strong> : © 2024. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-50304-1 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2024 ; 15 ; 1 ; 6212 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39043636">39043636</a> </li>
</ul>
<h3 id="cartagene-pheweb">CARTaGENE PheWeb</h3>
<ul>
<li><strong>NAME</strong> : CARTaGENE PheWeb </li>
<li><strong>URL</strong> : <a href="https://cerc-genomic-medicine.ca/pheweb/cartagene/"> https://cerc-genomic-medicine.ca/pheweb/cartagene/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#cartagene-biobank">cartagene biobank</a> </li>
</ul>
<h3 id="china-kadoorie-biobank-ckb">China Kadoorie Biobank (CKB)</h3>
<ul>
<li><strong>NAME</strong> : China Kadoorie Biobank (CKB) </li>
<li><strong>URL</strong> : <a href="https://pheweb.ckbiobank.org/"> https://pheweb.ckbiobank.org/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#china-kadoorie-biobank">china kadoorie biobank</a> </li>
</ul>
<h3 id="finmetseq">FinMetSeq</h3>
<ul>
<li><strong>NAME</strong> : FinMetSeq </li>
<li><strong>URL</strong> : <a href="https://pheweb.sph.umich.edu/FinMetSeq/"> https://pheweb.sph.umich.edu/FinMetSeq/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
</ul>
<h3 id="finngen-ukbb-meta-analysis">FinnGen-UKBB meta-analysis</h3>
<ul>
<li><strong>NAME</strong> : FinnGen-UKBB meta-analysis </li>
<li><strong>URL</strong> : <a href="https://public-metaresults-fg-ukbb.finngen.fi"> https://public-metaresults-fg-ukbb.finngen.fi</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="finngen-10-december-18-2023">Finngen 10 (December 18 2023)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 10 (December 18 2023) </li>
<li><strong>URL</strong> : <a href="https://r10.finngen.fi/">https://r10.finngen.fi/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-11-june-24-2024">Finngen 11 (June 24 2024)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 11 (June 24 2024) </li>
<li><strong>URL</strong> : <a href="https://r11.finngen.fi/">https://r11.finngen.fi/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-4-november-30-2020">Finngen 4 (November 30 2020)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 4 (November 30 2020) </li>
<li><strong>URL</strong> : <a href="https://r4.finngen.fi/about"> https://r4.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-5-may-11-2021">Finngen 5 (May 11 2021)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 5 (May 11 2021) </li>
<li><strong>URL</strong> : <a href="https://r5.finngen.fi/about"> https://r5.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-6-january-24-2022">Finngen 6 (January 24 2022)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 6 (January 24 2022) </li>
<li><strong>URL</strong> : <a href="https://r6.finngen.fi/about"> https://r6.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-7-june-1-2022">Finngen 7 (June 1 2022)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 7 (June 1 2022) </li>
<li><strong>URL</strong> : <a href="https://r7.finngen.fi/about"> https://r7.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-8-dec-1-2022">Finngen 8 (Dec 1 2022)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 8 (Dec 1 2022) </li>
<li><strong>URL</strong> : <a href="https://r8.finngen.fi/about">https://r8.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="finngen-9-may-11-2023">Finngen 9 (May 11 2023)</h3>
<ul>
<li><strong>NAME</strong> : Finngen 9 (May 11 2023) </li>
<li><strong>URL</strong> : <a href="https://r9.finngen.fi/about">https://r9.finngen.fi/about</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#finngen">Finngen</a> </li>
<li><strong>TITLE</strong> : FinnGen provides genetic insights from a well-phenotyped isolated population. </li>
<li><strong>CITATION</strong> : Kurki MI, ...&amp;, Palotie A. (2023) FinnGen provides genetic insights from a well-phenotyped isolated population. Nature, 613 (7944) 508-518. doi:10.1038/s41586-022-05473-8. PMID 36653562 </li>
<li><strong>MESH_MAJOR</strong> : Disease, Gene Frequency, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Estonia, Finland, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Meta-Analysis as Topic, Middle Aged, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Population isolates such as those in Finland benefit genetic research because deleterious alleles are often concentrated on a small number of low-frequency variants (0.1% ≤ minor allele frequency &lt; 5%). These variants survived the founding bottleneck rather than being distributed over a large number of ultrarare variants. Although this effect is well established in Mendelian genetics, its value in common disease genetics is less explored<sup>1,2</sup>. FinnGen aims to study the genome and national health register data of 500,000 Finnish individuals. Given the relatively high median age of participants (63 years) and the substantial fraction of hospital-based recruitment, FinnGen is enriched for disease end points. Here we analyse data from 224,737 participants from FinnGen and study 15 diseases that have previously been investigated in large genome-wide association studies (GWASs). We also include meta-analyses of biobank data from Estonia and the United Kingdom. We identified 30 new associations, primarily low-frequency variants, enriched in the Finnish population. A GWAS of 1,932 diseases also identified 2,733 genome-wide significant associations (893 phenome-wide significant (PWS), P &lt; 2.6 × 10<sup>-11</sup>) at 2,496 (771 PWS) independent loci with 807 (247 PWS) end points. Among these, fine-mapping implicated 148 (73 PWS) coding variants associated with 83 (42 PWS) end points. Moreover, 91 (47 PWS) had an allele frequency of &lt;5% in non-Finnish European individuals, of which 62 (32 PWS) were enriched by more than twofold in Finland. These findings demonstrate the power of bottlenecked populations to find entry points into the biology of common diseases through low-frequency, high impact variants. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-022-05473-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 613 ; 7944 ; 508-518 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36653562">36653562</a> </li>
</ul>
<h3 id="generation-scotland">Generation Scotland</h3>
<ul>
<li><strong>NAME</strong> : Generation Scotland </li>
<li><strong>URL</strong> : <a href="https://datashare.ed.ac.uk/handle/10283/844"> https://datashare.ed.ac.uk/handle/10283/844</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
</ul>
<h3 id="global-biobank">Global Biobank</h3>
<ul>
<li><strong>NAME</strong> : Global Biobank </li>
<li><strong>URL</strong> : <a href="http://results.globalbiobankmeta.org/">http://results.globalbiobankmeta.org/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : ALL </li>
<li><strong>TITLE</strong> : Global Biobank Meta-analysis Initiative: Powering genetic discovery across human disease. </li>
<li><strong>CITATION</strong> : Zhou W, ...&amp;, Neale BM. (2022) Global Biobank Meta-analysis Initiative: Powering genetic discovery across human disease. Cell Genom, 2 (10) 100192. doi:10.1016/j.xgen.2022.100192. PMID 36777996 </li>
<li><strong>ABSTRACT</strong> : Biobanks facilitate genome-wide association studies (GWASs), which have mapped genomic loci across a range of human diseases and traits. However, most biobanks are primarily composed of individuals of European ancestry. We introduce the Global Biobank Meta-analysis Initiative (GBMI)-a collaborative network of 23 biobanks from 4 continents representing more than 2.2 million consented individuals with genetic data linked to electronic health records. GBMI meta-analyzes summary statistics from GWASs generated using harmonized genotypes and phenotypes from member biobanks for 14 exemplar diseases and endpoints. This strategy validates that GWASs conducted in diverse biobanks can be integrated despite heterogeneity in case definitions, recruitment strategies, and baseline characteristics. This collaborative effort improves GWAS power for diseases, benefits understudied diseases, and improves risk prediction while also enabling the nomination of disease genes and drug candidates by incorporating gene and protein expression data and providing insight into the underlying biology of human diseases and traits. </li>
<li><strong>COPYRIGHT</strong> : © 2022. </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2022.100192 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom ; 2022 ; 2 ; 10 ; 100192 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36777996">36777996</a> </li>
</ul>
<h3 id="koges-pheweb">KoGES Pheweb</h3>
<ul>
<li><strong>NAME</strong> : KoGES Pheweb </li>
<li><strong>URL</strong> : <a href="https://koges.leelabsg.org/"> https://koges.leelabsg.org/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
</ul>
<h3 id="koreanchip">KoreanChip</h3>
<ul>
<li><strong>NAME</strong> : KoreanChip </li>
<li><strong>URL</strong> : <a href="https://www.koreanchip.org/downloads"> https://www.koreanchip.org/downloads</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
</ul>
<h3 id="mgi-1">MGI 1</h3>
<ul>
<li><strong>NAME</strong> : MGI 1 </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/MGI-freeze1/"> https://pheweb.org/MGI-freeze1/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
</ul>
<h3 id="mgi-2">MGI 2</h3>
<ul>
<li><strong>NAME</strong> : MGI 2 </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/MGI-freeze2/"> https://pheweb.org/MGI-freeze2/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
</ul>
<h3 id="mgi-biouv">MGI BioUV</h3>
<ul>
<li><strong>NAME</strong> : MGI BioUV </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/MGI-BioVU/"> https://pheweb.org/MGI-BioVU/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
</ul>
<h3 id="pan-ukb">Pan-UKB</h3>
<ul>
<li><strong>NAME</strong> : Pan-UKB </li>
<li><strong>URL</strong> : <a href="https://pan.ukbb.broadinstitute.org/"> https://pan.ukbb.broadinstitute.org/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="taiwan-biobank-pheweb">Taiwan BioBank Pheweb</h3>
<ul>
<li><strong>NAME</strong> : Taiwan BioBank Pheweb </li>
<li><strong>URL</strong> : <a href="https://taiwanview.twbiobank.org.tw/pheweb.php"> https://taiwanview.twbiobank.org.tw/pheweb.php</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#taiwan-biobank">taiwan biobank</a> </li>
</ul>
<h3 id="tohoku-medical-megabank-tmm-jmorp">Tohoku Medical Megabank (TMM) Jmorp</h3>
<ul>
<li><strong>NAME</strong> : Tohoku Medical Megabank (TMM) Jmorp </li>
<li><strong>URL</strong> : <a href="https://jmorp.megabank.tohoku.ac.jp/202109/gwas/"> https://jmorp.megabank.tohoku.ac.jp/202109/gwas/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#tohoku-medical-megabank">tohoku medical megabank</a> </li>
</ul>
<h3 id="ukb-neale">UKB Neale</h3>
<ul>
<li><strong>NAME</strong> : UKB Neale </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/UKB-Neale/"> https://pheweb.org/UKB-Neale/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="ukb-topmed">UKB TOPMed</h3>
<ul>
<li><strong>NAME</strong> : UKB TOPMed </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/UKB-TOPMed/"> https://pheweb.org/UKB-TOPMed/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="ukb-exome">UKB exome</h3>
<ul>
<li><strong>NAME</strong> : UKB exome </li>
<li><strong>URL</strong> : <a href="https://azphewas.com/"> https://azphewas.com/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="ukb-fastgwa">UKB fastgwa</h3>
<ul>
<li><strong>NAME</strong> : UKB fastgwa </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/resources/ukb_fastgwa/imp/"> https://yanglab.westlake.edu.cn/resources/ukb_fastgwa/imp/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="ukb-fastgwa-glmm">UKB fastgwa-glmm</h3>
<ul>
<li><strong>NAME</strong> : UKB fastgwa-glmm </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/resources/ukb_fastgwa/imp_binary/"> https://yanglab.westlake.edu.cn/resources/ukb_fastgwa/imp_binary/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h3 id="ukb-gene-based-genebass">UKB gene-based (Genebass)</h3>
<ul>
<li><strong>NAME</strong> : UKB gene-based (Genebass) </li>
<li><strong>URL</strong> : <a href="https://genebass.org/"> https://genebass.org/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Systematic single-variant and gene-based association testing of thousands of phenotypes in 394,841 UK Biobank exomes. </li>
<li><strong>CITATION</strong> : Karczewski KJ, ...&amp;, Neale BM. (2022) Systematic single-variant and gene-based association testing of thousands of phenotypes in 394,841 UK Biobank exomes. Cell Genom, 2 (9) 100168. doi:10.1016/j.xgen.2022.100168. PMID 36778668 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies have successfully discovered thousands of common variants associated with human diseases and traits, but the landscape of rare variations in human disease has not been explored at scale. Exome-sequencing studies of population biobanks provide an opportunity to systematically evaluate the impact of rare coding variations across a wide range of phenotypes to discover genes and allelic series relevant to human health and disease. Here, we present results from systematic association analyses of 4,529 phenotypes using single-variant and gene tests of 394,841 individuals in the UK Biobank with exome-sequence data. We find that the discovery of genetic associations is tightly linked to frequency and is correlated with metrics of deleteriousness and natural selection. We highlight biological findings elucidated by these data and release the dataset as a public resource alongside the Genebass browser for rapidly exploring rare-variant association results. </li>
<li><strong>COPYRIGHT</strong> : © 2022 The Author(s). </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2022.100168 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom ; 2022 ; 2 ; 9 ; 100168 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36778668">36778668</a> </li>
</ul>
<h3 id="ukb-saige">UKB saige</h3>
<ul>
<li><strong>NAME</strong> : UKB saige </li>
<li><strong>URL</strong> : <a href="https://pheweb.org/UKB-SAIGE/"> https://pheweb.org/UKB-SAIGE/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h2 id="consortiums">Consortiums</h2>
<h3 id="diagram">DIAGRAM</h3>
<ul>
<li><strong>NAME</strong> : DIAGRAM </li>
<li><strong>URL</strong> : <a href="https://www.diagram-consortium.org/downloads.html"> https://www.diagram-consortium.org/downloads.html</a> </li>
</ul>
<h3 id="giant-genetic-investigation-of-anthropometric-traits">GIANT (Genetic Investigation of ANthropometric Traits)</h3>
<ul>
<li><strong>NAME</strong> : GIANT (Genetic Investigation of ANthropometric Traits) </li>
<li><strong>URL</strong> : <a href="https://portals.broadinstitute.org/collaboration/giant/index.php/Main_Page"> https://portals.broadinstitute.org/collaboration/giant/index.php/Main_Page</a> </li>
</ul>
<h3 id="glgc-global-lipids-genetics-consortium">GLGC (Global Lipids Genetics Consortium)</h3>
<ul>
<li><strong>NAME</strong> : GLGC (Global Lipids Genetics Consortium) </li>
<li><strong>URL</strong> : <a href="http://csg.sph.umich.edu/willer/public/glgc-lipids2021/"> http://csg.sph.umich.edu/willer/public/glgc-lipids2021/</a> </li>
</ul>
<h3 id="megastroke">Megastroke</h3>
<ul>
<li><strong>NAME</strong> : Megastroke </li>
<li><strong>URL</strong> : <a href="https://www.megastroke.org/index.html"> https://www.megastroke.org/index.html</a> </li>
</ul>
<h3 id="pgc-psychiatric-genomics-consortium">PGC (Psychiatric Genomics Consortium)</h3>
<ul>
<li><strong>NAME</strong> : PGC (Psychiatric Genomics Consortium) </li>
<li><strong>URL</strong> : <a href="https://www.med.unc.edu/pgc/download-results/"> https://www.med.unc.edu/pgc/download-results/</a> </li>
</ul>
<h2 id="institution">Institution</h2>
<h3 id="cncr-ctglab">CNCR CTGLAB</h3>
<ul>
<li><strong>NAME</strong> : CNCR CTGLAB </li>
<li><strong>URL</strong> : <a href="https://cncr.nl/research/summary_statistics/">https://cncr.nl/research/summary_statistics/</a> </li>
</ul>
<h3 id="cnsgenomics">CNSGENOMICS</h3>
<ul>
<li><strong>NAME</strong> : CNSGENOMICS </li>
<li><strong>DESCRIPTION</strong> : Program in Complex Trait Genomics, IMB, The University of Queensland </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/content/data">https://cnsgenomics.com/content/data</a> </li>
</ul>
<h2 id="platform">Platform</h2>
<h3 id="cardiovascular-disease-knowledge-portal">Cardiovascular Disease Knowledge Portal</h3>
<ul>
<li><strong>NAME</strong> : Cardiovascular Disease Knowledge Portal </li>
<li><strong>URL</strong> : <a href="https://cvd.hugeamp.org/"> https://cvd.hugeamp.org/</a> </li>
<li><strong>TITLE</strong> : Cardiovascular Disease Knowledge Portal: A Community Resource for Cardiovascular Disease Research. </li>
<li><strong>CITATION</strong> : Costanzo MC, ...&amp;, Burtt NP. (2023) Cardiovascular Disease Knowledge Portal: A Community Resource for Cardiovascular Disease Research. Circ Genom Precis Med, 16 (6) e004181. doi:10.1161/CIRCGEN.123.004181. PMID 37814896 </li>
<li><strong>MESH_MAJOR</strong> : Cardiovascular Diseases, Myocardial Infarction </li>
<li><strong>MESH_MINOR</strong> : Community Resources, Epigenomics, Humans </li>
<li><strong>DOI</strong> : 10.1161/CIRCGEN.123.004181 </li>
<li><strong>JOURNAL_INFO</strong> : Circulation. Genomic and precision medicine ; Circ Genom Precis Med ; 2023 ; 16 ; 6 ; e004181 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37814896">37814896</a> </li>
</ul>
<h3 id="gwas-catalog">GWAS catalog</h3>
<ul>
<li><strong>NAME</strong> : GWAS catalog </li>
<li><strong>URL</strong> : <a href="https://www.ebi.ac.uk/gwas/">https://www.ebi.ac.uk/gwas/</a> </li>
<li><strong>TITLE</strong> : The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. </li>
<li><strong>CITATION</strong> : Sollis E, ...&amp;, Harris LW. (2023) The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. Nucleic Acids Res, 51 (D1) D977-D985. doi:10.1093/nar/gkac1010. PMID 36350656 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Knowledge Bases </li>
<li><strong>MESH_MINOR</strong> : Animals, DNA Copy Number Variations, Humans, Mice, National Human Genome Research Institute (U.S.), Phenotype, Polymorphism, Single Nucleotide, Software, United States </li>
<li><strong>ABSTRACT</strong> : The NHGRI-EBI GWAS Catalog (www.ebi.ac.uk/gwas) is a FAIR knowledgebase providing detailed, structured, standardised and interoperable genome-wide association study (GWAS) data to &gt;200 000 users per year from academic research, healthcare and industry. The Catalog contains variant-trait associations and supporting metadata for &gt;45 000 published GWAS across &gt;5000 human traits, and &gt;40 000 full P-value summary statistics datasets. Content is curated from publications or acquired via author submission of prepublication summary statistics through a new submission portal and validation tool. GWAS data volume has vastly increased in recent years. We have updated our software to meet this scaling challenge and to enable rapid release of submitted summary statistics. The scope of the repository has expanded to include additional data types of high interest to the community, including sequencing-based GWAS, gene-based analyses and copy number variation analyses. Community outreach has increased the number of shared datasets from under-represented traits, e.g. cancer, and we continue to contribute to awareness of the lack of population diversity in GWAS. Interoperability of the Catalog has been enhanced through links to other resources including the Polygenic Score Catalog and the International Mouse Phenotyping Consortium, refinements to GWAS trait annotation, and the development of a standard format for GWAS data. </li>
<li><strong>COPYRIGHT</strong> : © The Author(s) 2022. Published by Oxford University Press on behalf of Nucleic Acids Research. </li>
<li><strong>DOI</strong> : 10.1093/nar/gkac1010 </li>
<li><strong>JOURNAL_INFO</strong> : Nucleic acids research ; Nucleic Acids Res ; 2023 ; 51 ; D1 ; D977-D985 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36350656">36350656</a> </li>
</ul>
<h3 id="opengwas">OpenGWAS</h3>
<ul>
<li><strong>NAME</strong> : OpenGWAS </li>
<li><strong>URL</strong> : <a href="https://gwas.mrcieu.ac.uk/">https://gwas.mrcieu.ac.uk/</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2020.08.10.244293 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Elsworth, B., Lyon, M., Alexander, T., Liu, Y., Matthews, P., Hallett, J., ... &amp; Hemani, G. (2020). The MRC IEU OpenGWAS data infrastructure. BioRxiv, 2020-08. </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>