
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Sumstats_Metabolomics/">
      
      
        <link rel="next" href="../Tools/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>Proteomics - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Proteomics
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Biobanks/" class="md-tabs__link">
          
  
  Biobanks

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../References/" class="md-tabs__link">
          
  
  References

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Single_Cell/" class="md-tabs__link">
          
  
  Single_Cell

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Sumstats/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Tools/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Biobanks
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AFRICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AFRICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AMERICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AMERICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_ASIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ASIA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_EUROPE/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    EUROPE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_OCIENIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    OCIENIA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    References
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            References
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    References
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Epigenetics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Gene/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Genome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Phenotype/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Variant/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_scRNA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    scRNA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Single_Cell
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Framework/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Framework
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Harmonization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Harmonization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Network/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Network
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_QC/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    QC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Trajectory/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Trajectory
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" checked>
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_General/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    General
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Imaging/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Metabolomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comparison" class="md-nav__link">
    <span class="md-ellipsis">
      Comparison
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Comparison">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#eldjarn-gh-37794188" class="md-nav__link">
    <span class="md-ellipsis">
      Eldjarn GH-37794188
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortium" class="md-nav__link">
    <span class="md-ellipsis">
      Consortium
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortium">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      SCALLOP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#carland-c-37550624" class="md-nav__link">
    <span class="md-ellipsis">
      Carland C-37550624
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caron-b-35264221" class="md-nav__link">
    <span class="md-ellipsis">
      Caron B-35264221
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dhindsa-rs-37794183" class="md-nav__link">
    <span class="md-ellipsis">
      Dhindsa RS-37794183
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gilly-a-37778719" class="md-nav__link">
    <span class="md-ellipsis">
      Gilly A-37778719
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hansson-o-36504281" class="md-nav__link">
    <span class="md-ellipsis">
      Hansson O-36504281
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koprulu-m-36823471" class="md-nav__link">
    <span class="md-ellipsis">
      Koprulu M-36823471
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macdonald-dunlop" class="md-nav__link">
    <span class="md-ellipsis">
      Macdonald-Dunlop
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#said" class="md-nav__link">
    <span class="md-ellipsis">
      Said
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre-k-38412862" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre K-38412862
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun-bb-37794186" class="md-nav__link">
    <span class="md-ellipsis">
      Sun BB-37794186
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wang-39317738" class="md-nav__link">
    <span class="md-ellipsis">
      Wang-39317738
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu-f-36797296" class="md-nav__link">
    <span class="md-ellipsis">
      Xu F-36797296
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#krishna-c-39085222" class="md-nav__link">
    <span class="md-ellipsis">
      Krishna C-39085222
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#omicspred-portal" class="md-nav__link">
    <span class="md-ellipsis">
      OmicsPred portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proteome-phewas-browser" class="md-nav__link">
    <span class="md-ellipsis">
      Proteome PheWAS browser
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgwas-server" class="md-nav__link">
    <span class="md-ellipsis">
      pGWAS server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#post-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      Post-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Post-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deng-yt-39579765" class="md-nav__link">
    <span class="md-ellipsis">
      Deng YT-39579765
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-suhre-k-32860016" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Suhre K-32860016
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Admixture/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6_4" >
        
          
          <label class="md-nav__link" for="__nav_6_4" id="__nav_6_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_6_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6_4">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Functional_prediction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Functional_prediction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Genealogy/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Phylogenetic_tree/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Selection/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comparison" class="md-nav__link">
    <span class="md-ellipsis">
      Comparison
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Comparison">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#eldjarn-gh-37794188" class="md-nav__link">
    <span class="md-ellipsis">
      Eldjarn GH-37794188
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortium" class="md-nav__link">
    <span class="md-ellipsis">
      Consortium
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortium">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      SCALLOP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#carland-c-37550624" class="md-nav__link">
    <span class="md-ellipsis">
      Carland C-37550624
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caron-b-35264221" class="md-nav__link">
    <span class="md-ellipsis">
      Caron B-35264221
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dhindsa-rs-37794183" class="md-nav__link">
    <span class="md-ellipsis">
      Dhindsa RS-37794183
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gilly-a-37778719" class="md-nav__link">
    <span class="md-ellipsis">
      Gilly A-37778719
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hansson-o-36504281" class="md-nav__link">
    <span class="md-ellipsis">
      Hansson O-36504281
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koprulu-m-36823471" class="md-nav__link">
    <span class="md-ellipsis">
      Koprulu M-36823471
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macdonald-dunlop" class="md-nav__link">
    <span class="md-ellipsis">
      Macdonald-Dunlop
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#said" class="md-nav__link">
    <span class="md-ellipsis">
      Said
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre-k-38412862" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre K-38412862
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun-bb-37794186" class="md-nav__link">
    <span class="md-ellipsis">
      Sun BB-37794186
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wang-39317738" class="md-nav__link">
    <span class="md-ellipsis">
      Wang-39317738
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu-f-36797296" class="md-nav__link">
    <span class="md-ellipsis">
      Xu F-36797296
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#krishna-c-39085222" class="md-nav__link">
    <span class="md-ellipsis">
      Krishna C-39085222
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#omicspred-portal" class="md-nav__link">
    <span class="md-ellipsis">
      OmicsPred portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proteome-phewas-browser" class="md-nav__link">
    <span class="md-ellipsis">
      Proteome PheWAS browser
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgwas-server" class="md-nav__link">
    <span class="md-ellipsis">
      pGWAS server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#post-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      Post-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Post-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deng-yt-39579765" class="md-nav__link">
    <span class="md-ellipsis">
      Deng YT-39579765
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-suhre-k-32860016" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Suhre K-32860016
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Proteomics</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">PLATFORM</th>
<th style="text-align: left;">YEAR</th>
<th style="text-align: left;">TITLE</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#eldjarn-gh-37794188">Eldjarn GH-37794188</a></td>
<td style="text-align: left;">Comparison</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Large-scale plasma proteomics comparisons through genetics and disease associations</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#scallop">SCALLOP</a></td>
<td style="text-align: left;">Consortium</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#carland-c-37550624">Carland C-37550624</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#caron-b-35264221">Caron B-35264221</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#dhindsa-rs-37794183">Dhindsa RS-37794183</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Rare variant associations with plasma protein levels in the UK Biobank</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gilly-a-37778719">Gilly A-37778719</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Genome-wide meta-analysis of 92 cardiometabolic protein serum levels</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hansson-o-36504281">Hansson O-36504281</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">The genetic regulation of protein expression in cerebrospinal fluid</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#koprulu-m-36823471">Koprulu M-36823471</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Proteogenomic links to human metabolic diseases</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#macdonald-dunlop">Macdonald-Dunlop</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#said">Said</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#suhre-k-38412862">Suhre K-38412862</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2024</td>
<td style="text-align: left;">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sun-bb-37794186">Sun BB-37794186</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Plasma proteomic associations with genetics and health in the UK Biobank</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wang-39317738">Wang-39317738</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#xu-f-36797296">Xu F-36797296</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">MS</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#krishna-c-39085222">Krishna C-39085222</a></td>
<td style="text-align: left;">HLA-GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2024</td>
<td style="text-align: left;">The influence of HLA genetic variation on plasma protein expression</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#omicspred-portal">OmicsPred portal</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">An atlas of genetic scores to predict multi-omic traits</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#proteome-phewas-browser">Proteome PheWAS browser</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgwas-server">pGWAS server</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Connecting genetic risk to disease end points through the human blood plasma proteome</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#deng-yt-39579765">Deng YT-39579765</a></td>
<td style="text-align: left;">Post-GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2024</td>
<td style="text-align: left;">Atlas of the plasma proteome in health and disease in 53,026 adults</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-suhre-k-32860016">Review-Suhre K-32860016</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Genetics meets proteomics: perspectives for large population-based studies</td>
</tr>
</tbody>
</table>
<h2 id="comparison">Comparison</h2>
<h3 id="eldjarn-gh-37794188">Eldjarn GH-37794188</h3>
<ul>
<li><strong>NAME</strong> : Eldjarn GH-37794188 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_EUROPE/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Large-scale plasma proteomics comparisons through genetics and disease associations </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06563-x </li>
<li><strong>ABSTRACT</strong> : High-throughput proteomics platforms measuring thousands of proteins in plasma combined with genomic and phenotypic information have the power to bridge the gap between the genome and diseases. Here we performed association studies of Olink Explore 3072 data generated by the UK Biobank Pharma Proteomics Project1 on plasma samples from more than 50,000 UK Biobank participants with phenotypic and genotypic data, stratifying on British or Irish, African and South Asian ancestries. We compared the results with those of a SomaScan v4 study on plasma from 36,000 Icelandic people2, for 1,514 of whom Olink data were also available. We found modest correlation between the two platforms. Although cis protein quantitative trait loci were detected for a similar absolute number of assays on the two platforms (2,101 on Olink versus 2,120 on SomaScan), the proportion of assays with such supporting evidence for assay performance was higher on the Olink platform (72% versus 43%). A considerable number of proteins had genomic associations that differed between the platforms. We provide examples where differences between platforms may influence conclusions drawn from the integration of protein levels with the study of diseases. We demonstrate how leveraging the diverse ancestries of participants in the UK Biobank helps to detect novel associations and refine genomic location. Our results show the value of the information provided by the two most commonly used high-throughput proteomics platforms and demonstrate the differences between them that at times provides useful complementarity. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Eldjarn GH, Ferkingstad E, Lund SH, Helgason H, ...&amp;, Stefansson K. (2023) Large-scale plasma proteomics comparisons through genetics and disease associations Nature, 622 (7982) 348-358. doi:10.1038/s41586-023-06563-x. PMID 37794188 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 348-358 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794188">37794188</a> </li>
</ul>
<h2 id="consortium">Consortium</h2>
<h3 id="scallop">SCALLOP</h3>
<ul>
<li><strong>NAME</strong> : SCALLOP </li>
<li><strong>DESCRIPTION</strong> : The SCALLOP consortium (Systematic and Combined AnaLysis of Olink Proteins) is a collaborative framework for discovery and follow-up of genetic associations with proteins on the Olink Proteomics platform. To date, 35 PIs from 28 research institutions have joined the effort, which now comprises summary level data for more than 70,000 patients and controls from 45 cohort studies. SCALLOP welcomes new members. </li>
<li><strong>URL</strong> : <a href="http://www.scallop-consortium.com/">http://www.scallop-consortium.com/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_EUROPE/#uk-biobank">UK Biobank</a> </li>
</ul>
<h2 id="gwas">GWAS</h2>
<h3 id="carland-c-37550624">Carland C-37550624</h3>
<ul>
<li><strong>NAME</strong> : Carland C-37550624 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases </li>
<li><strong>DOI</strong> : 10.1186/s12014-023-09421-0 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: Human plasma contains a wide variety of circulating proteins. These proteins can be important clinical biomarkers in disease and also possible drug targets. Large scale genomics studies of circulating proteins can identify genetic variants that lead to relative protein abundance. METHODS: We conducted a meta-analysis on genome-wide association studies of autosomal chromosomes in 22,997 individuals of primarily European ancestry across 12 cohorts to identify protein quantitative trait loci (pQTL) for 92 cardiometabolic associated plasma proteins. RESULTS: We identified 503 (337 cis and 166 trans) conditionally independent pQTLs, including several novel variants not reported in the literature. We conducted a sex-stratified analysis and found that 118 (23.5%) of pQTLs demonstrated heterogeneity between sexes. The direction of effect was preserved but there were differences in effect size and significance. Additionally, we annotate trans-pQTLs with nearest genes and report plausible biological relationships. Using Mendelian randomization, we identified causal associations for 18 proteins across 19 phenotypes, of which 10 have additional genetic colocalization evidence. We highlight proteins associated with a constellation of cardiometabolic traits including angiopoietin-related protein 7 (ANGPTL7) and Semaphorin 3F (SEMA3F). CONCLUSION: Through large-scale analysis of protein quantitative trait loci, we provide a comprehensive overview of common variants associated with plasma proteins. We highlight possible biological relationships which may serve as a basis for further investigation into possible causal roles in cardiometabolic diseases. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Carland C, Png G, Malarstig A, Kho PF, ...&amp;, Assimes T. (2023) Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases Clin. Proteomics, 20 (1) 31. doi:10.1186/s12014-023-09421-0. PMID 37550624 </li>
<li><strong>JOURNAL_INFO</strong> : Clinical proteomics ; Clin. Proteomics ; 2023 ; 20 ; 1 ; 31 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37550624">37550624</a> </li>
</ul>
<h3 id="caron-b-35264221">Caron B-35264221</h3>
<ul>
<li><strong>NAME</strong> : Caron B-35264221 </li>
<li><strong>TITLE</strong> : Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes </li>
<li><strong>DOI</strong> : 10.1186/s13073-022-01032-y </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: Blood plasma proteins play an important role in immune defense against pathogens, including cytokine signaling, the complement system, and the acute-phase response. Recent large-scale studies have reported genetic (i.e., protein quantitative trait loci, pQTLs) and non-genetic factors, such as age and sex, as major determinants to inter-individual variability in immune response variation. However, the contribution of blood-cell composition to plasma protein heterogeneity has not been fully characterized and may act as a mediating factor in association studies. METHODS: Here, we evaluated plasma protein levels from 400 unrelated healthy individuals of western European ancestry, who were stratified by sex and two decades of life (20-29 and 60-69 years), from the Milieu Intrieur cohort. We quantified 229 proteins by Luminex in a clinically certified laboratory and their levels of variation were analyzed together with 5.2 million single-nucleotide polymorphisms. With respect to non-genetic variables, we included 254 lifestyle and biochemical factors, as well as counts of seven circulating immune cell populations measured by hemogram and standardized flow cytometry. RESULTS: Collectively, we found 152 significant associations involving 49 proteins and 20 non-genetic variables. Consistent with previous studies, age and sex showed a global, pervasive impact on plasma protein heterogeneity, while body mass index and other health status variables were among the non-genetic factors with the highest number of associations. After controlling for these covariates, we identified 100 and 12 pQTLs acting in cis and trans, respectively, collectively associated with 87 plasma proteins and including 19 novel genetic associations. Genetic factors explained the largest fraction of the variability of plasma protein levels, as compared to non-genetic factors. In addition, blood-cell fractions, including leukocytes, lymphocytes, monocytes, neutrophils, eosinophils, basophils, and platelets, had a larger contribution to inter-individual variability than age and sex and appeared as confounders of specific genetic associations. Finally, we identified new genetic associations with plasma protein levels of five monogenic Mendelian disease genes including two primary immunodeficiency genes (Ficolin-3 and FAS). CONCLUSIONS: Our study identified novel genetic and non-genetic factors associated to plasma protein levels which may inform health status and disease management. </li>
<li><strong>CITATION</strong> : Caron B, Patin E, Rotival M, Charbit B, ...&amp;, Milieu Intrieur Consortium. (2022) Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes Genome Med., 14 (1) 28. doi:10.1186/s13073-022-01032-y. PMID 35264221 </li>
<li><strong>JOURNAL_INFO</strong> : Genome medicine ; Genome Med. ; 2022 ; 14 ; 1 ; 28 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35264221">35264221</a> </li>
</ul>
<h3 id="dhindsa-rs-37794183">Dhindsa RS-37794183</h3>
<ul>
<li><strong>NAME</strong> : Dhindsa RS-37794183 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_EUROPE/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Rare variant associations with plasma protein levels in the UK Biobank </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06547-x </li>
<li><strong>ABSTRACT</strong> : Integrating human genomics and proteomics can help elucidate disease mechanisms, identify clinical biomarkers and discover drug targets1-4. Because previous proteogenomic studies have focused on common variation via genome-wide association studies, the contribution of rare variants to the plasma proteome remains largely unknown. Here we identify associations between rare protein-coding variants and 2,923 plasma protein abundances measured in 49,736 UK Biobank individuals. Our variant-level exome-wide association study identified 5,433 rare genotype-protein associations, of which 81% were undetected in a previous genome-wide association study of the same cohort5. We then looked at aggregate signals using gene-level collapsing analysis, which revealed 1,962 gene-protein associations. Of the 691 gene-level signals from protein-truncating variants, 99.4% were associated with decreased protein levels. STAB1 and STAB2, encoding scavenger receptors involved in plasma protein clearance, emerged as pleiotropic loci, with 77 and 41 protein associations, respectively. We demonstrate the utility of our publicly accessible resource through several applications. These include detailing an allelic series in NLRC4, identifying potential biomarkers for a fatty liver disease-associated variant in HSD17B13 and bolstering phenome-wide association studies by integrating protein quantitative trait loci with protein-truncating variants in collapsing analyses. Finally, we uncover distinct proteomic consequences of clonal haematopoiesis (CH), including an association between TET2-CH and increased FLT3 levels. Our results highlight a considerable role for rare variation in plasma protein abundance and the value of proteogenomics in therapeutic discovery. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Dhindsa RS, Burren OS, Sun BB, Prins BP, ...&amp;, Petrovski S. (2023) Rare variant associations with plasma protein levels in the UK Biobank Nature, 622 (7982) 339-347. doi:10.1038/s41586-023-06547-x. PMID 37794183 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 339-347 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794183">37794183</a> </li>
</ul>
<h3 id="gilly-a-37778719">Gilly A-37778719</h3>
<ul>
<li><strong>NAME</strong> : Gilly A-37778719 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Genome-wide meta-analysis of 92 cardiometabolic protein serum levels </li>
<li><strong>DOI</strong> : 10.1016/j.molmet.2023.101810 </li>
<li><strong>ABSTRACT</strong> : OBJECTIVES: Global cardiometabolic disease prevalence has grown rapidly over the years, making it the leading cause of death worldwide. Proteins are crucial components in biological pathways dysregulated in disease states. Identifying genetic components that influence circulating protein levels may lead to the discovery of biomarkers for early stages of disease or offer opportunities as therapeutic targets. METHODS: Here, we carry out a genome-wide association study (GWAS) utilising whole genome sequencing data in 3,005 individuals from the HELIC founder populations cohort, across 92 proteins of cardiometabolic relevance. RESULTS: We report 322 protein quantitative trait loci (pQTL) signals across 92 proteins, of which 76 are located in or near the coding gene (cis-pQTL). We link those association signals with changes in protein expression and cardiometabolic disease risk using colocalisation and Mendelian randomisation (MR) analyses. CONCLUSIONS: The majority of previously unknown signals we describe point to proteins or protein interactions involved in inflammation and immune response, providing genetic evidence for the contributing role of inflammation in cardiometabolic disease processes. </li>
<li><strong>CITATION</strong> : Gilly A, Park YC, Tsafantakis E, Karaleftheri M, ...&amp;, Zeggini E. (2023) Genome-wide meta-analysis of 92 cardiometabolic protein serum levels Mol. Metab., 78 () 101810. doi:10.1016/j.molmet.2023.101810. PMID 37778719 </li>
<li><strong>JOURNAL_INFO</strong> : Molecular metabolism ; Mol. Metab. ; 2023 ; 78 ; <NA> ; 101810 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37778719">37778719</a> </li>
</ul>
<h3 id="hansson-o-36504281">Hansson O-36504281</h3>
<ul>
<li><strong>NAME</strong> : Hansson O-36504281 </li>
<li><strong>TITLE</strong> : The genetic regulation of protein expression in cerebrospinal fluid </li>
<li><strong>DOI</strong> : 10.15252/emmm.202216359 </li>
<li><strong>ABSTRACT</strong> : Studies of the genetic regulation of cerebrospinal fluid (CSF) proteins may reveal pathways for treatment of neurological diseases. 398 proteins in CSF were measured in 1,591 participants from the BioFINDER study. Protein quantitative trait loci (pQTL) were identified as associations between genetic variants and proteins, with 176 pQTLs for 145 CSF proteins (P &lt; 1.25  10-10 , 117 cis-pQTLs and 59 trans-pQTLs). Ventricular volume (measured with brain magnetic resonance imaging) was a confounder for several pQTLs. pQTLs for CSF and plasma proteins were overall correlated, but CSF-specific pQTLs were also observed. Mendelian randomization analyses suggested causal roles for several proteins, for example, ApoE, CD33, and GRN in Alzheimer's disease, MMP-10 in preclinical Alzheimer's disease, SIGLEC9 in amyotrophic lateral sclerosis, and CD38, GPNMB, and ADAM15 in Parkinson's disease. CSF levels of GRN, MMP-10, and GPNMB were altered in Alzheimer's disease, preclinical Alzheimer's disease, and Parkinson's disease, respectively. These findings point to pathways to be explored for novel therapies. The novel finding that ventricular volume confounded pQTLs has implications for design of future studies of the genetic regulation of the CSF proteome. </li>
<li><strong>CITATION</strong> : Hansson O, Kumar A, Janelidze S, Stomrud E, ...&amp;, Mattsson-Carlgren N. (2023) The genetic regulation of protein expression in cerebrospinal fluid EMBO Mol. Med., 15 (1) e16359. doi:10.15252/emmm.202216359. PMID 36504281 </li>
<li><strong>JOURNAL_INFO</strong> : EMBO molecular medicine ; EMBO Mol. Med. ; 2023 ; 15 ; 1 ; e16359 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36504281">36504281</a> </li>
</ul>
<h3 id="koprulu-m-36823471">Koprulu M-36823471</h3>
<ul>
<li><strong>NAME</strong> : Koprulu M-36823471 </li>
<li><strong>TITLE</strong> : Proteogenomic links to human metabolic diseases </li>
<li><strong>DOI</strong> : 10.1038/s42255-023-00753-7 </li>
<li><strong>ABSTRACT</strong> : Studying the plasma proteome as the intermediate layer between the genome and the phenome has the potential to identify new disease processes. Here, we conducted a cis-focused proteogenomic analysis of 2,923 plasma proteins measured in 1,180 individuals using antibody-based assays. We (1) identify 256 unreported protein quantitative trait loci (pQTL); (2) demonstrate shared genetic regulation of 224 cis-pQTLs with 575 specific health outcomes, revealing examples for notable metabolic diseases (such as gastrin-releasing peptide as a potential therapeutic target for type 2 diabetes); (3) improve causal gene assignment at 40% (n = 192) of overlapping risk loci; and (4) observe convergence of phenotypic consequences of cis-pQTLs and rare loss-of-function gene burden for 12 proteins, such as TIMD4 for lipoprotein metabolism. Our findings demonstrate the value of integrating complementary proteomic technologies with genomics even at moderate scale to identify new mediators of metabolic diseases with the potential for therapeutic interventions. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Koprulu M, Carrasco-Zanini J, Wheeler E, Lockhart S, ...&amp;, Langenberg C. (2023) Proteogenomic links to human metabolic diseases Nat. Metab., 5 (3) 516-528. doi:10.1038/s42255-023-00753-7. PMID 36823471 </li>
<li><strong>JOURNAL_INFO</strong> : Nature metabolism ; Nat. Metab. ; 2023 ; 5 ; 3 ; 516-528 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36823471">36823471</a> </li>
</ul>
<h3 id="macdonald-dunlop">Macdonald-Dunlop</h3>
<ul>
<li><strong>NAME</strong> : Macdonald-Dunlop </li>
<li><strong>PREPRINT_DOI</strong> : 2021.08.03.21261494 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>CITATION</strong> : Macdonald-Dunlop, E. et al. Mapping genetic determinants of 184 circulating proteins in 26,494 individuals to connect proteins and diseases. bioRxiv (2021) doi:10.1101/2021.08.03.21261494. </li>
</ul>
<h3 id="said">Said</h3>
<ul>
<li><strong>NAME</strong> : Said </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.11.13.23298365 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_ASIA/#china-kadoorie-biobank">China Kadoorie Biobank</a> </li>
<li><strong>CITATION</strong> : Said, S. et al. Ancestry diversity in the genetic determinants of the human plasma proteome and associated new drug targets. bioRxiv (2023) doi:10.1101/2023.11.13.23298365. </li>
</ul>
<h3 id="suhre-k-38412862">Suhre K-38412862</h3>
<ul>
<li><strong>NAME</strong> : Suhre K-38412862 </li>
<li><strong>DESCRIPTION</strong> : rQTLs </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_EUROPE/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2024.100506 </li>
<li><strong>ABSTRACT</strong> : Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development because they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease endpoints. Using Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank, we identified 4,248 associations with 2,821 ratios between protein levels (rQTLs). rQTLs were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. Conducting a GWAS on ratios increased the number of discovered genetic signals by 24.7%. The approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large-scale GWAS. </li>
<li><strong>CITATION</strong> : Suhre K. (2024) Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions Cell Genom, 4 (3) 100506. doi:10.1016/j.xgen.2024.100506. PMID 38412862 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom ; 2024 ; 4 ; 3 ; 100506 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38412862">38412862</a> </li>
</ul>
<h3 id="sun-bb-37794186">Sun BB-37794186</h3>
<ul>
<li><strong>NAME</strong> : Sun BB-37794186 </li>
<li><strong>TITLE</strong> : Plasma proteomic associations with genetics and health in the UK Biobank </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06592-6 </li>
<li><strong>ABSTRACT</strong> : The Pharma Proteomics Project is a precompetitive biopharmaceutical consortium characterizing the plasma proteomic profiles of 54,219 UK Biobank participants. Here we provide a detailed summary of this initiative, including technical and biological validations, insights into proteomic disease signatures, and prediction modelling for various demographic and health indicators. We present comprehensive protein quantitative trait locus (pQTL) mapping of 2,923 proteins that identifies 14,287 primary genetic associations, of which 81% are previously undescribed, alongside ancestry-specific pQTL mapping in non-European individuals. The study provides an updated characterization of the genetic architecture of the plasma proteome, contextualized with projected pQTL discovery rates as sample sizes and proteomic assay coverages increase over time. We offer extensive insights into trans pQTLs across multiple biological domains, highlight genetic influences on ligand-receptor interactions and pathway perturbations across a diverse collection of cytokines and complement networks, and illustrate long-range epistatic effects of ABO blood group and FUT2 secretor status on proteins with gastrointestinal tissue-enriched expression. We demonstrate the utility of these data for drug discovery by extending the genetic proxied effects of protein targets, such as PCSK9, on additional endpoints, and disentangle specific genes and proteins perturbed at loci associated with COVID-19 susceptibility. This public-private partnership provides the scientific community with an open-access proteomics resource of considerable breadth and depth to help to elucidate the biological mechanisms underlying proteo-genomic discoveries and accelerate the development of biomarkers, predictive models and therapeutics1. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Sun BB, Chiou J, Traylor M, Benner C, ...&amp;, Whelan CD. (2023) Plasma proteomic associations with genetics and health in the UK Biobank Nature, 622 (7982) 329-338. doi:10.1038/s41586-023-06592-6. PMID 37794186 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 329-338 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794186">37794186</a> </li>
</ul>
<h3 id="wang-39317738">Wang-39317738</h3>
<ul>
<li><strong>NAME</strong> : Wang-39317738 </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>CITATION</strong> : Wang, Q. S. et al. Statistically and functionally fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct regulation patterns and disease relevance. Nat. Genet. 56, 20542067 (2024). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39317738">39317738</a> </li>
</ul>
<h3 id="xu-f-36797296">Xu F-36797296</h3>
<ul>
<li><strong>NAME</strong> : Xu F-36797296 </li>
<li><strong>URL</strong> : <a href="https://omics.lab.westlake.edu.cn/data/proteins/">https://omics.lab.westlake.edu.cn/data/proteins/</a> </li>
<li><strong>TITLE</strong> : Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-36491-3 </li>
<li><strong>ABSTRACT</strong> : Identification of protein quantitative trait loci (pQTL) helps understand the underlying mechanisms of diseases and discover promising targets for pharmacological intervention. For most important class of drug targets, genetic evidence needs to be generalizable to diverse populations. Given that the majority of the previous studies were conducted in European ancestry populations, little is known about the protein-associated genetic variants in East Asians. Based on data-independent acquisition mass spectrometry technique, we conduct genome-wide association analyses for 304 unique proteins in 2,958 Han Chinese participants. We identify 195 genetic variant-protein associations. Colocalization and Mendelian randomization analyses highlight 60 gene-protein-phenotype associations, 45 of which (75%) have not been prioritized in Europeans previously. Further cross-ancestry analyses uncover key proteins that contributed to the differences in the obesity-induced diabetes and coronary artery disease susceptibility. These findings provide novel druggable proteins as well as a unique resource for the trans-ancestry evaluation of protein-targeted drug discovery. </li>
<li><strong>CITATION</strong> : Xu F, Yu EY, Cai X, Yue L, ...&amp;, Zheng JS. (2023) Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility Nat. Commun., 14 (1) 896. doi:10.1038/s41467-023-36491-3. PMID 36797296 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 896 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36797296">36797296</a> </li>
</ul>
<h2 id="hla-gwas">HLA-GWAS</h2>
<h3 id="krishna-c-39085222">Krishna C-39085222</h3>
<ul>
<li><strong>NAME</strong> : Krishna C-39085222 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Biobanks_EUROPE/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : The influence of HLA genetic variation on plasma protein expression </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-50583-8 </li>
<li><strong>ABSTRACT</strong> : Genetic variation in the human leukocyte antigen (HLA) loci is associated with risk of immune-mediated diseases, but the molecular effects of HLA polymorphism are unclear. Here we examined the effects of HLA genetic variation on the expression of 2940 plasma proteins across 45,330 Europeans in the UK Biobank, with replication analyses across multiple ancestry groups. We detected 504 proteins affected by HLA variants (HLA-pQTL), including widespread trans effects by autoimmune disease risk alleles. More than 80% of the HLA-pQTL fine-mapped to amino acid positions in the peptide binding groove. HLA-I and II affected proteins expressed in similar cell types but in different pathways of both adaptive and innate immunity. Finally, we investigated potential HLA-pQTL effects on disease by integrating HLA-pQTL with fine-mapped HLA-disease signals in the UK Biobank. Our data reveal the diverse effects of HLA genetic variation and aid the interpretation of associations between HLA alleles and immune-mediated diseases. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Krishna C, Chiou J, Sakaue S, Kang JB, ...&amp;, Hu X. (2024) The influence of HLA genetic variation on plasma protein expression Nat. Commun., 15 (1) 6469. doi:10.1038/s41467-024-50583-8. PMID 39085222 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2024 ; 15 ; 1 ; 6469 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39085222">39085222</a> </li>
</ul>
<h2 id="platform">Platform</h2>
<h3 id="omicspred-portal">OmicsPred portal</h3>
<ul>
<li><strong>NAME</strong> : OmicsPred portal </li>
<li><strong>URL</strong> : <a href="https://www.omicspred.org/">https://www.omicspred.org/</a> </li>
<li><strong>TITLE</strong> : An atlas of genetic scores to predict multi-omic traits </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-05844-9 </li>
<li><strong>ABSTRACT</strong> : The use of omic modalities to dissect the molecular underpinnings of common diseases and traits is becoming increasingly common. But multi-omic traits can be genetically predicted, which enables highly cost-effective and powerful analyses for studies that do not have multi-omics1. Here we examine a large cohort (the INTERVAL study2; n = 50,000 participants) with extensive multi-omic data for plasma proteomics (SomaScan, n = 3,175; Olink, n = 4,822), plasma metabolomics (Metabolon HD4, n = 8,153), serum metabolomics (Nightingale, n = 37,359) and whole-blood Illumina RNA sequencing (n = 4,136), and use machine learning to train genetic scores for 17,227 molecular traits, including 10,521 that reach Bonferroni-adjusted significance. We evaluate the performance of genetic scores through external validation across cohorts of individuals of European, Asian and African American ancestries. In addition, we show the utility of these multi-omic genetic scores by quantifying the genetic control of biological pathways and by generating a synthetic multi-omic dataset of the UK Biobank3 to identify disease associations using a phenome-wide scan. We highlight a series of biological insights with regard to genetic mechanisms in metabolism and canonical pathway associations with disease; for example, JAK-STAT signalling and coronary atherosclerosis. Finally, we develop a portal ( https://www.omicspred.org/ ) to facilitate public access to all genetic scores and validation results, as well as to serve as a platform for future extensions and enhancements of multi-omic genetic scores. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Xu Y, Ritchie SC, Liang Y, Timmers PRHJ, ...&amp;, Inouye M. (2023) An atlas of genetic scores to predict multi-omic traits Nature, 616 (7955) 123-131. doi:10.1038/s41586-023-05844-9. PMID 36991119 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 616 ; 7955 ; 123-131 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36991119">36991119</a> </li>
</ul>
<h3 id="proteome-phewas-browser">Proteome PheWAS browser</h3>
<ul>
<li><strong>NAME</strong> : Proteome PheWAS browser </li>
<li><strong>TITLE</strong> : Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases </li>
<li><strong>DOI</strong> : 10.1038/s41588-020-0682-6 </li>
<li><strong>ABSTRACT</strong> : The human proteome is a major source of therapeutic targets. Recent genetic association analyses of the plasma proteome enable systematic evaluation of the causal consequences of variation in plasma protein levels. Here we estimated the effects of 1,002 proteins on 225 phenotypes using two-sample Mendelian randomization (MR) and colocalization. Of 413 associations supported by evidence from MR, 130 (31.5%) were not supported by results of colocalization analyses, suggesting that genetic confounding due to linkage disequilibrium is widespread in nave phenome-wide association studies of proteins. Combining MR and colocalization evidence in cis-only analyses, we identified 111 putatively causal effects between 65 proteins and 52 disease-related phenotypes ( https://www.epigraphdb.org/pqtl/ ). Evaluation of data from historic drug development programs showed that target-indication pairs with MR and colocalization support were more likely to be approved, evidencing the value of this approach in identifying and prioritizing potential therapeutic targets. </li>
<li><strong>CITATION</strong> : Zheng J, Haberland V, Baird D, Walker V, ...&amp;, Gaunt TR. (2020) Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases Nat. Genet., 52 (10) 1122-1131. doi:10.1038/s41588-020-0682-6. PMID 32895551 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2020 ; 52 ; 10 ; 1122-1131 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32895551">32895551</a> </li>
</ul>
<h3 id="pgwas-server">pGWAS server</h3>
<ul>
<li><strong>NAME</strong> : pGWAS server </li>
<li><strong>DESCRIPTION</strong> : In our study, we performed a genome-wide association study with protein levels (pGWAS). Using a highly multiplexed, aptamer-based, affinity proteomics platform (SOMAscan), we quantified levels of 1,124 proteins in blood plasma samples from 1,000 German individuals (KORA cohort) and 338 Arab or Asian individuals (QMDiab cohort). We identified 539 independent, genome-wide significant SNP-to-protein associations, which can be investigated using this webserver. </li>
<li><strong>URL</strong> : <a href="https://metabolomics.helmholtz-muenchen.de/pgwas/">https://metabolomics.helmholtz-muenchen.de/pgwas/</a> </li>
<li><strong>TITLE</strong> : Connecting genetic risk to disease end points through the human blood plasma proteome </li>
<li><strong>DOI</strong> : 10.1038/ncomms14357 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) with intermediate phenotypes, like changes in metabolite and protein levels, provide functional evidence to map disease associations and translate them into clinical applications. However, although hundreds of genetic variants have been associated with complex disorders, the underlying molecular pathways often remain elusive. Associations with intermediate traits are key in establishing functional links between GWAS-identified risk-variants and disease end points. Here we describe a GWAS using a highly multiplexed aptamer-based affinity proteomics platform. We quantify 539 associations between protein levels and gene variants (pQTLs) in a German cohort and replicate over half of them in an Arab and Asian cohort. Fifty-five of the replicated pQTLs are located in trans. Our associations overlap with 57 genetic risk loci for 42 unique disease end points. We integrate this information into a genome-proteome network and provide an interactive web-tool for interrogations. Our results provide a basis for novel approaches to pharmaceutical and diagnostic applications. </li>
<li><strong>CITATION</strong> : Suhre K, Arnold M, Bhagwat AM, Cotton RJ, ...&amp;, Graumann J. (2017) Connecting genetic risk to disease end points through the human blood plasma proteome Nat. Commun., 8 () 14357. doi:10.1038/ncomms14357. PMID 28240269 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2017 ; 8 ; <NA> ; 14357 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28240269">28240269</a> </li>
</ul>
<h2 id="post-gwas">Post-GWAS</h2>
<h3 id="deng-yt-39579765">Deng YT-39579765</h3>
<ul>
<li><strong>NAME</strong> : Deng YT-39579765 </li>
<li><strong>URL</strong> : <a href="https://proteome-phenome-atlas.com/">https://proteome-phenome-atlas.com/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>TITLE</strong> : Atlas of the plasma proteome in health and disease in 53,026 adults </li>
<li><strong>DOI</strong> : 10.1016/j.cell.2024.10.045 </li>
<li><strong>ABSTRACT</strong> : SummaryLarge-scale proteomics studies can refine our understanding of health and disease and enable precision medicine. Here, we provide a detailed atlas of 2,920 plasma proteins linking to diseases (406 prevalent and 660 incident) and 986 health-related traits in 53,026 individuals (median follow-up: 14.8 years) from the UK Biobank, representing the most comprehensive proteome profiles to date. This atlas revealed 168,100 protein-disease associations and 554,488 protein-trait associations. Over 650 proteins were shared among at least 50 diseases, and over 1,000 showed sex and age heterogeneity. Furthermore, proteins demonstrated promising potential in disease discrimination (area under the curve [AUC] &gt; 0.80 in 183 diseases). Finally, integrating protein quantitative trait locus data determined 474 causal proteins, providing 37 drug-repurposing opportunities and 26 promising targets with favorable safety profiles. These results provide an open-access comprehensive proteome-phenome resource (https://proteome-phenome-atlas.com/) to help elucidate the biological mechanisms of diseases and accelerate the development of disease biomarkers, prediction models, and therapeutic targets. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Deng YT, You J, He Y, Zhang Y, ...&amp;, Yu JT. (2024) Atlas of the plasma proteome in health and disease in 53,026 adults Cell, 0 (0) . doi:10.1016/j.cell.2024.10.045. PMID 39579765 </li>
<li><strong>JOURNAL_INFO</strong> : Cell ; Cell ; 2024 ; 0 ; 0 ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39579765">39579765</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-suhre-k-32860016">Review-Suhre K-32860016</h3>
<ul>
<li><strong>NAME</strong> : Review-Suhre K-32860016 </li>
<li><strong>DESCRIPTION</strong> : A Table of all published GWAS with proteomics </li>
<li><strong>URL</strong> : <a href="http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/">http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/</a> </li>
<li><strong>TITLE</strong> : Genetics meets proteomics: perspectives for large population-based studies </li>
<li><strong>DOI</strong> : 10.1038/s41576-020-0268-2 </li>
<li><strong>ABSTRACT</strong> : Proteomic analysis of cells, tissues and body fluids has generated valuable insights into the complex processes influencing human biology. Proteins represent intermediate phenotypes for disease and provide insight into how genetic and non-genetic risk factors are mechanistically linked to clinical outcomes. Associations between protein levels and DNA sequence variants that colocalize with risk alleles for common diseases can expose disease-associated pathways, revealing novel drug targets and translational biomarkers. However, genome-wide, population-scale analyses of proteomic data are only now emerging. Here, we review current findings from studies of the plasma proteome and discuss their potential for advancing biomedical translation through the interpretation of genome-wide association analyses. We highlight the challenges faced by currently available technologies and provide perspectives relevant to their future application in large-scale biobank studies. </li>
<li><strong>CITATION</strong> : Suhre K, McCarthy MI, Schwenk JM. (2021) Genetics meets proteomics: perspectives for large population-based studies Nat. Rev. Genet., 22 (1) 19-37. doi:10.1038/s41576-020-0268-2. PMID 32860016 </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Genetics ; Nat. Rev. Genet. ; 2021 ; 22 ; 1 ; 19-37 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32860016">32860016</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>