
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Dimension_reduction_README/">
      
      
        <link rel="next" href="../Tools_Fine_mapping_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Drug_discovery - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug_discovery
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Genome_Genome_README/" class="md-tabs__link">
          
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#drug-targetor" class="md-nav__link">
    <span class="md-ellipsis">
      DRUG TARGETOR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grep" class="md-nav__link">
    <span class="md-ellipsis">
      GREP
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#guideline-namba" class="md-nav__link">
    <span class="md-ellipsis">
      Guideline-Namba
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#open-targets-genetics" class="md-nav__link">
    <span class="md-ellipsis">
      Open Targets Genetics
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pes" class="md-nav__link">
    <span class="md-ellipsis">
      PES
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ps4dr" class="md-nav__link">
    <span class="md-ellipsis">
      PS4DR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#priority-index" class="md-nav__link">
    <span class="md-ellipsis">
      Priority index
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trans-phar" class="md-nav__link">
    <span class="md-ellipsis">
      Trans-Phar
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Genealogy_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#drug-targetor" class="md-nav__link">
    <span class="md-ellipsis">
      DRUG TARGETOR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grep" class="md-nav__link">
    <span class="md-ellipsis">
      GREP
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#guideline-namba" class="md-nav__link">
    <span class="md-ellipsis">
      Guideline-Namba
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#open-targets-genetics" class="md-nav__link">
    <span class="md-ellipsis">
      Open Targets Genetics
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pes" class="md-nav__link">
    <span class="md-ellipsis">
      PES
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ps4dr" class="md-nav__link">
    <span class="md-ellipsis">
      PS4DR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#priority-index" class="md-nav__link">
    <span class="md-ellipsis">
      Priority index
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trans-phar" class="md-nav__link">
    <span class="md-ellipsis">
      Trans-Phar
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Drug_discovery</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: right;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#drug-targetor">DRUG TARGETOR</a></td>
<td style="text-align: left;">Gaspar HA, Hübel C, Breen G. (2019) Drug Targetor: a web interface to investigate the human druggome for over 500 phenotypes Bioinformatics, 35 (14) 2515-2517. doi:10.1093/bioinformatics/bty982. PMID 30517594</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#grep">GREP</a></td>
<td style="text-align: left;">Sakaue S, Okada Y. (2019) GREP: genome for REPositioning drugs Bioinformatics, 35 (19) 3821-3823. doi:10.1093/bioinformatics/btz166. PMID 30859178</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#guideline-namba">Guideline-Namba</a></td>
<td style="text-align: left;">Namba S, Konuma T, Wu KH, Zhou W, ...&amp;, Okada Y. (2022) A practical guideline of genomics-driven drug discovery in the era of global biobank meta-analysis Cell Genom, 2 (10) 100190. doi:10.1016/j.xgen.2022.100190. PMID 36778001</td>
<td style="text-align: right;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#open-targets-genetics">Open Targets Genetics</a></td>
<td style="text-align: left;">Mountjoy E, Schmidt EM, Carmona M, Schwartzentruber J, ...&amp;, Ghoussaini M. (2021) An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci Nat. Genet., 53 (11) 1527-1533. doi:10.1038/s41588-021-00945-5. PMID 34711957</td>
<td style="text-align: right;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pes">PES</a></td>
<td style="text-align: left;">Reay WR, Atkins JR, Carr VJ, Green MJ, ...&amp;, Cairns MJ. (2020) Pharmacological enrichment of polygenic risk for precision medicine in complex disorders Sci. Rep., 10 (1) 879. doi:10.1038/s41598-020-57795-0. PMID 31964963</td>
<td style="text-align: right;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ps4dr">PS4DR</a></td>
<td style="text-align: left;">Emon MA, Domingo-Fernández D, Hoyt CT, Hofmann-Apitius M. (2020) PS4DR: a multimodal workflow for identification and prioritization of drugs based on pathway signatures BMC Bioinformatics, 21 (1) 231. doi:10.1186/s12859-020-03568-5. PMID 32503412</td>
<td style="text-align: right;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#priority-index">Priority index</a></td>
<td style="text-align: left;">Fang H, ULTRA-DD Consortium, De Wolf H, Knezevic B, ...&amp;, Knight JC. (2019) A genetics-led approach defines the drug target landscape of 30 immune-related traits Nat. Genet., 51 (7) 1082-1091. doi:10.1038/s41588-019-0456-1. PMID 31253980</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#trans-phar">Trans-Phar</a></td>
<td style="text-align: left;">Konuma T, Ogawa K, Okada Y. (2021) Integration of genetically regulated gene expression and pharmacological library provides therapeutic drug candidates Hum. Mol. Genet., 30 (3-4) 294-304. doi:10.1093/hmg/ddab049. PMID 33577681</td>
<td style="text-align: right;">2021</td>
</tr>
</tbody>
</table>
<h2 id="drug-targetor">DRUG TARGETOR</h2>
<ul>
<li><strong>NAME</strong> : DRUG TARGETOR </li>
<li><strong>SHORT NAME</strong> : DRUG TARGETOR </li>
<li><strong>FULL NAME</strong> : DRUG TARGETOR </li>
<li><strong>DESCRIPTION</strong> : This website harnesses results from genome-wide association studies (GWAS), and drug bioactivity data, to prioritize drugs and targets for a given phenotype. Drug Targetor networks are constructed using genetically scored drugs and genes, connected by the type of drug-target or drug-gene interaction </li>
<li><strong>URL</strong> : <a href="https://drugtargetor.com/index_v1.21.html">https://drugtargetor.com/index_v1.21.html</a> </li>
<li><strong>TITLE</strong> : Drug Targetor: a web interface to investigate the human druggome for over 500 phenotypes </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/bty982 </li>
<li><strong>ABSTRACT</strong> : SUMMARY: Results from hundreds of genome-wide association studies (GWAS) are now freely available and offer a catalogue of the association between phenotypes across medicine with variants in the genome. With the aim of using this data to better understand therapeutic mechanisms, we have developed Drug Targetor, a web interface that allows the generation and exploration of drug-target networks of hundreds of phenotypes using GWAS data. Drug Targetor networks consist of drug and target nodes ordered by genetic association and connected by drug-target or drug-gene relationship. We show that Drug Targetor can help prioritize drugs, targets and drug-target interactions for a specific phenotype based on genetic evidence. AVAILABILITY AND IMPLEMENTATION: Drug Targetor v1.21 is a web application freely available online at drugtargetor.com and under MIT licence. The source code can be found at https://github.com/hagax8/drugtargetor. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Gaspar HA, Hübel C, Breen G. (2019) Drug Targetor: a web interface to investigate the human druggome for over 500 phenotypes Bioinformatics, 35 (14) 2515-2517. doi:10.1093/bioinformatics/bty982. PMID 30517594 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2019 ; 35 ; 14 ; 2515-2517 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30517594">30517594</a> </li>
</ul>
<h2 id="grep">GREP</h2>
<ul>
<li><strong>NAME</strong> : GREP </li>
<li><strong>SHORT NAME</strong> : GREP </li>
<li><strong>FULL NAME</strong> : Genome for REPositioning drugs </li>
<li><strong>DESCRIPTION</strong> : GREP can quantify an enrichment of the user-defined set of genes in the target of clinical indication categories and capture potentially repositionable drugs targeting the gene set. Both can be run in a few seconds! </li>
<li><strong>URL</strong> : <a href="https://github.com/saorisakaue/GREP">https://github.com/saorisakaue/GREP</a> </li>
<li><strong>TITLE</strong> : GREP: genome for REPositioning drugs </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btz166 </li>
<li><strong>ABSTRACT</strong> : SUMMARY: Making use of accumulated genetic knowledge for clinical practice is our next goal in human genetics. Here we introduce GREP (Genome for REPositioning drugs), a standalone python software to quantify an enrichment of the user-defined set of genes in the target of clinical indication categories and to capture potentially repositionable drugs targeting the gene set. We show that genes identified by the large-scale genome-wide association studies were robustly enriched in the approved drugs to treat the trait of interest. This enrichment analysis was also highly applicable to other sets of biological genes such as those identified by gene expression studies and genes somatically mutated in cancers. This software should accelerate investigators to reposition drugs to other indications with the guidance of known genomics. AVAILABILITY AND IMPLEMENTATION: GREP is available at https://github.com/saorisakaue/GREP as a python source code. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>CITATION</strong> : Sakaue S, Okada Y. (2019) GREP: genome for REPositioning drugs Bioinformatics, 35 (19) 3821-3823. doi:10.1093/bioinformatics/btz166. PMID 30859178 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics  ; Bioinformatics ; 2019 ; 35 ; 19 ; 3821-3823 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30859178">30859178</a> </li>
</ul>
<h2 id="guideline-namba">Guideline-Namba</h2>
<ul>
<li><strong>NAME</strong> : Guideline-Namba </li>
<li><strong>DESCRIPTION</strong> : a practical guideline for genomics-driven drug discovery for cross-population meta-analysis, as lessons from the Global Biobank Meta-analysis Initiative (GBMI) </li>
<li><strong>TITLE</strong> : A practical guideline of genomics-driven drug discovery in the era of global biobank meta-analysis </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2022.100190 </li>
<li><strong>ABSTRACT</strong> : Genomics-driven drug discovery is indispensable for accelerating the development of novel therapeutic targets. However, the drug discovery framework based on evidence from genome-wide association studies (GWASs) has not been established, especially for cross-population GWAS meta-analysis. Here, we introduce a practical guideline for genomics-driven drug discovery for cross-population meta-analysis, as lessons from the Global Biobank Meta-analysis Initiative (GBMI). Our drug discovery framework encompassed three methodologies and was applied to the 13 common diseases targeted by GBMI (N mean = 1,329,242). Individual methodologies complementarily prioritized drugs and drug targets, which were systematically validated by referring previously known drug-disease relationships. Integration of the three methodologies provided a comprehensive catalog of candidate drugs for repositioning, nominating promising drug candidates targeting the genes involved in the coagulation process for venous thromboembolism and the interleukin-4 and interleukin-13 signaling pathway for gout. Our study highlighted key factors for successful genomics-driven drug discovery using cross-population meta-analyses. </li>
<li><strong>CITATION</strong> : Namba S, Konuma T, Wu KH, Zhou W, ...&amp;, Okada Y. (2022) A practical guideline of genomics-driven drug discovery in the era of global biobank meta-analysis Cell Genom, 2 (10) 100190. doi:10.1016/j.xgen.2022.100190. PMID 36778001 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom ; 2022 ; 2 ; 10 ; 100190 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36778001">36778001</a> </li>
</ul>
<h2 id="open-targets-genetics">Open Targets Genetics</h2>
<ul>
<li><strong>NAME</strong> : Open Targets Genetics </li>
<li><strong>SHORT NAME</strong> : Open Targets Genetics </li>
<li><strong>FULL NAME</strong> : Open Targets Genetics </li>
<li><strong>DESCRIPTION</strong> : Open Targets Genetics is a comprehensive tool highlighting variant-centric statistical evidence to allow both prioritisation of candidate causal variants at trait-associated loci and identification of potential drug targets. </li>
<li><strong>URL</strong> : <a href="https://genetics.opentargets.org/">https://genetics.opentargets.org/</a> </li>
<li><strong>TITLE</strong> : An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00945-5 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWASs) have identified many variants associated with complex traits, but identifying the causal gene(s) is a major challenge. In the present study, we present an open resource that provides systematic fine mapping and gene prioritization across 133,441 published human GWAS loci. We integrate genetics (GWAS Catalog and UK Biobank) with transcriptomic, proteomic and epigenomic data, including systematic disease-disease and disease-molecular trait colocalization results across 92 cell types and tissues. We identify 729 loci fine mapped to a single-coding causal variant and colocalized with a single gene. We trained a machine-learning model using the fine-mapped genetics and functional genomics data and 445 gold-standard curated GWAS loci to distinguish causal genes from neighboring genes, outperforming a naive distance-based model. Our prioritized genes were enriched for known approved drug targets (odds ratio = 8.1, 95% confidence interval = 5.7, 11.5). These results are publicly available through a web portal ( http://genetics.opentargets.org ), enabling users to easily prioritize genes at disease-associated loci and assess their potential as drug targets. </li>
<li><strong>CITATION</strong> : Mountjoy E, Schmidt EM, Carmona M, Schwartzentruber J, ...&amp;, Ghoussaini M. (2021) An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci Nat. Genet., 53 (11) 1527-1533. doi:10.1038/s41588-021-00945-5. PMID 34711957 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 11 ; 1527-1533 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34711957">34711957</a> </li>
</ul>
<h2 id="pes">PES</h2>
<ul>
<li><strong>NAME</strong> : PES </li>
<li><strong>SHORT NAME</strong> : PES </li>
<li><strong>FULL NAME</strong> : Pharmagenic_enrichment_score </li>
<li><strong>DESCRIPTION</strong> : a framework to quantify an individual’s common variant enrichment in clinically actionable systems responsive to existing drugs. </li>
<li><strong>TITLE</strong> : Pharmacological enrichment of polygenic risk for precision medicine in complex disorders </li>
<li><strong>DOI</strong> : 10.1038/s41598-020-57795-0 </li>
<li><strong>ABSTRACT</strong> : Individuals with complex disorders typically have a heritable burden of common variation that can be expressed as a polygenic risk score (PRS). While PRS has some predictive utility, it lacks the molecular specificity to be directly informative for clinical interventions. We therefore sought to develop a framework to quantify an individual's common variant enrichment in clinically actionable systems responsive to existing drugs. This was achieved with a metric designated the pharmagenic enrichment score (PES), which we demonstrate for individual SNP profiles in a cohort of cases with schizophrenia. A large proportion of these had elevated PES in one or more of eight clinically actionable gene-sets enriched with schizophrenia associated common variation. Notable candidates targeting these pathways included vitamins, antioxidants, insulin modulating agents, and cholinergic drugs. Interestingly, elevated PES was also observed in individuals with otherwise low common variant burden. The biological saliency of PES profiles were observed directly through their impact on gene expression in a subset of the cohort with matched transcriptomic data, supporting our assertion that this gene-set orientated approach could integrate an individual's common variant risk to inform personalised interventions, including drug repositioning, for complex disorders such as schizophrenia. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Reay WR, Atkins JR, Carr VJ, Green MJ, ...&amp;, Cairns MJ. (2020) Pharmacological enrichment of polygenic risk for precision medicine in complex disorders Sci. Rep., 10 (1) 879. doi:10.1038/s41598-020-57795-0. PMID 31964963 </li>
<li><strong>JOURNAL_INFO</strong> : Scientific reports ; Sci. Rep. ; 2020 ; 10 ; 1 ; 879 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31964963">31964963</a> </li>
</ul>
<h2 id="ps4dr">PS4DR</h2>
<ul>
<li><strong>NAME</strong> : PS4DR </li>
<li><strong>SHORT NAME</strong> : PS4DR </li>
<li><strong>FULL NAME</strong> : Pathway Signatures for Drug Repositioning </li>
<li><strong>DESCRIPTION</strong> : This package comprises a modular workflow designed to identify drug repositioning candidates using multi-omics data sets. A schematic figure of the workflow is presented below. The R scripts necessary to run the MSDRP pipeline are located in the R directory. </li>
<li><strong>URL</strong> : <a href="https://github.com/ps4dr/ps4dr">https://github.com/ps4dr/ps4dr</a> </li>
<li><strong>TITLE</strong> : PS4DR: a multimodal workflow for identification and prioritization of drugs based on pathway signatures </li>
<li><strong>DOI</strong> : 10.1186/s12859-020-03568-5 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: During the last decade, there has been a surge towards computational drug repositioning owing to constantly increasing -omics data in the biomedical research field. While numerous existing methods focus on the integration of heterogeneous data to propose candidate drugs, it is still challenging to substantiate their results with mechanistic insights of these candidate drugs. Therefore, there is a need for more innovative and efficient methods which can enable better integration of data and knowledge for drug repositioning. RESULTS: Here, we present a customizable workflow (PS4DR) which not only integrates high-throughput data such as genome-wide association study (GWAS) data and gene expression signatures from disease and drug perturbations but also takes pathway knowledge into consideration to predict drug candidates for repositioning. We have collected and integrated publicly available GWAS data and gene expression signatures for several diseases and hundreds of FDA-approved drugs or those under clinical trial in this study. Additionally, different pathway databases were used for mechanistic knowledge integration in the workflow. Using this systematic consolidation of data and knowledge, the workflow computes pathway signatures that assist in the prediction of new indications for approved and investigational drugs. CONCLUSION: We showcase PS4DR with applications demonstrating how this tool can be used for repositioning and identifying new drugs as well as proposing drugs that can simulate disease dysregulations. We were able to validate our workflow by demonstrating its capability to predict FDA-approved drugs for their known indications for several diseases. Further, PS4DR returned many potential drug candidates for repositioning that were backed up by epidemiological evidence extracted from scientific literature. Source code is freely available at https://github.com/ps4dr/ps4dr. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Emon MA, Domingo-Fernández D, Hoyt CT, Hofmann-Apitius M. (2020) PS4DR: a multimodal workflow for identification and prioritization of drugs based on pathway signatures BMC Bioinformatics, 21 (1) 231. doi:10.1186/s12859-020-03568-5. PMID 32503412 </li>
<li><strong>JOURNAL_INFO</strong> : BMC bioinformatics ; BMC Bioinformatics ; 2020 ; 21 ; 1 ; 231 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32503412">32503412</a> </li>
</ul>
<h2 id="priority-index">Priority index</h2>
<ul>
<li><strong>NAME</strong> : Priority index </li>
<li><strong>SHORT NAME</strong> : Priority index </li>
<li><strong>FULL NAME</strong> : Priority index </li>
<li><strong>DESCRIPTION</strong> : A Comprehensive Resource for Genetic Targets in Immune-Mediated Disease </li>
<li><strong>URL</strong> : <a href="http://pi.well.ox.ac.uk:3010/">http://pi.well.ox.ac.uk:3010/</a> </li>
<li><strong>TITLE</strong> : A genetics-led approach defines the drug target landscape of 30 immune-related traits </li>
<li><strong>DOI</strong> : 10.1038/s41588-019-0456-1 </li>
<li><strong>ABSTRACT</strong> : Most candidate drugs currently fail later-stage clinical trials, largely due to poor prediction of efficacy on early target selection1. Drug targets with genetic support are more likely to be therapeutically valid2,3, but the translational use of genome-scale data such as from genome-wide association studies for drug target discovery in complex diseases remains challenging4-6. Here, we show that integration of functional genomic and immune-related annotations, together with knowledge of network connectivity, maximizes the informativeness of genetics for target validation, defining the target prioritization landscape for 30 immune traits at the gene and pathway level. We demonstrate how our genetics-led drug target prioritization approach (the priority index) successfully identifies current therapeutics, predicts activity in high-throughput cellular screens (including L1000, CRISPR, mutagenesis and patient-derived cell assays), enables prioritization of under-explored targets and allows for determination of target-level trait relationships. The priority index is an open-access, scalable system accelerating early-stage drug target selection for immune-mediated disease. </li>
<li><strong>CITATION</strong> : Fang H, ULTRA-DD Consortium, De Wolf H, Knezevic B, ...&amp;, Knight JC. (2019) A genetics-led approach defines the drug target landscape of 30 immune-related traits Nat. Genet., 51 (7) 1082-1091. doi:10.1038/s41588-019-0456-1. PMID 31253980 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2019 ; 51 ; 7 ; 1082-1091 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31253980">31253980</a> </li>
</ul>
<h2 id="trans-phar">Trans-Phar</h2>
<ul>
<li><strong>NAME</strong> : Trans-Phar </li>
<li><strong>SHORT NAME</strong> : Trans-Phar </li>
<li><strong>FULL NAME</strong> : integration of Transcriptome-wide association study and Pharmacological database </li>
<li><strong>DESCRIPTION</strong> : This software achieves in silico screening of chemical compounds, which have inverse effects in expression profiles compared with genetically regulated gene expression of common diseases, from large-scale pharmacological database (Connectivity Map [CMap] L1000 library). </li>
<li><strong>URL</strong> : <a href="https://github.com/konumat/Trans-Phar">https://github.com/konumat/Trans-Phar</a> </li>
<li><strong>TITLE</strong> : Integration of genetically regulated gene expression and pharmacological library provides therapeutic drug candidates </li>
<li><strong>DOI</strong> : 10.1093/hmg/ddab049 </li>
<li><strong>ABSTRACT</strong> : Approaches toward new therapeutics using disease genomics, such as genome-wide association study (GWAS), are anticipated. Here, we developed Trans-Phar [integration of transcriptome-wide association study (TWAS) and pharmacological database], achieving in silico screening of compounds from a large-scale pharmacological database (L1000 Connectivity Map), which have inverse expression profiles compared with tissue-specific genetically regulated gene expression. Firstly we confirmed the statistical robustness by the application of the null GWAS data and enrichment in the true-positive drug-disease relationships by the application of UK-Biobank GWAS summary statistics in broad disease categories, then we applied the GWAS summary statistics of large-scale European meta-analysis (17 traits; naverage = 201 849) and the hospitalized COVID-19 (n = 900 687), which has urgent need for drug development. We detected potential therapeutic compounds as well as anisomycin in schizophrenia (false discovery rate (FDR)-q = 0.056) and verapamil in hospitalized COVID-19 (FDR-q = 0.068) as top-associated compounds. This approach could be effective in disease genomics-driven drug development. </li>
<li><strong>CITATION</strong> : Konuma T, Ogawa K, Okada Y. (2021) Integration of genetically regulated gene expression and pharmacological library provides therapeutic drug candidates Hum. Mol. Genet., 30 (3-4) 294-304. doi:10.1093/hmg/ddab049. PMID 33577681 </li>
<li><strong>JOURNAL_INFO</strong> : Human molecular genetics ; Hum. Mol. Genet. ; 2021 ; 30 ; 3-4 ; 294-304 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33577681">33577681</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>