# Contents
- FUSION
- PrediXcan
- S-PrediXcan
- MultiXcan
- webTWAS
- TWAShub
- cTWAS
- PTWAS

## FUSION
- FULL NAME: Functional Summary-based Imputation
- SHORT NAME: FUSION
- YEAR: 2016
- DESCRIPTION:FUSION is a suite of tools for performing transcriptome-wide and regulome-wide association studies (TWAS and RWAS). FUSION builds predictive models of the genetic component of a functional/molecular phenotype and predicts and tests that component for association with disease using GWAS summary statistics. The goal is to identify associations between a GWAS phenotype and a functional phenotype that was only measured in reference data. We provide precomputed predictive models from multiple studies to facilitate this analysis.
- URL : [http://gusevlab.org/projects/fusion/](http://gusevlab.org/projects/fusion/)
- CITATION:Gusev, A., Ko, A., Shi, H., Bhatia, G., Chung, W., Penninx, B. W., ... & Pasaniuc, B. (2016). Integrative approaches for large-scale transcriptome-wide association studies. Nature genetics, 48(3), 245-252.

## PrediXcan
- FULL NAME:  PrediXcan
- SHORT NAME:  PrediXcan
- YEAR: 2015
- DESCRIPTION: PrediXcan is a gene-based association test that prioritizes genes that are likely to be causal for the phenotype.
- URL : [https://github.com/hakyimlab/PrediXcan](https://github.com/hakyimlab/PrediXcan)
- CITATION: Gamazon, E. R., Wheeler, H. E., Shah, K. P., Mozaffari, S. V., Aquino-Michaels, K., Carroll, R. J., ... & Im, H. K. (2015). A gene-based association method for mapping traits using reference transcriptome data. Nature genetics, 47(9), 1091-1098.

## S-PrediXcan
- FULL NAME: S-PrediXcan
- SHORT NAME:  S-PrediXcan
- YEAR: 2018
- DESCRIPTION:  a mathematical expression to compute PrediXcan (a gene mapping approach) results using summary data (S-PrediXcan) 
- URL : [https://github.com/hakyimlab/MetaXcan](https://github.com/hakyimlab/MetaXcan)
- CITATION: Barbeira, A. N., Dickinson, S. P., Bonazzola, R., Zheng, J., Wheeler, H. E., Torres, J. M., ... & Im, H. K. (2018). Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics. Nature communications, 9(1), 1-20.

## MultiXcan
- FULL NAME: MultiXcan
- SHORT NAME:  MultiXcan
- YEAR: 2018
- DESCRIPTION: an efficient statistical method (MultiXcan) that leverages the substantial sharing of eQTLs across tissues and contexts to improve our ability to identify potential target genes. MultiXcan integrates evidence across multiple panels using multivariate regression, which naturally takes into account the correlation structure.
- URL :  [https://github.com/hakyimlab/MetaXcan](https://github.com/hakyimlab/MetaXcan)
- CITATION: Barbeira, A. N., Pividori, M., Zheng, J., Wheeler, H. E., Nicolae, D. L., & Im, H. K. (2019). Integrating predicted transcriptome from multiple tissues improves association detection. PLoS genetics, 15(1), e1007889.

## webTWAS
- FULL NAME: webTWAS
- SHORT NAME:webTWAS
- YEAR:2022
- DESCRIPTION: a resource for disease candidate susceptibility genes identified by transcriptome-wide association study
- URL:[http://www.webtwas.net/#/](http://www.webtwas.net/#/)
- CITATION: Cao, C., Wang, J., Kwok, D., Cui, F., Zhang, Z., Zhao, D., ... & Zou, Q. (2022). webTWAS: a resource for disease candidate susceptibility genes identified by transcriptome-wide association study. Nucleic acids research, 50(D1), D1123-D1130.

## TWAS hub
- FULL NAME:TWAS hub
- SHORT NAME:TWAS hub
- YEAR:2018
- DESCRIPTION:TWAS hub is an interactive browser of results from integrative analyses of GWAS and functional data for hundreds of traits and >100k expression models. The aim is facilitate the investigation of individual TWAS associations; pleiotropic disease/trait associations for a given gene of interest; predicted gene associations for a given disease/trait of interest with detailed per-locus statistics; and pleiotropic relationships between traits based on shared associated genes.
- URL:[http://twas-hub.org/](http://twas-hub.org/)
- CITATION: Mancuso, Nicholas, et al. "Integrating gene expression with summary association statistics to identify genes associated with 30 complex traits." The American Journal of Human Genetics 100.3 (2017): 473-487.

## cTWAS
- FULL NAME:causal-TWAS
- SHORT NAME: cTWAS
- YEAR: 2024
- DESCRIPTION: Expression Quantitative Trait Loci (eQTLs) have often been used to nominate candidate genes from Genome-wide association studies (GWAS). However, commonly used methods are susceptible to false positives largely due to Linkage Disequilibrium of eQTLs with causal variants acting on the phenotype directly. Our method, causal-TWAS (cTWAS), addressed this challenge by borrowing ideas from statistical fine-mapping. It is a generalization of Transcriptome-wide association studies (TWAS), but when analyzing any gene, it adjusts for other nearby genes and all nearby genetic variants.
- URL: https://xinhe-lab.github.io/ctwas/
- CITATION: Zhao, S., Crouse, W., Qian, S., Luo, K., Stephens, M., & He, X. (2024). Adjusting for genetic confounders in transcriptome-wide association studies improves discovery of risk genes of complex traits. Nature Genetics, 1-12.
- KEYWORDS: TWAS, fine-mapping

## PTWAS
- FULL NAME: probabilistic TWAS
- SHORT NAME: PTWAS
- YEAR: 2020
- URL: https://github.com/xqwen/ptwas
- CITATION: Zhang, Y., Quick, C., Yu, K., Barbeira, A., GTEx Consortium, Luca, F., ... & Wen, X. (2020). PTWAS: investigating tissue-relevant causal molecular mechanisms of complex traits using probabilistic TWAS analysis. Genome biology, 21, 1-26.
- KEYWORDS: TWAS, instrumental variables
