
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Imputation_README/">
      
      
        <link rel="next" href="../Tools_Meta_and_Multi_triat_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Mendelian_randomization - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Mendelian_randomization
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Reference_data_README/" class="md-tabs__link">
          
  
  Reference_data

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          <label class="md-nav__link" for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#conceptsprincipals" class="md-nav__link">
    <span class="md-ellipsis">
      Concepts&amp;Principals
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Concepts&Principals">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-bowden_j" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Bowden_J
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-hemani_g" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Hemani_G
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-howe_lj" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Howe_LJ
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-pierce_bl" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Pierce_BL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-smith_gd" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Smith_GD
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-wu_y" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wu_Y
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-zhu_z" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Zhu_Z
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#two-sample-mr" class="md-nav__link">
    <span class="md-ellipsis">
      Two-sample-MR
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#reviewstutorials" class="md-nav__link">
    <span class="md-ellipsis">
      Reviews&amp;Tutorials
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Reviews&Tutorials">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-benn_m" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Benn_M
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-davies_nm" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Davies_NM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-emdin_ca" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Emdin_CA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-lawlor_da" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Lawlor_DA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-minelli_c" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Minelli_C
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-sanderson_e" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Sanderson_E
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-teumer_a" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Teumer_A
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-zheng_j" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Zheng_J
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Reference_data
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Allele_Frequency_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Allele_Frequency
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Epigenomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Expression_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Expression
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Gene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            Gene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_Ensembl_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Ensembl
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_GENCODE_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GENCODE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_HGNC_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HGNC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_NCBI_refSeq_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    NCBI_refSeq
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5_6" >
        
          
          <label class="md-nav__link" for="__nav_4_5_6" id="__nav_4_5_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_5_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5_6">
            <span class="md-nav__icon md-icon"></span>
            UCSC_refGene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ucsc_genes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_knownGenes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    knownGenes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_6" >
        
          
          <label class="md-nav__link" for="__nav_4_6" id="__nav_4_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Genome
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_6">
            <span class="md-nav__icon md-icon"></span>
            Genome
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_hg38_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    hg38
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Phenotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_xQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    xQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Karyotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Karyotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Venn_diagram_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Venn_diagram
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Workflow_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Workflow
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#conceptsprincipals" class="md-nav__link">
    <span class="md-ellipsis">
      Concepts&amp;Principals
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Concepts&Principals">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-bowden_j" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Bowden_J
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-hemani_g" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Hemani_G
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-howe_lj" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Howe_LJ
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-pierce_bl" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Pierce_BL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-smith_gd" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Smith_GD
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-wu_y" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wu_Y
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-zhu_z" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Zhu_Z
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#two-sample-mr" class="md-nav__link">
    <span class="md-ellipsis">
      Two-sample-MR
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#reviewstutorials" class="md-nav__link">
    <span class="md-ellipsis">
      Reviews&amp;Tutorials
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Reviews&Tutorials">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-benn_m" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Benn_M
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-davies_nm" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Davies_NM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-emdin_ca" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Emdin_CA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-lawlor_da" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Lawlor_DA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-minelli_c" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Minelli_C
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-sanderson_e" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Sanderson_E
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-teumer_a" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Teumer_A
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-zheng_j" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Zheng_J
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Mendelian_randomization</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: right;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#review-bowden_j">Review-Bowden_J</a></td>
<td style="text-align: left;">Concepts&amp;Principals</td>
<td style="text-align: left;">Bowden J, ...&amp;, Burgess S. (2015) Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. Int J Epidemiol, 44 (2) 512-25. doi:10.1093/ije/dyv080. PMID 26050253</td>
<td style="text-align: right;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-hemani_g">Review-Hemani_G</a></td>
<td style="text-align: left;">Concepts&amp;Principals</td>
<td style="text-align: left;">Hemani G, ...&amp;, Davey Smith G. (2017) Orienting the causal relationship between imprecisely measured traits using GWAS summary data. PLoS Genet, 13 (11) e1007081. doi:10.1371/journal.pgen.1007081. PMID 29149188</td>
<td style="text-align: right;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-howe_lj">Review-Howe_LJ</a></td>
<td style="text-align: left;">Concepts&amp;Principals</td>
<td style="text-align: left;">Howe LJ, ...&amp;, Davies NM. (2022) Interpreting Mendelian-randomization estimates of the effects of categorical exposures such as disease status and educational attainment. Int J Epidemiol, 51 (3) 948-957. doi:10.1093/ije/dyab208. PMID 34570226</td>
<td style="text-align: right;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-pierce_bl">Review-Pierce_BL</a></td>
<td style="text-align: left;">Concepts&amp;Principals</td>
<td style="text-align: left;">Pierce BL, Burgess S. (2013) Efficient design for Mendelian randomization studies: subsample and 2-sample instrumental variable estimators. Am J Epidemiol, 178 (7) 1177-84. doi:10.1093/aje/kwt084. PMID 23863760</td>
<td style="text-align: right;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-smith_gd">Review-Smith_GD</a></td>
<td style="text-align: left;">Concepts&amp;Principals</td>
<td style="text-align: left;">Davey Smith, G., &amp; Ebrahim, S. (2003). ‘Mendelian randomization’: can genetic epidemiology contribute to understanding environmental determinants of disease?. International journal of epidemiology, 32(1), 1-22.</td>
<td style="text-align: right;">2003</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-wu_y">Review-Wu_Y</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Wu Y, ...&amp;, Yang J. (2018) Integrative analysis of omics summary data reveals putative mechanisms underlying complex traits. Nat Commun, 9 (1) 918. doi:10.1038/s41467-018-03371-0. PMID 29500431</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-zhu_z">Review-Zhu_Z</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Zhu Z, ...&amp;, Yang J. (2016) Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Nat Genet, 48 (5) 481-7. doi:10.1038/ng.3538. PMID 27019110</td>
<td style="text-align: right;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#two-sample-mr">Two-sample-MR</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Hemani G, ...&amp;, Haycock PC. (2018) The MR-Base platform supports systematic causal inference across the human phenome. Elife, 7  . doi:10.7554/eLife.34408. PMID 29846171</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-benn_m">Review-Benn_M</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Benn M, Nordestgaard BG. (2018) From genome-wide association studies to Mendelian randomization: novel opportunities for understanding cardiovascular disease causality, pathogenesis, prevention, and treatment. Cardiovasc Res, 114 (9) 1192-1208. doi:10.1093/cvr/cvy045. PMID 29471399</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-davies_nm">Review-Davies_NM</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Davies NM, ...&amp;, Davey Smith G. (2018) Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians. BMJ, 362  k601. doi:10.1136/bmj.k601. PMID 30002074</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-emdin_ca">Review-Emdin_CA</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Emdin CA, ...&amp;, Kathiresan S. (2017) Mendelian Randomization. JAMA, 318 (19) 1925-1926. doi:10.1001/jama.2017.17219. PMID 29164242</td>
<td style="text-align: right;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-lawlor_da">Review-Lawlor_DA</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Lawlor DA. (2016) Commentary: Two-sample Mendelian randomization: opportunities and challenges. Int J Epidemiol, 45 (3) 908-15. doi:10.1093/ije/dyw127. PMID 27427429</td>
<td style="text-align: right;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-minelli_c">Review-Minelli_C</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Minelli C, ...&amp;, Thompson J. (2021) The use of two-sample methods for Mendelian randomization analyses on single large datasets. Int J Epidemiol, 50 (5) 1651-1659. doi:10.1093/ije/dyab084. PMID 33899104</td>
<td style="text-align: right;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-sanderson_e">Review-Sanderson_E</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Sanderson E, ...&amp;, Smith GD. (2022) Mendelian randomization. Nat Rev Methods Primers, 2  . doi:10.1038/s43586-021-00092-5. PMID 37325194</td>
<td style="text-align: right;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-teumer_a">Review-Teumer_A</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Teumer A. (2018) Common Methods for Performing Mendelian Randomization. Front Cardiovasc Med, 5  51. doi:10.3389/fcvm.2018.00051. PMID 29892602</td>
<td style="text-align: right;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-zheng_j">Review-Zheng_J</a></td>
<td style="text-align: left;">Reviews&amp;Tutorials</td>
<td style="text-align: left;">Zheng J, ...&amp;, Smith GD. (2017) Recent Developments in Mendelian Randomization Studies. Curr Epidemiol Rep, 4 (4) 330-345. doi:10.1007/s40471-017-0128-6. PMID 29226067</td>
<td style="text-align: right;">2017</td>
</tr>
</tbody>
</table>
<h2 id="conceptsprincipals">Concepts&amp;Principals</h2>
<h3 id="review-bowden_j">Review-Bowden_J</h3>
<ul>
<li><strong>NAME</strong> : Review-Bowden_J </li>
<li><strong>TITLE</strong> : Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. </li>
<li><strong>CITATION</strong> : Bowden J, ...&amp;, Burgess S. (2015) Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. Int J Epidemiol, 44 (2) 512-25. doi:10.1093/ije/dyv080. PMID 26050253 </li>
<li><strong>MESH_MAJOR</strong> :  </li>
<li><strong>MESH_MINOR</strong> : Bias, Blood Pressure, Body Height, Causality, Coronary Artery Disease, Genetic Pleiotropy, Genetic Variation, Humans, Mendelian Randomization Analysis, Meta-Analysis as Topic, Models, Biological, Regression Analysis, Respiration, Risk Assessment, Statistics as Topic </li>
<li><strong>ABSTRACT</strong> : The number of Mendelian randomization analyses including large numbers of genetic variants is rapidly increasing. This is due to the proliferation of genome-wide association studies, and the desire to obtain more precise estimates of causal effects. However, some genetic variants may not be valid instrumental variables, in particular due to them having more than one proximal phenotypic correlate (pleiotropy).|We view Mendelian randomization with multiple instruments as a meta-analysis, and show that bias caused by pleiotropy can be regarded as analogous to small study bias. Causal estimates using each instrument can be displayed visually by a funnel plot to assess potential asymmetry. Egger regression, a tool to detect small study bias in meta-analysis, can be adapted to test for bias from pleiotropy, and the slope coefficient from Egger regression provides an estimate of the causal effect. Under the assumption that the association of each genetic variant with the exposure is independent of the pleiotropic effect of the variant (not via the exposure), Egger's test gives a valid test of the null causal hypothesis and a consistent causal effect estimate even when all the genetic variants are invalid instrumental variables.|We illustrate the use of this approach by re-analysing two published Mendelian randomization studies of the causal effect of height on lung function, and the causal effect of blood pressure on coronary artery disease risk. The conservative nature of this approach is illustrated with these examples.|An adaption of Egger regression (which we call MR-Egger) can detect some violations of the standard instrumental variable assumptions, and provide an effect estimate which is not subject to these violations. The approach provides a sensitivity analysis for the robustness of the findings from a Mendelian randomization investigation. </li>
<li><strong>COPYRIGHT</strong> : © The Author 2015; Published by Oxford University Press on behalf of the International Epidemiological Association. </li>
<li><strong>DOI</strong> : 10.1093/ije/dyv080 </li>
<li><strong>JOURNAL_INFO</strong> : International journal of epidemiology ; Int J Epidemiol ; 2015 ; 44 ; 2 ; 512-25 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26050253">26050253</a> </li>
</ul>
<h3 id="review-hemani_g">Review-Hemani_G</h3>
<ul>
<li><strong>NAME</strong> : Review-Hemani_G </li>
<li><strong>TITLE</strong> : Orienting the causal relationship between imprecisely measured traits using GWAS summary data. </li>
<li><strong>CITATION</strong> : Hemani G, ...&amp;, Davey Smith G. (2017) Orienting the causal relationship between imprecisely measured traits using GWAS summary data. PLoS Genet, 13 (11) e1007081. doi:10.1371/journal.pgen.1007081. PMID 29149188 </li>
<li><strong>MESH_MAJOR</strong> :  </li>
<li><strong>MESH_MINOR</strong> : Bias, DNA Methylation, Gene Expression Regulation, Genetic Association Studies, Genome-Wide Association Study, Mendelian Randomization Analysis, Phenotype, Sample Size </li>
<li><strong>ABSTRACT</strong> : Inference about the causal structure that induces correlations between two traits can be achieved by combining genetic associations with a mediation-based approach, as is done in the causal inference test (CIT). However, we show that measurement error in the phenotypes can lead to the CIT inferring the wrong causal direction, and that increasing sample sizes has the adverse effect of increasing confidence in the wrong answer. This problem is likely to be general to other mediation-based approaches. Here we introduce an extension to Mendelian randomisation, a method that uses genetic associations in an instrumentation framework, that enables inference of the causal direction between traits, with some advantages. First, it can be performed using only summary level data from genome-wide association studies; second, it is less susceptible to bias in the presence of measurement error or unmeasured confounding. We apply the method to infer the causal direction between DNA methylation and gene expression levels. Our results demonstrate that, in general, DNA methylation is more likely to be the causal factor, but this result is highly susceptible to bias induced by systematic differences in measurement error between the platforms, and by horizontal pleiotropy. We emphasise that, where possible, implementing MR and appropriate sensitivity analyses alongside other approaches such as CIT is important to triangulate reliable conclusions about causality. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1007081 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2017 ; 13 ; 11 ; e1007081 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29149188">29149188</a> </li>
</ul>
<h3 id="review-howe_lj">Review-Howe_LJ</h3>
<ul>
<li><strong>NAME</strong> : Review-Howe_LJ </li>
<li><strong>TITLE</strong> : Interpreting Mendelian-randomization estimates of the effects of categorical exposures such as disease status and educational attainment. </li>
<li><strong>CITATION</strong> : Howe LJ, ...&amp;, Davies NM. (2022) Interpreting Mendelian-randomization estimates of the effects of categorical exposures such as disease status and educational attainment. Int J Epidemiol, 51 (3) 948-957. doi:10.1093/ije/dyab208. PMID 34570226 </li>
<li><strong>MESH_MAJOR</strong> : Diabetes Mellitus, Type 2 </li>
<li><strong>MESH_MINOR</strong> : Adult, Educational Status, Genome-Wide Association Study, Humans, Mendelian Randomization Analysis, Multifactorial Inheritance, Random Allocation </li>
<li><strong>ABSTRACT</strong> : Mendelian randomization has been previously used to estimate the effects of binary and ordinal categorical exposures-e.g. Type 2 diabetes or educational attainment defined by qualification-on outcomes. Binary and categorical phenotypes can be modelled in terms of liability-an underlying latent continuous variable with liability thresholds separating individuals into categories. Genetic variants influence an individual's categorical exposure via their effects on liability, thus Mendelian-randomization analyses with categorical exposures will capture effects of liability that act independently of exposure category.|We discuss how groups in which the categorical exposure is invariant can be used to detect liability effects acting independently of exposure category. For example, associations between an adult educational-attainment polygenic score (PGS) and body mass index measured before the minimum school leaving age (e.g. age 10 years), cannot indicate the effects of years in full-time education on this outcome. Using UK Biobank data, we show that a higher educational-attainment PGS is strongly associated with lower smoking initiation and higher odds of glasses use at age 15 years. These associations were replicated in sibling models. An orthogonal approach using the raising of the school leaving age (ROSLA) policy change found that individuals who chose to remain in education to age 16 years before the reform likely had higher liability to educational attainment than those who were compelled to remain in education to age 16 years after the reform, and had higher income, lower pack-years of smoking, higher odds of glasses use and lower deprivation in adulthood. These results suggest that liability to educational attainment is associated with health and social outcomes independently of years in full-time education.|Mendelian-randomization studies with non-continuous exposures should be interpreted in terms of liability, which may affect the outcome via changes in exposure category and/or independently. </li>
<li><strong>COPYRIGHT</strong> : © The Author(s) 2021. Published by Oxford University Press on behalf of the International Epidemiological Association. </li>
<li><strong>DOI</strong> : 10.1093/ije/dyab208 </li>
<li><strong>JOURNAL_INFO</strong> : International journal of epidemiology ; Int J Epidemiol ; 2022 ; 51 ; 3 ; 948-957 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34570226">34570226</a> </li>
</ul>
<h3 id="review-pierce_bl">Review-Pierce_BL</h3>
<ul>
<li><strong>NAME</strong> : Review-Pierce_BL </li>
<li><strong>TITLE</strong> : Efficient design for Mendelian randomization studies: subsample and 2-sample instrumental variable estimators. </li>
<li><strong>CITATION</strong> : Pierce BL, Burgess S. (2013) Efficient design for Mendelian randomization studies: subsample and 2-sample instrumental variable estimators. Am J Epidemiol, 178 (7) 1177-84. doi:10.1093/aje/kwt084. PMID 23863760 </li>
<li><strong>MESH_MAJOR</strong> :  </li>
<li><strong>MESH_MINOR</strong> : Computer Simulation, Humans, Mendelian Randomization Analysis, Sample Size </li>
<li><strong>ABSTRACT</strong> : Mendelian randomization (MR) is a method for estimating the causal relationship between an exposure and an outcome using a genetic factor as an instrumental variable (IV) for the exposure. In the traditional MR setting, data on the IV, exposure, and outcome are available for all participants. However, obtaining complete exposure data may be difficult in some settings, due to high measurement costs or lack of appropriate biospecimens. We used simulated data sets to assess statistical power and bias for MR when exposure data are available for a subset (or an independent set) of participants. We show that obtaining exposure data for a subset of participants is a cost-efficient strategy, often having negligible effects on power in comparison with a traditional complete-data analysis. The size of the subset needed to achieve maximum power depends on IV strength, and maximum power is approximately equal to the power of traditional IV estimators. Weak IVs are shown to lead to bias towards the null when the subsample is small and towards the confounded association when the subset is relatively large. Various approaches for confidence interval calculation are considered. These results have important implications for reducing the costs and increasing the feasibility of MR studies. </li>
<li><strong>DOI</strong> : 10.1093/aje/kwt084 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of epidemiology ; Am J Epidemiol ; 2013 ; 178 ; 7 ; 1177-84 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23863760">23863760</a> </li>
</ul>
<h3 id="review-smith_gd">Review-Smith_GD</h3>
<ul>
<li><strong>NAME</strong> : Review-Smith_GD </li>
<li><strong>TITLE</strong> : 'Mendelian randomization': can genetic epidemiology contribute to understanding environmental determinants of disease? </li>
<li><strong>CITATION</strong> : Davey Smith, G., &amp; Ebrahim, S. (2003). ‘Mendelian randomization’: can genetic epidemiology contribute to understanding environmental determinants of disease?. International journal of epidemiology, 32(1), 1-22. </li>
<li><strong>MESH_MAJOR</strong> : Environmental Exposure, Molecular Epidemiology </li>
<li><strong>MESH_MINOR</strong> : Animals, Disease, Genotype, Humans, Polymorphism, Genetic, Risk Assessment </li>
<li><strong>ABSTRACT</strong> : Associations between modifiable exposures and disease seen in observational epidemiology are sometimes confounded and thus misleading, despite our best efforts to improve the design and analysis of studies. Mendelian randomization-the random assortment of genes from parents to offspring that occurs during gamete formation and conception-provides one method for assessing the causal nature of some environmental exposures. The association between a disease and a polymorphism that mimics the biological link between a proposed exposure and disease is not generally susceptible to the reverse causation or confounding that may distort interpretations of conventional observational studies. Several examples where the phenotypic effects of polymorphisms are well documented provide encouraging evidence of the explanatory power of Mendelian randomization and are described. The limitations of the approach include confounding by polymorphisms in linkage disequilibrium with the polymorphism under study, that polymorphisms may have several phenotypic effects associated with disease, the lack of suitable polymorphisms for studying modifiable exposures of interest, and canalization-the buffering of the effects of genetic variation during development. Nevertheless, Mendelian randomization provides new opportunities to test causality and demonstrates how investment in the human genome project may contribute to understanding and preventing the adverse effects on human health of modifiable exposures. </li>
<li><strong>JOURNAL_INFO</strong> : International journal of epidemiology ; Int J Epidemiol ; 2003 ; 32 ; 1 ; 1-22 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/12689998">12689998</a> </li>
</ul>
<h2 id="misc">MISC</h2>
<h3 id="review-wu_y">Review-Wu_Y</h3>
<ul>
<li><strong>NAME</strong> : Review-Wu_Y </li>
<li><strong>SHORT NAME</strong> : SMR-multi </li>
<li><strong>TITLE</strong> : Integrative analysis of omics summary data reveals putative mechanisms underlying complex traits. </li>
<li><strong>CITATION</strong> : Wu Y, ...&amp;, Yang J. (2018) Integrative analysis of omics summary data reveals putative mechanisms underlying complex traits. Nat Commun, 9 (1) 918. doi:10.1038/s41467-018-03371-0. PMID 29500431 </li>
<li><strong>MESH_MAJOR</strong> : Quantitative Trait, Heritable </li>
<li><strong>MESH_MINOR</strong> : DNA Methylation, Genetic Predisposition to Disease, Genome-Wide Association Study, Genomics, Humans, Organ Specificity, Quantitative Trait Loci, RNA, Messenger, Regulatory Sequences, Nucleic Acid, Schizophrenia, Transcriptome </li>
<li><strong>ABSTRACT</strong> : The identification of genes and regulatory elements underlying the associations discovered by GWAS is essential to understanding the aetiology of complex traits (including diseases). Here, we demonstrate an analytical paradigm of prioritizing genes and regulatory elements at GWAS loci for follow-up functional studies. We perform an integrative analysis that uses summary-level SNP data from multi-omics studies to detect DNA methylation (DNAm) sites associated with gene expression and phenotype through shared genetic effects (i.e., pleiotropy). We identify pleiotropic associations between 7858 DNAm sites and 2733 genes. These DNAm sites are enriched in enhancers and promoters, and &gt;40% of them are mapped to distal genes. Further pleiotropic association analyses, which link both the methylome and transcriptome to 12 complex traits, identify 149 DNAm sites and 66 genes, indicating a plausible mechanism whereby the effect of a genetic variant on phenotype is mediated by genetic regulation of transcription through DNAm. </li>
<li><strong>DOI</strong> : 10.1038/s41467-018-03371-0 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2018 ; 9 ; 1 ; 918 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29500431">29500431</a> </li>
</ul>
<h3 id="review-zhu_z">Review-Zhu_Z</h3>
<ul>
<li><strong>NAME</strong> : Review-Zhu_Z </li>
<li><strong>SHORT NAME</strong> : SMR </li>
<li><strong>FULL NAME</strong> : Summary-data-based Mendelian Randomization </li>
<li><strong>DESCRIPTION</strong> : The SMR software tool was originally developed to implement the SMR &amp; HEIDI methods to test for pleiotropic association between the expression level of a gene and a complex trait of interest using summary-level data from GWAS and expression quantitative trait loci (eQTL) studies (Zhu et al. 2016 Nature Genetics). The SMR &amp; HEIDI methodology can be interpreted as an analysis to test if the effect size of a SNP on the phenotype is mediated by gene expression. This tool can therefore be used to prioritize genes underlying GWAS hits for follow-up functional studies. The methods are applicable to all kinds of molecular QTL (xQTL) data, including DNA methylation QTL (mQTL) and protein abundance QTL (pQTL). </li>
<li><strong>URL</strong> : https://yanglab.westlake.edu.cn/software/smr/#Overview </li>
<li><strong>KEYWORDS</strong> : pleiotropy or causality, xQTL, eQTL, MR, HEIDI, linkage </li>
<li><strong>TITLE</strong> : Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. </li>
<li><strong>CITATION</strong> : Zhu Z, ...&amp;, Yang J. (2016) Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Nat Genet, 48 (5) 481-7. doi:10.1038/ng.3538. PMID 27019110 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Pleiotropy, Genetic Techniques, Genome-Wide Association Study, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Data Interpretation, Statistical, Gene Expression Regulation, Genetic Linkage, Genetic Variation, Humans, Transcriptome </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) have identified thousands of genetic variants associated with human complex traits. However, the genes or functional DNA elements through which these variants exert their effects on the traits are often unknown. We propose a method (called SMR) that integrates summary-level data from GWAS with data from expression quantitative trait locus (eQTL) studies to identify genes whose expression levels are associated with a complex trait because of pleiotropy. We apply the method to five human complex traits using GWAS data on up to 339,224 individuals and eQTL data on 5,311 individuals, and we prioritize 126 genes (for example, TRAF1 and ANKRD55 for rheumatoid arthritis and SNX19 and NMRAL1 for schizophrenia), of which 25 genes are new candidates; 77 genes are not the nearest annotated gene to the top associated GWAS SNP. These genes provide important leads to design future functional studies to understand the mechanism whereby DNA variation leads to complex trait variation. </li>
<li><strong>DOI</strong> : 10.1038/ng.3538 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2016 ; 48 ; 5 ; 481-7 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27019110">27019110</a> </li>
</ul>
<h3 id="two-sample-mr">Two-sample-MR</h3>
<ul>
<li><strong>NAME</strong> : Two-sample-MR </li>
<li><strong>SHORT NAME</strong> : Two-sample MR </li>
<li><strong>FULL NAME</strong> : Two-sample MR </li>
<li><strong>DESCRIPTION</strong> : Two sample Mendelian randomisation (2SMR) is a method to estimate the causal effect of an exposure on an outcome using only summary statistics from genome wide association studies (GWAS). Though conceptually straightforward, there are a number of steps that are required to perform the analysis properly, and they can be cumbersome </li>
<li><strong>URL</strong> : https://mrcieu.github.io/TwoSampleMR/articles/introduction.html </li>
<li><strong>TITLE</strong> : The MR-Base platform supports systematic causal inference across the human phenome. </li>
<li><strong>CITATION</strong> : Hemani G, ...&amp;, Haycock PC. (2018) The MR-Base platform supports systematic causal inference across the human phenome. Elife, 7  . doi:10.7554/eLife.34408. PMID 29846171 </li>
<li><strong>MESH_MAJOR</strong> : Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Cholesterol, LDL, Coronary Disease, Databases, Genetic, Genetic Pleiotropy, Genome-Wide Association Study, Humans, Models, Genetic, Phenotype, Polymorphism, Single Nucleotide </li>
<li><strong>ABSTRACT</strong> : Results from genome-wide association studies (GWAS) can be used to infer causal relationships between phenotypes, using a strategy known as 2-sample Mendelian randomization (2SMR) and bypassing the need for individual-level data. However, 2SMR methods are evolving rapidly and GWAS results are often insufficiently curated, undermining efficient implementation of the approach. We therefore developed MR-Base (<ext-link ext-link-type="uri" xlink:href="http://www.mrbase.org">http://www.mrbase.org</ext-link>): a platform that integrates a curated database of complete GWAS results (no restrictions according to statistical significance) with an application programming interface, web app and R packages that automate 2SMR. The software includes several sensitivity analyses for assessing the impact of horizontal pleiotropy and other violations of assumptions. The database currently comprises 11 billion single nucleotide polymorphism-trait associations from 1673 GWAS and is updated on a regular basis. Integrating data with software ensures more rigorous application of hypothesis-driven analyses and allows millions of potential causal relationships to be efficiently evaluated in phenome-wide association studies. </li>
<li><strong>COPYRIGHT</strong> : © 2018, Hemani et al. </li>
<li><strong>DOI</strong> : 10.7554/eLife.34408 </li>
<li><strong>JOURNAL_INFO</strong> : eLife ; Elife ; 2018 ; 7 ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29846171">29846171</a> </li>
</ul>
<h2 id="reviewstutorials">Reviews&amp;Tutorials</h2>
<h3 id="review-benn_m">Review-Benn_M</h3>
<ul>
<li><strong>NAME</strong> : Review-Benn_M </li>
<li><strong>TITLE</strong> : From genome-wide association studies to Mendelian randomization: novel opportunities for understanding cardiovascular disease causality, pathogenesis, prevention, and treatment. </li>
<li><strong>CITATION</strong> : Benn M, Nordestgaard BG. (2018) From genome-wide association studies to Mendelian randomization: novel opportunities for understanding cardiovascular disease causality, pathogenesis, prevention, and treatment. Cardiovasc Res, 114 (9) 1192-1208. doi:10.1093/cvr/cvy045. PMID 29471399 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Variation, Genome-Wide Association Study, Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Cardiovascular Diseases, Gene-Environment Interaction, Genetic Predisposition to Disease, Humans, Life Style, Phenotype, Predictive Value of Tests, Prognosis, Risk Assessment, Risk Factors </li>
<li><strong>ABSTRACT</strong> : The Mendelian randomization approach is an epidemiological study design incorporating genetic information into traditional epidemiological studies to infer causality of biomarkers, risk factors, or lifestyle factors on disease risk. Mendelian randomization studies often draw on novel information generated in genome-wide association studies on causal associations between genetic variants and a risk factor or lifestyle factor. Such information can then be used in a largely unconfounded study design free of reverse causation to understand if and how risk factors and lifestyle factors cause cardiovascular disease. If causation is demonstrated, an opportunity for prevention of disease is identified; importantly however, before prevention or treatment can be implemented, randomized intervention trials altering risk factor levels or improving deleterious lifestyle factors needs to document reductions in cardiovascular disease in a safe and side-effect sparse manner. Documentation of causality can also inform on potential drug targets, more likely to be successful than prior approaches often relying on animal or cell studies mainly. The present review summarizes the history and background of Mendelian randomization, the study design, assumptions for using the design, and the most common caveats, followed by a discussion on advantages and disadvantages of different types of Mendelian randomization studies using one or more samples and different levels of information on study participants. The review also provides an overview of results on many of the risk factors and lifestyle factors for cardiovascular disease examined to date using the Mendelian randomization study design. </li>
<li><strong>DOI</strong> : 10.1093/cvr/cvy045 </li>
<li><strong>JOURNAL_INFO</strong> : Cardiovascular research ; Cardiovasc Res ; 2018 ; 114 ; 9 ; 1192-1208 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29471399">29471399</a> </li>
</ul>
<h3 id="review-davies_nm">Review-Davies_NM</h3>
<ul>
<li><strong>NAME</strong> : Review-Davies_NM </li>
<li><strong>TITLE</strong> : Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians. </li>
<li><strong>CITATION</strong> : Davies NM, ...&amp;, Davey Smith G. (2018) Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians. BMJ, 362  k601. doi:10.1136/bmj.k601. PMID 30002074 </li>
<li><strong>MESH_MAJOR</strong> : Causality, Confounding Factors, Epidemiologic, Effect Modifier, Epidemiologic, Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Genetic Variation, Genome-Wide Association Study, Humans, Observational Studies as Topic, Outcome Assessment, Health Care, Public Health </li>
<li><strong>ABSTRACT</strong> : Mendelian randomisation uses genetic variation as a natural experiment to investigate the causal relations between potentially modifiable risk factors and health outcomes in observational data. As with all epidemiological approaches, findings from Mendelian randomisation studies depend on specific assumptions. We provide explanations of the information typically reported in Mendelian randomisation studies that can be used to assess the plausibility of these assumptions and guidance on how to interpret findings from Mendelian randomisation studies in the context of other sources of evidence </li>
<li><strong>DOI</strong> : 10.1136/bmj.k601 </li>
<li><strong>JOURNAL_INFO</strong> : BMJ (Clinical research ed.) ; BMJ ; 2018 ; 362 ; <NA> ; k601 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30002074">30002074</a> </li>
</ul>
<h3 id="review-emdin_ca">Review-Emdin_CA</h3>
<ul>
<li><strong>NAME</strong> : Review-Emdin_CA </li>
<li><strong>TITLE</strong> : Mendelian Randomization. </li>
<li><strong>CITATION</strong> : Emdin CA, ...&amp;, Kathiresan S. (2017) Mendelian Randomization. JAMA, 318 (19) 1925-1926. doi:10.1001/jama.2017.17219. PMID 29164242 </li>
<li><strong>MESH_MAJOR</strong> : Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Cholesterol, HDL, Coronary Disease, Humans </li>
<li><strong>DOI</strong> : 10.1001/jama.2017.17219 </li>
<li><strong>JOURNAL_INFO</strong> : JAMA ; JAMA ; 2017 ; 318 ; 19 ; 1925-1926 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29164242">29164242</a> </li>
</ul>
<h3 id="review-lawlor_da">Review-Lawlor_DA</h3>
<ul>
<li><strong>NAME</strong> : Review-Lawlor_DA </li>
<li><strong>TITLE</strong> : Commentary: Two-sample Mendelian randomization: opportunities and challenges. </li>
<li><strong>CITATION</strong> : Lawlor DA. (2016) Commentary: Two-sample Mendelian randomization: opportunities and challenges. Int J Epidemiol, 45 (3) 908-15. doi:10.1093/ije/dyw127. PMID 27427429 </li>
<li><strong>MESH_MAJOR</strong> : Mendelian Randomization Analysis, Random Allocation </li>
<li><strong>MESH_MINOR</strong> : Genotype, Humans </li>
<li><strong>DOI</strong> : 10.1093/ije/dyw127 </li>
<li><strong>JOURNAL_INFO</strong> : International journal of epidemiology ; Int J Epidemiol ; 2016 ; 45 ; 3 ; 908-15 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27427429">27427429</a> </li>
</ul>
<h3 id="review-minelli_c">Review-Minelli_C</h3>
<ul>
<li><strong>NAME</strong> : Review-Minelli_C </li>
<li><strong>TITLE</strong> : The use of two-sample methods for Mendelian randomization analyses on single large datasets. </li>
<li><strong>CITATION</strong> : Minelli C, ...&amp;, Thompson J. (2021) The use of two-sample methods for Mendelian randomization analyses on single large datasets. Int J Epidemiol, 50 (5) 1651-1659. doi:10.1093/ije/dyab084. PMID 33899104 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Bias, Causality, Humans, Polymorphism, Single Nucleotide, Research Design </li>
<li><strong>ABSTRACT</strong> : With genome-wide association data for many exposures and outcomes now available from large biobanks, one-sample Mendelian randomization (MR) is increasingly used to investigate causal relationships. Many robust MR methods are available to address pleiotropy, but these assume independence between the gene-exposure and gene-outcome association estimates. Unlike in two-sample MR, in one-sample MR the two estimates are obtained from the same individuals, and the assumption of independence does not hold in the presence of confounding.|With simulations mimicking a typical study in UK Biobank, we assessed the performance, in terms of bias and precision of the MR estimate, of the fixed-effect and (multiplicative) random-effects meta-analysis method, weighted median estimator, weighted mode estimator and MR-Egger regression, used in both one-sample and two-sample data. We considered scenarios differing by the: presence/absence of a true causal effect; amount of confounding; and presence and type of pleiotropy (none, balanced or directional).|Even in the presence of substantial correlation due to confounding, all two-sample methods used in one-sample MR performed similarly to when used in two-sample MR, except for MR-Egger which resulted in bias reflecting direction and magnitude of the confounding. Such bias was much reduced in the presence of very high variability in instrument strength across variants (IGX2 of 97%).|Two-sample MR methods can be safely used for one-sample MR performed within large biobanks, expect for MR-Egger. MR-Egger is not recommended for one-sample MR unless the correlation between the gene-exposure and gene-outcome estimates due to confounding can be kept low, or the variability in instrument strength is very high. </li>
<li><strong>COPYRIGHT</strong> : © The Author(s) 2021. Published by Oxford University Press on behalf of the International Epidemiological Association. </li>
<li><strong>DOI</strong> : 10.1093/ije/dyab084 </li>
<li><strong>JOURNAL_INFO</strong> : International journal of epidemiology ; Int J Epidemiol ; 2021 ; 50 ; 5 ; 1651-1659 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33899104">33899104</a> </li>
</ul>
<h3 id="review-sanderson_e">Review-Sanderson_E</h3>
<ul>
<li><strong>NAME</strong> : Review-Sanderson_E </li>
<li><strong>TITLE</strong> : Mendelian randomization. </li>
<li><strong>CITATION</strong> : Sanderson E, ...&amp;, Smith GD. (2022) Mendelian randomization. Nat Rev Methods Primers, 2  . doi:10.1038/s43586-021-00092-5. PMID 37325194 </li>
<li><strong>ABSTRACT</strong> : Mendelian randomization (MR) is a term that applies to the use of genetic variation to address causal questions about how modifiable exposures influence different outcomes. The principles of MR are based on Mendel's laws of inheritance and instrumental variable estimation methods, which enable the inference of causal effects in the presence of unobserved confounding. In this Primer, we outline the principles of MR, the instrumental variable conditions underlying MR estimation and some of the methods used for estimation. We go on to discuss how the assumptions underlying an MR study can be assessed and give methods of estimation that are robust to certain violations of these assumptions. We give examples of a range of studies in which MR has been applied, the limitations of current methods of analysis and the outlook for MR in the future. The difference between the assumptions required for MR analysis and other forms of non-interventional epidemiological studies means that MR can be used as part of a triangulation across multiple sources of evidence for causal inference. </li>
<li><strong>DOI</strong> : 10.1038/s43586-021-00092-5 </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Methods primers ; Nat Rev Methods Primers ; 2022 ; 2 ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37325194">37325194</a> </li>
</ul>
<h3 id="review-teumer_a">Review-Teumer_A</h3>
<ul>
<li><strong>NAME</strong> : Review-Teumer_A </li>
<li><strong>TITLE</strong> : Common Methods for Performing Mendelian Randomization. </li>
<li><strong>CITATION</strong> : Teumer A. (2018) Common Methods for Performing Mendelian Randomization. Front Cardiovasc Med, 5  51. doi:10.3389/fcvm.2018.00051. PMID 29892602 </li>
<li><strong>ABSTRACT</strong> : Mendelian randomization (MR) is a framework for assessing causal inference using cross-sectional data in combination with genetic information. This paper summarizes statistical methods commonly applied and strait forward to use for conducting MR analyses including those taking advantage of the rich dataset of SNP-trait associations that were revealed in the last decade through large-scale genome-wide association studies. Using these data, powerful MR studies are possible. However, the causal estimate may be biased in case the assumptions of MR are violated. The source and the type of this bias are described while providing a summary of the mathematical formulas that should help estimating the magnitude and direction of the potential bias depending on the specific research setting. Finally, methods for relaxing the assumptions and for conducting sensitivity analyses are discussed. Future researches in the field of MR include the assessment of non-linear causal effects, and automatic detection of invalid instruments. </li>
<li><strong>DOI</strong> : 10.3389/fcvm.2018.00051 </li>
<li><strong>JOURNAL_INFO</strong> : Frontiers in cardiovascular medicine ; Front Cardiovasc Med ; 2018 ; 5 ; <NA> ; 51 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29892602">29892602</a> </li>
</ul>
<h3 id="review-zheng_j">Review-Zheng_J</h3>
<ul>
<li><strong>NAME</strong> : Review-Zheng_J </li>
<li><strong>TITLE</strong> : Recent Developments in Mendelian Randomization Studies. </li>
<li><strong>CITATION</strong> : Zheng J, ...&amp;, Smith GD. (2017) Recent Developments in Mendelian Randomization Studies. Curr Epidemiol Rep, 4 (4) 330-345. doi:10.1007/s40471-017-0128-6. PMID 29226067 </li>
<li><strong>ABSTRACT</strong> : Mendelian randomization (MR) is a strategy for evaluating causality in observational epidemiological studies. MR exploits the fact that genotypes are not generally susceptible to reverse causation and confounding, due to their fixed nature and Mendel's First and Second Laws of Inheritance. MR has the potential to provide information on causality in many situations where randomized controlled trials are not possible, but the results of MR studies must be interpreted carefully to avoid drawing erroneous conclusions.|In this review, we outline the principles behind MR, as well as assumptions and limitations of the method. Extensions to the basic approach are discussed, including two-sample MR, bidirectional MR, two-step MR, multivariable MR, and factorial MR. We also consider some new applications and recent developments in the methodology, including its ability to inform drug development, automation of the method using tools such as MR-Base, and phenome-wide and hypothesis-free MR.|In conjunction with the growing availability of large-scale genomic databases, higher level of automation and increased robustness of the methods, MR promises to be a valuable strategy to examine causality in complex biological/omics networks, inform drug development and prioritize intervention targets for disease prevention in the future. </li>
<li><strong>DOI</strong> : 10.1007/s40471-017-0128-6 </li>
<li><strong>JOURNAL_INFO</strong> : Current epidemiology reports ; Curr Epidemiol Rep ; 2017 ; 4 ; 4 ; 330-345 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29226067">29226067</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>