
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Population_Genetics_Admixture_README/">
      
      
        <link rel="next" href="../Population_Genetics_Phylogenetic_tree_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.13">
    
    
      
        <title>Genealogy - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Genealogy
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Phenotype_Phenotype_README/" class="md-tabs__link">
          
  
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Imaging
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Annotation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    RWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    TWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    eQTL
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    sQTL
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Colocalization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data_processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Dimension_reduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Drug_discovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Fine_mapping
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_prioritization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GxE_interactions
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    HLA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Imputation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mendelian_randomization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Power_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simulation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Winners_curse
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chromosome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Forest_plot
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heatmap
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    LD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variants_on_protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
      
        
        
      
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" checked>
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Admixture
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Genealogy
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Genealogy
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arg-needle" class="md-nav__link">
    <span class="md-ellipsis">
      ARG-Needle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arginfer" class="md-nav__link">
    <span class="md-ellipsis">
      ARGinfer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argweaver" class="md-nav__link">
    <span class="md-ellipsis">
      ARGweaver
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argweaver-d" class="md-nav__link">
    <span class="md-ellipsis">
      ARGweaver-D
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#asmc" class="md-nav__link">
    <span class="md-ellipsis">
      ASMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arbores" class="md-nav__link">
    <span class="md-ellipsis">
      Arbores
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kwarg" class="md-nav__link">
    <span class="md-ellipsis">
      KwARG
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#msmc" class="md-nav__link">
    <span class="md-ellipsis">
      MSMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#psmc" class="md-nav__link">
    <span class="md-ellipsis">
      PSMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rent" class="md-nav__link">
    <span class="md-ellipsis">
      RENT+
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relate" class="md-nav__link">
    <span class="md-ellipsis">
      Relate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sarge" class="md-nav__link">
    <span class="md-ellipsis">
      SARGE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsdate" class="md-nav__link">
    <span class="md-ellipsis">
      tsdate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsinfer" class="md-nav__link">
    <span class="md-ellipsis">
      tsinfer
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phylogenetic_tree
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Selection
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Reference
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Phenotype_Phenotype_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phenotype
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Gene_Gene_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variant
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Protein_Protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arg-needle" class="md-nav__link">
    <span class="md-ellipsis">
      ARG-Needle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arginfer" class="md-nav__link">
    <span class="md-ellipsis">
      ARGinfer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argweaver" class="md-nav__link">
    <span class="md-ellipsis">
      ARGweaver
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argweaver-d" class="md-nav__link">
    <span class="md-ellipsis">
      ARGweaver-D
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#asmc" class="md-nav__link">
    <span class="md-ellipsis">
      ASMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arbores" class="md-nav__link">
    <span class="md-ellipsis">
      Arbores
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kwarg" class="md-nav__link">
    <span class="md-ellipsis">
      KwARG
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#msmc" class="md-nav__link">
    <span class="md-ellipsis">
      MSMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#psmc" class="md-nav__link">
    <span class="md-ellipsis">
      PSMC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rent" class="md-nav__link">
    <span class="md-ellipsis">
      RENT+
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relate" class="md-nav__link">
    <span class="md-ellipsis">
      Relate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sarge" class="md-nav__link">
    <span class="md-ellipsis">
      SARGE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsdate" class="md-nav__link">
    <span class="md-ellipsis">
      tsdate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsinfer" class="md-nav__link">
    <span class="md-ellipsis">
      tsinfer
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>Genealogy</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#arg-needle">ARG-Needle</a></td>
<td style="text-align: left;">Zhang BC, Biddanda A, Gunnarsson ÁF, Cooper F, ...&amp;, Palamara PF. (2023) Biobank-scale inference of ancestral recombination graphs enables genealogical analysis of complex traits Nat. Genet., 55 (5) 768-776. doi:10.1038/s41588-023-01379-x. PMID 37127670</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#arginfer">ARGinfer</a></td>
<td style="text-align: left;">Mahmoudi A, Koskela J, Kelleher J, Chan YB, ...&amp;, Balding D. (2022) Bayesian inference of ancestral recombination graphs PLoS Comput. Biol., 18 (3) e1009960. doi:10.1371/journal.pcbi.1009960. PMID 35263345</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#argweaver-d">ARGweaver-D</a></td>
<td style="text-align: left;">Hubisz MJ, Williams AL, Siepel A. (2020) Mapping gene flow between ancient hominins through demography-aware inference of the ancestral recombination graph PLoS Genet., 16 (8) e1008895. doi:10.1371/journal.pgen.1008895. PMID 32760067</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#argweaver">ARGweaver</a></td>
<td style="text-align: left;">Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. (2014) Genome-wide inference of ancestral recombination graphs PLoS Genet., 10 (5) e1004342. doi:10.1371/journal.pgen.1004342. PMID 24831947</td>
<td style="text-align: left;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#asmc">ASMC</a></td>
<td style="text-align: left;">Palamara PF, Terhorst J, Song YS, Price AL. (2018) High-throughput inference of pairwise coalescence times identifies signals of selection and enriched disease heritability Nat. Genet., 50 (9) 1311-1317. doi:10.1038/s41588-018-0177-x. PMID 30104759</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#arbores">Arbores</a></td>
<td style="text-align: left;">Heine K, Beskos A, Jasra A, Balding D, ...&amp;, De Iorio M. (2018) Bridging trees for posterior inference on ancestral recombination graphs Proc. Math. Phys. Eng. Sci., 474 (2220) 20180568. doi:10.1098/rspa.2018.0568. PMID 30602937</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#kwarg">KwARG</a></td>
<td style="text-align: left;">Ignatieva A, Lyngsø RB, Jenkins PA, Hein J. (2021) KwARG: parsimonious reconstruction of ancestral recombination graphs with recurrent mutation Bioinformatics, 37 (19) 3277-3284. doi:10.1093/bioinformatics/btab351. PMID 33970217</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#msmc">MSMC</a></td>
<td style="text-align: left;">Schiffels, S. &amp; Durbin, R. Inferring human population size and separation history from multiple genome sequences. Nat. Genet. 46, 919–925 (2014).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#psmc">PSMC</a></td>
<td style="text-align: left;">Li H, Durbin R. (2011) Inference of human population history from individual whole-genome sequences Nature, 475 (7357) 493-496. doi:10.1038/nature10231. PMID 21753753</td>
<td style="text-align: left;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#rent">RENT+</a></td>
<td style="text-align: left;">Mirzaei S, Wu Y. (2017) RENT+: an improved method for inferring local genealogical trees from haplotypes with recombination Bioinformatics, 33 (7) 1021-1030. doi:10.1093/bioinformatics/btw735. PMID 28065901</td>
<td style="text-align: left;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#relate">Relate</a></td>
<td style="text-align: left;">Speidel L, Forest M, Shi S, Myers SR. (2019) A method for genome-wide genealogy estimation for thousands of samples Nat. Genet., 51 (9) 1321-1329. doi:10.1038/s41588-019-0484-x. PMID 31477933</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sarge">SARGE</a></td>
<td style="text-align: left;">Schaefer NK, Shapiro B, Green RE. (2021) An ancestral recombination graph of human, Neanderthal, and Denisovan genomes Sci. Adv., 7 (29) eabc0776. doi:10.1126/sciadv.abc0776. PMID 34272242</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tsdate">tsdate</a></td>
<td style="text-align: left;">Wohns AW, Wong Y, Jeffery B, Akbari A, ...&amp;, McVean G. (2022) A unified genealogy of modern and ancient genomes Science, 375 (6583) eabi8264. doi:10.1126/science.abi8264. PMID 35201891</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tsinfer">tsinfer</a></td>
<td style="text-align: left;">Kelleher J, Wong Y, Wohns AW, Fadil C, ...&amp;, McVean G. (2019) Inferring whole-genome histories in large population datasets Nat. Genet., 51 (9) 1330-1338. doi:10.1038/s41588-019-0483-y. PMID 31477934</td>
<td style="text-align: left;">2019</td>
</tr>
</tbody>
</table>
<h2 id="arg-needle">ARG-Needle</h2>
<ul>
<li><strong>NAME</strong> : ARG-Needle </li>
<li><strong>SHORT NAME</strong> : ARG-Needle </li>
<li><strong>URL</strong> : <a href="https://palamaralab.github.io/software/argneedle/">https://palamaralab.github.io/software/argneedle/</a> </li>
<li><strong>TITLE</strong> : Biobank-scale inference of ancestral recombination graphs enables genealogical analysis of complex traits </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01379-x </li>
<li><strong>ABSTRACT</strong> : Genome-wide genealogies compactly represent the evolutionary history of a set of genomes and inferring them from genetic data has the potential to facilitate a wide range of analyses. We introduce a method, ARG-Needle, for accurately inferring biobank-scale genealogies from sequencing or genotyping array data, as well as strategies to utilize genealogies to perform association and other complex trait analyses. We use these methods to build genome-wide genealogies using genotyping data for 337,464 UK Biobank individuals and test for association across seven complex traits. Genealogy-based association detects more rare and ultra-rare signals (N = 134, frequency range 0.0007-0.1%) than genotype imputation using ~65,000 sequenced haplotypes (N = 64). In a subset of 138,039 exome sequencing samples, these associations strongly tag (average r = 0.72) underlying sequencing variants enriched (4.8×) for loss-of-function variation. These results demonstrate that inferred genome-wide genealogies may be leveraged in the analysis of complex traits, complementing approaches that require the availability of large, population-specific sequencing panels. </li>
<li><strong>CITATION</strong> : Zhang BC, Biddanda A, Gunnarsson ÁF, Cooper F, ...&amp;, Palamara PF. (2023) Biobank-scale inference of ancestral recombination graphs enables genealogical analysis of complex traits Nat. Genet., 55 (5) 768-776. doi:10.1038/s41588-023-01379-x. PMID 37127670 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2023 ; 55 ; 5 ; 768-776 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37127670">37127670</a> </li>
</ul>
<h2 id="arginfer">ARGinfer</h2>
<ul>
<li><strong>NAME</strong> : ARGinfer </li>
<li><strong>SHORT NAME</strong> : ARGinfer </li>
<li><strong>TITLE</strong> : Bayesian inference of ancestral recombination graphs </li>
<li><strong>DOI</strong> : 10.1371/journal.pcbi.1009960 </li>
<li><strong>ABSTRACT</strong> : We present a novel algorithm, implemented in the software ARGinfer, for probabilistic inference of the Ancestral Recombination Graph under the Coalescent with Recombination. Our Markov Chain Monte Carlo algorithm takes advantage of the Succinct Tree Sequence data structure that has allowed great advances in simulation and point estimation, but not yet probabilistic inference. Unlike previous methods, which employ the Sequentially Markov Coalescent approximation, ARGinfer uses the Coalescent with Recombination, allowing more accurate inference of key evolutionary parameters. We show using simulations that ARGinfer can accurately estimate many properties of the evolutionary history of the sample, including the topology and branch lengths of the genealogical tree at each sequence site, and the times and locations of mutation and recombination events. ARGinfer approximates posterior probability distributions for these and other quantities, providing interpretable assessments of uncertainty that we show to be well calibrated. ARGinfer is currently limited to tens of DNA sequences of several hundreds of kilobases, but has scope for further computational improvements to increase its applicability. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Mahmoudi A, Koskela J, Kelleher J, Chan YB, ...&amp;, Balding D. (2022) Bayesian inference of ancestral recombination graphs PLoS Comput. Biol., 18 (3) e1009960. doi:10.1371/journal.pcbi.1009960. PMID 35263345 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS computational biology ; PLoS Comput. Biol. ; 2022 ; 18 ; 3 ; e1009960 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35263345">35263345</a> </li>
</ul>
<h2 id="argweaver">ARGweaver</h2>
<ul>
<li><strong>NAME</strong> : ARGweaver </li>
<li><strong>SHORT NAME</strong> : ARGweaver </li>
<li><strong>DESCRIPTION</strong> : The ARGweaver software package contains programs and libraries for sampling and manipulating ancestral recombination graphs (ARGs). An ARG is a rich data structure for representing the ancestry of DNA sequences undergoing coalescence and recombination. </li>
<li><strong>URL</strong> : <a href="https://github.com/mdrasmus/argweaver">https://github.com/mdrasmus/argweaver</a> </li>
<li><strong>TITLE</strong> : Genome-wide inference of ancestral recombination graphs </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1004342 </li>
<li><strong>ABSTRACT</strong> : The complex correlation structure of a collection of orthologous DNA sequences is uniquely captured by the "ancestral recombination graph" (ARG), a complete record of coalescence and recombination events in the history of the sample. However, existing methods for ARG inference are computationally intensive, highly approximate, or limited to small numbers of sequences, and, as a consequence, explicit ARG inference is rarely used in applied population genomics. Here, we introduce a new algorithm for ARG inference that is efficient enough to apply to dozens of complete mammalian genomes. The key idea of our approach is to sample an ARG of [Formula: see text] chromosomes conditional on an ARG of [Formula: see text] chromosomes, an operation we call "threading." Using techniques based on hidden Markov models, we can perform this threading operation exactly, up to the assumptions of the sequentially Markov coalescent and a discretization of time. An extension allows for threading of subtrees instead of individual sequences. Repeated application of these threading operations results in highly efficient Markov chain Monte Carlo samplers for ARGs. We have implemented these methods in a computer program called ARGweaver. Experiments with simulated data indicate that ARGweaver converges rapidly to the posterior distribution over ARGs and is effective in recovering various features of the ARG for dozens of sequences generated under realistic parameters for human populations. In applications of ARGweaver to 54 human genome sequences from Complete Genomics, we find clear signatures of natural selection, including regions of unusually ancient ancestry associated with balancing selection and reductions in allele age in sites under directional selection. The patterns we observe near protein-coding genes are consistent with a primary influence from background selection rather than hitchhiking, although we cannot rule out a contribution from recurrent selective sweeps. </li>
<li><strong>CITATION</strong> : Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. (2014) Genome-wide inference of ancestral recombination graphs PLoS Genet., 10 (5) e1004342. doi:10.1371/journal.pgen.1004342. PMID 24831947 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2014 ; 10 ; 5 ; e1004342 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24831947">24831947</a> </li>
</ul>
<h2 id="argweaver-d">ARGweaver-D</h2>
<ul>
<li><strong>NAME</strong> : ARGweaver-D </li>
<li><strong>SHORT NAME</strong> : ARGweaver-D </li>
<li><strong>TITLE</strong> : Mapping gene flow between ancient hominins through demography-aware inference of the ancestral recombination graph </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1008895 </li>
<li><strong>ABSTRACT</strong> : The sequencing of Neanderthal and Denisovan genomes has yielded many new insights about interbreeding events between extinct hominins and the ancestors of modern humans. While much attention has been paid to the relatively recent gene flow from Neanderthals and Denisovans into modern humans, other instances of introgression leave more subtle genomic evidence and have received less attention. Here, we present a major extension of the ARGweaver algorithm, called ARGweaver-D, which can infer local genetic relationships under a user-defined demographic model that includes population splits and migration events. This Bayesian algorithm probabilistically samples ancestral recombination graphs (ARGs) that specify not only tree topologies and branch lengths along the genome, but also indicate migrant lineages. The sampled ARGs can therefore be parsed to produce probabilities of introgression along the genome. We show that this method is well powered to detect the archaic migration into modern humans, even with only a few samples. We then show that the method can also detect introgressed regions stemming from older migration events, or from unsampled populations. We apply it to human, Neanderthal, and Denisovan genomes, looking for signatures of older proposed migration events, including ancient humans into Neanderthal, and unknown archaic hominins into Denisovans. We identify 3% of the Neanderthal genome that is putatively introgressed from ancient humans, and estimate that the gene flow occurred between 200-300kya. We find no convincing evidence that negative selection acted against these regions. Finally, we predict that 1% of the Denisovan genome was introgressed from an unsequenced, but highly diverged, archaic hominin ancestor. About 15% of these "super-archaic" regions-comprising at least about 4Mb-were, in turn, introgressed into modern humans and continue to exist in the genomes of people alive today. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Hubisz MJ, Williams AL, Siepel A. (2020) Mapping gene flow between ancient hominins through demography-aware inference of the ancestral recombination graph PLoS Genet., 16 (8) e1008895. doi:10.1371/journal.pgen.1008895. PMID 32760067 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2020 ; 16 ; 8 ; e1008895 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32760067">32760067</a> </li>
</ul>
<h2 id="asmc">ASMC</h2>
<ul>
<li><strong>NAME</strong> : ASMC </li>
<li><strong>SHORT NAME</strong> : ASMC </li>
<li><strong>FULL NAME</strong> : Ascertained Sequentially Markovian Coalescent </li>
<li><strong>DESCRIPTION</strong> : The Ascertained Sequentially Markovian Coalescent is a method to efficiently estimate pairwise coalescence time along the genome. It can be run using SNP array or whole-genome sequencing (WGS) data. </li>
<li><strong>URL</strong> : <a href="https://github.com/pierpal/ASMC">https://github.com/pierpal/ASMC</a> </li>
<li><strong>TITLE</strong> : High-throughput inference of pairwise coalescence times identifies signals of selection and enriched disease heritability </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0177-x </li>
<li><strong>ABSTRACT</strong> : Interest in reconstructing demographic histories has motivated the development of methods to estimate locus-specific pairwise coalescence times from whole-genome sequencing data. Here we introduce a powerful new method, ASMC, that can estimate coalescence times using only SNP array data, and is orders of magnitude faster than previous approaches. We applied ASMC to detect recent positive selection in 113,851 phased British samples from the UK Biobank, and detected 12 genome-wide significant signals, including 6 novel loci. We also applied ASMC to sequencing data from 498 Dutch individuals to detect background selection at deeper time scales. We detected strong heritability enrichment in regions of high background selection in an analysis of 20 independent diseases and complex traits using stratified linkage disequilibrium score regression, conditioned on a broad set of functional annotations (including other background selection annotations). These results underscore the widespread effects of background selection on the genetic architecture of complex traits. </li>
<li><strong>CITATION</strong> : Palamara PF, Terhorst J, Song YS, Price AL. (2018) High-throughput inference of pairwise coalescence times identifies signals of selection and enriched disease heritability Nat. Genet., 50 (9) 1311-1317. doi:10.1038/s41588-018-0177-x. PMID 30104759 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2018 ; 50 ; 9 ; 1311-1317 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30104759">30104759</a> </li>
</ul>
<h2 id="arbores">Arbores</h2>
<ul>
<li><strong>NAME</strong> : Arbores </li>
<li><strong>SHORT NAME</strong> : Arbores </li>
<li><strong>TITLE</strong> : Bridging trees for posterior inference on ancestral recombination graphs </li>
<li><strong>DOI</strong> : 10.1098/rspa.2018.0568 </li>
<li><strong>ABSTRACT</strong> : We present a new Markov chain Monte Carlo algorithm, implemented in the software Arbores, for inferring the history of a sample of DNA sequences. Our principal innovation is a bridging procedure, previously applied only for simple stochastic processes, in which the local computations within a bridge can proceed independently of the rest of the DNA sequence, facilitating large-scale parallelization. </li>
<li><strong>COPYRIGHT</strong> : https://royalsociety.org/-/media/journals/author/Licence-to-Publish-20062019-final.pdf </li>
<li><strong>CITATION</strong> : Heine K, Beskos A, Jasra A, Balding D, ...&amp;, De Iorio M. (2018) Bridging trees for posterior inference on ancestral recombination graphs Proc. Math. Phys. Eng. Sci., 474 (2220) 20180568. doi:10.1098/rspa.2018.0568. PMID 30602937 </li>
<li><strong>JOURNAL_INFO</strong> : Proceedings. Mathematical, physical, and engineering sciences ; Proc. Math. Phys. Eng. Sci. ; 2018 ; 474 ; 2220 ; 20180568 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30602937">30602937</a> </li>
</ul>
<h2 id="kwarg">KwARG</h2>
<ul>
<li><strong>NAME</strong> : KwARG </li>
<li><strong>SHORT NAME</strong> : KwARG </li>
<li><strong>TITLE</strong> : KwARG: parsimonious reconstruction of ancestral recombination graphs with recurrent mutation </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btab351 </li>
<li><strong>ABSTRACT</strong> : MOTIVATION: The reconstruction of possible histories given a sample of genetic data in the presence of recombination and recurrent mutation is a challenging problem, but can provide key insights into the evolution of a population. We present KwARG, which implements a parsimony-based greedy heuristic algorithm for finding plausible genealogical histories (ancestral recombination graphs) that are minimal or near-minimal in the number of posited recombination and mutation events. RESULTS: Given an input dataset of aligned sequences, KwARG outputs a list of possible candidate solutions, each comprising a list of mutation and recombination events that could have generated the dataset; the relative proportion of recombinations and recurrent mutations in a solution can be controlled via specifying a set of 'cost' parameters. We demonstrate that the algorithm performs well when compared against existing methods. AVAILABILITY AND IMPLEMENTATION: The software is available at https://github.com/a-ignatieva/kwarg. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Ignatieva A, Lyngsø RB, Jenkins PA, Hein J. (2021) KwARG: parsimonious reconstruction of ancestral recombination graphs with recurrent mutation Bioinformatics, 37 (19) 3277-3284. doi:10.1093/bioinformatics/btab351. PMID 33970217 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2021 ; 37 ; 19 ; 3277-3284 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33970217">33970217</a> </li>
</ul>
<h2 id="msmc">MSMC</h2>
<ul>
<li><strong>NAME</strong> : MSMC </li>
<li><strong>SHORT NAME</strong> : MSMC </li>
<li><strong>URL</strong> : <a href="https://github.com/stschiff/msmc">https://github.com/stschiff/msmc</a> </li>
<li><strong>CITATION</strong> : Schiffels, S. &amp; Durbin, R. Inferring human population size and separation history from multiple genome sequences. Nat. Genet. 46, 919–925 (2014). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24952747">24952747</a> </li>
</ul>
<h2 id="psmc">PSMC</h2>
<ul>
<li><strong>NAME</strong> : PSMC </li>
<li><strong>SHORT NAME</strong> : PSMC </li>
<li><strong>DESCRIPTION</strong> : This software package infers population size history from a diploid sequence
using the Pairwise Sequentially Markovian Coalescent (PSMC) model. </li>
<li><strong>URL</strong> : <a href="https://github.com/lh3/psmc">https://github.com/lh3/psmc</a> </li>
<li><strong>TITLE</strong> : Inference of human population history from individual whole-genome sequences </li>
<li><strong>DOI</strong> : 10.1038/nature10231 </li>
<li><strong>ABSTRACT</strong> : The history of human population size is important for understanding human evolution. Various studies have found evidence for a founder event (bottleneck) in East Asian and European populations, associated with the human dispersal out-of-Africa event around 60 thousand years (kyr) ago. However, these studies have had to assume simplified demographic models with few parameters, and they do not provide a precise date for the start and stop times of the bottleneck. Here, with fewer assumptions on population size changes, we present a more detailed history of human population sizes between approximately ten thousand and a million years ago, using the pairwise sequentially Markovian coalescent model applied to the complete diploid genome sequences of a Chinese male (YH), a Korean male (SJK), three European individuals (J. C. Venter, NA12891 and NA12878 (ref. 9)) and two Yoruba males (NA18507 (ref. 10) and NA19239). We infer that European and Chinese populations had very similar population-size histories before 10-20 kyr ago. Both populations experienced a severe bottleneck 10-60 kyr ago, whereas African populations experienced a milder bottleneck from which they recovered earlier. All three populations have an elevated effective population size between 60 and 250 kyr ago, possibly due to population substructure. We also infer that the differentiation of genetically modern humans may have started as early as 100-120 kyr ago, but considerable genetic exchanges may still have occurred until 20-40 kyr ago. </li>
<li><strong>CITATION</strong> : Li H, Durbin R. (2011) Inference of human population history from individual whole-genome sequences Nature, 475 (7357) 493-496. doi:10.1038/nature10231. PMID 21753753 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2011 ; 475 ; 7357 ; 493-496 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21753753">21753753</a> </li>
</ul>
<h2 id="rent">RENT+</h2>
<ul>
<li><strong>NAME</strong> : RENT+ </li>
<li><strong>SHORT NAME</strong> : RENT+ </li>
<li><strong>TITLE</strong> : RENT+: an improved method for inferring local genealogical trees from haplotypes with recombination </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btw735 </li>
<li><strong>ABSTRACT</strong> : Motivation: : Haplotypes from one or multiple related populations share a common genealogical history. If this shared genealogy can be inferred from haplotypes, it can be very useful for many population genetics problems. However, with the presence of recombination, the genealogical history of haplotypes is complex and cannot be represented by a single genealogical tree. Therefore, inference of genealogical history with recombination is much more challenging than the case of no recombination. Results: : In this paper, we present a new approach called RENT+ for the inference of local genealogical trees from haplotypes with the presence of recombination. RENT+ builds on a previous genealogy inference approach called RENT , which infers a set of related genealogical trees at different genomic positions. RENT+ represents a significant improvement over RENT in the sense that it is more effective in extracting information contained in the haplotype data about the underlying genealogy than RENT . The key components of RENT+ are several greatly enhanced genealogy inference rules. Through simulation, we show that RENT+ is more efficient and accurate than several existing genealogy inference methods. As an application, we apply RENT+ in the inference of population demographic history from haplotypes, which outperforms several existing methods. Availability and Implementation: : RENT+ is implemented in Java, and is freely available for download from: https://github.com/SajadMirzaei/RentPlus . Contacts: : sajad@engr.uconn.edu or ywu@engr.uconn.edu. Supplementary information: : Supplementary data are available at Bioinformatics online. </li>
<li><strong>CITATION</strong> : Mirzaei S, Wu Y. (2017) RENT+: an improved method for inferring local genealogical trees from haplotypes with recombination Bioinformatics, 33 (7) 1021-1030. doi:10.1093/bioinformatics/btw735. PMID 28065901 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2017 ; 33 ; 7 ; 1021-1030 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28065901">28065901</a> </li>
</ul>
<h2 id="relate">Relate</h2>
<ul>
<li><strong>NAME</strong> : Relate </li>
<li><strong>SHORT NAME</strong> : Relate </li>
<li><strong>DESCRIPTION</strong> : Relate estimates genome-wide genealogies in the form of trees that adapt to changes in local ancestry caused by recombination. The method, which is scalable to thousands of samples, is described in the following paper. Please cite this paper if you use our software in your study. </li>
<li><strong>URL</strong> : <a href="https://myersgroup.github.io/relate/">https://myersgroup.github.io/relate/</a> </li>
<li><strong>TITLE</strong> : A method for genome-wide genealogy estimation for thousands of samples </li>
<li><strong>DOI</strong> : 10.1038/s41588-019-0484-x </li>
<li><strong>ABSTRACT</strong> : Knowledge of genome-wide genealogies for thousands of individuals would simplify most evolutionary analyses for humans and other species, but has remained computationally infeasible. We have developed a method, Relate, scaling to &gt;10,000 sequences while simultaneously estimating branch lengths, mutational ages and variable historical population sizes, as well as allowing for data errors. Application to 1,000 Genomes Project haplotypes produces joint genealogical histories for 26 human populations. Highly diverged lineages are present in all groups, but most frequent in Africa. Outside Africa, these mainly reflect ancient introgression from groups related to Neanderthals and Denisovans, while African signals instead reflect unknown events unique to that continent. Our approach allows more powerful inferences of natural selection than has previously been possible. We identify multiple regions under strong positive selection, and multi-allelic traits including hair color, body mass index and blood pressure, showing strong evidence of directional selection, varying among human groups. </li>
<li><strong>CITATION</strong> : Speidel L, Forest M, Shi S, Myers SR. (2019) A method for genome-wide genealogy estimation for thousands of samples Nat. Genet., 51 (9) 1321-1329. doi:10.1038/s41588-019-0484-x. PMID 31477933 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2019 ; 51 ; 9 ; 1321-1329 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31477933">31477933</a> </li>
</ul>
<h2 id="sarge">SARGE</h2>
<ul>
<li><strong>NAME</strong> : SARGE </li>
<li><strong>SHORT NAME</strong> : SARGE </li>
<li><strong>TITLE</strong> : An ancestral recombination graph of human, Neanderthal, and Denisovan genomes </li>
<li><strong>DOI</strong> : 10.1126/sciadv.abc0776 </li>
<li><strong>ABSTRACT</strong> : Many humans carry genes from Neanderthals, a legacy of past admixture. Existing methods detect this archaic hominin ancestry within human genomes using patterns of linkage disequilibrium or direct comparison to Neanderthal genomes. Each of these methods is limited in sensitivity and scalability. We describe a new ancestral recombination graph inference algorithm that scales to large genome-wide datasets and demonstrate its accuracy on real and simulated data. We then generate a genome-wide ancestral recombination graph including human and archaic hominin genomes. From this, we generate a map within human genomes of archaic ancestry and of genomic regions not shared with archaic hominins either by admixture or incomplete lineage sorting. We find that only 1.5 to 7% of the modern human genome is uniquely human. We also find evidence of multiple bursts of adaptive changes specific to modern humans within the past 600,000 years involving genes related to brain development and function. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Schaefer NK, Shapiro B, Green RE. (2021) An ancestral recombination graph of human, Neanderthal, and Denisovan genomes Sci. Adv., 7 (29) eabc0776. doi:10.1126/sciadv.abc0776. PMID 34272242 </li>
<li><strong>JOURNAL_INFO</strong> : Science advances ; Sci. Adv. ; 2021 ; 7 ; 29 ; eabc0776 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34272242">34272242</a> </li>
</ul>
<h2 id="tsdate">tsdate</h2>
<ul>
<li><strong>NAME</strong> : tsdate </li>
<li><strong>SHORT NAME</strong> : tsdate </li>
<li><strong>DESCRIPTION</strong> : The tsdate program [Wohns et al., 2022] infers dates for nodes in a genetic genealogy, sometimes loosely known as an ancestral recombination graph or ARG [Wong et al., 2023]. More precisely, it takes a genealogy in tree sequence format as an input and returns a copy of that tree sequence with altered node and mutation times. These times have been estimated on the basis of the number of mutations along the edges connecting genomes in the genealogy (i.e. using the “molecular clock”). </li>
<li><strong>URL</strong> : <a href="https://tskit.dev/tsdate/docs/latest/introduction.html">https://tskit.dev/tsdate/docs/latest/introduction.html</a> </li>
<li><strong>TITLE</strong> : A unified genealogy of modern and ancient genomes </li>
<li><strong>DOI</strong> : 10.1126/science.abi8264 </li>
<li><strong>ABSTRACT</strong> : The sequencing of modern and ancient genomes from around the world has revolutionized our understanding of human history and evolution. However, the problem of how best to characterize ancestral relationships from the totality of human genomic variation remains unsolved. Here, we address this challenge with nonparametric methods that enable us to infer a unified genealogy of modern and ancient humans. This compact representation of multiple datasets explores the challenges of missing and erroneous data and uses ancient samples to constrain and date relationships. We demonstrate the power of the method to recover relationships between individuals and populations as well as to identify descendants of ancient samples. Finally, we introduce a simple nonparametric estimator of the geographical location of ancestors that recapitulates key events in human history. </li>
<li><strong>CITATION</strong> : Wohns AW, Wong Y, Jeffery B, Akbari A, ...&amp;, McVean G. (2022) A unified genealogy of modern and ancient genomes Science, 375 (6583) eabi8264. doi:10.1126/science.abi8264. PMID 35201891 </li>
<li><strong>JOURNAL_INFO</strong> : Science (New York, N.Y.) ; Science ; 2022 ; 375 ; 6583 ; eabi8264 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35201891">35201891</a> </li>
</ul>
<h2 id="tsinfer">tsinfer</h2>
<ul>
<li><strong>NAME</strong> : tsinfer </li>
<li><strong>SHORT NAME</strong> : tsinfer </li>
<li><strong>DESCRIPTION</strong> : Infer a tree sequence from genetic variation data </li>
<li><strong>URL</strong> : <a href="https://github.com/tskit-dev/tsinfer">https://github.com/tskit-dev/tsinfer</a> </li>
<li><strong>TITLE</strong> : Inferring whole-genome histories in large population datasets </li>
<li><strong>DOI</strong> : 10.1038/s41588-019-0483-y </li>
<li><strong>ABSTRACT</strong> : Inferring the full genealogical history of a set of DNA sequences is a core problem in evolutionary biology, because this history encodes information about the events and forces that have influenced a species. However, current methods are limited, and the most accurate techniques are able to process no more than a hundred samples. As datasets that consist of millions of genomes are now being collected, there is a need for scalable and efficient inference methods to fully utilize these resources. Here we introduce an algorithm that is able to not only infer whole-genome histories with comparable accuracy to the state-of-the-art but also process four orders of magnitude more sequences. The approach also provides an 'evolutionary encoding' of the data, enabling efficient calculation of relevant statistics. We apply the method to human data from the 1000 Genomes Project, Simons Genome Diversity Project and UK Biobank, showing that the inferred genealogies are rich in biological signal and efficient to process. </li>
<li><strong>CITATION</strong> : Kelleher J, Wong Y, Wohns AW, Fadil C, ...&amp;, McVean G. (2019) Inferring whole-genome histories in large population datasets Nat. Genet., 51 (9) 1330-1338. doi:10.1038/s41588-019-0483-y. PMID 31477934 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2019 ; 51 ; 9 ; 1330-1338 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31477934">31477934</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.13a4f30d.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>