
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Heritability_and_genetic_correlation_README/">
      
      
        <link rel="next" href="../Tools_Mendelian_randomization_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.13">
    
    
      
        <title>Imputation - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Imputation
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Phenotype_Phenotype_README/" class="md-tabs__link">
          
  
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Imaging
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
      
        
        
      
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Annotation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    RWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    TWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    eQTL
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    sQTL
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Colocalization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data_processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Dimension_reduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Drug_discovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Fine_mapping
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_prioritization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GxE_interactions
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    HLA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Imputation
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Imputation
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1000-genomes-phase-3-version-5-1kgp3v5" class="md-nav__link">
    <span class="md-ellipsis">
      1000 Genomes Phase 3 Version 5 (1KGp3v5)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1kg7k" class="md-nav__link">
    <span class="md-ellipsis">
      1KG+7K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ckb-reference-panel" class="md-nav__link">
    <span class="md-ellipsis">
      CKB reference panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chinamap-panel" class="md-nav__link">
    <span class="md-ellipsis">
      ChinaMAP panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomeasia-100k" class="md-nav__link">
    <span class="md-ellipsis">
      GenomeAsia 100K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hgdp1kgp" class="md-nav__link">
    <span class="md-ellipsis">
      HGDP+1kGP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hrc" class="md-nav__link">
    <span class="md-ellipsis">
      HRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard" class="md-nav__link">
    <span class="md-ellipsis">
      NARD
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard2" class="md-nav__link">
    <span class="md-ellipsis">
      NARD2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-genome-resource-phase-1" class="md-nav__link">
    <span class="md-ellipsis">
      Nyuwa Genome Resource Phase 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgghan-panel" class="md-nav__link">
    <span class="md-ellipsis">
      PGG.Han panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#south-and-east-asian-reference-database-sead" class="md-nav__link">
    <span class="md-ellipsis">
      South and East Asian Reference Database (SEAD)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wbbc-panel" class="md-nav__link">
    <span class="md-ellipsis">
      WBBC panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation server
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation server">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cngb-imputation-service" class="md-nav__link">
    <span class="md-ellipsis">
      CNGB Imputation Service
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chinamap" class="md-nav__link">
    <span class="md-ellipsis">
      ChinaMAP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#michigan-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Michigan Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      NyuWa Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgghan" class="md-nav__link">
    <span class="md-ellipsis">
      PGG.Han
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sanger" class="md-nav__link">
    <span class="md-ellipsis">
      Sanger
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed_1" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#westlake-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Westlake Imputation Server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-tools" class="md-nav__link">
    <span class="md-ellipsis">
      Other tools
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Other tools">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reshape" class="md-nav__link">
    <span class="md-ellipsis">
      RESHAPE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasing-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      Phasing &amp; Imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Phasing &amp; Imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle4" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-imputation" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Imputation)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Phasing)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle2" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimpse" class="md-nav__link">
    <span class="md-ellipsis">
      GLIMPSE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute2" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute4" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute5" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach-pre-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach pre-phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach2" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach3" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach4" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quilt1" class="md-nav__link">
    <span class="md-ellipsis">
      QUILT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quilt2" class="md-nav__link">
    <span class="md-ellipsis">
      QUILT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit1" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit2" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit3" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit5" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastphase" class="md-nav__link">
    <span class="md-ellipsis">
      fastPHASE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-das" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Das
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-li" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Li
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-marchini" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Marchini
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structural-variants-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Structural variants imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Structural variants imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1kg-sv-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      1KG SV imputation panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mendelian_randomization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Power_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simulation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Winners_curse
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chromosome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Forest_plot
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heatmap
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    LD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variants_on_protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Admixture
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Genealogy_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genealogy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phylogenetic_tree
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Selection
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Reference
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Phenotype_Phenotype_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phenotype
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Gene_Gene_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variant
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Protein_Protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1000-genomes-phase-3-version-5-1kgp3v5" class="md-nav__link">
    <span class="md-ellipsis">
      1000 Genomes Phase 3 Version 5 (1KGp3v5)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1kg7k" class="md-nav__link">
    <span class="md-ellipsis">
      1KG+7K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ckb-reference-panel" class="md-nav__link">
    <span class="md-ellipsis">
      CKB reference panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chinamap-panel" class="md-nav__link">
    <span class="md-ellipsis">
      ChinaMAP panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genomeasia-100k" class="md-nav__link">
    <span class="md-ellipsis">
      GenomeAsia 100K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hgdp1kgp" class="md-nav__link">
    <span class="md-ellipsis">
      HGDP+1kGP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hrc" class="md-nav__link">
    <span class="md-ellipsis">
      HRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard" class="md-nav__link">
    <span class="md-ellipsis">
      NARD
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard2" class="md-nav__link">
    <span class="md-ellipsis">
      NARD2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-genome-resource-phase-1" class="md-nav__link">
    <span class="md-ellipsis">
      Nyuwa Genome Resource Phase 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgghan-panel" class="md-nav__link">
    <span class="md-ellipsis">
      PGG.Han panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#south-and-east-asian-reference-database-sead" class="md-nav__link">
    <span class="md-ellipsis">
      South and East Asian Reference Database (SEAD)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wbbc-panel" class="md-nav__link">
    <span class="md-ellipsis">
      WBBC panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation server
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation server">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cngb-imputation-service" class="md-nav__link">
    <span class="md-ellipsis">
      CNGB Imputation Service
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chinamap" class="md-nav__link">
    <span class="md-ellipsis">
      ChinaMAP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#michigan-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Michigan Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      NyuWa Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgghan" class="md-nav__link">
    <span class="md-ellipsis">
      PGG.Han
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sanger" class="md-nav__link">
    <span class="md-ellipsis">
      Sanger
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed_1" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#westlake-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Westlake Imputation Server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-tools" class="md-nav__link">
    <span class="md-ellipsis">
      Other tools
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Other tools">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reshape" class="md-nav__link">
    <span class="md-ellipsis">
      RESHAPE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasing-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      Phasing &amp; Imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Phasing &amp; Imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle4" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-imputation" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Imputation)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Phasing)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle2" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimpse" class="md-nav__link">
    <span class="md-ellipsis">
      GLIMPSE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute2" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute4" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute5" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach-pre-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach pre-phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach2" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach3" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach4" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quilt1" class="md-nav__link">
    <span class="md-ellipsis">
      QUILT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#quilt2" class="md-nav__link">
    <span class="md-ellipsis">
      QUILT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit1" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit2" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit3" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit5" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastphase" class="md-nav__link">
    <span class="md-ellipsis">
      fastPHASE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-das" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Das
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-li" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Li
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-marchini" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Marchini
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structural-variants-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Structural variants imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Structural variants imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1kg-sv-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      1KG SV imputation panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>Imputation</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#1000-genomes-phase-3-version-5-1kgp3v5">1000 Genomes Phase 3 Version 5 (1KGp3v5)</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">1000 Genomes Project Consortium, Auton A, Brooks LD, Durbin RM, ...&amp;, Abecasis GR. (2015) A global reference for human genetic variation Nature, 526 (7571) 68-74. doi:10.1038/nature15393. PMID 26432245</td>
<td style="text-align: left;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#1kg7k">1KG+7K</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Terao, C., Flanagan, J., Tomizuka, K., Liu, X., Ortega-Reyes, D., Matoba, N., ... &amp; Horikoshi, M. (2023). Population-specific reference panel improves imputation quality and enhances locus discovery and fine-mapping.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ckb-reference-panel">CKB reference panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yu C, Lan X, Tao Y, Guo Y, ...&amp;, Li L. (2023) A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study Nucleic Acids Res., 51 (21) 11770-11782. doi:10.1093/nar/gkad779. PMID 37870428</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#chinamap-panel">ChinaMAP panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Li L, Huang P, Sun X, Wang S, ...&amp;, Wang W. (2021) The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations Cell Res., 31 (12) 1308-1310. doi:10.1038/s41422-021-00564-z. PMID 34489580</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#genomeasia-100k">GenomeAsia 100K</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">GenomeAsia100K Consortium. (2019) The GenomeAsia 100K Project enables genetic discoveries across Asia Nature, 576 (7785) 106-111. doi:10.1038/s41586-019-1793-z. PMID 31802016</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hgdp1kgp">HGDP+1kGP</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Koenig Z, Yohannes MT, Nkambule LL, Zhao X, ...&amp;, Martin AR. (2024) A harmonized public resource of deeply sequenced diverse human genomes Genome Res., 34 (5) 796-809. doi:10.1101/gr.278378.123. PMID 38749656</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hrc">HRC</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">McCarthy S, Das S, Kretzschmar W, Delaneau O, ...&amp;, Marchini J. (2016) A reference panel of 64,976 haplotypes for genotype imputation Nat. Genet., 48 (10) 1279-1283. doi:10.1038/ng.3643. PMID 27548312</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nard2">NARD2</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Choi J, Kim S, Kim J, Son HY, ...&amp;, Im SW. (2023) A whole-genome reference panel of 14,393 individuals for East Asian populations accelerates discovery of rare functional variants Sci Adv, 9 (32) eadg6319. doi:10.1126/sciadv.adg6319. PMID 37556544</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nard">NARD</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yoo SK, Kim CU, Kim HL, Kim S, ...&amp;, Seo JS. (2019) NARD: whole-genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low-frequency variants Genome Med., 11 (1) 64. doi:10.1186/s13073-019-0677-z. PMID 31640730</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nyuwa-genome-resource-phase-1">Nyuwa Genome Resource Phase 1</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Zhang P, Luo H, Li Y, Wang Y, ...&amp;, He S. (2021) NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population Cell Rep., 37 (7) 110017. doi:10.1016/j.celrep.2021.110017. PMID 34788621</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgghan-panel">PGG.Han panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Gao Y, Zhang C, Yuan L, Ling Y, ...&amp;, Xu S. (2020) PGG.Han: the Han Chinese genome database and analysis platform Nucleic Acids Res., 48 (D1) D971-D976. doi:10.1093/nar/gkz829. PMID 31584086</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#south-and-east-asian-reference-database-sead">South and East Asian Reference Database (SEAD)</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yang, M. Y., Zhong, J. D., Li, X., Bai, W. Y., Yuan, C. D., Qiu, M. C., ... &amp; Zheng, H. F. (2023). SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population. medRxiv, 2023-12.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#topmed">TOPMED</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Taliun D, Harris DN, Kessler MD, Carlson J, ...&amp;, Abecasis GR. (2021) Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program Nature, 590 (7845) 290-299. doi:10.1038/s41586-021-03205-y. PMID 33568819</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wbbc-panel">WBBC panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Cong PK, Bai WY, Li JC, Yang MY, ...&amp;, Zheng HF. (2022) Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project Nat. Commun., 13 (1) 2939. doi:10.1038/s41467-022-30526-x. PMID 35618720</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cngb-imputation-service">CNGB Imputation Service</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Yu C, Lan X, Tao Y, Guo Y, ...&amp;, Li L. (2023) A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study Nucleic Acids Res., 51 (21) 11770-11782. doi:10.1093/nar/gkad779. PMID 37870428</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#chinamap">ChinaMAP</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Li L, Huang P, Sun X, Wang S, ...&amp;, Wang W. (2021) The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations Cell Res., 31 (12) 1308-1310. doi:10.1038/s41422-021-00564-z. PMID 34489580</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#michigan-imputation-server">Michigan Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Das S, Forer L, Schönherr S, Sidore C, ...&amp;, Fuchsberger C. (2016) Next-generation genotype imputation service and methods Nat. Genet., 48 (10) 1284-1287. doi:10.1038/ng.3656. PMID 27571263</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nyuwa-imputation-server">NyuWa Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Zhang P, Luo H, Li Y, Wang Y, ...&amp;, He S. (2021) NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population Cell Rep., 37 (7) 110017. doi:10.1016/j.celrep.2021.110017. PMID 34788621</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgghan">PGG.Han</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Gao Y, Zhang C, Yuan L, Ling Y, ...&amp;, Xu S. (2020) PGG.Han: the Han Chinese genome database and analysis platform Nucleic Acids Res., 48 (D1) D971-D976. doi:10.1093/nar/gkz829. PMID 31584086</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sanger">Sanger</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">McCarthy S, Das S, Kretzschmar W, Delaneau O, ...&amp;, Marchini J. (2016) A reference panel of 64,976 haplotypes for genotype imputation Nat. Genet., 48 (10) 1279-1283. doi:10.1038/ng.3643. PMID 27548312</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#topmed">TOPMED</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Taliun D, Harris DN, Kessler MD, Carlson J, ...&amp;, Abecasis GR. (2021) Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program Nature, 590 (7845) 290-299. doi:10.1038/s41586-021-03205-y. PMID 33568819</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#westlake-imputation-server">Westlake Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Cong PK, Bai WY, Li JC, Yang MY, ...&amp;, Zheng HF. (2022) Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project Nat. Commun., 13 (1) 2939. doi:10.1038/s41467-022-30526-x. PMID 35618720</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#reshape">RESHAPE</a></td>
<td style="text-align: left;">Other tools</td>
<td style="text-align: left;">Cavinato T, Rubinacci S, Malaspinas AS, Delaneau O. (2024) A resampling-based approach to share reference panels Nat. Comput. Sci., 4 (5) 360-366. doi:10.1038/s43588-024-00630-7. PMID 38745108</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle4">BEAGLE4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning BL, Browning SR. (2016) Genotype imputation with millions of reference samples Am. J. Hum. Genet., 98 (1) 116-126. doi:10.1016/j.ajhg.2015.11.020. PMID 26748515</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle54-imputation">BEAGLE5.4 (Imputation)</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning BL, Zhou Y, Browning SR. (2018) A one-penny imputed genome from next-generation reference panels Am. J. Hum. Genet., 103 (3) 338-348. doi:10.1016/j.ajhg.2018.07.015. PMID 30100085</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle54-phasing">BEAGLE5.4 (Phasing)</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning BL, Tian X, Zhou Y, Browning SR. (2021) Fast two-stage phasing of large-scale sequence data Am. J. Hum. Genet., 108 (10) 1880-1890. doi:10.1016/j.ajhg.2021.08.005. PMID 34478634</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle">BEAGLE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning SR, Browning BL. (2007) Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering Am. J. Hum. Genet., 81 (5) 1084-1097. doi:10.1086/521987. PMID 17924348</td>
<td style="text-align: left;">2007</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#eagle2">EAGLE2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Loh PR, Danecek P, Palamara PF, Fuchsberger C, ...&amp;, L Price A. (2016) Reference-based phasing using the Haplotype Reference Consortium panel Nat. Genet., 48 (11) 1443-1448. doi:10.1038/ng.3679. PMID 27694958</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#eagle">EAGLE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Loh PR, Palamara PF, Price AL. (2016) Fast and accurate long-range phasing in a UK Biobank cohort Nat. Genet., 48 (7) 811-816. doi:10.1038/ng.3571. PMID 27270109</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#glimpse">GLIMPSE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Rubinacci S, Ribeiro DM, Hofmeister RJ, Delaneau O. (2021) Efficient phasing and imputation of low-coverage sequencing data using large reference panels Nat. Genet., 53 (1) 120-126. doi:10.1038/s41588-020-00756-0. PMID 33414550</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute2">IMPUTE2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Howie BN, Donnelly P, Marchini J. (2009) A flexible and accurate genotype imputation method for the next generation of genome-wide association studies PLoS Genet., 5 (6) e1000529. doi:10.1371/journal.pgen.1000529. PMID 19543373</td>
<td style="text-align: left;">2009</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute4">IMPUTE4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Bycroft C, Freeman C, Petkova D, Band G, ...&amp;, Marchini J. (2018) The UK Biobank resource with deep phenotyping and genomic data Nature, 562 (7726) 203-209. doi:10.1038/s41586-018-0579-z. PMID 30305743</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute5">IMPUTE5</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Rubinacci S, Delaneau O, Marchini J. (2020) Genotype imputation using the Positional Burrows Wheeler Transform PLoS Genet., 16 (11) e1009049. doi:10.1371/journal.pgen.1009049. PMID 33196638</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute">IMPUTE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Marchini J, Howie B, Myers S, McVean G, ...&amp;, Donnelly P. (2007) A new multipoint method for genome-wide association studies by imputation of genotypes Nat. Genet., 39 (7) 906-913. doi:10.1038/ng2088. PMID 17572673</td>
<td style="text-align: left;">2007</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach-pre-phasing">MACH / minimach pre-phasing</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Howie B, Fuchsberger C, Stephens M, Marchini J, ...&amp;, Abecasis GR. (2012) Fast and accurate genotype imputation in genome-wide association studies through pre-phasing Nat. Genet., 44 (8) 955-959. doi:10.1038/ng.2354. PMID 22820512</td>
<td style="text-align: left;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach2">MACH / minimach2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Fuchsberger C, Abecasis GR, Hinds DA. (2015) Minimac2: Faster genotype imputation Bioinformatics, 31 (5) 782-784. doi:10.1093/bioinformatics/btu704. PMID 25338720</td>
<td style="text-align: left;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach3">MACH / minimach3</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Das S, Forer L, Schönherr S, Sidore C, ...&amp;, Fuchsberger C. (2016) Next-generation genotype imputation service and methods Nat. Genet., 48 (10) 1284-1287. doi:10.1038/ng.3656. PMID 27571263</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach4">MACH / minimach4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach">MACH / minimach</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Li Y, Willer CJ, Ding J, Scheet P, ...&amp;, Abecasis GR. (2010) MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes Genet. Epidemiol., 34 (8) 816-834. doi:10.1002/gepi.20533. PMID 21058334</td>
<td style="text-align: left;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#quilt1">QUILT1</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Davies RW, Kucka M, Su D, Shi S, ...&amp;, Myers S. (2021) Rapid genotype imputation from sequence with reference panels Nat. Genet., 53 (7) 1104-1111. doi:10.1038/s41588-021-00877-0. PMID 34083788</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#quilt2">QUILT2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Li, Z., Albrechtsen, A. &amp; Davies, R. W. Rapid and accurate genotype imputation from low coverage short read, long read, and cell free DNA sequence. bioRxiv 2024.07.18.604149 (2024) doi:10.1101/2024.07.18.604149.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit1">SHAPEIT1</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Delaneau O, Marchini J, Zagury JF. (2011) A linear complexity phasing method for thousands of genomes Nat. Methods, 9 (2) 179-181. doi:10.1038/nmeth.1785. PMID 22138821</td>
<td style="text-align: left;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit2">SHAPEIT2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Delaneau O, Zagury JF, Marchini J. (2013) Improved whole-chromosome phasing for disease and population genetic studies Nat. Methods, 10 (1) 5-6. doi:10.1038/nmeth.2307. PMID 23269371</td>
<td style="text-align: left;">2013</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit3">SHAPEIT3</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">O'Connell J, Sharp K, Shrine N, Wain L, ...&amp;, Marchini J. (2016) Haplotype estimation for biobank-scale data sets Nat. Genet., 48 (7) 817-820. doi:10.1038/ng.3583. PMID 27270105</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit4">SHAPEIT4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Delaneau O, Zagury JF, Robinson MR, Marchini JL, ...&amp;, Dermitzakis ET. (2019) Accurate, scalable and integrative haplotype estimation Nat. Commun., 10 (1) 5436. doi:10.1038/s41467-019-13225-y. PMID 31780650</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit5">SHAPEIT5</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Hofmeister RJ, Ribeiro DM, Rubinacci S, Delaneau O. (2022) Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank Nat. Genet., 55 (7) 1243-1249. doi:10.1038/s41588-023-01415-w. PMID 37386248</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastphase">fastPHASE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Scheet P, Stephens M. (2006) A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase Am. J. Hum. Genet., 78 (4) 629-644. doi:10.1086/502802. PMID 16532393</td>
<td style="text-align: left;">2006</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-das">Review-Das</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Das S, Abecasis GR, Browning BL. (2018) Genotype imputation from large reference panels Annu. Rev. Genomics Hum. Genet., 19 (1) 73-96. doi:10.1146/annurev-genom-083117-021602. PMID 29799802</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-li">Review-Li</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Li Y, Willer C, Sanna S, Abecasis G. (2009) Genotype imputation Annu. Rev. Genomics Hum. Genet., 10 (1) 387-406. doi:10.1146/annurev.genom.9.081307.164242. PMID 19715440</td>
<td style="text-align: left;">2009</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-marchini">Review-Marchini</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Marchini J, Howie B. (2010) Genotype imputation for genome-wide association studies Nat. Rev. Genet., 11 (7) 499-511. doi:10.1038/nrg2796. PMID 20517342</td>
<td style="text-align: left;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#1kg-sv-imputation-panel">1KG SV imputation panel</a></td>
<td style="text-align: left;">Structural variants imputation panel</td>
<td style="text-align: left;">Noyvert, B., Erzurumluoglu, A. M., Drichel, D., Omland, S., Andlauer, T. F., Mueller, S., ... &amp; Ding, Z. (2023). Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations. medRxiv, 2023-12.</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="imputation-panel">Imputation panel</h2>
<h3 id="1000-genomes-phase-3-version-5-1kgp3v5">1000 Genomes Phase 3 Version 5 (1KGp3v5)</h3>
<ul>
<li><strong>NAME</strong> : 1000 Genomes Phase 3 Version 5 (1KGp3v5) </li>
<li><strong>SHORT NAME</strong> : 1KG </li>
<li><strong>URL</strong> : <a href="https://www.internationalgenome.org/">https://www.internationalgenome.org/</a> </li>
<li><strong>TITLE</strong> : A global reference for human genetic variation </li>
<li><strong>DOI</strong> : 10.1038/nature15393 </li>
<li><strong>ABSTRACT</strong> : The 1000 Genomes Project set out to provide a comprehensive description of common human genetic variation by applying whole-genome sequencing to a diverse set of individuals from multiple populations. Here we report completion of the project, having reconstructed the genomes of 2,504 individuals from 26 populations using a combination of low-coverage whole-genome sequencing, deep exome sequencing, and dense microarray genotyping. We characterized a broad spectrum of genetic variation, in total over 88 million variants (84.7 million single nucleotide polymorphisms (SNPs), 3.6 million short insertions/deletions (indels), and 60,000 structural variants), all phased onto high-quality haplotypes. This resource includes &gt;99% of SNP variants with a frequency of &gt;1% for a variety of ancestries. We describe the distribution of genetic variation across the global sample, and discuss the implications for common disease studies. </li>
<li><strong>CITATION</strong> : 1000 Genomes Project Consortium, Auton A, Brooks LD, Durbin RM, ...&amp;, Abecasis GR. (2015) A global reference for human genetic variation Nature, 526 (7571) 68-74. doi:10.1038/nature15393. PMID 26432245 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2015 ; 526 ; 7571 ; 68-74 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26432245">26432245</a> </li>
</ul>
<h3 id="1kg7k">1KG+7K</h3>
<ul>
<li><strong>NAME</strong> : 1KG+7K </li>
<li><strong>SHORT NAME</strong> : 1KG+7K </li>
<li><strong>FULL NAME</strong> : 1KG+7K </li>
<li><strong>KEYWORDS</strong> : Japanese population-specific reference panel </li>
<li><strong>PREPRINT_DOI</strong> : 10.21203/rs.3.rs-3194976/v1 </li>
<li><strong>SERVER</strong> : researchsquare </li>
<li><strong>CITATION</strong> : Terao, C., Flanagan, J., Tomizuka, K., Liu, X., Ortega-Reyes, D., Matoba, N., ... &amp; Horikoshi, M. (2023). Population-specific reference panel improves imputation quality and enhances locus discovery and fine-mapping. </li>
</ul>
<h3 id="ckb-reference-panel">CKB reference panel</h3>
<ul>
<li><strong>NAME</strong> : CKB reference panel </li>
<li><strong>SHORT NAME</strong> : CKB </li>
<li><strong>FULL NAME</strong> : China Kadoorie Biobank </li>
<li><strong>URL</strong> : <a href="https://db.cngb.org/imputation/">https://db.cngb.org/imputation/</a> </li>
<li><strong>TITLE</strong> : A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study </li>
<li><strong>DOI</strong> : 10.1093/nar/gkad779 </li>
<li><strong>ABSTRACT</strong> : Precision medicine depends on high-accuracy individual-level genotype data. However, the whole-genome sequencing (WGS) is still not suitable for gigantic studies due to budget constraints. It is particularly important to construct highly accurate haplotype reference panel for genotype imputation. In this study, we used 10 000 samples with medium-depth WGS to construct a reference panel that we named the CKB reference panel. By imputing microarray datasets, it showed that the CKB panel outperformed compared panels in terms of both the number of well-imputed variants and imputation accuracy. In addition, we have completed the imputation of 100 706 microarrays with the CKB panel, and the after-imputed data is the hitherto largest whole genome data of the Chinese population. Furthermore, in the GWAS analysis of real phenotype height, the number of tested SNPs tripled and the number of significant SNPs doubled after imputation. Finally, we developed an online server for offering free genotype imputation service based on the CKB reference panel (https://db.cngb.org/imputation/). We believe that the CKB panel is of great value for imputing microarray or low-coverage genotype data of Chinese population, and potentially mixed populations. The imputation-completed 100 706 microarray data are enormous and precious resources of population genetic studies for complex traits and diseases. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Yu C, Lan X, Tao Y, Guo Y, ...&amp;, Li L. (2023) A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study Nucleic Acids Res., 51 (21) 11770-11782. doi:10.1093/nar/gkad779. PMID 37870428 </li>
<li><strong>JOURNAL_INFO</strong> : Nucleic acids research ; Nucleic Acids Res. ; 2023 ; 51 ; 21 ; 11770-11782 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37870428">37870428</a> </li>
</ul>
<h3 id="chinamap-panel">ChinaMAP panel</h3>
<ul>
<li><strong>NAME</strong> : ChinaMAP panel </li>
<li><strong>SHORT NAME</strong> : ChinaMAP </li>
<li><strong>FULL NAME</strong> : China Metabolic Analytics Project </li>
<li><strong>URL</strong> : <a href="http://www.mbiobank.com/">http://www.mbiobank.com/</a> </li>
<li><strong>TITLE</strong> : The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations </li>
<li><strong>DOI</strong> : 10.1038/s41422-021-00564-z </li>
<li><strong>ABSTRACT</strong> : The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Li L, Huang P, Sun X, Wang S, ...&amp;, Wang W. (2021) The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations Cell Res., 31 (12) 1308-1310. doi:10.1038/s41422-021-00564-z. PMID 34489580 </li>
<li><strong>JOURNAL_INFO</strong> : Cell research ; Cell Res. ; 2021 ; 31 ; 12 ; 1308-1310 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34489580">34489580</a> </li>
</ul>
<h3 id="genomeasia-100k">GenomeAsia 100K</h3>
<ul>
<li><strong>NAME</strong> : GenomeAsia 100K </li>
<li><strong>SHORT NAME</strong> : GenomeAsia 100K </li>
<li><strong>URL</strong> : <a href="https://www.genomeasia100k.org/">https://www.genomeasia100k.org/</a> </li>
<li><strong>TITLE</strong> : The GenomeAsia 100K Project enables genetic discoveries across Asia </li>
<li><strong>DOI</strong> : 10.1038/s41586-019-1793-z </li>
<li><strong>ABSTRACT</strong> : The underrepresentation of non-Europeans in human genetic studies so far has limited the diversity of individuals in genomic datasets and led to reduced medical relevance for a large proportion of the world's population. Population-specific reference genome datasets as well as genome-wide association studies in diverse populations are needed to address this issue. Here we describe the pilot phase of the GenomeAsia 100K Project. This includes a whole-genome sequencing reference dataset from 1,739 individuals of 219 population groups and 64 countries across Asia. We catalogue genetic variation, population structure, disease associations and founder effects. We also explore the use of this dataset in imputation, to facilitate genetic studies in populations across Asia and worldwide. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : GenomeAsia100K Consortium. (2019) The GenomeAsia 100K Project enables genetic discoveries across Asia Nature, 576 (7785) 106-111. doi:10.1038/s41586-019-1793-z. PMID 31802016 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2019 ; 576 ; 7785 ; 106-111 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31802016">31802016</a> </li>
</ul>
<h3 id="hgdp1kgp">HGDP+1kGP</h3>
<ul>
<li><strong>NAME</strong> : HGDP+1kGP </li>
<li><strong>SHORT NAME</strong> : HGDP+1kGP </li>
<li><strong>FULL NAME</strong> : Human Genome Diversity Project + 1000 Genomes project </li>
<li><strong>URL</strong> : <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability/#the-gnomad-hgdp-and-1000-genomes-callset">https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability/#the-gnomad-hgdp-and-1000-genomes-callset</a> </li>
<li><strong>TITLE</strong> : A harmonized public resource of deeply sequenced diverse human genomes </li>
<li><strong>DOI</strong> : 10.1101/gr.278378.123 </li>
<li><strong>ABSTRACT</strong> : Underrepresented populations are often excluded from genomic studies owing in part to a lack of resources supporting their analyses. The 1000 Genomes Project (1kGP) and Human Genome Diversity Project (HGDP), which have recently been sequenced to high coverage, are valuable genomic resources because of the global diversity they capture and their open data sharing policies. Here, we harmonized a high-quality set of 4094 whole genomes from 80 populations in the HGDP and 1kGP with data from the Genome Aggregation Database (gnomAD) and identified over 153 million high-quality SNVs, indels, and SVs. We performed a detailed ancestry analysis of this cohort, characterizing population structure and patterns of admixture across populations, analyzing site frequency spectra, and measuring variant counts at global and subcontinental levels. We also show substantial added value from this data set compared with the prior versions of the component resources, typically combined via liftOver and variant intersection; for example, we catalog millions of new genetic variants, mostly rare, compared with previous releases. In addition to unrestricted individual-level public release, we provide detailed tutorials for conducting many of the most common quality-control steps and analyses with these data in a scalable cloud-computing environment and publicly release this new phased joint callset for use as a haplotype resource in phasing and imputation pipelines. This jointly called reference panel will serve as a key resource to support research of diverse ancestry populations. </li>
<li><strong>CITATION</strong> : Koenig Z, Yohannes MT, Nkambule LL, Zhao X, ...&amp;, Martin AR. (2024) A harmonized public resource of deeply sequenced diverse human genomes Genome Res., 34 (5) 796-809. doi:10.1101/gr.278378.123. PMID 38749656 </li>
<li><strong>JOURNAL_INFO</strong> : Genome research ; Genome Res. ; 2024 ; 34 ; 5 ; 796-809 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38749656">38749656</a> </li>
</ul>
<h3 id="hrc">HRC</h3>
<ul>
<li><strong>NAME</strong> : HRC </li>
<li><strong>SHORT NAME</strong> : HRC </li>
<li><strong>URL</strong> : <a href="http://www.haplotype-reference-consortium.org/">http://www.haplotype-reference-consortium.org/</a> </li>
<li><strong>TITLE</strong> : A reference panel of 64,976 haplotypes for genotype imputation </li>
<li><strong>DOI</strong> : 10.1038/ng.3643 </li>
<li><strong>ABSTRACT</strong> : We describe a reference panel of 64,976 human haplotypes at 39,235,157 SNPs constructed using whole-genome sequence data from 20 studies of predominantly European ancestry. Using this resource leads to accurate genotype imputation at minor allele frequencies as low as 0.1% and a large increase in the number of SNPs tested in association studies, and it can help to discover and refine causal loci. We describe remote server resources that allow researchers to carry out imputation and phasing consistently and efficiently. </li>
<li><strong>CITATION</strong> : McCarthy S, Das S, Kretzschmar W, Delaneau O, ...&amp;, Marchini J. (2016) A reference panel of 64,976 haplotypes for genotype imputation Nat. Genet., 48 (10) 1279-1283. doi:10.1038/ng.3643. PMID 27548312 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 10 ; 1279-1283 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27548312">27548312</a> </li>
</ul>
<h3 id="nard">NARD</h3>
<ul>
<li><strong>NAME</strong> : NARD </li>
<li><strong>SHORT NAME</strong> : NARD </li>
<li><strong>FULL NAME</strong> : Northeast Asian Reference Database </li>
<li><strong>URL</strong> : <a href="https://nard.macrogen.com/">https://nard.macrogen.com/</a> </li>
<li><strong>TITLE</strong> : NARD: whole-genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low-frequency variants </li>
<li><strong>DOI</strong> : 10.1186/s13073-019-0677-z </li>
<li><strong>ABSTRACT</strong> : Here, we present the Northeast Asian Reference Database (NARD), including whole-genome sequencing data of 1779 individuals from Korea, Mongolia, Japan, China, and Hong Kong. NARD provides the genetic diversity of Korean (n = 850) and Mongolian (n = 384) ancestries that were not present in the 1000 Genomes Project Phase 3 (1KGP3). We combined and re-phased the genotypes from NARD and 1KGP3 to construct a union set of haplotypes. This approach established a robust imputation reference panel for Northeast Asians, which yields the greatest imputation accuracy of rare and low-frequency variants compared with the existing panels. NARD imputation panel is available at https://nard.macrogen.com/ . </li>
<li><strong>CITATION</strong> : Yoo SK, Kim CU, Kim HL, Kim S, ...&amp;, Seo JS. (2019) NARD: whole-genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low-frequency variants Genome Med., 11 (1) 64. doi:10.1186/s13073-019-0677-z. PMID 31640730 </li>
<li><strong>JOURNAL_INFO</strong> : Genome medicine ; Genome Med. ; 2019 ; 11 ; 1 ; 64 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31640730">31640730</a> </li>
</ul>
<h3 id="nard2">NARD2</h3>
<ul>
<li><strong>NAME</strong> : NARD2 </li>
<li><strong>SHORT NAME</strong> : NARD2 </li>
<li><strong>FULL NAME</strong> : Northeast Asian Reference Database 2 </li>
<li><strong>URL</strong> : <a href="https://nard.macrogen.com/">https://nard.macrogen.com/</a> </li>
<li><strong>TITLE</strong> : A whole-genome reference panel of 14,393 individuals for East Asian populations accelerates discovery of rare functional variants </li>
<li><strong>DOI</strong> : 10.1126/sciadv.adg6319 </li>
<li><strong>ABSTRACT</strong> : Underrepresentation of non-European (EUR) populations hinders growth of global precision medicine. Resources such as imputation reference panels that match the study population are necessary to find low-frequency variants with substantial effects. We created a reference panel consisting of 14,393 whole-genome sequences including more than 11,000 Asian individuals. Genome-wide association studies were conducted using the reference panel and a population-specific genotype array of 72,298 subjects for eight phenotypes. This panel yields improved imputation accuracy of rare and low-frequency variants within East Asian populations compared with the largest reference panel. Thirty-nine previously unidentified associations were found, and more than half of the variants were East Asian specific. We discovered genes with rare protein-altering variants, including LTBP1 for height and GPR75 for body mass index, as well as putative regulatory mechanisms for rare noncoding variants with cell type-specific effects. We suggest that this dataset will add to the potential value of Asian precision medicine. </li>
<li><strong>CITATION</strong> : Choi J, Kim S, Kim J, Son HY, ...&amp;, Im SW. (2023) A whole-genome reference panel of 14,393 individuals for East Asian populations accelerates discovery of rare functional variants Sci Adv, 9 (32) eadg6319. doi:10.1126/sciadv.adg6319. PMID 37556544 </li>
<li><strong>JOURNAL_INFO</strong> : Science advances ; Sci Adv ; 2023 ; 9 ; 32 ; eadg6319 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37556544">37556544</a> </li>
</ul>
<h3 id="nyuwa-genome-resource-phase-1">Nyuwa Genome Resource Phase 1</h3>
<ul>
<li><strong>NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>SHORT NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>FULL NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>URL</strong> : <a href="http://bigdata.ibp.ac.cn/refpanel/getstarted.php">http://bigdata.ibp.ac.cn/refpanel/getstarted.php</a> </li>
<li><strong>TITLE</strong> : NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population </li>
<li><strong>DOI</strong> : 10.1016/j.celrep.2021.110017 </li>
<li><strong>ABSTRACT</strong> : The lack of haplotype reference panels and whole-genome sequencing resources specific to the Chinese population has greatly hindered genetic studies in the world's largest population. Here, we present the NyuWa genome resource, based on deep (26.2×) sequencing of 2,999 Chinese individuals, and construct a NyuWa reference panel of 5,804 haplotypes and 19.3 million variants, which is a high-quality publicly available Chinese population-specific reference panel with thousands of samples. Compared with other panels, the NyuWa reference panel reduces the Han Chinese imputation error rate by a margin ranging from 30% to 51%. Population structure and imputation simulation tests support the applicability of one integrated reference panel for northern and southern Chinese. In addition, a total of 22,504 loss-of-function variants in coding and noncoding genes are identified, including 11,493 novel variants. These results highlight the value of the NyuWa genome resource in facilitating genetic research in Chinese and Asian populations. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Zhang P, Luo H, Li Y, Wang Y, ...&amp;, He S. (2021) NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population Cell Rep., 37 (7) 110017. doi:10.1016/j.celrep.2021.110017. PMID 34788621 </li>
<li><strong>JOURNAL_INFO</strong> : Cell reports ; Cell Rep. ; 2021 ; 37 ; 7 ; 110017 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34788621">34788621</a> </li>
</ul>
<h3 id="pgghan-panel">PGG.Han panel</h3>
<ul>
<li><strong>NAME</strong> : PGG.Han panel </li>
<li><strong>SHORT NAME</strong> : PGG.Han </li>
<li><strong>URL</strong> : <a href="https://www.biosino.org/pgghan2/index#home1">https://www.biosino.org/pgghan2/index#home1</a> </li>
<li><strong>TITLE</strong> : PGG.Han: the Han Chinese genome database and analysis platform </li>
<li><strong>DOI</strong> : 10.1093/nar/gkz829 </li>
<li><strong>ABSTRACT</strong> : As the largest ethnic group in the world, the Han Chinese population is nonetheless underrepresented in global efforts to catalogue the genomic variability of natural populations. Here, we developed the PGG.Han, a population genome database to serve as the central repository for the genomic data of the Han Chinese Genome Initiative (Phase I). In its current version, the PGG.Han archives whole-genome sequences or high-density genome-wide single-nucleotide variants (SNVs) of 114 783 Han Chinese individuals (a.k.a. the Han100K), representing geographical sub-populations covering 33 of the 34 administrative divisions of China, as well as Singapore. The PGG.Han provides: (i) an interactive interface for visualization of the fine-scale genetic structure of the Han Chinese population; (ii) genome-wide allele frequencies of hierarchical sub-populations; (iii) ancestry inference for individual samples and controlling population stratification based on nested ancestry informative markers (AIMs) panels; (iv) population-structure-aware shared control data for genotype-phenotype association studies (e.g. GWASs) and (v) a Han-Chinese-specific reference panel for genotype imputation. Computational tools are implemented into the PGG.Han, and an online user-friendly interface is provided for data analysis and results visualization. The PGG.Han database is freely accessible via http://www.pgghan.org or https://www.hanchinesegenomes.org. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Gao Y, Zhang C, Yuan L, Ling Y, ...&amp;, Xu S. (2020) PGG.Han: the Han Chinese genome database and analysis platform Nucleic Acids Res., 48 (D1) D971-D976. doi:10.1093/nar/gkz829. PMID 31584086 </li>
<li><strong>JOURNAL_INFO</strong> : Nucleic acids research ; Nucleic Acids Res. ; 2020 ; 48 ; D1 ; D971-D976 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31584086">31584086</a> </li>
</ul>
<h3 id="south-and-east-asian-reference-database-sead">South and East Asian Reference Database (SEAD)</h3>
<ul>
<li><strong>NAME</strong> : South and East Asian Reference Database (SEAD) </li>
<li><strong>SHORT NAME</strong> : SEAD </li>
<li><strong>FULL NAME</strong> : South and East Asian Reference Database </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.12.23.23300480 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>CITATION</strong> : Yang, M. Y., Zhong, J. D., Li, X., Bai, W. Y., Yuan, C. D., Qiu, M. C., ... &amp; Zheng, H. F. (2023). SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population. medRxiv, 2023-12. </li>
</ul>
<h3 id="topmed">TOPMED</h3>
<ul>
<li><strong>NAME</strong> : TOPMED </li>
<li><strong>SHORT NAME</strong> : TOPMED </li>
<li><strong>FULL NAME</strong> : Trans-Omics for Precision Medicine </li>
<li><strong>URL</strong> : <a href="https://imputation.biodatacatalyst.nhlbi.nih.gov/#!">https://imputation.biodatacatalyst.nhlbi.nih.gov/#!</a> </li>
<li><strong>TITLE</strong> : Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program </li>
<li><strong>DOI</strong> : 10.1038/s41586-021-03205-y </li>
<li><strong>ABSTRACT</strong> : The Trans-Omics for Precision Medicine (TOPMed) programme seeks to elucidate the genetic architecture and biology of heart, lung, blood and sleep disorders, with the ultimate goal of improving diagnosis, treatment and prevention of these diseases. The initial phases of the programme focused on whole-genome sequencing of individuals with rich phenotypic data and diverse backgrounds. Here we describe the TOPMed goals and design as well as the available resources and early insights obtained from the sequence data. The resources include a variant browser, a genotype imputation server, and genomic and phenotypic data that are available through dbGaP (Database of Genotypes and Phenotypes)1. In the first 53,831 TOPMed samples, we detected more than 400 million single-nucleotide and insertion or deletion variants after alignment with the reference genome. Additional previously undescribed variants were detected through assembly of unmapped reads and customized analysis in highly variable loci. Among the more than 400 million detected variants, 97% have frequencies of less than 1% and 46% are singletons that are present in only one individual (53% among unrelated individuals). These rare variants provide insights into mutational processes and recent human evolutionary history. The extensive catalogue of genetic variation in TOPMed studies provides unique opportunities for exploring the contributions of rare and noncoding sequence variants to phenotypic variation. Furthermore, combining TOPMed haplotypes with modern imputation methods improves the power and reach of genome-wide association studies to include variants down to a frequency of approximately 0.01%. </li>
<li><strong>CITATION</strong> : Taliun D, Harris DN, Kessler MD, Carlson J, ...&amp;, Abecasis GR. (2021) Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program Nature, 590 (7845) 290-299. doi:10.1038/s41586-021-03205-y. PMID 33568819 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2021 ; 590 ; 7845 ; 290-299 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33568819">33568819</a> </li>
</ul>
<h3 id="wbbc-panel">WBBC panel</h3>
<ul>
<li><strong>NAME</strong> : WBBC panel </li>
<li><strong>SHORT NAME</strong> : WBBC </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>TITLE</strong> : Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project </li>
<li><strong>DOI</strong> : 10.1038/s41467-022-30526-x </li>
<li><strong>ABSTRACT</strong> : We initiate the Westlake BioBank for Chinese (WBBC) pilot project with 4,535 whole-genome sequencing (WGS) individuals and 5,841 high-density genotyping individuals, and identify 81.5 million SNPs and INDELs, of which 38.5% are absent in dbSNP Build 151. We provide a population-specific reference panel and an online imputation server ( https://wbbc.westlake.edu.cn/ ) which could yield substantial improvement of imputation performance in Chinese population, especially for low-frequency and rare variants. By analyzing the singleton density of the WGS data, we find selection signatures in SNX29, DNAH1 and WDR1 genes, and the derived alleles of the alcohol metabolism genes (ADH1A and ADH1B) emerge around 7,000 years ago and tend to be more common from 4,000 years ago in East Asia. Genetic evidence supports the corresponding geographical boundaries of the Qinling-Huaihe Line and Nanling Mountains, which separate the Han Chinese into subgroups, and we reveal that North Han was more homogeneous than South Han. </li>
<li><strong>CITATION</strong> : Cong PK, Bai WY, Li JC, Yang MY, ...&amp;, Zheng HF. (2022) Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project Nat. Commun., 13 (1) 2939. doi:10.1038/s41467-022-30526-x. PMID 35618720 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2022 ; 13 ; 1 ; 2939 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35618720">35618720</a> </li>
</ul>
<h2 id="imputation-server">Imputation server</h2>
<h3 id="cngb-imputation-service">CNGB Imputation Service</h3>
<ul>
<li><strong>NAME</strong> : CNGB Imputation Service </li>
<li><strong>SHORT NAME</strong> : CNGB </li>
<li><strong>FULL NAME</strong> : China National GeneBank </li>
<li><strong>URL</strong> : <a href="https://db.cngb.org/imputation/">https://db.cngb.org/imputation/</a> </li>
<li><strong>TITLE</strong> : A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study </li>
<li><strong>DOI</strong> : 10.1093/nar/gkad779 </li>
<li><strong>ABSTRACT</strong> : Precision medicine depends on high-accuracy individual-level genotype data. However, the whole-genome sequencing (WGS) is still not suitable for gigantic studies due to budget constraints. It is particularly important to construct highly accurate haplotype reference panel for genotype imputation. In this study, we used 10 000 samples with medium-depth WGS to construct a reference panel that we named the CKB reference panel. By imputing microarray datasets, it showed that the CKB panel outperformed compared panels in terms of both the number of well-imputed variants and imputation accuracy. In addition, we have completed the imputation of 100 706 microarrays with the CKB panel, and the after-imputed data is the hitherto largest whole genome data of the Chinese population. Furthermore, in the GWAS analysis of real phenotype height, the number of tested SNPs tripled and the number of significant SNPs doubled after imputation. Finally, we developed an online server for offering free genotype imputation service based on the CKB reference panel (https://db.cngb.org/imputation/). We believe that the CKB panel is of great value for imputing microarray or low-coverage genotype data of Chinese population, and potentially mixed populations. The imputation-completed 100 706 microarray data are enormous and precious resources of population genetic studies for complex traits and diseases. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Yu C, Lan X, Tao Y, Guo Y, ...&amp;, Li L. (2023) A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study Nucleic Acids Res., 51 (21) 11770-11782. doi:10.1093/nar/gkad779. PMID 37870428 </li>
<li><strong>JOURNAL_INFO</strong> : Nucleic acids research ; Nucleic Acids Res. ; 2023 ; 51 ; 21 ; 11770-11782 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37870428">37870428</a> </li>
</ul>
<h3 id="chinamap">ChinaMAP</h3>
<ul>
<li><strong>NAME</strong> : ChinaMAP </li>
<li><strong>SHORT NAME</strong> : ChinaMAP </li>
<li><strong>FULL NAME</strong> : China Metabolic Analytics Project </li>
<li><strong>URL</strong> : <a href="http://www.mbiobank.com/">http://www.mbiobank.com/</a> </li>
<li><strong>TITLE</strong> : The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations </li>
<li><strong>DOI</strong> : 10.1038/s41422-021-00564-z </li>
<li><strong>ABSTRACT</strong> : The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Li L, Huang P, Sun X, Wang S, ...&amp;, Wang W. (2021) The ChinaMAP reference panel for the accurate genotype imputation in Chinese populations Cell Res., 31 (12) 1308-1310. doi:10.1038/s41422-021-00564-z. PMID 34489580 </li>
<li><strong>JOURNAL_INFO</strong> : Cell research ; Cell Res. ; 2021 ; 31 ; 12 ; 1308-1310 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34489580">34489580</a> </li>
</ul>
<h3 id="michigan-imputation-server">Michigan Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : Michigan Imputation Server </li>
<li><strong>SHORT NAME</strong> : Michigan </li>
<li><strong>URL</strong> : <a href="https://imputationserver.sph.umich.edu/index.html#!">https://imputationserver.sph.umich.edu/index.html#!</a> </li>
<li><strong>TITLE</strong> : Next-generation genotype imputation service and methods </li>
<li><strong>DOI</strong> : 10.1038/ng.3656 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation is a key component of genetic association studies, where it increases power, facilitates meta-analysis, and aids interpretation of signals. Genotype imputation is computationally demanding and, with current tools, typically requires access to a high-performance computing cluster and to a reference panel of sequenced genomes. Here we describe improvements to imputation machinery that reduce computational requirements by more than an order of magnitude with no loss of accuracy in comparison to standard imputation tools. We also describe a new web-based service for imputation that facilitates access to new reference panels and greatly improves user experience and productivity. </li>
<li><strong>CITATION</strong> : Das S, Forer L, Schönherr S, Sidore C, ...&amp;, Fuchsberger C. (2016) Next-generation genotype imputation service and methods Nat. Genet., 48 (10) 1284-1287. doi:10.1038/ng.3656. PMID 27571263 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 10 ; 1284-1287 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27571263">27571263</a> </li>
</ul>
<h3 id="nyuwa-imputation-server">NyuWa Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : NyuWa Imputation Server </li>
<li><strong>SHORT NAME</strong> : NyuWa </li>
<li><strong>URL</strong> : <a href="http://bigdata.ibp.ac.cn/refpanel/getstarted.php">http://bigdata.ibp.ac.cn/refpanel/getstarted.php</a> </li>
<li><strong>TITLE</strong> : NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population </li>
<li><strong>DOI</strong> : 10.1016/j.celrep.2021.110017 </li>
<li><strong>ABSTRACT</strong> : The lack of haplotype reference panels and whole-genome sequencing resources specific to the Chinese population has greatly hindered genetic studies in the world's largest population. Here, we present the NyuWa genome resource, based on deep (26.2×) sequencing of 2,999 Chinese individuals, and construct a NyuWa reference panel of 5,804 haplotypes and 19.3 million variants, which is a high-quality publicly available Chinese population-specific reference panel with thousands of samples. Compared with other panels, the NyuWa reference panel reduces the Han Chinese imputation error rate by a margin ranging from 30% to 51%. Population structure and imputation simulation tests support the applicability of one integrated reference panel for northern and southern Chinese. In addition, a total of 22,504 loss-of-function variants in coding and noncoding genes are identified, including 11,493 novel variants. These results highlight the value of the NyuWa genome resource in facilitating genetic research in Chinese and Asian populations. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Zhang P, Luo H, Li Y, Wang Y, ...&amp;, He S. (2021) NyuWa Genome resource: A deep whole-genome sequencing-based variation profile and reference panel for the Chinese population Cell Rep., 37 (7) 110017. doi:10.1016/j.celrep.2021.110017. PMID 34788621 </li>
<li><strong>JOURNAL_INFO</strong> : Cell reports ; Cell Rep. ; 2021 ; 37 ; 7 ; 110017 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34788621">34788621</a> </li>
</ul>
<h3 id="pgghan">PGG.Han</h3>
<ul>
<li><strong>NAME</strong> : PGG.Han </li>
<li><strong>SHORT NAME</strong> : PGG.Han </li>
<li><strong>URL</strong> : <a href="https://www.biosino.org/pgghan2/login">https://www.biosino.org/pgghan2/login</a> </li>
<li><strong>TITLE</strong> : PGG.Han: the Han Chinese genome database and analysis platform </li>
<li><strong>DOI</strong> : 10.1093/nar/gkz829 </li>
<li><strong>ABSTRACT</strong> : As the largest ethnic group in the world, the Han Chinese population is nonetheless underrepresented in global efforts to catalogue the genomic variability of natural populations. Here, we developed the PGG.Han, a population genome database to serve as the central repository for the genomic data of the Han Chinese Genome Initiative (Phase I). In its current version, the PGG.Han archives whole-genome sequences or high-density genome-wide single-nucleotide variants (SNVs) of 114 783 Han Chinese individuals (a.k.a. the Han100K), representing geographical sub-populations covering 33 of the 34 administrative divisions of China, as well as Singapore. The PGG.Han provides: (i) an interactive interface for visualization of the fine-scale genetic structure of the Han Chinese population; (ii) genome-wide allele frequencies of hierarchical sub-populations; (iii) ancestry inference for individual samples and controlling population stratification based on nested ancestry informative markers (AIMs) panels; (iv) population-structure-aware shared control data for genotype-phenotype association studies (e.g. GWASs) and (v) a Han-Chinese-specific reference panel for genotype imputation. Computational tools are implemented into the PGG.Han, and an online user-friendly interface is provided for data analysis and results visualization. The PGG.Han database is freely accessible via http://www.pgghan.org or https://www.hanchinesegenomes.org. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc/4.0/ </li>
<li><strong>CITATION</strong> : Gao Y, Zhang C, Yuan L, Ling Y, ...&amp;, Xu S. (2020) PGG.Han: the Han Chinese genome database and analysis platform Nucleic Acids Res., 48 (D1) D971-D976. doi:10.1093/nar/gkz829. PMID 31584086 </li>
<li><strong>JOURNAL_INFO</strong> : Nucleic acids research ; Nucleic Acids Res. ; 2020 ; 48 ; D1 ; D971-D976 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31584086">31584086</a> </li>
</ul>
<h3 id="sanger">Sanger</h3>
<ul>
<li><strong>NAME</strong> : Sanger </li>
<li><strong>URL</strong> : <a href="https://imputation.sanger.ac.uk/">https://imputation.sanger.ac.uk/</a> </li>
<li><strong>TITLE</strong> : A reference panel of 64,976 haplotypes for genotype imputation </li>
<li><strong>DOI</strong> : 10.1038/ng.3643 </li>
<li><strong>ABSTRACT</strong> : We describe a reference panel of 64,976 human haplotypes at 39,235,157 SNPs constructed using whole-genome sequence data from 20 studies of predominantly European ancestry. Using this resource leads to accurate genotype imputation at minor allele frequencies as low as 0.1% and a large increase in the number of SNPs tested in association studies, and it can help to discover and refine causal loci. We describe remote server resources that allow researchers to carry out imputation and phasing consistently and efficiently. </li>
<li><strong>CITATION</strong> : McCarthy S, Das S, Kretzschmar W, Delaneau O, ...&amp;, Marchini J. (2016) A reference panel of 64,976 haplotypes for genotype imputation Nat. Genet., 48 (10) 1279-1283. doi:10.1038/ng.3643. PMID 27548312 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 10 ; 1279-1283 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27548312">27548312</a> </li>
</ul>
<h3 id="topmed_1">TOPMED</h3>
<ul>
<li><strong>NAME</strong> : TOPMED </li>
<li><strong>SHORT NAME</strong> : TOPMED </li>
<li><strong>FULL NAME</strong> : Trans-Omics for Precision Medicine </li>
<li><strong>URL</strong> : <a href="https://imputation.biodatacatalyst.nhlbi.nih.gov/#!">https://imputation.biodatacatalyst.nhlbi.nih.gov/#!</a> </li>
<li><strong>TITLE</strong> : Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program </li>
<li><strong>DOI</strong> : 10.1038/s41586-021-03205-y </li>
<li><strong>ABSTRACT</strong> : The Trans-Omics for Precision Medicine (TOPMed) programme seeks to elucidate the genetic architecture and biology of heart, lung, blood and sleep disorders, with the ultimate goal of improving diagnosis, treatment and prevention of these diseases. The initial phases of the programme focused on whole-genome sequencing of individuals with rich phenotypic data and diverse backgrounds. Here we describe the TOPMed goals and design as well as the available resources and early insights obtained from the sequence data. The resources include a variant browser, a genotype imputation server, and genomic and phenotypic data that are available through dbGaP (Database of Genotypes and Phenotypes)1. In the first 53,831 TOPMed samples, we detected more than 400 million single-nucleotide and insertion or deletion variants after alignment with the reference genome. Additional previously undescribed variants were detected through assembly of unmapped reads and customized analysis in highly variable loci. Among the more than 400 million detected variants, 97% have frequencies of less than 1% and 46% are singletons that are present in only one individual (53% among unrelated individuals). These rare variants provide insights into mutational processes and recent human evolutionary history. The extensive catalogue of genetic variation in TOPMed studies provides unique opportunities for exploring the contributions of rare and noncoding sequence variants to phenotypic variation. Furthermore, combining TOPMed haplotypes with modern imputation methods improves the power and reach of genome-wide association studies to include variants down to a frequency of approximately 0.01%. </li>
<li><strong>CITATION</strong> : Taliun D, Harris DN, Kessler MD, Carlson J, ...&amp;, Abecasis GR. (2021) Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program Nature, 590 (7845) 290-299. doi:10.1038/s41586-021-03205-y. PMID 33568819 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2021 ; 590 ; 7845 ; 290-299 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33568819">33568819</a> </li>
</ul>
<h3 id="westlake-imputation-server">Westlake Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : Westlake Imputation Server </li>
<li><strong>SHORT NAME</strong> : Westlake Imputation Server </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>TITLE</strong> : Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project </li>
<li><strong>DOI</strong> : 10.1038/s41467-022-30526-x </li>
<li><strong>ABSTRACT</strong> : We initiate the Westlake BioBank for Chinese (WBBC) pilot project with 4,535 whole-genome sequencing (WGS) individuals and 5,841 high-density genotyping individuals, and identify 81.5 million SNPs and INDELs, of which 38.5% are absent in dbSNP Build 151. We provide a population-specific reference panel and an online imputation server ( https://wbbc.westlake.edu.cn/ ) which could yield substantial improvement of imputation performance in Chinese population, especially for low-frequency and rare variants. By analyzing the singleton density of the WGS data, we find selection signatures in SNX29, DNAH1 and WDR1 genes, and the derived alleles of the alcohol metabolism genes (ADH1A and ADH1B) emerge around 7,000 years ago and tend to be more common from 4,000 years ago in East Asia. Genetic evidence supports the corresponding geographical boundaries of the Qinling-Huaihe Line and Nanling Mountains, which separate the Han Chinese into subgroups, and we reveal that North Han was more homogeneous than South Han. </li>
<li><strong>CITATION</strong> : Cong PK, Bai WY, Li JC, Yang MY, ...&amp;, Zheng HF. (2022) Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project Nat. Commun., 13 (1) 2939. doi:10.1038/s41467-022-30526-x. PMID 35618720 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2022 ; 13 ; 1 ; 2939 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35618720">35618720</a> </li>
</ul>
<h2 id="other-tools">Other tools</h2>
<h3 id="reshape">RESHAPE</h3>
<ul>
<li><strong>NAME</strong> : RESHAPE </li>
<li><strong>SHORT NAME</strong> : RESHAPE </li>
<li><strong>FULL NAME</strong> : REcombine and Share HAPlotypEs </li>
<li><strong>DESCRIPTION</strong> : RESHAPE removes sample-level genetic information from a reference panel to create a synthetic reference panel. By providing it with a genetic map and the VCF/BCF of a reference panel, RESHAPE outputs a VCF/BCF of the same size where each haplotypes corresponds to a mosaic of the original haplotypes of the reference panel. </li>
<li><strong>URL</strong> : <a href="https://github.com/TheoCavinato/RESHAPE">https://github.com/TheoCavinato/RESHAPE</a> </li>
<li><strong>TITLE</strong> : A resampling-based approach to share reference panels </li>
<li><strong>DOI</strong> : 10.1038/s43588-024-00630-7 </li>
<li><strong>ABSTRACT</strong> : For many genome-wide association studies, imputing genotypes from a haplotype reference panel is a necessary step. Over the past 15 years, reference panels have become larger and more diverse, leading to improvements in imputation accuracy. However, the latest generation of reference panels is subject to restrictions on data sharing due to concerns about privacy, limiting their usefulness for genotype imputation. In this context, here we propose RESHAPE, a method that employs a recombination Poisson process on a reference panel to simulate the genomes of hypothetical descendants after multiple generations. This data transformation helps to protect against re-identification threats and preserves data attributes, such as linkage disequilibrium patterns and, to some degree, identity-by-descent sharing, allowing for genotype imputation. Our experiments on gold-standard datasets show that simulated descendants up to eight generations can serve as reference panels without substantially reducing genotype imputation accuracy. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Cavinato T, Rubinacci S, Malaspinas AS, Delaneau O. (2024) A resampling-based approach to share reference panels Nat. Comput. Sci., 4 (5) 360-366. doi:10.1038/s43588-024-00630-7. PMID 38745108 </li>
<li><strong>JOURNAL_INFO</strong> : Nature computational science ; Nat. Comput. Sci. ; 2024 ; 4 ; 5 ; 360-366 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38745108">38745108</a> </li>
</ul>
<h2 id="phasing-imputation-tool">Phasing &amp; Imputation tool</h2>
<h3 id="beagle">BEAGLE</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE </li>
<li><strong>SHORT NAME</strong> : BEAGLE </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>TITLE</strong> : Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering </li>
<li><strong>DOI</strong> : 10.1086/521987 </li>
<li><strong>ABSTRACT</strong> : Whole-genome association studies present many new statistical and computational challenges due to the large quantity of data obtained. One of these challenges is haplotype inference; methods for haplotype inference designed for small data sets from candidate-gene studies do not scale well to the large number of individuals genotyped in whole-genome association studies. We present a new method and software for inference of haplotype phase and missing data that can accurately phase data from whole-genome association studies, and we present the first comparison of haplotype-inference methods for real and simulated data sets with thousands of genotyped individuals. We find that our method outperforms existing methods in terms of both speed and accuracy for large data sets with thousands of individuals and densely spaced genetic markers, and we use our method to phase a real data set of 3,002 individuals genotyped for 490,032 markers in 3.1 days of computing time, with 99% of masked alleles imputed correctly. Our method is implemented in the Beagle software package, which is freely available. </li>
<li><strong>COPYRIGHT</strong> : https://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Browning SR, Browning BL. (2007) Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering Am. J. Hum. Genet., 81 (5) 1084-1097. doi:10.1086/521987. PMID 17924348 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2007 ; 81 ; 5 ; 1084-1097 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17924348">17924348</a> </li>
</ul>
<h3 id="beagle4">BEAGLE4</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE4 </li>
<li><strong>SHORT NAME</strong> : BEAGLE4 </li>
<li><strong>DESCRIPTION</strong> : (beagle 4.1) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>TITLE</strong> : Genotype imputation with millions of reference samples </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2015.11.020 </li>
<li><strong>ABSTRACT</strong> : We present a genotype imputation method that scales to millions of reference samples. The imputation method, based on the Li and Stephens model and implemented in Beagle v.4.1, is parallelized and memory efficient, making it well suited to multi-core computer processors. It achieves fast, accurate, and memory-efficient genotype imputation by restricting the probability model to markers that are genotyped in the target samples and by performing linear interpolation to impute ungenotyped variants. We compare Beagle v.4.1 with Impute2 and Minimac3 by using 1000 Genomes Project data, UK10K Project data, and simulated data. All three methods have similar accuracy but different memory requirements and different computation times. When imputing 10 Mb of sequence data from 50,000 reference samples, Beagle's throughput was more than 100× greater than Impute2's throughput on our computer servers. When imputing 10 Mb of sequence data from 200,000 reference samples in VCF format, Minimac3 consumed 26× more memory per computational thread and 15× more CPU time than Beagle. We demonstrate that Beagle v.4.1 scales to much larger reference panels by performing imputation from a simulated reference panel having 5 million samples and a mean marker density of one marker per four base pairs. </li>
<li><strong>COPYRIGHT</strong> : https://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Browning BL, Browning SR. (2016) Genotype imputation with millions of reference samples Am. J. Hum. Genet., 98 (1) 116-126. doi:10.1016/j.ajhg.2015.11.020. PMID 26748515 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2016 ; 98 ; 1 ; 116-126 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26748515">26748515</a> </li>
</ul>
<h3 id="beagle54-imputation">BEAGLE5.4 (Imputation)</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE5.4 (Imputation) </li>
<li><strong>SHORT NAME</strong> : BEAGLE5.4 (Imputation) </li>
<li><strong>DESCRIPTION</strong> : (beagle 5.4 imputation) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>TITLE</strong> : A one-penny imputed genome from next-generation reference panels </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2018.07.015 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation is commonly performed in genome-wide association studies because it greatly increases the number of markers that can be tested for association with a trait. In general, one should perform genotype imputation using the largest reference panel that is available because the number of accurately imputed variants increases with reference panel size. However, one impediment to using larger reference panels is the increased computational cost of imputation. We present a new genotype imputation method, Beagle 5.0, which greatly reduces the computational cost of imputation from large reference panels. We compare Beagle 5.0 with Beagle 4.1, Impute4, Minimac3, and Minimac4 using 1000 Genomes Project data, Haplotype Reference Consortium data, and simulated data for 10k, 100k, 1M, and 10M reference samples. All methods produce nearly identical accuracy, but Beagle 5.0 has the lowest computation time and the best scaling of computation time with increasing reference panel size. For 10k, 100k, 1M, and 10M reference samples and 1,000 phased target samples, Beagle 5.0's computation time is 3× (10k), 12× (100k), 43× (1M), and 533× (10M) faster than the fastest alternative method. Cost data from the Amazon Elastic Compute Cloud show that Beagle 5.0 can perform genome-wide imputation from 10M reference samples into 1,000 phased target samples at a cost of less than one US cent per sample. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Browning BL, Zhou Y, Browning SR. (2018) A one-penny imputed genome from next-generation reference panels Am. J. Hum. Genet., 103 (3) 338-348. doi:10.1016/j.ajhg.2018.07.015. PMID 30100085 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2018 ; 103 ; 3 ; 338-348 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30100085">30100085</a> </li>
</ul>
<h3 id="beagle54-phasing">BEAGLE5.4 (Phasing)</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE5.4 (Phasing) </li>
<li><strong>SHORT NAME</strong> : BEAGLE5.4 (Phasing) </li>
<li><strong>DESCRIPTION</strong> : (beagle 5.4 phasing) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>TITLE</strong> : Fast two-stage phasing of large-scale sequence data </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2021.08.005 </li>
<li><strong>ABSTRACT</strong> : Haplotype phasing is the estimation of haplotypes from genotype data. We present a fast, accurate, and memory-efficient haplotype phasing method that scales to large-scale SNP array and sequence data. The method uses marker windowing and composite reference haplotypes to reduce memory usage and computation time. It incorporates a progressive phasing algorithm that identifies confidently phased heterozygotes in each iteration and fixes the phase of these heterozygotes in subsequent iterations. For data with many low-frequency variants, such as whole-genome sequence data, the method employs a two-stage phasing algorithm that phases high-frequency markers via progressive phasing in the first stage and phases low-frequency markers via genotype imputation in the second stage. This haplotype phasing method is implemented in the open-source Beagle 5.2 software package. We compare Beagle 5.2 and SHAPEIT 4.2.1 by using expanding subsets of 485,301 UK Biobank samples and 38,387 TOPMed samples. Both methods have very similar accuracy and computation time for UK Biobank SNP array data. However, for TOPMed sequence data, Beagle is more than 20 times faster than SHAPEIT, achieves similar accuracy, and scales to larger sample sizes. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Browning BL, Tian X, Zhou Y, Browning SR. (2021) Fast two-stage phasing of large-scale sequence data Am. J. Hum. Genet., 108 (10) 1880-1890. doi:10.1016/j.ajhg.2021.08.005. PMID 34478634 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2021 ; 108 ; 10 ; 1880-1890 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34478634">34478634</a> </li>
</ul>
<h3 id="eagle">EAGLE</h3>
<ul>
<li><strong>NAME</strong> : EAGLE </li>
<li><strong>SHORT NAME</strong> : EAGLE </li>
<li><strong>DESCRIPTION</strong> : (EAGLE1) </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/Eagle/">https://alkesgroup.broadinstitute.org/Eagle/</a> </li>
<li><strong>TITLE</strong> : Fast and accurate long-range phasing in a UK Biobank cohort </li>
<li><strong>DOI</strong> : 10.1038/ng.3571 </li>
<li><strong>ABSTRACT</strong> : Recent work has leveraged the extensive genotyping of the Icelandic population to perform long-range phasing (LRP), enabling accurate imputation and association analysis of rare variants in target samples typed on genotyping arrays. Here we develop a fast and accurate LRP method, Eagle, that extends this paradigm to populations with much smaller proportions of genotyped samples by harnessing long (&gt;4-cM) identical-by-descent (IBD) tracts shared among distantly related individuals. We applied Eagle to N ≈ 150,000 samples (0.2% of the British population) from the UK Biobank, and we determined that it is 1-2 orders of magnitude faster than existing methods while achieving similar or better phasing accuracy (switch error rate ≈ 0.3%, corresponding to perfect phase in a majority of 10-Mb segments). We also observed that, when used within an imputation pipeline, Eagle prephasing improved downstream imputation accuracy in comparison to prephasing in batches using existing methods, as necessary to achieve comparable computational cost. </li>
<li><strong>CITATION</strong> : Loh PR, Palamara PF, Price AL. (2016) Fast and accurate long-range phasing in a UK Biobank cohort Nat. Genet., 48 (7) 811-816. doi:10.1038/ng.3571. PMID 27270109 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 7 ; 811-816 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27270109">27270109</a> </li>
</ul>
<h3 id="eagle2">EAGLE2</h3>
<ul>
<li><strong>NAME</strong> : EAGLE2 </li>
<li><strong>SHORT NAME</strong> : EAGLE2 </li>
<li><strong>DESCRIPTION</strong> : (EAGLE2) </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/Eagle/">https://alkesgroup.broadinstitute.org/Eagle/</a> </li>
<li><strong>TITLE</strong> : Reference-based phasing using the Haplotype Reference Consortium panel </li>
<li><strong>DOI</strong> : 10.1038/ng.3679 </li>
<li><strong>ABSTRACT</strong> : Haplotype phasing is a fundamental problem in medical and population genetics. Phasing is generally performed via statistical phasing in a genotyped cohort, an approach that can yield high accuracy in very large cohorts but attains lower accuracy in smaller cohorts. Here we instead explore the paradigm of reference-based phasing. We introduce a new phasing algorithm, Eagle2, that attains high accuracy across a broad range of cohort sizes by efficiently leveraging information from large external reference panels (such as the Haplotype Reference Consortium; HRC) using a new data structure based on the positional Burrows-Wheeler transform. We demonstrate that Eagle2 attains a ∼20× speedup and ∼10% increase in accuracy compared to reference-based phasing using SHAPEIT2. On European-ancestry samples, Eagle2 with the HRC panel achieves &gt;2× the accuracy of 1000 Genomes-based phasing. Eagle2 is open source and freely available for HRC-based phasing via the Sanger Imputation Service and the Michigan Imputation Server. </li>
<li><strong>CITATION</strong> : Loh PR, Danecek P, Palamara PF, Fuchsberger C, ...&amp;, L Price A. (2016) Reference-based phasing using the Haplotype Reference Consortium panel Nat. Genet., 48 (11) 1443-1448. doi:10.1038/ng.3679. PMID 27694958 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 11 ; 1443-1448 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27694958">27694958</a> </li>
</ul>
<h3 id="glimpse">GLIMPSE</h3>
<ul>
<li><strong>NAME</strong> : GLIMPSE </li>
<li><strong>SHORT NAME</strong> : GLIMPSE </li>
<li><strong>FULL NAME</strong> : Genotype Likelihoods IMputation and PhaSing mEthod </li>
<li><strong>DESCRIPTION</strong> : GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies. </li>
<li><strong>URL</strong> : <a href="https://odelaneau.github.io/GLIMPSE/">https://odelaneau.github.io/GLIMPSE/</a> </li>
<li><strong>TITLE</strong> : Efficient phasing and imputation of low-coverage sequencing data using large reference panels </li>
<li><strong>DOI</strong> : 10.1038/s41588-020-00756-0 </li>
<li><strong>ABSTRACT</strong> : Low-coverage whole-genome sequencing followed by imputation has been proposed as a cost-effective genotyping approach for disease and population genetics studies. However, its competitiveness against SNP arrays is undermined because current imputation methods are computationally expensive and unable to leverage large reference panels. Here, we describe a method, GLIMPSE, for phasing and imputation of low-coverage sequencing datasets from modern reference panels. We demonstrate its remarkable performance across different coverages and human populations. GLIMPSE achieves imputation of a genome for less than US$1 in computational cost, considerably outperforming other methods and improving imputation accuracy over the full allele frequency range. As a proof of concept, we show that 1× coverage enables effective gene expression association studies and outperforms dense SNP arrays in rare variant burden tests. Overall, this study illustrates the promising potential of low-coverage imputation and suggests a paradigm shift in the design of future genomic studies. </li>
<li><strong>CITATION</strong> : Rubinacci S, Ribeiro DM, Hofmeister RJ, Delaneau O. (2021) Efficient phasing and imputation of low-coverage sequencing data using large reference panels Nat. Genet., 53 (1) 120-126. doi:10.1038/s41588-020-00756-0. PMID 33414550 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 1 ; 120-126 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33414550">33414550</a> </li>
</ul>
<h3 id="impute">IMPUTE</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE </li>
<li><strong>SHORT NAME</strong> : IMPUTE </li>
<li><strong>URL</strong> : <a href="https://jmarchini.org/software/">https://jmarchini.org/software/</a> </li>
<li><strong>TITLE</strong> : A new multipoint method for genome-wide association studies by imputation of genotypes </li>
<li><strong>DOI</strong> : 10.1038/ng2088 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies are set to become the method of choice for uncovering the genetic basis of human diseases. A central challenge in this area is the development of powerful multipoint methods that can detect causal variants that have not been directly genotyped. We propose a coherent analysis framework that treats the problem as one involving missing or uncertain genotypes. Central to our approach is a model-based imputation method for inferring genotypes at observed or unobserved SNPs, leading to improved power over existing methods for multipoint association mapping. Using real genome-wide association study data, we show that our approach (i) is accurate and well calibrated, (ii) provides detailed views of associated regions that facilitate follow-up studies and (iii) can be used to validate and correct data at genotyped markers. A notable future use of our method will be to boost power by combining data from genome-wide scans that use different SNP sets. </li>
<li><strong>CITATION</strong> : Marchini J, Howie B, Myers S, McVean G, ...&amp;, Donnelly P. (2007) A new multipoint method for genome-wide association studies by imputation of genotypes Nat. Genet., 39 (7) 906-913. doi:10.1038/ng2088. PMID 17572673 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2007 ; 39 ; 7 ; 906-913 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17572673">17572673</a> </li>
</ul>
<h3 id="impute2">IMPUTE2</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE2 </li>
<li><strong>SHORT NAME</strong> : IMPUTE2 </li>
<li><strong>TITLE</strong> : A flexible and accurate genotype imputation method for the next generation of genome-wide association studies </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1000529 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation methods are now being widely used in the analysis of genome-wide association studies. Most imputation analyses to date have used the HapMap as a reference dataset, but new reference panels (such as controls genotyped on multiple SNP chips and densely typed samples from the 1,000 Genomes Project) will soon allow a broader range of SNPs to be imputed with higher accuracy, thereby increasing power. We describe a genotype imputation method (IMPUTE version 2) that is designed to address the challenges presented by these new datasets. The main innovation of our approach is a flexible modelling framework that increases accuracy and combines information across multiple reference panels while remaining computationally feasible. We find that IMPUTE v2 attains higher accuracy than other methods when the HapMap provides the sole reference panel, but that the size of the panel constrains the improvements that can be made. We also find that imputation accuracy can be greatly enhanced by expanding the reference panel to contain thousands of chromosomes and that IMPUTE v2 outperforms other methods in this setting at both rare and common SNPs, with overall error rates that are 15%-20% lower than those of the closest competing method. One particularly challenging aspect of next-generation association studies is to integrate information across multiple reference panels genotyped on different sets of SNPs; we show that our approach to this problem has practical advantages over other suggested solutions. </li>
<li><strong>CITATION</strong> : Howie BN, Donnelly P, Marchini J. (2009) A flexible and accurate genotype imputation method for the next generation of genome-wide association studies PLoS Genet., 5 (6) e1000529. doi:10.1371/journal.pgen.1000529. PMID 19543373 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2009 ; 5 ; 6 ; e1000529 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19543373">19543373</a> </li>
</ul>
<h3 id="impute4">IMPUTE4</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE4 </li>
<li><strong>SHORT NAME</strong> : IMPUTE4 </li>
<li><strong>TITLE</strong> : The UK Biobank resource with deep phenotyping and genomic data </li>
<li><strong>DOI</strong> : 10.1038/s41586-018-0579-z </li>
<li><strong>ABSTRACT</strong> : The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. The open resource is unique in its size and scope. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits. Here we describe the centralized analysis of the genetic data, including genotype quality, properties of population structure and relatedness of the genetic data, and efficient phasing and genotype imputation that increases the number of testable variants to around 96 million. Classical allelic variation at 11 human leukocyte antigen genes was imputed, resulting in the recovery of signals with known associations between human leukocyte antigen alleles and many diseases. </li>
<li><strong>CITATION</strong> : Bycroft C, Freeman C, Petkova D, Band G, ...&amp;, Marchini J. (2018) The UK Biobank resource with deep phenotyping and genomic data Nature, 562 (7726) 203-209. doi:10.1038/s41586-018-0579-z. PMID 30305743 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2018 ; 562 ; 7726 ; 203-209 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30305743">30305743</a> </li>
</ul>
<h3 id="impute5">IMPUTE5</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE5 </li>
<li><strong>SHORT NAME</strong> : IMPUTE5 </li>
<li><strong>TITLE</strong> : Genotype imputation using the Positional Burrows Wheeler Transform </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1009049 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation is the process of predicting unobserved genotypes in a sample of individuals using a reference panel of haplotypes. In the last 10 years reference panels have increased in size by more than 100 fold. Increasing reference panel size improves accuracy of markers with low minor allele frequencies but poses ever increasing computational challenges for imputation methods. Here we present IMPUTE5, a genotype imputation method that can scale to reference panels with millions of samples. This method continues to refine the observation made in the IMPUTE2 method, that accuracy is optimized via use of a custom subset of haplotypes when imputing each individual. It achieves fast, accurate, and memory-efficient imputation by selecting haplotypes using the Positional Burrows Wheeler Transform (PBWT). By using the PBWT data structure at genotyped markers, IMPUTE5 identifies locally best matching haplotypes and long identical by state segments. The method then uses the selected haplotypes as conditioning states within the IMPUTE model. Using the HRC reference panel, which has ∼65,000 haplotypes, we show that IMPUTE5 is up to 30x faster than MINIMAC4 and up to 3x faster than BEAGLE5.1, and uses less memory than both these methods. Using simulated reference panels we show that IMPUTE5 scales sub-linearly with reference panel size. For example, keeping the number of imputed markers constant, increasing the reference panel size from 10,000 to 1 million haplotypes requires less than twice the computation time. As the reference panel increases in size IMPUTE5 is able to utilize a smaller number of reference haplotypes, thus reducing computational cost. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Rubinacci S, Delaneau O, Marchini J. (2020) Genotype imputation using the Positional Burrows Wheeler Transform PLoS Genet., 16 (11) e1009049. doi:10.1371/journal.pgen.1009049. PMID 33196638 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2020 ; 16 ; 11 ; e1009049 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33196638">33196638</a> </li>
</ul>
<h3 id="mach-minimach">MACH / minimach</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach </li>
<li><strong>SHORT NAME</strong> : MACH / minimach </li>
<li><strong>DESCRIPTION</strong> : (MACH) </li>
<li><strong>URL</strong> : <a href="http://csg.sph.umich.edu/abecasis/MaCH/index.html">http://csg.sph.umich.edu/abecasis/MaCH/index.html</a> </li>
<li><strong>TITLE</strong> : MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes </li>
<li><strong>DOI</strong> : 10.1002/gepi.20533 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) can identify common alleles that contribute to complex disease susceptibility. Despite the large number of SNPs assessed in each study, the effects of most common SNPs must be evaluated indirectly using either genotyped markers or haplotypes thereof as proxies. We have previously implemented a computationally efficient Markov Chain framework for genotype imputation and haplotyping in the freely available MaCH software package. The approach describes sampled chromosomes as mosaics of each other and uses available genotype and shotgun sequence data to estimate unobserved genotypes and haplotypes, together with useful measures of the quality of these estimates. Our approach is already widely used to facilitate comparison of results across studies as well as meta-analyses of GWAS. Here, we use simulations and experimental genotypes to evaluate its accuracy and utility, considering choices of genotyping panels, reference panel configurations, and designs where genotyping is replaced with shotgun sequencing. Importantly, we show that genotype imputation not only facilitates cross study analyses but also increases power of genetic association studies. We show that genotype imputation of common variants using HapMap haplotypes as a reference is very accurate using either genome-wide SNP data or smaller amounts of data typical in fine-mapping studies. Furthermore, we show the approach is applicable in a variety of populations. Finally, we illustrate how association analyses of unobserved variants will benefit from ongoing advances such as larger HapMap reference panels and whole genome shotgun sequencing technologies. </li>
<li><strong>COPYRIGHT</strong> : http://onlinelibrary.wiley.com/termsAndConditions#vor </li>
<li><strong>CITATION</strong> : Li Y, Willer CJ, Ding J, Scheet P, ...&amp;, Abecasis GR. (2010) MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes Genet. Epidemiol., 34 (8) 816-834. doi:10.1002/gepi.20533. PMID 21058334 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet. Epidemiol. ; 2010 ; 34 ; 8 ; 816-834 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21058334">21058334</a> </li>
</ul>
<h3 id="mach-minimach-pre-phasing">MACH / minimach pre-phasing</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach pre-phasing </li>
<li><strong>SHORT NAME</strong> : MACH / minimach pre-phasing </li>
<li><strong>DESCRIPTION</strong> : (pre-phasing, minimac) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac">https://genome.sph.umich.edu/wiki/Minimac</a> </li>
<li><strong>TITLE</strong> : Fast and accurate genotype imputation in genome-wide association studies through pre-phasing </li>
<li><strong>DOI</strong> : 10.1038/ng.2354 </li>
<li><strong>ABSTRACT</strong> : The 1000 Genomes Project and disease-specific sequencing efforts are producing large collections of haplotypes that can be used as reference panels for genotype imputation in genome-wide association studies (GWAS). However, imputing from large reference panels with existing methods imposes a high computational burden. We introduce a strategy called 'pre-phasing' that maintains the accuracy of leading methods while reducing computational costs. We first statistically estimate the haplotypes for each individual within the GWAS sample (pre-phasing) and then impute missing genotypes into these estimated haplotypes. This reduces the computational cost because (i) the GWAS samples must be phased only once, whereas standard methods would implicitly repeat phasing with each reference panel update, and (ii) it is much faster to match a phased GWAS haplotype to one reference haplotype than to match two unphased GWAS genotypes to a pair of reference haplotypes. We implemented our approach in the MaCH and IMPUTE2 frameworks, and we tested it on data sets from the Wellcome Trust Case Control Consortium 2 (WTCCC2), the Genetic Association Information Network (GAIN), the Women's Health Initiative (WHI) and the 1000 Genomes Project. This strategy will be particularly valuable for repeated imputation as reference panels evolve. </li>
<li><strong>CITATION</strong> : Howie B, Fuchsberger C, Stephens M, Marchini J, ...&amp;, Abecasis GR. (2012) Fast and accurate genotype imputation in genome-wide association studies through pre-phasing Nat. Genet., 44 (8) 955-959. doi:10.1038/ng.2354. PMID 22820512 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2012 ; 44 ; 8 ; 955-959 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22820512">22820512</a> </li>
</ul>
<h3 id="mach-minimach2">MACH / minimach2</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach2 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach2 </li>
<li><strong>DESCRIPTION</strong> : (minimac2) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac2">https://genome.sph.umich.edu/wiki/Minimac2</a> </li>
<li><strong>TITLE</strong> : Minimac2: Faster genotype imputation </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btu704 </li>
<li><strong>ABSTRACT</strong> : UNLABELLED: Genotype imputation is a key step in the analysis of genome-wide association studies. Upcoming very large reference panels, such as those from The 1000 Genomes Project and the Haplotype Consortium, will improve imputation quality of rare and less common variants, but will also increase the computational burden. Here, we demonstrate how the application of software engineering techniques can help to keep imputation broadly accessible. Overall, these improvements speed up imputation by an order of magnitude compared with our previous implementation. AVAILABILITY AND IMPLEMENTATION: minimac2, including source code, documentation, and examples is available at http://genome.sph.umich.edu/wiki/Minimac2 </li>
<li><strong>CITATION</strong> : Fuchsberger C, Abecasis GR, Hinds DA. (2015) Minimac2: Faster genotype imputation Bioinformatics, 31 (5) 782-784. doi:10.1093/bioinformatics/btu704. PMID 25338720 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2015 ; 31 ; 5 ; 782-784 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25338720">25338720</a> </li>
</ul>
<h3 id="mach-minimach3">MACH / minimach3</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach3 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach3 </li>
<li><strong>DESCRIPTION</strong> : (minimac3) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac3">https://genome.sph.umich.edu/wiki/Minimac3</a> </li>
<li><strong>TITLE</strong> : Next-generation genotype imputation service and methods </li>
<li><strong>DOI</strong> : 10.1038/ng.3656 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation is a key component of genetic association studies, where it increases power, facilitates meta-analysis, and aids interpretation of signals. Genotype imputation is computationally demanding and, with current tools, typically requires access to a high-performance computing cluster and to a reference panel of sequenced genomes. Here we describe improvements to imputation machinery that reduce computational requirements by more than an order of magnitude with no loss of accuracy in comparison to standard imputation tools. We also describe a new web-based service for imputation that facilitates access to new reference panels and greatly improves user experience and productivity. </li>
<li><strong>CITATION</strong> : Das S, Forer L, Schönherr S, Sidore C, ...&amp;, Fuchsberger C. (2016) Next-generation genotype imputation service and methods Nat. Genet., 48 (10) 1284-1287. doi:10.1038/ng.3656. PMID 27571263 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 10 ; 1284-1287 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27571263">27571263</a> </li>
</ul>
<h3 id="mach-minimach4">MACH / minimach4</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach4 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach4 </li>
<li><strong>DESCRIPTION</strong> : (minimac4) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac4">https://genome.sph.umich.edu/wiki/Minimac4</a> </li>
</ul>
<h3 id="quilt1">QUILT1</h3>
<ul>
<li><strong>NAME</strong> : QUILT1 </li>
<li><strong>SHORT NAME</strong> : QUILT1 </li>
<li><strong>URL</strong> : <a href="https://github.com/rwdavies/QUILT">https://github.com/rwdavies/QUILT</a> </li>
<li><strong>TITLE</strong> : Rapid genotype imputation from sequence with reference panels </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00877-0 </li>
<li><strong>ABSTRACT</strong> : Inexpensive genotyping methods are essential to modern genomics. Here we present QUILT, which performs diploid genotype imputation using low-coverage whole-genome sequence data. QUILT employs Gibbs sampling to partition reads into maternal and paternal sets, facilitating rapid haploid imputation using large reference panels. We show this partitioning to be accurate over many megabases, enabling highly accurate imputation close to theoretical limits and outperforming existing methods. Moreover, QUILT can impute accurately using diverse technologies, including long reads from Oxford Nanopore Technologies, and a new form of low-cost barcoded Illumina sequencing called haplotagging, with the latter showing improved accuracy at low coverages. Relative to DNA genotyping microarrays, QUILT offers improved accuracy at reduced cost, particularly for diverse populations that are traditionally underserved in modern genomic analyses, with accuracy nearly doubling at rare SNPs. Finally, QUILT can accurately impute (four-digit) human leukocyte antigen types, the first such method from low-coverage sequence data. </li>
<li><strong>CITATION</strong> : Davies RW, Kucka M, Su D, Shi S, ...&amp;, Myers S. (2021) Rapid genotype imputation from sequence with reference panels Nat. Genet., 53 (7) 1104-1111. doi:10.1038/s41588-021-00877-0. PMID 34083788 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 7 ; 1104-1111 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34083788">34083788</a> </li>
</ul>
<h3 id="quilt2">QUILT2</h3>
<ul>
<li><strong>NAME</strong> : QUILT2 </li>
<li><strong>SHORT NAME</strong> : QUILT2 </li>
<li><strong>DESCRIPTION</strong> : QUILT2 is a fast and memory-efficient method for imputation from low coverage sequence. Statistically, QUILT2 operates on a per-read basis, and is base quality aware, meaning it can accurately impute from diverse inputs, including short read (e.g. Illumina), long read sequencing (that might be noisy) (e.g. Oxford Nanopore Technologies), barcoded Illumina sequencing (e.g. Haplotagging) and ancient DNA. In addition, QUILT2 can impute both the mother and fetal genome using cfDNA NIPT data. </li>
<li><strong>URL</strong> : <a href="https://github.com/rwdavies/QUILT">https://github.com/rwdavies/QUILT</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.07.18.604149 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Li, Z., Albrechtsen, A. &amp; Davies, R. W. Rapid and accurate genotype imputation from low coverage short read, long read, and cell free DNA sequence. bioRxiv 2024.07.18.604149 (2024) doi:10.1101/2024.07.18.604149. </li>
</ul>
<h3 id="shapeit1">SHAPEIT1</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT1 </li>
<li><strong>SHORT NAME</strong> : SHAPEIT1 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT1) </li>
<li><strong>URL</strong> : <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</a> </li>
<li><strong>TITLE</strong> : A linear complexity phasing method for thousands of genomes </li>
<li><strong>DOI</strong> : 10.1038/nmeth.1785 </li>
<li><strong>ABSTRACT</strong> : Human-disease etiology can be better understood with phase information about diploid sequences. We present a method for estimating haplotypes, using genotype data from unrelated samples or small nuclear families, that leads to improved accuracy and speed compared to several widely used methods. The method, segmented haplotype estimation and imputation tool (SHAPEIT), scales linearly with the number of haplotypes used in each iteration and can be run efficiently on whole chromosomes. </li>
<li><strong>CITATION</strong> : Delaneau O, Marchini J, Zagury JF. (2011) A linear complexity phasing method for thousands of genomes Nat. Methods, 9 (2) 179-181. doi:10.1038/nmeth.1785. PMID 22138821 </li>
<li><strong>JOURNAL_INFO</strong> : Nature methods ; Nat. Methods ; 2011 ; 9 ; 2 ; 179-181 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22138821">22138821</a> </li>
</ul>
<h3 id="shapeit2">SHAPEIT2</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT2 </li>
<li><strong>SHORT NAME</strong> : SHAPEIT2 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT2) </li>
<li><strong>TITLE</strong> : Improved whole-chromosome phasing for disease and population genetic studies </li>
<li><strong>DOI</strong> : 10.1038/nmeth.2307 </li>
<li><strong>CITATION</strong> : Delaneau O, Zagury JF, Marchini J. (2013) Improved whole-chromosome phasing for disease and population genetic studies Nat. Methods, 10 (1) 5-6. doi:10.1038/nmeth.2307. PMID 23269371 </li>
<li><strong>JOURNAL_INFO</strong> : Nature methods ; Nat. Methods ; 2013 ; 10 ; 1 ; 5-6 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23269371">23269371</a> </li>
</ul>
<h3 id="shapeit3">SHAPEIT3</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT3 </li>
<li><strong>SHORT NAME</strong> : SHAPEIT3 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT3) </li>
<li><strong>URL</strong> : <a href="https://jmarchini.org/shapeit3/">https://jmarchini.org/shapeit3/</a> </li>
<li><strong>TITLE</strong> : Haplotype estimation for biobank-scale data sets </li>
<li><strong>DOI</strong> : 10.1038/ng.3583 </li>
<li><strong>ABSTRACT</strong> : The UK Biobank (UKB) has recently released genotypes on 152,328 individuals together with extensive phenotypic and lifestyle information. We present a new phasing method, SHAPEIT3, that can handle such biobank-scale data sets and results in switch error rates as low as ∼0.3%. The method exhibits O(NlogN) scaling with sample size N, enabling fast and accurate phasing of even larger cohorts. </li>
<li><strong>CITATION</strong> : O'Connell J, Sharp K, Shrine N, Wain L, ...&amp;, Marchini J. (2016) Haplotype estimation for biobank-scale data sets Nat. Genet., 48 (7) 817-820. doi:10.1038/ng.3583. PMID 27270105 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 7 ; 817-820 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27270105">27270105</a> </li>
</ul>
<h3 id="shapeit4">SHAPEIT4</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT4 </li>
<li><strong>SHORT NAME</strong> : SHAPEIT4 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT4) </li>
<li><strong>URL</strong> : <a href="https://odelaneau.github.io/shapeit4/">https://odelaneau.github.io/shapeit4/</a> </li>
<li><strong>TITLE</strong> : Accurate, scalable and integrative haplotype estimation </li>
<li><strong>DOI</strong> : 10.1038/s41467-019-13225-y </li>
<li><strong>ABSTRACT</strong> : The number of human genomes being genotyped or sequenced increases exponentially and efficient haplotype estimation methods able to handle this amount of data are now required. Here we present a method, SHAPEIT4, which substantially improves upon other methods to process large genotype and high coverage sequencing datasets. It notably exhibits sub-linear running times with sample size, provides highly accurate haplotypes and allows integrating external phasing information such as large reference panels of haplotypes, collections of pre-phased variants and long sequencing reads. We provide SHAPEIT4 in an open source format and demonstrate its performance in terms of accuracy and running times on two gold standard datasets: the UK Biobank data and the Genome In A Bottle. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Delaneau O, Zagury JF, Robinson MR, Marchini JL, ...&amp;, Dermitzakis ET. (2019) Accurate, scalable and integrative haplotype estimation Nat. Commun., 10 (1) 5436. doi:10.1038/s41467-019-13225-y. PMID 31780650 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2019 ; 10 ; 1 ; 5436 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31780650">31780650</a> </li>
</ul>
<h3 id="shapeit5">SHAPEIT5</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT5 </li>
<li><strong>SHORT NAME</strong> : SHAPEIT5 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT5) </li>
<li><strong>TITLE</strong> : Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01415-w </li>
<li><strong>ABSTRACT</strong> : Phasing involves distinguishing the two parentally inherited copies of each chromosome into haplotypes. Here, we introduce SHAPEIT5, a new phasing method that quickly and accurately processes large sequencing datasets and applied it to UK Biobank (UKB) whole-genome and whole-exome sequencing data. We demonstrate that SHAPEIT5 phases rare variants with low switch error rates of below 5% for variants present in just 1 sample out of 100,000. Furthermore, we outline a method for phasing singletons, which, although less precise, constitutes an important step towards future developments. We then demonstrate that the use of UKB as a reference panel improves the accuracy of genotype imputation, which is even more pronounced when phased with SHAPEIT5 compared with other methods. Finally, we screen the UKB data for loss-of-function compound heterozygous events and identify 549 genes where both gene copies are knocked out. These genes complement current knowledge of gene essentiality in the human genome. </li>
<li><strong>CITATION</strong> : Hofmeister RJ, Ribeiro DM, Rubinacci S, Delaneau O. (2022) Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank Nat. Genet., 55 (7) 1243-1249. doi:10.1038/s41588-023-01415-w. PMID 37386248 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2022 ; 55 ; 7 ; 1243-1249 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37386248">37386248</a> </li>
</ul>
<h3 id="fastphase">fastPHASE</h3>
<ul>
<li><strong>NAME</strong> : fastPHASE </li>
<li><strong>SHORT NAME</strong> : fastPHASE </li>
<li><strong>URL</strong> : <a href="http://scheet.org/software.html">http://scheet.org/software.html</a> </li>
<li><strong>TITLE</strong> : A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase </li>
<li><strong>DOI</strong> : 10.1086/502802 </li>
<li><strong>ABSTRACT</strong> : We present a statistical model for patterns of genetic variation in samples of unrelated individuals from natural populations. This model is based on the idea that, over short regions, haplotypes in a population tend to cluster into groups of similar haplotypes. To capture the fact that, because of recombination, this clustering tends to be local in nature, our model allows cluster memberships to change continuously along the chromosome according to a hidden Markov model. This approach is flexible, allowing for both "block-like" patterns of linkage disequilibrium (LD) and gradual decline in LD with distance. The resulting model is also fast and, as a result, is practicable for large data sets (e.g., thousands of individuals typed at hundreds of thousands of markers). We illustrate the utility of the model by applying it to dense single-nucleotide-polymorphism genotype data for the tasks of imputing missing genotypes and estimating haplotypic phase. For imputing missing genotypes, methods based on this model are as accurate or more accurate than existing methods. For haplotype estimation, the point estimates are slightly less accurate than those from the best existing methods (e.g., for unrelated Centre d'Etude du Polymorphisme Humain individuals from the HapMap project, switch error was 0.055 for our method vs. 0.051 for PHASE) but require a small fraction of the computational cost. In addition, we demonstrate that the model accurately reflects uncertainty in its estimates, in that probabilities computed using the model are approximately well calibrated. The methods described in this article are implemented in a software package, fastPHASE, which is available from the Stephens Lab Web site. </li>
<li><strong>COPYRIGHT</strong> : https://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Scheet P, Stephens M. (2006) A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase Am. J. Hum. Genet., 78 (4) 629-644. doi:10.1086/502802. PMID 16532393 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2006 ; 78 ; 4 ; 629-644 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/16532393">16532393</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-das">Review-Das</h3>
<ul>
<li><strong>NAME</strong> : Review-Das </li>
<li><strong>TITLE</strong> : Genotype imputation from large reference panels </li>
<li><strong>DOI</strong> : 10.1146/annurev-genom-083117-021602 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation has become a standard tool in genome-wide association studies because it enables researchers to inexpensively approximate whole-genome sequence data from genome-wide single-nucleotide polymorphism array data. Genotype imputation increases statistical power, facilitates fine mapping of causal variants, and plays a key role in meta-analyses of genome-wide association studies. Only variants that were previously observed in a reference panel of sequenced individuals can be imputed. However, the rapid increase in the number of deeply sequenced individuals will soon make it possible to assemble enormous reference panels that greatly increase the number of imputable variants. In this review, we present an overview of genotype imputation and describe the computational techniques that make it possible to impute genotypes from reference panels with millions of individuals. </li>
<li><strong>CITATION</strong> : Das S, Abecasis GR, Browning BL. (2018) Genotype imputation from large reference panels Annu. Rev. Genomics Hum. Genet., 19 (1) 73-96. doi:10.1146/annurev-genom-083117-021602. PMID 29799802 </li>
<li><strong>JOURNAL_INFO</strong> : Annual review of genomics and human genetics ; Annu. Rev. Genomics Hum. Genet. ; 2018 ; 19 ; 1 ; 73-96 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29799802">29799802</a> </li>
</ul>
<h3 id="review-li">Review-Li</h3>
<ul>
<li><strong>NAME</strong> : Review-Li </li>
<li><strong>TITLE</strong> : Genotype imputation </li>
<li><strong>DOI</strong> : 10.1146/annurev.genom.9.081307.164242 </li>
<li><strong>ABSTRACT</strong> : Genotype imputation is now an essential tool in the analysis of genome-wide association scans. This technique allows geneticists to accurately evaluate the evidence for association at genetic markers that are not directly genotyped. Genotype imputation is particularly useful for combining results across studies that rely on different genotyping platforms but also increases the power of individual scans. Here, we review the history and theoretical underpinnings of the technique. To illustrate performance of the approach, we summarize results from several gene mapping studies. Finally, we preview the role of genotype imputation in an era when whole genome resequencing is becoming increasingly common. </li>
<li><strong>CITATION</strong> : Li Y, Willer C, Sanna S, Abecasis G. (2009) Genotype imputation Annu. Rev. Genomics Hum. Genet., 10 (1) 387-406. doi:10.1146/annurev.genom.9.081307.164242. PMID 19715440 </li>
<li><strong>JOURNAL_INFO</strong> : Annual review of genomics and human genetics ; Annu. Rev. Genomics Hum. Genet. ; 2009 ; 10 ; 1 ; 387-406 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19715440">19715440</a> </li>
</ul>
<h3 id="review-marchini">Review-Marchini</h3>
<ul>
<li><strong>NAME</strong> : Review-Marchini </li>
<li><strong>TITLE</strong> : Genotype imputation for genome-wide association studies </li>
<li><strong>DOI</strong> : 10.1038/nrg2796 </li>
<li><strong>ABSTRACT</strong> : In the past few years genome-wide association (GWA) studies have uncovered a large number of convincingly replicated associations for many complex human diseases. Genotype imputation has been used widely in the analysis of GWA studies to boost power, fine-map associations and facilitate the combination of results across studies using meta-analysis. This Review describes the details of several different statistical methods for imputing genotypes, illustrates and discusses the factors that influence imputation performance, and reviews methods that can be used to assess imputation performance and test association at imputed SNPs. </li>
<li><strong>CITATION</strong> : Marchini J, Howie B. (2010) Genotype imputation for genome-wide association studies Nat. Rev. Genet., 11 (7) 499-511. doi:10.1038/nrg2796. PMID 20517342 </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Genetics ; Nat. Rev. Genet. ; 2010 ; 11 ; 7 ; 499-511 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20517342">20517342</a> </li>
</ul>
<h2 id="structural-variants-imputation-panel">Structural variants imputation panel</h2>
<h3 id="1kg-sv-imputation-panel">1KG SV imputation panel</h3>
<ul>
<li><strong>NAME</strong> : 1KG SV imputation panel </li>
<li><strong>SHORT NAME</strong> : 1KG SV </li>
<li><strong>KEYWORDS</strong> : structural variants, long-read </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.12.20.23300308 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>CITATION</strong> : Noyvert, B., Erzurumluoglu, A. M., Drichel, D., Omland, S., Andlauer, T. F., Mueller, S., ... &amp; Ding, Z. (2023). Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations. medRxiv, 2023-12. </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.13a4f30d.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>