
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Heritability_and_genetic_correlation_README/">
      
      
        <link rel="next" href="../Tools_Mendelian_randomization_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Imputation - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Imputation
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1000-genomes-1kg" class="md-nav__link">
    <span class="md-ellipsis">
      1000 Genomes (1KG)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1kg7k" class="md-nav__link">
    <span class="md-ellipsis">
      1KG+7K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ckb-reference-panel" class="md-nav__link">
    <span class="md-ellipsis">
      CKB reference panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hgdp1kgp" class="md-nav__link">
    <span class="md-ellipsis">
      HGDP+1kGP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hrc" class="md-nav__link">
    <span class="md-ellipsis">
      HRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard" class="md-nav__link">
    <span class="md-ellipsis">
      NARD
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard2" class="md-nav__link">
    <span class="md-ellipsis">
      NARD2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-genome-resource-phase-1" class="md-nav__link">
    <span class="md-ellipsis">
      Nyuwa Genome Resource Phase 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#south-and-east-asian-reference-database-sead" class="md-nav__link">
    <span class="md-ellipsis">
      South and East Asian Reference Database (SEAD)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wbbc-panel" class="md-nav__link">
    <span class="md-ellipsis">
      WBBC panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation server
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation server">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cngb-imputation-service" class="md-nav__link">
    <span class="md-ellipsis">
      CNGB Imputation Service
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#michigan-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Michigan Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      NyuWa Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sanger" class="md-nav__link">
    <span class="md-ellipsis">
      Sanger
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed_1" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#westlake-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Westlake Imputation Server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-tools" class="md-nav__link">
    <span class="md-ellipsis">
      Other tools
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Other tools">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reshape" class="md-nav__link">
    <span class="md-ellipsis">
      RESHAPE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasing-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      Phasing &amp; Imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Phasing & Imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle4" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-imputation" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Imputation)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Phasing)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle2" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimpse" class="md-nav__link">
    <span class="md-ellipsis">
      GLIMPSE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute2" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute4" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute5" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach-pre-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach pre-phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach2" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach3" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach4" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit1" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit2" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit3" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit5" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastphase" class="md-nav__link">
    <span class="md-ellipsis">
      fastPHASE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-das" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Das
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-li" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Li
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-marchini" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Marchini
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structural-variants-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Structural variants imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Structural variants imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1kg-sv-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      1KG SV imputation panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1000-genomes-1kg" class="md-nav__link">
    <span class="md-ellipsis">
      1000 Genomes (1KG)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#1kg7k" class="md-nav__link">
    <span class="md-ellipsis">
      1KG+7K
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ckb-reference-panel" class="md-nav__link">
    <span class="md-ellipsis">
      CKB reference panel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hgdp1kgp" class="md-nav__link">
    <span class="md-ellipsis">
      HGDP+1kGP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hrc" class="md-nav__link">
    <span class="md-ellipsis">
      HRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard" class="md-nav__link">
    <span class="md-ellipsis">
      NARD
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nard2" class="md-nav__link">
    <span class="md-ellipsis">
      NARD2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-genome-resource-phase-1" class="md-nav__link">
    <span class="md-ellipsis">
      Nyuwa Genome Resource Phase 1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#south-and-east-asian-reference-database-sead" class="md-nav__link">
    <span class="md-ellipsis">
      South and East Asian Reference Database (SEAD)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wbbc-panel" class="md-nav__link">
    <span class="md-ellipsis">
      WBBC panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Imputation server
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Imputation server">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cngb-imputation-service" class="md-nav__link">
    <span class="md-ellipsis">
      CNGB Imputation Service
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#michigan-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Michigan Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nyuwa-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      NyuWa Imputation Server
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sanger" class="md-nav__link">
    <span class="md-ellipsis">
      Sanger
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#topmed_1" class="md-nav__link">
    <span class="md-ellipsis">
      TOPMED
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#westlake-imputation-server" class="md-nav__link">
    <span class="md-ellipsis">
      Westlake Imputation Server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-tools" class="md-nav__link">
    <span class="md-ellipsis">
      Other tools
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Other tools">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reshape" class="md-nav__link">
    <span class="md-ellipsis">
      RESHAPE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasing-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      Phasing &amp; Imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Phasing & Imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle4" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-imputation" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Imputation)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#beagle54-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      BEAGLE5.4 (Phasing)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eagle2" class="md-nav__link">
    <span class="md-ellipsis">
      EAGLE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#glimpse" class="md-nav__link">
    <span class="md-ellipsis">
      GLIMPSE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute2" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute4" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#impute5" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach-pre-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach pre-phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach2" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach3" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mach-minimach4" class="md-nav__link">
    <span class="md-ellipsis">
      MACH / minimach4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit1" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT1
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit2" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit3" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT3
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shapeit5" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT5
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastphase" class="md-nav__link">
    <span class="md-ellipsis">
      fastPHASE
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-das" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Das
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-li" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Li
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-marchini" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Marchini
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structural-variants-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      Structural variants imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Structural variants imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1kg-sv-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      1KG SV imputation panel
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Imputation</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#1000-genomes-1kg">1000 Genomes (1KG)</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">1000 Genomes Project Consortium. "A global reference for human genetic variation." Nature 526.7571 (2015): 68.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#1kg7k">1KG+7K</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Terao, C., Flanagan, J., Tomizuka, K., Liu, X., Ortega-Reyes, D., Matoba, N., ... &amp; Horikoshi, M. (2023). Population-specific reference panel improves imputation quality and enhances locus discovery and fine-mapping.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ckb-reference-panel">CKB reference panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yu, C., Lan, X., Tao, Y., Guo, Y., Sun, D., Qian, P., ... &amp; Li, L. (2023). A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study. Nucleic Acids Research, 51(21), 11770-11782.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hgdp1kgp">HGDP+1kGP</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Koenig, Z., Yohannes, M. T., Nkambule, L. L., Goodrich, J. K., Kim, H. A., Zhao, X., ... &amp; Martin, A. R. (2023). A harmonized public resource of deeply sequenced diverse human genomes. bioRxiv, 2023-01.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hrc">HRC</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">the Haplotype Reference Consortium. "A reference panel of 64,976 haplotypes for genotype imputation". Nature genetics, 2016, 48(10): 1279-1283.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nard2">NARD2</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Choi, J., Kim, S., Kim, J., Son, H. Y., Yoo, S. K., Kim, C. U., ... &amp; Im, S. W. (2023). A whole-genome reference panel of 14,393 individuals for East Asian populations accelerates discovery of rare functional variants. Science Advances, 9(32), eadg6319.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nard">NARD</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yoo, S. K., Kim, C. U., Kim, H. L., Kim, S., Shin, J. Y., Kim, N., ... &amp; Seo, J. S. (2019). NARD: whole-genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low-frequency variants. Genome medicine, 11, 1-10.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nyuwa-genome-resource-phase-1">Nyuwa Genome Resource Phase 1</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Zhang, P., Luo, H., Li, Y., Wang, Y., Wang, J., Zheng, Y., ... &amp; Han100K Initiative. (2021). NyuWa Genome resource: a deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Reports, 37(7), 110017.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#south-and-east-asian-reference-database-sead">South and East Asian Reference Database (SEAD)</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Yang, M. Y., Zhong, J. D., Li, X., Bai, W. Y., Yuan, C. D., Qiu, M. C., ... &amp; Zheng, H. F. (2023). SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population. medRxiv, 2023-12.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#topmed">TOPMED</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Taliun, Daniel, et al. "Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program." Nature 590.7845 (2021): 290-299.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wbbc-panel">WBBC panel</a></td>
<td style="text-align: left;">Imputation panel</td>
<td style="text-align: left;">Cong, P. K., Bai, W. Y., Li, J. C., Yang, M. Y., Khederzadeh, S., Gai, S. R., ... &amp; Zheng, H. F. (2022). Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project. Nature Communications, 13(1), 1-15.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cngb-imputation-service">CNGB Imputation Service</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Yu, C., Lan, X., Tao, Y., Guo, Y., Sun, D., Qian, P., ... &amp; Li, L. (2022). A High-resolution Haplotype-resolved Reference Panel Constructed from the China Kadoorie Biobank Study. medRxiv, 2022-12.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#michigan-imputation-server">Michigan Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Das, Sayantan, et al. "Next-generation genotype imputation service and methods." Nature genetics 48.10 (2016): 1284-1287.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#nyuwa-imputation-server">NyuWa Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Zhang, P., Luo, H., Li, Y., Wang, Y., Wang, J., Zheng, Y., ... &amp; Han100K Initiative. (2021). NyuWa Genome resource: a deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Reports, 37(7), 110017.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sanger">Sanger</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">"A reference panel of 64,976 haplotypes for genotype imputation." Nature genetics 48, no. 10 (2016): 1279-1283.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#topmed">TOPMED</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Taliun, D., Harris, D. N., Kessler, M. D., Carlson, J., Szpiech, Z. A., Torres, R., ... &amp; Stilp, A. M. (2021). Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature, 590(7845), 290-299.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#westlake-imputation-server">Westlake Imputation Server</a></td>
<td style="text-align: left;">Imputation server</td>
<td style="text-align: left;">Cong, P. K., Bai, W. Y., Li, J. C., Yang, M. Y., Khederzadeh, S., Gai, S. R., ... &amp; Zheng, H. F. (2022). Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project. Nature Communications, 13(1), 1-15.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#reshape">RESHAPE</a></td>
<td style="text-align: left;">Other tools</td>
<td style="text-align: left;">Cavinato, T., Rubinacci, S., Malaspinas, A. S., &amp; Delaneau, O. (2023). A resampling-based approach to share reference panels. bioRxiv, 2023-04.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle4">BEAGLE4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning, Brian L., and Sharon R. Browning. "Genotype imputation with millions of reference samples." The American Journal of Human Genetics 98.1 (2016): 116-126.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle54-imputation">BEAGLE5.4 (Imputation)</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning, Brian L., Ying Zhou, and Sharon R. Browning. "A one-penny imputed genome from next-generation reference panels." The American Journal of Human Genetics 103.3 (2018): 338-348.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle54-phasing">BEAGLE5.4 (Phasing)</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Browning, Brian L., et al. "Fast two-stage phasing of large-scale sequence data." The American Journal of Human Genetics 108.10 (2021): 1880-1890.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#beagle">BEAGLE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">(beagle) Browning, Sharon R., and Brian L. Browning. "Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering." The American Journal of Human Genetics 81.5 (2007): 1084-1097.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#eagle2">EAGLE2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Loh, Po-Ru, et al. "Reference-based phasing using the Haplotype Reference Consortium panel." Nature genetics 48.11 (2016): 1443-1448.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#eagle">EAGLE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Loh, Po-Ru, Pier Francesco Palamara, and Alkes L. Price. "Fast and accurate long-range phasing in a UK Biobank cohort." Nature genetics 48.7 (2016): 811-816.<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#glimpse">GLIMPSE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Rubinacci, S., Ribeiro, D. M., Hofmeister, R. J., &amp; Delaneau, O. (2021). Efficient phasing and imputation of low-coverage sequencing data using large reference panels. Nature Genetics, 53(1), 120-126.<br><br>(using GLIMPSE for ancient DNA) Sousa da Mota, B., Rubinacci, S., Cruz Dvalos, D. I., G. Amorim, C. E., Sikora, M., Johannsen, N. N., ... &amp; Delaneau, O. (2023). Imputation of ancient human genomes. Nature Communications, 14(1), 3660.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute2">IMPUTE2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Howie, Bryan N., Peter Donnelly, and Jonathan Marchini. "A flexible and accurate genotype imputation method for the next generation of genome-wide association studies." PLoS genetics 5.6 (2009): e1000529.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute4">IMPUTE4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Bycroft, Clare, et al. "The UK Biobank resource with deep phenotyping and genomic data." Nature 562.7726 (2018): 203-209.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute5">IMPUTE5</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Rubinacci, Simone, Olivier Delaneau, and Jonathan Marchini. "Genotype imputation using the positional burrows wheeler transform." PLoS genetics 16.11 (2020): e1009049.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#impute">IMPUTE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Marchini, Jonathan, et al. "A new multipoint method for genome-wide association studies by imputation of genotypes." Nature genetics 39.7 (2007): 906-913.<br><br><br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach-pre-phasing">MACH / minimach pre-phasing</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Howie, Bryan, et al. "Fast and accurate genotype imputation in genome-wide association studies through pre-phasing." Nature genetics 44.8 (2012): 955-959.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach2">MACH / minimach2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Fuchsberger, Christian, Gonalo R. Abecasis, and David A. Hinds. "minimac2: faster genotype imputation." Bioinformatics 31.5 (2015): 782-784.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach3">MACH / minimach3</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Das, Sayantan, et al. "Next-generation genotype imputation service and methods." Nature genetics 48.10 (2016): 1284-1287.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach4">MACH / minimach4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mach-minimach">MACH / minimach</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Li, Yun, et al. "MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes." Genetic epidemiology 34.8 (2010): 816-834.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit1">SHAPEIT1</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">(SHAPEIT1) Delaneau, Olivier, Jonathan Marchini, and Jean-Franois Zagury. "A linear complexity phasing method for thousands of genomes." Nature methods 9.2 (2012): 179-181.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit2">SHAPEIT2</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">(SHAPEIT2) Delaneau, Olivier, Jean-Francois Zagury, and Jonathan Marchini. "Improved whole-chromosome phasing for disease and population genetic studies." Nature methods 10.1 (2013): 5-6.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit3">SHAPEIT3</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;"><br><br>(SHAPEIT3) O'Connell, Jared, et al. "Haplotype estimation for biobank-scale data sets." Nature genetics 48.7 (2016): 817-820.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit4">SHAPEIT4</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">(SHAPEIT4) Delaneau, Olivier, et al. "Accurate, scalable and integrative haplotype estimation." Nature communications 10.1 (2019): 1-10.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shapeit5">SHAPEIT5</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">(SHAPEIT5) Hofmeister, R. J., Ribeiro, D. M., Rubinacci, S., &amp; Delaneau, O. (2023). Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. Nature Genetics, 55(7), 1243-1249.<br><br>(SHAPEIT5) Hofmeister, R. J., Ribeiro, D. M., Rubinacci, S., &amp; Delaneau, O. (2022). Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. bioRxiv, 2022-10.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastphase">fastPHASE</a></td>
<td style="text-align: left;">Phasing &amp; Imputation tool</td>
<td style="text-align: left;">Scheet, Paul, and Matthew Stephens. "A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase." The American Journal of Human Genetics 78.4 (2006): 629-644.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-das">Review-Das</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Das, Sayantan, Gonalo R. Abecasis, and Brian L. Browning. "Genotype imputation from large reference panels." Annu Rev Genomics Hum Genet 19.1 (2018): 73-96.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-li">Review-Li</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Li, Yun, et al. "Genotype imputation." Annual review of genomics and human genetics 10 (2009): 387.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-marchini">Review-Marchini</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Marchini, Jonathan, and Bryan Howie. "Genotype imputation for genome-wide association studies." Nature Reviews Genetics 11.7 (2010): 499-511.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#1kg-sv-imputation-panel">1KG SV imputation panel</a></td>
<td style="text-align: left;">Structural variants imputation panel</td>
<td style="text-align: left;">Noyvert, B., Erzurumluoglu, A. M., Drichel, D., Omland, S., Andlauer, T. F., Mueller, S., ... &amp; Ding, Z. (2023). Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations. medRxiv, 2023-12.</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="imputation-panel">Imputation panel</h2>
<h3 id="1000-genomes-1kg">1000 Genomes (1KG)</h3>
<ul>
<li><strong>NAME</strong> : 1000 Genomes (1KG) </li>
<li><strong>SHORT NAME</strong> : 1KG </li>
<li><strong>CITATION</strong> : 1000 Genomes Project Consortium. "A global reference for human genetic variation." Nature 526.7571 (2015): 68. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26432245">26432245</a> </li>
</ul>
<h3 id="1kg7k">1KG+7K</h3>
<ul>
<li><strong>NAME</strong> : 1KG+7K </li>
<li><strong>SHORT NAME</strong> : 1KG+7K </li>
<li><strong>FULL NAME</strong> : 1KG+7K </li>
<li><strong>KEYWORDS</strong> : Japanese population-specific reference panel </li>
<li><strong>PREPRINT_DOI</strong> : 10.21203/rs.3.rs-3194976/v1 </li>
<li><strong>SERVER</strong> : researchsquare </li>
<li><strong>CITATION</strong> : Terao, C., Flanagan, J., Tomizuka, K., Liu, X., Ortega-Reyes, D., Matoba, N., ... &amp; Horikoshi, M. (2023). Population-specific reference panel improves imputation quality and enhances locus discovery and fine-mapping. </li>
</ul>
<h3 id="ckb-reference-panel">CKB reference panel</h3>
<ul>
<li><strong>NAME</strong> : CKB reference panel </li>
<li><strong>SHORT NAME</strong> : CKB </li>
<li><strong>FULL NAME</strong> : China Kadoorie Biobank </li>
<li><strong>URL</strong> : <a href="https://db.cngb.org/imputation/">https://db.cngb.org/imputation/</a> </li>
<li><strong>CITATION</strong> : Yu, C., Lan, X., Tao, Y., Guo, Y., Sun, D., Qian, P., ... &amp; Li, L. (2023). A high-resolution haplotype-resolved Reference panel constructed from the China Kadoorie Biobank Study. Nucleic Acids Research, 51(21), 11770-11782. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37870428">37870428</a> </li>
</ul>
<h3 id="hgdp1kgp">HGDP+1kGP</h3>
<ul>
<li><strong>NAME</strong> : HGDP+1kGP </li>
<li><strong>SHORT NAME</strong> : HGDP+1kGP </li>
<li><strong>FULL NAME</strong> : Human Genome Diversity Project + 1000 Genomes project </li>
<li><strong>URL</strong> : <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability/#the-gnomad-hgdp-and-1000-genomes-callset">https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability/#the-gnomad-hgdp-and-1000-genomes-callset</a> </li>
<li><strong>CITATION</strong> : Koenig, Z., Yohannes, M. T., Nkambule, L. L., Goodrich, J. K., Kim, H. A., Zhao, X., ... &amp; Martin, A. R. (2023). A harmonized public resource of deeply sequenced diverse human genomes. bioRxiv, 2023-01. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38749656">38749656</a> </li>
</ul>
<h3 id="hrc">HRC</h3>
<ul>
<li><strong>NAME</strong> : HRC </li>
<li><strong>SHORT NAME</strong> : HRC </li>
<li><strong>CITATION</strong> : the Haplotype Reference Consortium. "A reference panel of 64,976 haplotypes for genotype imputation". Nature genetics, 2016, 48(10): 1279-1283. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27548312">27548312</a> </li>
</ul>
<h3 id="nard">NARD</h3>
<ul>
<li><strong>NAME</strong> : NARD </li>
<li><strong>SHORT NAME</strong> : NARD </li>
<li><strong>FULL NAME</strong> : Northeast Asian Reference Database </li>
<li><strong>URL</strong> : <a href="https://nard.macrogen.com/">https://nard.macrogen.com/</a> </li>
<li><strong>CITATION</strong> : Yoo, S. K., Kim, C. U., Kim, H. L., Kim, S., Shin, J. Y., Kim, N., ... &amp; Seo, J. S. (2019). NARD: whole-genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low-frequency variants. Genome medicine, 11, 1-10. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31640730">31640730</a> </li>
</ul>
<h3 id="nard2">NARD2</h3>
<ul>
<li><strong>NAME</strong> : NARD2 </li>
<li><strong>SHORT NAME</strong> : NARD2 </li>
<li><strong>FULL NAME</strong> : Northeast Asian Reference Database 2 </li>
<li><strong>URL</strong> : <a href="https://nard.macrogen.com/">https://nard.macrogen.com/</a> </li>
<li><strong>CITATION</strong> : Choi, J., Kim, S., Kim, J., Son, H. Y., Yoo, S. K., Kim, C. U., ... &amp; Im, S. W. (2023). A whole-genome reference panel of 14,393 individuals for East Asian populations accelerates discovery of rare functional variants. Science Advances, 9(32), eadg6319. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37556544">37556544</a> </li>
</ul>
<h3 id="nyuwa-genome-resource-phase-1">Nyuwa Genome Resource Phase 1</h3>
<ul>
<li><strong>NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>SHORT NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>FULL NAME</strong> : Nyuwa Genome Resource Phase 1 </li>
<li><strong>URL</strong> : <a href="http://bigdata.ibp.ac.cn/refpanel/getstarted.php">http://bigdata.ibp.ac.cn/refpanel/getstarted.php</a> </li>
<li><strong>CITATION</strong> : Zhang, P., Luo, H., Li, Y., Wang, Y., Wang, J., Zheng, Y., ... &amp; Han100K Initiative. (2021). NyuWa Genome resource: a deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Reports, 37(7), 110017. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34788621">34788621</a> </li>
</ul>
<h3 id="south-and-east-asian-reference-database-sead">South and East Asian Reference Database (SEAD)</h3>
<ul>
<li><strong>NAME</strong> : South and East Asian Reference Database (SEAD) </li>
<li><strong>SHORT NAME</strong> : SEAD </li>
<li><strong>FULL NAME</strong> : South and East Asian Reference Database </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.12.23.23300480 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>CITATION</strong> : Yang, M. Y., Zhong, J. D., Li, X., Bai, W. Y., Yuan, C. D., Qiu, M. C., ... &amp; Zheng, H. F. (2023). SEAD: an augmented reference panel with 22,134 haplotypes boosts the rare variants imputation and GWAS analysis in Asian population. medRxiv, 2023-12. </li>
</ul>
<h3 id="topmed">TOPMED</h3>
<ul>
<li><strong>NAME</strong> : TOPMED </li>
<li><strong>CITATION</strong> : Taliun, Daniel, et al. "Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program." Nature 590.7845 (2021): 290-299. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33568819">33568819</a> </li>
</ul>
<h3 id="wbbc-panel">WBBC panel</h3>
<ul>
<li><strong>NAME</strong> : WBBC panel </li>
<li><strong>SHORT NAME</strong> : WBBC </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>CITATION</strong> : Cong, P. K., Bai, W. Y., Li, J. C., Yang, M. Y., Khederzadeh, S., Gai, S. R., ... &amp; Zheng, H. F. (2022). Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project. Nature Communications, 13(1), 1-15. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35618720">35618720</a> </li>
</ul>
<h2 id="imputation-server">Imputation server</h2>
<h3 id="cngb-imputation-service">CNGB Imputation Service</h3>
<ul>
<li><strong>NAME</strong> : CNGB Imputation Service </li>
<li><strong>SHORT NAME</strong> : CNGB </li>
<li><strong>FULL NAME</strong> : China National GeneBank </li>
<li><strong>URL</strong> : <a href="https://db.cngb.org/imputation/">https://db.cngb.org/imputation/</a> </li>
<li><strong>CITATION</strong> : Yu, C., Lan, X., Tao, Y., Guo, Y., Sun, D., Qian, P., ... &amp; Li, L. (2022). A High-resolution Haplotype-resolved Reference Panel Constructed from the China Kadoorie Biobank Study. medRxiv, 2022-12. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37870428">37870428</a> </li>
</ul>
<h3 id="michigan-imputation-server">Michigan Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : Michigan Imputation Server </li>
<li><strong>SHORT NAME</strong> : Michigan </li>
<li><strong>URL</strong> : <a href="https://imputationserver.sph.umich.edu/index.html#!">https://imputationserver.sph.umich.edu/index.html#!</a> </li>
<li><strong>CITATION</strong> : Das, Sayantan, et al. "Next-generation genotype imputation service and methods." Nature genetics 48.10 (2016): 1284-1287. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27571263">27571263</a> </li>
</ul>
<h3 id="nyuwa-imputation-server">NyuWa Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : NyuWa Imputation Server </li>
<li><strong>SHORT NAME</strong> : NyuWa </li>
<li><strong>URL</strong> : <a href="http://bigdata.ibp.ac.cn/refpanel/getstarted.php">http://bigdata.ibp.ac.cn/refpanel/getstarted.php</a> </li>
<li><strong>CITATION</strong> : Zhang, P., Luo, H., Li, Y., Wang, Y., Wang, J., Zheng, Y., ... &amp; Han100K Initiative. (2021). NyuWa Genome resource: a deep whole-genome sequencing-based variation profile and reference panel for the Chinese population. Cell Reports, 37(7), 110017. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34788621">34788621</a> </li>
</ul>
<h3 id="sanger">Sanger</h3>
<ul>
<li><strong>NAME</strong> : Sanger </li>
<li><strong>URL</strong> : <a href="https://imputation.sanger.ac.uk/">https://imputation.sanger.ac.uk/</a> </li>
<li><strong>CITATION</strong> : "A reference panel of 64,976 haplotypes for genotype imputation." Nature genetics 48, no. 10 (2016): 1279-1283. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27548312">27548312</a> </li>
</ul>
<h3 id="topmed_1">TOPMED</h3>
<ul>
<li><strong>NAME</strong> : TOPMED </li>
<li><strong>URL</strong> : <a href="https://imputation.biodatacatalyst.nhlbi.nih.gov/#!">https://imputation.biodatacatalyst.nhlbi.nih.gov/#!</a> </li>
<li><strong>CITATION</strong> : Taliun, D., Harris, D. N., Kessler, M. D., Carlson, J., Szpiech, Z. A., Torres, R., ... &amp; Stilp, A. M. (2021). Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature, 590(7845), 290-299. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33568819">33568819</a> </li>
</ul>
<h3 id="westlake-imputation-server">Westlake Imputation Server</h3>
<ul>
<li><strong>NAME</strong> : Westlake Imputation Server </li>
<li><strong>URL</strong> : <a href="https://imputationserver.westlake.edu.cn/">https://imputationserver.westlake.edu.cn/</a> </li>
<li><strong>CITATION</strong> : Cong, P. K., Bai, W. Y., Li, J. C., Yang, M. Y., Khederzadeh, S., Gai, S. R., ... &amp; Zheng, H. F. (2022). Genomic analyses of 10,376 individuals in the Westlake BioBank for Chinese (WBBC) pilot project. Nature Communications, 13(1), 1-15. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35618720">35618720</a> </li>
</ul>
<h2 id="other-tools">Other tools</h2>
<h3 id="reshape">RESHAPE</h3>
<ul>
<li><strong>NAME</strong> : RESHAPE </li>
<li><strong>SHORT NAME</strong> : RESHAPE </li>
<li><strong>FULL NAME</strong> : REcombine and Share HAPlotypEs </li>
<li><strong>DESCRIPTION</strong> : RESHAPE removes sample-level genetic information from a reference panel to create a synthetic reference panel. By providing it with a genetic map and the VCF/BCF of a reference panel, RESHAPE outputs a VCF/BCF of the same size where each haplotypes corresponds to a mosaic of the original haplotypes of the reference panel. </li>
<li><strong>URL</strong> : <a href="https://github.com/TheoCavinato/RESHAPE">https://github.com/TheoCavinato/RESHAPE</a> </li>
<li><strong>CITATION</strong> : Cavinato, T., Rubinacci, S., Malaspinas, A. S., &amp; Delaneau, O. (2023). A resampling-based approach to share reference panels. bioRxiv, 2023-04. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38745108">38745108</a> </li>
</ul>
<h2 id="phasing-imputation-tool">Phasing &amp; Imputation tool</h2>
<h3 id="beagle">BEAGLE</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE </li>
<li><strong>SHORT NAME</strong> : BEAGLE </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>CITATION</strong> : (beagle) Browning, Sharon R., and Brian L. Browning. "Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering." The American Journal of Human Genetics 81.5 (2007): 1084-1097. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17924348">17924348</a> </li>
</ul>
<h3 id="beagle4">BEAGLE4</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE4 </li>
<li><strong>SHORT NAME</strong> : BEAGLE4 </li>
<li><strong>DESCRIPTION</strong> : (beagle 4.1) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>CITATION</strong> : Browning, Brian L., and Sharon R. Browning. "Genotype imputation with millions of reference samples." The American Journal of Human Genetics 98.1 (2016): 116-126. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26748515">26748515</a> </li>
</ul>
<h3 id="beagle54-imputation">BEAGLE5.4 (Imputation)</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE5.4 (Imputation) </li>
<li><strong>SHORT NAME</strong> : BEAGLE5.4 (Imputation) </li>
<li><strong>DESCRIPTION</strong> : (beagle 5.4 imputation) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>CITATION</strong> : Browning, Brian L., Ying Zhou, and Sharon R. Browning. "A one-penny imputed genome from next-generation reference panels." The American Journal of Human Genetics 103.3 (2018): 338-348. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30100085">30100085</a> </li>
</ul>
<h3 id="beagle54-phasing">BEAGLE5.4 (Phasing)</h3>
<ul>
<li><strong>NAME</strong> : BEAGLE5.4 (Phasing) </li>
<li><strong>SHORT NAME</strong> : BEAGLE5.4 (Phasing) </li>
<li><strong>DESCRIPTION</strong> : (beagle 5.4 phasing) </li>
<li><strong>URL</strong> : <a href="https://faculty.washington.edu/browning/beagle/beagle.html">https://faculty.washington.edu/browning/beagle/beagle.html</a> </li>
<li><strong>CITATION</strong> : Browning, Brian L., et al. "Fast two-stage phasing of large-scale sequence data." The American Journal of Human Genetics 108.10 (2021): 1880-1890. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34478634">34478634</a> </li>
</ul>
<h3 id="eagle">EAGLE</h3>
<ul>
<li><strong>NAME</strong> : EAGLE </li>
<li><strong>SHORT NAME</strong> : EAGLE </li>
<li><strong>DESCRIPTION</strong> : (EAGLE1) </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/Eagle/">https://alkesgroup.broadinstitute.org/Eagle/</a> </li>
<li><strong>CITATION</strong> : Loh, Po-Ru, Pier Francesco Palamara, and Alkes L. Price. "Fast and accurate long-range phasing in a UK Biobank cohort." Nature genetics 48.7 (2016): 811-816.<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27270109">27270109</a> </li>
</ul>
<h3 id="eagle2">EAGLE2</h3>
<ul>
<li><strong>NAME</strong> : EAGLE2 </li>
<li><strong>SHORT NAME</strong> : EAGLE2 </li>
<li><strong>DESCRIPTION</strong> : (EAGLE2) </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/Eagle/">https://alkesgroup.broadinstitute.org/Eagle/</a> </li>
<li><strong>CITATION</strong> : Loh, Po-Ru, et al. "Reference-based phasing using the Haplotype Reference Consortium panel." Nature genetics 48.11 (2016): 1443-1448. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27694958">27694958</a> </li>
</ul>
<h3 id="glimpse">GLIMPSE</h3>
<ul>
<li><strong>NAME</strong> : GLIMPSE </li>
<li><strong>SHORT NAME</strong> : GLIMPSE </li>
<li><strong>FULL NAME</strong> : Genotype Likelihoods IMputation and PhaSing mEthod </li>
<li><strong>DESCRIPTION</strong> : GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies. </li>
<li><strong>URL</strong> : <a href="https://odelaneau.github.io/GLIMPSE/">https://odelaneau.github.io/GLIMPSE/</a> </li>
<li><strong>CITATION</strong> : Rubinacci, S., Ribeiro, D. M., Hofmeister, R. J., &amp; Delaneau, O. (2021). Efficient phasing and imputation of low-coverage sequencing data using large reference panels. Nature Genetics, 53(1), 120-126.<br><br>(using GLIMPSE for ancient DNA) Sousa da Mota, B., Rubinacci, S., Cruz Dvalos, D. I., G. Amorim, C. E., Sikora, M., Johannsen, N. N., ... &amp; Delaneau, O. (2023). Imputation of ancient human genomes. Nature Communications, 14(1), 3660. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33414550">33414550</a> </li>
</ul>
<h3 id="impute">IMPUTE</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE </li>
<li><strong>SHORT NAME</strong> : IMPUTE </li>
<li><strong>URL</strong> : <a href="https://jmarchini.org/software/">https://jmarchini.org/software/</a> </li>
<li><strong>CITATION</strong> : Marchini, Jonathan, et al. "A new multipoint method for genome-wide association studies by imputation of genotypes." Nature genetics 39.7 (2007): 906-913.<br><br><br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17572673">17572673</a> </li>
</ul>
<h3 id="impute2">IMPUTE2</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE2 </li>
<li><strong>SHORT NAME</strong> : IMPUTE2 </li>
<li><strong>CITATION</strong> : Howie, Bryan N., Peter Donnelly, and Jonathan Marchini. "A flexible and accurate genotype imputation method for the next generation of genome-wide association studies." PLoS genetics 5.6 (2009): e1000529. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19543373">19543373</a> </li>
</ul>
<h3 id="impute4">IMPUTE4</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE4 </li>
<li><strong>SHORT NAME</strong> : IMPUTE4 </li>
<li><strong>CITATION</strong> : Bycroft, Clare, et al. "The UK Biobank resource with deep phenotyping and genomic data." Nature 562.7726 (2018): 203-209. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30305743">30305743</a> </li>
</ul>
<h3 id="impute5">IMPUTE5</h3>
<ul>
<li><strong>NAME</strong> : IMPUTE5 </li>
<li><strong>SHORT NAME</strong> : IMPUTE5 </li>
<li><strong>CITATION</strong> : Rubinacci, Simone, Olivier Delaneau, and Jonathan Marchini. "Genotype imputation using the positional burrows wheeler transform." PLoS genetics 16.11 (2020): e1009049. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33196638">33196638</a> </li>
</ul>
<h3 id="mach-minimach">MACH / minimach</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach </li>
<li><strong>SHORT NAME</strong> : MACH / minimach </li>
<li><strong>DESCRIPTION</strong> : (MACH) </li>
<li><strong>URL</strong> : [http://csg.sph.umich.edu/abecasis/MaCH/index.html</li>
</ul>
<p>](http://csg.sph.umich.edu/abecasis/MaCH/index.html</p>
<p>) 
 - <strong>CITATION</strong> : Li, Yun, et al. "MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes." Genetic epidemiology 34.8 (2010): 816-834. 
 - <strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21058334">21058334</a> </p>
<h3 id="mach-minimach-pre-phasing">MACH / minimach pre-phasing</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach pre-phasing </li>
<li><strong>SHORT NAME</strong> : MACH / minimach pre-phasing </li>
<li><strong>DESCRIPTION</strong> : (pre-phasing, minimac) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac">https://genome.sph.umich.edu/wiki/Minimac</a> </li>
<li><strong>CITATION</strong> : Howie, Bryan, et al. "Fast and accurate genotype imputation in genome-wide association studies through pre-phasing." Nature genetics 44.8 (2012): 955-959. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22820512">22820512</a> </li>
</ul>
<h3 id="mach-minimach2">MACH / minimach2</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach2 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach2 </li>
<li><strong>DESCRIPTION</strong> : (minimac2) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac2">https://genome.sph.umich.edu/wiki/Minimac2</a> </li>
<li><strong>CITATION</strong> : Fuchsberger, Christian, Gonalo R. Abecasis, and David A. Hinds. "minimac2: faster genotype imputation." Bioinformatics 31.5 (2015): 782-784. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25338720">25338720</a> </li>
</ul>
<h3 id="mach-minimach3">MACH / minimach3</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach3 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach3 </li>
<li><strong>DESCRIPTION</strong> : (minimac3) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac3">https://genome.sph.umich.edu/wiki/Minimac3</a> </li>
<li><strong>CITATION</strong> : Das, Sayantan, et al. "Next-generation genotype imputation service and methods." Nature genetics 48.10 (2016): 1284-1287. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27571263">27571263</a> </li>
</ul>
<h3 id="mach-minimach4">MACH / minimach4</h3>
<ul>
<li><strong>NAME</strong> : MACH / minimach4 </li>
<li><strong>SHORT NAME</strong> : MACH / minimach4 </li>
<li><strong>DESCRIPTION</strong> : (minimac4) </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/Minimac4">https://genome.sph.umich.edu/wiki/Minimac4</a> </li>
</ul>
<h3 id="shapeit1">SHAPEIT1</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT1 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT1) </li>
<li><strong>URL</strong> : <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</a> </li>
<li><strong>CITATION</strong> : (SHAPEIT1) Delaneau, Olivier, Jonathan Marchini, and Jean-Franois Zagury. "A linear complexity phasing method for thousands of genomes." Nature methods 9.2 (2012): 179-181. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22138821">22138821</a> </li>
</ul>
<h3 id="shapeit2">SHAPEIT2</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT2 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT2) </li>
<li><strong>CITATION</strong> : (SHAPEIT2) Delaneau, Olivier, Jean-Francois Zagury, and Jonathan Marchini. "Improved whole-chromosome phasing for disease and population genetic studies." Nature methods 10.1 (2013): 5-6. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23269371">23269371</a> </li>
</ul>
<h3 id="shapeit3">SHAPEIT3</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT3 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT3) </li>
<li><strong>URL</strong> : <a href="https://jmarchini.org/shapeit3/">https://jmarchini.org/shapeit3/</a> </li>
<li><strong>CITATION</strong> : <br><br>(SHAPEIT3) O'Connell, Jared, et al. "Haplotype estimation for biobank-scale data sets." Nature genetics 48.7 (2016): 817-820. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27270105">27270105</a> </li>
</ul>
<h3 id="shapeit4">SHAPEIT4</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT4 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT4) </li>
<li><strong>URL</strong> : <a href="https://odelaneau.github.io/shapeit4/">https://odelaneau.github.io/shapeit4/</a> </li>
<li><strong>CITATION</strong> : (SHAPEIT4) Delaneau, Olivier, et al. "Accurate, scalable and integrative haplotype estimation." Nature communications 10.1 (2019): 1-10. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31780650">31780650</a> </li>
</ul>
<h3 id="shapeit5">SHAPEIT5</h3>
<ul>
<li><strong>NAME</strong> : SHAPEIT5 </li>
<li><strong>DESCRIPTION</strong> : (SHAPEIT5) </li>
<li><strong>CITATION</strong> : (SHAPEIT5) Hofmeister, R. J., Ribeiro, D. M., Rubinacci, S., &amp; Delaneau, O. (2023). Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. Nature Genetics, 55(7), 1243-1249.<br><br>(SHAPEIT5) Hofmeister, R. J., Ribeiro, D. M., Rubinacci, S., &amp; Delaneau, O. (2022). Accurate rare variant phasing of whole-genome and whole-exome sequencing data in the UK Biobank. bioRxiv, 2022-10. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37386248">37386248</a> </li>
</ul>
<h3 id="fastphase">fastPHASE</h3>
<ul>
<li><strong>NAME</strong> : fastPHASE </li>
<li><strong>SHORT NAME</strong> : fastPHASE </li>
<li><strong>URL</strong> : <a href="http://scheet.org/software.html">http://scheet.org/software.html</a> </li>
<li><strong>CITATION</strong> : Scheet, Paul, and Matthew Stephens. "A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase." The American Journal of Human Genetics 78.4 (2006): 629-644. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/16532393">16532393</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-das">Review-Das</h3>
<ul>
<li><strong>NAME</strong> : Review-Das </li>
<li><strong>CITATION</strong> : Das, Sayantan, Gonalo R. Abecasis, and Brian L. Browning. "Genotype imputation from large reference panels." Annu Rev Genomics Hum Genet 19.1 (2018): 73-96. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29799802">29799802</a> </li>
</ul>
<h3 id="review-li">Review-Li</h3>
<ul>
<li><strong>NAME</strong> : Review-Li </li>
<li><strong>CITATION</strong> : Li, Yun, et al. "Genotype imputation." Annual review of genomics and human genetics 10 (2009): 387. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/19715440">19715440</a> </li>
</ul>
<h3 id="review-marchini">Review-Marchini</h3>
<ul>
<li><strong>NAME</strong> : Review-Marchini </li>
<li><strong>CITATION</strong> : Marchini, Jonathan, and Bryan Howie. "Genotype imputation for genome-wide association studies." Nature Reviews Genetics 11.7 (2010): 499-511. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20517342">20517342</a> </li>
</ul>
<h2 id="structural-variants-imputation-panel">Structural variants imputation panel</h2>
<h3 id="1kg-sv-imputation-panel">1KG SV imputation panel</h3>
<ul>
<li><strong>NAME</strong> : 1KG SV imputation panel </li>
<li><strong>SHORT NAME</strong> : 1KG SV </li>
<li><strong>KEYWORDS</strong> : structural variants, long-read </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.12.20.23300308 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>CITATION</strong> : Noyvert, B., Erzurumluoglu, A. M., Drichel, D., Omland, S., Andlauer, T. F., Mueller, S., ... &amp; Ding, Z. (2023). Imputation of structural variants using a multi-ancestry long-read sequencing panel enables identification of disease associations. medRxiv, 2023-12. </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>