# QTL
## QTLbase
- URL: http://www.mulinlab.org/qtlbase
- CITATION: Zheng, Z., Huang, D., Wang, J., Zhao, K., Zhou, Y., Guo, Z., ... & Li, M. J. (2020). QTLbase: an integrative resource for quantitative trait loci across multiple human molecular phenotypes. Nucleic acids research, 48(D1), D983-D991.

------------------------------------------------------------------
# Transcriptional regulation
## eQTL - gene expression 

### GTEx
- URL : https://gtexportal.org/home/datasets

### v9 Long Read RNASeq data from 88 GTEx tissue and cell line samples
- CITATION: Glinos, D. A., Garborcauskas, G., Hoffman, P., Ehsan, N., Jiang, L., Gokden, A., ... & Cummings, B. B. (2022). Transcriptome variation in human tissues revealed by long-read sequencing. Nature, 608(7922), 353-359.

### v8 Single-Tissue e/sQTL Data
- CITATION: GTEx Consortium. (2020). The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science, 369(6509), 1318-1330. 

### eQTLGen Consortium
- URL : https://www.eqtlgen.org/
- CITATION: Võsa, U., Claringbould, A., Westra, H. J., Bonder, M. J., Deelen, P., Zeng, B., ... & Franke, L. (2021). Large-scale cis-and trans-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression. Nature genetics, 53(9), 1300-1310.

### sc-eQTLGen 
- URL: [https://eqtlgen.org/sc/](https://eqtlgen.org/sc/)
- CITATION: van der Wijst, Monique GP, et al. "Science Forum: The single-cell eQTLGen consortium." elife 9 (2020): e52155.

### PsychENCODE consortium
- URL : https://www.nimhgenetics.org/resources/psychencode
- CITATION: Akbarian, S., Liu, C., Knowles, J. et al. The PsychENCODE project. Nat Neurosci 18, 1707–1712 (2015). https://doi.org/10.1038/nn.4156

## sQTL - alternative splicing
### GTEx
### v8 Single-Tissue e/sQTL Data
- CITATION: GTEx Consortium. (2020). The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science, 369(6509), 1318-1330. 

### BrainMeta sQTL & eQTL portal
- DESCRIPTION: We developed a method, THISTLE, which uses individual-level genotype and RNA-seq data or summary-level isoform-eQTL data for splicing QTL (sQTL) mapping (Qi et al. 2022). We applied THISTLE, in combination with a complementary sQTL mapping strategy, for sQTL mapping using RNA-seq data of 2,865 brain cortex samples from 2,443 unrelated individuals of European ancestry with genome-wide SNP data. We also applied QTLtools for eQTL mapping using the same dataset. See “Tutorial” for the instructions to use this online portal. See “BESD files” for details to download the full sQTL or eQTL summary statistics in SMR binary (BESD) format.
- URL :https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/
------------------------------------------------------------------
# Post-transcriptional regulation
## reQTL - RNA editing
## cerQTL - competing endogenous RNA expression
------------------------------------------------------------------
# Translational regulation
## pQTL - protein expression
## riboQTL - ribosome occupancy 
------------------------------------------------------------------
# Post-translational regulation
## metaQTL - cell metabolism
------------------------------------------------------------------
# Epigenetic regulation
## mQTL - DNA methylation
### iMETHYL
## caQTL - chromatin accessibility
## hQTL - histone modification
