
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_GxE_interactions/">
      
      
        <link rel="next" href="../Tools_Heritability_and_genetic_correlation/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>HLA - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              HLA
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Biobanks/" class="md-tabs__link">
          
  
  Biobanks

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../References/" class="md-tabs__link">
          
  
  References

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Single_Cell/" class="md-tabs__link">
          
  
  Single_Cell

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Biobanks
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AFRICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AFRICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AMERICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AMERICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_ASIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ASIA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_EUROPE/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    EUROPE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_OCIENIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    OCIENIA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    References
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            References
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    References
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Epigenetics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Gene/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Genome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Phenotype/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Variant/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_scRNA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    scRNA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Single_Cell
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Framework/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Framework
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Harmonization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Harmonization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Network/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Network
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_QC/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    QC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Trajectory/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Trajectory
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_General/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    General
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Imaging/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Metabolomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Proteomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" checked>
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Admixture/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6_4" >
        
          
          <label class="md-nav__link" for="__nav_6_4" id="__nav_6_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_6_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6_4">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Functional_prediction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Functional_prediction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Genealogy/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    HLA
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-analyses-tutorial" class="md-nav__link">
    <span class="md-ellipsis">
      HLA analyses tutorial
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA analyses tutorial">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#sakaue" class="md-nav__link">
    <span class="md-ellipsis">
      Sakaue
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#four-digit-multi-ethnic-hla-v1-2021" class="md-nav__link">
    <span class="md-ellipsis">
      Four-digit Multi-ethnic HLA v1 (2021)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#four-digit-multi-ethnic-hla-v2-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Four-digit Multi-ethnic HLA v2 (2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#han-mhc" class="md-nav__link">
    <span class="md-ellipsis">
      Han-MHC
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-pipeline" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation pipeline
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation pipeline">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#hla-tapas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-TAPAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cookhla" class="md-nav__link">
    <span class="md-ellipsis">
      CookHLA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deephla" class="md-nav__link">
    <span class="md-ellipsis">
      DEEP*HLA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hibag" class="md-nav__link">
    <span class="md-ellipsis">
      HIBAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hlarimnt" class="md-nav__link">
    <span class="md-ellipsis">
      HLARIMNT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snp2hla" class="md-nav__link">
    <span class="md-ellipsis">
      SNP2HLA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Phylogenetic_tree/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Selection/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-analyses-tutorial" class="md-nav__link">
    <span class="md-ellipsis">
      HLA analyses tutorial
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA analyses tutorial">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#sakaue" class="md-nav__link">
    <span class="md-ellipsis">
      Sakaue
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-panel" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation panel
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation panel">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#four-digit-multi-ethnic-hla-v1-2021" class="md-nav__link">
    <span class="md-ellipsis">
      Four-digit Multi-ethnic HLA v1 (2021)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#four-digit-multi-ethnic-hla-v2-2022" class="md-nav__link">
    <span class="md-ellipsis">
      Four-digit Multi-ethnic HLA v2 (2022)
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#han-mhc" class="md-nav__link">
    <span class="md-ellipsis">
      Han-MHC
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-pipeline" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation pipeline
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation pipeline">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#hla-tapas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-TAPAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-imputation-tool" class="md-nav__link">
    <span class="md-ellipsis">
      HLA imputation tool
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA imputation tool">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cookhla" class="md-nav__link">
    <span class="md-ellipsis">
      CookHLA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deephla" class="md-nav__link">
    <span class="md-ellipsis">
      DEEP*HLA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hibag" class="md-nav__link">
    <span class="md-ellipsis">
      HIBAG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hlarimnt" class="md-nav__link">
    <span class="md-ellipsis">
      HLARIMNT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snp2hla" class="md-nav__link">
    <span class="md-ellipsis">
      SNP2HLA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>HLA</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#sakaue">Sakaue</a></td>
<td style="text-align: left;">HLA analyses tutorial</td>
<td style="text-align: left;">Sakaue S, Gurajala S, Curtis M, Luo Y, ...&amp;, Raychaudhuri S. (2023) Tutorial: a statistical genetics guide to identifying HLA alleles driving complex disease Nat. Protoc., 18 (9) 2625-2641. doi:10.1038/s41596-023-00853-4. PMID 37495751</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#four-digit-multi-ethnic-hla-v1-2021">Four-digit Multi-ethnic HLA v1 (2021)</a></td>
<td style="text-align: left;">HLA imputation panel</td>
<td style="text-align: left;">Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#four-digit-multi-ethnic-hla-v2-2022">Four-digit Multi-ethnic HLA v2 (2022)</a></td>
<td style="text-align: left;">HLA imputation panel</td>
<td style="text-align: left;">Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#han-mhc">Han-MHC</a></td>
<td style="text-align: left;">HLA imputation panel</td>
<td style="text-align: left;">Zhou F, Cao H, Zuo X, Zhang T, ...&amp;, Zhang X. (2016) Deep sequencing of the MHC region in the Chinese population contributes to studies of complex disease Nat. Genet., 48 (7) 740-746. doi:10.1038/ng.3576. PMID 27213287</td>
<td style="text-align: left;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hla-tapas">HLA-TAPAS</a></td>
<td style="text-align: left;">HLA imputation pipeline</td>
<td style="text-align: left;">Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cookhla">CookHLA</a></td>
<td style="text-align: left;">HLA imputation tool</td>
<td style="text-align: left;">Cook S, Choi W, Lim H, Luo Y, ...&amp;, Han B. (2021) Accurate imputation of human leukocyte antigens with CookHLA Nat. Commun., 12 (1) 1264. doi:10.1038/s41467-021-21541-5. PMID 33627654</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#deephla">DEEP*HLA</a></td>
<td style="text-align: left;">HLA imputation tool</td>
<td style="text-align: left;">Naito T, Suzuki K, Hirata J, Kamatani Y, ...&amp;, Okada Y. (2021) A deep learning method for HLA imputation and trans-ethnic MHC fine-mapping of type 1 diabetes Nat. Commun., 12 (1) 1639. doi:10.1038/s41467-021-21975-x. PMID 33712626</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hibag">HIBAG</a></td>
<td style="text-align: left;">HLA imputation tool</td>
<td style="text-align: left;">Zheng X, Shen J, Cox C, Wakefield JC, ...&amp;, Weir BS. (2014) HIBAG--HLA genotype imputation with attribute bagging Pharmacogenomics J., 14 (2) 192-200. doi:10.1038/tpj.2013.18. PMID 23712092</td>
<td style="text-align: left;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hlarimnt">HLARIMNT</a></td>
<td style="text-align: left;">HLA imputation tool</td>
<td style="text-align: left;">Tanaka, K., Kato, K., Nonaka, N., &amp; Seita, J. (2022). Efficient HLA imputation from sequential SNPs data by Transformer. arXiv preprint arXiv:2211.06430.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#snp2hla">SNP2HLA</a></td>
<td style="text-align: left;">HLA imputation tool</td>
<td style="text-align: left;">Jia X, Han B, Onengut-Gumuscu S, Chen WM, ...&amp;, de Bakker PI. (2013) Imputing amino acid polymorphisms in human leukocyte antigens PLoS One, 8 (6) e64683. doi:10.1371/journal.pone.0064683. PMID 23762245</td>
<td style="text-align: left;">2013</td>
</tr>
</tbody>
</table>
<h2 id="hla-analyses-tutorial">HLA analyses tutorial</h2>
<h3 id="sakaue">Sakaue</h3>
<ul>
<li><strong>NAME</strong> : Sakaue </li>
<li><strong>URL</strong> : <a href="https://github.com/immunogenomics/HLA_analyses_tutorial">https://github.com/immunogenomics/HLA_analyses_tutorial</a> </li>
<li><strong>KEYWORDS</strong> : HLA analyses tutorial </li>
<li><strong>TITLE</strong> : Tutorial: a statistical genetics guide to identifying HLA alleles driving complex disease </li>
<li><strong>DOI</strong> : 10.1038/s41596-023-00853-4 </li>
<li><strong>ABSTRACT</strong> : The human leukocyte antigen (HLA) locus is associated with more complex diseases than any other locus in the human genome. In many diseases, HLA explains more heritability than all other known loci combined. In silico HLA imputation methods enable rapid and accurate estimation of HLA alleles in the millions of individuals that are already genotyped on microarrays. HLA imputation has been used to define causal variation in autoimmune diseases, such as type I diabetes, and in human immunodeficiency virus infection control. However, there are few guidelines on performing HLA imputation, association testing, and fine mapping. Here, we present a comprehensive tutorial to impute HLA alleles from genotype data. We provide detailed guidance on performing standard quality control measures for input genotyping data and describe options to impute HLA alleles and amino acids either locally or using the web-based Michigan Imputation Server, which hosts a multi-ancestry HLA imputation reference panel. We also offer best practice recommendations to conduct association tests to define the alleles, amino acids, and haplotypes that affect human traits. Along with the pipeline, we provide a step-by-step online guide with scripts and available software ( https://github.com/immunogenomics/HLA_analyses_tutorial ). This tutorial will be broadly applicable to large-scale genotyping data and will contribute to defining the role of HLA in human diseases across global populations. </li>
<li><strong>CITATION</strong> : Sakaue S, Gurajala S, Curtis M, Luo Y, ...&amp;, Raychaudhuri S. (2023) Tutorial: a statistical genetics guide to identifying HLA alleles driving complex disease Nat. Protoc., 18 (9) 2625-2641. doi:10.1038/s41596-023-00853-4. PMID 37495751 </li>
<li><strong>JOURNAL_INFO</strong> : Nature protocols ; Nat. Protoc. ; 2023 ; 18 ; 9 ; 2625-2641 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37495751">37495751</a> </li>
</ul>
<h2 id="hla-imputation-panel">HLA imputation panel</h2>
<h3 id="four-digit-multi-ethnic-hla-v1-2021">Four-digit Multi-ethnic HLA v1 (2021)</h3>
<ul>
<li><strong>NAME</strong> : Four-digit Multi-ethnic HLA v1 (2021) </li>
<li><strong>SHORT NAME</strong> : Four-digit Multi-ethnic HLA v1 (2021) </li>
<li><strong>DESCRIPTION</strong> : Available on Michigan imputation server </li>
<li><strong>URL</strong> : <a href="https://github.com/immunogenomics/HLA-TAPAS/">https://github.com/immunogenomics/HLA-TAPAS/</a> </li>
<li><strong>TITLE</strong> : A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00935-7 </li>
<li><strong>ABSTRACT</strong> : Fine-mapping to plausible causal variation may be more effective in multi-ancestry cohorts, particularly in the MHC, which has population-specific structure. To enable such studies, we constructed a large (n = 21,546) HLA reference panel spanning five global populations based on whole-genome sequences. Despite population-specific long-range haplotypes, we demonstrated accurate imputation at G-group resolution (94.2%, 93.7%, 97.8% and 93.7% in admixed African (AA), East Asian (EAS), European (EUR) and Latino (LAT) populations). Applying HLA imputation to genome-wide association study data for HIV-1 viral load in three populations (EUR, AA and LAT), we obviated effects of previously reported associations from population-specific HIV studies and discovered a novel association at position 156 in HLA-B. We pinpointed the MHC association to three amino acid positions (97, 67 and 156) marking three consecutive pockets (C, B and D) within the HLA-B peptide-binding groove, explaining 12.9% of trait variance. </li>
<li><strong>CITATION</strong> : Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 10 ; 1504-1516 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34611364">34611364</a> </li>
</ul>
<h3 id="four-digit-multi-ethnic-hla-v2-2022">Four-digit Multi-ethnic HLA v2 (2022)</h3>
<ul>
<li><strong>NAME</strong> : Four-digit Multi-ethnic HLA v2 (2022) </li>
<li><strong>SHORT NAME</strong> : Four-digit Multi-ethnic HLA v2 (2022) </li>
<li><strong>DESCRIPTION</strong> : Available on Michigan imputation server </li>
<li><strong>URL</strong> : <a href="https://github.com/immunogenomics/HLA-TAPAS/">https://github.com/immunogenomics/HLA-TAPAS/</a> </li>
<li><strong>TITLE</strong> : A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00935-7 </li>
<li><strong>ABSTRACT</strong> : Fine-mapping to plausible causal variation may be more effective in multi-ancestry cohorts, particularly in the MHC, which has population-specific structure. To enable such studies, we constructed a large (n = 21,546) HLA reference panel spanning five global populations based on whole-genome sequences. Despite population-specific long-range haplotypes, we demonstrated accurate imputation at G-group resolution (94.2%, 93.7%, 97.8% and 93.7% in admixed African (AA), East Asian (EAS), European (EUR) and Latino (LAT) populations). Applying HLA imputation to genome-wide association study data for HIV-1 viral load in three populations (EUR, AA and LAT), we obviated effects of previously reported associations from population-specific HIV studies and discovered a novel association at position 156 in HLA-B. We pinpointed the MHC association to three amino acid positions (97, 67 and 156) marking three consecutive pockets (C, B and D) within the HLA-B peptide-binding groove, explaining 12.9% of trait variance. </li>
<li><strong>CITATION</strong> : Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 10 ; 1504-1516 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34611364">34611364</a> </li>
</ul>
<h3 id="han-mhc">Han-MHC</h3>
<ul>
<li><strong>NAME</strong> : Han-MHC </li>
<li><strong>SHORT NAME</strong> : Han-MHC </li>
<li><strong>URL</strong> : <a href="http://gigadb.org/dataset/100156">http://gigadb.org/dataset/100156</a> </li>
<li><strong>TITLE</strong> : Deep sequencing of the MHC region in the Chinese population contributes to studies of complex disease </li>
<li><strong>DOI</strong> : 10.1038/ng.3576 </li>
<li><strong>ABSTRACT</strong> : The human major histocompatibility complex (MHC) region has been shown to be associated with numerous diseases. However, it remains a challenge to pinpoint the causal variants for these associations because of the extreme complexity of the region. We thus sequenced the entire 5-Mb MHC region in 20,635 individuals of Han Chinese ancestry (10,689 controls and 9,946 patients with psoriasis) and constructed a Han-MHC database that includes both variants and HLA gene typing results of high accuracy. We further identified multiple independent new susceptibility loci in HLA-C, HLA-B, HLA-DPB1 and BTNL2 and an intergenic variant, rs118179173, associated with psoriasis and confirmed the well-established risk allele HLA-C*06:02. We anticipate that our Han-MHC reference panel built by deep sequencing of a large number of samples will serve as a useful tool for investigating the role of the MHC region in a variety of diseases and thus advance understanding of the pathogenesis of these disorders. </li>
<li><strong>CITATION</strong> : Zhou F, Cao H, Zuo X, Zhang T, ...&amp;, Zhang X. (2016) Deep sequencing of the MHC region in the Chinese population contributes to studies of complex disease Nat. Genet., 48 (7) 740-746. doi:10.1038/ng.3576. PMID 27213287 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2016 ; 48 ; 7 ; 740-746 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27213287">27213287</a> </li>
</ul>
<h2 id="hla-imputation-pipeline">HLA imputation pipeline</h2>
<h3 id="hla-tapas">HLA-TAPAS</h3>
<ul>
<li><strong>NAME</strong> : HLA-TAPAS </li>
<li><strong>SHORT NAME</strong> : HLA-TAPAS </li>
<li><strong>FULL NAME</strong> : HLA-Typing At Protein for Association Studie </li>
<li><strong>DESCRIPTION</strong> : HLA-TAPAS (HLA-Typing At Protein for Association Studies) is an HLA-focused pipeline that can handle HLA reference panel construction (MakeReference), HLA imputation (SNP2HLA), and HLA association (HLAassoc). </li>
<li><strong>URL</strong> : <a href="https://github.com/immunogenomics/HLA-TAPAS">https://github.com/immunogenomics/HLA-TAPAS</a> </li>
<li><strong>KEYWORDS</strong> : HLA pipeline </li>
<li><strong>TITLE</strong> : A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00935-7 </li>
<li><strong>ABSTRACT</strong> : Fine-mapping to plausible causal variation may be more effective in multi-ancestry cohorts, particularly in the MHC, which has population-specific structure. To enable such studies, we constructed a large (n = 21,546) HLA reference panel spanning five global populations based on whole-genome sequences. Despite population-specific long-range haplotypes, we demonstrated accurate imputation at G-group resolution (94.2%, 93.7%, 97.8% and 93.7% in admixed African (AA), East Asian (EAS), European (EUR) and Latino (LAT) populations). Applying HLA imputation to genome-wide association study data for HIV-1 viral load in three populations (EUR, AA and LAT), we obviated effects of previously reported associations from population-specific HIV studies and discovered a novel association at position 156 in HLA-B. We pinpointed the MHC association to three amino acid positions (97, 67 and 156) marking three consecutive pockets (C, B and D) within the HLA-B peptide-binding groove, explaining 12.9% of trait variance. </li>
<li><strong>CITATION</strong> : Luo Y, Kanai M, Choi W, Li X, ...&amp;, Raychaudhuri S. (2021) A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response Nat. Genet., 53 (10) 1504-1516. doi:10.1038/s41588-021-00935-7. PMID 34611364 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 10 ; 1504-1516 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34611364">34611364</a> </li>
</ul>
<h2 id="hla-imputation-tool">HLA imputation tool</h2>
<h3 id="cookhla">CookHLA</h3>
<ul>
<li><strong>NAME</strong> : CookHLA </li>
<li><strong>URL</strong> : <a href="https://github.com/WansonChoi/CookHLA">https://github.com/WansonChoi/CookHLA</a> </li>
<li><strong>TITLE</strong> : Accurate imputation of human leukocyte antigens with CookHLA </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-21541-5 </li>
<li><strong>ABSTRACT</strong> : The recent development of imputation methods enabled the prediction of human leukocyte antigen (HLA) alleles from intergenic SNP data, allowing studies to fine-map HLA for immune phenotypes. Here we report an accurate HLA imputation method, CookHLA, which has superior imputation accuracy compared to previous methods. CookHLA differs from other approaches in that it locally embeds prediction markers into highly polymorphic exons to account for exonic variability, and in that it adaptively learns the genetic map within MHC from the data to facilitate imputation. Our benchmarking with real datasets shows that our method achieves high imputation accuracy in a wide range of scenarios, including situations where the reference panel is small or ethnically unmatched. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Cook S, Choi W, Lim H, Luo Y, ...&amp;, Han B. (2021) Accurate imputation of human leukocyte antigens with CookHLA Nat. Commun., 12 (1) 1264. doi:10.1038/s41467-021-21541-5. PMID 33627654 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 1264 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33627654">33627654</a> </li>
</ul>
<h3 id="deephla">DEEP*HLA</h3>
<ul>
<li><strong>NAME</strong> : DEEP*HLA </li>
<li><strong>URL</strong> : <a href="https://github.com/tatsuhikonaito/DEEP-HLA">https://github.com/tatsuhikonaito/DEEP-HLA</a> </li>
<li><strong>TITLE</strong> : A deep learning method for HLA imputation and trans-ethnic MHC fine-mapping of type 1 diabetes </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-21975-x </li>
<li><strong>ABSTRACT</strong> : Conventional human leukocyte antigen (HLA) imputation methods drop their performance for infrequent alleles, which is one of the factors that reduce the reliability of trans-ethnic major histocompatibility complex (MHC) fine-mapping due to inter-ethnic heterogeneity in allele frequency spectra. We develop DEEP<em>HLA, a deep learning method for imputing HLA genotypes. Through validation using the Japanese and European HLA reference panels (n = 1,118 and 5,122), DEEP</em>HLA achieves the highest accuracies with significant superiority for low-frequency and rare alleles. DEEP<em>HLA is less dependent on distance-dependent linkage disequilibrium decay of the target alleles and might capture the complicated region-wide information. We apply DEEP</em>HLA to type 1 diabetes GWAS data from BioBank Japan (n = 62,387) and UK Biobank (n = 354,459), and successfully disentangle independently associated class I and II HLA variants with shared risk among diverse populations (the top signal at amino acid position 71 of HLA-DRβ1; P = 7.5 × 10-120). Our study illustrates the value of deep learning in genotype imputation and trans-ethnic MHC fine-mapping. </li>
<li><strong>CITATION</strong> : Naito T, Suzuki K, Hirata J, Kamatani Y, ...&amp;, Okada Y. (2021) A deep learning method for HLA imputation and trans-ethnic MHC fine-mapping of type 1 diabetes Nat. Commun., 12 (1) 1639. doi:10.1038/s41467-021-21975-x. PMID 33712626 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 1639 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33712626">33712626</a> </li>
</ul>
<h3 id="hibag">HIBAG</h3>
<ul>
<li><strong>NAME</strong> : HIBAG </li>
<li><strong>URL</strong> : <a href="https://github.com/zhengxwen/HIBAG">https://github.com/zhengxwen/HIBAG</a> </li>
<li><strong>TITLE</strong> : HIBAG--HLA genotype imputation with attribute bagging </li>
<li><strong>DOI</strong> : 10.1038/tpj.2013.18 </li>
<li><strong>ABSTRACT</strong> : Genotyping of classical human leukocyte antigen (HLA) alleles is an essential tool in the analysis of diseases and adverse drug reactions with associations mapping to the major histocompatibility complex (MHC). However, deriving high-resolution HLA types subsequent to whole-genome single-nucleotide polymorphism (SNP) typing or sequencing is often cost prohibitive for large samples. An alternative approach takes advantage of the extended haplotype structure within the MHC to predict HLA alleles using dense SNP genotypes, such as those available from genome-wide SNP panels. Current methods for HLA imputation are difficult to apply or may require the user to have access to large training data sets with SNP and HLA types. We propose HIBAG, HLA Imputation using attribute BAGging, that makes predictions by averaging HLA-type posterior probabilities over an ensemble of classifiers built on bootstrap samples. We assess the performance of HIBAG using our study data (n=2668 subjects of European ancestry) as a training set and HLA data from the British 1958 birth cohort study (n≈1000 subjects) as independent validation samples. Prediction accuracies for HLA-A, B, C, DRB1 and DQB1 range from 92.2% to 98.1% using a set of SNP markers common to the Illumina 1M Duo, OmniQuad, OmniExpress, 660K and 550K platforms. HIBAG performed well compared with the other two leading methods, HLA*IMP and BEAGLE. This method is implemented in a freely available HIBAG R package that includes pre-fit classifiers for European, Asian, Hispanic and African ancestries, providing a readily available imputation approach without the need to have access to large training data sets. </li>
<li><strong>CITATION</strong> : Zheng X, Shen J, Cox C, Wakefield JC, ...&amp;, Weir BS. (2014) HIBAG--HLA genotype imputation with attribute bagging Pharmacogenomics J., 14 (2) 192-200. doi:10.1038/tpj.2013.18. PMID 23712092 </li>
<li><strong>JOURNAL_INFO</strong> : The pharmacogenomics journal ; Pharmacogenomics J. ; 2014 ; 14 ; 2 ; 192-200 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23712092">23712092</a> </li>
</ul>
<h3 id="hlarimnt">HLARIMNT</h3>
<ul>
<li><strong>NAME</strong> : HLARIMNT </li>
<li><strong>SHORT NAME</strong> : HLARIMNT </li>
<li><strong>FULL NAME</strong> : HLA Reliable IMputatioN by Transformer </li>
<li><strong>URL</strong> : <a href="https://github.com/seitalab/HLARIMNT">https://github.com/seitalab/HLARIMNT</a> </li>
<li><strong>KEYWORDS</strong> : HLA, imputation </li>
<li><strong>CITATION</strong> : Tanaka, K., Kato, K., Nonaka, N., &amp; Seita, J. (2022). Efficient HLA imputation from sequential SNPs data by Transformer. arXiv preprint arXiv:2211.06430. </li>
</ul>
<h3 id="snp2hla">SNP2HLA</h3>
<ul>
<li><strong>NAME</strong> : SNP2HLA </li>
<li><strong>URL</strong> : <a href="http://software.broadinstitute.org/mpg/snp2hla/">http://software.broadinstitute.org/mpg/snp2hla/</a> </li>
<li><strong>TITLE</strong> : Imputing amino acid polymorphisms in human leukocyte antigens </li>
<li><strong>DOI</strong> : 10.1371/journal.pone.0064683 </li>
<li><strong>ABSTRACT</strong> : DNA sequence variation within human leukocyte antigen (HLA) genes mediate susceptibility to a wide range of human diseases. The complex genetic structure of the major histocompatibility complex (MHC) makes it difficult, however, to collect genotyping data in large cohorts. Long-range linkage disequilibrium between HLA loci and SNP markers across the major histocompatibility complex (MHC) region offers an alternative approach through imputation to interrogate HLA variation in existing GWAS data sets. Here we describe a computational strategy, SNP2HLA, to impute classical alleles and amino acid polymorphisms at class I (HLA-A, -B, -C) and class II (-DPA1, -DPB1, -DQA1, -DQB1, and -DRB1) loci. To characterize performance of SNP2HLA, we constructed two European ancestry reference panels, one based on data collected in HapMap-CEPH pedigrees (90 individuals) and another based on data collected by the Type 1 Diabetes Genetics Consortium (T1DGC, 5,225 individuals). We imputed HLA alleles in an independent data set from the British 1958 Birth Cohort (N = 918) with gold standard four-digit HLA types and SNPs genotyped using the Affymetrix GeneChip 500 K and Illumina Immunochip microarrays. We demonstrate that the sample size of the reference panel, rather than SNP density of the genotyping platform, is critical to achieve high imputation accuracy. Using the larger T1DGC reference panel, the average accuracy at four-digit resolution is 94.7% using the low-density Affymetrix GeneChip 500 K, and 96.7% using the high-density Illumina Immunochip. For amino acid polymorphisms within HLA genes, we achieve 98.6% and 99.3% accuracy using the Affymetrix GeneChip 500 K and Illumina Immunochip, respectively. Finally, we demonstrate how imputation and association testing at amino acid resolution can facilitate fine-mapping of primary MHC association signals, giving a specific example from type 1 diabetes. </li>
<li><strong>CITATION</strong> : Jia X, Han B, Onengut-Gumuscu S, Chen WM, ...&amp;, de Bakker PI. (2013) Imputing amino acid polymorphisms in human leukocyte antigens PLoS One, 8 (6) e64683. doi:10.1371/journal.pone.0064683. PMID 23762245 </li>
<li><strong>JOURNAL_INFO</strong> : PloS one ; PLoS One ; 2013 ; 8 ; 6 ; e64683 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23762245">23762245</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>