
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_HLA_README/">
      
      
        <link rel="next" href="../Tools_Imputation_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Heritability_and_genetic_correlation - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Heritability_and_genetic_correlation
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Genome_Genome_README/" class="md-tabs__link">
          
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#s-ldxr" class="md-nav__link">
    <span class="md-ellipsis">
      S-LDXR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cross-trait-ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      cross-trait LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popcorn" class="md-nav__link">
    <span class="md-ellipsis">
      popcorn
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#heritability" class="md-nav__link">
    <span class="md-ellipsis">
      Heritability
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Heritability">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesrr-rc" class="md-nav__link">
    <span class="md-ellipsis">
      BayesRR-RC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-binary" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Binary
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-bivariate" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Bivariate
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-ldms" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-LDMS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-partition" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Partition
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-quantitative" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Quantitative
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdl" class="md-nav__link">
    <span class="md-ellipsis">
      HDL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumher" class="md-nav__link">
    <span class="md-ellipsis">
      SumHer
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetraher" class="md-nav__link">
    <span class="md-ellipsis">
      TetraHer
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#local-heritabilitygenetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Local heritability/genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Local heritability/genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gnova" class="md-nav__link">
    <span class="md-ellipsis">
      GNOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heels" class="md-nav__link">
    <span class="md-ellipsis">
      HEELS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hess" class="md-nav__link">
    <span class="md-ellipsis">
      HESS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lava" class="md-nav__link">
    <span class="md-ellipsis">
      LAVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supergnova" class="md-nav__link">
    <span class="md-ellipsis">
      SUPERGNOVA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Genealogy_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#s-ldxr" class="md-nav__link">
    <span class="md-ellipsis">
      S-LDXR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cross-trait-ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      cross-trait LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popcorn" class="md-nav__link">
    <span class="md-ellipsis">
      popcorn
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#heritability" class="md-nav__link">
    <span class="md-ellipsis">
      Heritability
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Heritability">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesrr-rc" class="md-nav__link">
    <span class="md-ellipsis">
      BayesRR-RC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-binary" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Binary
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-bivariate" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Bivariate
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-ldms" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-LDMS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-partition" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Partition
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-quantitative" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Quantitative
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdl" class="md-nav__link">
    <span class="md-ellipsis">
      HDL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumher" class="md-nav__link">
    <span class="md-ellipsis">
      SumHer
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetraher" class="md-nav__link">
    <span class="md-ellipsis">
      TetraHer
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#local-heritabilitygenetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Local heritability/genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Local heritability/genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gnova" class="md-nav__link">
    <span class="md-ellipsis">
      GNOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heels" class="md-nav__link">
    <span class="md-ellipsis">
      HEELS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hess" class="md-nav__link">
    <span class="md-ellipsis">
      HESS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lava" class="md-nav__link">
    <span class="md-ellipsis">
      LAVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supergnova" class="md-nav__link">
    <span class="md-ellipsis">
      SUPERGNOVA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Heritability_and_genetic_correlation</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: right;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#s-ldxr">S-LDXR</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Shi H, Gazal S, Kanai M, Koch EM, ...&amp;, Price AL. (2021) Population-specific causal disease effect sizes in functionally important regions impacted by selection Nat. Commun., 12 (1) 1098. doi:10.1038/s41467-021-21286-1. PMID 33597505</td>
<td style="text-align: right;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cross-trait-ldsc">cross-trait LDSC</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Bulik-Sullivan B, Finucane HK, Anttila V, Gusev A, ...&amp;, Neale BM. (2015) An atlas of genetic correlations across human diseases and traits Nat. Genet., 47 (11) 1236-1241. doi:10.1038/ng.3406. PMID 26414676</td>
<td style="text-align: right;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#popcorn">popcorn</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Brown BC, Asian Genetic Epidemiology Network Type 2 Diabetes Consortium, Ye CJ, Price AL, ...&amp;, Zaitlen N. (2016) Transethnic Genetic-Correlation Estimates from Summary Statistics Am. J. Hum. Genet., 99 (1) 76-88. doi:10.1016/j.ajhg.2016.05.001. PMID 27321947</td>
<td style="text-align: right;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bayesrr-rc">BayesRR-RC</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Patxot M, Banos DT, Kousathanas A, Orliac EJ, ...&amp;, Robinson MR. (2021) Probabilistic inference of the genetic architecture underlying functional enrichment of complex traits Nat. Commun., 12 (1) 6972. doi:10.1038/s41467-021-27258-9. PMID 34848700</td>
<td style="text-align: right;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-binary">GCTA-GREML-Binary</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Lee SH, Wray NR, Goddard ME, Visscher PM. (2011) Estimating missing heritability for disease from genome-wide association studies Am. J. Hum. Genet., 88 (3) 294-305. doi:10.1016/j.ajhg.2011.02.002. PMID 21376301</td>
<td style="text-align: right;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-bivariate">GCTA-GREML-Bivariate</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Lee SH, Yang J, Goddard ME, Visscher PM, ...&amp;, Wray NR. (2012) Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood Bioinformatics, 28 (19) 2540-2542. doi:10.1093/bioinformatics/bts474. PMID 22843982</td>
<td style="text-align: right;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-ldms">GCTA-GREML-LDMS</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang J, Bakshi A, Zhu Z, Hemani G, ...&amp;, Visscher PM. (2015) Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index Nat. Genet., 47 (10) 1114-1120. doi:10.1038/ng.3390. PMID 26323059</td>
<td style="text-align: right;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-partition">GCTA-GREML-Partition</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang J, Manolio TA, Pasquale LR, Boerwinkle E, ...&amp;, Visscher PM. (2011) Genome partitioning of genetic variation for complex traits using common SNPs Nat. Genet., 43 (6) 519-525. doi:10.1038/ng.823. PMID 21552263</td>
<td style="text-align: right;">2011</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-quantitative">GCTA-GREML-Quantitative</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang J, Benyamin B, McEvoy BP, Gordon S, ...&amp;, Visscher PM. (2010) Common SNPs explain a large proportion of the heritability for human height Nat. Genet., 42 (7) 565-569. doi:10.1038/ng.608. PMID 20562875</td>
<td style="text-align: right;">2010</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hdl">HDL</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Ning Z, Pawitan Y, Shen X. (2020) High-definition likelihood inference of genetic correlations across human complex traits Nat. Genet., 52 (8) 859-864. doi:10.1038/s41588-020-0653-y. PMID 32601477</td>
<td style="text-align: right;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldak">LDAK</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Speed D, Hemani G, Johnson MR, Balding DJ. (2012) Improved heritability estimation from genome-wide SNPs Am. J. Hum. Genet., 91 (6) 1011-1021. doi:10.1016/j.ajhg.2012.10.010. PMID 23217325</td>
<td style="text-align: right;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldsc">LDSC</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Bulik-Sullivan B, Loh PR, Finucane HK, Ripke S, ...&amp;, O'Donovan MC. (2015) LD Score regression distinguishes confounding from polygenicity in genome-wide association studies Nat. Genet., 47 (3) 291-295. doi:10.1038/ng.3211. PMID 25642630</td>
<td style="text-align: right;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sumher">SumHer</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Speed D, Balding DJ. (2019) SumHer better estimates the SNP heritability of complex traits from summary statistics Nat. Genet., 51 (2) 277-284. doi:10.1038/s41588-018-0279-5. PMID 30510236</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tetraher">TetraHer</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Speed D, Evans DM. (2024) Estimating disease heritability from complex pedigrees allowing for ascertainment and covariates Am. J. Hum. Genet., 111 (4) 680-690. doi:10.1016/j.ajhg.2024.02.010. PMID 38490208</td>
<td style="text-align: right;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gnova">GNOVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Lu Q, Li B, Ou D, Erlendsdottir M, ...&amp;, Zhao H. (2017) A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics Am. J. Hum. Genet., 101 (6) 939-964. doi:10.1016/j.ajhg.2017.11.001. PMID 29220677</td>
<td style="text-align: right;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#heels">HEELS</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Li H, Mazumder R, Lin X. (2023) Accurate and efficient estimation of local heritability using summary statistics and the linkage disequilibrium matrix Nat. Commun., 14 (1) 7954. doi:10.1038/s41467-023-43565-9. PMID 38040712</td>
<td style="text-align: right;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hess">HESS</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Shi H, Kichaev G, Pasaniuc B. (2016) Contrasting the genetic architecture of 30 complex traits from summary association data Am. J. Hum. Genet., 99 (1) 139-153. doi:10.1016/j.ajhg.2016.05.013. PMID 27346688</td>
<td style="text-align: right;">2016</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lava">LAVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Werme J, van der Sluis S, Posthuma D, de Leeuw CA. (2022) An integrated framework for local genetic correlation analysis Nat. Genet., 54 (3) 274-282. doi:10.1038/s41588-022-01017-y. PMID 35288712</td>
<td style="text-align: right;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#supergnova">SUPERGNOVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Zhang Y, Lu Q, Ye Y, Huang K, ...&amp;, Zhao H. (2021) SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits Genome Biol., 22 (1) 262. doi:10.1186/s13059-021-02478-w. PMID 34493297</td>
<td style="text-align: right;">2021</td>
</tr>
</tbody>
</table>
<h2 id="genetic-correlation">Genetic correlation</h2>
<h3 id="s-ldxr">S-LDXR</h3>
<ul>
<li><strong>NAME</strong> : S-LDXR </li>
<li><strong>SHORT NAME</strong> : S-LDXR </li>
<li><strong>FULL NAME</strong> : S-LDXR </li>
<li><strong>DESCRIPTION</strong> : S-LDXR is a software for estimating enrichment of stratified squared trans-ethnic genetic correlation across genomic annotations from GWAS summary statistics data. </li>
<li><strong>URL</strong> : <a href="https://huwenboshi.github.io/s-ldxr/">https://huwenboshi.github.io/s-ldxr/</a> </li>
<li><strong>KEYWORDS</strong> : trans-ethnic, stratified, functional categories </li>
<li><strong>TITLE</strong> : Population-specific causal disease effect sizes in functionally important regions impacted by selection </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-21286-1 </li>
<li><strong>ABSTRACT</strong> : Many diseases exhibit population-specific causal effect sizes with trans-ethnic genetic correlations significantly less than 1, limiting trans-ethnic polygenic risk prediction. We develop a new method, S-LDXR, for stratifying squared trans-ethnic genetic correlation across genomic annotations, and apply S-LDXR to genome-wide summary statistics for 31 diseases and complex traits in East Asians (average N = 90K) and Europeans (average N = 267K) with an average trans-ethnic genetic correlation of 0.85. We determine that squared trans-ethnic genetic correlation is 0.82Ã— (s.e. 0.01) depleted in the top quintile of background selection statistic, implying more population-specific causal effect sizes. Accordingly, causal effect sizes are more population-specific in functionally important regions, including conserved and regulatory regions. In regions surrounding specifically expressed genes, causal effect sizes are most population-specific for skin and immune genes, and least population-specific for brain genes. Our results could potentially be explained by stronger gene-environment interaction at loci impacted by selection, particularly positive selection. </li>
<li><strong>CITATION</strong> : Shi H, Gazal S, Kanai M, Koch EM, ...&amp;, Price AL. (2021) Population-specific causal disease effect sizes in functionally important regions impacted by selection Nat. Commun., 12 (1) 1098. doi:10.1038/s41467-021-21286-1. PMID 33597505 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 1098 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33597505">33597505</a> </li>
</ul>
<h3 id="cross-trait-ldsc">cross-trait LDSC</h3>
<ul>
<li><strong>NAME</strong> : cross-trait LDSC </li>
<li><strong>SHORT NAME</strong> : cross-trait LDSC </li>
<li><strong>FULL NAME</strong> : cross-trait LD Score Regression </li>
<li><strong>DESCRIPTION</strong> : ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores. </li>
<li><strong>URL</strong> : <a href="https://github.com/bulik/ldsc">https://github.com/bulik/ldsc</a> </li>
<li><strong>KEYWORDS</strong> : cross-trait, LD score regression </li>
<li><strong>TITLE</strong> : An atlas of genetic correlations across human diseases and traits </li>
<li><strong>DOI</strong> : 10.1038/ng.3406 </li>
<li><strong>ABSTRACT</strong> : Identifying genetic correlations between complex traits and diseases can provide useful etiological insights and help prioritize likely causal relationships. The major challenges preventing estimation of genetic correlation from genome-wide association study (GWAS) data with current methods are the lack of availability of individual-level genotype data and widespread sample overlap among meta-analyses. We circumvent these difficulties by introducing a technique-cross-trait LD Score regression-for estimating genetic correlation that requires only GWAS summary statistics and is not biased by sample overlap. We use this method to estimate 276 genetic correlations among 24 traits. The results include genetic correlations between anorexia nervosa and schizophrenia, anorexia and obesity, and educational attainment and several diseases. These results highlight the power of genome-wide analyses, as there currently are no significantly associated SNPs for anorexia nervosa and only three for educational attainment. </li>
<li><strong>CITATION</strong> : Bulik-Sullivan B, Finucane HK, Anttila V, Gusev A, ...&amp;, Neale BM. (2015) An atlas of genetic correlations across human diseases and traits Nat. Genet., 47 (11) 1236-1241. doi:10.1038/ng.3406. PMID 26414676 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2015 ; 47 ; 11 ; 1236-1241 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26414676">26414676</a> </li>
</ul>
<h3 id="popcorn">popcorn</h3>
<ul>
<li><strong>NAME</strong> : popcorn </li>
<li><strong>SHORT NAME</strong> : popcorn </li>
<li><strong>FULL NAME</strong> : popcorn </li>
<li><strong>DESCRIPTION</strong> : Popcorn is a program for estimaing the correlation of causal variant effect. This is the python3 version of Popcorn and still under development sizes across populations in GWAS. </li>
<li><strong>URL</strong> : <a href="https://github.com/brielin/Popcorn">https://github.com/brielin/Popcorn</a> </li>
<li><strong>KEYWORDS</strong> : trans-ethnic </li>
<li><strong>TITLE</strong> : Transethnic Genetic-Correlation Estimates from Summary Statistics </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2016.05.001 </li>
<li><strong>ABSTRACT</strong> : The increasing number of genetic association studies conducted in multiple populations provides an unprecedented opportunity to study how the genetic architecture of complex phenotypes varies between populations, a problem important for both medical and population genetics. Here, we have developed a method for estimating the transethnic genetic correlation: the correlation of causal-variant effect sizes at SNPs common in populations. This methods takes advantage of the entire spectrum of SNP associations and uses only summary-level data from genome-wide association studies. This avoids the computational costs and privacy concerns associated with genotype-level information while remaining scalable to hundreds of thousands of individuals and millions of SNPs. We applied our method to data on gene expression, rheumatoid arthritis, and type 2 diabetes and overwhelmingly found that the genetic correlation was significantly less than 1. Our method is implemented in a Python package called Popcorn. </li>
<li><strong>CITATION</strong> : Brown BC, Asian Genetic Epidemiology Network Type 2 Diabetes Consortium, Ye CJ, Price AL, ...&amp;, Zaitlen N. (2016) Transethnic Genetic-Correlation Estimates from Summary Statistics Am. J. Hum. Genet., 99 (1) 76-88. doi:10.1016/j.ajhg.2016.05.001. PMID 27321947 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2016 ; 99 ; 1 ; 76-88 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27321947">27321947</a> </li>
</ul>
<h2 id="heritability">Heritability</h2>
<h3 id="bayesrr-rc">BayesRR-RC</h3>
<ul>
<li><strong>NAME</strong> : BayesRR-RC </li>
<li><strong>SHORT NAME</strong> : BayesRR-RC </li>
<li><strong>FULL NAME</strong> : BayesRR-RC </li>
<li><strong>DESCRIPTION</strong> : gmrm is hybrid-parallel software for a Bayesian grouped mixture of regressions model for genome-wide association studies (GWAS). It is written in C++ using extensive optimisations and code vectorisation. It relies on plink's .bed format. It can handle multiple traits simultaneously. </li>
<li><strong>URL</strong> : <a href="https://github.com/medical-genomics-group/gmrm">https://github.com/medical-genomics-group/gmrm</a> </li>
<li><strong>TITLE</strong> : Probabilistic inference of the genetic architecture underlying functional enrichment of complex traits </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-27258-9 </li>
<li><strong>ABSTRACT</strong> : We develop a Bayesian model (BayesRR-RC) that provides robust SNP-heritability estimation, an alternative to marker discovery, and accurate genomic prediction, taking 22 seconds per iteration to estimate 8.4 million SNP-effects and 78 SNP-heritability parameters in the UK Biobank. We find that only â‰¤10% of the genetic variation captured for height, body mass index, cardiovascular disease, and type 2 diabetes is attributable to proximal regulatory regions within 10kb upstream of genes, while 12-25% is attributed to coding regions, 32-44% to introns, and 22-28% to distal 10-500kb upstream regions. Up to 24% of all cis and coding regions of each chromosome are associated with each trait, with over 3,100 independent exonic and intronic regions and over 5,400 independent regulatory regions having â‰¥95% probability of contributing â‰¥0.001% to the genetic variance of these four traits. Our open-source software (GMRM) provides a scalable alternative to current approaches for biobank data. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Patxot M, Banos DT, Kousathanas A, Orliac EJ, ...&amp;, Robinson MR. (2021) Probabilistic inference of the genetic architecture underlying functional enrichment of complex traits Nat. Commun., 12 (1) 6972. doi:10.1038/s41467-021-27258-9. PMID 34848700 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 6972 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34848700">34848700</a> </li>
</ul>
<h3 id="gcta-greml-binary">GCTA-GREML-Binary</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Binary </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (case-control) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>TITLE</strong> : Estimating missing heritability for disease from genome-wide association studies </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2011.02.002 </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies are designed to discover SNPs that are associated with a complex trait. Employing strict significance thresholds when testing individual SNPs avoids false positives at the expense of increasing false negatives. Recently, we developed a method for quantitative traits that estimates the variation accounted for when fitting all SNPs simultaneously. Here we develop this method further for case-control studies. We use a linear mixed model for analysis of binary traits and transform the estimates to a liability scale by adjusting both for scale and for ascertainment of the case samples. We show by theory and simulation that the method is unbiased. We apply the method to data from the Wellcome Trust Case Control Consortium and show that a substantial proportion of variation in liability for Crohn disease, bipolar disorder, and type I diabetes is tagged by common SNPs. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Lee SH, Wray NR, Goddard ME, Visscher PM. (2011) Estimating missing heritability for disease from genome-wide association studies Am. J. Hum. Genet., 88 (3) 294-305. doi:10.1016/j.ajhg.2011.02.002. PMID 21376301 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2011 ; 88 ; 3 ; 294-305 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21376301">21376301</a> </li>
</ul>
<h3 id="gcta-greml-bivariate">GCTA-GREML-Bivariate</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Bivariate </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (Bivariate GREML) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>KEYWORDS</strong> : bivariate </li>
<li><strong>TITLE</strong> : Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/bts474 </li>
<li><strong>ABSTRACT</strong> : SUMMARY: Genetic correlations are the genome-wide aggregate effects of causal variants affecting multiple traits. Traditionally, genetic correlations between complex traits are estimated from pedigree studies, but such estimates can be confounded by shared environmental factors. Moreover, for diseases, low prevalence rates imply that even if the true genetic correlation between disorders was high, co-aggregation of disorders in families might not occur or could not be distinguished from chance. We have developed and implemented statistical methods based on linear mixed models to obtain unbiased estimates of the genetic correlation between pairs of quantitative traits or pairs of binary traits of complex diseases using population-based case-control studies with genome-wide single-nucleotide polymorphism data. The method is validated in a simulation study and applied to estimate genetic correlation between various diseases from Wellcome Trust Case Control Consortium data in a series of bivariate analyses. We estimate a significant positive genetic correlation between risk of Type 2 diabetes and hypertension of ~0.31 (SE 0.14, P = 0.024). AVAILABILITY: Our methods, appropriate for both quantitative and binary traits, are implemented in the freely available software GCTA (http://www.complextraitgenomics.com/software/gcta/reml_bivar.html). CONTACT: hong.lee@uq.edu.au SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>CITATION</strong> : Lee SH, Yang J, Goddard ME, Visscher PM, ...&amp;, Wray NR. (2012) Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood Bioinformatics, 28 (19) 2540-2542. doi:10.1093/bioinformatics/bts474. PMID 22843982 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2012 ; 28 ; 19 ; 2540-2542 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22843982">22843982</a> </li>
</ul>
<h3 id="gcta-greml-ldms">GCTA-GREML-LDMS</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>SHORT NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>FULL NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>DESCRIPTION</strong> : (GREML-LDMS) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>TITLE</strong> : Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index </li>
<li><strong>DOI</strong> : 10.1038/ng.3390 </li>
<li><strong>ABSTRACT</strong> : We propose a method (GREML-LDMS) to estimate heritability for human complex traits in unrelated individuals using whole-genome sequencing data. We demonstrate using simulations based on whole-genome sequencing data that âˆ¼97% and âˆ¼68% of variation at common and rare variants, respectively, can be captured by imputation. Using the GREML-LDMS method, we estimate from 44,126 unrelated individuals that all âˆ¼17 million imputed variants explain 56% (standard error (s.e.) = 2.3%) of variance for height and 27% (s.e. = 2.5%) of variance for body mass index (BMI), and we find evidence that height- and BMI-associated variants have been under natural selection. Considering the imperfect tagging of imputation and potential overestimation of heritability from previous family-based studies, heritability is likely to be 60-70% for height and 30-40% for BMI. Therefore, the missing heritability is small for both traits. For further discovery of genes associated with complex traits, a study design with SNP arrays followed by imputation is more cost-effective than whole-genome sequencing at current prices. </li>
<li><strong>CITATION</strong> : Yang J, Bakshi A, Zhu Z, Hemani G, ...&amp;, Visscher PM. (2015) Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index Nat. Genet., 47 (10) 1114-1120. doi:10.1038/ng.3390. PMID 26323059 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2015 ; 47 ; 10 ; 1114-1120 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26323059">26323059</a> </li>
</ul>
<h3 id="gcta-greml-partition">GCTA-GREML-Partition</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Partition </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (partition the genetic variance into individual chromosomes and genomic segments) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>TITLE</strong> : Genome partitioning of genetic variation for complex traits using common SNPs </li>
<li><strong>DOI</strong> : 10.1038/ng.823 </li>
<li><strong>ABSTRACT</strong> : We estimate and partition genetic variation for height, body mass index (BMI), von Willebrand factor and QT interval (QTi) using 586,898 SNPs genotyped on 11,586 unrelated individuals. We estimate that âˆ¼45%, âˆ¼17%, âˆ¼25% and âˆ¼21% of the variance in height, BMI, von Willebrand factor and QTi, respectively, can be explained by all autosomal SNPs and a further âˆ¼0.5-1% can be explained by X chromosome SNPs. We show that the variance explained by each chromosome is proportional to its length, and that SNPs in or near genes explain more variation than SNPs between genes. We propose a new approach to estimate variation due to cryptic relatedness and population stratification. Our results provide further evidence that a substantial proportion of heritability is captured by common SNPs, that height, BMI and QTi are highly polygenic traits, and that the additive variation explained by a part of the genome is approximately proportional to the total length of DNA contained within genes therein. </li>
<li><strong>CITATION</strong> : Yang J, Manolio TA, Pasquale LR, Boerwinkle E, ...&amp;, Visscher PM. (2011) Genome partitioning of genetic variation for complex traits using common SNPs Nat. Genet., 43 (6) 519-525. doi:10.1038/ng.823. PMID 21552263 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2011 ; 43 ; 6 ; 519-525 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21552263">21552263</a> </li>
</ul>
<h3 id="gcta-greml-quantitative">GCTA-GREML-Quantitative</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Quantitative </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : GCTA-GREML analysis: estimating the variance explained by the SNPs / GCTA-GREML analysis for a case-control study </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>TITLE</strong> : Common SNPs explain a large proportion of the heritability for human height </li>
<li><strong>DOI</strong> : 10.1038/ng.608 </li>
<li><strong>ABSTRACT</strong> : SNPs discovered by genome-wide association studies (GWASs) account for only a small fraction of the genetic variation of complex traits in human populations. Where is the remaining heritability? We estimated the proportion of variance for human height explained by 294,831 SNPs genotyped on 3,925 unrelated individuals using a linear model analysis, and validated the estimation method with simulations based on the observed genotype data. We show that 45% of variance can be explained by considering all SNPs simultaneously. Thus, most of the heritability is not missing but has not previously been detected because the individual effects are too small to pass stringent significance tests. We provide evidence that the remaining heritability is due to incomplete linkage disequilibrium between causal variants and genotyped SNPs, exacerbated by causal variants having lower minor allele frequency than the SNPs explored to date. </li>
<li><strong>CITATION</strong> : Yang J, Benyamin B, McEvoy BP, Gordon S, ...&amp;, Visscher PM. (2010) Common SNPs explain a large proportion of the heritability for human height Nat. Genet., 42 (7) 565-569. doi:10.1038/ng.608. PMID 20562875 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2010 ; 42 ; 7 ; 565-569 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20562875">20562875</a> </li>
</ul>
<h3 id="hdl">HDL</h3>
<ul>
<li><strong>NAME</strong> : HDL </li>
<li><strong>SHORT NAME</strong> : HDL </li>
<li><strong>FULL NAME</strong> : High-Definition Likelihood </li>
<li><strong>DESCRIPTION</strong> : High-Definition Likelihood (HDL) is a likelihood-based method for estimating genetic correlation using GWAS summary statistics. Compared to LD Score regression (LDSC), It reduces the variance of a genetic correlation estimate by about 60%. </li>
<li><strong>URL</strong> : <a href="https://github.com/zhenin/HDL/">https://github.com/zhenin/HDL/</a> </li>
<li><strong>TITLE</strong> : High-definition likelihood inference of genetic correlations across human complex traits </li>
<li><strong>DOI</strong> : 10.1038/s41588-020-0653-y </li>
<li><strong>ABSTRACT</strong> : Genetic correlation is a central parameter for understanding shared genetic architecture between complex traits. By using summary statistics from genome-wide association studies (GWAS), linkage disequilibrium score regression (LDSC) was developed for unbiased estimation of genetic correlations. Although easy to use, LDSC only partially utilizes LD information. By fully accounting for LD across the genome, we develop a high-definition likelihood (HDL) method to improve precision in genetic correlation estimation. Compared to LDSC, HDL reduces the variance of genetic correlation estimates by about 60%, equivalent to a 2.5-fold increase in sample size. We apply HDL and LDSC to estimate 435 genetic correlations among 30 behavioral and disease-related phenotypes measured in the UK Biobank (UKBB). In addition to 154 significant genetic correlations observed for both methods, HDL identified another 57 significant genetic correlations, compared to only another 2 significant genetic correlations identified by LDSC. HDL brings more power to genomic analyses and better reveals the underlying connections across human complex traits. </li>
<li><strong>CITATION</strong> : Ning Z, Pawitan Y, Shen X. (2020) High-definition likelihood inference of genetic correlations across human complex traits Nat. Genet., 52 (8) 859-864. doi:10.1038/s41588-020-0653-y. PMID 32601477 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2020 ; 52 ; 8 ; 859-864 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32601477">32601477</a> </li>
</ul>
<h3 id="ldak">LDAK</h3>
<ul>
<li><strong>NAME</strong> : LDAK </li>
<li><strong>SHORT NAME</strong> : LDAK </li>
<li><strong>FULL NAME</strong> : LD-adjusted kinships </li>
<li><strong>DESCRIPTION</strong> : LDAK is a software package for analysing association study data. </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>TITLE</strong> : Improved heritability estimation from genome-wide SNPs </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2012.10.010 </li>
<li><strong>ABSTRACT</strong> : Estimation of narrow-sense heritability, h(2), from genome-wide SNPs genotyped in unrelated individuals has recently attracted interest and offers several advantages over traditional pedigree-based methods. With the use of this approach, it has been estimated that over half the heritability of human height can be attributed to the ~300,000 SNPs on a genome-wide genotyping array. In comparison, only 5%-10% can be explained by SNPs reaching genome-wide significance. We investigated via simulation the validity of several key assumptions underpinning the mixed-model analysis used in SNP-based h(2) estimation. Although we found that the method is reasonably robust to violations of four key assumptions, it can be highly sensitive to uneven linkage disequilibrium (LD) between SNPs: contributions to h(2) are overestimated from causal variants in regions of high LD and are underestimated in regions of low LD. The overall direction of the bias can be up or down depending on the genetic architecture of the trait, but it can be substantial in realistic scenarios. We propose a modified kinship matrix in which SNPs are weighted according to local LD. We show that this correction greatly reduces the bias and increases the precision of h(2) estimates. We demonstrate the impact of our method on the first seven diseases studied by the Wellcome Trust Case Control Consortium. Our LD adjustment revises downward the h(2) estimate for immune-related diseases, as expected because of high LD in the major-histocompatibility region, but increases it for some nonimmune diseases. To calculate our revised kinship matrix, we developed LDAK, software for computing LD-adjusted kinships. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/3.0/ </li>
<li><strong>CITATION</strong> : Speed D, Hemani G, Johnson MR, Balding DJ. (2012) Improved heritability estimation from genome-wide SNPs Am. J. Hum. Genet., 91 (6) 1011-1021. doi:10.1016/j.ajhg.2012.10.010. PMID 23217325 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2012 ; 91 ; 6 ; 1011-1021 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23217325">23217325</a> </li>
</ul>
<h3 id="ldsc">LDSC</h3>
<ul>
<li><strong>NAME</strong> : LDSC </li>
<li><strong>SHORT NAME</strong> : LDSC </li>
<li><strong>FULL NAME</strong> : LD Score Regression </li>
<li><strong>DESCRIPTION</strong> : ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores. </li>
<li><strong>URL</strong> : <a href="https://github.com/bulik/ldsc">https://github.com/bulik/ldsc</a> </li>
<li><strong>TITLE</strong> : LD Score regression distinguishes confounding from polygenicity in genome-wide association studies </li>
<li><strong>DOI</strong> : 10.1038/ng.3211 </li>
<li><strong>ABSTRACT</strong> : Both polygenicity (many small genetic effects) and confounding biases, such as cryptic relatedness and population stratification, can yield an inflated distribution of test statistics in genome-wide association studies (GWAS). However, current methods cannot distinguish between inflation from a true polygenic signal and bias. We have developed an approach, LD Score regression, that quantifies the contribution of each by examining the relationship between test statistics and linkage disequilibrium (LD). The LD Score regression intercept can be used to estimate a more powerful and accurate correction factor than genomic control. We find strong evidence that polygenicity accounts for the majority of the inflation in test statistics in many GWAS of large sample size. </li>
<li><strong>CITATION</strong> : Bulik-Sullivan B, Loh PR, Finucane HK, Ripke S, ...&amp;, O'Donovan MC. (2015) LD Score regression distinguishes confounding from polygenicity in genome-wide association studies Nat. Genet., 47 (3) 291-295. doi:10.1038/ng.3211. PMID 25642630 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2015 ; 47 ; 3 ; 291-295 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25642630">25642630</a> </li>
</ul>
<h3 id="sumher">SumHer</h3>
<ul>
<li><strong>NAME</strong> : SumHer </li>
<li><strong>SHORT NAME</strong> : SumHer </li>
<li><strong>FULL NAME</strong> : SumHer </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>TITLE</strong> : SumHer better estimates the SNP heritability of complex traits from summary statistics </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0279-5 </li>
<li><strong>ABSTRACT</strong> : We present SumHer, software for estimating confounding bias, SNP heritability, enrichments of heritability and genetic correlations using summary statistics from genome-wide association studies. The key difference between SumHer and the existing software LD Score Regression (LDSC) is that SumHer allows the user to specify the heritability model. We apply SumHer to results from 24 large-scale association studies (average sample size 121,000) using our recommended heritability model. We show that these studies tended to substantially over-correct for confounding, and as a result the number of genome-wide significant loci was under-reported by about a quarter. We also estimate enrichments for 24 categories of SNPs defined by functional annotations. A previous study using LDSC reported that conserved regions were 13-fold enriched, and found a further six categories with above threefold enrichment. By contrast, our analysis using SumHer finds that none of the categories have enrichment above twofold. SumHer provides an improved understanding of the genetic architecture of complex traits, which enables more efficient analysis of future genetic data. </li>
<li><strong>CITATION</strong> : Speed D, Balding DJ. (2019) SumHer better estimates the SNP heritability of complex traits from summary statistics Nat. Genet., 51 (2) 277-284. doi:10.1038/s41588-018-0279-5. PMID 30510236 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2019 ; 51 ; 2 ; 277-284 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30510236">30510236</a> </li>
</ul>
<h3 id="tetraher">TetraHer</h3>
<ul>
<li><strong>NAME</strong> : TetraHer </li>
<li><strong>SHORT NAME</strong> : TetraHer </li>
<li><strong>FULL NAME</strong> : TetraHer </li>
<li><strong>DESCRIPTION</strong> : a method for estimating the liability heritability of binary phenotypes </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>TITLE</strong> : Estimating disease heritability from complex pedigrees allowing for ascertainment and covariates </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2024.02.010 </li>
<li><strong>ABSTRACT</strong> : We propose TetraHer, a method for estimating the liability heritability of binary phenotypes. TetraHer has five key features. First, it can be applied to data from complex pedigrees that contain multiple types of relationships. Second, it can correct for ascertainment of cases or controls. Third, it can accommodate covariates. Fourth, it can model the contribution of common environment. Fifth, it produces a likelihood that can be used for significance testing. We first demonstrate the validity of TetraHer on simulated data. We then use TetraHer to estimate liability heritability for 229 codes from the tenth International Classification of Diseases (ICD-10). We identify 107 codes with significant heritability (p &lt; 0.05/229), which can be used in future analyses for investigating the genetic architecture of human diseases. </li>
<li><strong>CITATION</strong> : Speed D, Evans DM. (2024) Estimating disease heritability from complex pedigrees allowing for ascertainment and covariates Am. J. Hum. Genet., 111 (4) 680-690. doi:10.1016/j.ajhg.2024.02.010. PMID 38490208 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2024 ; 111 ; 4 ; 680-690 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38490208">38490208</a> </li>
</ul>
<h2 id="local-heritabilitygenetic-correlation">Local heritability/genetic correlation</h2>
<h3 id="gnova">GNOVA</h3>
<ul>
<li><strong>NAME</strong> : GNOVA </li>
<li><strong>SHORT NAME</strong> : GNOVA </li>
<li><strong>FULL NAME</strong> : GeNetic cOVariance Analyzer </li>
<li><strong>DESCRIPTION</strong> : A principled framework to estimate annotation-stratified genetic covariance using GWAS summary statistics. </li>
<li><strong>URL</strong> : <a href="https://github.com/xtonyjiang/GNOVA">https://github.com/xtonyjiang/GNOVA</a> </li>
<li><strong>TITLE</strong> : A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2017.11.001 </li>
<li><strong>ABSTRACT</strong> : Despite the success of large-scale genome-wide association studies (GWASs) on complex traits, our understanding of their genetic architecture is far from complete. Jointly modeling multiple traits' genetic profiles has provided insights into the shared genetic basis of many complex traits. However, large-scale inference sets a high bar for both statistical power and biological interpretability. Here we introduce a principled framework to estimate annotation-stratified genetic covariance between traits using GWAS summary statistics. Through theoretical and numerical analyses, we demonstrate that our method provides accurate covariance estimates, thereby enabling researchers to dissect both the shared and distinct genetic architecture across traits to better understand their etiologies. Among 50 complex traits with publicly accessible GWAS summary statistics (Ntotalâ‰ˆ 4.5 million), we identified more than 170 pairs with statistically significant genetic covariance. In particular, we found strong genetic covariance between late-onset Alzheimer disease (LOAD) and amyotrophic lateral sclerosis (ALS), two major neurodegenerative diseases, in single-nucleotide polymorphisms (SNPs) with high minor allele frequencies and in SNPs located in the predicted functional genome. Joint analysis of LOAD, ALS, and other traits highlights LOAD's correlation with cognitive traits and hints at an autoimmune component for ALS. </li>
<li><strong>COPYRIGHT</strong> : https://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Lu Q, Li B, Ou D, Erlendsdottir M, ...&amp;, Zhao H. (2017) A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics Am. J. Hum. Genet., 101 (6) 939-964. doi:10.1016/j.ajhg.2017.11.001. PMID 29220677 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2017 ; 101 ; 6 ; 939-964 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29220677">29220677</a> </li>
</ul>
<h3 id="heels">HEELS</h3>
<ul>
<li><strong>NAME</strong> : HEELS </li>
<li><strong>SHORT NAME</strong> : HEELS </li>
<li><strong>FULL NAME</strong> : Heritability Estimation with high Efficiency using LD and association Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : HEELS is a Python-based command line tool that produce accurate and precise local heritability estimates using summary-level statistics (marginal association test statistics along with the empirical (in-sample) LD statistics). </li>
<li><strong>URL</strong> : <a href="https://github.com/huilisabrina/HEELS">https://github.com/huilisabrina/HEELS</a> </li>
<li><strong>TITLE</strong> : Accurate and efficient estimation of local heritability using summary statistics and the linkage disequilibrium matrix </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-43565-9 </li>
<li><strong>ABSTRACT</strong> : Existing SNP-heritability estimators that leverage summary statistics from genome-wide association studies (GWAS) are much less efficient (i.e., have larger standard errors) than the restricted maximum likelihood (REML) estimators which require access to individual-level data. We introduce a new method for local heritability estimation-Heritability Estimation with high Efficiency using LD and association Summary Statistics (HEELS)-that significantly improves the statistical efficiency of summary-statistics-based heritability estimator and attains comparable statistical efficiency as REML (with a relative statistical efficiency &gt;92%). Moreover, we propose representing the empirical LD matrix as the sum of a low-rank matrix and a banded matrix. We show that this way of modeling the LD can not only reduce the storage and memory cost, but also improve the computational efficiency of heritability estimation. We demonstrate the statistical efficiency of HEELS and the advantages of our proposed LD approximation strategies both in simulations and through empirical analyses of the UK Biobank data. </li>
<li><strong>CITATION</strong> : Li H, Mazumder R, Lin X. (2023) Accurate and efficient estimation of local heritability using summary statistics and the linkage disequilibrium matrix Nat. Commun., 14 (1) 7954. doi:10.1038/s41467-023-43565-9. PMID 38040712 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 7954 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38040712">38040712</a> </li>
</ul>
<h3 id="hess">HESS</h3>
<ul>
<li><strong>NAME</strong> : HESS </li>
<li><strong>SHORT NAME</strong> : HESS </li>
<li><strong>FULL NAME</strong> : Heritability Estimation from Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : HESS (Heritability Estimation from Summary Statistics) is a software package for estimating and visualizing local SNP-heritability and genetic covariance (correlation) from GWAS summary association data. </li>
<li><strong>URL</strong> : <a href="https://huwenboshi.github.io/hess/">https://huwenboshi.github.io/hess/</a> </li>
<li><strong>TITLE</strong> : Contrasting the genetic architecture of 30 complex traits from summary association data </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2016.05.013 </li>
<li><strong>ABSTRACT</strong> : Variance-component methods that estimate the aggregate contribution of large sets of variants to the heritability of complex traits have yielded important insights into the genetic architecture of common diseases. Here, we introduce methods that estimate the total trait variance explained by the typed variants at a single locus in the genome (local SNP heritability) from genome-wide association study (GWAS) summary data while accounting for linkage disequilibrium among variants. We applied our estimator to ultra-large-scale GWAS summary data of 30 common traits and diseases to gain insights into their local genetic architecture. First, we found that common SNPs have a high contribution to the heritability of all studied traits. Second, we identified traits for which the majority of the SNP heritability can be confined to a small percentage of the genome. Third, we identified GWAS risk loci where the entire locus explains significantly more variance in the trait than the GWAS reported variants. Finally, we identified loci that explain a significant amount of heritability across multiple traits. </li>
<li><strong>CITATION</strong> : Shi H, Kichaev G, Pasaniuc B. (2016) Contrasting the genetic architecture of 30 complex traits from summary association data Am. J. Hum. Genet., 99 (1) 139-153. doi:10.1016/j.ajhg.2016.05.013. PMID 27346688 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2016 ; 99 ; 1 ; 139-153 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27346688">27346688</a> </li>
</ul>
<h3 id="lava">LAVA</h3>
<ul>
<li><strong>NAME</strong> : LAVA </li>
<li><strong>SHORT NAME</strong> : LAVA </li>
<li><strong>FULL NAME</strong> : Local Analysis of [co]Variant Association </li>
<li><strong>DESCRIPTION</strong> : LAVA is a tool to conduct genome-wide, local genetic correlation analysis on multiple traits, using GWAS summary statistics as input. </li>
<li><strong>URL</strong> : <a href="https://ctg.cncr.nl/software/lava">https://ctg.cncr.nl/software/lava</a> </li>
<li><strong>TITLE</strong> : An integrated framework for local genetic correlation analysis </li>
<li><strong>DOI</strong> : 10.1038/s41588-022-01017-y </li>
<li><strong>ABSTRACT</strong> : Genetic correlation (rg) analysis is used to identify phenotypes that may have a shared genetic basis. Traditionally, rg is studied globally, considering only the average of the shared signal across the genome, although this approach may fail when the rg is confined to particular genomic regions or in opposing directions at different loci. Current tools for local rg analysis are restricted to analysis of two phenotypes. Here we introduce LAVA, an integrated framework for local rg analysis that, in addition to testing the standard bivariate local rgs between two phenotypes, can evaluate local heritabilities and analyze conditional genetic relations between several phenotypes using partial correlation and multiple regression. Applied to 25 behavioral and health phenotypes, we show considerable heterogeneity in the bivariate local rgs across the genome, which is often masked by the global rg patterns, and demonstrate how our conditional approaches can elucidate more complex, multivariate genetic relations. </li>
<li><strong>CITATION</strong> : Werme J, van der Sluis S, Posthuma D, de Leeuw CA. (2022) An integrated framework for local genetic correlation analysis Nat. Genet., 54 (3) 274-282. doi:10.1038/s41588-022-01017-y. PMID 35288712 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2022 ; 54 ; 3 ; 274-282 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35288712">35288712</a> </li>
</ul>
<h3 id="supergnova">SUPERGNOVA</h3>
<ul>
<li><strong>NAME</strong> : SUPERGNOVA </li>
<li><strong>SHORT NAME</strong> : SUPERGNOVA </li>
<li><strong>FULL NAME</strong> : SUPER GeNetic cOVariance Analyzer </li>
<li><strong>DESCRIPTION</strong> : SUPERGNOVA (SUPER GeNetic cOVariance Analyzer) is a statistical framework to perform local genetic covariance analysis. </li>
<li><strong>URL</strong> : <a href="https://github.com/qlu-lab/SUPERGNOVA">https://github.com/qlu-lab/SUPERGNOVA</a> </li>
<li><strong>TITLE</strong> : SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits </li>
<li><strong>DOI</strong> : 10.1186/s13059-021-02478-w </li>
<li><strong>ABSTRACT</strong> : Local genetic correlation quantifies the genetic similarity of complex traits in specific genomic regions. However, accurate estimation of local genetic correlation remains challenging, due to linkage disequilibrium in local genomic regions and sample overlap across studies. We introduce SUPERGNOVA, a statistical framework to estimate local genetic correlations using summary statistics from genome-wide association studies. We demonstrate that SUPERGNOVA outperforms existing methods through simulations and analyses of 30 complex traits. In particular, we show that the positive yet paradoxical genetic correlation between autism spectrum disorder and cognitive performance could be explained by two etiologically distinct genetic signatures with bidirectional local genetic correlations. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Zhang Y, Lu Q, Ye Y, Huang K, ...&amp;, Zhao H. (2021) SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits Genome Biol., 22 (1) 262. doi:10.1186/s13059-021-02478-w. PMID 34493297 </li>
<li><strong>JOURNAL_INFO</strong> : Genome biology ; Genome Biol. ; 2021 ; 22 ; 1 ; 262 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34493297">34493297</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>