<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <meta name="author" content="HE Yunye">
    
    <link rel="shortcut icon" href="../img/favicon.ico">

    
    <title>Heritability_and_genetic_correlation - CTGCatalog</title>
    

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/v4-shims.css">
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/hack-font@3.3.0/build/web/hack.min.css">
    <link href='//rsms.me/inter/inter.css' rel='stylesheet' type='text/css'>
    <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin' rel='stylesheet' type='text/css'>
    <link href="../css/bootstrap-custom.min.css" rel="stylesheet">
    <link href="../css/base.min.css" rel="stylesheet">
    <link href="../css/cinder.min.css" rel="stylesheet">

    
        
        <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/styles/github.min.css">
        
    

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
            <script src="https://cdn.jsdelivr.net/npm/html5shiv@3.7.3/dist/html5shiv.min.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/respond.js@1.4.2/dest/respond.min.js"></script>
        <![endif]-->

    

     
</head>

<body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">

        <!-- Collapsed navigation -->
        <div class="navbar-header">
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            

            <!-- Main title -->

            
              <a class="navbar-brand" href="..">CTGCatalog</a>
            
        </div>

        <!-- Expanded navigation -->
        <div class="navbar-collapse collapse">
                <!-- Main navigation -->
                <ul class="nav navbar-nav">
                
                
                    <li >
                        <a href="..">Home</a>
                    </li>
                
                
                
                    <li class="dropdown active">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Tools <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                        
                            
<li >
    <a href="../Tools_README/">Tools</a>
</li>

                        
                            
<li >
    <a href="../Tools_Visualization_README/">Visualization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Drug_discovery_README/">Drug_discovery</a>
</li>

                        
                            
<li >
    <a href="../Tools_Gene_prioritization_README/">Gene_prioritization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Polygenic_risk_scores_README/">Polygenic_risk_scores</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Association_tests</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Tools_Association_tests_README/">Association_tests</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_TWAS_README/">TWAS</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_sQTL_README/">sQTL</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_eQTL_README/">eQTL</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_RWAS_README/">RWAS</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Tools_Colocalization_README/">Colocalization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Simulation_README/">Simulation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Data_processing_README/">Data_processing</a>
</li>

                        
                            
<li >
    <a href="../Tools_Annotation_README/">Annotation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Fine_mapping_README/">Fine_mapping</a>
</li>

                        
                            
<li >
    <a href="../Tools_Gene_set_pathway_analysis_README/">Gene_set_pathway_analysis</a>
</li>

                        
                            
<li >
    <a href="../Tools_Tissue_and_single_cell_README/">Tissue_and_single_cell</a>
</li>

                        
                            
<li >
    <a href="../Tools_Imputation_README/">Imputation</a>
</li>

                        
                            
<li class="active">
    <a href="./">Heritability_and_genetic_correlation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Proteomics_README/">Proteomics</a>
</li>

                        
                            
<li >
    <a href="../Tools_GxE_interactions_README/">GxE_interactions</a>
</li>

                        
                            
<li >
    <a href="../Tools_Mendelian_randomization_README/">Mendelian_randomization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Meta_and_Multi_triat_README/">Meta_and_Multi_triat</a>
</li>

                        
                        </ul>
                    </li>
                
                
                
                    <li >
                        <a href="../Sumstats_README/">Sumstats</a>
                    </li>
                
                
                
                    <li class="dropdown">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Reference_data <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                        
                            
<li >
    <a href="../Reference_data_README/">Reference_data</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Biobanks_Cohorts_README/">Biobanks_Cohorts</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Protein_README/">Protein</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Gene</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Gene_README/">Gene</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_NCBI_refSeq_README/">NCBI_refSeq</a>
</li>

        
            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">UCSC_refGene</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Gene_UCSC_refGene_README/">UCSC_refGene</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/">ucsc_genes</a>
</li>

        
    </ul>
  </li>

        
            
<li >
    <a href="../Reference_data_Gene_HGNC_README/">HGNC</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_knownGenes_README/">knownGenes</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_Ensembl_README/">Ensembl</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_GENCODE_README/">GENCODE</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Reference_data_Phenotype_README/">Phenotype</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Variant_README/">Variant</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Epigenomics_README/">Epigenomics</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Expression_README/">Expression</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Genome</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Genome_README/">Genome</a>
</li>

        
            
<li >
    <a href="../Reference_data_Genome_hg38_README/">hg38</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Reference_data_xQTL_README/">xQTL</a>
</li>

                        
                        </ul>
                    </li>
                
                
                </ul>

            <ul class="nav navbar-nav navbar-right">
                    <li>
                        <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
                            <i class="fas fa-search"></i> Search
                        </a>
                    </li>
                    <li >
                        <a rel="prev" href="../Tools_Imputation_README/">
                            <i class="fas fa-arrow-left"></i> Previous
                        </a>
                    </li>
                    <li >
                        <a rel="next" href="../Tools_Proteomics_README/">
                            Next <i class="fas fa-arrow-right"></i>
                        </a>
                    </li>
                    <li>
                        <a href="https://github.com/Cloufield/CTGCatalog/"><i class="fab fa-github"></i> GitHub</a>
                    </li>
            </ul>
        </div>
    </div>
</div>

    <div class="container">
        
        
        <div class="col-md-3"><div class="bs-sidebar hidden-print affix well" role="complementary">
    <ul class="nav bs-sidenav">
        <li class="first-level active"><a href="#contents">Contents</a></li>
        <li class="first-level "><a href="#heritabilty">Heritabilty</a></li>
            <li class="second-level"><a href="#ldsc">LDSC</a></li>
                
            <li class="second-level"><a href="#gcta-greml">GCTA - GREML</a></li>
                
        <li class="first-level "><a href="#local-heritability-local-genetic-correlation">Local Heritability/ Local genetic correlation</a></li>
            <li class="second-level"><a href="#lava">LAVA</a></li>
                
            <li class="second-level"><a href="#hess">HESS</a></li>
                
            <li class="second-level"><a href="#gnova">GNOVA</a></li>
                
            <li class="second-level"><a href="#supergnova">SUPERGNOVA</a></li>
                
        <li class="first-level "><a href="#genetic-correlation">Genetic correlation</a></li>
            <li class="second-level"><a href="#cross-trait-ldsc">cross-trait LDSC</a></li>
                
            <li class="second-level"><a href="#popcorn">popcorn</a></li>
                
    </ul>
</div></div>
        <div class="col-md-9" role="main">

<h1 id="contents">Contents</h1>
<ul>
<li>Heritability</li>
<li>Local Heritability/ Local genetic correlation</li>
<li>Genetic correlation</li>
</ul>
<h1 id="heritabilty">Heritabilty</h1>
<h2 id="ldsc">LDSC</h2>
<ul>
<li>SHORT NAME:LDSC</li>
<li>FULL NAME:LD Score Regression</li>
<li>YEAR:2015</li>
<li>DESCRIPTION:ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores.</li>
<li>URL:https://github.com/bulik/ldsc</li>
<li>CITATION :Bulik-Sullivan, Brendan K., et al. "LD Score regression distinguishes confounding from polygenicity in genome-wide association studies." Nature genetics 47.3 (2015): 291-295.</li>
</ul>
<h2 id="gcta-greml">GCTA - GREML</h2>
<ul>
<li>SHORT NAME: GREML</li>
<li>FULL NAME:Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li>YEAR:2010</li>
<li>DESCRIPTION:GCTA-GREML analysis: estimating the variance explained by the SNPs / GCTA-GREML analysis for a case-control study</li>
<li>URL: https://yanglab.westlake.edu.cn/software/gcta/#GREML</li>
<li>CITATION :(quantitative)Yang, Jian, et al. "Common SNPs explain a large proportion of the heritability for human height." Nature genetics 42.7 (2010): 565-569.</li>
<li>CITATION :(case-control )Lee, Sang Hong, et al. "Estimating missing heritability for disease from genome-wide association studies." The American Journal of Human Genetics 88.3 (2011): 294-305.</li>
<li>CITATION :(partition the genetic variance into individual chromosomes and genomic segments) Yang, Jian, et al. "Genome partitioning of genetic variation for complex traits using common SNPs." Nature genetics 43.6 (2011): 519-525.</li>
<li>CITATION :(Bivariate GREML) Lee, Sang Hong, et al. "Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood." Bioinformatics 28.19 (2012): 2540-2542.</li>
<li>CITATION :(GREML-LDMS) Yang, Jian, et al. "Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index." Nature genetics 47.10 (2015): 1114-1120.</li>
</ul>
<h1 id="local-heritability-local-genetic-correlation">Local Heritability/ Local genetic correlation</h1>
<h2 id="lava">LAVA</h2>
<ul>
<li>DESCRIPTION:LAVA is a tool to conduct genome-wide, local genetic correlation analysis on multiple traits, using GWAS summary statistics as input.</li>
<li>URL: https://ctg.cncr.nl/software/lava</li>
<li>YEAR: 2022</li>
<li>CITATION:Werme, J., van der Sluis, S., Posthuma, D. et al. An integrated framework for local genetic correlation analysis. Nat Genet 54, 274–282 (2022). https://doi.org/10.1038/s41588-022-01017-y</li>
</ul>
<h2 id="hess">HESS</h2>
<ul>
<li>FULL NAME: Heritability Estimation from Summary Statistics</li>
<li>DESCRIPTION:HESS (Heritability Estimation from Summary Statistics) is a software package for estimating and visualizing local SNP-heritability and genetic covariance (correlation) from GWAS summary association data.</li>
<li>YEAR: 2016</li>
<li>URL: https://huwenboshi.github.io/hess/</li>
<li>CITATION:  Shi, Huwenbo, Gleb Kichaev, and Bogdan Pasaniuc. "Contrasting the genetic architecture of 30 complex traits from summary association data." The American Journal of Human Genetics 99.1 (2016): 139-153.</li>
<li>CITATION: Shi, Huwenbo, et al. "Local genetic correlation gives insights into the shared genetic architecture of complex traits." The American Journal of Human Genetics 101.5 (2017): 737-751.</li>
</ul>
<h2 id="gnova">GNOVA</h2>
<ul>
<li>SHORT NAME:GNOVA</li>
<li>FULL NAME:GeNetic cOVariance Analyzer</li>
<li>YEAR:2017</li>
<li>DESCRIPTION:A principled framework to estimate annotation-stratified genetic covariance using GWAS summary statistics.</li>
<li>URL:https://github.com/xtonyjiang/GNOVA</li>
<li>CITATION :Lu, Qiongshi, et al. "A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics." The American Journal of Human Genetics 101.6 (2017): 939-964.</li>
</ul>
<h2 id="supergnova">SUPERGNOVA</h2>
<ul>
<li>SHORT NAME:SUPERGNOVA</li>
<li>FULL NAME:SUPER GeNetic cOVariance Analyzer</li>
<li>YEAR:2021</li>
<li>DESCRIPTION:SUPERGNOVA (SUPER GeNetic cOVariance Analyzer) is a statistical framework to perform local genetic covariance analysis. </li>
<li>URL:https://github.com/qlu-lab/SUPERGNOVA</li>
<li>CITATION :Zhang, Yiliang, et al. "SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits." Genome biology 22.1 (2021): 1-30.</li>
</ul>
<h1 id="genetic-correlation">Genetic correlation</h1>
<h2 id="cross-trait-ldsc">cross-trait LDSC</h2>
<ul>
<li>SHORT NAME: cross-trait LDSC</li>
<li>FULL NAME:cross-trait LD Score Regression</li>
<li>YEAR: 2015</li>
<li>DESCRIPTION:ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores.</li>
<li>URL: https://github.com/bulik/ldsc</li>
<li>CITATION : Bulik-Sullivan, Brendan, et al. "An atlas of genetic correlations across human diseases and traits." Nature genetics 47.11 (2015): 1236-1241.</li>
</ul>
<h2 id="popcorn">popcorn</h2>
<ul>
<li>SHORT NAME:popcorn</li>
<li>FULL NAME:popcorn</li>
<li>YEAR:2016</li>
<li>DESCRIPTION:Popcorn is a program for estimaing the correlation of causal variant effect. This is the python3 version of Popcorn and still under development sizes across populations in GWAS.</li>
<li>URL:https://github.com/brielin/Popcorn</li>
<li>CITATION :Brown, Brielin C., et al. "Transethnic genetic-correlation estimates from summary statistics." The American Journal of Human Genetics 99.1 (2016): 76-88.</li>
</ul></div>
        
        
    </div>

    
      <footer class="col-md-12 text-center">
          
          
            <hr>
            <p>
            <small>CTGCatalog is licensed under the MIT license</small><br>
            
            <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</small>
            </p>
          

          
          
      </footer>
    
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="../js/bootstrap-3.0.3.min.js"></script>

    
    <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/highlight.min.js"></script>
        
    <script>hljs.initHighlightingOnLoad();</script>
    

    <script>var base_url = ".."</script>
    
    <script src="../js/base.js"></script>
    <script src="../search/main.js"></script>

    <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal">
                    <span aria-hidden="true">&times;</span>
                    <span class="sr-only">Close</span>
                </button>
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
            </div>
            <div class="modal-body">
                <p>
                    From here you can search these documents. Enter
                    your search terms below.
                </p>
                <form>
                    <div class="form-group">
                        <input type="text" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>
    </body>

</html>
