
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_HLA_README/">
      
      
        <link rel="next" href="../Tools_Imputation_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Heritability_and_genetic_correlation - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Heritability_and_genetic_correlation
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#s-ldxr" class="md-nav__link">
    <span class="md-ellipsis">
      S-LDXR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cross-trait-ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      cross-trait LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popcorn" class="md-nav__link">
    <span class="md-ellipsis">
      popcorn
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#heritability" class="md-nav__link">
    <span class="md-ellipsis">
      Heritability
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Heritability">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesrr-rc" class="md-nav__link">
    <span class="md-ellipsis">
      BayesRR-RC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-binary" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Binary
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-bivariate" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Bivariate
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-ldms" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-LDMS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-partition" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Partition
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-quantitative" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Quantitative
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdl" class="md-nav__link">
    <span class="md-ellipsis">
      HDL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumher" class="md-nav__link">
    <span class="md-ellipsis">
      SumHer
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetraher" class="md-nav__link">
    <span class="md-ellipsis">
      TetraHer
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#local-heritabilitygenetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Local heritability/genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Local heritability/genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gnova" class="md-nav__link">
    <span class="md-ellipsis">
      GNOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heels" class="md-nav__link">
    <span class="md-ellipsis">
      HEELS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hess" class="md-nav__link">
    <span class="md-ellipsis">
      HESS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lava" class="md-nav__link">
    <span class="md-ellipsis">
      LAVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supergnova" class="md-nav__link">
    <span class="md-ellipsis">
      SUPERGNOVA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#s-ldxr" class="md-nav__link">
    <span class="md-ellipsis">
      S-LDXR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cross-trait-ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      cross-trait LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#popcorn" class="md-nav__link">
    <span class="md-ellipsis">
      popcorn
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#heritability" class="md-nav__link">
    <span class="md-ellipsis">
      Heritability
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Heritability">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesrr-rc" class="md-nav__link">
    <span class="md-ellipsis">
      BayesRR-RC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-binary" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Binary
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-bivariate" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Bivariate
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-ldms" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-LDMS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-partition" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Partition
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gcta-greml-quantitative" class="md-nav__link">
    <span class="md-ellipsis">
      GCTA-GREML-Quantitative
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hdl" class="md-nav__link">
    <span class="md-ellipsis">
      HDL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldsc" class="md-nav__link">
    <span class="md-ellipsis">
      LDSC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sumher" class="md-nav__link">
    <span class="md-ellipsis">
      SumHer
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tetraher" class="md-nav__link">
    <span class="md-ellipsis">
      TetraHer
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#local-heritabilitygenetic-correlation" class="md-nav__link">
    <span class="md-ellipsis">
      Local heritability/genetic correlation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Local heritability/genetic correlation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gnova" class="md-nav__link">
    <span class="md-ellipsis">
      GNOVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heels" class="md-nav__link">
    <span class="md-ellipsis">
      HEELS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hess" class="md-nav__link">
    <span class="md-ellipsis">
      HESS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lava" class="md-nav__link">
    <span class="md-ellipsis">
      LAVA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#supergnova" class="md-nav__link">
    <span class="md-ellipsis">
      SUPERGNOVA
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Heritability_and_genetic_correlation</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#s-ldxr">S-LDXR</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Shi, H., Gazal, S., Kanai, M., Koch, E. M., Schoech, A. P., Siewert, K. M., ... &amp; Price, A. L. (2021). Population-specific causal disease effect sizes in functionally important regions impacted by selection. Nature communications, 12(1), 1098.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cross-trait-ldsc">cross-trait LDSC</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Bulik-Sullivan, Brendan, et al. "An atlas of genetic correlations across human diseases and traits." Nature genetics 47.11 (2015): 1236-1241.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#popcorn">popcorn</a></td>
<td style="text-align: left;">Genetic correlation</td>
<td style="text-align: left;">Brown, Brielin C., et al. "Transethnic genetic-correlation estimates from summary statistics." The American Journal of Human Genetics 99.1 (2016): 76-88.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bayesrr-rc">BayesRR-RC</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;"><br><br>Patxot, M. et al. Probabilistic inference of the genetic architecture underlying functional enrichment of complex traits. Nat. Commun. 12, 6972 (2021).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-binary">GCTA-GREML-Binary</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Lee, Sang Hong, et al. "Estimating missing heritability for disease from genome-wide association studies." The American Journal of Human Genetics 88.3 (2011): 294-305.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-bivariate">GCTA-GREML-Bivariate</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Lee, Sang Hong, et al. "Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood." Bioinformatics 28.19 (2012): 2540-2542.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-ldms">GCTA-GREML-LDMS</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang, Jian, et al. "Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index." Nature genetics 47.10 (2015): 1114-1120.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-partition">GCTA-GREML-Partition</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang, Jian, et al. "Genome partitioning of genetic variation for complex traits using common SNPs." Nature genetics 43.6 (2011): 519-525.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gcta-greml-quantitative">GCTA-GREML-Quantitative</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Yang, Jian, et al. "Common SNPs explain a large proportion of the heritability for human height." Nature genetics 42.7 (2010): 565-569.<br><br><br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hdl">HDL</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Ning, Z., Pawitan, Y., &amp; Shen, X. (2020). High-definition likelihood inference of genetic correlations across human complex traits. Nature genetics, 52(8), 859-864.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldak">LDAK</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;"><br><br>Speed, D., Hemani, G., Johnson, M. R. &amp; Balding, D. J. Improved heritability estimation from genome-wide SNPs. Am. J. Hum. Genet. 91, 1011–1021 (2012).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldsc">LDSC</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;">Bulik-Sullivan, Brendan K., et al. "LD Score regression distinguishes confounding from polygenicity in genome-wide association studies." Nature genetics 47.3 (2015): 291-295.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sumher">SumHer</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;"><br><br>Speed, D. &amp; Balding, D. J. SumHer better estimates the SNP heritability of complex traits from summary statistics. Nat. Genet. 51, 277–284 (2019).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tetraher">TetraHer</a></td>
<td style="text-align: left;">Heritability</td>
<td style="text-align: left;"><br><br>Speed, D. &amp; Evans, D. M. Estimating disease heritability from complex pedigrees allowing for ascertainment and covariates. Am. J. Hum. Genet. 111, 680–690 (2024).<br><br>  <br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gnova">GNOVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Lu, Qiongshi, et al. "A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics." The American Journal of Human Genetics 101.6 (2017): 939-964.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#heels">HEELS</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Li, H., Mazumder, R., &amp; Lin, X. (2023). Accurate and efficient estimation of local heritability using summary statistics and the linkage disequilibrium matrix. Nature Communications, 14(1), 7954.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hess">HESS</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Shi, Huwenbo, Gleb Kichaev, and Bogdan Pasaniuc. "Contrasting the genetic architecture of 30 complex traits from summary association data." The American Journal of Human Genetics 99.1 (2016): 139-153.<br><br>Shi, Huwenbo, et al. "Local genetic correlation gives insights into the shared genetic architecture of complex traits." The American Journal of Human Genetics 101.5 (2017): 737-751.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lava">LAVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Werme, J., van der Sluis, S., Posthuma, D. et al. An integrated framework for local genetic correlation analysis. Nat Genet 54, 274–282 (2022). https://doi.org/10.1038/s41588-022-01017-y</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#supergnova">SUPERGNOVA</a></td>
<td style="text-align: left;">Local heritability/genetic correlation</td>
<td style="text-align: left;">Zhang, Yiliang, et al. "SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits." Genome biology 22.1 (2021): 1-30.</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="genetic-correlation">Genetic correlation</h2>
<h3 id="s-ldxr">S-LDXR</h3>
<ul>
<li><strong>NAME</strong> : S-LDXR </li>
<li><strong>SHORT NAME</strong> : S-LDXR </li>
<li><strong>FULL NAME</strong> : S-LDXR </li>
<li><strong>DESCRIPTION</strong> : S-LDXR is a software for estimating enrichment of stratified squared trans-ethnic genetic correlation across genomic annotations from GWAS summary statistics data. </li>
<li><strong>URL</strong> : <a href="https://huwenboshi.github.io/s-ldxr/">https://huwenboshi.github.io/s-ldxr/</a> </li>
<li><strong>KEYWORDS</strong> : trans-ethnic, stratified, functional categories </li>
<li><strong>CITATION</strong> : Shi, H., Gazal, S., Kanai, M., Koch, E. M., Schoech, A. P., Siewert, K. M., ... &amp; Price, A. L. (2021). Population-specific causal disease effect sizes in functionally important regions impacted by selection. Nature communications, 12(1), 1098. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33597505">33597505</a> </li>
</ul>
<h3 id="cross-trait-ldsc">cross-trait LDSC</h3>
<ul>
<li><strong>NAME</strong> : cross-trait LDSC </li>
<li><strong>SHORT NAME</strong> : cross-trait LDSC </li>
<li><strong>FULL NAME</strong> : cross-trait LD Score Regression </li>
<li><strong>DESCRIPTION</strong> : ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores. </li>
<li><strong>URL</strong> : <a href="https://github.com/bulik/ldsc">https://github.com/bulik/ldsc</a> </li>
<li><strong>KEYWORDS</strong> : cross-trait, LD score regression </li>
<li><strong>CITATION</strong> : Bulik-Sullivan, Brendan, et al. "An atlas of genetic correlations across human diseases and traits." Nature genetics 47.11 (2015): 1236-1241. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26414676">26414676</a> </li>
</ul>
<h3 id="popcorn">popcorn</h3>
<ul>
<li><strong>NAME</strong> : popcorn </li>
<li><strong>SHORT NAME</strong> : popcorn </li>
<li><strong>FULL NAME</strong> : popcorn </li>
<li><strong>DESCRIPTION</strong> : Popcorn is a program for estimaing the correlation of causal variant effect. This is the python3 version of Popcorn and still under development sizes across populations in GWAS. </li>
<li><strong>URL</strong> : <a href="https://github.com/brielin/Popcorn">https://github.com/brielin/Popcorn</a> </li>
<li><strong>KEYWORDS</strong> : trans-ethnic </li>
<li><strong>CITATION</strong> : Brown, Brielin C., et al. "Transethnic genetic-correlation estimates from summary statistics." The American Journal of Human Genetics 99.1 (2016): 76-88. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27321947">27321947</a> </li>
</ul>
<h2 id="heritability">Heritability</h2>
<h3 id="bayesrr-rc">BayesRR-RC</h3>
<ul>
<li><strong>NAME</strong> : BayesRR-RC </li>
<li><strong>SHORT NAME</strong> : BayesRR-RC </li>
<li><strong>FULL NAME</strong> : BayesRR-RC </li>
<li><strong>DESCRIPTION</strong> : gmrm is hybrid-parallel software for a Bayesian grouped mixture of regressions model for genome-wide association studies (GWAS). It is written in C++ using extensive optimisations and code vectorisation. It relies on plink's .bed format. It can handle multiple traits simultaneously. </li>
<li><strong>URL</strong> : <a href="https://github.com/medical-genomics-group/gmrm">https://github.com/medical-genomics-group/gmrm</a> </li>
<li><strong>CITATION</strong> : <br><br>Patxot, M. et al. Probabilistic inference of the genetic architecture underlying functional enrichment of complex traits. Nat. Commun. 12, 6972 (2021).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34848700">34848700</a> </li>
</ul>
<h3 id="gcta-greml-binary">GCTA-GREML-Binary</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Binary </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (case-control) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>CITATION</strong> : Lee, Sang Hong, et al. "Estimating missing heritability for disease from genome-wide association studies." The American Journal of Human Genetics 88.3 (2011): 294-305. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21376301">21376301</a> </li>
</ul>
<h3 id="gcta-greml-bivariate">GCTA-GREML-Bivariate</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Bivariate </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (Bivariate GREML) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>KEYWORDS</strong> : bivariate </li>
<li><strong>CITATION</strong> : Lee, Sang Hong, et al. "Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood." Bioinformatics 28.19 (2012): 2540-2542. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22843982">22843982</a> </li>
</ul>
<h3 id="gcta-greml-ldms">GCTA-GREML-LDMS</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>SHORT NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>FULL NAME</strong> : GCTA-GREML-LDMS </li>
<li><strong>DESCRIPTION</strong> : (GREML-LDMS) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>CITATION</strong> : Yang, Jian, et al. "Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index." Nature genetics 47.10 (2015): 1114-1120. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26323059">26323059</a> </li>
</ul>
<h3 id="gcta-greml-partition">GCTA-GREML-Partition</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Partition </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : (partition the genetic variance into individual chromosomes and genomic segments) </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>CITATION</strong> : Yang, Jian, et al. "Genome partitioning of genetic variation for complex traits using common SNPs." Nature genetics 43.6 (2011): 519-525. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21552263">21552263</a> </li>
</ul>
<h3 id="gcta-greml-quantitative">GCTA-GREML-Quantitative</h3>
<ul>
<li><strong>NAME</strong> : GCTA-GREML-Quantitative </li>
<li><strong>SHORT NAME</strong> : GREML </li>
<li><strong>FULL NAME</strong> : Genome-wide complex trait analysis (GCTA) Genome-based restricted maximum likelihood (GREML) </li>
<li><strong>DESCRIPTION</strong> : GCTA-GREML analysis: estimating the variance explained by the SNPs / GCTA-GREML analysis for a case-control study </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#GREML">https://yanglab.westlake.edu.cn/software/gcta/#GREML</a> </li>
<li><strong>CITATION</strong> : Yang, Jian, et al. "Common SNPs explain a large proportion of the heritability for human height." Nature genetics 42.7 (2010): 565-569.<br><br><br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20562875">20562875</a> </li>
</ul>
<h3 id="hdl">HDL</h3>
<ul>
<li><strong>NAME</strong> : HDL </li>
<li><strong>SHORT NAME</strong> : HDL </li>
<li><strong>FULL NAME</strong> : High-Definition Likelihood </li>
<li><strong>DESCRIPTION</strong> : High-Definition Likelihood (HDL) is a likelihood-based method for estimating genetic correlation using GWAS summary statistics. Compared to LD Score regression (LDSC), It reduces the variance of a genetic correlation estimate by about 60%. </li>
<li><strong>URL</strong> : <a href="https://github.com/zhenin/HDL/">https://github.com/zhenin/HDL/</a> </li>
<li><strong>CITATION</strong> : Ning, Z., Pawitan, Y., &amp; Shen, X. (2020). High-definition likelihood inference of genetic correlations across human complex traits. Nature genetics, 52(8), 859-864. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32601477">32601477</a> </li>
</ul>
<h3 id="ldak">LDAK</h3>
<ul>
<li><strong>NAME</strong> : LDAK </li>
<li><strong>SHORT NAME</strong> : LDAK </li>
<li><strong>FULL NAME</strong> : LD-adjusted kinships </li>
<li><strong>DESCRIPTION</strong> : LDAK is a software package for analysing association study data. </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>CITATION</strong> : <br><br>Speed, D., Hemani, G., Johnson, M. R. &amp; Balding, D. J. Improved heritability estimation from genome-wide SNPs. Am. J. Hum. Genet. 91, 1011–1021 (2012).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23217325">23217325</a> </li>
</ul>
<h3 id="ldsc">LDSC</h3>
<ul>
<li><strong>NAME</strong> : LDSC </li>
<li><strong>SHORT NAME</strong> : LDSC </li>
<li><strong>FULL NAME</strong> : LD Score Regression </li>
<li><strong>DESCRIPTION</strong> : ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores. </li>
<li><strong>URL</strong> : <a href="https://github.com/bulik/ldsc">https://github.com/bulik/ldsc</a> </li>
<li><strong>CITATION</strong> : Bulik-Sullivan, Brendan K., et al. "LD Score regression distinguishes confounding from polygenicity in genome-wide association studies." Nature genetics 47.3 (2015): 291-295. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25642630">25642630</a> </li>
</ul>
<h3 id="sumher">SumHer</h3>
<ul>
<li><strong>NAME</strong> : SumHer </li>
<li><strong>SHORT NAME</strong> : SumHer </li>
<li><strong>FULL NAME</strong> : SumHer </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>CITATION</strong> : <br><br>Speed, D. &amp; Balding, D. J. SumHer better estimates the SNP heritability of complex traits from summary statistics. Nat. Genet. 51, 277–284 (2019).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30510236">30510236</a> </li>
</ul>
<h3 id="tetraher">TetraHer</h3>
<ul>
<li><strong>NAME</strong> : TetraHer </li>
<li><strong>SHORT NAME</strong> : TetraHer </li>
<li><strong>FULL NAME</strong> : TetraHer </li>
<li><strong>DESCRIPTION</strong> : a method for estimating the liability heritability of binary phenotypes </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>CITATION</strong> : <br><br>Speed, D. &amp; Evans, D. M. Estimating disease heritability from complex pedigrees allowing for ascertainment and covariates. Am. J. Hum. Genet. 111, 680–690 (2024).<br><br>  <br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38490208">38490208</a> </li>
</ul>
<h2 id="local-heritabilitygenetic-correlation">Local heritability/genetic correlation</h2>
<h3 id="gnova">GNOVA</h3>
<ul>
<li><strong>NAME</strong> : GNOVA </li>
<li><strong>SHORT NAME</strong> : GNOVA </li>
<li><strong>FULL NAME</strong> : GeNetic cOVariance Analyzer </li>
<li><strong>DESCRIPTION</strong> : A principled framework to estimate annotation-stratified genetic covariance using GWAS summary statistics. </li>
<li><strong>URL</strong> : <a href="https://github.com/xtonyjiang/GNOVA">https://github.com/xtonyjiang/GNOVA</a> </li>
<li><strong>CITATION</strong> : Lu, Qiongshi, et al. "A powerful approach to estimating annotation-stratified genetic covariance via GWAS summary statistics." The American Journal of Human Genetics 101.6 (2017): 939-964. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29220677">29220677</a> </li>
</ul>
<h3 id="heels">HEELS</h3>
<ul>
<li><strong>NAME</strong> : HEELS </li>
<li><strong>SHORT NAME</strong> : HEELS </li>
<li><strong>FULL NAME</strong> : Heritability Estimation with high Efficiency using LD and association Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : HEELS is a Python-based command line tool that produce accurate and precise local heritability estimates using summary-level statistics (marginal association test statistics along with the empirical (in-sample) LD statistics). </li>
<li><strong>URL</strong> : <a href="https://github.com/huilisabrina/HEELS">https://github.com/huilisabrina/HEELS</a> </li>
<li><strong>CITATION</strong> : Li, H., Mazumder, R., &amp; Lin, X. (2023). Accurate and efficient estimation of local heritability using summary statistics and the linkage disequilibrium matrix. Nature Communications, 14(1), 7954. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38040712">38040712</a> </li>
</ul>
<h3 id="hess">HESS</h3>
<ul>
<li><strong>NAME</strong> : HESS </li>
<li><strong>SHORT NAME</strong> : HESS </li>
<li><strong>FULL NAME</strong> : Heritability Estimation from Summary Statistics </li>
<li><strong>DESCRIPTION</strong> : HESS (Heritability Estimation from Summary Statistics) is a software package for estimating and visualizing local SNP-heritability and genetic covariance (correlation) from GWAS summary association data. </li>
<li><strong>URL</strong> : <a href="https://huwenboshi.github.io/hess/">https://huwenboshi.github.io/hess/</a> </li>
<li><strong>CITATION</strong> : Shi, Huwenbo, Gleb Kichaev, and Bogdan Pasaniuc. "Contrasting the genetic architecture of 30 complex traits from summary association data." The American Journal of Human Genetics 99.1 (2016): 139-153.<br><br>Shi, Huwenbo, et al. "Local genetic correlation gives insights into the shared genetic architecture of complex traits." The American Journal of Human Genetics 101.5 (2017): 737-751. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27346688">27346688</a> </li>
</ul>
<h3 id="lava">LAVA</h3>
<ul>
<li><strong>NAME</strong> : LAVA </li>
<li><strong>SHORT NAME</strong> : LAVA </li>
<li><strong>FULL NAME</strong> : Local Analysis of [co]Variant Association </li>
<li><strong>DESCRIPTION</strong> : LAVA is a tool to conduct genome-wide, local genetic correlation analysis on multiple traits, using GWAS summary statistics as input. </li>
<li><strong>URL</strong> : <a href="https://ctg.cncr.nl/software/lava">https://ctg.cncr.nl/software/lava</a> </li>
<li><strong>CITATION</strong> : Werme, J., van der Sluis, S., Posthuma, D. et al. An integrated framework for local genetic correlation analysis. Nat Genet 54, 274–282 (2022). https://doi.org/10.1038/s41588-022-01017-y </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35288712">35288712</a> </li>
</ul>
<h3 id="supergnova">SUPERGNOVA</h3>
<ul>
<li><strong>NAME</strong> : SUPERGNOVA </li>
<li><strong>SHORT NAME</strong> : SUPERGNOVA </li>
<li><strong>FULL NAME</strong> : SUPER GeNetic cOVariance Analyzer </li>
<li><strong>DESCRIPTION</strong> : SUPERGNOVA (SUPER GeNetic cOVariance Analyzer) is a statistical framework to perform local genetic covariance analysis. </li>
<li><strong>URL</strong> : <a href="https://github.com/qlu-lab/SUPERGNOVA">https://github.com/qlu-lab/SUPERGNOVA</a> </li>
<li><strong>CITATION</strong> : Zhang, Yiliang, et al. "SUPERGNOVA: local genetic correlation analysis reveals heterogeneous etiologic sharing of complex traits." Genome biology 22.1 (2021): 1-30. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34493297">34493297</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>