
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Drug_discovery_README/">
      
      
        <link rel="next" href="../Tools_Gene_prioritization_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Fine_mapping - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Fine_mapping
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Reference_data_README/" class="md-tabs__link">
          
  
  Reference_data

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          <label class="md-nav__link" for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cafeh" class="md-nav__link">
    <span class="md-ellipsis">
      CAFEH
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#caviar" class="md-nav__link">
    <span class="md-ellipsis">
      CAVIAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#caviarbf" class="md-nav__link">
    <span class="md-ellipsis">
      CAVIARBF
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#finemap" class="md-nav__link">
    <span class="md-ellipsis">
      FINEMAP
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jam" class="md-nav__link">
    <span class="md-ellipsis">
      JAM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mesusie" class="md-nav__link">
    <span class="md-ellipsis">
      MESuSiE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mscaviar" class="md-nav__link">
    <span class="md-ellipsis">
      MsCAVIAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multisusie" class="md-nav__link">
    <span class="md-ellipsis">
      MultiSuSiE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paintor" class="md-nav__link">
    <span class="md-ellipsis">
      PAINTOR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rfr-susie-inf-finemap-inf" class="md-nav__link">
    <span class="md-ellipsis">
      RFR SuSiE-inf FINEMAP-inf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susie" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susie-rss" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIE-RSS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susiex" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIEx
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sparsepro" class="md-nav__link">
    <span class="md-ellipsis">
      SparsePro
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mjam" class="md-nav__link">
    <span class="md-ellipsis">
      mJAM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mvsusie" class="md-nav__link">
    <span class="md-ellipsis">
      mvSuSiE
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Reference_data
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Allele_Frequency_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Allele_Frequency
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Epigenomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Expression_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Expression
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Gene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            Gene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_Ensembl_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Ensembl
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_GENCODE_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GENCODE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_HGNC_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HGNC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_NCBI_refSeq_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    NCBI_refSeq
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5_6" >
        
          
          <label class="md-nav__link" for="__nav_4_5_6" id="__nav_4_5_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_5_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5_6">
            <span class="md-nav__icon md-icon"></span>
            UCSC_refGene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ucsc_genes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_knownGenes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    knownGenes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_6" >
        
          
          <label class="md-nav__link" for="__nav_4_6" id="__nav_4_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Genome
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_6">
            <span class="md-nav__icon md-icon"></span>
            Genome
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_hg38_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    hg38
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Phenotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_xQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    xQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Karyotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Karyotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Venn_diagram_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Venn_diagram
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Workflow_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Workflow
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cafeh" class="md-nav__link">
    <span class="md-ellipsis">
      CAFEH
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#caviar" class="md-nav__link">
    <span class="md-ellipsis">
      CAVIAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#caviarbf" class="md-nav__link">
    <span class="md-ellipsis">
      CAVIARBF
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#finemap" class="md-nav__link">
    <span class="md-ellipsis">
      FINEMAP
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jam" class="md-nav__link">
    <span class="md-ellipsis">
      JAM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mesusie" class="md-nav__link">
    <span class="md-ellipsis">
      MESuSiE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mr-mega" class="md-nav__link">
    <span class="md-ellipsis">
      MR-MEGA
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mscaviar" class="md-nav__link">
    <span class="md-ellipsis">
      MsCAVIAR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multisusie" class="md-nav__link">
    <span class="md-ellipsis">
      MultiSuSiE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paintor" class="md-nav__link">
    <span class="md-ellipsis">
      PAINTOR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rfr-susie-inf-finemap-inf" class="md-nav__link">
    <span class="md-ellipsis">
      RFR SuSiE-inf FINEMAP-inf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susie" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIE
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susie-rss" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIE-RSS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#susiex" class="md-nav__link">
    <span class="md-ellipsis">
      SUSIEx
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sparsepro" class="md-nav__link">
    <span class="md-ellipsis">
      SparsePro
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mjam" class="md-nav__link">
    <span class="md-ellipsis">
      mJAM
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mvsusie" class="md-nav__link">
    <span class="md-ellipsis">
      mvSuSiE
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Fine_mapping</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CITATION</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#cafeh">CAFEH</a></td>
<td style="text-align: left;">Arvanitis, M., Tayeb, K., Strober, B. J., &amp; Battle, A. (2022). Redefining tissue specificity of genetic regulation of gene expression in the presence of allelic heterogeneity. The American Journal of Human Genetics, 109(2), 223-239.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#caviarbf">CAVIARBF</a></td>
<td style="text-align: left;">Chen, W., Larrabee, B. R., Ovsyannikova, I. G., Kennedy, R. B., Haralambieva, I. H., Poland, G. A., &amp; Schaid, D. J. (2015). Fine mapping causal variants with an approximate Bayesian method using marginal test statistics. Genetics, 200(3), 719-736.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#caviar">CAVIAR</a></td>
<td style="text-align: left;">Hormozdiari, F., Kostem, E., Kang, E. Y., Pasaniuc, B., &amp; Eskin, E. (2014). Identifying causal variants at loci with multiple signals of association. Genetics, 198(2), 497-508.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#finemap">FINEMAP</a></td>
<td style="text-align: left;">Benner, C., Spencer, C. C., Havulinna, A. S., Salomaa, V., Ripatti, S., &amp; Pirinen, M. (2016). FINEMAP: efficient variable selection using summary data from genome-wide association studies. Bioinformatics, 32(10), 1493-1501.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#jam">JAM</a></td>
<td style="text-align: left;">Newcombe, Paul J., David V. Conti, and Sylvia Richardson. "JAM: a scalable Bayesian framework for joint analysis of marginal SNP effects." Genetic epidemiology 40.3 (2016): 188-201.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mesusie">MESuSiE</a></td>
<td style="text-align: left;">Gao, B., Zhou, X. MESuSiE enables scalable and powerful multi-ancestry fine-mapping of causal variants in genome-wide association studies. Nat Genet (2024). https://doi.org/10.1038/s41588-023-01604-7</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mr-mega">MR-MEGA</a></td>
<td style="text-align: left;">Mägi, R., Horikoshi, M., Sofer, T., Mahajan, A., Kitajima, H., Franceschini, N., ... &amp; Morris, A. P. (2017). Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Human molecular genetics, 26(18), 3639-3650.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mscaviar">MsCAVIAR</a></td>
<td style="text-align: left;">LaPierre, N., Taraszka, K., Huang, H., He, R., Hormozdiari, F., &amp; Eskin, E. (2021). Identifying causal variants by fine mapping across multiple studies. PLoS genetics, 17(9), e1009733.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multisusie">MultiSuSiE</a></td>
<td style="text-align: left;">Rossen, J. et al. MultiSuSiE improves multi-ancestry fine-mapping in All of Us whole-genome sequencing data. medRxiv 2024.05.13.24307291 (2024) doi:10.1101/2024.05.13.24307291</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#paintor">PAINTOR</a></td>
<td style="text-align: left;">Kichaev, G., Yang, W. Y., Lindstrom, S., Hormozdiari, F., Eskin, E., Price, A. L., ... &amp; Pasaniuc, B. (2014). Integrating functional data to prioritize causal variants in statistical fine-mapping studies. PLoS genetics, 10(10), e1004722.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#rfr susie-inf finemap-inf">RFR SuSiE-inf FINEMAP-inf</a></td>
<td style="text-align: left;">Cui, R., Elzur, R. A., Kanai, M., Ulirsch, J. C., Weissbrod, O., Daly, M., ... &amp; Finucane, H. K. (2022). Improving fine-mapping by modeling infinitesimal effects. bioRxiv.<br><br>Cui, R., Elzur, R. A., Kanai, M., Ulirsch, J. C., Weissbrod, O., Daly, M. J., ... &amp; Finucane, H. K. (2023). Improving fine-mapping by modeling infinitesimal effects. Nature Genetics, 1-8.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#susie-rss">SUSIE-RSS</a></td>
<td style="text-align: left;">Zou, Yuxin, et al. "Fine-mapping from summary data with the “Sum of Single Effects” model." PLoS Genetics 18.7 (2022): e1010299.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#susie">SUSIE</a></td>
<td style="text-align: left;">Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. (2020). A simple new approach to variable selection in regression, with application to genetic fine mapping. Journal of the Royal Statistical Society, Series B 82, 1273–1300. https://doi.org/10.1111/rssb.12388</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#susiex">SUSIEx</a></td>
<td style="text-align: left;">https://www.medrxiv.org/content/10.1101/2023.01.07.23284293v2</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sparsepro">SparsePro</a></td>
<td style="text-align: left;">Zhang, W., Najafabadi, H. S., &amp; Li, Y. (2021). SparsePro: an efficient genome-wide fine-mapping method integrating summary statistics and functional annotations. bioRxiv.<br><br>Zhang, W., Najafabadi, H., &amp; Li, Y. (2023). SparsePro: An efficient fine-mapping method integrating summary statistics and functional annotations. PLoS genetics, 19(12), e1011104.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mjam">mJAM</a></td>
<td style="text-align: left;">Shen, J., Jiang, L., Wang, K., Wang, A., Chen, F., Newcombe, P. J., ... &amp; Conti, D. V. (2022). Fine-mapping and credible set construction using a multi-population joint analysis of marginal summary statistics from genome-wide association studies. bioRxiv, 2022-12.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mvsusie">mvSuSiE</a></td>
<td style="text-align: left;">Zou, Y., Carbonetto, P., Xie, D., Wang, G., &amp; Stephens, M. (2023). Fast and flexible joint fine-mapping of multiple traits via the Sum of Single Effects model. bioRxiv, 2023-04.</td>
</tr>
</tbody>
</table>
<h2 id="cafeh">CAFEH</h2>
<ul>
<li>NAME : CAFEH </li>
<li>SHORT NAME : CAFEH </li>
<li>FULL NAME : colocalization and fine-mapping in the presence of allelic heterogeneity </li>
<li>YEAR : 2022 </li>
<li>DESCRIPTION : CAFEH is a method that performs finemapping and colocalization jointly over multiple phenotypes. CAFEH can be run with 10s of phenotypes and 1000s of variants in a few minutes. </li>
<li>URL : https://github.com/karltayeb/cafeh </li>
<li>CITATION : Arvanitis, M., Tayeb, K., Strober, B. J., &amp; Battle, A. (2022). Redefining tissue specificity of genetic regulation of gene expression in the presence of allelic heterogeneity. The American Journal of Human Genetics, 109(2), 223-239. </li>
<li>KEYWORDS : multi-trait, finemapping, colocalization </li>
</ul>
<h2 id="caviar">CAVIAR</h2>
<ul>
<li>NAME : CAVIAR </li>
<li>SHORT NAME : CAVIAR </li>
<li>FULL NAME : causal variants identification in associated regions </li>
<li>YEAR : 2014 </li>
<li>DESCRIPTION : a statistical framework that quantifies the probability of each variant to be causal while allowing an arbitrary number of causal variants. </li>
<li>URL : http://genetics.cs.ucla.edu/caviar/ </li>
<li>CITATION : Hormozdiari, F., Kostem, E., Kang, E. Y., Pasaniuc, B., &amp; Eskin, E. (2014). Identifying causal variants at loci with multiple signals of association. Genetics, 198(2), 497-508. </li>
</ul>
<h2 id="caviarbf">CAVIARBF</h2>
<ul>
<li>NAME : CAVIARBF </li>
<li>SHORT NAME : CAVIARBF </li>
<li>FULL NAME : CAVIAR Bayes factor </li>
<li>YEAR : 2015 </li>
<li>DESCRIPTION : a fine-mapping method using marginal test statistics in the Bayesian framework </li>
<li>URL : https://bitbucket.org/Wenan/caviarbf/src/master/ </li>
<li>CITATION : Chen, W., Larrabee, B. R., Ovsyannikova, I. G., Kennedy, R. B., Haralambieva, I. H., Poland, G. A., &amp; Schaid, D. J. (2015). Fine mapping causal variants with an approximate Bayesian method using marginal test statistics. Genetics, 200(3), 719-736. </li>
<li>KEYWORDS : Bayes factor </li>
</ul>
<h2 id="finemap">FINEMAP</h2>
<ul>
<li>NAME : FINEMAP </li>
<li>SHORT NAME : FINEMAP </li>
<li>FULL NAME : FINEMAP </li>
<li>YEAR : 2016 </li>
<li>DESCRIPTION : FINEMAP is a program for 1.identifying causal SNPs, 2. estimating effect sizes of causal SNPs, 3 estimating the heritability contribution of causal SNPs </li>
<li>URL : http://www.christianbenner.com/ </li>
<li>CITATION : Benner, C., Spencer, C. C., Havulinna, A. S., Salomaa, V., Ripatti, S., &amp; Pirinen, M. (2016). FINEMAP: efficient variable selection using summary data from genome-wide association studies. Bioinformatics, 32(10), 1493-1501. </li>
<li>KEYWORDS : Shotgun Stochastic Search (SSS) </li>
</ul>
<h2 id="jam">JAM</h2>
<ul>
<li>NAME : JAM </li>
<li>SHORT NAME : JAM </li>
<li>FULL NAME : joint analysis of marginal summary statistics </li>
<li>YEAR : 2016 </li>
<li>DESCRIPTION : Bayesian variable selection under a range of likelihoods, including linear regression for continuous outcomes, logistic regression for binary outcomes, Weibull regression for survival outcomes binary and survial outcomes, and the "JAM" model for summary genetic association data. </li>
<li>URL : https://github.com/pjnewcombe/R2BGLiMS </li>
<li>CITATION : Newcombe, Paul J., David V. Conti, and Sylvia Richardson. "JAM: a scalable Bayesian framework for joint analysis of marginal SNP effects." Genetic epidemiology 40.3 (2016): 188-201. </li>
</ul>
<h2 id="mesusie">MESuSiE</h2>
<ul>
<li>NAME : MESuSiE </li>
<li>SHORT NAME : MESuSiE </li>
<li>FULL NAME : multi-ancestry sum of the single effects model </li>
<li>YEAR : 2024 </li>
<li>DESCRIPTION : MESuSiE relies on GWAS summary statistics from multiple ancestries, properly accounts for the LD structure of the local genomic region in multiple ancestries, and explicitly models both shared and ancestry-specific causal signals to accommodate causal effect size similarity as well as heterogeneity across ancestries. MESuSiE outputs posterior inclusion probability of variant being shared or ancestry-specific causal variants. </li>
<li>URL : https://github.com/borangao/MESuSiE </li>
<li>CITATION : Gao, B., Zhou, X. MESuSiE enables scalable and powerful multi-ancestry fine-mapping of causal variants in genome-wide association studies. Nat Genet (2024). https://doi.org/10.1038/s41588-023-01604-7 </li>
<li>KEYWORDS : multi-trait, fine-mapping </li>
</ul>
<h2 id="mr-mega">MR-MEGA</h2>
<ul>
<li>NAME : MR-MEGA </li>
<li>SHORT NAME : MR-MEGA </li>
<li>FULL NAME : Meta-Regression of Multi-AncEstry Genetic Association </li>
<li>YEAR : 2017 </li>
<li>DESCRIPTION : MR-MEGA (Meta-Regression of Multi-AncEstry Genetic Association) is a tool to detect and fine-map complex trait association signals via multi-ancestry meta-regression.  This approach uses genome-wide metrics of diversity between populations to derive axes of genetic variation via multi-dimensional scaling [Purcell 2007].  Allelic effects of a variant across GWAS, weighted by their corresponding standard errors, can then be modelled in a linear regression framework, including the axes of genetic variation as covariates.  The flexibility of this model enables partitioning of the heterogeneity into components due to ancestry and residual variation, which would be expected to improve fine-mapping resolution. </li>
<li>URL : https://genomics.ut.ee/en/tools </li>
<li>CITATION : Mägi, R., Horikoshi, M., Sofer, T., Mahajan, A., Kitajima, H., Franceschini, N., ... &amp; Morris, A. P. (2017). Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Human molecular genetics, 26(18), 3639-3650. </li>
<li>KEYWORDS : Multi-AncEstry </li>
</ul>
<h2 id="mscaviar">MsCAVIAR</h2>
<ul>
<li>NAME : MsCAVIAR </li>
<li>SHORT NAME : MsCAVIAR </li>
<li>FULL NAME : multiple study causal variants identification in associated regions </li>
<li>YEAR : 2021 </li>
<li>DESCRIPTION : MsCAVIAR is a method for fine-mapping (identifying causal variants among GWAS associated variants) by leveraging information from multiple studies. One important application area is trans-ethnic fine mapping. </li>
<li>URL : https://github.com/nlapier2/MsCAVIAR </li>
<li>CITATION : LaPierre, N., Taraszka, K., Huang, H., He, R., Hormozdiari, F., &amp; Eskin, E. (2021). Identifying causal variants by fine mapping across multiple studies. PLoS genetics, 17(9), e1009733. </li>
<li>KEYWORDS : multi-study finemapping </li>
</ul>
<h2 id="multisusie">MultiSuSiE</h2>
<ul>
<li>NAME : MultiSuSiE </li>
<li>SHORT NAME : MultiSuSiE </li>
<li>FULL NAME : MultiSuSiE </li>
<li>YEAR : 2024 </li>
<li>DESCRIPTION : MultiSuSiE is a multi-ancestry extension of the Sum of Single Effects model (Wang et al. 2020 J. R. Statist. Soc. B, Zou et al. 2022 PLoS Genet.) implemented in Python. </li>
<li>URL : https://github.com/jordanero/MultiSuSiE </li>
<li>CITATION : Rossen, J. et al. MultiSuSiE improves multi-ancestry fine-mapping in All of Us whole-genome sequencing data. medRxiv 2024.05.13.24307291 (2024) doi:10.1101/2024.05.13.24307291 </li>
<li>KEYWORDS : cross-ancestry, fine-mapping </li>
</ul>
<h2 id="paintor">PAINTOR</h2>
<ul>
<li>NAME : PAINTOR </li>
<li>SHORT NAME : PAINTOR </li>
<li>FULL NAME : Probabilistic Annotation INtegraTOR </li>
<li>YEAR : 2014 </li>
<li>DESCRIPTION : Finding causal variants that underlie known risk loci is one of the main post-GWAS challenges. Here we present PAINTOR (Probabilistic Annotation INtegraTOR), a probabilistic framework that integrates association strength with genomic functional annotation data to improve accuracy in selecting plausible causal variants for functional validation. The main output of PAINTOR are probabilities for every variant to be causal that can be used for prioritization in functional assays to establish biological causality. </li>
<li>URL : https://bogdan.dgsom.ucla.edu/pages/paintor/ </li>
<li>CITATION : Kichaev, G., Yang, W. Y., Lindstrom, S., Hormozdiari, F., Eskin, E., Price, A. L., ... &amp; Pasaniuc, B. (2014). Integrating functional data to prioritize causal variants in statistical fine-mapping studies. PLoS genetics, 10(10), e1004722. </li>
<li>KEYWORDS : Empirical Bayes prior </li>
</ul>
<h2 id="rfr-susie-inf-finemap-inf">RFR SuSiE-inf FINEMAP-inf</h2>
<ul>
<li>NAME : RFR SuSiE-inf FINEMAP-inf </li>
<li>SHORT NAME : RFR </li>
<li>FULL NAME : Replication Failure Rate </li>
<li>YEAR : 2022 </li>
<li>DESCRIPTION : Replication Failure Rate (RFR), a metric to assess the consistency of fine-mapping results based on downsampling a large cohort. SuSiE-inf and FINEMAP-inf, that extend SuSiE and FINEMAP to incorporate a term for infinitesimal effects in addition to a small number of larger causal effects of interest. </li>
<li>URL : https://github.com/FinucaneLab/fine-mapping-inf </li>
<li>CITATION : Cui, R., Elzur, R. A., Kanai, M., Ulirsch, J. C., Weissbrod, O., Daly, M., ... &amp; Finucane, H. K. (2022). Improving fine-mapping by modeling infinitesimal effects. bioRxiv.
Cui, R., Elzur, R. A., Kanai, M., Ulirsch, J. C., Weissbrod, O., Daly, M. J., ... &amp; Finucane, H. K. (2023). Improving fine-mapping by modeling infinitesimal effects. Nature Genetics, 1-8. </li>
</ul>
<h2 id="susie">SUSIE</h2>
<ul>
<li>NAME : SUSIE </li>
<li>SHORT NAME : SUSIE </li>
<li>FULL NAME : sum of single effects </li>
<li>YEAR : 2020 </li>
<li>DESCRIPTION : The susieR package implements a simple new way to perform variable selection in multiple regression (y = Xb + e). The methods implemented here are particularly well-suited to settings where some of the X variables are highly correlated, and the true effects are highly sparse (e.g. &lt;20 non-zero effects in the vector b). One example of this is genetic fine-mapping applications, and this application was a major motivation for developing these methods. </li>
<li>URL : https://stephenslab.github.io/susieR/index.html </li>
<li>CITATION : Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. (2020). A simple new approach to variable selection in regression, with application to genetic fine mapping. Journal of the Royal Statistical Society, Series B 82, 1273–1300. https://doi.org/10.1111/rssb.12388 </li>
<li>KEYWORDS : fine-mapping, sum of single-effects (SuSiE) regression, iterative Bayesian stepwise selection (IBSS) </li>
</ul>
<h2 id="susie-rss">SUSIE-RSS</h2>
<ul>
<li>NAME : SUSIE-RSS </li>
<li>SHORT NAME : SUSIE-RSS </li>
<li>FULL NAME : sum of single effects regression with summary statistics </li>
<li>YEAR : 2022 </li>
<li>DESCRIPTION : The susieR package implements a simple new way to perform variable selection in multiple regression (y = Xb + e). The methods implemented here are particularly well-suited to settings where some of the X variables are highly correlated, and the true effects are highly sparse (e.g. &lt;20 non-zero effects in the vector b). One example of this is genetic fine-mapping applications, and this application was a major motivation for developing these methods. </li>
<li>URL : https://stephenslab.github.io/susieR/index.html </li>
<li>CITATION : Zou, Yuxin, et al. "Fine-mapping from summary data with the “Sum of Single Effects” model." PLoS Genetics 18.7 (2022): e1010299. </li>
<li>KEYWORDS : fine-mapping, summary statistics </li>
</ul>
<h2 id="susiex">SUSIEx</h2>
<ul>
<li>NAME : SUSIEx </li>
<li>SHORT NAME : SUSIEx </li>
<li>FULL NAME : SUSIEx </li>
<li>YEAR : 2023 </li>
<li>DESCRIPTION : SuSiEx is a Python based command line tool that performs cross-ethnic fine-mapping using GWAS summary statistics and LD reference panels. The method is built on the Sum of Single Effects (SuSiE) model. </li>
<li>URL : https://github.com/getian107/SuSiEx </li>
<li>CITATION : https://www.medrxiv.org/content/10.1101/2023.01.07.23284293v2 </li>
<li>KEYWORDS : cross-ancestry, fine-mapping </li>
</ul>
<h2 id="sparsepro">SparsePro</h2>
<ul>
<li>NAME : SparsePro </li>
<li>SHORT NAME : SparsePro </li>
<li>FULL NAME : SparsePro </li>
<li>YEAR : 2021 </li>
<li>DESCRIPTION : SparsePro is a command line tool for efficiently conducting genome-wide fine-mapping. Our method has two key features: First, by creating a sparse low-dimensional projection of the high-dimensional genotype, we enable a linear search of causal variants instead of an exponential search of causal configurations in most existing methods; Second, we adopt a probabilistic framework with a highly efficient variational expectation-maximization algorithm to integrate statistical associations and functional priors. </li>
<li>URL : https://github.com/zhwm/SparsePro </li>
<li>CITATION : Zhang, W., Najafabadi, H. S., &amp; Li, Y. (2021). SparsePro: an efficient genome-wide fine-mapping method integrating summary statistics and functional annotations. bioRxiv.
Zhang, W., Najafabadi, H., &amp; Li, Y. (2023). SparsePro: An efficient fine-mapping method integrating summary statistics and functional annotations. PLoS genetics, 19(12), e1011104. </li>
</ul>
<h2 id="mjam">mJAM</h2>
<ul>
<li>NAME : mJAM </li>
<li>SHORT NAME : mJAM </li>
<li>FULL NAME : multi-population JAM </li>
<li>YEAR : 2022 </li>
<li>URL : https://github.com/USCbiostats/hJAM/R </li>
<li>CITATION : Shen, J., Jiang, L., Wang, K., Wang, A., Chen, F., Newcombe, P. J., ... &amp; Conti, D. V. (2022). Fine-mapping and credible set construction using a multi-population joint analysis of marginal summary statistics from genome-wide association studies. bioRxiv, 2022-12. </li>
<li>KEYWORDS : multi-population </li>
</ul>
<h2 id="mvsusie">mvSuSiE</h2>
<ul>
<li>NAME : mvSuSiE </li>
<li>SHORT NAME : mvSuSiE </li>
<li>FULL NAME : mvSuSiE </li>
<li>YEAR : 2023 </li>
<li>DESCRIPTION : Implements a multivariate generalization of the "Sum of Single Effects" (SuSiE) model for variable selection in multivariate linear regression. </li>
<li>URL : https://github.com/stephenslab/mvsusieR </li>
<li>CITATION : Zou, Y., Carbonetto, P., Xie, D., Wang, G., &amp; Stephens, M. (2023). Fast and flexible joint fine-mapping of multiple traits via the Sum of Single Effects model. bioRxiv, 2023-04. </li>
<li>KEYWORDS : multi-trait, fine-mapping </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>