
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Metabolomics_Metabolomics_README/">
      
      
        <link rel="next" href="../Imaging_Imaging_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>Proteomics - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Proteomics
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comparison" class="md-nav__link">
    <span class="md-ellipsis">
      Comparison
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Comparison">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#eldjarn_gh-37794188" class="md-nav__link">
    <span class="md-ellipsis">
      Eldjarn_GH-37794188
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortium" class="md-nav__link">
    <span class="md-ellipsis">
      Consortium
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortium">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      SCALLOP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ahola-olli_av-27989323" class="md-nav__link">
    <span class="md-ellipsis">
      Ahola-Olli_AV-27989323
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bretherick_ad-32628676" class="md-nav__link">
    <span class="md-ellipsis">
      Bretherick_AD-32628676
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carayol_j-29234017" class="md-nav__link">
    <span class="md-ellipsis">
      Carayol_J-29234017
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carland_c-37550624" class="md-nav__link">
    <span class="md-ellipsis">
      Carland_C-37550624
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caron_b-35264221" class="md-nav__link">
    <span class="md-ellipsis">
      Caron_B-35264221
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deming_y-28247064" class="md-nav__link">
    <span class="md-ellipsis">
      Deming_Y-28247064
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dhindsa_rs-37794183" class="md-nav__link">
    <span class="md-ellipsis">
      Dhindsa_RS-37794183
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emilsson_v-30072576" class="md-nav__link">
    <span class="md-ellipsis">
      Emilsson_V-30072576
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enroth_s-25147954" class="md-nav__link">
    <span class="md-ellipsis">
      Enroth_S-25147954
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ferkingstad_e-34857953" class="md-nav__link">
    <span class="md-ellipsis">
      Ferkingstad_E-34857953
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#folkersen_l-28369058" class="md-nav__link">
    <span class="md-ellipsis">
      Folkersen_L-28369058
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#folkersen_l-33067605" class="md-nav__link">
    <span class="md-ellipsis">
      Folkersen_L-33067605
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gilly_a-37778719" class="md-nav__link">
    <span class="md-ellipsis">
      Gilly_A-37778719
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gudjonsson_a-35078996" class="md-nav__link">
    <span class="md-ellipsis">
      Gudjonsson_A-35078996
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hansson_o-36504281" class="md-nav__link">
    <span class="md-ellipsis">
      Hansson_O-36504281
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hillary_rf-31320639" class="md-nav__link">
    <span class="md-ellipsis">
      Hillary_RF-31320639
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hillary_rf-32641083" class="md-nav__link">
    <span class="md-ellipsis">
      Hillary_RF-32641083
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#johansson_a-23487758" class="md-nav__link">
    <span class="md-ellipsis">
      Johansson_Å-23487758
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#katz_dh-34814699" class="md-nav__link">
    <span class="md-ellipsis">
      Katz_DH-34814699
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#katz_dh-35984888" class="md-nav__link">
    <span class="md-ellipsis">
      Katz_DH-35984888
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kauwe_js-25340798" class="md-nav__link">
    <span class="md-ellipsis">
      Kauwe_JS-25340798
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kim_s-23894628" class="md-nav__link">
    <span class="md-ellipsis">
      Kim_S-23894628
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koprulu_m-36823471" class="md-nav__link">
    <span class="md-ellipsis">
      Koprulu_M-36823471
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macdonald-dunlop" class="md-nav__link">
    <span class="md-ellipsis">
      Macdonald-Dunlop
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#melzer_d-18464913" class="md-nav__link">
    <span class="md-ellipsis">
      Melzer_D-18464913
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pietzner_m-34648354" class="md-nav__link">
    <span class="md-ellipsis">
      Pietzner_M-34648354
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#png_g-34857772" class="md-nav__link">
    <span class="md-ellipsis">
      Png_G-34857772
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruffieux_h-32492067" class="md-nav__link">
    <span class="md-ellipsis">
      Ruffieux_H-32492067
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#said" class="md-nav__link">
    <span class="md-ellipsis">
      Said
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sasayama_d-28031287" class="md-nav__link">
    <span class="md-ellipsis">
      Sasayama_D-28031287
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre_k-28240269" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre_K-28240269
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre_k-38412862" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre_K-38412862
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_bb-29875488" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_BB-29875488
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_bb-37794186" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_BB-37794186
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_w-27532455" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_W-27532455
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#surapaneni_a-35870639" class="md-nav__link">
    <span class="md-ellipsis">
      Surapaneni_A-35870639
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#thareja_g-36168886" class="md-nav__link">
    <span class="md-ellipsis">
      Thareja_G-36168886
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu_f-36797296" class="md-nav__link">
    <span class="md-ellipsis">
      Xu_F-36797296
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang_c-34239129" class="md-nav__link">
    <span class="md-ellipsis">
      Yang_C-34239129
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yao_c-30111768" class="md-nav__link">
    <span class="md-ellipsis">
      Yao_C-30111768
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zhong_w-32576278" class="md-nav__link">
    <span class="md-ellipsis">
      Zhong_W-32576278
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#krishna_c-39085222" class="md-nav__link">
    <span class="md-ellipsis">
      Krishna_C-39085222
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#omicspred-portal" class="md-nav__link">
    <span class="md-ellipsis">
      OmicsPred portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proteome-phewas-browser" class="md-nav__link">
    <span class="md-ellipsis">
      Proteome PheWAS browser
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgwas-server" class="md-nav__link">
    <span class="md-ellipsis">
      pGWAS server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-suhre_k-32860016" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Suhre_K-32860016
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comparison" class="md-nav__link">
    <span class="md-ellipsis">
      Comparison
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Comparison">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#eldjarn_gh-37794188" class="md-nav__link">
    <span class="md-ellipsis">
      Eldjarn_GH-37794188
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#consortium" class="md-nav__link">
    <span class="md-ellipsis">
      Consortium
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Consortium">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      SCALLOP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ahola-olli_av-27989323" class="md-nav__link">
    <span class="md-ellipsis">
      Ahola-Olli_AV-27989323
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bretherick_ad-32628676" class="md-nav__link">
    <span class="md-ellipsis">
      Bretherick_AD-32628676
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carayol_j-29234017" class="md-nav__link">
    <span class="md-ellipsis">
      Carayol_J-29234017
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#carland_c-37550624" class="md-nav__link">
    <span class="md-ellipsis">
      Carland_C-37550624
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#caron_b-35264221" class="md-nav__link">
    <span class="md-ellipsis">
      Caron_B-35264221
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#deming_y-28247064" class="md-nav__link">
    <span class="md-ellipsis">
      Deming_Y-28247064
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dhindsa_rs-37794183" class="md-nav__link">
    <span class="md-ellipsis">
      Dhindsa_RS-37794183
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emilsson_v-30072576" class="md-nav__link">
    <span class="md-ellipsis">
      Emilsson_V-30072576
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enroth_s-25147954" class="md-nav__link">
    <span class="md-ellipsis">
      Enroth_S-25147954
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ferkingstad_e-34857953" class="md-nav__link">
    <span class="md-ellipsis">
      Ferkingstad_E-34857953
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#folkersen_l-28369058" class="md-nav__link">
    <span class="md-ellipsis">
      Folkersen_L-28369058
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#folkersen_l-33067605" class="md-nav__link">
    <span class="md-ellipsis">
      Folkersen_L-33067605
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gilly_a-37778719" class="md-nav__link">
    <span class="md-ellipsis">
      Gilly_A-37778719
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gudjonsson_a-35078996" class="md-nav__link">
    <span class="md-ellipsis">
      Gudjonsson_A-35078996
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hansson_o-36504281" class="md-nav__link">
    <span class="md-ellipsis">
      Hansson_O-36504281
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hillary_rf-31320639" class="md-nav__link">
    <span class="md-ellipsis">
      Hillary_RF-31320639
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#hillary_rf-32641083" class="md-nav__link">
    <span class="md-ellipsis">
      Hillary_RF-32641083
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#johansson_a-23487758" class="md-nav__link">
    <span class="md-ellipsis">
      Johansson_Å-23487758
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#katz_dh-34814699" class="md-nav__link">
    <span class="md-ellipsis">
      Katz_DH-34814699
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#katz_dh-35984888" class="md-nav__link">
    <span class="md-ellipsis">
      Katz_DH-35984888
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kauwe_js-25340798" class="md-nav__link">
    <span class="md-ellipsis">
      Kauwe_JS-25340798
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kim_s-23894628" class="md-nav__link">
    <span class="md-ellipsis">
      Kim_S-23894628
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#koprulu_m-36823471" class="md-nav__link">
    <span class="md-ellipsis">
      Koprulu_M-36823471
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#macdonald-dunlop" class="md-nav__link">
    <span class="md-ellipsis">
      Macdonald-Dunlop
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#melzer_d-18464913" class="md-nav__link">
    <span class="md-ellipsis">
      Melzer_D-18464913
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pietzner_m-34648354" class="md-nav__link">
    <span class="md-ellipsis">
      Pietzner_M-34648354
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#png_g-34857772" class="md-nav__link">
    <span class="md-ellipsis">
      Png_G-34857772
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ruffieux_h-32492067" class="md-nav__link">
    <span class="md-ellipsis">
      Ruffieux_H-32492067
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#said" class="md-nav__link">
    <span class="md-ellipsis">
      Said
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sasayama_d-28031287" class="md-nav__link">
    <span class="md-ellipsis">
      Sasayama_D-28031287
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre_k-28240269" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre_K-28240269
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#suhre_k-38412862" class="md-nav__link">
    <span class="md-ellipsis">
      Suhre_K-38412862
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_bb-29875488" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_BB-29875488
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_bb-37794186" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_BB-37794186
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun_w-27532455" class="md-nav__link">
    <span class="md-ellipsis">
      Sun_W-27532455
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#surapaneni_a-35870639" class="md-nav__link">
    <span class="md-ellipsis">
      Surapaneni_A-35870639
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#thareja_g-36168886" class="md-nav__link">
    <span class="md-ellipsis">
      Thareja_G-36168886
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu_f-36797296" class="md-nav__link">
    <span class="md-ellipsis">
      Xu_F-36797296
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yang_c-34239129" class="md-nav__link">
    <span class="md-ellipsis">
      Yang_C-34239129
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yao_c-30111768" class="md-nav__link">
    <span class="md-ellipsis">
      Yao_C-30111768
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#zhong_w-32576278" class="md-nav__link">
    <span class="md-ellipsis">
      Zhong_W-32576278
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hla-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      HLA-GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="HLA-GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#krishna_c-39085222" class="md-nav__link">
    <span class="md-ellipsis">
      Krishna_C-39085222
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#omicspred-portal" class="md-nav__link">
    <span class="md-ellipsis">
      OmicsPred portal
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#proteome-phewas-browser" class="md-nav__link">
    <span class="md-ellipsis">
      Proteome PheWAS browser
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgwas-server" class="md-nav__link">
    <span class="md-ellipsis">
      pGWAS server
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-suhre_k-32860016" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Suhre_K-32860016
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Proteomics</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">PLATFORM</th>
<th style="text-align: left;">YEAR</th>
<th style="text-align: left;">TITLE</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#eldjarngh-37794188">Eldjarn_GH-37794188</a></td>
<td style="text-align: left;">Comparison</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Large-scale plasma proteomics comparisons through genetics and disease associations.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#scallop">SCALLOP</a></td>
<td style="text-align: left;">Consortium</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ahola-olliav-27989323">Ahola-Olli_AV-27989323</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Genome-wide Association Study Identifies 27 Loci Influencing Concentrations of Circulating Cytokines and Growth Factors.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bretherickad-32628676">Bretherick_AD-32628676</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Linking protein to phenotype with Mendelian Randomization detects 38 proteins with causal roles in human diseases and traits.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#carayolj-29234017">Carayol_J-29234017</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Protein quantitative trait locus study in obesity during weight-loss identifies a leptin regulator.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#carlandc-37550624">Carland_C-37550624</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#caronb-35264221">Caron_B-35264221</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#demingy-28247064">Deming_Y-28247064</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Genome-wide association study identifies four novel loci associated with Alzheimer's endophenotypes and disease modifiers.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#dhindsars-37794183">Dhindsa_RS-37794183</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Rare variant associations with plasma protein levels in the UK Biobank.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#emilssonv-30072576">Emilsson_V-30072576</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2018</td>
<td style="text-align: left;">Co-regulatory networks of human serum proteins link genetics to disease.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#enroths-25147954">Enroth_S-25147954</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2014</td>
<td style="text-align: left;">Strong effects of genetic and lifestyle factors on biomarker variation and use of personalized cutoffs.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ferkingstade-34857953">Ferkingstad_E-34857953</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Large-scale integration of the plasma proteome with genetics and disease.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#folkersenl-28369058">Folkersen_L-28369058</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Mapping of 79 loci for 83 plasma protein biomarkers in cardiovascular disease.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#folkersenl-33067605">Folkersen_L-33067605</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Genomic and drug target evaluation of 90 cardiovascular proteins in 30,931 individuals.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gillya-37778719">Gilly_A-37778719</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Genome-wide meta-analysis of 92 cardiometabolic protein serum levels.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gudjonssona-35078996">Gudjonsson_A-35078996</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">A genome-wide association study of serum proteins reveals shared loci with common diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hanssono-36504281">Hansson_O-36504281</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">The genetic regulation of protein expression in cerebrospinal fluid.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hillaryrf-31320639">Hillary_RF-31320639</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2019</td>
<td style="text-align: left;">Genome and epigenome wide studies of neurological protein biomarkers in the Lothian Birth Cohort 1936.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#hillaryrf-32641083">Hillary_RF-32641083</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Multi-method genome- and epigenome-wide studies of inflammatory protein levels in healthy older adults.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#johansson-23487758">Johansson_Å-23487758</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2013</td>
<td style="text-align: left;">Identification of genetic variants influencing the human plasma proteome.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#katzdh-34814699">Katz_DH-34814699</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">Whole Genome Sequence Analysis of the Plasma Proteome in Black Adults Provides Novel Insights Into Cardiovascular Disease.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#katzdh-35984888">Katz_DH-35984888</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">Proteomic profiling platforms head to head: Leveraging genetics and clinical traits to compare aptamer- and antibody-based methods.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#kauwejs-25340798">Kauwe_JS-25340798</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2014</td>
<td style="text-align: left;">Genome-wide association study of CSF levels of 59 alzheimer's disease candidate proteins: significant associations with proteins involved in amyloid processing and inflammation.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#kims-23894628">Kim_S-23894628</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2013</td>
<td style="text-align: left;">Influence of genetic variation on plasma protein levels in older adults using a multi-analyte panel.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#koprulum-36823471">Koprulu_M-36823471</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Proteogenomic links to human metabolic diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#macdonald-dunlop">Macdonald-Dunlop</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#melzerd-18464913">Melzer_D-18464913</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2008</td>
<td style="text-align: left;">A genome-wide association study identifies protein quantitative trait loci (pQTLs).</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pietznerm-34648354">Pietzner_M-34648354</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Mapping the proteo-genomic convergence of human diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pngg-34857772">Png_G-34857772</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Mapping the serum proteome to neurological diseases using whole genome sequencing.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ruffieuxh-32492067">Ruffieux_H-32492067</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">A fully joint Bayesian quantitative trait locus mapping of human protein abundance in plasma.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#said">Said</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sasayamad-28031287">Sasayama_D-28031287</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Genome-wide quantitative trait loci mapping of the human cerebrospinal fluid proteome.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#suhrek-28240269">Suhre_K-28240269</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Connecting genetic risk to disease end points through the human blood plasma proteome.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#suhrek-38412862">Suhre_K-38412862</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2024</td>
<td style="text-align: left;">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sunbb-29875488">Sun_BB-29875488</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2018</td>
<td style="text-align: left;">Genomic atlas of the human plasma proteome.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sunbb-37794186">Sun_BB-37794186</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">OLINK</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Plasma proteomic associations with genetics and health in the UK Biobank.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sunw-27532455">Sun_W-27532455</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2016</td>
<td style="text-align: left;">Common Genetic Polymorphisms Influence Blood Biomarker Measurements in COPD.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#surapanenia-35870639">Surapaneni_A-35870639</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2022</td>
<td style="text-align: left;">Identification of 969 protein quantitative trait loci in an African American population with kidney disease attributed to hypertension.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tharejag-36168886">Thareja_G-36168886</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Differences and commonalities in the genetic architecture of protein quantitative trait loci in European and Arab populations.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#xuf-36797296">Xu_F-36797296</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">MS</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#yangc-34239129">Yang_C-34239129</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Somascan</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Genomic atlas of the proteome from brain, CSF and plasma prioritizes proteins implicated in neurological disorders.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#yaoc-30111768">Yao_C-30111768</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">IA</td>
<td style="text-align: left;">2018</td>
<td style="text-align: left;">Genome-wide mapping of plasma protein QTLs identifies putatively causal genes and pathways for cardiovascular disease.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#zhongw-32576278">Zhong_W-32576278</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Whole-genome sequence association analysis of blood proteins in a longitudinal wellness cohort.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#krishnac-39085222">Krishna_C-39085222</a></td>
<td style="text-align: left;">HLA-GWAS</td>
<td style="text-align: left;">PEA/OLINK</td>
<td style="text-align: left;">2024</td>
<td style="text-align: left;">The influence of HLA genetic variation on plasma protein expression.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#omicspred-portal">OmicsPred portal</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2023</td>
<td style="text-align: left;">An atlas of genetic scores to predict multi-omic traits.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#proteome-phewas-browser">Proteome PheWAS browser</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2020</td>
<td style="text-align: left;">Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgwas-server">pGWAS server</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2017</td>
<td style="text-align: left;">Connecting genetic risk to disease end points through the human blood plasma proteome.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-suhrek-32860016">Review-Suhre_K-32860016</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">2021</td>
<td style="text-align: left;">Genetics meets proteomics: perspectives for large population-based studies.</td>
</tr>
</tbody>
</table>
<h2 id="comparison">Comparison</h2>
<h3 id="eldjarn_gh-37794188">Eldjarn_GH-37794188</h3>
<ul>
<li><strong>NAME</strong> : Eldjarn_GH-37794188 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Large-scale plasma proteomics comparisons through genetics and disease associations. </li>
<li><strong>CITATION</strong> : Eldjarn GH, ...&amp;, Stefansson K. (2023) Large-scale plasma proteomics comparisons through genetics and disease associations. Nature, 622 (7982) 348-358. doi:10.1038/s41586-023-06563-x. PMID 37794188 </li>
<li><strong>MESH_MAJOR</strong> : Blood Proteins, Disease Susceptibility, Genomics, Genotype, Phenotype, Proteomics </li>
<li><strong>MESH_MINOR</strong> : Africa, Asia, Southern, Biological Specimen Banks, Datasets as Topic, Genome, Human, Humans, Iceland, Ireland, Plasma, Proteome, Quantitative Trait Loci, United Kingdom </li>
<li><strong>ABSTRACT</strong> : High-throughput proteomics platforms measuring thousands of proteins in plasma combined with genomic and phenotypic information have the power to bridge the gap between the genome and diseases. Here we performed association studies of Olink Explore 3072 data generated by the UK Biobank Pharma Proteomics Project<sup>1</sup> on plasma samples from more than 50,000 UK Biobank participants with phenotypic and genotypic data, stratifying on British or Irish, African and South Asian ancestries. We compared the results with those of a SomaScan v4 study on plasma from 36,000 Icelandic people<sup>2</sup>, for 1,514 of whom Olink data were also available. We found modest correlation between the two platforms. Although cis protein quantitative trait loci were detected for a similar absolute number of assays on the two platforms (2,101 on Olink versus 2,120 on SomaScan), the proportion of assays with such supporting evidence for assay performance was higher on the Olink platform (72% versus 43%). A considerable number of proteins had genomic associations that differed between the platforms. We provide examples where differences between platforms may influence conclusions drawn from the integration of protein levels with the study of diseases. We demonstrate how leveraging the diverse ancestries of participants in the UK Biobank helps to detect novel associations and refine genomic location. Our results show the value of the information provided by the two most commonly used high-throughput proteomics platforms and demonstrate the differences between them that at times provides useful complementarity. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06563-x </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 348-358 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794188">37794188</a> </li>
</ul>
<h2 id="consortium">Consortium</h2>
<h3 id="scallop">SCALLOP</h3>
<ul>
<li><strong>NAME</strong> : SCALLOP </li>
<li><strong>DESCRIPTION</strong> : The SCALLOP consortium (Systematic and Combined AnaLysis of Olink Proteins) is a collaborative framework for discovery and follow-up of genetic associations with proteins on the Olink Proteomics platform. To date, 35 PIs from 28 research institutions have joined the effort, which now comprises summary level data for more than 70,000 patients and controls from 45 cohort studies. SCALLOP welcomes new members. </li>
<li><strong>URL</strong> : <a href="http://www.scallop-consortium.com/">http://www.scallop-consortium.com/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
</ul>
<h2 id="gwas">GWAS</h2>
<h3 id="ahola-olli_av-27989323">Ahola-Olli_AV-27989323</h3>
<ul>
<li><strong>NAME</strong> : Ahola-Olli_AV-27989323 </li>
<li><strong>TITLE</strong> : Genome-wide Association Study Identifies 27 Loci Influencing Concentrations of Circulating Cytokines and Growth Factors. </li>
<li><strong>CITATION</strong> : Ahola-Olli AV, ...&amp;, Raitakari OT. (2017) Genome-wide Association Study Identifies 27 Loci Influencing Concentrations of Circulating Cytokines and Growth Factors. Am J Hum Genet, 100 (1) 40-50. doi:10.1016/j.ajhg.2016.11.007. PMID 27989323 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Autoimmune Diseases, Colitis, Ulcerative, Crohn Disease, Cytokines, Female, Humans, Inflammation, Intercellular Signaling Peptides and Proteins, Male, Multiple Sclerosis, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Circulating cytokines and growth factors are regulators of inflammation and have been implicated in autoimmune and metabolic diseases. In this genome-wide association study (GWAS) of up to 8,293 Finns we identified 27 genome-widely significant loci (p &lt; 1.2 × 10<sup>-9</sup>) for one or more cytokines. Fifteen of the associated variants had expression quantitative trait loci in whole blood. We provide genetic instruments to clarify the causal roles of cytokine signaling and upstream inflammation in immune-related and other chronic diseases. We further link inflammatory markers with variants previously associated with autoimmune diseases such as Crohn disease, multiple sclerosis, and ulcerative colitis and hereby elucidate the molecular mechanisms underpinning these diseases and suggest potential drug targets. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2017 American Society of Human Genetics. Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2016.11.007 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am J Hum Genet ; 2017 ; 100 ; 1 ; 40-50 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27989323">27989323</a> </li>
</ul>
<h3 id="bretherick_ad-32628676">Bretherick_AD-32628676</h3>
<ul>
<li><strong>NAME</strong> : Bretherick_AD-32628676 </li>
<li><strong>TITLE</strong> : Linking protein to phenotype with Mendelian Randomization detects 38 proteins with causal roles in human diseases and traits. </li>
<li><strong>CITATION</strong> : Bretherick AD, ...&amp;, Haley C. (2020) Linking protein to phenotype with Mendelian Randomization detects 38 proteins with causal roles in human diseases and traits. PLoS Genet, 16 (7) e1008785. doi:10.1371/journal.pgen.1008785. PMID 32628676 </li>
<li><strong>MESH_MAJOR</strong> : Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Antigens, Differentiation, Cardiovascular Diseases, Fatty Acid-Binding Proteins, Female, Fibroblast Growth Factor 5, Genetic Association Studies, Humans, Lipoprotein Lipase, Lymphotoxin-alpha, Male, Proteome, Quantitative Trait Loci, Receptors, Immunologic, Receptors, Interleukin-6, Schizophrenia </li>
<li><strong>ABSTRACT</strong> : To efficiently transform genetic associations into drug targets requires evidence that a particular gene, and its encoded protein, contribute causally to a disease. To achieve this, we employ a three-step proteome-by-phenome Mendelian Randomization (MR) approach. In step one, 154 protein quantitative trait loci (pQTLs) were identified and independently replicated. From these pQTLs, 64 replicated locally-acting variants were used as instrumental variables for proteome-by-phenome MR across 846 traits (step two). When its assumptions are met, proteome-by-phenome MR, is equivalent to simultaneously running many randomized controlled trials. Step 2 yielded 38 proteins that significantly predicted variation in traits and diseases in 509 instances. Step 3 revealed that amongst the 271 instances from GeneAtlas (UK Biobank), 77 showed little evidence of pleiotropy (HEIDI), and 92 evidence of colocalization (eCAVIAR). Results were wide ranging: including, for example, new evidence for a causal role of tyrosine-protein phosphatase non-receptor type substrate 1 (SHPS1; SIRPA) in schizophrenia, and a new finding that intestinal fatty acid binding protein (FABP2) abundance contributes to the pathogenesis of cardiovascular disease. We also demonstrated confirmatory evidence for the causal role of four further proteins (FGF5, IL6R, LPL, LTA) in cardiovascular disease risk. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1008785 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2020 ; 16 ; 7 ; e1008785 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32628676">32628676</a> </li>
</ul>
<h3 id="carayol_j-29234017">Carayol_J-29234017</h3>
<ul>
<li><strong>NAME</strong> : Carayol_J-29234017 </li>
<li><strong>TITLE</strong> : Protein quantitative trait locus study in obesity during weight-loss identifies a leptin regulator. </li>
<li><strong>CITATION</strong> : Carayol J, ...&amp;, Hager J. (2017) Protein quantitative trait locus study in obesity during weight-loss identifies a leptin regulator. Nat Commun, 8 (1) 2084. doi:10.1038/s41467-017-02182-z. PMID 29234017 </li>
<li><strong>MESH_MAJOR</strong> : Body Mass Index, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Adolescent, Adult, Female, Gene Regulatory Networks, Humans, Leptin, Male, Middle Aged, Obesity, Polynucleotide Adenylyltransferase, Proteins, Proteomics, Regulatory Elements, Transcriptional, Weight Loss, Young Adult </li>
<li><strong>ABSTRACT</strong> : Thousands of genetic variants have been associated with complex traits through genome-wide association studies. However, the functional variants or mechanistic consequences remain elusive. Intermediate traits such as gene expression or protein levels are good proxies of the metabolic state of an organism. Proteome analysis especially can provide new insights into the molecular mechanisms of complex traits like obesity. The role of genetic variation in determining protein level variation has not been assessed in obesity. To address this, we design a large-scale protein quantitative trait locus (pQTL) analysis based on a set of 1129 proteins from 494 obese subjects before and after a weight loss intervention. This reveals 55 BMI-associated cis-pQTLs and trans-pQTLs at baseline and 3 trans-pQTLs after the intervention. We provide evidence for distinct genetic mechanisms regulating BMI-associated proteins before and after weight loss. Finally, by functional analysis, we identify and validate FAM46A as a trans regulator for leptin. </li>
<li><strong>DOI</strong> : 10.1038/s41467-017-02182-z </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2017 ; 8 ; 1 ; 2084 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29234017">29234017</a> </li>
</ul>
<h3 id="carland_c-37550624">Carland_C-37550624</h3>
<ul>
<li><strong>NAME</strong> : Carland_C-37550624 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases. </li>
<li><strong>CITATION</strong> : Carland C, ...&amp;, Assimes T. (2023) Proteomic analysis of 92 circulating proteins and their effects in cardiometabolic diseases. Clin Proteomics, 20 (1) 31. doi:10.1186/s12014-023-09421-0. PMID 37550624 </li>
<li><strong>ABSTRACT</strong> : Human plasma contains a wide variety of circulating proteins. These proteins can be important clinical biomarkers in disease and also possible drug targets. Large scale genomics studies of circulating proteins can identify genetic variants that lead to relative protein abundance.|We conducted a meta-analysis on genome-wide association studies of autosomal chromosomes in 22,997 individuals of primarily European ancestry across 12 cohorts to identify protein quantitative trait loci (pQTL) for 92 cardiometabolic associated plasma proteins.|We identified 503 (337 cis and 166 trans) conditionally independent pQTLs, including several novel variants not reported in the literature. We conducted a sex-stratified analysis and found that 118 (23.5%) of pQTLs demonstrated heterogeneity between sexes. The direction of effect was preserved but there were differences in effect size and significance. Additionally, we annotate trans-pQTLs with nearest genes and report plausible biological relationships. Using Mendelian randomization, we identified causal associations for 18 proteins across 19 phenotypes, of which 10 have additional genetic colocalization evidence. We highlight proteins associated with a constellation of cardiometabolic traits including angiopoietin-related protein 7 (ANGPTL7) and Semaphorin 3F (SEMA3F).|Through large-scale analysis of protein quantitative trait loci, we provide a comprehensive overview of common variants associated with plasma proteins. We highlight possible biological relationships which may serve as a basis for further investigation into possible causal roles in cardiometabolic diseases. </li>
<li><strong>COPYRIGHT</strong> : © 2023. BioMed Central Ltd., part of Springer Nature. </li>
<li><strong>DOI</strong> : 10.1186/s12014-023-09421-0 </li>
<li><strong>JOURNAL_INFO</strong> : Clinical proteomics ; Clin Proteomics ; 2023 ; 20 ; 1 ; 31 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37550624">37550624</a> </li>
</ul>
<h3 id="caron_b-35264221">Caron_B-35264221</h3>
<ul>
<li><strong>NAME</strong> : Caron_B-35264221 </li>
<li><strong>TITLE</strong> : Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes. </li>
<li><strong>CITATION</strong> : . (2022) Integrative genetic and immune cell analysis of plasma proteins in healthy donors identifies novel associations involving primary immune deficiency genes. Genome Med, 14 (1) 28. doi:10.1186/s13073-022-01032-y. PMID 35264221 </li>
<li><strong>MESH_MAJOR</strong> : Blood Proteins, Primary Immunodeficiency Diseases </li>
<li><strong>MESH_MINOR</strong> : Genome-Wide Association Study, Health Status, Humans, Polymorphism, Single Nucleotide, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Blood plasma proteins play an important role in immune defense against pathogens, including cytokine signaling, the complement system, and the acute-phase response. Recent large-scale studies have reported genetic (i.e., protein quantitative trait loci, pQTLs) and non-genetic factors, such as age and sex, as major determinants to inter-individual variability in immune response variation. However, the contribution of blood-cell composition to plasma protein heterogeneity has not been fully characterized and may act as a mediating factor in association studies.|Here, we evaluated plasma protein levels from 400 unrelated healthy individuals of western European ancestry, who were stratified by sex and two decades of life (20-29 and 60-69 years), from the Milieu Intérieur cohort. We quantified 229 proteins by Luminex in a clinically certified laboratory and their levels of variation were analyzed together with 5.2 million single-nucleotide polymorphisms. With respect to non-genetic variables, we included 254 lifestyle and biochemical factors, as well as counts of seven circulating immune cell populations measured by hemogram and standardized flow cytometry.|Collectively, we found 152 significant associations involving 49 proteins and 20 non-genetic variables. Consistent with previous studies, age and sex showed a global, pervasive impact on plasma protein heterogeneity, while body mass index and other health status variables were among the non-genetic factors with the highest number of associations. After controlling for these covariates, we identified 100 and 12 pQTLs acting in cis and trans, respectively, collectively associated with 87 plasma proteins and including 19 novel genetic associations. Genetic factors explained the largest fraction of the variability of plasma protein levels, as compared to non-genetic factors. In addition, blood-cell fractions, including leukocytes, lymphocytes, monocytes, neutrophils, eosinophils, basophils, and platelets, had a larger contribution to inter-individual variability than age and sex and appeared as confounders of specific genetic associations. Finally, we identified new genetic associations with plasma protein levels of five monogenic Mendelian disease genes including two primary immunodeficiency genes (Ficolin-3 and FAS).|Our study identified novel genetic and non-genetic factors associated to plasma protein levels which may inform health status and disease management. </li>
<li><strong>COPYRIGHT</strong> : © 2022. The Author(s). </li>
<li><strong>DOI</strong> : 10.1186/s13073-022-01032-y </li>
<li><strong>JOURNAL_INFO</strong> : Genome medicine ; Genome Med ; 2022 ; 14 ; 1 ; 28 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35264221">35264221</a> </li>
</ul>
<h3 id="deming_y-28247064">Deming_Y-28247064</h3>
<ul>
<li><strong>NAME</strong> : Deming_Y-28247064 </li>
<li><strong>TITLE</strong> : Genome-wide association study identifies four novel loci associated with Alzheimer's endophenotypes and disease modifiers. </li>
<li><strong>CITATION</strong> : Deming Y, ...&amp;, Cruchaga C. (2017) Genome-wide association study identifies four novel loci associated with Alzheimer's endophenotypes and disease modifiers. Acta Neuropathol, 133 (5) 839-856. doi:10.1007/s00401-017-1685-y. PMID 28247064 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Predisposition to Disease, Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Adult, Aged, Aged, 80 and over, Alzheimer Disease, Amyloid beta-Peptides, Apolipoproteins E, Biomarkers, Disease Progression, Endophenotypes, Genetic Loci, Genotype, Humans, Middle Aged, Risk Factors, tau Proteins </li>
<li><strong>ABSTRACT</strong> : More than 20 genetic loci have been associated with risk for Alzheimer's disease (AD), but reported genome-wide significant loci do not account for all the estimated heritability and provide little information about underlying biological mechanisms. Genetic studies using intermediate quantitative traits such as biomarkers, or endophenotypes, benefit from increased statistical power to identify variants that may not pass the stringent multiple test correction in case-control studies. Endophenotypes also contain additional information helpful for identifying variants and genes associated with other aspects of disease, such as rate of progression or onset, and provide context to interpret the results from genome-wide association studies (GWAS). We conducted GWAS of amyloid beta (Aβ<sub>42</sub>), tau, and phosphorylated tau (ptau<sub>181</sub>) levels in cerebrospinal fluid (CSF) from 3146 participants across nine studies to identify novel variants associated with AD. Five genome-wide significant loci (two novel) were associated with ptau<sub>181</sub>, including loci that have also been associated with AD risk or brain-related phenotypes. Two novel loci associated with Aβ<sub>42</sub> near GLIS1 on 1p32.3 (β = -0.059, P = 2.08 × 10<sup>-8</sup>) and within SERPINB1 on 6p25 (β = -0.025, P = 1.72 × 10<sup>-8</sup>) were also associated with AD risk (GLIS1: OR = 1.105, P = 3.43 × 10<sup>-2</sup>), disease progression (GLIS1: β = 0.277, P = 1.92 × 10<sup>-2</sup>), and age at onset (SERPINB1: β = 0.043, P = 4.62 × 10<sup>-3</sup>). Bioinformatics indicate that the intronic SERPINB1 variant (rs316341) affects expression of SERPINB1 in various tissues, including the hippocampus, suggesting that SERPINB1 influences AD through an Aβ-associated mechanism. Analyses of known AD risk loci suggest CLU and FERMT2 may influence CSF Aβ<sub>42</sub> (P = 0.001 and P = 0.009, respectively) and the INPP5D locus may affect ptau<sub>181</sub> levels (P = 0.009); larger studies are necessary to verify these results. Together the findings from this study can be used to inform future AD studies. </li>
<li><strong>DOI</strong> : 10.1007/s00401-017-1685-y </li>
<li><strong>JOURNAL_INFO</strong> : Acta neuropathologica ; Acta Neuropathol ; 2017 ; 133 ; 5 ; 839-856 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28247064">28247064</a> </li>
</ul>
<h3 id="dhindsa_rs-37794183">Dhindsa_RS-37794183</h3>
<ul>
<li><strong>NAME</strong> : Dhindsa_RS-37794183 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Rare variant associations with plasma protein levels in the UK Biobank. </li>
<li><strong>CITATION</strong> : Dhindsa RS, ...&amp;, Petrovski S. (2023) Rare variant associations with plasma protein levels in the UK Biobank. Nature, 622 (7982) 339-347. doi:10.1038/s41586-023-06547-x. PMID 37794183 </li>
<li><strong>MESH_MAJOR</strong> : Biological Specimen Banks, Blood Proteins, Genetic Association Studies, Genomics, Proteomics </li>
<li><strong>MESH_MINOR</strong> : Alleles, Biomarkers, Databases, Factual, Exome, Hematopoiesis, Humans, Mutation, Plasma, United Kingdom </li>
<li><strong>ABSTRACT</strong> : Integrating human genomics and proteomics can help elucidate disease mechanisms, identify clinical biomarkers and discover drug targets<sup>1-4</sup>. Because previous proteogenomic studies have focused on common variation via genome-wide association studies, the contribution of rare variants to the plasma proteome remains largely unknown. Here we identify associations between rare protein-coding variants and 2,923 plasma protein abundances measured in 49,736 UK Biobank individuals. Our variant-level exome-wide association study identified 5,433 rare genotype-protein associations, of which 81% were undetected in a previous genome-wide association study of the same cohort<sup>5</sup>. We then looked at aggregate signals using gene-level collapsing analysis, which revealed 1,962 gene-protein associations. Of the 691 gene-level signals from protein-truncating variants, 99.4% were associated with decreased protein levels. STAB1 and STAB2, encoding scavenger receptors involved in plasma protein clearance, emerged as pleiotropic loci, with 77 and 41 protein associations, respectively. We demonstrate the utility of our publicly accessible resource through several applications. These include detailing an allelic series in NLRC4, identifying potential biomarkers for a fatty liver disease-associated variant in HSD17B13 and bolstering phenome-wide association studies by integrating protein quantitative trait loci with protein-truncating variants in collapsing analyses. Finally, we uncover distinct proteomic consequences of clonal haematopoiesis (CH), including an association between TET2-CH and increased FLT3 levels. Our results highlight a considerable role for rare variation in plasma protein abundance and the value of proteogenomics in therapeutic discovery. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06547-x </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 339-347 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794183">37794183</a> </li>
</ul>
<h3 id="emilsson_v-30072576">Emilsson_V-30072576</h3>
<ul>
<li><strong>NAME</strong> : Emilsson_V-30072576 </li>
<li><strong>TITLE</strong> : Co-regulatory networks of human serum proteins link genetics to disease. </li>
<li><strong>CITATION</strong> : Emilsson V, ...&amp;, Gudnason V. (2018) Co-regulatory networks of human serum proteins link genetics to disease. Science, 361 (6404) 769-773. doi:10.1126/science.aaq1327. PMID 30072576 </li>
<li><strong>MESH_MINOR</strong> : Aptamers, Nucleotide, Blood Proteins, Cardiovascular Diseases, Genetic Predisposition to Disease, Genetic Variation, Humans, Iceland, Metabolic Diseases, Metabolic Networks and Pathways, Proteome, Proteomics </li>
<li><strong>ABSTRACT</strong> : Proteins circulating in the blood are critical for age-related disease processes; however, the serum proteome has remained largely unexplored. To this end, 4137 proteins covering most predicted extracellular proteins were measured in the serum of 5457 Icelanders over 65 years of age. Pairwise correlation between proteins as they varied across individuals revealed 27 different network modules of serum proteins, many of which were associated with cardiovascular and metabolic disease states, as well as overall survival. The protein modules were controlled by cis- and trans-acting genetic variants, which in many cases were also associated with complex disease. This revealed co-regulated groups of circulating proteins that incorporated regulatory control between tissues and demonstrated close relationships to past, current, and future disease states. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2018 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. </li>
<li><strong>DOI</strong> : 10.1126/science.aaq1327 </li>
<li><strong>JOURNAL_INFO</strong> : Science (New York, N.Y.) ; Science ; 2018 ; 361 ; 6404 ; 769-773 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30072576">30072576</a> </li>
</ul>
<h3 id="enroth_s-25147954">Enroth_S-25147954</h3>
<ul>
<li><strong>NAME</strong> : Enroth_S-25147954 </li>
<li><strong>TITLE</strong> : Strong effects of genetic and lifestyle factors on biomarker variation and use of personalized cutoffs. </li>
<li><strong>CITATION</strong> : Enroth S, ...&amp;, Gyllensten U. (2014) Strong effects of genetic and lifestyle factors on biomarker variation and use of personalized cutoffs. Nat Commun, 5  4684. doi:10.1038/ncomms5684. PMID 25147954 </li>
<li><strong>MESH_MAJOR</strong> : Life Style, Polymorphism, Single Nucleotide </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Biomarkers, Tumor, Cohort Studies, Female, Genome-Wide Association Study, Humans, Inflammation, Male, Reference Values, Sweden </li>
<li><strong>ABSTRACT</strong> : Ideal biomarkers used for disease diagnosis should display deviating levels in affected individuals only and be robust to factors unrelated to the disease. Here we show the impact of genetic, clinical and lifestyle factors on circulating levels of 92 protein biomarkers for cancer and inflammation, using a population-based cohort of 1,005 individuals. For 75% of the biomarkers, the levels are significantly heritable and genome-wide association studies identifies 16 novel loci and replicate 2 previously known loci with strong effects on one or several of the biomarkers with P-values down to 4.4 × 10(-58). Integrative analysis attributes as much as 56.3% of the observed variance to non-disease factors. We propose that information on the biomarker-specific profile of major genetic, clinical and lifestyle factors should be used to establish personalized clinical cutoffs, and that this would increase the sensitivity of using biomarkers for prediction of clinical end points. </li>
<li><strong>DOI</strong> : 10.1038/ncomms5684 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2014 ; 5 ; <NA> ; 4684 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25147954">25147954</a> </li>
</ul>
<h3 id="ferkingstad_e-34857953">Ferkingstad_E-34857953</h3>
<ul>
<li><strong>NAME</strong> : Ferkingstad_E-34857953 </li>
<li><strong>TITLE</strong> : Large-scale integration of the plasma proteome with genetics and disease. </li>
<li><strong>CITATION</strong> : Ferkingstad E, ...&amp;, Stefansson K. (2021) Large-scale integration of the plasma proteome with genetics and disease. Nat Genet, 53 (12) 1712-1721. doi:10.1038/s41588-021-00978-w. PMID 34857953 </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Blood Proteins, Disease, Female, Gene Frequency, Genetic Variation, Genome-Wide Association Study, Humans, Male, Middle Aged, Proteome, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : The plasma proteome can help bridge the gap between the genome and diseases. Here we describe genome-wide association studies (GWASs) of plasma protein levels measured with 4,907 aptamers in 35,559 Icelanders. We found 18,084 associations between sequence variants and levels of proteins in plasma (protein quantitative trait loci; pQTL), of which 19% were with rare variants (minor allele frequency (MAF) &lt; 1%). We tested plasma protein levels for association with 373 diseases and other traits and identified 257,490 associations. We integrated pQTL and genetic associations with diseases and other traits and found that 12% of 45,334 lead associations in the GWAS Catalog are with variants in high linkage disequilibrium with pQTL. We identified 938 genes encoding potential drug targets with variants that influence levels of possible biomarkers. Combining proteomics, genomics and transcriptomics, we provide a valuable resource that can be used to improve understanding of disease pathogenesis and to assist with drug discovery and development. </li>
<li><strong>COPYRIGHT</strong> : © 2021. The Author(s), under exclusive licence to Springer Nature America, Inc. </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00978-w </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2021 ; 53 ; 12 ; 1712-1721 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34857953">34857953</a> </li>
</ul>
<h3 id="folkersen_l-28369058">Folkersen_L-28369058</h3>
<ul>
<li><strong>NAME</strong> : Folkersen_L-28369058 </li>
<li><strong>TITLE</strong> : Mapping of 79 loci for 83 plasma protein biomarkers in cardiovascular disease. </li>
<li><strong>CITATION</strong> : Folkersen L, ...&amp;, Mälarstig A. (2017) Mapping of 79 loci for 83 plasma protein biomarkers in cardiovascular disease. PLoS Genet, 13 (4) e1006706. doi:10.1371/journal.pgen.1006706. PMID 28369058 </li>
<li><strong>MESH_MAJOR</strong> : Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Blood Proteins, Cardiovascular Diseases, Coronary Artery Disease, Female, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Male </li>
<li><strong>ABSTRACT</strong> : Recent advances in highly multiplexed immunoassays have allowed systematic large-scale measurement of hundreds of plasma proteins in large cohort studies. In combination with genotyping, such studies offer the prospect to 1) identify mechanisms involved with regulation of protein expression in plasma, and 2) determine whether the plasma proteins are likely to be causally implicated in disease. We report here the results of genome-wide association (GWA) studies of 83 proteins considered relevant to cardiovascular disease (CVD), measured in 3,394 individuals with multiple CVD risk factors. We identified 79 genome-wide significant (p&lt;5e-8) association signals, 55 of which replicated at P&lt;0.0007 in separate validation studies (n = 2,639 individuals). Using automated text mining, manual curation, and network-based methods incorporating information on expression quantitative trait loci (eQTL), we propose plausible causal mechanisms for 25 trans-acting loci, including a potential post-translational regulation of stem cell factor by matrix metalloproteinase 9 and receptor-ligand pairs such as RANK-RANK ligand. Using public GWA study data, we further evaluate all 79 loci for their causal effect on coronary artery disease, and highlight several potentially causal associations. Overall, a majority of the plasma proteins studied showed evidence of regulation at the genetic level. Our results enable future studies of the causal architecture of human disease, which in turn should aid discovery of new drug targets. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1006706 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2017 ; 13 ; 4 ; e1006706 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28369058">28369058</a> </li>
</ul>
<h3 id="folkersen_l-33067605">Folkersen_L-33067605</h3>
<ul>
<li><strong>NAME</strong> : Folkersen_L-33067605 </li>
<li><strong>TITLE</strong> : Genomic and drug target evaluation of 90 cardiovascular proteins in 30,931 individuals. </li>
<li><strong>CITATION</strong> : Folkersen L, ...&amp;, Mälarstig A. (2020) Genomic and drug target evaluation of 90 cardiovascular proteins in 30,931 individuals. Nat Metab, 2 (10) 1135-1148. doi:10.1038/s42255-020-00287-2. PMID 33067605 </li>
<li><strong>MESH_MAJOR</strong> : Chromosome Mapping, Drug Delivery Systems, Genomics </li>
<li><strong>MESH_MINOR</strong> : ATP Binding Cassette Transporter 1, Asthma, Cardiovascular System, Gene Knockdown Techniques, Genome-Wide Association Study, Humans, Inflammatory Bowel Diseases, Interleukin-1 Receptor-Like 1 Protein, Intracellular Signaling Peptides and Proteins, Linkage Disequilibrium, Mendelian Randomization Analysis, Protein Serine-Threonine Kinases, Proteome, Quantitative Trait Loci, Receptors, CCR2, Receptors, CCR5 </li>
<li><strong>ABSTRACT</strong> : Circulating proteins are vital in human health and disease and are frequently used as biomarkers for clinical decision-making or as targets for pharmacological intervention. Here, we map and replicate protein quantitative trait loci (pQTL) for 90 cardiovascular proteins in over 30,000 individuals, resulting in 451 pQTLs for 85 proteins. For each protein, we further perform pathway mapping to obtain trans-pQTL gene and regulatory designations. We substantiate these regulatory findings with orthogonal evidence for trans-pQTLs using mouse knockdown experiments (ABCA1 and TRIB1) and clinical trial results (chemokine receptors CCR2 and CCR5), with consistent regulation. Finally, we evaluate known drug targets, and suggest new target candidates or repositioning opportunities using Mendelian randomization. This identifies 11 proteins with causal evidence of involvement in human disease that have not previously been targeted, including EGF, IL-16, PAPPA, SPON1, F3, ADM, CASP-8, CHI3L1, CXCL16, GDF15 and MMP-12. Taken together, these findings demonstrate the utility of large-scale mapping of the genetics of the proteome and provide a resource for future precision studies of circulating proteins in human health. </li>
<li><strong>DOI</strong> : 10.1038/s42255-020-00287-2 </li>
<li><strong>JOURNAL_INFO</strong> : Nature metabolism ; Nat Metab ; 2020 ; 2 ; 10 ; 1135-1148 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33067605">33067605</a> </li>
</ul>
<h3 id="gilly_a-37778719">Gilly_A-37778719</h3>
<ul>
<li><strong>NAME</strong> : Gilly_A-37778719 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Genome-wide meta-analysis of 92 cardiometabolic protein serum levels. </li>
<li><strong>CITATION</strong> : Gilly A, ...&amp;, Zeggini E. (2023) Genome-wide meta-analysis of 92 cardiometabolic protein serum levels. Mol Metab, 78  101810. doi:10.1016/j.molmet.2023.101810. PMID 37778719 </li>
<li><strong>MESH_MAJOR</strong> : Cardiovascular Diseases, Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Blood Proteins, Genetic Predisposition to Disease, Humans, Inflammation, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Global cardiometabolic disease prevalence has grown rapidly over the years, making it the leading cause of death worldwide. Proteins are crucial components in biological pathways dysregulated in disease states. Identifying genetic components that influence circulating protein levels may lead to the discovery of biomarkers for early stages of disease or offer opportunities as therapeutic targets.|Here, we carry out a genome-wide association study (GWAS) utilising whole genome sequencing data in 3,005 individuals from the HELIC founder populations cohort, across 92 proteins of cardiometabolic relevance.|We report 322 protein quantitative trait loci (pQTL) signals across 92 proteins, of which 76 are located in or near the coding gene (cis-pQTL). We link those association signals with changes in protein expression and cardiometabolic disease risk using colocalisation and Mendelian randomisation (MR) analyses.|The majority of previously unknown signals we describe point to proteins or protein interactions involved in inflammation and immune response, providing genetic evidence for the contributing role of inflammation in cardiometabolic disease processes. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2023 The Author(s). Published by Elsevier GmbH.. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.molmet.2023.101810 </li>
<li><strong>JOURNAL_INFO</strong> : Molecular metabolism ; Mol Metab ; 2023 ; 78 ; <NA> ; 101810 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37778719">37778719</a> </li>
</ul>
<h3 id="gudjonsson_a-35078996">Gudjonsson_A-35078996</h3>
<ul>
<li><strong>NAME</strong> : Gudjonsson_A-35078996 </li>
<li><strong>TITLE</strong> : A genome-wide association study of serum proteins reveals shared loci with common diseases. </li>
<li><strong>CITATION</strong> : Gudjonsson A, ...&amp;, Gudnason V. (2022) A genome-wide association study of serum proteins reveals shared loci with common diseases. Nat Commun, 13 (1) 480. doi:10.1038/s41467-021-27850-z. PMID 35078996 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Predisposition to Disease, Genome, Human, Polymorphism, Single Nucleotide, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Aged, Aged, 80 and over, Blood Proteins, Cohort Studies, Disease, Female, Genome-Wide Association Study, Humans, Iceland, Male </li>
<li><strong>ABSTRACT</strong> : With the growing number of genetic association studies, the genotype-phenotype atlas has become increasingly more complex, yet the functional consequences of most disease associated alleles is not understood. The measurement of protein level variation in solid tissues and biofluids integrated with genetic variants offers a path to deeper functional insights. Here we present a large-scale proteogenomic study in 5,368 individuals, revealing 4,035 independent associations between genetic variants and 2,091 serum proteins, of which 36% are previously unreported. The majority of both cis- and trans-acting genetic signals are unique for a single protein, although our results also highlight numerous highly pleiotropic genetic effects on protein levels and demonstrate that a protein's genetic association profile reflects certain characteristics of the protein, including its location in protein networks, tissue specificity and intolerance to loss of function mutations. Integrating protein measurements with deep phenotyping of the cohort, we observe substantial enrichment of phenotype associations for serum proteins regulated by established GWAS loci, and offer new insights into the interplay between genetics, serum protein levels and complex disease. </li>
<li><strong>COPYRIGHT</strong> : © 2022. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-27850-z </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2022 ; 13 ; 1 ; 480 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35078996">35078996</a> </li>
</ul>
<h3 id="hansson_o-36504281">Hansson_O-36504281</h3>
<ul>
<li><strong>NAME</strong> : Hansson_O-36504281 </li>
<li><strong>TITLE</strong> : The genetic regulation of protein expression in cerebrospinal fluid. </li>
<li><strong>CITATION</strong> : Hansson O, ...&amp;, Mattsson-Carlgren N. (2023) The genetic regulation of protein expression in cerebrospinal fluid. EMBO Mol Med, 15 (1) e16359. doi:10.15252/emmm.202216359. PMID 36504281 </li>
<li><strong>MESH_MAJOR</strong> : Alzheimer Disease, Parkinson Disease </li>
<li><strong>MESH_MINOR</strong> : ADAM Proteins, Antigens, CD, Biomarkers, Humans, Matrix Metalloproteinase 10, Membrane Glycoproteins, Membrane Proteins, Proteomics, Quantitative Trait Loci, Sialic Acid Binding Immunoglobulin-like Lectins </li>
<li><strong>ABSTRACT</strong> : Studies of the genetic regulation of cerebrospinal fluid (CSF) proteins may reveal pathways for treatment of neurological diseases. 398 proteins in CSF were measured in 1,591 participants from the BioFINDER study. Protein quantitative trait loci (pQTL) were identified as associations between genetic variants and proteins, with 176 pQTLs for 145 CSF proteins (P &lt; 1.25 × 10<sup>-10</sup> , 117 cis-pQTLs and 59 trans-pQTLs). Ventricular volume (measured with brain magnetic resonance imaging) was a confounder for several pQTLs. pQTLs for CSF and plasma proteins were overall correlated, but CSF-specific pQTLs were also observed. Mendelian randomization analyses suggested causal roles for several proteins, for example, ApoE, CD33, and GRN in Alzheimer's disease, MMP-10 in preclinical Alzheimer's disease, SIGLEC9 in amyotrophic lateral sclerosis, and CD38, GPNMB, and ADAM15 in Parkinson's disease. CSF levels of GRN, MMP-10, and GPNMB were altered in Alzheimer's disease, preclinical Alzheimer's disease, and Parkinson's disease, respectively. These findings point to pathways to be explored for novel therapies. The novel finding that ventricular volume confounded pQTLs has implications for design of future studies of the genetic regulation of the CSF proteome. </li>
<li><strong>COPYRIGHT</strong> : © 2022 The Authors. Published under the terms of the CC BY 4.0 license. </li>
<li><strong>DOI</strong> : 10.15252/emmm.202216359 </li>
<li><strong>JOURNAL_INFO</strong> : EMBO molecular medicine ; EMBO Mol Med ; 2023 ; 15 ; 1 ; e16359 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36504281">36504281</a> </li>
</ul>
<h3 id="hillary_rf-31320639">Hillary_RF-31320639</h3>
<ul>
<li><strong>NAME</strong> : Hillary_RF-31320639 </li>
<li><strong>TITLE</strong> : Genome and epigenome wide studies of neurological protein biomarkers in the Lothian Birth Cohort 1936. </li>
<li><strong>CITATION</strong> : Hillary RF, ...&amp;, Marioni RE. (2019) Genome and epigenome wide studies of neurological protein biomarkers in the Lothian Birth Cohort 1936. Nat Commun, 10 (1) 3160. doi:10.1038/s41467-019-11177-x. PMID 31320639 </li>
<li><strong>MESH_MINOR</strong> : Aged, Biomarkers, Blood Proteins, CpG Islands, DNA Methylation, Epigenesis, Genetic, Female, Genome, Human, Genome-Wide Association Study, Humans, Male, Nervous System Diseases, Quantitative Trait Loci, Scotland </li>
<li><strong>ABSTRACT</strong> : Although plasma proteins may serve as markers of neurological disease risk, the molecular mechanisms responsible for inter-individual variation in plasma protein levels are poorly understood. Therefore, we conduct genome- and epigenome-wide association studies on the levels of 92 neurological proteins to identify genetic and epigenetic loci associated with their plasma concentrations (n = 750 healthy older adults). We identify 41 independent genome-wide significant (P &lt; 5.4 × 10<sup>-10</sup>) loci for 33 proteins and 26 epigenome-wide significant (P &lt; 3.9 × 10<sup>-10</sup>) sites associated with the levels of 9 proteins. Using this information, we identify biological pathways in which putative neurological biomarkers are implicated (neurological, immunological and extracellular matrix metabolic pathways). We also observe causal relationships (by Mendelian randomisation analysis) between changes in gene expression (DRAXIN, MDGA1 and KYNU), or DNA methylation profiles (MATN3, MDGA1 and NEP), and altered plasma protein levels. Together, this may help inform causal relationships between biomarkers and neurological diseases. </li>
<li><strong>DOI</strong> : 10.1038/s41467-019-11177-x </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2019 ; 10 ; 1 ; 3160 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31320639">31320639</a> </li>
</ul>
<h3 id="hillary_rf-32641083">Hillary_RF-32641083</h3>
<ul>
<li><strong>NAME</strong> : Hillary_RF-32641083 </li>
<li><strong>TITLE</strong> : Multi-method genome- and epigenome-wide studies of inflammatory protein levels in healthy older adults. </li>
<li><strong>CITATION</strong> : Hillary RF, ...&amp;, Marioni RE. (2020) Multi-method genome- and epigenome-wide studies of inflammatory protein levels in healthy older adults. Genome Med, 12 (1) 60. doi:10.1186/s13073-020-00754-1. PMID 32641083 </li>
<li><strong>MESH_MAJOR</strong> : Biomarkers, Epigenomics, Genome-Wide Association Study, Genomics </li>
<li><strong>MESH_MINOR</strong> : Age Factors, Aged, Aged, 80 and over, Blood Proteins, Computational Biology, DNA Methylation, Disease Susceptibility, Epigenesis, Genetic, Female, Gene Expression Regulation, Healthy Volunteers, Humans, Inflammation, Inflammation Mediators, Male, Middle Aged, Polymorphism, Single Nucleotide, Proteins, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : The molecular factors which control circulating levels of inflammatory proteins are not well understood. Furthermore, association studies between molecular probes and human traits are often performed by linear model-based methods which may fail to account for complex structure and interrelationships within molecular datasets.|In this study, we perform genome- and epigenome-wide association studies (GWAS/EWAS) on the levels of 70 plasma-derived inflammatory protein biomarkers in healthy older adults (Lothian Birth Cohort 1936; n = 876; Olink® inflammation panel). We employ a Bayesian framework (BayesR+) which can account for issues pertaining to data structure and unknown confounding variables (with sensitivity analyses using ordinary least squares- (OLS) and mixed model-based approaches).|We identified 13 SNPs associated with 13 proteins (n = 1 SNP each) concordant across OLS and Bayesian methods. We identified 3 CpG sites spread across 3 proteins (n = 1 CpG each) that were concordant across OLS, mixed-model and Bayesian analyses. Tagged genetic variants accounted for up to 45% of variance in protein levels (for MCP2, 36% of variance alone attributable to 1 polymorphism). Methylation data accounted for up to 46% of variation in protein levels (for CXCL10). Up to 66% of variation in protein levels (for VEGFA) was explained using genetic and epigenetic data combined. We demonstrated putative causal relationships between CD6 and IL18R1 with inflammatory bowel disease and between IL12B and Crohn's disease.|Our data may aid understanding of the molecular regulation of the circulating inflammatory proteome as well as causal relationships between inflammatory mediators and disease. </li>
<li><strong>DOI</strong> : 10.1186/s13073-020-00754-1 </li>
<li><strong>JOURNAL_INFO</strong> : Genome medicine ; Genome Med ; 2020 ; 12 ; 1 ; 60 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32641083">32641083</a> </li>
</ul>
<h3 id="johansson_a-23487758">Johansson_Å-23487758</h3>
<ul>
<li><strong>NAME</strong> : Johansson_Å-23487758 </li>
<li><strong>TITLE</strong> : Identification of genetic variants influencing the human plasma proteome. </li>
<li><strong>CITATION</strong> : Johansson Å, ...&amp;, Gyllensten U. (2013) Identification of genetic variants influencing the human plasma proteome. Proc Natl Acad Sci U S A, 110 (12) 4673-8. doi:10.1073/pnas.1217238110. PMID 23487758 </li>
<li><strong>MESH_MAJOR</strong> : Alleles, Blood Proteins, Polymorphism, Single Nucleotide, Proteome </li>
<li><strong>MESH_MINOR</strong> : Adolescent, Adult, Biomarkers, Cohort Studies, Female, Genetic Markers, Genome-Wide Association Study, Humans, Male, Middle Aged </li>
<li><strong>ABSTRACT</strong> : Genetic variants influencing the transcriptome have been extensively studied. However, the impact of the genetic factors on the human proteome is largely unexplored, mainly due to lack of suitable high-throughput methods. Here we present unique and comprehensive identification of genetic variants affecting the human plasma protein profile by combining high-throughput and high-resolution mass spectrometry (MS) with genome-wide SNP data. We identified and quantified the abundance of 1,056 tryptic-digested peptides, representing 163 proteins in the plasma of 1,060 individuals from two population-based cohorts. The abundance level of almost one-fifth (19%) of the peptides was found to be heritable, with heritability ranging from 0.08 to 0.43. The levels of 60 peptides from 25 proteins, 15% of the proteins studied, were influenced by cis-acting SNPs. We identified and replicated individual cis-acting SNPs (combined P value ranging from 3.1 × 10(-52) to 2.9 × 10(-12)) influencing 11 peptides from 5 individual proteins. These SNPs represent both regulatory SNPs and nonsynonymous changes defining well-studied disease alleles such as the ε4 allele of apolipoprotein E (APOE), which has been shown to increase risk of Alzheimer's disease. Our results show that high-throughput mass spectrometry represents a promising method for large-scale characterization of the human proteome, allowing for both quantification and sequencing of individual proteins. Abundance and peptide composition of a protein plays an important role in the etiology, diagnosis, and treatment of a number of diseases. A better understanding of the genetic impact on the plasma proteome is therefore important for evaluating potential biomarkers and therapeutic agents for common diseases. </li>
<li><strong>DOI</strong> : 10.1073/pnas.1217238110 </li>
<li><strong>JOURNAL_INFO</strong> : Proceedings of the National Academy of Sciences of the United States of America ; Proc Natl Acad Sci U S A ; 2013 ; 110 ; 12 ; 4673-8 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23487758">23487758</a> </li>
</ul>
<h3 id="katz_dh-34814699">Katz_DH-34814699</h3>
<ul>
<li><strong>NAME</strong> : Katz_DH-34814699 </li>
<li><strong>TITLE</strong> : Whole Genome Sequence Analysis of the Plasma Proteome in Black Adults Provides Novel Insights Into Cardiovascular Disease. </li>
<li><strong>CITATION</strong> : . (2022) Whole Genome Sequence Analysis of the Plasma Proteome in Black Adults Provides Novel Insights Into Cardiovascular Disease. Circulation, 145 (5) 357-370. doi:10.1161/CIRCULATIONAHA.121.055117. PMID 34814699 </li>
<li><strong>MESH_MINOR</strong> : Adult, Black People, Cardiovascular Diseases, Female, Genome-Wide Association Study, Humans, Male, Proteome </li>
<li><strong>ABSTRACT</strong> : Plasma proteins are critical mediators of cardiovascular processes and are the targets of many drugs. Previous efforts to characterize the genetic architecture of the plasma proteome have been limited by a focus on individuals of European descent and leveraged genotyping arrays and imputation. Here we describe whole genome sequence analysis of the plasma proteome in individuals with greater African ancestry, increasing our power to identify novel genetic determinants.|Proteomic profiling of 1301 proteins was performed in 1852 Black adults from the Jackson Heart Study using aptamer-based proteomics (SomaScan). Whole genome sequencing association analysis was ascertained for all variants with minor allele count ≥5. Results were validated using an alternative, antibody-based, proteomic platform (Olink) as well as replicated in the Multi-Ethnic Study of Atherosclerosis and the HERITAGE Family Study (Health, Risk Factors, Exercise Training and Genetics).|We identify 569 genetic associations between 479 proteins and 438 unique genetic regions at a Bonferroni-adjusted significance level of 3.8×10<sup>-11</sup>. These associations include 114 novel locus-protein relationships and an additional 217 novel sentinel variant-protein relationships. Novel cardiovascular findings include new protein associations at the <i>APOE</i> gene locus including ZAP70 (sentinel single nucleotide polymorphism [SNP] rs7412-T, β=0.61±0.05, <i>P</i>=3.27×10<sup>-30</sup>) and MMP-3 (β=-0.60±0.05, <i>P</i>=1.67×10<sup>-32</sup>), as well as a completely novel pleiotropic locus at the <i>HPX</i> gene, associated with 9 proteins. Further, the associations suggest new mechanisms of genetically mediated cardiovascular disease linked to African ancestry; we identify a novel association between variants linked to APOL1-associated chronic kidney and heart disease and the protein CKAP2 (rs73885319-G, β=0.34±0.04, <i>P</i>=1.34×10<sup>-17</sup>) as well as an association between ATTR amyloidosis and RBP4 levels in community-dwelling individuals without heart failure.|Taken together, these results provide evidence for the functional importance of variants in non-European populations, and suggest new biological mechanisms for ancestry-specific determinants of lipids, coagulation, and myocardial function. </li>
<li><strong>DOI</strong> : 10.1161/CIRCULATIONAHA.121.055117 </li>
<li><strong>JOURNAL_INFO</strong> : Circulation ; Circulation ; 2022 ; 145 ; 5 ; 357-370 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34814699">34814699</a> </li>
</ul>
<h3 id="katz_dh-35984888">Katz_DH-35984888</h3>
<ul>
<li><strong>NAME</strong> : Katz_DH-35984888 </li>
<li><strong>TITLE</strong> : Proteomic profiling platforms head to head: Leveraging genetics and clinical traits to compare aptamer- and antibody-based methods. </li>
<li><strong>CITATION</strong> : Katz DH, ...&amp;, Gerszten RE. (2022) Proteomic profiling platforms head to head: Leveraging genetics and clinical traits to compare aptamer- and antibody-based methods. Sci Adv, 8 (33) eabm5164. doi:10.1126/sciadv.abm5164. PMID 35984888 </li>
<li><strong>MESH_MAJOR</strong> : Proteome, Proteomics </li>
<li><strong>MESH_MINOR</strong> : Adult, Antibodies, Aptamers, Peptide, Humans, Longitudinal Studies, Phenotype </li>
<li><strong>ABSTRACT</strong> : High-throughput proteomic profiling using antibody or aptamer-based affinity reagents is used increasingly in human studies. However, direct analyses to address the relative strengths and weaknesses of these platforms are lacking. We assessed findings from the SomaScan1.3K (<i>N</i> = 1301 reagents), the SomaScan5K platform (<i>N</i> = 4979 reagents), and the Olink Explore (<i>N</i> = 1472 reagents) profiling techniques in 568 adults from the Jackson Heart Study and 219 participants in the HERITAGE Family Study across four performance domains: precision, accuracy, analytic breadth, and phenotypic associations leveraging detailed clinical phenotyping and genetic data. Across these studies, we show evidence supporting more reliable protein target specificity and a higher number of phenotypic associations for the Olink platform, while the Soma platforms benefit from greater measurement precision and analytic breadth across the proteome. </li>
<li><strong>DOI</strong> : 10.1126/sciadv.abm5164 </li>
<li><strong>JOURNAL_INFO</strong> : Science advances ; Sci Adv ; 2022 ; 8 ; 33 ; eabm5164 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35984888">35984888</a> </li>
</ul>
<h3 id="kauwe_js-25340798">Kauwe_JS-25340798</h3>
<ul>
<li><strong>NAME</strong> : Kauwe_JS-25340798 </li>
<li><strong>TITLE</strong> : Genome-wide association study of CSF levels of 59 alzheimer's disease candidate proteins: significant associations with proteins involved in amyloid processing and inflammation. </li>
<li><strong>CITATION</strong> : Kauwe JS, ...&amp;, Goate AM. (2014) Genome-wide association study of CSF levels of 59 alzheimer's disease candidate proteins: significant associations with proteins involved in amyloid processing and inflammation. PLoS Genet, 10 (10) e1004758. doi:10.1371/journal.pgen.1004758. PMID 25340798 </li>
<li><strong>MESH_MINOR</strong> : Alzheimer Disease, Amyloid beta-Peptides, Blood Proteins, Chemokine CCL2, Chemokine CCL4, Female, Genome-Wide Association Study, Humans, Male, Matrix Metalloproteinase 3, Nerve Growth Factor, Polymorphism, Single Nucleotide, Receptors, Interleukin-6, Receptors, Lipoprotein, Renin, tau Proteins </li>
<li><strong>ABSTRACT</strong> : Cerebrospinal fluid (CSF) 42 amino acid species of amyloid beta (Aβ42) and tau levels are strongly correlated with the presence of Alzheimer's disease (AD) neuropathology including amyloid plaques and neurodegeneration and have been successfully used as endophenotypes for genetic studies of AD. Additional CSF analytes may also serve as useful endophenotypes that capture other aspects of AD pathophysiology. Here we have conducted a genome-wide association study of CSF levels of 59 AD-related analytes. All analytes were measured using the Rules Based Medicine Human DiscoveryMAP Panel, which includes analytes relevant to several disease-related processes. Data from two independently collected and measured datasets, the Knight Alzheimer's Disease Research Center (ADRC) and Alzheimer's Disease Neuroimaging Initiative (ADNI), were analyzed separately, and combined results were obtained using meta-analysis. We identified genetic associations with CSF levels of 5 proteins (Angiotensin-converting enzyme (ACE), Chemokine (C-C motif) ligand 2 (CCL2), Chemokine (C-C motif) ligand 4 (CCL4), Interleukin 6 receptor (IL6R) and Matrix metalloproteinase-3 (MMP3)) with study-wide significant p-values (p&lt;1.46×10-10) and significant, consistent evidence for association in both the Knight ADRC and the ADNI samples. These proteins are involved in amyloid processing and pro-inflammatory signaling. SNPs associated with ACE, IL6R and MMP3 protein levels are located within the coding regions of the corresponding structural gene. The SNPs associated with CSF levels of CCL4 and CCL2 are located in known chemokine binding proteins. The genetic associations reported here are novel and suggest mechanisms for genetic control of CSF and plasma levels of these disease-related proteins. Significant SNPs in ACE and MMP3 also showed association with AD risk. Our findings suggest that these proteins/pathways may be valuable therapeutic targets for AD. Robust associations in cognitively normal individuals suggest that these SNPs also influence regulation of these proteins more generally and may therefore be relevant to other diseases. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1004758 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2014 ; 10 ; 10 ; e1004758 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25340798">25340798</a> </li>
</ul>
<h3 id="kim_s-23894628">Kim_S-23894628</h3>
<ul>
<li><strong>NAME</strong> : Kim_S-23894628 </li>
<li><strong>TITLE</strong> : Influence of genetic variation on plasma protein levels in older adults using a multi-analyte panel. </li>
<li><strong>CITATION</strong> : . (2013) Influence of genetic variation on plasma protein levels in older adults using a multi-analyte panel. PLoS One, 8 (7) e70269. doi:10.1371/journal.pone.0070269. PMID 23894628 </li>
<li><strong>MESH_MAJOR</strong> : Polymorphism, Single Nucleotide </li>
<li><strong>MESH_MINOR</strong> : Aged, Aged, 80 and over, Alzheimer Disease, Apolipoproteins E, Complement C3b Inactivator Proteins, Female, Genome-Wide Association Study, Humans, Linear Models, Male, Middle Aged </li>
<li><strong>ABSTRACT</strong> : Proteins, widely studied as potential biomarkers, play important roles in numerous physiological functions and diseases. Genetic variation may modulate corresponding protein levels and point to the role of these variants in disease pathophysiology. Effects of individual single nucleotide polymorphisms (SNPs) within a gene were analyzed for corresponding plasma protein levels using genome-wide association study (GWAS) genotype data and proteomic panel data with 132 quality-controlled analytes from 521 Caucasian participants in the Alzheimer's Disease Neuroimaging Initiative (ADNI) cohort. Linear regression analysis detected 112 significant (Bonferroni threshold p=2.44×10(-5)) associations between 27 analytes and 112 SNPs. 107 out of these 112 associations were tested in the Indiana Memory and Aging Study (IMAS) cohort for replication and 50 associations were replicated at uncorrected p&lt;0.05 in the same direction of effect as those in the ADNI. We identified multiple novel associations including the association of rs7517126 with plasma complement factor H-related protein 1 (CFHR1) level at p&lt;1.46×10(-60), accounting for 40 percent of total variation of the protein level. We serendipitously found the association of rs6677604 with the same protein at p&lt;9.29×10(-112). Although these two SNPs were not in the strong linkage disequilibrium, 61 percent of total variation of CFHR1 was accounted for by rs6677604 without additional variation by rs7517126 when both SNPs were tested together. 78 other SNP-protein associations in the ADNI sample exceeded genome-wide significance (5×10(-8)). Our results confirmed previously identified gene-protein associations for interleukin-6 receptor, chemokine CC-4, angiotensin-converting enzyme, and angiotensinogen, although the direction of effect was reversed in some cases. This study is among the first analyses of gene-protein product relationships integrating multiplex-panel proteomics and targeted genes extracted from a GWAS array. With intensive searches taking place for proteomic biomarkers for many diseases, the role of genetic variation takes on new importance and should be considered in interpretation of proteomic results. </li>
<li><strong>DOI</strong> : 10.1371/journal.pone.0070269 </li>
<li><strong>JOURNAL_INFO</strong> : PloS one ; PLoS One ; 2013 ; 8 ; 7 ; e70269 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23894628">23894628</a> </li>
</ul>
<h3 id="koprulu_m-36823471">Koprulu_M-36823471</h3>
<ul>
<li><strong>NAME</strong> : Koprulu_M-36823471 </li>
<li><strong>TITLE</strong> : Proteogenomic links to human metabolic diseases. </li>
<li><strong>CITATION</strong> : Koprulu M, ...&amp;, Langenberg C. (2023) Proteogenomic links to human metabolic diseases. Nat Metab, 5 (3) 516-528. doi:10.1038/s42255-023-00753-7. PMID 36823471 </li>
<li><strong>MESH_MAJOR</strong> : Diabetes Mellitus, Type 2, Proteogenomics </li>
<li><strong>MESH_MINOR</strong> : Blood Proteins, Humans, Proteomics, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Studying the plasma proteome as the intermediate layer between the genome and the phenome has the potential to identify new disease processes. Here, we conducted a cis-focused proteogenomic analysis of 2,923 plasma proteins measured in 1,180 individuals using antibody-based assays. We (1) identify 256 unreported protein quantitative trait loci (pQTL); (2) demonstrate shared genetic regulation of 224 cis-pQTLs with 575 specific health outcomes, revealing examples for notable metabolic diseases (such as gastrin-releasing peptide as a potential therapeutic target for type 2 diabetes); (3) improve causal gene assignment at 40% (n = 192) of overlapping risk loci; and (4) observe convergence of phenotypic consequences of cis-pQTLs and rare loss-of-function gene burden for 12 proteins, such as TIMD4 for lipoprotein metabolism. Our findings demonstrate the value of integrating complementary proteomic technologies with genomics even at moderate scale to identify new mediators of metabolic diseases with the potential for therapeutic interventions. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s), under exclusive licence to Springer Nature Limited. </li>
<li><strong>DOI</strong> : 10.1038/s42255-023-00753-7 </li>
<li><strong>JOURNAL_INFO</strong> : Nature metabolism ; Nat Metab ; 2023 ; 5 ; 3 ; 516-528 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36823471">36823471</a> </li>
</ul>
<h3 id="macdonald-dunlop">Macdonald-Dunlop</h3>
<ul>
<li><strong>NAME</strong> : Macdonald-Dunlop </li>
<li><strong>PREPRINT_DOI</strong> : 2021.08.03.21261494 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>CITATION</strong> : <br><br>Macdonald-Dunlop, E. et al. Mapping genetic determinants of 184 circulating proteins in 26,494 individuals to connect proteins and diseases. bioRxiv (2021) doi:10.1101/2021.08.03.21261494.<br><br> </li>
</ul>
<h3 id="melzer_d-18464913">Melzer_D-18464913</h3>
<ul>
<li><strong>NAME</strong> : Melzer_D-18464913 </li>
<li><strong>TITLE</strong> : A genome-wide association study identifies protein quantitative trait loci (pQTLs). </li>
<li><strong>CITATION</strong> : Melzer D, ...&amp;, Ferrucci L. (2008) A genome-wide association study identifies protein quantitative trait loci (pQTLs). PLoS Genet, 4 (5) e1000072. doi:10.1371/journal.pgen.1000072. PMID 18464913 </li>
<li><strong>MESH_MAJOR</strong> : Genome, Human, Polymorphism, Single Nucleotide, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Adult, Aged, Aged, 80 and over, Blood Proteins, Female, Gene Dosage, Genetic Linkage, Genetic Variation, Genotype, Humans, Male, Middle Aged, Transcription, Genetic </li>
<li><strong>ABSTRACT</strong> : There is considerable evidence that human genetic variation influences gene expression. Genome-wide studies have revealed that mRNA levels are associated with genetic variation in or close to the gene coding for those mRNA transcripts - cis effects, and elsewhere in the genome - trans effects. The role of genetic variation in determining protein levels has not been systematically assessed. Using a genome-wide association approach we show that common genetic variation influences levels of clinically relevant proteins in human serum and plasma. We evaluated the role of 496,032 polymorphisms on levels of 42 proteins measured in 1200 fasting individuals from the population based InCHIANTI study. Proteins included insulin, several interleukins, adipokines, chemokines, and liver function markers that are implicated in many common diseases including metabolic, inflammatory, and infectious conditions. We identified eight Cis effects, including variants in or near the IL6R (p = 1.8x10(-57)), CCL4L1 (p = 3.9x10(-21)), IL18 (p = 6.8x10(-13)), LPA (p = 4.4x10(-10)), GGT1 (p = 1.5x10(-7)), SHBG (p = 3.1x10(-7)), CRP (p = 6.4x10(-6)) and IL1RN (p = 7.3x10(-6)) genes, all associated with their respective protein products with effect sizes ranging from 0.19 to 0.69 standard deviations per allele. Mechanisms implicated include altered rates of cleavage of bound to unbound soluble receptor (IL6R), altered secretion rates of different sized proteins (LPA), variation in gene copy number (CCL4L1) and altered transcription (GGT1). We identified one novel trans effect that was an association between ABO blood group and tumour necrosis factor alpha (TNF-alpha) levels (p = 6.8x10(-40)), but this finding was not present when TNF-alpha was measured using a different assay , or in a second study, suggesting an assay-specific association. Our results show that protein levels share some of the features of the genetics of gene expression. These include the presence of strong genetic effects in cis locations. The identification of protein quantitative trait loci (pQTLs) may be a powerful complementary method of improving our understanding of disease pathways. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1000072 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2008 ; 4 ; 5 ; e1000072 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/18464913">18464913</a> </li>
</ul>
<h3 id="pietzner_m-34648354">Pietzner_M-34648354</h3>
<ul>
<li><strong>NAME</strong> : Pietzner_M-34648354 </li>
<li><strong>TITLE</strong> : Mapping the proteo-genomic convergence of human diseases. </li>
<li><strong>CITATION</strong> : Pietzner M, ...&amp;, Langenberg C. (2021) Mapping the proteo-genomic convergence of human diseases. Science, 374 (6569) eabj1541. doi:10.1126/science.abj1541. PMID 34648354 </li>
<li><strong>MESH_MAJOR</strong> : Genome, Human, Genomics, Proteome </li>
<li><strong>MESH_MINOR</strong> : Aging, Alternative Splicing, Blood Proteins, COVID-19, Connective Tissue Diseases, Disease, Drug Development, Female, Gallstones, Genetic Association Studies, Genetic Variation, Genome-Wide Association Study, Humans, Internet, Male, Phenotype, Proteins, Quantitative Trait Loci, Sex Characteristics </li>
<li><strong>ABSTRACT</strong> : Characterization of the genetic regulation of proteins is essential for understanding disease etiology and developing therapies. We identified 10,674 genetic associations for 3892 plasma proteins to create a cis-anchored gene-protein-disease map of 1859 connections that highlights strong cross-disease biological convergence. This proteo-genomic map provides a framework to connect etiologically related diseases, to provide biological context for new or emerging disorders, and to integrate different biological domains to establish mechanisms for known gene-disease links. Our results identify proteo-genomic connections within and between diseases and establish the value of cis-protein variants for annotation of likely causal disease genes at loci identified in genome-wide association studies, thereby addressing a major barrier to experimental validation and clinical translation of genetic discoveries. </li>
<li><strong>DOI</strong> : 10.1126/science.abj1541 </li>
<li><strong>JOURNAL_INFO</strong> : Science (New York, N.Y.) ; Science ; 2021 ; 374 ; 6569 ; eabj1541 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34648354">34648354</a> </li>
</ul>
<h3 id="png_g-34857772">Png_G-34857772</h3>
<ul>
<li><strong>NAME</strong> : Png_G-34857772 </li>
<li><strong>TITLE</strong> : Mapping the serum proteome to neurological diseases using whole genome sequencing. </li>
<li><strong>CITATION</strong> : Png G, ...&amp;, Zeggini E. (2021) Mapping the serum proteome to neurological diseases using whole genome sequencing. Nat Commun, 12 (1) 7042. doi:10.1038/s41467-021-27387-1. PMID 34857772 </li>
<li><strong>MESH_MINOR</strong> : Alzheimer Disease, Biomarkers, Cohort Studies, Gene Expression, Gene Ontology, Genetic Predisposition to Disease, Genome, Human, Humans, Membrane Glycoproteins, Mendelian Randomization Analysis, Molecular Sequence Annotation, Parkinson Disease, Proteome, Quantitative Trait Loci, Scavenger Receptors, Class A, Schizophrenia, Sialic Acid Binding Ig-like Lectin 3, Whole Genome Sequencing </li>
<li><strong>ABSTRACT</strong> : Despite the increasing global burden of neurological disorders, there is a lack of effective diagnostic and therapeutic biomarkers. Proteins are often dysregulated in disease and have a strong genetic component. Here, we carry out a protein quantitative trait locus analysis of 184 neurologically-relevant proteins, using whole genome sequencing data from two isolated population-based cohorts (N = 2893). In doing so, we elucidate the genetic landscape of the circulating proteome and its connection to neurological disorders. We detect 214 independently-associated variants for 107 proteins, the majority of which (76%) are cis-acting, including 114 variants that have not been previously identified. Using two-sample Mendelian randomisation, we identify causal associations between serum CD33 and Alzheimer's disease, GPNMB and Parkinson's disease, and MSR1 and schizophrenia, describing their clinical potential and highlighting drug repurposing opportunities. </li>
<li><strong>COPYRIGHT</strong> : © 2021. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-27387-1 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2021 ; 12 ; 1 ; 7042 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34857772">34857772</a> </li>
</ul>
<h3 id="ruffieux_h-32492067">Ruffieux_H-32492067</h3>
<ul>
<li><strong>NAME</strong> : Ruffieux_H-32492067 </li>
<li><strong>TITLE</strong> : A fully joint Bayesian quantitative trait locus mapping of human protein abundance in plasma. </li>
<li><strong>CITATION</strong> : Ruffieux H, ...&amp;, Valsesia A. (2020) A fully joint Bayesian quantitative trait locus mapping of human protein abundance in plasma. PLoS Comput Biol, 16 (6) e1007882. doi:10.1371/journal.pcbi.1007882. PMID 32492067 </li>
<li><strong>MESH_MAJOR</strong> : Bayes Theorem, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Blood Proteins, Genome-Wide Association Study, Humans </li>
<li><strong>ABSTRACT</strong> : Molecular quantitative trait locus (QTL) analyses are increasingly popular to explore the genetic architecture of complex traits, but existing studies do not leverage shared regulatory patterns and suffer from a large multiplicity burden, which hampers the detection of weak signals such as trans associations. Here, we present a fully multivariate proteomic QTL (pQTL) analysis performed with our recently proposed Bayesian method LOCUS on data from two clinical cohorts, with plasma protein levels quantified by mass-spectrometry and aptamer-based assays. Our two-stage study identifies 136 pQTL associations in the first cohort, of which &gt;80% replicate in the second independent cohort and have significant enrichment with functional genomic elements and disease risk loci. Moreover, 78% of the pQTLs whose protein abundance was quantified by both proteomic techniques are confirmed across assays. Our thorough comparisons with standard univariate QTL mapping on (1) these data and (2) synthetic data emulating the real data show how LOCUS borrows strength across correlated protein levels and markers on a genome-wide scale to effectively increase statistical power. Notably, 15% of the pQTLs uncovered by LOCUS would be missed by the univariate approach, including several trans and pleiotropic hits with successful independent validation. Finally, the analysis of extensive clinical data from the two cohorts indicates that the genetically-driven proteins identified by LOCUS are enriched in associations with low-grade inflammation, insulin resistance and dyslipidemia and might therefore act as endophenotypes for metabolic diseases. While considerations on the clinical role of the pQTLs are beyond the scope of our work, these findings generate useful hypotheses to be explored in future research; all results are accessible online from our searchable database. Thanks to its efficient variational Bayes implementation, LOCUS can analyze jointly thousands of traits and millions of markers. Its applicability goes beyond pQTL studies, opening new perspectives for large-scale genome-wide association and QTL analyses. Diet, Obesity and Genes (DiOGenes) trial registration number: NCT00390637. </li>
<li><strong>DOI</strong> : 10.1371/journal.pcbi.1007882 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS computational biology ; PLoS Comput Biol ; 2020 ; 16 ; 6 ; e1007882 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32492067">32492067</a> </li>
</ul>
<h3 id="said">Said</h3>
<ul>
<li><strong>NAME</strong> : Said </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2023.11.13.23298365 </li>
<li><strong>SERVER</strong> : medrxiv </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#china-kadoorie-biobank">China Kadoorie Biobank</a> </li>
<li><strong>CITATION</strong> : <br><br>Said, S. et al. Ancestry diversity in the genetic determinants of the human plasma proteome and associated new drug targets. bioRxiv (2023) doi:10.1101/2023.11.13.23298365.<br><br> </li>
</ul>
<h3 id="sasayama_d-28031287">Sasayama_D-28031287</h3>
<ul>
<li><strong>NAME</strong> : Sasayama_D-28031287 </li>
<li><strong>TITLE</strong> : Genome-wide quantitative trait loci mapping of the human cerebrospinal fluid proteome. </li>
<li><strong>CITATION</strong> : Sasayama D, ...&amp;, Kunugi H. (2017) Genome-wide quantitative trait loci mapping of the human cerebrospinal fluid proteome. Hum Mol Genet, 26 (1) 44-51. doi:10.1093/hmg/ddw366. PMID 28031287 </li>
<li><strong>MESH_MAJOR</strong> : Genome, Human </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Genome-Wide Association Study, Humans, Mental Disorders, Phenotype, Polymorphism, Single Nucleotide, Protein Array Analysis, Proteome, Proteomics, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Cerebrospinal fluid (CSF) is virtually the only one accessible source of proteins derived from the central nervous system (CNS) of living humans and possibly reflects the pathophysiology of a variety of neuropsychiatric diseases. However, little is known regarding the genetic basis of variation in protein levels of human CSF. We examined CSF levels of 1,126 proteins in 133 subjects and performed a genome-wide association analysis of 514,227 single nucleotide polymorphisms (SNPs) to detect protein quantitative trait loci (pQTLs). To be conservative, Spearman's correlation was used to identify an association between genotypes of SNPs and protein levels. A total of 421 cis and 25 trans SNP-protein pairs were significantly correlated at a false discovery rate (FDR) of less than 0.01 (nominal P &lt; 7.66 × 10-9). Cis-only analysis revealed additional 580 SNP-protein pairs with FDR &lt; 0.01 (nominal P &lt; 2.13 × 10-5). pQTL SNPs were more likely, compared to non-pQTL SNPs, to be a disease/trait-associated variants identified by previous genome-wide association studies. The present findings suggest that genetic variations play an important role in the regulation of protein expression in the CNS. The obtained database may serve as a valuable resource to understand the genetic bases for CNS protein expression pattern in humans. </li>
<li><strong>COPYRIGHT</strong> : © The Author 2016. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com. </li>
<li><strong>DOI</strong> : 10.1093/hmg/ddw366 </li>
<li><strong>JOURNAL_INFO</strong> : Human molecular genetics ; Hum Mol Genet ; 2017 ; 26 ; 1 ; 44-51 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28031287">28031287</a> </li>
</ul>
<h3 id="suhre_k-28240269">Suhre_K-28240269</h3>
<ul>
<li><strong>NAME</strong> : Suhre_K-28240269 </li>
<li><strong>TITLE</strong> : Connecting genetic risk to disease end points through the human blood plasma proteome. </li>
<li><strong>CITATION</strong> : Suhre K, ...&amp;, Graumann J. (2017) Connecting genetic risk to disease end points through the human blood plasma proteome. Nat Commun, 8  14357. doi:10.1038/ncomms14357. PMID 28240269 </li>
<li><strong>MESH_MAJOR</strong> : Endpoint Determination, Genetic Predisposition to Disease </li>
<li><strong>MESH_MINOR</strong> : Alleles, Blood Proteins, Complement System Proteins, Drug Delivery Systems, Gene Regulatory Networks, Genetic Variation, Genome, Human, Genome-Wide Association Study, Glycoproteins, Heme, Humans, Molecular Sequence Annotation, Pharmacogenetics, Protein Processing, Post-Translational, Proteome, Quantitative Trait Loci, RNA Splicing, RNA, Messenger, Reproducibility of Results, Risk Factors </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) with intermediate phenotypes, like changes in metabolite and protein levels, provide functional evidence to map disease associations and translate them into clinical applications. However, although hundreds of genetic variants have been associated with complex disorders, the underlying molecular pathways often remain elusive. Associations with intermediate traits are key in establishing functional links between GWAS-identified risk-variants and disease end points. Here we describe a GWAS using a highly multiplexed aptamer-based affinity proteomics platform. We quantify 539 associations between protein levels and gene variants (pQTLs) in a German cohort and replicate over half of them in an Arab and Asian cohort. Fifty-five of the replicated pQTLs are located in trans. Our associations overlap with 57 genetic risk loci for 42 unique disease end points. We integrate this information into a genome-proteome network and provide an interactive web-tool for interrogations. Our results provide a basis for novel approaches to pharmaceutical and diagnostic applications. </li>
<li><strong>DOI</strong> : 10.1038/ncomms14357 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2017 ; 8 ; <NA> ; 14357 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28240269">28240269</a> </li>
</ul>
<h3 id="suhre_k-38412862">Suhre_K-38412862</h3>
<ul>
<li><strong>NAME</strong> : Suhre_K-38412862 </li>
<li><strong>DESCRIPTION</strong> : rQTLs </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions. </li>
<li><strong>CITATION</strong> : Suhre K. (2024) Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions. Cell Genom, 4 (3) 100506. doi:10.1016/j.xgen.2024.100506. PMID 38412862 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Predisposition to Disease, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Humans, Proteins </li>
<li><strong>ABSTRACT</strong> : Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development because they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease endpoints. Using Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank, we identified 4,248 associations with 2,821 ratios between protein levels (rQTLs). rQTLs were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. Conducting a GWAS on ratios increased the number of discovered genetic signals by 24.7%. The approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large-scale GWAS. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2024 The Author(s). Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2024.100506 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom ; 2024 ; 4 ; 3 ; 100506 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38412862">38412862</a> </li>
</ul>
<h3 id="sun_bb-29875488">Sun_BB-29875488</h3>
<ul>
<li><strong>NAME</strong> : Sun_BB-29875488 </li>
<li><strong>TITLE</strong> : Genomic atlas of the human plasma proteome. </li>
<li><strong>CITATION</strong> : Sun BB, ...&amp;, Butterworth AS. (2018) Genomic atlas of the human plasma proteome. Nature, 558 (7708) 73-79. doi:10.1038/s41586-018-0175-2. PMID 29875488 </li>
<li><strong>MESH_MAJOR</strong> : Genomics </li>
<li><strong>MESH_MINOR</strong> : Blood Proteins, Female, Hepatocyte Growth Factor, Humans, Inflammatory Bowel Diseases, Male, Mutation, Missense, Myeloblastin, Positive Regulatory Domain I-Binding Factor 1, Proteome, Proto-Oncogene Proteins, Quantitative Trait Loci, Vasculitis, alpha 1-Antitrypsin </li>
<li><strong>ABSTRACT</strong> : Although plasma proteins have important roles in biological processes and are the direct targets of many drugs, the genetic factors that control inter-individual variation in plasma protein levels are not well understood. Here we characterize the genetic architecture of the human plasma proteome in healthy blood donors from the INTERVAL study. We identify 1,927 genetic associations with 1,478 proteins, a fourfold increase on existing knowledge, including trans associations for 1,104 proteins. To understand the consequences of perturbations in plasma protein levels, we apply an integrated approach that links genetic variation with biological pathway, disease, and drug databases. We show that protein quantitative trait loci overlap with gene expression quantitative trait loci, as well as with disease-associated loci, and find evidence that protein biomarkers have causal roles in disease using Mendelian randomization analysis. By linking genetic factors to diseases via specific proteins, our analyses highlight potential therapeutic targets, opportunities for matching existing drugs with new disease indications, and potential safety concerns for drugs under development. </li>
<li><strong>DOI</strong> : 10.1038/s41586-018-0175-2 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2018 ; 558 ; 7708 ; 73-79 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29875488">29875488</a> </li>
</ul>
<h3 id="sun_bb-37794186">Sun_BB-37794186</h3>
<ul>
<li><strong>NAME</strong> : Sun_BB-37794186 </li>
<li><strong>TITLE</strong> : Plasma proteomic associations with genetics and health in the UK Biobank. </li>
<li><strong>CITATION</strong> : Sun BB, ...&amp;, Whelan CD. (2023) Plasma proteomic associations with genetics and health in the UK Biobank. Nature, 622 (7982) 329-338. doi:10.1038/s41586-023-06592-6. PMID 37794186 </li>
<li><strong>MESH_MAJOR</strong> : Biological Specimen Banks, Blood Proteins, Databases, Factual, Genomics, Health, Proteome, Proteomics </li>
<li><strong>MESH_MINOR</strong> : ABO Blood-Group System, COVID-19, Drug Discovery, Epistasis, Genetic, Fucosyltransferases, Galactoside 2-alpha-L-fucosyltransferase, Genetic Predisposition to Disease, Humans, Plasma, Proprotein Convertase 9, Public-Private Sector Partnerships, Quantitative Trait Loci, United Kingdom </li>
<li><strong>ABSTRACT</strong> : The Pharma Proteomics Project is a precompetitive biopharmaceutical consortium characterizing the plasma proteomic profiles of 54,219 UK Biobank participants. Here we provide a detailed summary of this initiative, including technical and biological validations, insights into proteomic disease signatures, and prediction modelling for various demographic and health indicators. We present comprehensive protein quantitative trait locus (pQTL) mapping of 2,923 proteins that identifies 14,287 primary genetic associations, of which 81% are previously undescribed, alongside ancestry-specific pQTL mapping in non-European individuals. The study provides an updated characterization of the genetic architecture of the plasma proteome, contextualized with projected pQTL discovery rates as sample sizes and proteomic assay coverages increase over time. We offer extensive insights into trans pQTLs across multiple biological domains, highlight genetic influences on ligand-receptor interactions and pathway perturbations across a diverse collection of cytokines and complement networks, and illustrate long-range epistatic effects of ABO blood group and FUT2 secretor status on proteins with gastrointestinal tissue-enriched expression. We demonstrate the utility of these data for drug discovery by extending the genetic proxied effects of protein targets, such as PCSK9, on additional endpoints, and disentangle specific genes and proteins perturbed at loci associated with COVID-19 susceptibility. This public-private partnership provides the scientific community with an open-access proteomics resource of considerable breadth and depth to help to elucidate the biological mechanisms underlying proteo-genomic discoveries and accelerate the development of biomarkers, predictive models and therapeutics<sup>1</sup>. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-06592-6 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 622 ; 7982 ; 329-338 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37794186">37794186</a> </li>
</ul>
<h3 id="sun_w-27532455">Sun_W-27532455</h3>
<ul>
<li><strong>NAME</strong> : Sun_W-27532455 </li>
<li><strong>TITLE</strong> : Common Genetic Polymorphisms Influence Blood Biomarker Measurements in COPD. </li>
<li><strong>CITATION</strong> : . (2016) Common Genetic Polymorphisms Influence Blood Biomarker Measurements in COPD. PLoS Genet, 12 (8) e1006011. doi:10.1371/journal.pgen.1006011. PMID 27532455 </li>
<li><strong>MESH_MINOR</strong> : ABO Blood-Group System, Biomarkers, Blood Proteins, Emphysema, Genetic Predisposition to Disease, Genome-Wide Association Study, Genotype, Humans, Polymorphism, Single Nucleotide, Pulmonary Disease, Chronic Obstructive, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Implementing precision medicine for complex diseases such as chronic obstructive lung disease (COPD) will require extensive use of biomarkers and an in-depth understanding of how genetic, epigenetic, and environmental variations contribute to phenotypic diversity and disease progression. A meta-analysis from two large cohorts of current and former smokers with and without COPD [SPIROMICS (N = 750); COPDGene (N = 590)] was used to identify single nucleotide polymorphisms (SNPs) associated with measurement of 88 blood proteins (protein quantitative trait loci; pQTLs). PQTLs consistently replicated between the two cohorts. Features of pQTLs were compared to previously reported expression QTLs (eQTLs). Inference of causal relations of pQTL genotypes, biomarker measurements, and four clinical COPD phenotypes (airflow obstruction, emphysema, exacerbation history, and chronic bronchitis) were explored using conditional independence tests. We identified 527 highly significant (p &lt; 8 X 10-10) pQTLs in 38 (43%) of blood proteins tested. Most pQTL SNPs were novel with low overlap to eQTL SNPs. The pQTL SNPs explained &gt;10% of measured variation in 13 protein biomarkers, with a single SNP (rs7041; p = 10-392) explaining 71%-75% of the measured variation in vitamin D binding protein (gene = GC). Some of these pQTLs [e.g., pQTLs for VDBP, sRAGE (gene = AGER), surfactant protein D (gene = SFTPD), and TNFRSF10C] have been previously associated with COPD phenotypes. Most pQTLs were local (cis), but distant (trans) pQTL SNPs in the ABO blood group locus were the top pQTL SNPs for five proteins. The inclusion of pQTL SNPs improved the clinical predictive value for the established association of sRAGE and emphysema, and the explanation of variance (R2) for emphysema improved from 0.3 to 0.4 when the pQTL SNP was included in the model along with clinical covariates. Causal modeling provided insight into specific pQTL-disease relationships for airflow obstruction and emphysema. In conclusion, given the frequency of highly significant local pQTLs, the large amount of variance potentially explained by pQTL, and the differences observed between pQTLs and eQTLs SNPs, we recommend that protein biomarker-disease association studies take into account the potential effect of common local SNPs and that pQTLs be integrated along with eQTLs to uncover disease mechanisms. Large-scale blood biomarker studies would also benefit from close attention to the ABO blood group. </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1006011 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet ; 2016 ; 12 ; 8 ; e1006011 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/27532455">27532455</a> </li>
</ul>
<h3 id="surapaneni_a-35870639">Surapaneni_A-35870639</h3>
<ul>
<li><strong>NAME</strong> : Surapaneni_A-35870639 </li>
<li><strong>TITLE</strong> : Identification of 969 protein quantitative trait loci in an African American population with kidney disease attributed to hypertension. </li>
<li><strong>CITATION</strong> : Surapaneni A, ...&amp;, Grams ME. (2022) Identification of 969 protein quantitative trait loci in an African American population with kidney disease attributed to hypertension. Kidney Int, 102 (5) 1167-1177. doi:10.1016/j.kint.2022.07.005. PMID 35870639 </li>
<li><strong>MESH_MAJOR</strong> : Hypertension, Kidney Diseases </li>
<li><strong>MESH_MINOR</strong> : Black or African American, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Polymorphism, Single Nucleotide, Proteome, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Investigations into the causal underpinnings of disease processes can be aided by the incorporation of genetic information. Genetic studies require populations varied in both ancestry and prevalent disease in order to optimize discovery and ensure generalizability of findings to the global population. Here, we report the genetic determinants of the serum proteome in 466 African Americans with chronic kidney disease attributed to hypertension from the richly phenotyped African American Study of Kidney Disease and Hypertension (AASK) study. Using the largest aptamer-based protein profiling platform to date (6,790 proteins or protein complexes), we identified 969 genetic associations with 900 unique proteins; including 52 novel cis (local) associations and 379 novel trans (distant) associations. The genetic effects of previously published cis-protein quantitative trait loci (pQTLs) were found to be highly reproducible, and we found evidence that our novel genetic signals colocalize with gene expression and disease processes. Many trans- pQTLs were found to reflect associations mediated by the circulating cis protein, and the common trans-pQTLs are enriched for processes involving extracellular vesicles, highlighting a plausible mechanism for distal regulation of the levels of secreted proteins. Thus, our study generates a valuable resource of genetic associations linking variants to protein levels and disease in an understudied patient population to inform future studies of drug targets and physiology. </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2022 International Society of Nephrology. Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.kint.2022.07.005 </li>
<li><strong>JOURNAL_INFO</strong> : Kidney international ; Kidney Int ; 2022 ; 102 ; 5 ; 1167-1177 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35870639">35870639</a> </li>
</ul>
<h3 id="thareja_g-36168886">Thareja_G-36168886</h3>
<ul>
<li><strong>NAME</strong> : Thareja_G-36168886 </li>
<li><strong>TITLE</strong> : Differences and commonalities in the genetic architecture of protein quantitative trait loci in European and Arab populations. </li>
<li><strong>CITATION</strong> : Thareja G, ...&amp;, Suhre K. (2023) Differences and commonalities in the genetic architecture of protein quantitative trait loci in European and Arab populations. Hum Mol Genet, 32 (6) 907-916. doi:10.1093/hmg/ddac243. PMID 36168886 </li>
<li><strong>MESH_MAJOR</strong> : Arabs, Quantitative Trait Loci, White People </li>
<li><strong>MESH_MINOR</strong> : Genetics, Population, Genome-Wide Association Study, Humans </li>
<li><strong>ABSTRACT</strong> : Polygenic scores (PGS) can identify individuals at risk of adverse health events and guide genetics-based personalized medicine. However, it is not clear how well PGS translate between different populations, limiting their application to well-studied ethnicities. Proteins are intermediate traits linking genetic predisposition and environmental factors to disease, with numerous blood circulating protein levels representing functional readouts of disease-related processes. We hypothesized that studying the genetic architecture of a comprehensive set of blood-circulating proteins between a European and an Arab population could shed fresh light on the translatability of PGS to understudied populations. We therefore conducted a genome-wide association study with whole-genome sequencing data using 1301 proteins measured on the SOMAscan aptamer-based affinity proteomics platform in 2935 samples of Qatar Biobank and evaluated the replication of protein quantitative traits (pQTLs) from European studies in an Arab population. Then, we investigated the colocalization of shared pQTL signals between the two populations. Finally, we compared the performance of protein PGS derived from a Caucasian population in a European and an Arab cohort. We found that the majority of shared pQTL signals (81.8%) colocalized between both populations. About one-third of the genetic protein heritability was explained by protein PGS derived from a European cohort, with protein PGS performing ~20% better in Europeans when compared to Arabs. Our results are relevant for the translation of PGS to non-Caucasian populations, as well as for future efforts to extend genetic research to understudied populations. </li>
<li><strong>COPYRIGHT</strong> : © The Author(s) 2022. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com. </li>
<li><strong>DOI</strong> : 10.1093/hmg/ddac243 </li>
<li><strong>JOURNAL_INFO</strong> : Human molecular genetics ; Hum Mol Genet ; 2023 ; 32 ; 6 ; 907-916 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36168886">36168886</a> </li>
</ul>
<h3 id="xu_f-36797296">Xu_F-36797296</h3>
<ul>
<li><strong>NAME</strong> : Xu_F-36797296 </li>
<li><strong>TITLE</strong> : Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility. </li>
<li><strong>CITATION</strong> : Xu F, ...&amp;, Zheng JS. (2023) Genome-wide genotype-serum proteome mapping provides insights into the cross-ancestry differences in cardiometabolic disease susceptibility. Nat Commun, 14 (1) 896. doi:10.1038/s41467-023-36491-3. PMID 36797296 </li>
<li><strong>MESH_MAJOR</strong> : Cardiovascular Diseases, Proteome </li>
<li><strong>MESH_MINOR</strong> : Genetic Predisposition to Disease, Genome-Wide Association Study, Genotype, Humans, Phenotype, Polymorphism, Single Nucleotide </li>
<li><strong>ABSTRACT</strong> : Identification of protein quantitative trait loci (pQTL) helps understand the underlying mechanisms of diseases and discover promising targets for pharmacological intervention. For most important class of drug targets, genetic evidence needs to be generalizable to diverse populations. Given that the majority of the previous studies were conducted in European ancestry populations, little is known about the protein-associated genetic variants in East Asians. Based on data-independent acquisition mass spectrometry technique, we conduct genome-wide association analyses for 304 unique proteins in 2,958 Han Chinese participants. We identify 195 genetic variant-protein associations. Colocalization and Mendelian randomization analyses highlight 60 gene-protein-phenotype associations, 45 of which (75%) have not been prioritized in Europeans previously. Further cross-ancestry analyses uncover key proteins that contributed to the differences in the obesity-induced diabetes and coronary artery disease susceptibility. These findings provide novel druggable proteins as well as a unique resource for the trans-ancestry evaluation of protein-targeted drug discovery. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-36491-3 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2023 ; 14 ; 1 ; 896 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36797296">36797296</a> </li>
</ul>
<h3 id="yang_c-34239129">Yang_C-34239129</h3>
<ul>
<li><strong>NAME</strong> : Yang_C-34239129 </li>
<li><strong>TITLE</strong> : Genomic atlas of the proteome from brain, CSF and plasma prioritizes proteins implicated in neurological disorders. </li>
<li><strong>CITATION</strong> : Yang C, ...&amp;, Cruchaga C. (2021) Genomic atlas of the proteome from brain, CSF and plasma prioritizes proteins implicated in neurological disorders. Nat Neurosci, 24 (9) 1302-1312. doi:10.1038/s41593-021-00886-6. PMID 34239129 </li>
<li><strong>MESH_MAJOR</strong> : Alzheimer Disease, Quantitative Trait Loci </li>
<li><strong>MESH_MINOR</strong> : Aged, Brain, Cerebrospinal Fluid, Female, Gene Expression Regulation, High-Throughput Screening Assays, Humans, Male, Middle Aged, Plasma, Proteome, Proteomics </li>
<li><strong>ABSTRACT</strong> : Understanding the tissue-specific genetic controls of protein levels is essential to uncover mechanisms of post-transcriptional gene regulation. In this study, we generated a genomic atlas of protein levels in three tissues relevant to neurological disorders (brain, cerebrospinal fluid and plasma) by profiling thousands of proteins from participants with and without Alzheimer's disease. We identified 274, 127 and 32 protein quantitative trait loci (pQTLs) for cerebrospinal fluid, plasma and brain, respectively. cis-pQTLs were more likely to be tissue shared, but trans-pQTLs tended to be tissue specific. Between 48.0% and 76.6% of pQTLs did not co-localize with expression, splicing, DNA methylation or histone acetylation QTLs. Using Mendelian randomization, we nominated proteins implicated in neurological diseases, including Alzheimer's disease, Parkinson's disease and stroke. This first multi-tissue study will be instrumental to map signals from genome-wide association studies onto functional genes, to discover pathways and to identify drug targets for neurological diseases. </li>
<li><strong>COPYRIGHT</strong> : © 2021. The Author(s), under exclusive licence to Springer Nature America, Inc. </li>
<li><strong>DOI</strong> : 10.1038/s41593-021-00886-6 </li>
<li><strong>JOURNAL_INFO</strong> : Nature neuroscience ; Nat Neurosci ; 2021 ; 24 ; 9 ; 1302-1312 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34239129">34239129</a> </li>
</ul>
<h3 id="yao_c-30111768">Yao_C-30111768</h3>
<ul>
<li><strong>NAME</strong> : Yao_C-30111768 </li>
<li><strong>TITLE</strong> : Genome-wide mapping of plasma protein QTLs identifies putatively causal genes and pathways for cardiovascular disease. </li>
<li><strong>CITATION</strong> : Yao C, ...&amp;, Levy D. (2018) Genome-wide mapping of plasma protein QTLs identifies putatively causal genes and pathways for cardiovascular disease. Nat Commun, 9 (1) 3268. doi:10.1038/s41467-018-05512-x. PMID 30111768 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study </li>
<li><strong>MESH_MINOR</strong> : Adult, Blood Proteins, Cardiovascular Diseases, Chromosome Mapping, Female, Gene Expression Profiling, Genetic Predisposition to Disease, Humans, Male, Middle Aged, Polymorphism, Single Nucleotide, Quantitative Trait Loci, Risk Factors, Signal Transduction </li>
<li><strong>ABSTRACT</strong> : Identifying genetic variants associated with circulating protein concentrations (protein quantitative trait loci; pQTLs) and integrating them with variants from genome-wide association studies (GWAS) may illuminate the proteome's causal role in disease and bridge a knowledge gap regarding SNP-disease associations. We provide the results of GWAS of 71 high-value cardiovascular disease proteins in 6861 Framingham Heart Study participants and independent external replication. We report the mapping of over 16,000 pQTL variants and their functional relevance. We provide an integrated plasma protein-QTL database. Thirteen proteins harbor pQTL variants that match coronary disease-risk variants from GWAS or test causal for coronary disease by Mendelian randomization. Eight of these proteins predict new-onset cardiovascular disease events in Framingham participants. We demonstrate that identifying pQTLs, integrating them with GWAS results, employing Mendelian randomization, and prospectively testing protein-trait associations holds potential for elucidating causal genes, proteins, and pathways for cardiovascular disease and may identify targets for its prevention and treatment. </li>
<li><strong>DOI</strong> : 10.1038/s41467-018-05512-x </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2018 ; 9 ; 1 ; 3268 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30111768">30111768</a> </li>
</ul>
<h3 id="zhong_w-32576278">Zhong_W-32576278</h3>
<ul>
<li><strong>NAME</strong> : Zhong_W-32576278 </li>
<li><strong>TITLE</strong> : Whole-genome sequence association analysis of blood proteins in a longitudinal wellness cohort. </li>
<li><strong>CITATION</strong> : Zhong W, ...&amp;, Uhlén M. (2020) Whole-genome sequence association analysis of blood proteins in a longitudinal wellness cohort. Genome Med, 12 (1) 53. doi:10.1186/s13073-020-00755-0. PMID 32576278 </li>
<li><strong>MESH_MINOR</strong> : Aged, Blood Proteins, Cohort Studies, Female, Genetic Predisposition to Disease, Genetic Variation, Genome-Wide Association Study, Humans, Male, Middle Aged, Proteome, Whole Genome Sequencing </li>
<li><strong>ABSTRACT</strong> : The human plasma proteome is important for many biological processes and targets for diagnostics and therapy. It is therefore of great interest to understand the interplay of genetic and environmental factors to determine the specific protein levels in individuals and to gain a deeper insight of the importance of genetic architecture related to the individual variability of plasma levels of proteins during adult life.|We have combined whole-genome sequencing, multiplex plasma protein profiling, and extensive clinical phenotyping in a longitudinal 2-year wellness study of 101 healthy individuals with repeated sampling. Analyses of genetic and non-genetic associations related to the variability of blood levels of proteins in these individuals were performed.|The analyses showed that each individual has a unique protein profile, and we report on the intra-individual as well as inter-individual variation for 794 plasma proteins. A genome-wide association study (GWAS) using 7.3 million genetic variants identified by whole-genome sequencing revealed 144 independent variants across 107 proteins that showed strong association (P &lt; 6 × 10<sup>-11</sup>) between genetics and the inter-individual variability on protein levels. Many proteins not reported before were identified (67 out of 107) with individual plasma level affected by genetics. Our longitudinal analysis further demonstrates that these levels are stable during the 2-year study period. The variability of protein profiles as a consequence of environmental factors was also analyzed with focus on the effects of weight loss and infections.|We show that the adult blood levels of many proteins are determined at birth by genetics, which is important for efforts aimed to understand the relationship between plasma proteome profiles and human biology and disease. </li>
<li><strong>DOI</strong> : 10.1186/s13073-020-00755-0 </li>
<li><strong>JOURNAL_INFO</strong> : Genome medicine ; Genome Med ; 2020 ; 12 ; 1 ; 53 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32576278">32576278</a> </li>
</ul>
<h2 id="hla-gwas">HLA-GWAS</h2>
<h3 id="krishna_c-39085222">Krishna_C-39085222</h3>
<ul>
<li><strong>NAME</strong> : Krishna_C-39085222 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : <a href="../Sumstats_Biobanks_Cohorts_README/#uk-biobank">UK Biobank</a> </li>
<li><strong>TITLE</strong> : The influence of HLA genetic variation on plasma protein expression. </li>
<li><strong>CITATION</strong> : Krishna C, ...&amp;, Hu X. (2024) The influence of HLA genetic variation on plasma protein expression. Nat Commun, 15 (1) 6469. doi:10.1038/s41467-024-50583-8. PMID 39085222 </li>
<li><strong>MESH_MAJOR</strong> : Alleles, Blood Proteins, Genetic Variation, HLA Antigens </li>
<li><strong>MESH_MINOR</strong> : Autoimmune Diseases, Genetic Predisposition to Disease, Humans, Immunity, Innate, Polymorphism, Single Nucleotide, United Kingdom, White People </li>
<li><strong>ABSTRACT</strong> : Genetic variation in the human leukocyte antigen (HLA) loci is associated with risk of immune-mediated diseases, but the molecular effects of HLA polymorphism are unclear. Here we examined the effects of HLA genetic variation on the expression of 2940 plasma proteins across 45,330 Europeans in the UK Biobank, with replication analyses across multiple ancestry groups. We detected 504 proteins affected by HLA variants (HLA-pQTL), including widespread trans effects by autoimmune disease risk alleles. More than 80% of the HLA-pQTL fine-mapped to amino acid positions in the peptide binding groove. HLA-I and II affected proteins expressed in similar cell types but in different pathways of both adaptive and innate immunity. Finally, we investigated potential HLA-pQTL effects on disease by integrating HLA-pQTL with fine-mapped HLA-disease signals in the UK Biobank. Our data reveal the diverse effects of HLA genetic variation and aid the interpretation of associations between HLA alleles and immune-mediated diseases. </li>
<li><strong>COPYRIGHT</strong> : © 2024. The Author(s). </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-50583-8 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2024 ; 15 ; 1 ; 6469 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39085222">39085222</a> </li>
</ul>
<h2 id="platform">Platform</h2>
<h3 id="omicspred-portal">OmicsPred portal</h3>
<ul>
<li><strong>NAME</strong> : OmicsPred portal </li>
<li><strong>URL</strong> : <a href="https://www.omicspred.org/">https://www.omicspred.org/</a> </li>
<li><strong>TITLE</strong> : An atlas of genetic scores to predict multi-omic traits. </li>
<li><strong>CITATION</strong> : Xu Y, ...&amp;, Inouye M. (2023) An atlas of genetic scores to predict multi-omic traits. Nature, 616 (7955) 123-131. doi:10.1038/s41586-023-05844-9. PMID 36991119 </li>
<li><strong>MESH_MAJOR</strong> : Coronary Artery Disease, Multiomics </li>
<li><strong>MESH_MINOR</strong> : Asian, Black or African American, Cohort Studies, Databases, Factual, Datasets as Topic, European People, Humans, Internet, Machine Learning, Metabolome, Metabolomics, Phenotype, Plasma, Proteome, Proteomics, Reproducibility of Results, United Kingdom </li>
<li><strong>ABSTRACT</strong> : The use of omic modalities to dissect the molecular underpinnings of common diseases and traits is becoming increasingly common. But multi-omic traits can be genetically predicted, which enables highly cost-effective and powerful analyses for studies that do not have multi-omics<sup>1</sup>. Here we examine a large cohort (the INTERVAL study<sup>2</sup>; n = 50,000 participants) with extensive multi-omic data for plasma proteomics (SomaScan, n = 3,175; Olink, n = 4,822), plasma metabolomics (Metabolon HD4, n = 8,153), serum metabolomics (Nightingale, n = 37,359) and whole-blood Illumina RNA sequencing (n = 4,136), and use machine learning to train genetic scores for 17,227 molecular traits, including 10,521 that reach Bonferroni-adjusted significance. We evaluate the performance of genetic scores through external validation across cohorts of individuals of European, Asian and African American ancestries. In addition, we show the utility of these multi-omic genetic scores by quantifying the genetic control of biological pathways and by generating a synthetic multi-omic dataset of the UK Biobank<sup>3</sup> to identify disease associations using a phenome-wide scan. We highlight a series of biological insights with regard to genetic mechanisms in metabolism and canonical pathway associations with disease; for example, JAK-STAT signalling and coronary atherosclerosis. Finally, we develop a portal ( https://www.omicspred.org/ ) to facilitate public access to all genetic scores and validation results, as well as to serve as a platform for future extensions and enhancements of multi-omic genetic scores. </li>
<li><strong>COPYRIGHT</strong> : © 2023. The Author(s), under exclusive licence to Springer Nature Limited. </li>
<li><strong>DOI</strong> : 10.1038/s41586-023-05844-9 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2023 ; 616 ; 7955 ; 123-131 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36991119">36991119</a> </li>
</ul>
<h3 id="proteome-phewas-browser">Proteome PheWAS browser</h3>
<ul>
<li><strong>NAME</strong> : Proteome PheWAS browser </li>
<li><strong>TITLE</strong> : Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases. </li>
<li><strong>CITATION</strong> : Zheng J, ...&amp;, Gaunt TR. (2020) Phenome-wide Mendelian randomization mapping the influence of the plasma proteome on complex diseases. Nat Genet, 52 (10) 1122-1131. doi:10.1038/s41588-020-0682-6. PMID 32895551 </li>
<li><strong>MESH_MAJOR</strong> : Genetic Predisposition to Disease, Mendelian Randomization Analysis </li>
<li><strong>MESH_MINOR</strong> : Blood Proteins, Genome-Wide Association Study, Humans, Phenotype, Polymorphism, Single Nucleotide, Proteome </li>
<li><strong>ABSTRACT</strong> : The human proteome is a major source of therapeutic targets. Recent genetic association analyses of the plasma proteome enable systematic evaluation of the causal consequences of variation in plasma protein levels. Here we estimated the effects of 1,002 proteins on 225 phenotypes using two-sample Mendelian randomization (MR) and colocalization. Of 413 associations supported by evidence from MR, 130 (31.5%) were not supported by results of colocalization analyses, suggesting that genetic confounding due to linkage disequilibrium is widespread in naïve phenome-wide association studies of proteins. Combining MR and colocalization evidence in cis-only analyses, we identified 111 putatively causal effects between 65 proteins and 52 disease-related phenotypes ( https://www.epigraphdb.org/pqtl/ ). Evaluation of data from historic drug development programs showed that target-indication pairs with MR and colocalization support were more likely to be approved, evidencing the value of this approach in identifying and prioritizing potential therapeutic targets. </li>
<li><strong>DOI</strong> : 10.1038/s41588-020-0682-6 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2020 ; 52 ; 10 ; 1122-1131 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32895551">32895551</a> </li>
</ul>
<h3 id="pgwas-server">pGWAS server</h3>
<ul>
<li><strong>NAME</strong> : pGWAS server </li>
<li><strong>DESCRIPTION</strong> : In our study, we performed a genome-wide association study with protein levels (pGWAS). Using a highly multiplexed, aptamer-based, affinity proteomics platform (SOMAscan™), we quantified levels of 1,124 proteins in blood plasma samples from 1,000 German individuals (KORA cohort) and 338 Arab or Asian individuals (QMDiab cohort). We identified 539 independent, genome-wide significant SNP-to-protein associations, which can be investigated using this webserver. </li>
<li><strong>URL</strong> : <a href="https://metabolomics.helmholtz-muenchen.de/pgwas/">https://metabolomics.helmholtz-muenchen.de/pgwas/</a> </li>
<li><strong>TITLE</strong> : Connecting genetic risk to disease end points through the human blood plasma proteome. </li>
<li><strong>CITATION</strong> : Suhre K, ...&amp;, Graumann J. (2017) Connecting genetic risk to disease end points through the human blood plasma proteome. Nat Commun, 8  14357. doi:10.1038/ncomms14357. PMID 28240269 </li>
<li><strong>MESH_MAJOR</strong> : Endpoint Determination, Genetic Predisposition to Disease </li>
<li><strong>MESH_MINOR</strong> : Alleles, Blood Proteins, Complement System Proteins, Drug Delivery Systems, Gene Regulatory Networks, Genetic Variation, Genome, Human, Genome-Wide Association Study, Glycoproteins, Heme, Humans, Molecular Sequence Annotation, Pharmacogenetics, Protein Processing, Post-Translational, Proteome, Quantitative Trait Loci, RNA Splicing, RNA, Messenger, Reproducibility of Results, Risk Factors </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies (GWAS) with intermediate phenotypes, like changes in metabolite and protein levels, provide functional evidence to map disease associations and translate them into clinical applications. However, although hundreds of genetic variants have been associated with complex disorders, the underlying molecular pathways often remain elusive. Associations with intermediate traits are key in establishing functional links between GWAS-identified risk-variants and disease end points. Here we describe a GWAS using a highly multiplexed aptamer-based affinity proteomics platform. We quantify 539 associations between protein levels and gene variants (pQTLs) in a German cohort and replicate over half of them in an Arab and Asian cohort. Fifty-five of the replicated pQTLs are located in trans. Our associations overlap with 57 genetic risk loci for 42 unique disease end points. We integrate this information into a genome-proteome network and provide an interactive web-tool for interrogations. Our results provide a basis for novel approaches to pharmaceutical and diagnostic applications. </li>
<li><strong>DOI</strong> : 10.1038/ncomms14357 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat Commun ; 2017 ; 8 ; <NA> ; 14357 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28240269">28240269</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-suhre_k-32860016">Review-Suhre_K-32860016</h3>
<ul>
<li><strong>NAME</strong> : Review-Suhre_K-32860016 </li>
<li><strong>DESCRIPTION</strong> : A Table of all published GWAS with proteomics </li>
<li><strong>URL</strong> : <a href="http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/">http://www.metabolomix.com/a-table-of-all-published-gwas-with-proteomics/</a> </li>
<li><strong>TITLE</strong> : Genetics meets proteomics: perspectives for large population-based studies. </li>
<li><strong>CITATION</strong> : Suhre K, ...&amp;, Schwenk JM. (2021) Genetics meets proteomics: perspectives for large population-based studies. Nat Rev Genet, 22 (1) 19-37. doi:10.1038/s41576-020-0268-2. PMID 32860016 </li>
<li><strong>MESH_MAJOR</strong> : Genome-Wide Association Study, Proteomics </li>
<li><strong>MESH_MINOR</strong> : Biomarkers, Blood Proteins, Humans, Phenotype, Proteome </li>
<li><strong>ABSTRACT</strong> : Proteomic analysis of cells, tissues and body fluids has generated valuable insights into the complex processes influencing human biology. Proteins represent intermediate phenotypes for disease and provide insight into how genetic and non-genetic risk factors are mechanistically linked to clinical outcomes. Associations between protein levels and DNA sequence variants that colocalize with risk alleles for common diseases can expose disease-associated pathways, revealing novel drug targets and translational biomarkers. However, genome-wide, population-scale analyses of proteomic data are only now emerging. Here, we review current findings from studies of the plasma proteome and discuss their potential for advancing biomedical translation through the interpretation of genome-wide association analyses. We highlight the challenges faced by currently available technologies and provide perspectives relevant to their future application in large-scale biobank studies. </li>
<li><strong>DOI</strong> : 10.1038/s41576-020-0268-2 </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Genetics ; Nat Rev Genet ; 2021 ; 22 ; 1 ; 19-37 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32860016">32860016</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>