
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Phylogenetic_tree/">
      
      
        <link rel="next" href="../Tools_Power_analysis/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>Polygenic_risk_scores - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Polygenic_risk_scores
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Biobanks/" class="md-tabs__link">
          
  
  Biobanks

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../References/" class="md-tabs__link">
          
  
  References

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Single_Cell/" class="md-tabs__link">
          
  
  Single_Cell

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Biobanks
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AFRICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AFRICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_AMERICA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    AMERICA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_ASIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ASIA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_EUROPE/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    EUROPE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Biobanks_OCIENIA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    OCIENIA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    References
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            References
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    References
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Epigenetics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Gene/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Genome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Phenotype/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_Variant/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../References_scRNA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    scRNA
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Single_Cell
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Single_Cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Framework/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Framework
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Harmonization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Harmonization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Network/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Network
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_QC/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    QC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Single_Cell_Trajectory/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Trajectory
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_General/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    General
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Imaging/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Metabolomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Proteomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" checked>
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Admixture/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6_4" >
        
          
          <label class="md-nav__link" for="__nav_6_4" id="__nav_6_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_6_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6_4">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Functional_prediction/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Functional_prediction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Genealogy/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genealogy
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Phylogenetic_tree/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#benchmark" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Benchmark">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#benchmark-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark-Wang
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bias" class="md-nav__link">
    <span class="md-ellipsis">
      Bias
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Bias">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ellis-ca" class="md-nav__link">
    <span class="md-ellipsis">
      Ellis CA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-credible-intervals" class="md-nav__link">
    <span class="md-ellipsis">
      PRS credible intervals
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cross-population" class="md-nav__link">
    <span class="md-ellipsis">
      Cross-population
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Cross-population">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bridgeprs" class="md-nav__link">
    <span class="md-ellipsis">
      BridgePRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ct-sleb" class="md-nav__link">
    <span class="md-ellipsis">
      CT-SLEB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prosper" class="md-nav__link">
    <span class="md-ellipsis">
      PROSPER
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-csx" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CSx
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-fh" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-FH
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdprx" class="md-nav__link">
    <span class="md-ellipsis">
      SDPRX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tl-prs" class="md-nav__link">
    <span class="md-ellipsis">
      TL-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shaprs" class="md-nav__link">
    <span class="md-ellipsis">
      shaPRS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cross-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Cross-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Cross-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ddx-prs" class="md-nav__link">
    <span class="md-ellipsis">
      DDx-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixer" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multi-pgs" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-PGS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#puma-cubs" class="md-nav__link">
    <span class="md-ellipsis">
      PUMA-CUBS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wmt-sblup" class="md-nav__link">
    <span class="md-ellipsis">
      wMT-SBLUP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scprs" class="md-nav__link">
    <span class="md-ellipsis">
      scPRS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pathway" class="md-nav__link">
    <span class="md-ellipsis">
      Pathway
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Pathway">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#prset" class="md-nav__link">
    <span class="md-ellipsis">
      PRSet
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pipeline" class="md-nav__link">
    <span class="md-ellipsis">
      Pipeline
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Pipeline">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prsice-2" class="md-nav__link">
    <span class="md-ellipsis">
      PRSice-2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgsc_calc" class="md-nav__link">
    <span class="md-ellipsis">
      pgsc_calc
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cancer-prsweb" class="md-nav__link">
    <span class="md-ellipsis">
      Cancer PRSweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exprsweb" class="md-nav__link">
    <span class="md-ellipsis">
      ExPRSweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgscatalog" class="md-nav__link">
    <span class="md-ellipsis">
      PGSCatalog
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgsfusion" class="md-nav__link">
    <span class="md-ellipsis">
      PGSFusion
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      PRS atlas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metabolites-prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      metabolites PRS atlas
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#polygenicity" class="md-nav__link">
    <span class="md-ellipsis">
      Polygenicity
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Polygenicity">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesr" class="md-nav__link">
    <span class="md-ellipsis">
      BayesR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayess" class="md-nav__link">
    <span class="md-ellipsis">
      BayesS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayesr" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayesrc" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayess" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-kachuri" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Kachuri
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-peter" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Peter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wang
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#single-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Single-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Single-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#all-sum" class="md-nav__link">
    <span class="md-ellipsis">
      ALL-Sum
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#calpred" class="md-nav__link">
    <span class="md-ellipsis">
      CalPred
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbslmm" class="md-nav__link">
    <span class="md-ellipsis">
      DBSLMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmrm" class="md-nav__link">
    <span class="md-ellipsis">
      GMRM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grpa-prs" class="md-nav__link">
    <span class="md-ellipsis">
      GRPa-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genoboost" class="md-nav__link">
    <span class="md-ellipsis">
      GenoBoost
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred-funct" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred-funct
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred2" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred2-auto" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2-auto
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megaprs" class="md-nav__link">
    <span class="md-ellipsis">
      MegaPRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixer_1" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiblup" class="md-nav__link">
    <span class="md-ellipsis">
      MultiBLUP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prsmix_aoi" class="md-nav__link">
    <span class="md-ellipsis">
      PRSMix_AOI
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs_to_abs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS_to_Abs
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prstuning" class="md-nav__link">
    <span class="md-ellipsis">
      PRStuning
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdpr" class="md-nav__link">
    <span class="md-ellipsis">
      SDPR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#viprs" class="md-nav__link">
    <span class="md-ellipsis">
      VIPRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lassosum" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lassosum2" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meta-prs" class="md-nav__link">
    <span class="md-ellipsis">
      meta-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rtprs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      rtPRS-CS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#standards" class="md-nav__link">
    <span class="md-ellipsis">
      Standards
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Standards">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#prs-rs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-RS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorial" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Tutorial">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#tutorial-choi" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial-Choi
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Selection/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#benchmark" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Benchmark">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#benchmark-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark-Wang
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bias" class="md-nav__link">
    <span class="md-ellipsis">
      Bias
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Bias">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ellis-ca" class="md-nav__link">
    <span class="md-ellipsis">
      Ellis CA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-credible-intervals" class="md-nav__link">
    <span class="md-ellipsis">
      PRS credible intervals
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cross-population" class="md-nav__link">
    <span class="md-ellipsis">
      Cross-population
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Cross-population">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bridgeprs" class="md-nav__link">
    <span class="md-ellipsis">
      BridgePRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ct-sleb" class="md-nav__link">
    <span class="md-ellipsis">
      CT-SLEB
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prosper" class="md-nav__link">
    <span class="md-ellipsis">
      PROSPER
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-csx" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CSx
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-fh" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-FH
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdprx" class="md-nav__link">
    <span class="md-ellipsis">
      SDPRX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tl-prs" class="md-nav__link">
    <span class="md-ellipsis">
      TL-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shaprs" class="md-nav__link">
    <span class="md-ellipsis">
      shaPRS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cross-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Cross-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Cross-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ddx-prs" class="md-nav__link">
    <span class="md-ellipsis">
      DDx-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixer" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multi-pgs" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-PGS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#puma-cubs" class="md-nav__link">
    <span class="md-ellipsis">
      PUMA-CUBS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wmt-sblup" class="md-nav__link">
    <span class="md-ellipsis">
      wMT-SBLUP
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#scprs" class="md-nav__link">
    <span class="md-ellipsis">
      scPRS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pathway" class="md-nav__link">
    <span class="md-ellipsis">
      Pathway
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Pathway">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#prset" class="md-nav__link">
    <span class="md-ellipsis">
      PRSet
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pipeline" class="md-nav__link">
    <span class="md-ellipsis">
      Pipeline
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Pipeline">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prsice-2" class="md-nav__link">
    <span class="md-ellipsis">
      PRSice-2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgsc_calc" class="md-nav__link">
    <span class="md-ellipsis">
      pgsc_calc
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#platform" class="md-nav__link">
    <span class="md-ellipsis">
      Platform
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Platform">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cancer-prsweb" class="md-nav__link">
    <span class="md-ellipsis">
      Cancer PRSweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exprsweb" class="md-nav__link">
    <span class="md-ellipsis">
      ExPRSweb
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgscatalog" class="md-nav__link">
    <span class="md-ellipsis">
      PGSCatalog
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgsfusion" class="md-nav__link">
    <span class="md-ellipsis">
      PGSFusion
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      PRS atlas
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#metabolites-prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      metabolites PRS atlas
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#polygenicity" class="md-nav__link">
    <span class="md-ellipsis">
      Polygenicity
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Polygenicity">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bayesr" class="md-nav__link">
    <span class="md-ellipsis">
      BayesR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bayess" class="md-nav__link">
    <span class="md-ellipsis">
      BayesS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayesr" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayesrc" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesRC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sbayess" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-kachuri" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Kachuri
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-peter" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Peter
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wang
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#single-trait" class="md-nav__link">
    <span class="md-ellipsis">
      Single-trait
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Single-trait">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#all-sum" class="md-nav__link">
    <span class="md-ellipsis">
      ALL-Sum
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#calpred" class="md-nav__link">
    <span class="md-ellipsis">
      CalPred
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dbslmm" class="md-nav__link">
    <span class="md-ellipsis">
      DBSLMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gmrm" class="md-nav__link">
    <span class="md-ellipsis">
      GMRM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#grpa-prs" class="md-nav__link">
    <span class="md-ellipsis">
      GRPa-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#genoboost" class="md-nav__link">
    <span class="md-ellipsis">
      GenoBoost
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred-funct" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred-funct
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred2" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldpred2-auto" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2-auto
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megaprs" class="md-nav__link">
    <span class="md-ellipsis">
      MegaPRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mixer_1" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#multiblup" class="md-nav__link">
    <span class="md-ellipsis">
      MultiBLUP
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prsmix_aoi" class="md-nav__link">
    <span class="md-ellipsis">
      PRSMix_AOI
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prs_to_abs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS_to_Abs
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#prstuning" class="md-nav__link">
    <span class="md-ellipsis">
      PRStuning
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sdpr" class="md-nav__link">
    <span class="md-ellipsis">
      SDPR
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#viprs" class="md-nav__link">
    <span class="md-ellipsis">
      VIPRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lassosum" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lassosum2" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meta-prs" class="md-nav__link">
    <span class="md-ellipsis">
      meta-PRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rtprs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      rtPRS-CS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#standards" class="md-nav__link">
    <span class="md-ellipsis">
      Standards
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Standards">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#prs-rs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-RS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorial" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Tutorial">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#tutorial-choi" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial-Choi
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Polygenic_risk_scores</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#benchmark-wang">Benchmark-Wang</a></td>
<td style="text-align: left;">Benchmark</td>
<td style="text-align: left;">Wang C, Zhang J, Veldsman WP, Zhou X, ...&amp;, Zhang L. (2022) A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants Brief. Bioinform.,  () . doi:10.1093/bib/bbac552. PMID 36585786</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ellis-ca">Ellis CA</a></td>
<td style="text-align: left;">Bias</td>
<td style="text-align: left;">Ellis CA, Oliver KL, Harris RV, Ottman R, ...&amp;, Bahlo M. (2024) Inflation of polygenic risk scores caused by sample overlap and relatedness: Examples of a major risk of bias Am. J. Hum. Genet., 0 (0) . doi:10.1016/j.ajhg.2024.07.014. PMID 39168121</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-credible-intervals">PRS credible intervals</a></td>
<td style="text-align: left;">Bias</td>
<td style="text-align: left;">Ding Y, Hou K, Burch KS, Lapinska S, ...&amp;, Pasaniuc B. (2022) Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification Nat. Genet., 54 (1) 30-39. doi:10.1038/s41588-021-00961-5. PMID 34931067</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bridgeprs">BridgePRS</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Hoggart CJ, Choi SW, García-González J, Souaiaia T, ...&amp;, O'Reilly PF. (2023) BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability Nat. Genet., 56 (1) 180-186. doi:10.1038/s41588-023-01583-9. PMID 38123642</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ct-sleb">CT-SLEB</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Zhang H, Zhan J, Jin J, Zhang J, ...&amp;, Chatterjee N. (2023) A new method for multiancestry polygenic prediction improves performance across diverse populations Nat. Genet.,  () . doi:10.1038/s41588-023-01501-z. PMID 37749244</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prosper">PROSPER</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Zhang J, Zhan J, Jin J, Ma C, ...&amp;, Chatterjee N. (2024) An ensemble penalized regression method for multi-ancestry polygenic risk prediction Nat. Commun., 15 (1) 3238. doi:10.1038/s41467-024-47357-7. PMID 38622117</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-csx">PRS-CSx</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Ruan Y, Lin YF, Feng YC, Chen CY, ...&amp;, Ge T. (2022) Improving polygenic prediction in ancestrally diverse populations Nat. Genet., 54 (5) 573-580. doi:10.1038/s41588-022-01054-7. PMID 35513724</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-fh">PRS-FH</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Hujoel MLA, Loh PR, Neale BM, Price AL. (2022) Incorporating family history of disease improves polygenic risk scores in diverse populations Cell Genom., 2 (7) 100152. doi:10.1016/j.xgen.2022.100152. PMID 35935918</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sdprx">SDPRX</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Zhou G, Chen T, Zhao H. (2023) SDPRX: A statistical method for cross-population prediction of complex traits Am. J. Hum. Genet., 110 (1) 13-22. doi:10.1016/j.ajhg.2022.11.007. PMID 36460009</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tl-prs">TL-PRS</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Zhao Z, Fritsche LG, Smith JA, Mukherjee B, ...&amp;, Lee S. (2022) The construction of cross-population polygenic risk scores using transfer learning Am. J. Hum. Genet., 109 (11) 1998-2008. doi:10.1016/j.ajhg.2022.09.010. PMID 36240765</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shaprs">shaPRS</a></td>
<td style="text-align: left;">Cross-population</td>
<td style="text-align: left;">Kelemen M, Vigorito E, Fachal L, Anderson CA, ...&amp;, Wallace C. (2024) shaPRS: Leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores Am. J. Hum. Genet.,  () . doi:10.1016/j.ajhg.2024.04.009. PMID 38703768</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ddx-prs">DDx-PRS</a></td>
<td style="text-align: left;">Cross-trait</td>
<td style="text-align: left;">Peyrot, W. J., Panagiotaropoulou, G., Olde Loohuis, L. M., Adams, M., Awasthi, S., Ge, T., ... &amp; Price, A. L. (2024). Distinguishing different psychiatric disorders using DDx-PRS. medRxiv, 2024-02.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mixer">MiXeR</a></td>
<td style="text-align: left;">Cross-trait</td>
<td style="text-align: left;">Frei O, Holland D, Smeland OB, Shadrin AA, ...&amp;, Dale AM. (2019) Bivariate causal mixture model quantifies polygenic overlap between complex traits beyond genetic correlation Nat. Commun., 10 (1) 2417. doi:10.1038/s41467-019-10310-0. PMID 31160569</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multi-pgs">Multi-PGS</a></td>
<td style="text-align: left;">Cross-trait</td>
<td style="text-align: left;">Albiñana C, Zhu Z, Schork AJ, Ingason A, ...&amp;, Vilhjálmsson BJ. (2023) Multi-PGS enhances polygenic prediction by combining 937 polygenic scores Nat. Commun., 14 (1) 4702. doi:10.1038/s41467-023-40330-w. PMID 37543680</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#puma-cubs">PUMA-CUBS</a></td>
<td style="text-align: left;">Cross-trait</td>
<td style="text-align: left;">Zhao, Zijie, et al. "Optimizing and benchmarking polygenic risk scores with GWAS summary statistics." bioRxiv (2022).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wmt-sblup">wMT-SBLUP</a></td>
<td style="text-align: left;">Cross-trait</td>
<td style="text-align: left;">Maier RM, Zhu Z, Lee SH, Trzaskowski M, ...&amp;, Robinson MR. (2018) Improving genetic prediction by leveraging genetic correlations among human diseases and traits Nat. Commun., 9 (1) 989. doi:10.1038/s41467-017-02769-6. PMID 29515099</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#scprs">scPRS</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Zhang, S. et al. Single-cell polygenic risk scores dissect cellular and molecular heterogeneity of complex human diseases. Nat. Biotechnol. 1–17 (2025).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prset">PRSet</a></td>
<td style="text-align: left;">Pathway</td>
<td style="text-align: left;">Choi SW, García-González J, Ruan Y, Wu HM, ...&amp;, O'Reilly PF. (2023) PRSet: Pathway-based polygenic risk score analyses and software PLoS Genet., 19 (2) e1010624. doi:10.1371/journal.pgen.1010624. PMID 36749789</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#plink2">PLINK2</a></td>
<td style="text-align: left;">Pipeline</td>
<td style="text-align: left;">Chang CC, Chow CC, Tellier LC, Vattikuti S, ...&amp;, Lee JJ. (2015) Second-generation PLINK: rising to the challenge of larger and richer datasets Gigascience, 4 (1) 7. doi:10.1186/s13742-015-0047-8. PMID 25722852</td>
<td style="text-align: left;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prsice-2">PRSice-2</a></td>
<td style="text-align: left;">Pipeline</td>
<td style="text-align: left;">Choi SW, O'Reilly PF. (2019) PRSice-2: Polygenic Risk Score software for biobank-scale data Gigascience, 8 (7) 1-6. doi:10.1093/gigascience/giz082. PMID 31307061</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgsccalc">pgsc_calc</a></td>
<td style="text-align: left;">Pipeline</td>
<td style="text-align: left;">Lambert, Wingfield et al. (2024) The Polygenic Score Catalog: new functionality and tools to enable FAIR research. medRxiv. doi:10.1101/2024.05.29.24307783.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cancer-prsweb">Cancer PRSweb</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Fritsche LG, Patil S, Beesley LJ, VandeHaar P, ...&amp;, Mukherjee B. (2020) Cancer PRSweb: An online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks Am. J. Hum. Genet., 107 (5) 815-836. doi:10.1016/j.ajhg.2020.08.025. PMID 32991828</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#exprsweb">ExPRSweb</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Ma Y, Patil S, Zhou X, Mukherjee B, ...&amp;, Fritsche LG. (2022) ExPRSweb: An online repository with polygenic risk scores for common health-related exposures Am. J. Hum. Genet., 109 (10) 1742-1760. doi:10.1016/j.ajhg.2022.09.001. PMID 36152628</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgscatalog">PGSCatalog</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Lambert SA, Gil L, Jupp S, Ritchie SC, ...&amp;, Inouye M. (2021) The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation Nat. Genet., 53 (4) 420-425. doi:10.1038/s41588-021-00783-5. PMID 33692568</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgsfusion">PGSFusion</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Yang, S., Ye, X., Ji, X., Li, Z., Tian, M., Huang, P., &amp; Cao, C. (2024). PGSFusion streamlines polygenic score construction and epidemiological applications in biobank-scale cohorts. bioRxiv, 2024-08.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-atlas">PRS atlas</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Richardson TG, Harrison S, Hemani G, Davey Smith G. (2019) An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome Elife, 8 () . doi:10.7554/eLife.43657. PMID 30835202</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metabolites-prs-atlas">metabolites PRS atlas</a></td>
<td style="text-align: left;">Platform</td>
<td style="text-align: left;">Fang S, Holmes MV, Gaunt TR, Davey Smith G, ...&amp;, Richardson TG. (2022) Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers Elife, 11 () e73951. doi:10.7554/eLife.73951. PMID 36219204</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bayesr">BayesR</a></td>
<td style="text-align: left;">Polygenicity</td>
<td style="text-align: left;">Moser G, Lee SH, Hayes BJ, Goddard ME, ...&amp;, Visscher PM. (2015) Simultaneous discovery, estimation and prediction analysis of complex traits using a bayesian mixture model PLoS Genet., 11 (4) e1004969. doi:10.1371/journal.pgen.1004969. PMID 25849665</td>
<td style="text-align: left;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bayess">BayesS</a></td>
<td style="text-align: left;">Polygenicity</td>
<td style="text-align: left;">Zeng J, de Vlaming R, Wu Y, Robinson MR, ...&amp;, Yang J. (2018) Signatures of negative selection in the genetic architecture of human complex traits Nat. Genet., 50 (5) 746-753. doi:10.1038/s41588-018-0101-4. PMID 29662166</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayesrc">SBayesRC</a></td>
<td style="text-align: left;">Polygenicity</td>
<td style="text-align: left;">Zheng Z, Liu S, Sidorenko J, Wang Y, ...&amp;, Zeng J. (2024) Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries Nat. Genet., 56 (5) 767-777. doi:10.1038/s41588-024-01704-y. PMID 38689000</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayesr">SBayesR</a></td>
<td style="text-align: left;">Polygenicity</td>
<td style="text-align: left;">Lloyd-Jones LR, Zeng J, Sidorenko J, Yengo L, ...&amp;, Visscher PM. (2019) Improved polygenic prediction by Bayesian multiple regression on summary statistics Nat. Commun., 10 (1) 5086. doi:10.1038/s41467-019-12653-0. PMID 31704910</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayess">SBayesS</a></td>
<td style="text-align: left;">Polygenicity</td>
<td style="text-align: left;">Zeng J, Xue A, Jiang L, Lloyd-Jones LR, ...&amp;, Yang J. (2021) Widespread signatures of natural selection across human complex traits and functional genomic categories Nat. Commun., 12 (1) 1164. doi:10.1038/s41467-021-21446-3. PMID 33608517</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-kachuri">Review-Kachuri</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Kachuri L, Chatterjee N, Hirbo J, Schaid DJ, ...&amp;, Ge T. (2023) Principles and methods for transferring polygenic risk scores across global populations Nat. Rev. Genet., 25 (1) 8-25. doi:10.1038/s41576-023-00637-2. PMID 37620596</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-peter">Review-Peter</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Visscher PM, Yengo L, Cox NJ, Wray NR. (2021) Discovery and implications of polygenicity of common diseases Science, 373 (6562) 1468-1473. doi:10.1126/science.abi8206. PMID 34554790</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-wang">Review-Wang</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Wang Y, Tsuo K, Kanai M, Neale BM, ...&amp;, Martin AR. (2022) Challenges and opportunities for developing more generalizable polygenic risk scores Annu. Rev. Biomed. Data Sci., 5 (1) 293-320. doi:10.1146/annurev-biodatasci-111721-074830. PMID 35576555</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#all-sum">ALL-Sum</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Chen T, Zhang H, Mazumder R, Lin X. (2024) Fast and scalable ensemble learning method for versatile polygenic risk prediction Proc. Natl. Acad. Sci. U. S. A., 121 (33) e2403210121. doi:10.1073/pnas.2403210121. PMID 39110727</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#calpred">CalPred</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Hou K, Xu Z, Ding Y, Mandla R, ...&amp;, Pasaniuc B. (2024) Calibrated prediction intervals for polygenic scores across diverse contexts Nat. Genet., 56 (7) 1386-1396. doi:10.1038/s41588-024-01792-w. PMID 38886587</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#dbslmm">DBSLMM</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Yang S, Zhou X. (2020) Accurate and scalable construction of polygenic scores in large biobank data sets Am. J. Hum. Genet., 106 (5) 679-693. doi:10.1016/j.ajhg.2020.03.013. PMID 32330416</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gmrm">GMRM</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Orliac EJ, Trejo Banos D, Ojavee SE, Läll K, ...&amp;, Robinson MR. (2022) Improving GWAS discovery and genomic prediction accuracy in biobank data Proc. Natl. Acad. Sci. U. S. A., 119 (31) e2121279119. doi:10.1073/pnas.2121279119. PMID 35905320</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#grpa-prs">GRPa-PRS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Li X, Fernandes BS, Liu A, Chen J, ...&amp;, Dai Y. (2024) GRPa-PRS: A risk stratification method to identify genetically-regulated pathways in polygenic diseases medRxiv,  () 2023.06.19.23291621. doi:10.1101/2023.06.19.23291621. PMID 37425929</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#genoboost">GenoBoost</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Ohta R, Tanigawa Y, Suzuki Y, Kellis M, ...&amp;, Morishita S. (2024) A polygenic score method boosted by non-additive models Nat. Commun., 15 (1) 4433. doi:10.1038/s41467-024-48654-x. PMID 38811555</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred-funct">LDpred-funct</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Márquez-Luna C, Gazal S, Loh PR, Kim SS, ...&amp;, Price AL. (2021) Incorporating functional priors improves polygenic prediction accuracy in UK Biobank and 23andMe data sets Nat. Commun., 12 (1) 6052. doi:10.1038/s41467-021-25171-9. PMID 34663819</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred2-auto">LDpred2-auto</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Privé F, Albiñana C, Arbel J, Pasaniuc B, ...&amp;, Vilhjálmsson BJ. (2023) Inferring disease architecture and predictive ability with LDpred2-auto Am. J. Hum. Genet., 110 (12) 2042-2055. doi:10.1016/j.ajhg.2023.10.010. PMID 37944514</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred2">LDpred2</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Privé F, Arbel J, Vilhjálmsson BJ. (2021) LDpred2: better, faster, stronger Bioinformatics, 36 (22-23) 5424-5431. doi:10.1093/bioinformatics/btaa1029. PMID 33326037</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred">LDpred</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Vilhjálmsson BJ, Yang J, Finucane HK, Gusev A, ...&amp;, Price AL. (2015) Modeling linkage disequilibrium increases accuracy of polygenic risk scores Am. J. Hum. Genet., 97 (4) 576-592. doi:10.1016/j.ajhg.2015.09.001. PMID 26430803</td>
<td style="text-align: left;">2015</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#megaprs">MegaPRS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Zhang Q, Privé F, Vilhjálmsson B, Speed D. (2021) Improved genetic prediction of complex traits from individual-level data or summary statistics Nat. Commun., 12 (1) 4192. doi:10.1038/s41467-021-24485-y. PMID 34234142</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mixer">MiXeR</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Holland D, Frei O, Desikan R, Fan CC, ...&amp;, Dale AM. (2020) Beyond SNP heritability: Polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model PLoS Genet., 16 (5) e1008612. doi:10.1371/journal.pgen.1008612. PMID 32427991</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multiblup">MultiBLUP</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Speed D, Balding DJ. (2014) MultiBLUP: improved SNP-based prediction for complex traits Genome Res., 24 (9) 1550-1557. doi:10.1101/gr.169375.113. PMID 24963154</td>
<td style="text-align: left;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-cs">PRS-CS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Ge T, Chen CY, Ni Y, Feng YA, ...&amp;, Smoller JW. (2019) Polygenic prediction via Bayesian regression and continuous shrinkage priors Nat. Commun., 10 (1) 1776. doi:10.1038/s41467-019-09718-5. PMID 30992449</td>
<td style="text-align: left;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prsmixaoi">PRSMix_AOI</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Misra, A. et al. Instability of high polygenic risk classification and mitigation by integrative scoring. bioRxiv 2024.07.24.24310897 (2024) doi:10.1101/2024.07.24.24310897.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prstoabs">PRS_to_Abs</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Pain O, Gillett AC, Austin JC, Folkersen L, ...&amp;, Lewis CM. (2022) A tool for translating polygenic scores onto the absolute scale using summary statistics Eur. J. Hum. Genet., 30 (3) 339-348. doi:10.1038/s41431-021-01028-z. PMID 34983942</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prstuning">PRStuning</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Jiang, W., Chen, L., Girgenti, M. J., &amp; Zhao, H. (2023). Tuning Parameters for Polygenic Risk Score Methods Using GWAS Summary Statistics from Training Data. Research Square.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sdpr">SDPR</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Zhou G, Zhao H. (2021) A fast and robust Bayesian nonparametric method for prediction of complex traits using summary statistics PLoS Genet., 17 (7) e1009697. doi:10.1371/journal.pgen.1009697. PMID 34310601</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#viprs">VIPRS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Zabad S, Gravel S, Li Y. (2023) Fast and accurate Bayesian polygenic risk modeling with variational inference Am. J. Hum. Genet., 110 (5) 741-761. doi:10.1016/j.ajhg.2023.03.009. PMID 37030289</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lassosum2">lassosum2</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Privé F, Arbel J, Aschard H, Vilhjálmsson BJ. (2022) Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores HGG Adv, 3 (4) 100136. doi:10.1016/j.xhgg.2022.100136. PMID 36105883</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lassosum">lassosum</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Mak TSH, Porsch RM, Choi SW, Zhou X, ...&amp;, Sham PC. (2017) Polygenic scores via penalized regression on summary statistics Genet. Epidemiol., 41 (6) 469-480. doi:10.1002/gepi.22050. PMID 28480976</td>
<td style="text-align: left;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#meta-prs">meta-PRS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Albiñana C, Grove J, McGrath JJ, Agerbo E, ...&amp;, Vilhjálmsson BJ. (2021) Leveraging both individual-level genetic data and GWAS summary statistics increases polygenic prediction Am. J. Hum. Genet., 108 (6) 1001-1011. doi:10.1016/j.ajhg.2021.04.014. PMID 33964208</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#rtprs-cs">rtPRS-CS</a></td>
<td style="text-align: left;">Single-trait</td>
<td style="text-align: left;">Tubbs, J. D., Chen, Y., Duan, R., Huang, H. &amp; Ge, T. Real-time dynamic polygenic prediction for streaming data. bioRxiv 2024.07.12.24310357 (2024) doi:10.1101/2024.07.12.24310357.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-rs">PRS-RS</a></td>
<td style="text-align: left;">Standards</td>
<td style="text-align: left;">Wand H, Lambert SA, Tamburro C, Iacocca MA, ...&amp;, Wojcik GL. (2021) Improving reporting standards for polygenic scores in risk prediction studies Nature, 591 (7849) 211-219. doi:10.1038/s41586-021-03243-6. PMID 33692554</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tutorial-choi">Tutorial-Choi</a></td>
<td style="text-align: left;">Tutorial</td>
<td style="text-align: left;">Choi SW, Mak TS, O'Reilly PF. (2020) Tutorial: a guide to performing polygenic risk score analyses Nat. Protoc., 15 (9) 2759-2772. doi:10.1038/s41596-020-0353-1. PMID 32709988</td>
<td style="text-align: left;">2020</td>
</tr>
</tbody>
</table>
<h2 id="benchmark">Benchmark</h2>
<h3 id="benchmark-wang">Benchmark-Wang</h3>
<ul>
<li><strong>NAME</strong> : Benchmark-Wang </li>
<li><strong>TITLE</strong> : A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants </li>
<li><strong>DOI</strong> : 10.1093/bib/bbac552 </li>
<li><strong>ABSTRACT</strong> : Quantifying an individual's risk for common diseases is an important goal of precision health. The polygenic risk score (PRS), which aggregates multiple risk alleles of candidate diseases, has emerged as a standard approach for identifying high-risk individuals. Although several studies have been performed to benchmark the PRS calculation tools and assess their potential to guide future clinical applications, some issues remain to be further investigated, such as lacking (i) various simulated data with different genetic effects; (ii) evaluation of machine learning models and (iii) evaluation on multiple ancestries studies. In this study, we systematically validated and compared 13 statistical methods, 5 machine learning models and 2 ensemble models using simulated data with additive and genetic interaction models, 22 common diseases with internal training sets, 4 common diseases with external summary statistics and 3 common diseases for trans-ancestry studies in UK Biobank. The statistical methods were better in simulated data from additive models and machine learning models have edges for data that include genetic interactions. Ensemble models are generally the best choice by integrating various statistical methods. LDpred2 outperformed the other standalone tools, whereas PRS-CS, lassosum and DBSLMM showed comparable performance. We also identified that disease heritability strongly affected the predictive performance of all methods. Both the number and effect sizes of risk SNPs are important; and sample size strongly influences the performance of all methods. For the trans-ancestry studies, we found that the performance of most methods became worse when training and testing sets were from different populations. </li>
<li><strong>CITATION</strong> : Wang C, Zhang J, Veldsman WP, Zhou X, ...&amp;, Zhang L. (2022) A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants Brief. Bioinform.,  () . doi:10.1093/bib/bbac552. PMID 36585786 </li>
<li><strong>JOURNAL_INFO</strong> : Briefings in bioinformatics ; Brief. Bioinform. ; 2022 ; <NA> ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36585786">36585786</a> </li>
</ul>
<h2 id="bias">Bias</h2>
<h3 id="ellis-ca">Ellis CA</h3>
<ul>
<li><strong>NAME</strong> : Ellis CA </li>
<li><strong>TITLE</strong> : Inflation of polygenic risk scores caused by sample overlap and relatedness: Examples of a major risk of bias </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2024.07.014 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs) are an important tool for understanding the role of common genetic variants in human disease. Standard best practices recommend that PRSs be analyzed in cohorts that are independent of the genome-wide association study (GWAS) used to derive the scores without sample overlap or relatedness between the two cohorts. However, identifying sample overlap and relatedness can be challenging in an era of GWASs performed by large biobanks and international research consortia. Although most genomics researchers are aware of best practices and theoretical concerns about sample overlap and relatedness between GWAS and PRS cohorts, the prevailing assumption is that the risk of bias is small for very large GWASs. Here, we present two real-world examples demonstrating that sample overlap and relatedness is not a minor or theoretical concern but an important potential source of bias in PRS studies. Using a recently developed statistical adjustment tool, we found that excluding overlapping and related samples was equal to or more powerful than adjusting for overlap bias. Our goal is to make genomics researchers aware of the magnitude of risk of bias from sample overlap and relatedness and to highlight the need for mitigation tools, including independent validation cohorts in PRS studies, continued development of statistical adjustment methods, and tools for researchers to test their cohorts for overlap and relatedness with GWAS cohorts without sharing individual-level data. </li>
<li><strong>CITATION</strong> : Ellis CA, Oliver KL, Harris RV, Ottman R, ...&amp;, Bahlo M. (2024) Inflation of polygenic risk scores caused by sample overlap and relatedness: Examples of a major risk of bias Am. J. Hum. Genet., 0 (0) . doi:10.1016/j.ajhg.2024.07.014. PMID 39168121 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2024 ; 0 ; 0 ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39168121">39168121</a> </li>
</ul>
<h3 id="prs-credible-intervals">PRS credible intervals</h3>
<ul>
<li><strong>NAME</strong> : PRS credible intervals </li>
<li><strong>SHORT NAME</strong> : PRS credible intervals </li>
<li><strong>FULL NAME</strong> : PRS credible intervals </li>
<li><strong>URL</strong> : <a href="https://privefl.github.io/bigsnpr/articles/prs_uncertainty.html">https://privefl.github.io/bigsnpr/articles/prs_uncertainty.html</a> </li>
<li><strong>KEYWORDS</strong> : uncertainty </li>
<li><strong>TITLE</strong> : Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00961-5 </li>
<li><strong>ABSTRACT</strong> : Although the cohort-level accuracy of polygenic risk scores (PRSs)-estimates of genetic value at the individual level-has been widely assessed, uncertainty in PRSs remains underexplored. In the present study, we show that Bayesian PRS methods can estimate the variance of an individual's PRS and can yield well-calibrated credible intervals via posterior sampling. For 13 real traits in the UK Biobank (n = 291,273 unrelated 'white British'), we observe large variances in individual PRS estimates which impact interpretation of PRS-based stratification; averaging across traits, only 0.8% (s.d. = 1.6%) of individuals with PRS point estimates in the top decile have corresponding 95% credible intervals fully contained in the top decile. We provide an analytical estimator for the expectation of individual PRS variance as a function of SNP heritability, number of causal SNPs and sample size. Our results showcase the importance of incorporating uncertainty in individual PRS estimates into subsequent analyses. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Ding Y, Hou K, Burch KS, Lapinska S, ...&amp;, Pasaniuc B. (2022) Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification Nat. Genet., 54 (1) 30-39. doi:10.1038/s41588-021-00961-5. PMID 34931067 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2022 ; 54 ; 1 ; 30-39 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34931067">34931067</a> </li>
</ul>
<h2 id="cross-population">Cross-population</h2>
<h3 id="bridgeprs">BridgePRS</h3>
<ul>
<li><strong>NAME</strong> : BridgePRS </li>
<li><strong>SHORT NAME</strong> : BridgePRS </li>
<li><strong>FULL NAME</strong> : BridgePRS </li>
<li><strong>DESCRIPTION</strong> : BridgePRS is a Bayesian-ridge (Bridge) approach, which "bridges" the PRS between two populations of different ancestry, developed to tackle the "PRS Portability Problem". The PRS Portability Problem causes lower accuracy PRS in underrepresented populations due to the biased sampling in GWAS data collection. </li>
<li><strong>URL</strong> : <a href="https://www.bridgeprs.net/">https://www.bridgeprs.net/</a> </li>
<li><strong>TITLE</strong> : BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01583-9 </li>
<li><strong>ABSTRACT</strong> : Here we present BridgePRS, a novel Bayesian polygenic risk score (PRS) method that leverages shared genetic effects across ancestries to increase PRS portability. We evaluate BridgePRS via simulations and real UK Biobank data across 19 traits in individuals of African, South Asian and East Asian ancestry, using both UK Biobank and Biobank Japan genome-wide association study summary statistics; out-of-cohort validation is performed in the Mount Sinai (New York) BioMe biobank. BridgePRS is compared with the leading alternative, PRS-CSx, and two other PRS methods. Simulations suggest that the performance of BridgePRS relative to PRS-CSx increases as uncertainty increases: with lower trait heritability, higher polygenicity and greater between-population genetic diversity; and when causal variants are not present in the data. In real data, BridgePRS has a 61% larger average R2 than PRS-CSx in out-of-cohort prediction of African ancestry samples in BioMe (P = 6 × 10-5). BridgePRS is a computationally efficient, user-friendly and powerful approach for PRS analyses in non-European ancestries. </li>
<li><strong>CITATION</strong> : Hoggart CJ, Choi SW, García-González J, Souaiaia T, ...&amp;, O'Reilly PF. (2023) BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability Nat. Genet., 56 (1) 180-186. doi:10.1038/s41588-023-01583-9. PMID 38123642 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2023 ; 56 ; 1 ; 180-186 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38123642">38123642</a> </li>
</ul>
<h3 id="ct-sleb">CT-SLEB</h3>
<ul>
<li><strong>NAME</strong> : CT-SLEB </li>
<li><strong>SHORT NAME</strong> : CT-SLEB </li>
<li><strong>DESCRIPTION</strong> : CT-SLEB is a method designed to generate multi-ancestry PRSs that incorporate existing large GWAS from EUR populations and smaller GWAS from non-EUR populations. The method has three key steps: 1. Clumping and Thresholding for selecting SNPs to be included in a PRS for the target population; 2. Empirical-Bayes method for estimating the coefficients of the SNPs; 3. Super-learning model to combine a series of PRSs generated under different SNP selection thresholds. </li>
<li><strong>URL</strong> : <a href="https://github.com/andrewhaoyu/CTSLEB">https://github.com/andrewhaoyu/CTSLEB</a> </li>
<li><strong>TITLE</strong> : A new method for multiancestry polygenic prediction improves performance across diverse populations </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01501-z </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs) increasingly predict complex traits; however, suboptimal performance in non-European populations raise concerns about clinical applications and health inequities. We developed CT-SLEB, a powerful and scalable method to calculate PRSs, using ancestry-specific genome-wide association study summary statistics from multiancestry training samples, integrating clumping and thresholding, empirical Bayes and superlearning. We evaluated CT-SLEB and nine alternative methods with large-scale simulated genome-wide association studies (~19 million common variants) and datasets from 23andMe, Inc., the Global Lipids Genetics Consortium, All of Us and UK Biobank, involving 5.1 million individuals of diverse ancestry, with 1.18 million individuals from four non-European populations across 13 complex traits. Results demonstrated that CT-SLEB significantly improves PRS performance in non-European populations compared with simple alternatives, with comparable or superior performance to a recent, computationally intensive method. Moreover, our simulation studies offered insights into sample size requirements and SNP density effects on multiancestry risk prediction. </li>
<li><strong>CITATION</strong> : Zhang H, Zhan J, Jin J, Zhang J, ...&amp;, Chatterjee N. (2023) A new method for multiancestry polygenic prediction improves performance across diverse populations Nat. Genet.,  () . doi:10.1038/s41588-023-01501-z. PMID 37749244 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2023 ; <NA> ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37749244">37749244</a> </li>
</ul>
<h3 id="prosper">PROSPER</h3>
<ul>
<li><strong>NAME</strong> : PROSPER </li>
<li><strong>SHORT NAME</strong> : PROSPER </li>
<li><strong>FULL NAME</strong> : Polygenic Risk scOres based on enSemble of PEnalized Regression models </li>
<li><strong>DESCRIPTION</strong> : PROSPER is a new multi-ancestry PRS method with penalized regression followed by ensemble learning. This software is a command line tool based on R programming language. Large-scale benchmarking study shows that PROSPER could be the leading method to reduce the disparity of PRS performance across ancestry groups </li>
<li><strong>URL</strong> : <a href="https://github.com/Jingning-Zhang/PROSPER">https://github.com/Jingning-Zhang/PROSPER</a> </li>
<li><strong>TITLE</strong> : An ensemble penalized regression method for multi-ancestry polygenic risk prediction </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-47357-7 </li>
<li><strong>ABSTRACT</strong> : Great efforts are being made to develop advanced polygenic risk scores (PRS) to improve the prediction of complex traits and diseases. However, most existing PRS are primarily trained on European ancestry populations, limiting their transferability to non-European populations. In this article, we propose a novel method for generating multi-ancestry Polygenic Risk scOres based on enSemble of PEnalized Regression models (PROSPER). PROSPER integrates genome-wide association studies (GWAS) summary statistics from diverse populations to develop ancestry-specific PRS with improved predictive power for minority populations. The method uses a combination of L 1 (lasso) and L 2 (ridge) penalty functions, a parsimonious specification of the penalty parameters across populations, and an ensemble step to combine PRS generated across different penalty parameters. We evaluate the performance of PROSPER and other existing methods on large-scale simulated and real datasets, including those from 23andMe Inc., the Global Lipids Genetics Consortium, and All of Us. Results show that PROSPER can substantially improve multi-ancestry polygenic prediction compared to alternative methods across a wide variety of genetic architectures. In real data analyses, for example, PROSPER increased out-of-sample prediction R2 for continuous traits by an average of 70% compared to a state-of-the-art Bayesian method (PRS-CSx) in the African ancestry population. Further, PROSPER is computationally highly scalable for the analysis of large SNP contents and many diverse populations. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Zhang J, Zhan J, Jin J, Ma C, ...&amp;, Chatterjee N. (2024) An ensemble penalized regression method for multi-ancestry polygenic risk prediction Nat. Commun., 15 (1) 3238. doi:10.1038/s41467-024-47357-7. PMID 38622117 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2024 ; 15 ; 1 ; 3238 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38622117">38622117</a> </li>
</ul>
<h3 id="prs-csx">PRS-CSx</h3>
<ul>
<li><strong>NAME</strong> : PRS-CSx </li>
<li><strong>SHORT NAME</strong> : PRS-CSx </li>
<li><strong>FULL NAME</strong> : PRS-CSx </li>
<li><strong>DESCRIPTION</strong> : PRS-CSx is a Python based command line tool that integrates GWAS summary statistics and external LD reference panels from multiple populations to improve cross-population polygenic prediction. Posterior SNP effect sizes are inferred under coupled continuous shrinkage (CS) priors across populations. </li>
<li><strong>URL</strong> : <a href="https://github.com/getian107/PRScsx">https://github.com/getian107/PRScsx</a> </li>
<li><strong>KEYWORDS</strong> : continuous shrinkage (CS) prior,  cross-population </li>
<li><strong>TITLE</strong> : Improving polygenic prediction in ancestrally diverse populations </li>
<li><strong>DOI</strong> : 10.1038/s41588-022-01054-7 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRS) have attenuated cross-population predictive performance. As existing genome-wide association studies (GWAS) have been conducted predominantly in individuals of European descent, the limited transferability of PRS reduces their clinical value in non-European populations, and may exacerbate healthcare disparities. Recent efforts to level ancestry imbalance in genomic research have expanded the scale of non-European GWAS, although most remain underpowered. Here, we present a new PRS construction method, PRS-CSx, which improves cross-population polygenic prediction by integrating GWAS summary statistics from multiple populations. PRS-CSx couples genetic effects across populations via a shared continuous shrinkage (CS) prior, enabling more accurate effect size estimation by sharing information between summary statistics and leveraging linkage disequilibrium diversity across discovery samples, while inheriting computational efficiency and robustness from PRS-CS. We show that PRS-CSx outperforms alternative methods across traits with a wide range of genetic architectures, cross-population genetic overlaps and discovery GWAS sample sizes in simulations, and improves the prediction of quantitative traits and schizophrenia risk in non-European populations. </li>
<li><strong>CITATION</strong> : Ruan Y, Lin YF, Feng YC, Chen CY, ...&amp;, Ge T. (2022) Improving polygenic prediction in ancestrally diverse populations Nat. Genet., 54 (5) 573-580. doi:10.1038/s41588-022-01054-7. PMID 35513724 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2022 ; 54 ; 5 ; 573-580 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35513724">35513724</a> </li>
</ul>
<h3 id="prs-fh">PRS-FH</h3>
<ul>
<li><strong>NAME</strong> : PRS-FH </li>
<li><strong>SHORT NAME</strong> : PRS-FH </li>
<li><strong>FULL NAME</strong> : family history </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/UKBB/PRSFH/PRSFH/">https://alkesgroup.broadinstitute.org/UKBB/PRSFH/PRSFH/</a> </li>
<li><strong>KEYWORDS</strong> : family history </li>
<li><strong>TITLE</strong> : Incorporating family history of disease improves polygenic risk scores in diverse populations </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2022.100152 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs) derived from genotype data and family history (FH) of disease provide valuable information for predicting disease risk, but PRSs perform poorly when applied to diverse populations. Here, we explore methods for combining both types of information (PRS-FH) in UK Biobank data. PRSs were trained using all British individuals (n = 409,000), and target samples consisted of unrelated non-British Europeans (n = 42,000), South Asians (n = 7,000), or Africans (n = 7,000). We evaluated PRS, FH, and PRS-FH using liability-scale R 2, primarily focusing on 3 well-powered diseases (type 2 diabetes, hypertension, and depression). PRS attained average prediction R 2s of 5.8%, 4.0%, and 0.53% in non-British Europeans, South Asians, and Africans, confirming poor cross-population transferability. In contrast, PRS-FH attained average prediction R 2s of 13%, 12%, and 10%, respectively, representing a large improvement in Europeans and an extremely large improvement in Africans. In conclusion, including family history improves the accuracy of polygenic risk scores, particularly in diverse populations. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Hujoel MLA, Loh PR, Neale BM, Price AL. (2022) Incorporating family history of disease improves polygenic risk scores in diverse populations Cell Genom., 2 (7) 100152. doi:10.1016/j.xgen.2022.100152. PMID 35935918 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom. ; 2022 ; 2 ; 7 ; 100152 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35935918">35935918</a> </li>
</ul>
<h3 id="sdprx">SDPRX</h3>
<ul>
<li><strong>NAME</strong> : SDPRX </li>
<li><strong>SHORT NAME</strong> : SDPRX </li>
<li><strong>FULL NAME</strong> : SDPRX </li>
<li><strong>DESCRIPTION</strong> : SDPRX is a statistical method for cross-population prediction of complex traits. It integrates GWAS summary statistics and LD matrices from two populations (EUR and non-EUR) to compuate polygenic risk scores. </li>
<li><strong>URL</strong> : <a href="https://github.com/eldronzhou/SDPRX">https://github.com/eldronzhou/SDPRX</a> </li>
<li><strong>TITLE</strong> : SDPRX: A statistical method for cross-population prediction of complex traits </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2022.11.007 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk score (PRS) has demonstrated its great utility in biomedical research through identifying high-risk individuals for different diseases from their genotypes. However, the broader application of PRS to the general population is hindered by the limited transferability of PRS developed in Europeans to non-European populations. To improve PRS prediction accuracy in non-European populations, we develop a statistical method called SDPRX that can effectively integrate genome wide association study summary statistics from different populations. SDPRX automatically adjusts for linkage disequilibrium differences between populations and characterizes the joint distribution of the effect sizes of a variant in two populations to be both null, population specific, or shared with correlation. Through simulations and applications to real traits, we show that SDPRX improves the prediction performance over existing methods in non-European populations. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Zhou G, Chen T, Zhao H. (2023) SDPRX: A statistical method for cross-population prediction of complex traits Am. J. Hum. Genet., 110 (1) 13-22. doi:10.1016/j.ajhg.2022.11.007. PMID 36460009 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2023 ; 110 ; 1 ; 13-22 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36460009">36460009</a> </li>
</ul>
<h3 id="tl-prs">TL-PRS</h3>
<ul>
<li><strong>NAME</strong> : TL-PRS </li>
<li><strong>SHORT NAME</strong> : TL-PRS </li>
<li><strong>FULL NAME</strong> : transfer learning PRS </li>
<li><strong>DESCRIPTION</strong> : This R package helps users to construct multi-ethnic polygenic risk score (PRS) using transfer learning. It can help predict PRS of minor ancestry using summary statistics from exsiting resources, such as UK Biobank. </li>
<li><strong>URL</strong> : <a href="https://github.com/ZhangchenZhao/TLPRS">https://github.com/ZhangchenZhao/TLPRS</a> </li>
<li><strong>TITLE</strong> : The construction of cross-population polygenic risk scores using transfer learning </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2022.09.010 </li>
<li><strong>ABSTRACT</strong> : As most existing genome-wide association studies (GWASs) were conducted in European-ancestry cohorts, and as the existing polygenic risk score (PRS) models have limited transferability across ancestry groups, PRS research on non-European-ancestry groups needs to make efficient use of available data until we attain large sample sizes across all ancestry groups. Here we propose a PRS method using transfer learning techniques. Our approach, TL-PRS, uses gradient descent to fine-tune the baseline PRS model from an ancestry group with large sample GWASs to the dataset of target ancestry. In our application of constructing PRS for seven quantitative and two dichotomous traits for 10,285 individuals of South Asian ancestry and 8,168 individuals of African ancestry in UK Biobank, TL-PRS using PRS-CS as a baseline method obtained 25% average relative improvement for South Asian samples and 29% for African samples compared to the standard PRS-CS method in terms of predicted R2. Our approach increases the transferability of PRSs across ancestries and thereby helps reduce existing inequities in genetics research. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Zhao Z, Fritsche LG, Smith JA, Mukherjee B, ...&amp;, Lee S. (2022) The construction of cross-population polygenic risk scores using transfer learning Am. J. Hum. Genet., 109 (11) 1998-2008. doi:10.1016/j.ajhg.2022.09.010. PMID 36240765 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2022 ; 109 ; 11 ; 1998-2008 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36240765">36240765</a> </li>
</ul>
<h3 id="shaprs">shaPRS</h3>
<ul>
<li><strong>NAME</strong> : shaPRS </li>
<li><strong>SHORT NAME</strong> : shaPRS </li>
<li><strong>FULL NAME</strong> : shaPRS </li>
<li><strong>DESCRIPTION</strong> : Leveraging shared genetic effects across traits and ancestries improves accuracy of polygenic scores </li>
<li><strong>URL</strong> : <a href="https://github.com/mkelcb/shaprs">https://github.com/mkelcb/shaprs</a> </li>
<li><strong>KEYWORDS</strong> : cross-ancestry, genetic correlation </li>
<li><strong>TITLE</strong> : shaPRS: Leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2024.04.009 </li>
<li><strong>ABSTRACT</strong> : We present shaPRS, a method that leverages widespread pleiotropy between traits or shared genetic effects across ancestries, to improve the accuracy of polygenic scores. The method uses genome-wide summary statistics from two diseases or ancestries to improve the genetic effect estimate and standard error at SNPs where there is homogeneity of effect between the two datasets. When there is significant evidence of heterogeneity, the genetic effect from the disease or population closest to the target population is maintained. We show via simulation and a series of real-world examples that shaPRS substantially enhances the accuracy of polygenic risk scores (PRSs) for complex diseases and greatly improves PRS performance across ancestries. shaPRS is a PRS pre-processing method that is agnostic to the actual PRS generation method, and as a result, it can be integrated into existing PRS generation pipelines and continue to be applied as more performant PRS methods are developed over time. </li>
<li><strong>CITATION</strong> : Kelemen M, Vigorito E, Fachal L, Anderson CA, ...&amp;, Wallace C. (2024) shaPRS: Leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores Am. J. Hum. Genet.,  () . doi:10.1016/j.ajhg.2024.04.009. PMID 38703768 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2024 ; <NA> ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38703768">38703768</a> </li>
</ul>
<h2 id="cross-trait">Cross-trait</h2>
<h3 id="ddx-prs">DDx-PRS</h3>
<ul>
<li><strong>NAME</strong> : DDx-PRS </li>
<li><strong>SHORT NAME</strong> : DDx-PRS </li>
<li><strong>FULL NAME</strong> : Differential Diagnosis-Polygenic Risk Score </li>
<li><strong>DESCRIPTION</strong> : The DDxPRS R function provides a tool for distuingishing different disorders based on polygenic prediction. </li>
<li><strong>URL</strong> : <a href="https://github.com/wouterpeyrot/DDxPRS">https://github.com/wouterpeyrot/DDxPRS</a> </li>
<li><strong>CITATION</strong> : Peyrot, W. J., Panagiotaropoulou, G., Olde Loohuis, L. M., Adams, M., Awasthi, S., Ge, T., ... &amp; Price, A. L. (2024). Distinguishing different psychiatric disorders using DDx-PRS. medRxiv, 2024-02. </li>
</ul>
<h3 id="mixer">MiXeR</h3>
<ul>
<li><strong>NAME</strong> : MiXeR </li>
<li><strong>SHORT NAME</strong> : MiXeR </li>
<li><strong>FULL NAME</strong> : MiXeR（cross-trait analysis） </li>
<li><strong>DESCRIPTION</strong> : Causal Mixture Model for GWAS summary statistics </li>
<li><strong>URL</strong> : <a href="https://github.com/precimed/mixer">https://github.com/precimed/mixer</a> </li>
<li><strong>TITLE</strong> : Bivariate causal mixture model quantifies polygenic overlap between complex traits beyond genetic correlation </li>
<li><strong>DOI</strong> : 10.1038/s41467-019-10310-0 </li>
<li><strong>ABSTRACT</strong> : Accumulating evidence from genome wide association studies (GWAS) suggests an abundance of shared genetic influences among complex human traits and disorders, such as mental disorders. Here we introduce a statistical tool, MiXeR, which quantifies polygenic overlap irrespective of genetic correlation, using GWAS summary statistics. MiXeR results are presented as a Venn diagram of unique and shared polygenic components across traits. At 90% of SNP-heritability explained for each phenotype, MiXeR estimates that 8.3 K variants causally influence schizophrenia and 6.4 K influence bipolar disorder. Among these variants, 6.2 K are shared between the disorders, which have a high genetic correlation. Further, MiXeR uncovers polygenic overlap between schizophrenia and educational attainment. Despite a genetic correlation close to zero, the phenotypes share 8.3 K causal variants, while 2.5 K additional variants influence only educational attainment. By considering the polygenicity, discoverability and heritability of complex phenotypes, MiXeR analysis may improve our understanding of cross-trait genetic architectures. </li>
<li><strong>CITATION</strong> : Frei O, Holland D, Smeland OB, Shadrin AA, ...&amp;, Dale AM. (2019) Bivariate causal mixture model quantifies polygenic overlap between complex traits beyond genetic correlation Nat. Commun., 10 (1) 2417. doi:10.1038/s41467-019-10310-0. PMID 31160569 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2019 ; 10 ; 1 ; 2417 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31160569">31160569</a> </li>
</ul>
<h3 id="multi-pgs">Multi-PGS</h3>
<ul>
<li><strong>NAME</strong> : Multi-PGS </li>
<li><strong>SHORT NAME</strong> : Multi-PGS </li>
<li><strong>FULL NAME</strong> : Multi-PGS </li>
<li><strong>DESCRIPTION</strong> : a framework to generate enriched PGS from a wealth of publicly available genome-wide association studies, combining thousands of studies focused on many different phenotypes, into a multi-PGS </li>
<li><strong>URL</strong> : <a href="https://github.com/ClaraAlbi/paper_multiPGS">https://github.com/ClaraAlbi/paper_multiPGS</a> </li>
<li><strong>TITLE</strong> : Multi-PGS enhances polygenic prediction by combining 937 polygenic scores </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-40330-w </li>
<li><strong>ABSTRACT</strong> : The predictive performance of polygenic scores (PGS) is largely dependent on the number of samples available to train the PGS. Increasing the sample size for a specific phenotype is expensive and takes time, but this sample size can be effectively increased by using genetically correlated phenotypes. We propose a framework to generate multi-PGS from thousands of publicly available genome-wide association studies (GWAS) with no need to individually select the most relevant ones. In this study, the multi-PGS framework increases prediction accuracy over single PGS for all included psychiatric disorders and other available outcomes, with prediction R2 increases of up to 9-fold for attention-deficit/hyperactivity disorder compared to a single PGS. We also generate multi-PGS for phenotypes without an existing GWAS and for case-case predictions. We benchmark the multi-PGS framework against other methods and highlight its potential application to new emerging biobanks. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Albiñana C, Zhu Z, Schork AJ, Ingason A, ...&amp;, Vilhjálmsson BJ. (2023) Multi-PGS enhances polygenic prediction by combining 937 polygenic scores Nat. Commun., 14 (1) 4702. doi:10.1038/s41467-023-40330-w. PMID 37543680 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 4702 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37543680">37543680</a> </li>
</ul>
<h3 id="puma-cubs">PUMA-CUBS</h3>
<ul>
<li><strong>NAME</strong> : PUMA-CUBS </li>
<li><strong>SHORT NAME</strong> : PUMA-CUBS </li>
<li><strong>FULL NAME</strong> : PUMA-CUBS </li>
<li><strong>DESCRIPTION</strong> : an ensemble learning strategy named PUMACUBS to combine multiple PRS models into an ensemble score without requiring external data for model fitting. </li>
<li><strong>URL</strong> : <a href="https://github.com/qlu-lab/PUMAS">https://github.com/qlu-lab/PUMAS</a> </li>
<li><strong>CITATION</strong> : Zhao, Zijie, et al. "Optimizing and benchmarking polygenic risk scores with GWAS summary statistics." bioRxiv (2022). </li>
</ul>
<h3 id="wmt-sblup">wMT-SBLUP</h3>
<ul>
<li><strong>NAME</strong> : wMT-SBLUP </li>
<li><strong>SHORT NAME</strong> : wMT-SBLUP </li>
<li><strong>FULL NAME</strong> : weighted approximate multi-trait summary statistic BLUP </li>
<li><strong>URL</strong> : <a href="https://github.com/uqrmaie1/smtpred">https://github.com/uqrmaie1/smtpred</a> </li>
<li><strong>TITLE</strong> : Improving genetic prediction by leveraging genetic correlations among human diseases and traits </li>
<li><strong>DOI</strong> : 10.1038/s41467-017-02769-6 </li>
<li><strong>ABSTRACT</strong> : Genomic prediction has the potential to contribute to precision medicine. However, to date, the utility of such predictors is limited due to low accuracy for most traits. Here theory and simulation study are used to demonstrate that widespread pleiotropy among phenotypes can be utilised to improve genomic risk prediction. We show how a genetic predictor can be created as a weighted index that combines published genome-wide association study (GWAS) summary statistics across many different traits. We apply this framework to predict risk of schizophrenia and bipolar disorder in the Psychiatric Genomics consortium data, finding substantial heterogeneity in prediction accuracy increases across cohorts. For six additional phenotypes in the UK Biobank data, we find increases in prediction accuracy ranging from 0.7% for height to 47% for type 2 diabetes, when using a multi-trait predictor that combines published summary statistics from multiple traits, as compared to a predictor based only on one trait. </li>
<li><strong>CITATION</strong> : Maier RM, Zhu Z, Lee SH, Trzaskowski M, ...&amp;, Robinson MR. (2018) Improving genetic prediction by leveraging genetic correlations among human diseases and traits Nat. Commun., 9 (1) 989. doi:10.1038/s41467-017-02769-6. PMID 29515099 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2018 ; 9 ; 1 ; 989 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29515099">29515099</a> </li>
</ul>
<h2 id="misc">MISC</h2>
<h3 id="scprs">scPRS</h3>
<ul>
<li><strong>NAME</strong> : scPRS </li>
<li><strong>SHORT NAME</strong> : scPRS </li>
<li><strong>FULL NAME</strong> : scPRS </li>
<li><strong>DESCRIPTION</strong> : We introduce scPRS, an interpretable geometric deep learning model that contructs single-cell-resolved PRS leveraging reference single-cell ATAC-seq data for enhanced disease prediction and biological discovery. </li>
<li><strong>URL</strong> : <a href="https://github.com/szhang1112/scPRS">https://github.com/szhang1112/scPRS</a> </li>
<li><strong>KEYWORDS</strong> : GWAS, scATAC, cell-resolved PRS, GNN </li>
<li><strong>CITATION</strong> : Zhang, S. et al. Single-cell polygenic risk scores dissect cellular and molecular heterogeneity of complex human diseases. Nat. Biotechnol. 1–17 (2025). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/40715455">40715455</a> </li>
</ul>
<h2 id="pathway">Pathway</h2>
<h3 id="prset">PRSet</h3>
<ul>
<li><strong>NAME</strong> : PRSet </li>
<li><strong>SHORT NAME</strong> : PRSet </li>
<li><strong>FULL NAME</strong> : PRSet </li>
<li><strong>DESCRIPTION</strong> : A new feature of PRSice is the ability to perform set base/pathway based analysis. This new feature is called PRSet. </li>
<li><strong>URL</strong> : <a href="https://www.prsice.info/quick_start_prset/">https://www.prsice.info/quick_start_prset/</a> </li>
<li><strong>KEYWORDS</strong> : pathway-based </li>
<li><strong>TITLE</strong> : PRSet: Pathway-based polygenic risk score analyses and software </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1010624 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs) have been among the leading advances in biomedicine in recent years. As a proxy of genetic liability, PRSs are utilised across multiple fields and applications. While numerous statistical and machine learning methods have been developed to optimise their predictive accuracy, these typically distil genetic liability to a single number based on aggregation of an individual's genome-wide risk alleles. This results in a key loss of information about an individual's genetic profile, which could be critical given the functional sub-structure of the genome and the heterogeneity of complex disease. In this manuscript, we introduce a 'pathway polygenic' paradigm of disease risk, in which multiple genetic liabilities underlie complex diseases, rather than a single genome-wide liability. We describe a method and accompanying software, PRSet, for computing and analysing pathway-based PRSs, in which polygenic scores are calculated across genomic pathways for each individual. We evaluate the potential of pathway PRSs in two distinct ways, creating two major sections: (1) In the first section, we benchmark PRSet as a pathway enrichment tool, evaluating its capacity to capture GWAS signal in pathways. We find that for target sample sizes of &gt;10,000 individuals, pathway PRSs have similar power for evaluating pathway enrichment as leading methods MAGMA and LD score regression, with the distinct advantage of providing individual-level estimates of genetic liability for each pathway -opening up a range of pathway-based PRS applications, (2) In the second section, we evaluate the performance of pathway PRSs for disease stratification. We show that using a supervised disease stratification approach, pathway PRSs (computed by PRSet) outperform two standard genome-wide PRSs (computed by C+T and lassosum) for classifying disease subtypes in 20 of 21 scenarios tested. As the definition and functional annotation of pathways becomes increasingly refined, we expect pathway PRSs to offer key insights into the heterogeneity of complex disease and treatment response, to generate biologically tractable therapeutic targets from polygenic signal, and, ultimately, to provide a powerful path to precision medicine. </li>
<li><strong>CITATION</strong> : Choi SW, García-González J, Ruan Y, Wu HM, ...&amp;, O'Reilly PF. (2023) PRSet: Pathway-based polygenic risk score analyses and software PLoS Genet., 19 (2) e1010624. doi:10.1371/journal.pgen.1010624. PMID 36749789 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2023 ; 19 ; 2 ; e1010624 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36749789">36749789</a> </li>
</ul>
<h2 id="pipeline">Pipeline</h2>
<h3 id="plink2">PLINK2</h3>
<ul>
<li><strong>NAME</strong> : PLINK2 </li>
<li><strong>SHORT NAME</strong> : PLINK2 </li>
<li><strong>FULL NAME</strong> : PLINK2 </li>
<li><strong>DESCRIPTION</strong> : The second-generation versions of PLINK will offer dramatic improvements in performance and compatibility. </li>
<li><strong>URL</strong> : <a href="https://www.cog-genomics.org/plink/2.0/">https://www.cog-genomics.org/plink/2.0/</a> </li>
<li><strong>USE</strong> : calculate PRS using genotype data. </li>
<li><strong>TITLE</strong> : Second-generation PLINK: rising to the challenge of larger and richer datasets </li>
<li><strong>DOI</strong> : 10.1186/s13742-015-0047-8 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: PLINK 1 is a widely used open-source C/C++ toolset for genome-wide association studies (GWAS) and research in population genetics. However, the steady accumulation of data from imputation and whole-genome sequencing studies has exposed a strong need for faster and scalable implementations of key functions, such as logistic regression, linkage disequilibrium estimation, and genomic distance evaluation. In addition, GWAS and population-genetic data now frequently contain genotype likelihoods, phase information, and/or multiallelic variants, none of which can be represented by PLINK 1's primary data format. FINDINGS: To address these issues, we are developing a second-generation codebase for PLINK. The first major release from this codebase, PLINK 1.9, introduces extensive use of bit-level parallelism, [Formula: see text]-time/constant-space Hardy-Weinberg equilibrium and Fisher's exact tests, and many other algorithmic improvements. In combination, these changes accelerate most operations by 1-4 orders of magnitude, and allow the program to handle datasets too large to fit in RAM. We have also developed an extension to the data format which adds low-overhead support for genotype likelihoods, phase, multiallelic variants, and reference vs. alternate alleles, which is the basis of our planned second release (PLINK 2.0). CONCLUSIONS: The second-generation versions of PLINK will offer dramatic improvements in performance and compatibility. For the first time, users without access to high-end computing resources can perform several essential analyses of the feature-rich and very large genetic datasets coming into use. </li>
<li><strong>CITATION</strong> : Chang CC, Chow CC, Tellier LC, Vattikuti S, ...&amp;, Lee JJ. (2015) Second-generation PLINK: rising to the challenge of larger and richer datasets Gigascience, 4 (1) 7. doi:10.1186/s13742-015-0047-8. PMID 25722852 </li>
<li><strong>JOURNAL_INFO</strong> : GigaScience ; Gigascience ; 2015 ; 4 ; 1 ; 7 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25722852">25722852</a> </li>
</ul>
<h3 id="prsice-2">PRSice-2</h3>
<ul>
<li><strong>NAME</strong> : PRSice-2 </li>
<li><strong>SHORT NAME</strong> : PRSice-2 </li>
<li><strong>FULL NAME</strong> : PRSice-2 </li>
<li><strong>DESCRIPTION</strong> : PRSice (pronounced 'precise') is a Polygenic Risk Score software for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS) analyses. </li>
<li><strong>URL</strong> : <a href="https://www.prsice.info/">https://www.prsice.info/</a> </li>
<li><strong>TITLE</strong> : PRSice-2: Polygenic Risk Score software for biobank-scale data </li>
<li><strong>DOI</strong> : 10.1093/gigascience/giz082 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: Polygenic risk score (PRS) analyses have become an integral part of biomedical research, exploited to gain insights into shared aetiology among traits, to control for genomic profile in experimental studies, and to strengthen causal inference, among a range of applications. Substantial efforts are now devoted to biobank projects to collect large genetic and phenotypic data, providing unprecedented opportunity for genetic discovery and applications. To process the large-scale data provided by such biobank resources, highly efficient and scalable methods and software are required. RESULTS: Here we introduce PRSice-2, an efficient and scalable software program for automating and simplifying PRS analyses on large-scale data. PRSice-2 handles both genotyped and imputed data, provides empirical association P-values free from inflation due to overfitting, supports different inheritance models, and can evaluate multiple continuous and binary target traits simultaneously. We demonstrate that PRSice-2 is dramatically faster and more memory-efficient than PRSice-1 and alternative PRS software, LDpred and lassosum, while having comparable predictive power. CONCLUSION: PRSice-2's combination of efficiency and power will be increasingly important as data sizes grow and as the applications of PRS become more sophisticated, e.g., when incorporated into high-dimensional or gene set-based analyses. PRSice-2 is written in C++, with an R script for plotting, and is freely available for download from http://PRSice.info. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Choi SW, O'Reilly PF. (2019) PRSice-2: Polygenic Risk Score software for biobank-scale data Gigascience, 8 (7) 1-6. doi:10.1093/gigascience/giz082. PMID 31307061 </li>
<li><strong>JOURNAL_INFO</strong> : GigaScience ; Gigascience ; 2019 ; 8 ; 7 ; 1-6 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31307061">31307061</a> </li>
</ul>
<h3 id="pgsc_calc">pgsc_calc</h3>
<ul>
<li><strong>NAME</strong> : pgsc_calc </li>
<li><strong>SHORT NAME</strong> : pgsc_calc </li>
<li><strong>FULL NAME</strong> : The Polygenic Score Catalog Calculator </li>
<li><strong>DESCRIPTION</strong> : pgsc_calc is a bioinformatics best-practice analysis pipeline for calculating polygenic [risk] scores on samples with imputed genotypes using existing scoring files from the Polygenic Score (PGS) Catalog and/or user-defined PGS/PRS. </li>
<li><strong>URL</strong> : <a href="https://github.com/PGScatalog/pgsc_calc">https://github.com/PGScatalog/pgsc_calc</a> </li>
<li><strong>KEYWORDS</strong> : PRS calculation pipeline </li>
<li><strong>CITATION</strong> : Lambert, Wingfield et al. (2024) The Polygenic Score Catalog: new functionality and tools to enable FAIR research. medRxiv. doi:10.1101/2024.05.29.24307783. </li>
</ul>
<h2 id="platform">Platform</h2>
<h3 id="cancer-prsweb">Cancer PRSweb</h3>
<ul>
<li><strong>NAME</strong> : Cancer PRSweb </li>
<li><strong>SHORT NAME</strong> : Cancer PRSweb </li>
<li><strong>FULL NAME</strong> : Cancer PRSweb </li>
<li><strong>DESCRIPTION</strong> : Our framework condenses these summary statistics into PRS using linkage disequilibrium pruning and p-value thresholding (fixed or data-adaptively optimized thresholds) or penalized, genome-wide effect size weighting. We evaluate them in the cancer-enriched cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each PRS construct, measures on performance, calibration, and discrimination are provided. Beyond the cancer PRS evaluation in MGI and UKB, the PRSweb platform features construct downloads, risk evaluation in the top percentiles, and phenome-wide PRS association studies (PRS-PheWAS) for a subset of PRS that are predictive for the primary cancer. </li>
<li><strong>URL</strong> : <a href="https://prsweb.sph.umich.edu:8443/">https://prsweb.sph.umich.edu:8443/</a> </li>
<li><strong>KEYWORDS</strong> : Cancer PRS </li>
<li><strong>TITLE</strong> : Cancer PRSweb: An online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2020.08.025 </li>
<li><strong>ABSTRACT</strong> : To facilitate scientific collaboration on polygenic risk scores (PRSs) research, we created an extensive PRS online repository for 35 common cancer traits integrating freely available genome-wide association studies (GWASs) summary statistics from three sources: published GWASs, the NHGRI-EBI GWAS Catalog, and UK Biobank-based GWASs. Our framework condenses these summary statistics into PRSs using various approaches such as linkage disequilibrium pruning/p value thresholding (fixed or data-adaptively optimized thresholds) and penalized, genome-wide effect size weighting. We evaluated the PRSs in two biobanks: the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and the population-based UK Biobank (UKB). For each PRS construct, we provide measures on predictive performance and discrimination. Besides PRS evaluation, the Cancer-PRSweb platform features construct downloads and phenome-wide PRS association study results (PRS-PheWAS) for predictive PRSs. We expect this integrated platform to accelerate PRS-related cancer research. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Fritsche LG, Patil S, Beesley LJ, VandeHaar P, ...&amp;, Mukherjee B. (2020) Cancer PRSweb: An online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks Am. J. Hum. Genet., 107 (5) 815-836. doi:10.1016/j.ajhg.2020.08.025. PMID 32991828 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2020 ; 107 ; 5 ; 815-836 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32991828">32991828</a> </li>
</ul>
<h3 id="exprsweb">ExPRSweb</h3>
<ul>
<li><strong>NAME</strong> : ExPRSweb </li>
<li><strong>SHORT NAME</strong> : ExPRSweb </li>
<li><strong>FULL NAME</strong> : exposure polygenic risk scores (ExPRSs) </li>
<li><strong>DESCRIPTION</strong> : Integrating published and freely available genome-wide association studies (GWAS) summary statistics from multiple sources (published GWAS, the NHGRI-EBI GWAS Catalog, FinnGen- or UKB-based GWAS), we created an online repository for exposure polygenic risk scores (ExPRS) for health-related exposure traits. Our framework condenses these summary statistics into ExPRS using linkage disequilibrium pruning and p-value thresholding (P&amp;T) or penalized, genome-wide effect size weighting. We evaluate them in the cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each ExPRS construct, measures on performance, accuracy, and discrimination are provided. Beyond the ExPRS evaluation in MGI and UKB, the ExPRSweb platform features construct downloads, evaluation in the top percentiles, and phenome-wide ExPRS association studies (ExPRS-PheWAS) for a subset of ExPRS that are predictive for the corresponding exposure. </li>
<li><strong>URL</strong> : <a href="https://exprsweb.sph.umich.edu:8443/">https://exprsweb.sph.umich.edu:8443/</a> </li>
<li><strong>KEYWORDS</strong> : exposure PRS </li>
<li><strong>TITLE</strong> : ExPRSweb: An online repository with polygenic risk scores for common health-related exposures </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2022.09.001 </li>
<li><strong>ABSTRACT</strong> : Complex traits are influenced by genetic risk factors, lifestyle, and environmental variables, so-called exposures. Some exposures, e.g., smoking or lipid levels, have common genetic modifiers identified in genome-wide association studies. Because measurements are often unfeasible, exposure polygenic risk scores (ExPRSs) offer an alternative to study the influence of exposures on various phenotypes. Here, we collected publicly available summary statistics for 28 exposures and applied four common PRS methods to generate ExPRSs in two large biobanks: the Michigan Genomics Initiative and the UK Biobank. We established ExPRSs for 27 exposures and demonstrated their applicability in phenome-wide association studies and as predictors for common chronic conditions. Especially the addition of multiple ExPRSs showed, for several chronic conditions, an improvement compared to prediction models that only included traditional, disease-focused PRSs. To facilitate follow-up studies, we share all ExPRS constructs and generated results via an online repository called ExPRSweb. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Ma Y, Patil S, Zhou X, Mukherjee B, ...&amp;, Fritsche LG. (2022) ExPRSweb: An online repository with polygenic risk scores for common health-related exposures Am. J. Hum. Genet., 109 (10) 1742-1760. doi:10.1016/j.ajhg.2022.09.001. PMID 36152628 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2022 ; 109 ; 10 ; 1742-1760 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36152628">36152628</a> </li>
</ul>
<h3 id="pgscatalog">PGSCatalog</h3>
<ul>
<li><strong>NAME</strong> : PGSCatalog </li>
<li><strong>SHORT NAME</strong> : PGS Catalog </li>
<li><strong>FULL NAME</strong> : PGS Catalog </li>
<li><strong>DESCRIPTION</strong> : The PGS Catalog is an open database of published polygenic scores (PGS). Each PGS in the Catalog is consistently annotated with relevant metadata; including scoring files (variants, effect alleles/weights), annotations of how the PGS was developed and applied, and evaluations of their predictive performance. </li>
<li><strong>URL</strong> : <a href="https://www.pgscatalog.org/">https://www.pgscatalog.org/</a> </li>
<li><strong>KEYWORDS</strong> : PGS database </li>
<li><strong>TITLE</strong> : The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation </li>
<li><strong>DOI</strong> : 10.1038/s41588-021-00783-5 </li>
<li><strong>ABSTRACT</strong> : We present the Polygenic Score (PGS) Catalog (https://www.PGSCatalog.org), an open resource of published scores (including variants, alleles and weights) and consistently curated metadata required for reproducibility and independent applications. The PGS Catalog has capabilities for user deposition, expert curation and programmatic access, thus providing the community with a platform for PGS dissemination, research and translation. </li>
<li><strong>CITATION</strong> : Lambert SA, Gil L, Jupp S, Ritchie SC, ...&amp;, Inouye M. (2021) The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation Nat. Genet., 53 (4) 420-425. doi:10.1038/s41588-021-00783-5. PMID 33692568 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2021 ; 53 ; 4 ; 420-425 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33692568">33692568</a> </li>
</ul>
<h3 id="pgsfusion">PGSFusion</h3>
<ul>
<li><strong>NAME</strong> : PGSFusion </li>
<li><strong>SHORT NAME</strong> : PGSFusion </li>
<li><strong>DESCRIPTION</strong> : PGSFusion is your free comprehensive webserver for constructing polygenic scores (PGS)evaluating performance, and unlocking epidemiological insights. This server implements 16 leading summary statistics-based PGS methods in a standardized interface, and rigorously assesses their predictive capabilities using the UK Biobank dataset. </li>
<li><strong>URL</strong> : <a href="http://www.pgsfusion.net/#/">http://www.pgsfusion.net/#/</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.08.05.606619 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Yang, S., Ye, X., Ji, X., Li, Z., Tian, M., Huang, P., &amp; Cao, C. (2024). PGSFusion streamlines polygenic score construction and epidemiological applications in biobank-scale cohorts. bioRxiv, 2024-08. </li>
</ul>
<h3 id="prs-atlas">PRS atlas</h3>
<ul>
<li><strong>NAME</strong> : PRS atlas </li>
<li><strong>SHORT NAME</strong> : PRS atlas </li>
<li><strong>FULL NAME</strong> : PRS atlas </li>
<li><strong>DESCRIPTION</strong> : This web application can be used to query findings from an analysis of 162 polygenic risk scores and 551 complex traits using data from the UK Biobank study1. Traits were selected based on the heritability analysis conducted by the Neale Lab2 (P&lt;0.05). We encourage users of this resource to conduct follow-up analyses of associations to robustly identify causal relationships between complex traits. </li>
<li><strong>URL</strong> : <a href="http://mrcieu.mrsoftware.org/PRS_atlas/">http://mrcieu.mrsoftware.org/PRS_atlas/</a> </li>
<li><strong>TITLE</strong> : An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome </li>
<li><strong>DOI</strong> : 10.7554/eLife.43657 </li>
<li><strong>ABSTRACT</strong> : The age of large-scale genome-wide association studies (GWAS) has provided us with an unprecedented opportunity to evaluate the genetic liability of complex disease using polygenic risk scores (PRS). In this study, we have analysed 162 PRS (p&lt;5×10-05) derived from GWAS and 551 heritable traits from the UK Biobank study (N = 334,398). Findings can be investigated using a web application (http:‌//‌mrcieu.‌mrsoftware.org/‌PRS‌_atlas/), which we envisage will help uncover both known and novel mechanisms which contribute towards disease susceptibility. To demonstrate this, we have investigated the results from a phenome-wide evaluation of schizophrenia genetic liability. Amongst findings were inverse associations with measures of cognitive function which extensive follow-up analyses using Mendelian randomization (MR) provided evidence of a causal relationship. We have also investigated the effect of multiple risk factors on disease using mediation and multivariable MR frameworks. Our atlas provides a resource for future endeavours seeking to unravel the causal determinants of complex disease. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Richardson TG, Harrison S, Hemani G, Davey Smith G. (2019) An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome Elife, 8 () . doi:10.7554/eLife.43657. PMID 30835202 </li>
<li><strong>JOURNAL_INFO</strong> : eLife ; Elife ; 2019 ; 8 ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30835202">30835202</a> </li>
</ul>
<h3 id="metabolites-prs-atlas">metabolites PRS atlas</h3>
<ul>
<li><strong>NAME</strong> : metabolites PRS atlas </li>
<li><strong>SHORT NAME</strong> : metabolites PRS atlas </li>
<li><strong>FULL NAME</strong> : metabolites PRS atlas </li>
<li><strong>DESCRIPTION</strong> : This web application can be used to query findings from a systematic analysis of 129 polygenic risk scores and 249 circulating metabolits using high-throughput nuclear magnetic resonance data from the UK Biobank study1,2. We encourage users of this resource to conduct follow-up analyses of associations to investigate potential causal and non-causal metabolic biomarkers. Age-stratified results can be used to investigate how potential sources of collider bias (e.g. statin therapy) may influence findings in the full sample </li>
<li><strong>URL</strong> : <a href="http://mrcieu.mrsoftware.org/metabolites_PRS_atlas/">http://mrcieu.mrsoftware.org/metabolites_PRS_atlas/</a> </li>
<li><strong>TITLE</strong> : Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers </li>
<li><strong>DOI</strong> : 10.7554/eLife.73951 </li>
<li><strong>ABSTRACT</strong> : Background: Polygenic scores (PGS) are becoming an increasingly popular approach to predict complex disease risk, although they also hold the potential to develop insight into the molecular profiles of patients with an elevated genetic predisposition to disease. Methods: We sought to construct an atlas of associations between 125 different PGS derived using results from genome-wide association studies and 249 circulating metabolites in up to 83,004 participants from the UK Biobank. Results: As an exemplar to demonstrate the value of this atlas, we conducted a hypothesis-free evaluation of all associations with glycoprotein acetyls (GlycA), an inflammatory biomarker. Using bidirectional Mendelian randomization, we find that the associations highlighted likely reflect the effect of risk factors, such as adiposity or liability towards smoking, on systemic inflammation as opposed to the converse direction. Moreover, we repeated all analyses in our atlas within age strata to investigate potential sources of collider bias, such as medication usage. This was exemplified by comparing associations between lipoprotein lipid profiles and the coronary artery disease PGS in the youngest and oldest age strata, which had differing proportions of individuals undergoing statin therapy. Lastly, we generated all PGS-metabolite associations stratified by sex and separately after excluding 13 established lipid-associated loci to further evaluate the robustness of findings. Conclusions: We envisage that the atlas of results constructed in our study will motivate future hypothesis generation and help prioritize and deprioritize circulating metabolic traits for in-depth investigations. All results can be visualized and downloaded at http://mrcieu.mrsoftware.org/metabolites_PRS_atlas. Funding: This work is supported by funding from the Wellcome Trust, the British Heart Foundation, and the Medical Research Council Integrative Epidemiology Unit. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Fang S, Holmes MV, Gaunt TR, Davey Smith G, ...&amp;, Richardson TG. (2022) Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers Elife, 11 () e73951. doi:10.7554/eLife.73951. PMID 36219204 </li>
<li><strong>JOURNAL_INFO</strong> : eLife ; Elife ; 2022 ; 11 ; <NA> ; e73951 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36219204">36219204</a> </li>
</ul>
<h2 id="polygenicity">Polygenicity</h2>
<h3 id="bayesr">BayesR</h3>
<ul>
<li><strong>NAME</strong> : BayesR </li>
<li><strong>SHORT NAME</strong> : BayesR </li>
<li><strong>FULL NAME</strong> : BayesR </li>
<li><strong>DESCRIPTION</strong> : Bayesian mixture model to dissect genetic variation for disease in human populations and to construct more powerful risk predictors </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a> </li>
<li><strong>TITLE</strong> : Simultaneous discovery, estimation and prediction analysis of complex traits using a bayesian mixture model </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1004969 </li>
<li><strong>ABSTRACT</strong> : Gene discovery, estimation of heritability captured by SNP arrays, inference on genetic architecture and prediction analyses of complex traits are usually performed using different statistical models and methods, leading to inefficiency and loss of power. Here we use a Bayesian mixture model that simultaneously allows variant discovery, estimation of genetic variance explained by all variants and prediction of unobserved phenotypes in new samples. We apply the method to simulated data of quantitative traits and Welcome Trust Case Control Consortium (WTCCC) data on disease and show that it provides accurate estimates of SNP-based heritability, produces unbiased estimators of risk in new samples, and that it can estimate genetic architecture by partitioning variation across hundreds to thousands of SNPs. We estimated that, depending on the trait, 2,633 to 9,411 SNPs explain all of the SNP-based heritability in the WTCCC diseases. The majority of those SNPs (&gt;96%) had small effects, confirming a substantial polygenic component to common diseases. The proportion of the SNP-based variance explained by large effects (each SNP explaining 1% of the variance) varied markedly between diseases, ranging from almost zero for bipolar disorder to 72% for type 1 diabetes. Prediction analyses demonstrate that for diseases with major loci, such as type 1 diabetes and rheumatoid arthritis, Bayesian methods outperform profile scoring or mixed model approaches. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Moser G, Lee SH, Hayes BJ, Goddard ME, ...&amp;, Visscher PM. (2015) Simultaneous discovery, estimation and prediction analysis of complex traits using a bayesian mixture model PLoS Genet., 11 (4) e1004969. doi:10.1371/journal.pgen.1004969. PMID 25849665 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2015 ; 11 ; 4 ; e1004969 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25849665">25849665</a> </li>
</ul>
<h3 id="bayess">BayesS</h3>
<ul>
<li><strong>NAME</strong> : BayesS </li>
<li><strong>SHORT NAME</strong> : BayesS </li>
<li><strong>FULL NAME</strong> : BayesS </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a> </li>
<li><strong>TITLE</strong> : Signatures of negative selection in the genetic architecture of human complex traits </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0101-4 </li>
<li><strong>ABSTRACT</strong> : We develop a Bayesian mixed linear model that simultaneously estimates single-nucleotide polymorphism (SNP)-based heritability, polygenicity (proportion of SNPs with nonzero effects), and the relationship between SNP effect size and minor allele frequency for complex traits in conventionally unrelated individuals using genome-wide SNP data. We apply the method to 28 complex traits in the UK Biobank data (N = 126,752) and show that on average, 6% of SNPs have nonzero effects, which in total explain 22% of phenotypic variance. We detect significant (P &lt; 0.05/28) signatures of natural selection in the genetic architecture of 23 traits, including reproductive, cardiovascular, and anthropometric traits, as well as educational attainment. The significant estimates of the relationship between effect size and minor allele frequency in complex traits are consistent with a model of negative (or purifying) selection, as confirmed by forward simulation. We conclude that negative selection acts pervasively on the genetic variants associated with human complex traits. </li>
<li><strong>CITATION</strong> : Zeng J, de Vlaming R, Wu Y, Robinson MR, ...&amp;, Yang J. (2018) Signatures of negative selection in the genetic architecture of human complex traits Nat. Genet., 50 (5) 746-753. doi:10.1038/s41588-018-0101-4. PMID 29662166 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2018 ; 50 ; 5 ; 746-753 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/29662166">29662166</a> </li>
</ul>
<h3 id="sbayesr">SBayesR</h3>
<ul>
<li><strong>NAME</strong> : SBayesR </li>
<li><strong>SHORT NAME</strong> : SBayesR </li>
<li><strong>FULL NAME</strong> : SBayesR </li>
<li><strong>DESCRIPTION</strong> : extend a powerful individual-level data Bayesian multiple regression model (BayesR) to one that utilises summary statistics from genome-wide association studies, SBayesR. </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a> </li>
<li><strong>TITLE</strong> : Improved polygenic prediction by Bayesian multiple regression on summary statistics </li>
<li><strong>DOI</strong> : 10.1038/s41467-019-12653-0 </li>
<li><strong>ABSTRACT</strong> : Accurate prediction of an individual's phenotype from their DNA sequence is one of the great promises of genomics and precision medicine. We extend a powerful individual-level data Bayesian multiple regression model (BayesR) to one that utilises summary statistics from genome-wide association studies (GWAS), SBayesR. In simulation and cross-validation using 12 real traits and 1.1 million variants on 350,000 individuals from the UK Biobank, SBayesR improves prediction accuracy relative to commonly used state-of-the-art summary statistics methods at a fraction of the computational resources. Furthermore, using summary statistics for variants from the largest GWAS meta-analysis (n ≈ 700, 000) on height and BMI, we show that on average across traits and two independent data sets that SBayesR improves prediction R2 by 5.2% relative to LDpred and by 26.5% relative to clumping and p value thresholding. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Lloyd-Jones LR, Zeng J, Sidorenko J, Yengo L, ...&amp;, Visscher PM. (2019) Improved polygenic prediction by Bayesian multiple regression on summary statistics Nat. Commun., 10 (1) 5086. doi:10.1038/s41467-019-12653-0. PMID 31704910 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2019 ; 10 ; 1 ; 5086 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31704910">31704910</a> </li>
</ul>
<h3 id="sbayesrc">SBayesRC</h3>
<ul>
<li><strong>NAME</strong> : SBayesRC </li>
<li><strong>SHORT NAME</strong> : SBayesRC </li>
<li><strong>FULL NAME</strong> : SBayesRC </li>
<li><strong>DESCRIPTION</strong> : SBayesRC integrates GWAS summary statistics with functional genomic annotations to improve polygenic prediction of complex traits. </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a> </li>
<li><strong>KEYWORDS</strong> : functional genomic annotation, whole-genome variants,  cross-ancestry </li>
<li><strong>TITLE</strong> : Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries </li>
<li><strong>DOI</strong> : 10.1038/s41588-024-01704-y </li>
<li><strong>ABSTRACT</strong> : We develop a method, SBayesRC, that integrates genome-wide association study (GWAS) summary statistics with functional genomic annotations to improve polygenic prediction of complex traits. Our method is scalable to whole-genome variant analysis and refines signals from functional annotations by allowing them to affect both causal variant probability and causal effect distribution. We analyze 50 complex traits and diseases using ∼7 million common single-nucleotide polymorphisms (SNPs) and 96 annotations. SBayesRC improves prediction accuracy by 14% in European ancestry and up to 34% in cross-ancestry prediction compared to the baseline method SBayesR, which does not use annotations, and outperforms other methods, including LDpred2, LDpred-funct, MegaPRS, PolyPred-S and PRS-CSx. Investigation of factors affecting prediction accuracy identifies a significant interaction between SNP density and annotation information, suggesting whole-genome sequence variants with annotations may further improve prediction. Functional partitioning analysis highlights a major contribution of evolutionary constrained regions to prediction accuracy and the largest per-SNP contribution from nonsynonymous SNPs. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Zheng Z, Liu S, Sidorenko J, Wang Y, ...&amp;, Zeng J. (2024) Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries Nat. Genet., 56 (5) 767-777. doi:10.1038/s41588-024-01704-y. PMID 38689000 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2024 ; 56 ; 5 ; 767-777 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38689000">38689000</a> </li>
</ul>
<h3 id="sbayess">SBayesS</h3>
<ul>
<li><strong>NAME</strong> : SBayesS </li>
<li><strong>SHORT NAME</strong> : SBayesS </li>
<li><strong>FULL NAME</strong> : SBayesS </li>
<li><strong>DESCRIPTION</strong> : estimate multiple genetic architecture parameters including selection signature using only GWAS summary statistics </li>
<li><strong>URL</strong> : <a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a> </li>
<li><strong>TITLE</strong> : Widespread signatures of natural selection across human complex traits and functional genomic categories </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-21446-3 </li>
<li><strong>ABSTRACT</strong> : Understanding how natural selection has shaped genetic architecture of complex traits is of importance in medical and evolutionary genetics. Bayesian methods have been developed using individual-level GWAS data to estimate multiple genetic architecture parameters including selection signature. Here, we present a method (SBayesS) that only requires GWAS summary statistics. We analyse data for 155 complex traits (n = 27k-547k) and project the estimates onto those obtained from evolutionary simulations. We estimate that, on average across traits, about 1% of human genome sequence are mutational targets with a mean selection coefficient of ~0.001. Common diseases, on average, show a smaller number of mutational targets and have been under stronger selection, compared to other traits. SBayesS analyses incorporating functional annotations reveal that selection signatures vary across genomic regions, among which coding regions have the strongest selection signature and are enriched for both the number of associated variants and the magnitude of effect sizes. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Zeng J, Xue A, Jiang L, Lloyd-Jones LR, ...&amp;, Yang J. (2021) Widespread signatures of natural selection across human complex traits and functional genomic categories Nat. Commun., 12 (1) 1164. doi:10.1038/s41467-021-21446-3. PMID 33608517 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 1164 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33608517">33608517</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-kachuri">Review-Kachuri</h3>
<ul>
<li><strong>NAME</strong> : Review-Kachuri </li>
<li><strong>TITLE</strong> : Principles and methods for transferring polygenic risk scores across global populations </li>
<li><strong>DOI</strong> : 10.1038/s41576-023-00637-2 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs) summarize the genetic predisposition of a complex human trait or disease and may become a valuable tool for advancing precision medicine. However, PRSs that are developed in populations of predominantly European genetic ancestries can increase health disparities due to poor predictive performance in individuals of diverse and complex genetic ancestries. We describe genetic and modifiable risk factors that limit the transferability of PRSs across populations and review the strengths and weaknesses of existing PRS construction methods for diverse ancestries. Developing PRSs that benefit global populations in research and clinical settings provides an opportunity for innovation and is essential for health equity. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Kachuri L, Chatterjee N, Hirbo J, Schaid DJ, ...&amp;, Ge T. (2023) Principles and methods for transferring polygenic risk scores across global populations Nat. Rev. Genet., 25 (1) 8-25. doi:10.1038/s41576-023-00637-2. PMID 37620596 </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Genetics ; Nat. Rev. Genet. ; 2023 ; 25 ; 1 ; 8-25 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37620596">37620596</a> </li>
</ul>
<h3 id="review-peter">Review-Peter</h3>
<ul>
<li><strong>NAME</strong> : Review-Peter </li>
<li><strong>TITLE</strong> : Discovery and implications of polygenicity of common diseases </li>
<li><strong>DOI</strong> : 10.1126/science.abi8206 </li>
<li><strong>ABSTRACT</strong> : The sequencing of the human genome has allowed the study of the genetic architecture of common diseases: the number of genomic variants that contribute to risk of disease and their joint frequency and effect size distribution. Common diseases are polygenic, with many loci contributing to phenotype, and the cumulative burden of risk alleles determines individual risk in conjunction with environmental factors. Most risk loci occur in noncoding regions of the genome regulating cell- and context-specific gene expression. Although the effect sizes of most risk alleles are small, their cumulative effects in individuals, quantified as a polygenic (risk) score, can identify people at increased risk of disease, thereby facilitating prevention or early intervention. </li>
<li><strong>CITATION</strong> : Visscher PM, Yengo L, Cox NJ, Wray NR. (2021) Discovery and implications of polygenicity of common diseases Science, 373 (6562) 1468-1473. doi:10.1126/science.abi8206. PMID 34554790 </li>
<li><strong>JOURNAL_INFO</strong> : Science ; Science ; 2021 ; 373 ; 6562 ; 1468-1473 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34554790">34554790</a> </li>
</ul>
<h3 id="review-wang">Review-Wang</h3>
<ul>
<li><strong>NAME</strong> : Review-Wang </li>
<li><strong>TITLE</strong> : Challenges and opportunities for developing more generalizable polygenic risk scores </li>
<li><strong>DOI</strong> : 10.1146/annurev-biodatasci-111721-074830 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRS) estimate an individual's genetic likelihood of complex traits and diseases by aggregating information across multiple genetic variants identified from genome-wide association studies. PRS can predict a broad spectrum of diseases and have therefore been widely used in research settings. Some work has investigated their potential applications as biomarkers in preventative medicine, but significant work is still needed to definitively establish and communicate absolute risk to patients for genetic and modifiable risk factors across demographic groups. However, the biggest limitation of PRS currently is that they show poor generalizability across diverse ancestries and cohorts. Major efforts are underway through methodological development and data generation initiatives to improve their generalizability. This review aims to comprehensively discuss current progress on the development of PRS, the factors that affect their generalizability, and promising areas for improving their accuracy, portability, and implementation. </li>
<li><strong>CITATION</strong> : Wang Y, Tsuo K, Kanai M, Neale BM, ...&amp;, Martin AR. (2022) Challenges and opportunities for developing more generalizable polygenic risk scores Annu. Rev. Biomed. Data Sci., 5 (1) 293-320. doi:10.1146/annurev-biodatasci-111721-074830. PMID 35576555 </li>
<li><strong>JOURNAL_INFO</strong> : Annual review of biomedical data science ; Annu. Rev. Biomed. Data Sci. ; 2022 ; 5 ; 1 ; 293-320 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35576555">35576555</a> </li>
</ul>
<h2 id="single-trait">Single-trait</h2>
<h3 id="all-sum">ALL-Sum</h3>
<ul>
<li><strong>NAME</strong> : ALL-Sum </li>
<li><strong>SHORT NAME</strong> : ALL-Sum </li>
<li><strong>FULL NAME</strong> : Aggregated L0Learn using Summary-level data </li>
<li><strong>DESCRIPTION</strong> : ALL - Sum leverages  a L0L2 penalized regression and ensemble learning across tuning parameters to flexibly  model traits with diverse genetic architectures. </li>
<li><strong>URL</strong> : <a href="https://github.com/chen-tony/ALL-Sum/">https://github.com/chen-tony/ALL-Sum/</a> </li>
<li><strong>KEYWORDS</strong> : ensemble learning </li>
<li><strong>TITLE</strong> : Fast and scalable ensemble learning method for versatile polygenic risk prediction </li>
<li><strong>DOI</strong> : 10.1073/pnas.2403210121 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, but existing methods face several limitations, encompassing issues related to computational burden, predictive accuracy, and adaptability to a wide range of genetic architectures. To address these issues, we propose Aggregated L0Learn using Summary-level data (ALL-Sum), a fast and scalable ensemble learning method for computing PRS using summary statistics from genome-wide association studies (GWAS). ALL-Sum leverages a L0L2 penalized regression and ensemble learning across tuning parameters to flexibly model traits with diverse genetic architectures. In extensive large-scale simulations across a wide range of polygenicity and GWAS sample sizes, ALL-Sum consistently outperformed popular alternative methods in terms of prediction accuracy, runtime, and memory usage by 10%, 20-fold, and threefold, respectively, and demonstrated robustness to diverse genetic architectures. We validated the performance of ALL-Sum in real data analysis of 11 complex traits using GWAS summary statistics from nine data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen Biobank, with validation in the UK Biobank. Our results show that on average, ALL-Sum obtained PRS with 25% higher accuracy on average, with 15 times faster computation and half the memory than the current state-of-the-art methods, and had robust performance across a wide range of traits and diseases. Furthermore, our method demonstrates stable prediction when using linkage disequilibrium computed from different data sources. ALL-Sum is available as a user-friendly R software package with publicly available reference data for streamlined analysis. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Chen T, Zhang H, Mazumder R, Lin X. (2024) Fast and scalable ensemble learning method for versatile polygenic risk prediction Proc. Natl. Acad. Sci. U. S. A., 121 (33) e2403210121. doi:10.1073/pnas.2403210121. PMID 39110727 </li>
<li><strong>JOURNAL_INFO</strong> : Proceedings of the National Academy of Sciences of the United States of America ; Proc. Natl. Acad. Sci. U. S. A. ; 2024 ; 121 ; 33 ; e2403210121 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39110727">39110727</a> </li>
</ul>
<h3 id="calpred">CalPred</h3>
<ul>
<li><strong>NAME</strong> : CalPred </li>
<li><strong>SHORT NAME</strong> : CalPred </li>
<li><strong>FULL NAME</strong> : Calibrated prediction intervals </li>
<li><strong>DESCRIPTION</strong> : a statistical framework that jointly models the effects of all contexts on PGS accuracy with parameters learned in a calibration dataset </li>
<li><strong>URL</strong> : <a href="https://github.com/KangchengHou/calpred">https://github.com/KangchengHou/calpred</a> </li>
<li><strong>KEYWORDS</strong> : trait prediction intervals </li>
<li><strong>TITLE</strong> : Calibrated prediction intervals for polygenic scores across diverse contexts </li>
<li><strong>DOI</strong> : 10.1038/s41588-024-01792-w </li>
<li><strong>ABSTRACT</strong> : Polygenic scores (PGS) have emerged as the tool of choice for genomic prediction in a wide range of fields. We show that PGS performance varies broadly across contexts and biobanks. Contexts such as age, sex and income can impact PGS accuracy with similar magnitudes as genetic ancestry. Here we introduce an approach (CalPred) that models all contexts jointly to produce prediction intervals that vary across contexts to achieve calibration (include the trait with 90% probability), whereas existing methods are miscalibrated. In analyses of 72 traits across large and diverse biobanks (All of Us and UK Biobank), we find that prediction intervals required adjustment by up to 80% for quantitative traits. For disease traits, PGS-based predictions were miscalibrated across socioeconomic contexts such as annual household income levels, further highlighting the need of accounting for context information in PGS-based prediction across diverse populations. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Hou K, Xu Z, Ding Y, Mandla R, ...&amp;, Pasaniuc B. (2024) Calibrated prediction intervals for polygenic scores across diverse contexts Nat. Genet., 56 (7) 1386-1396. doi:10.1038/s41588-024-01792-w. PMID 38886587 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2024 ; 56 ; 7 ; 1386-1396 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38886587">38886587</a> </li>
</ul>
<h3 id="dbslmm">DBSLMM</h3>
<ul>
<li><strong>NAME</strong> : DBSLMM </li>
<li><strong>SHORT NAME</strong> : DBSLMM </li>
<li><strong>FULL NAME</strong> : Deterministic Bayesian Sparse Linear Mixed Model </li>
<li><strong>DESCRIPTION</strong> : There are two versions of DBSLMM: the tuning version and the deterministic version. The tuning version examines three different heritability choices and requires a validation data to tune the heritability hyper-parameter. The deterministic version uses one heritability estimate and directly fit the model in the training data without a separate validation data. Both versions requires a reference data to compute the SNP correlation matrix. In our experience, the tuning version may work more accurately than the deterministic version. </li>
<li><strong>URL</strong> : <a href="https://github.com/biostat0903/DBSLMM">https://github.com/biostat0903/DBSLMM</a> </li>
<li><strong>TITLE</strong> : Accurate and scalable construction of polygenic scores in large biobank data sets </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2020.03.013 </li>
<li><strong>ABSTRACT</strong> : Accurate construction of polygenic scores (PGS) can enable early diagnosis of diseases and facilitate the development of personalized medicine. Accurate PGS construction requires prediction models that are both adaptive to different genetic architectures and scalable to biobank scale datasets with millions of individuals and tens of millions of genetic variants. Here, we develop such a method called Deterministic Bayesian Sparse Linear Mixed Model (DBSLMM). DBSLMM relies on a flexible modeling assumption on the effect size distribution to achieve robust and accurate prediction performance across a range of genetic architectures. DBSLMM also relies on a simple deterministic search algorithm to yield an approximate analytic estimation solution using summary statistics only. The deterministic search algorithm, when paired with further algebraic innovations, results in substantial computational savings. With simulations, we show that DBSLMM achieves scalable and accurate prediction performance across a range of realistic genetic architectures. We then apply DBSLMM to analyze 25 traits in UK Biobank. For these traits, compared to existing approaches, DBSLMM achieves an average of 2.03%-101.09% accuracy gain in internal cross-validations. In external validations on two separate datasets, including one from BioBank Japan, DBSLMM achieves an average of 14.74%-522.74% accuracy gain. In these real data applications, DBSLMM is 1.03-28.11 times faster and uses only 7.4%-24.8% of physical memory as compared to other multiple regression-based PGS methods. Overall, DBSLMM represents an accurate and scalable method for constructing PGS in biobank scale datasets. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Yang S, Zhou X. (2020) Accurate and scalable construction of polygenic scores in large biobank data sets Am. J. Hum. Genet., 106 (5) 679-693. doi:10.1016/j.ajhg.2020.03.013. PMID 32330416 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2020 ; 106 ; 5 ; 679-693 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32330416">32330416</a> </li>
</ul>
<h3 id="gmrm">GMRM</h3>
<ul>
<li><strong>NAME</strong> : GMRM </li>
<li><strong>SHORT NAME</strong> : GMRM </li>
<li><strong>FULL NAME</strong> : Bayesian grouped mixture of regressions model </li>
<li><strong>DESCRIPTION</strong> : gmrm is hybrid-parallel software for a Bayesian grouped mixture of regressions model for genome-wide association studies (GWAS). It is written in C++ using extensive optimisations and code vectorisation. It relies on plink's .bed format. It can handle multiple traits simultaneously. </li>
<li><strong>URL</strong> : <a href="https://github.com/medical-genomics-group/gmrm">https://github.com/medical-genomics-group/gmrm</a> </li>
<li><strong>TITLE</strong> : Improving GWAS discovery and genomic prediction accuracy in biobank data </li>
<li><strong>DOI</strong> : 10.1073/pnas.2121279119 </li>
<li><strong>ABSTRACT</strong> : Genetically informed, deep-phenotyped biobanks are an important research resource and it is imperative that the most powerful, versatile, and efficient analysis approaches are used. Here, we apply our recently developed Bayesian grouped mixture of regressions model (GMRM) in the UK and Estonian Biobanks and obtain the highest genomic prediction accuracy reported to date across 21 heritable traits. When compared to other approaches, GMRM accuracy was greater than annotation prediction models run in the LDAK or LDPred-funct software by 15% (SE 7%) and 14% (SE 2%), respectively, and was 18% (SE 3%) greater than a baseline BayesR model without single-nucleotide polymorphism (SNP) markers grouped into minor allele frequency-linkage disequilibrium (MAF-LD) annotation categories. For height, the prediction accuracy R2 was 47% in a UK Biobank holdout sample, which was 76% of the estimated [Formula: see text]. We then extend our GMRM prediction model to provide mixed-linear model association (MLMA) SNP marker estimates for genome-wide association (GWAS) discovery, which increased the independent loci detected to 16,162 in unrelated UK Biobank individuals, compared to 10,550 from BoltLMM and 10,095 from Regenie, a 62 and 65% increase, respectively. The average [Formula: see text] value of the leading markers increased by 15.24 (SE 0.41) for every 1% increase in prediction accuracy gained over a baseline BayesR model across the traits. Thus, we show that modeling genetic associations accounting for MAF and LD differences among SNP markers, and incorporating prior knowledge of genomic function, is important for both genomic prediction and discovery in large-scale individual-level studies. </li>
<li><strong>CITATION</strong> : Orliac EJ, Trejo Banos D, Ojavee SE, Läll K, ...&amp;, Robinson MR. (2022) Improving GWAS discovery and genomic prediction accuracy in biobank data Proc. Natl. Acad. Sci. U. S. A., 119 (31) e2121279119. doi:10.1073/pnas.2121279119. PMID 35905320 </li>
<li><strong>JOURNAL_INFO</strong> : Proceedings of the National Academy of Sciences of the United States of America ; Proc. Natl. Acad. Sci. U. S. A. ; 2022 ; 119 ; 31 ; e2121279119 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35905320">35905320</a> </li>
</ul>
<h3 id="grpa-prs">GRPa-PRS</h3>
<ul>
<li><strong>NAME</strong> : GRPa-PRS </li>
<li><strong>SHORT NAME</strong> : GRPa-PRS </li>
<li><strong>FULL NAME</strong> : genetically-regulated pathways </li>
<li><strong>URL</strong> : <a href="https://github.com/davidroad/GRPa-PRS">https://github.com/davidroad/GRPa-PRS</a> </li>
<li><strong>TITLE</strong> : GRPa-PRS: A risk stratification method to identify genetically-regulated pathways in polygenic diseases </li>
<li><strong>DOI</strong> : 10.1101/2023.06.19.23291621 </li>
<li><strong>ABSTRACT</strong> : Background: Polygenic risk scores (PRS) are tools used to evaluate an individual's susceptibility to polygenic diseases based on their genetic profile. A considerable proportion of people carry a high genetic risk but evade the disease. On the other hand, some individuals with a low risk of eventually developing the disease. We hypothesized that unknown counterfactors might be involved in reversing the PRS prediction, which might provide new insights into the pathogenesis, prevention, and early intervention of diseases. Methods: We built a novel computational framework to identify genetically-regulated pathways (GRPas) using PRS-based stratification for each cohort. We curated two AD cohorts with genotyping data; the discovery (disc) and the replication (rep) datasets include 2722 and 2854 individuals, respectively. First, we calculated the optimized PRS model based on the three recent AD GWAS summary statistics for each cohort. Then, we stratified the individuals by their PRS and clinical diagnosis into six biologically meaningful PRS strata, such as AD cases with low/high risk and cognitively normal (CN) with low/high risk. Lastly, we imputed individual genetically-regulated expression (GReX) and identified differential GReX and GRPas between risk strata using gene-set enrichment and variational analyses in two models, with and without APOE effects. An orthogonality test was further conducted to verify those GRPas are independent of PRS risk. To verify the generalizability of other polygenic diseases, we further applied a default model of GRPa-PRS for schizophrenia (SCZ). Results: For each stratum, we conducted the same procedures in both the disc and rep datasets for comparison. In AD, we identified several well-known AD-related pathways, including amyloid-beta clearance, tau protein binding, and astrocyte response to oxidative stress. Additionally, we discovered resilience-related GRPs that are orthogonal to AD PRS, such as the calcium signaling pathway and divalent inorganic cation homeostasis. In SCZ, pathways related to mitochondrial function and muscle development were highlighted. Finally, our GRPa-PRS method identified more consistent differential pathways compared to another variant-based pathway PRS method. Conclusions: We developed a framework, GRPa-PRS, to systematically explore the differential GReX and GRPas among individuals stratified by their estimated PRS. The GReX-level comparison among those strata unveiled new insights into the pathways associated with disease risk and resilience. Our framework is extendable to other polygenic complex diseases. </li>
<li><strong>CITATION</strong> : Li X, Fernandes BS, Liu A, Chen J, ...&amp;, Dai Y. (2024) GRPa-PRS: A risk stratification method to identify genetically-regulated pathways in polygenic diseases medRxiv,  () 2023.06.19.23291621. doi:10.1101/2023.06.19.23291621. PMID 37425929 </li>
<li><strong>JOURNAL_INFO</strong> : medRxiv: the preprint server for health sciences ; medRxiv ; 2024 ; <NA> ; <NA> ; 2023.06.19.23291621 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37425929">37425929</a> </li>
</ul>
<h3 id="genoboost">GenoBoost</h3>
<ul>
<li><strong>NAME</strong> : GenoBoost </li>
<li><strong>SHORT NAME</strong> : GenoBoost </li>
<li><strong>FULL NAME</strong> : GenoBoost </li>
<li><strong>DESCRIPTION</strong> : GenoBoost is a polygenic score method to capture additive and non-additive genetic inheritance effects. </li>
<li><strong>URL</strong> : <a href="https://github.com/rickyota/genoboost">https://github.com/rickyota/genoboost</a> </li>
<li><strong>KEYWORDS</strong> : additive effects, non-additive effects, statistical boosting </li>
<li><strong>TITLE</strong> : A polygenic score method boosted by non-additive models </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-48654-x </li>
<li><strong>ABSTRACT</strong> : Dominance heritability in complex traits has received increasing recognition. However, most polygenic score (PGS) approaches do not incorporate non-additive effects. Here, we present GenoBoost, a flexible PGS modeling framework capable of considering both additive and non-additive effects, specifically focusing on genetic dominance. Building on statistical boosting theory, we derive provably optimal GenoBoost scores and provide its efficient implementation for analyzing large-scale cohorts. We benchmark it against seven commonly used PGS methods and demonstrate its competitive predictive performance. GenoBoost is ranked the best for four traits and second-best for three traits among twelve tested disease outcomes in UK Biobank. We reveal that GenoBoost improves prediction for autoimmune diseases by incorporating non-additive effects localized in the MHC locus and, more broadly, works best in less polygenic traits. We further demonstrate that GenoBoost can infer the mode of genetic inheritance without requiring prior knowledge. For example, GenoBoost finds non-zero genetic dominance effects for 602 of 900 selected genetic variants, resulting in 2.5% improvements in predicting psoriasis cases. Lastly, we show that GenoBoost can prioritize genetic loci with genetic dominance not previously reported in the GWAS catalog. Our results highlight the increased accuracy and biological insights from incorporating non-additive effects in PGS models. </li>
<li><strong>CITATION</strong> : Ohta R, Tanigawa Y, Suzuki Y, Kellis M, ...&amp;, Morishita S. (2024) A polygenic score method boosted by non-additive models Nat. Commun., 15 (1) 4433. doi:10.1038/s41467-024-48654-x. PMID 38811555 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2024 ; 15 ; 1 ; 4433 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38811555">38811555</a> </li>
</ul>
<h3 id="ldpred">LDpred</h3>
<ul>
<li><strong>NAME</strong> : LDpred </li>
<li><strong>SHORT NAME</strong> : LDpred </li>
<li><strong>FULL NAME</strong> : LDpred </li>
<li><strong>DESCRIPTION</strong> : LDpred is a Python based software package that adjusts GWAS summary statistics for the effects of linkage disequilibrium (LD). </li>
<li><strong>URL</strong> : <a href="https://github.com/bvilhjal/ldpred">https://github.com/bvilhjal/ldpred</a> </li>
<li><strong>KEYWORDS</strong> : Bayesian, Gaussian infinitesimal prior, python </li>
<li><strong>TITLE</strong> : Modeling linkage disequilibrium increases accuracy of polygenic risk scores </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2015.09.001 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores have shown great promise in predicting complex disease risk and will become more accurate as training sample sizes increase. The standard approach for calculating risk scores involves linkage disequilibrium (LD)-based marker pruning and applying a p value threshold to association statistics, but this discards information and can reduce predictive accuracy. We introduce LDpred, a method that infers the posterior mean effect size of each marker by using a prior on effect sizes and LD information from an external reference panel. Theory and simulations show that LDpred outperforms the approach of pruning followed by thresholding, particularly at large sample sizes. Accordingly, predicted R(2) increased from 20.1% to 25.3% in a large schizophrenia dataset and from 9.8% to 12.0% in a large multiple sclerosis dataset. A similar relative improvement in accuracy was observed for three additional large disease datasets and for non-European schizophrenia samples. The advantage of LDpred over existing methods will grow as sample sizes increase. </li>
<li><strong>CITATION</strong> : Vilhjálmsson BJ, Yang J, Finucane HK, Gusev A, ...&amp;, Price AL. (2015) Modeling linkage disequilibrium increases accuracy of polygenic risk scores Am. J. Hum. Genet., 97 (4) 576-592. doi:10.1016/j.ajhg.2015.09.001. PMID 26430803 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2015 ; 97 ; 4 ; 576-592 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/26430803">26430803</a> </li>
</ul>
<h3 id="ldpred-funct">LDpred-funct</h3>
<ul>
<li><strong>NAME</strong> : LDpred-funct </li>
<li><strong>SHORT NAME</strong> : LDpred-funct </li>
<li><strong>FULL NAME</strong> : LDpred-funct </li>
<li><strong>DESCRIPTION</strong> : LDpred-funct is a method for polygenic prediction that leverages trait-specific functional priors to increase prediction accuracy. </li>
<li><strong>URL</strong> : <a href="https://github.com/carlaml/LDpred-funct">https://github.com/carlaml/LDpred-funct</a> </li>
<li><strong>KEYWORDS</strong> : Bayesian, functional priors </li>
<li><strong>TITLE</strong> : Incorporating functional priors improves polygenic prediction accuracy in UK Biobank and 23andMe data sets </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-25171-9 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk prediction is a widely investigated topic because of its promising clinical applications. Genetic variants in functional regions of the genome are enriched for complex trait heritability. Here, we introduce a method for polygenic prediction, LDpred-funct, that leverages trait-specific functional priors to increase prediction accuracy. We fit priors using the recently developed baseline-LD model, including coding, conserved, regulatory, and LD-related annotations. We analytically estimate posterior mean causal effect sizes and then use cross-validation to regularize these estimates, improving prediction accuracy for sparse architectures. We applied LDpred-funct to predict 21 highly heritable traits in the UK Biobank (avg N = 373 K as training data). LDpred-funct attained a +4.6% relative improvement in average prediction accuracy (avg prediction R2 = 0.144; highest R2 = 0.413 for height) compared to SBayesR (the best method that does not incorporate functional information). For height, meta-analyzing training data from UK Biobank and 23andMe cohorts (N = 1107 K) increased prediction R2 to 0.431. Our results show that incorporating functional priors improves polygenic prediction accuracy, consistent with the functional architecture of complex traits. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Márquez-Luna C, Gazal S, Loh PR, Kim SS, ...&amp;, Price AL. (2021) Incorporating functional priors improves polygenic prediction accuracy in UK Biobank and 23andMe data sets Nat. Commun., 12 (1) 6052. doi:10.1038/s41467-021-25171-9. PMID 34663819 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 6052 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34663819">34663819</a> </li>
</ul>
<h3 id="ldpred2">LDpred2</h3>
<ul>
<li><strong>NAME</strong> : LDpred2 </li>
<li><strong>SHORT NAME</strong> : LDpred2 </li>
<li><strong>FULL NAME</strong> : LDpred2 </li>
<li><strong>DESCRIPTION</strong> : LDpred-2 is one of the dedicated PRS programs which is an R package that uses a Bayesian approach to polygenic risk scoring. </li>
<li><strong>URL</strong> : <a href="https://privefl.github.io/bigsnpr/articles/LDpred2.html">https://privefl.github.io/bigsnpr/articles/LDpred2.html</a> </li>
<li><strong>KEYWORDS</strong> : Bayesian, R, LDpred2-grid (LDpred2), LDpred2-auto, LDpred2-sparse </li>
<li><strong>TITLE</strong> : LDpred2: better, faster, stronger </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btaa1029 </li>
<li><strong>ABSTRACT</strong> : MOTIVATION: Polygenic scores have become a central tool in human genetics research. LDpred is a popular method for deriving polygenic scores based on summary statistics and a matrix of correlation between genetic variants. However, LDpred has limitations that may reduce its predictive performance. RESULTS: Here, we present LDpred2, a new version of LDpred that addresses these issues. We also provide two new options in LDpred2: a 'sparse' option that can learn effects that are exactly 0, and an 'auto' option that directly learns the two LDpred parameters from data. We benchmark predictive performance of LDpred2 against the previous version on simulated and real data, demonstrating substantial improvements in robustness and predictive accuracy compared to LDpred1. We then show that LDpred2 also outperforms other polygenic score methods recently developed, with a mean AUC over the 8 real traits analyzed here of 65.1%, compared to 63.8% for lassosum, 62.9% for PRS-CS and 61.5% for SBayesR. Note that LDpred2 provides more accurate polygenic scores when run genome-wide, instead of per chromosome. AVAILABILITY AND IMPLEMENTATION: LDpred2 is implemented in R package bigsnpr. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Privé F, Arbel J, Vilhjálmsson BJ. (2021) LDpred2: better, faster, stronger Bioinformatics, 36 (22-23) 5424-5431. doi:10.1093/bioinformatics/btaa1029. PMID 33326037 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2021 ; 36 ; 22-23 ; 5424-5431 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33326037">33326037</a> </li>
</ul>
<h3 id="ldpred2-auto">LDpred2-auto</h3>
<ul>
<li><strong>NAME</strong> : LDpred2-auto </li>
<li><strong>SHORT NAME</strong> : LDpred2-auto </li>
<li><strong>FULL NAME</strong> : LDpred2-auto </li>
<li><strong>DESCRIPTION</strong> : LDpred2 is a widely used Bayesian method for building polygenic scores (PGS). LDpred2-auto can infer the two parameters from the LDpred model, h^2 and p, so that it does not require an additional validation dataset to choose best-performing parameters. Here, we present a new version of LDpred2-auto, which adds a third parameter alpha to its model for modeling negative selection. Additional changes are also made to provide better sampling of these parameters. </li>
<li><strong>URL</strong> : <a href="https://privefl.github.io/bigsnpr/articles/LDpred2.html">https://privefl.github.io/bigsnpr/articles/LDpred2.html</a> </li>
<li><strong>KEYWORDS</strong> : Bayesian, new LDpred2-auto, α (relationship between MAF and beta) </li>
<li><strong>TITLE</strong> : Inferring disease architecture and predictive ability with LDpred2-auto </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2023.10.010 </li>
<li><strong>ABSTRACT</strong> : LDpred2 is a widely used Bayesian method for building polygenic scores (PGSs). LDpred2-auto can infer the two parameters from the LDpred model, the SNP heritability h2 and polygenicity p, so that it does not require an additional validation dataset to choose best-performing parameters. The main aim of this paper is to properly validate the use of LDpred2-auto for inferring multiple genetic parameters. Here, we present a new version of LDpred2-auto that adds an optional third parameter α to its model, for modeling negative selection. We then validate the inference of these three parameters (or two, when using the previous model). We also show that LDpred2-auto provides per-variant probabilities of being causal that are well calibrated and can therefore be used for fine-mapping purposes. We also introduce a formula to infer the out-of-sample predictive performance r2 of the resulting PGS directly from the Gibbs sampler of LDpred2-auto. Finally, we extend the set of HapMap3 variants recommended to use with LDpred2 with 37% more variants to improve the coverage of this set, and we show that this new set of variants captures 12% more heritability and provides 6% more predictive performance, on average, in UK Biobank analyses. </li>
<li><strong>COPYRIGHT</strong> : http://www.elsevier.com/open-access/userlicense/1.0/ </li>
<li><strong>CITATION</strong> : Privé F, Albiñana C, Arbel J, Pasaniuc B, ...&amp;, Vilhjálmsson BJ. (2023) Inferring disease architecture and predictive ability with LDpred2-auto Am. J. Hum. Genet., 110 (12) 2042-2055. doi:10.1016/j.ajhg.2023.10.010. PMID 37944514 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2023 ; 110 ; 12 ; 2042-2055 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37944514">37944514</a> </li>
</ul>
<h3 id="megaprs">MegaPRS</h3>
<ul>
<li><strong>NAME</strong> : MegaPRS </li>
<li><strong>SHORT NAME</strong> : MegaPRS </li>
<li><strong>FULL NAME</strong> : MegaPRS </li>
<li><strong>DESCRIPTION</strong> : individual level: big_spLinReg, LDAK-Ridge-Predict, LDAK-Bolt-Predict and LDAK-BayesR-Predict
sumstats: LDAK-Lasso-SS, LDAK-Ridge-SS, LDAK-Bolt-SS and LDAK-BayesR-SS </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>TITLE</strong> : Improved genetic prediction of complex traits from individual-level data or summary statistics </li>
<li><strong>DOI</strong> : 10.1038/s41467-021-24485-y </li>
<li><strong>ABSTRACT</strong> : Most existing tools for constructing genetic prediction models begin with the assumption that all genetic variants contribute equally towards the phenotype. However, this represents a suboptimal model for how heritability is distributed across the genome. Therefore, we develop prediction tools that allow the user to specify the heritability model. We compare individual-level data prediction tools using 14 UK Biobank phenotypes; our new tool LDAK-Bolt-Predict outperforms the existing tools Lasso, BLUP, Bolt-LMM and BayesR for all 14 phenotypes. We compare summary statistic prediction tools using 225 UK Biobank phenotypes; our new tool LDAK-BayesR-SS outperforms the existing tools lassosum, sBLUP, LDpred and SBayesR for 223 of the 225 phenotypes. When we improve the heritability model, the proportion of phenotypic variance explained increases by on average 14%, which is equivalent to increasing the sample size by a quarter. </li>
<li><strong>CITATION</strong> : Zhang Q, Privé F, Vilhjálmsson B, Speed D. (2021) Improved genetic prediction of complex traits from individual-level data or summary statistics Nat. Commun., 12 (1) 4192. doi:10.1038/s41467-021-24485-y. PMID 34234142 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2021 ; 12 ; 1 ; 4192 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34234142">34234142</a> </li>
</ul>
<h3 id="mixer_1">MiXeR</h3>
<ul>
<li><strong>NAME</strong> : MiXeR </li>
<li><strong>SHORT NAME</strong> : MiXeR </li>
<li><strong>FULL NAME</strong> : MiXeR（univariate） </li>
<li><strong>DESCRIPTION</strong> : Causal Mixture Model for GWAS summary statistics </li>
<li><strong>URL</strong> : <a href="https://github.com/precimed/mixer">https://github.com/precimed/mixer</a> </li>
<li><strong>TITLE</strong> : Beyond SNP heritability: Polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1008612 </li>
<li><strong>ABSTRACT</strong> : Estimating the polygenicity (proportion of causally associated single nucleotide polymorphisms (SNPs)) and discoverability (effect size variance) of causal SNPs for human traits is currently of considerable interest. SNP-heritability is proportional to the product of these quantities. We present a basic model, using detailed linkage disequilibrium structure from a reference panel of 11 million SNPs, to estimate these quantities from genome-wide association studies (GWAS) summary statistics. We apply the model to diverse phenotypes and validate the implementation with simulations. We find model polygenicities (as a fraction of the reference panel) ranging from ≃ 2 × 10-5 to ≃ 4 × 10-3, with discoverabilities similarly ranging over two orders of magnitude. A power analysis allows us to estimate the proportions of phenotypic variance explained additively by causal SNPs reaching genome-wide significance at current sample sizes, and map out sample sizes required to explain larger portions of additive SNP heritability. The model also allows for estimating residual inflation (or deflation from over-correcting of z-scores), and assessing compatibility of replication and discovery GWAS summary statistics. </li>
<li><strong>CITATION</strong> : Holland D, Frei O, Desikan R, Fan CC, ...&amp;, Dale AM. (2020) Beyond SNP heritability: Polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model PLoS Genet., 16 (5) e1008612. doi:10.1371/journal.pgen.1008612. PMID 32427991 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2020 ; 16 ; 5 ; e1008612 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32427991">32427991</a> </li>
</ul>
<h3 id="multiblup">MultiBLUP</h3>
<ul>
<li><strong>NAME</strong> : MultiBLUP </li>
<li><strong>SHORT NAME</strong> : MultiBLUP </li>
<li><strong>FULL NAME</strong> : MultiBLUP </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>TITLE</strong> : MultiBLUP: improved SNP-based prediction for complex traits </li>
<li><strong>DOI</strong> : 10.1101/gr.169375.113 </li>
<li><strong>ABSTRACT</strong> : BLUP (best linear unbiased prediction) is widely used to predict complex traits in plant and animal breeding, and increasingly in human genetics. The BLUP mathematical model, which consists of a single random effect term, was adequate when kinships were measured from pedigrees. However, when genome-wide SNPs are used to measure kinships, the BLUP model implicitly assumes that all SNPs have the same effect-size distribution, which is a severe and unnecessary limitation. We propose MultiBLUP, which extends the BLUP model to include multiple random effects, allowing greatly improved prediction when the random effects correspond to classes of SNPs with distinct effect-size variances. The SNP classes can be specified in advance, for example, based on SNP functional annotations, and we also provide an adaptive procedure for determining a suitable partition of SNPs. We apply MultiBLUP to genome-wide association data from the Wellcome Trust Case Control Consortium (seven diseases), and from much larger studies of celiac disease and inflammatory bowel disease, finding that it consistently provides better prediction than alternative methods. Moreover, MultiBLUP is computationally very efficient; for the largest data set, which includes 12,678 individuals and 1.5 M SNPs, the total analysis can be run on a single desktop PC in less than a day and can be parallelized to run even faster. Tools to perform MultiBLUP are freely available in our software LDAK. </li>
<li><strong>CITATION</strong> : Speed D, Balding DJ. (2014) MultiBLUP: improved SNP-based prediction for complex traits Genome Res., 24 (9) 1550-1557. doi:10.1101/gr.169375.113. PMID 24963154 </li>
<li><strong>JOURNAL_INFO</strong> : Genome research ; Genome Res. ; 2014 ; 24 ; 9 ; 1550-1557 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24963154">24963154</a> </li>
</ul>
<h3 id="prs-cs">PRS-CS</h3>
<ul>
<li><strong>NAME</strong> : PRS-CS </li>
<li><strong>SHORT NAME</strong> : PRS-CS </li>
<li><strong>FULL NAME</strong> : PRS-CS </li>
<li><strong>DESCRIPTION</strong> : PRS-CS is a Python based command line tool that infers posterior SNP effect sizes under continuous shrinkage (CS) priors using GWAS summary statistics and an external LD reference panel. </li>
<li><strong>URL</strong> : <a href="https://github.com/getian107/PRScs">https://github.com/getian107/PRScs</a> </li>
<li><strong>KEYWORDS</strong> : continuous shrinkage (CS) prior </li>
<li><strong>TITLE</strong> : Polygenic prediction via Bayesian regression and continuous shrinkage priors </li>
<li><strong>DOI</strong> : 10.1038/s41467-019-09718-5 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRS) have shown promise in predicting human complex traits and diseases. Here, we present PRS-CS, a polygenic prediction method that infers posterior effect sizes of single nucleotide polymorphisms (SNPs) using genome-wide association summary statistics and an external linkage disequilibrium (LD) reference panel. PRS-CS utilizes a high-dimensional Bayesian regression framework, and is distinct from previous work by placing a continuous shrinkage (CS) prior on SNP effect sizes, which is robust to varying genetic architectures, provides substantial computational advantages, and enables multivariate modeling of local LD patterns. Simulation studies using data from the UK Biobank show that PRS-CS outperforms existing methods across a wide range of genetic architectures, especially when the training sample size is large. We apply PRS-CS to predict six common complex diseases and six quantitative traits in the Partners HealthCare Biobank, and further demonstrate the improvement of PRS-CS in prediction accuracy over alternative methods. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Ge T, Chen CY, Ni Y, Feng YA, ...&amp;, Smoller JW. (2019) Polygenic prediction via Bayesian regression and continuous shrinkage priors Nat. Commun., 10 (1) 1776. doi:10.1038/s41467-019-09718-5. PMID 30992449 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2019 ; 10 ; 1 ; 1776 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30992449">30992449</a> </li>
</ul>
<h3 id="prsmix_aoi">PRSMix_AOI</h3>
<ul>
<li><strong>NAME</strong> : PRSMix_AOI </li>
<li><strong>SHORT NAME</strong> : PRSMix_AOI </li>
<li><strong>FULL NAME</strong> : add -one-in (AOI) </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.07.24.24310897 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Misra, A. et al. Instability of high polygenic risk classification and mitigation by integrative scoring. bioRxiv 2024.07.24.24310897 (2024) doi:10.1101/2024.07.24.24310897. </li>
</ul>
<h3 id="prs_to_abs">PRS_to_Abs</h3>
<ul>
<li><strong>NAME</strong> : PRS_to_Abs </li>
<li><strong>SHORT NAME</strong> : PRS_to_Abs </li>
<li><strong>FULL NAME</strong> : PRS_to_Abs </li>
<li><strong>DESCRIPTION</strong> : Converting Polygenic Score to Absolute Scale </li>
<li><strong>URL</strong> : <a href="https://opain.github.io/GenoPred/PRS_to_Abs_tool.html">https://opain.github.io/GenoPred/PRS_to_Abs_tool.html</a> </li>
<li><strong>TITLE</strong> : A tool for translating polygenic scores onto the absolute scale using summary statistics </li>
<li><strong>DOI</strong> : 10.1038/s41431-021-01028-z </li>
<li><strong>ABSTRACT</strong> : There is growing interest in the clinical application of polygenic scores as their predictive utility increases for a range of health-related phenotypes. However, providing polygenic score predictions on the absolute scale is an important step for their safe interpretation. We have developed a method to convert polygenic scores to the absolute scale for binary and normally distributed phenotypes. This method uses summary statistics, requiring only the area-under-the-ROC curve (AUC) or variance explained (R2) by the polygenic score, and the prevalence of binary phenotypes, or mean and standard deviation of normally distributed phenotypes. Polygenic scores are converted using normal distribution theory. We also evaluate methods for estimating polygenic score AUC/R2 from genome-wide association study (GWAS) summary statistics alone. We validate the absolute risk conversion and AUC/R2 estimation using data for eight binary and three continuous phenotypes in the UK Biobank sample. When the AUC/R2 of the polygenic score is known, the observed and estimated absolute values were highly concordant. Estimates of AUC/R2 from the lassosum pseudovalidation method were most similar to the observed AUC/R2 values, though estimated values deviated substantially from the observed for autoimmune disorders. This study enables accurate interpretation of polygenic scores using only summary statistics, providing a useful tool for educational and clinical purposes. Furthermore, we have created interactive webtools implementing the conversion to the absolute ( https://opain.github.io/GenoPred/PRS_to_Abs_tool.html ). Several further barriers must be addressed before clinical implementation of polygenic scores, such as ensuring target individuals are well represented by the GWAS sample. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Pain O, Gillett AC, Austin JC, Folkersen L, ...&amp;, Lewis CM. (2022) A tool for translating polygenic scores onto the absolute scale using summary statistics Eur. J. Hum. Genet., 30 (3) 339-348. doi:10.1038/s41431-021-01028-z. PMID 34983942 </li>
<li><strong>JOURNAL_INFO</strong> : European journal of human genetics: EJHG ; Eur. J. Hum. Genet. ; 2022 ; 30 ; 3 ; 339-348 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34983942">34983942</a> </li>
</ul>
<h3 id="prstuning">PRStuning</h3>
<ul>
<li><strong>NAME</strong> : PRStuning </li>
<li><strong>SHORT NAME</strong> : PRStuning </li>
<li><strong>FULL NAME</strong> : PRStuning </li>
<li><strong>DESCRIPTION</strong> : Estimate Testing AUC for Binary Phenotype Using GWAS Summary Statistics from the Training Data </li>
<li><strong>CITATION</strong> : Jiang, W., Chen, L., Girgenti, M. J., &amp; Zhao, H. (2023). Tuning Parameters for Polygenic Risk Score Methods Using GWAS Summary Statistics from Training Data. Research Square. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37398263">37398263</a> </li>
</ul>
<h3 id="sdpr">SDPR</h3>
<ul>
<li><strong>NAME</strong> : SDPR </li>
<li><strong>SHORT NAME</strong> : SDPR </li>
<li><strong>FULL NAME</strong> : SDPR </li>
<li><strong>DESCRIPTION</strong> : SDPR (Summary statistics based Dirichelt Process Regression) is a method to compute polygenic risk score (PRS) from summary statistics. It is the extension of Dirichlet Process Regression (DPR) to the use of summary statistics </li>
<li><strong>URL</strong> : <a href="https://github.com/eldronzhou/SDPR">https://github.com/eldronzhou/SDPR</a> </li>
<li><strong>TITLE</strong> : A fast and robust Bayesian nonparametric method for prediction of complex traits using summary statistics </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1009697 </li>
<li><strong>ABSTRACT</strong> : Genetic prediction of complex traits has great promise for disease prevention, monitoring, and treatment. The development of accurate risk prediction models is hindered by the wide diversity of genetic architecture across different traits, limited access to individual level data for training and parameter tuning, and the demand for computational resources. To overcome the limitations of the most existing methods that make explicit assumptions on the underlying genetic architecture and need a separate validation data set for parameter tuning, we develop a summary statistics-based nonparametric method that does not rely on validation datasets to tune parameters. In our implementation, we refine the commonly used likelihood assumption to deal with the discrepancy between summary statistics and external reference panel. We also leverage the block structure of the reference linkage disequilibrium matrix for implementation of a parallel algorithm. Through simulations and applications to twelve traits, we show that our method is adaptive to different genetic architectures, statistically robust, and computationally efficient. Our method is available at https://github.com/eldronzhou/SDPR. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Zhou G, Zhao H. (2021) A fast and robust Bayesian nonparametric method for prediction of complex traits using summary statistics PLoS Genet., 17 (7) e1009697. doi:10.1371/journal.pgen.1009697. PMID 34310601 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2021 ; 17 ; 7 ; e1009697 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34310601">34310601</a> </li>
</ul>
<h3 id="viprs">VIPRS</h3>
<ul>
<li><strong>NAME</strong> : VIPRS </li>
<li><strong>SHORT NAME</strong> : VIPRS </li>
<li><strong>FULL NAME</strong> : Variational inference of polygenic risk scores </li>
<li><strong>DESCRIPTION</strong> : viprs is a python package that implements scripts and utilities for running variational inference algorithms on genome-wide association study (GWAS) data for the purposes polygenic risk estimation. </li>
<li><strong>URL</strong> : <a href="https://github.com/shz9/viprs">https://github.com/shz9/viprs</a> </li>
<li><strong>KEYWORDS</strong> : Variational Inference (VI) </li>
<li><strong>TITLE</strong> : Fast and accurate Bayesian polygenic risk modeling with variational inference </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2023.03.009 </li>
<li><strong>ABSTRACT</strong> : The advent of large-scale genome-wide association studies (GWASs) has motivated the development of statistical methods for phenotype prediction with single-nucleotide polymorphism (SNP) array data. These polygenic risk score (PRS) methods use a multiple linear regression framework to infer joint effect sizes of all genetic variants on the trait. Among the subset of PRS methods that operate on GWAS summary statistics, sparse Bayesian methods have shown competitive predictive ability. However, most existing Bayesian approaches employ Markov chain Monte Carlo (MCMC) algorithms, which are computationally inefficient and do not scale favorably to higher dimensions, for posterior inference. Here, we introduce variational inference of polygenic risk scores (VIPRS), a Bayesian summary statistics-based PRS method that utilizes variational inference techniques to approximate the posterior distribution for the effect sizes. Our experiments with 36 simulation configurations and 12 real phenotypes from the UK Biobank dataset demonstrated that VIPRS is consistently competitive with the state-of-the-art in prediction accuracy while being more than twice as fast as popular MCMC-based approaches. This performance advantage is robust across a variety of genetic architectures, SNP heritabilities, and independent GWAS cohorts. In addition to its competitive accuracy on the "White British" samples, VIPRS showed improved transferability when applied to other ethnic groups, with up to 1.7-fold increase in R2 among individuals of Nigerian ancestry for low-density lipoprotein (LDL) cholesterol. To illustrate its scalability, we applied VIPRS to a dataset of 9.6 million genetic markers, which conferred further improvements in prediction accuracy for highly polygenic traits, such as height. </li>
<li><strong>CITATION</strong> : Zabad S, Gravel S, Li Y. (2023) Fast and accurate Bayesian polygenic risk modeling with variational inference Am. J. Hum. Genet., 110 (5) 741-761. doi:10.1016/j.ajhg.2023.03.009. PMID 37030289 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am. J. Hum. Genet. ; 2023 ; 110 ; 5 ; 741-761 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37030289">37030289</a> </li>
</ul>
<h3 id="lassosum">lassosum</h3>
<ul>
<li><strong>NAME</strong> : lassosum </li>
<li><strong>SHORT NAME</strong> : lassosum </li>
<li><strong>FULL NAME</strong> : lassosum </li>
<li><strong>DESCRIPTION</strong> : lassosum is a method for computing LASSO/Elastic Net estimates of a linear regression problem given summary statistics from GWAS and Genome-wide meta-analyses, accounting for Linkage Disequilibrium (LD), via a reference panel. </li>
<li><strong>URL</strong> : <a href="https://github.com/tshmak/lassosum">https://github.com/tshmak/lassosum</a> </li>
<li><strong>KEYWORDS</strong> : penalized regression </li>
<li><strong>TITLE</strong> : Polygenic scores via penalized regression on summary statistics </li>
<li><strong>DOI</strong> : 10.1002/gepi.22050 </li>
<li><strong>ABSTRACT</strong> : Polygenic scores (PGS) summarize the genetic contribution of a person's genotype to a disease or phenotype. They can be used to group participants into different risk categories for diseases, and are also used as covariates in epidemiological analyses. A number of possible ways of calculating PGS have been proposed, and recently there is much interest in methods that incorporate information available in published summary statistics. As there is no inherent information on linkage disequilibrium (LD) in summary statistics, a pertinent question is how we can use LD information available elsewhere to supplement such analyses. To answer this question, we propose a method for constructing PGS using summary statistics and a reference panel in a penalized regression framework, which we call lassosum. We also propose a general method for choosing the value of the tuning parameter in the absence of validation data. In our simulations, we showed that pseudovalidation often resulted in prediction accuracy that is comparable to using a dataset with validation phenotype and was clearly superior to the conservative option of setting the tuning parameter of lassosum to its lowest value. We also showed that lassosum achieved better prediction accuracy than simple clumping and P-value thresholding in almost all scenarios. It was also substantially faster and more accurate than the recently proposed LDpred. </li>
<li><strong>CITATION</strong> : Mak TSH, Porsch RM, Choi SW, Zhou X, ...&amp;, Sham PC. (2017) Polygenic scores via penalized regression on summary statistics Genet. Epidemiol., 41 (6) 469-480. doi:10.1002/gepi.22050. PMID 28480976 </li>
<li><strong>JOURNAL_INFO</strong> : Genetic epidemiology ; Genet. Epidemiol. ; 2017 ; 41 ; 6 ; 469-480 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28480976">28480976</a> </li>
</ul>
<h3 id="lassosum2">lassosum2</h3>
<ul>
<li><strong>NAME</strong> : lassosum2 </li>
<li><strong>SHORT NAME</strong> : lassosum2 </li>
<li><strong>FULL NAME</strong> : lassosum2 </li>
<li><strong>DESCRIPTION</strong> : lassosum2 is a re-implementation of the lassosum model that now uses the exact same input parameters as LDpred2 (corr and df_beta). It should be fast to run. It can be run next to LDpred2 and the best model can be chosen using the validation set. Note that parameter ‘s’ from lassosum has been replaced by a new parameter ‘delta’ in lassosum2, in order to better reflect that the lassosum model also uses L2-regularization (therefore, elastic-net regularization). </li>
<li><strong>URL</strong> : <a href="https://privefl.github.io/bigsnpr/articles/LDpred2.html#lassosum2-grid-of-models">https://privefl.github.io/bigsnpr/articles/LDpred2.html#lassosum2-grid-of-models</a> </li>
<li><strong>TITLE</strong> : Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores </li>
<li><strong>DOI</strong> : 10.1016/j.xhgg.2022.100136 </li>
<li><strong>ABSTRACT</strong> : Publicly available genome-wide association studies (GWAS) summary statistics exhibit uneven quality, which can impact the validity of follow-up analyses. First, we present an overview of possible misspecifications that come with GWAS summary statistics. Then, in both simulations and real-data analyses, we show that additional information such as imputation INFO scores, allele frequencies, and per-variant sample sizes in GWAS summary statistics can be used to detect possible issues and correct for misspecifications in the GWAS summary statistics. One important motivation for us is to improve the predictive performance of polygenic scores built from these summary statistics. Unfortunately, owing to the lack of reporting standards for GWAS summary statistics, this additional information is not systematically reported. We also show that using well-matched linkage disequilibrium (LD) references can improve model fit and translate into more accurate prediction. Finally, we discuss how to make polygenic score methods such as lassosum and LDpred2 more robust to these misspecifications to improve their predictive power. </li>
<li><strong>CITATION</strong> : Privé F, Arbel J, Aschard H, Vilhjálmsson BJ. (2022) Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores HGG Adv, 3 (4) 100136. doi:10.1016/j.xhgg.2022.100136. PMID 36105883 </li>
<li><strong>JOURNAL_INFO</strong> : HGG advances ; HGG Adv ; 2022 ; 3 ; 4 ; 100136 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36105883">36105883</a> </li>
</ul>
<h3 id="meta-prs">meta-PRS</h3>
<ul>
<li><strong>NAME</strong> : meta-PRS </li>
<li><strong>SHORT NAME</strong> : meta-PRS </li>
<li><strong>FULL NAME</strong> : linear combination of PRSs </li>
<li><strong>URL</strong> : <a href="https://github.com/ClaraAlbi/paper_MetaPRS">https://github.com/ClaraAlbi/paper_MetaPRS</a> </li>
<li><strong>TITLE</strong> : Leveraging both individual-level genetic data and GWAS summary statistics increases polygenic prediction </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2021.04.014 </li>
<li><strong>ABSTRACT</strong> : The accuracy of polygenic risk scores (PRSs) to predict complex diseases increases with the training sample size. PRSs are generally derived based on summary statistics from large meta-analyses of multiple genome-wide association studies (GWASs). However, it is now common for researchers to have access to large individual-level data as well, such as the UK Biobank data. To the best of our knowledge, it has not yet been explored how best to combine both types of data (summary statistics and individual-level data) to optimize polygenic prediction. The most widely used approach to combine data is the meta-analysis of GWAS summary statistics (meta-GWAS), but we show that it does not always provide the most accurate PRS. Through simulations and using 12 real case-control and quantitative traits from both iPSYCH and UK Biobank along with external GWAS summary statistics, we compare meta-GWAS with two alternative data-combining approaches, stacked clumping and thresholding (SCT) and meta-PRS. We find that, when large individual-level data are available, the linear combination of PRSs (meta-PRS) is both a simple alternative to meta-GWAS and often more accurate. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Albiñana C, Grove J, McGrath JJ, Agerbo E, ...&amp;, Vilhjálmsson BJ. (2021) Leveraging both individual-level genetic data and GWAS summary statistics increases polygenic prediction Am. J. Hum. Genet., 108 (6) 1001-1011. doi:10.1016/j.ajhg.2021.04.014. PMID 33964208 </li>
<li><strong>JOURNAL_INFO</strong> : The American Journal of Human Genetics ; Am. J. Hum. Genet. ; 2021 ; 108 ; 6 ; 1001-1011 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33964208">33964208</a> </li>
</ul>
<h3 id="rtprs-cs">rtPRS-CS</h3>
<ul>
<li><strong>NAME</strong> : rtPRS-CS </li>
<li><strong>SHORT NAME</strong> : rtPRS-CS </li>
<li><strong>FULL NAME</strong> : real-time PRS-CS </li>
<li><strong>DESCRIPTION</strong> : rtPRS-CS is a python-based command line tool that performs real-time online updating of polygenic risk score (PRS) weights in a target dataset, one sample at-a-time. Given the most recent set of SNP weights, for each new target sample with both phenotypic and genetic information, rtPRS-CS uses stochastic gradient descent to update the SNP weights, adjusting for the effect of a set of covariates. </li>
<li><strong>URL</strong> : <a href="https://github.com/getian107/rtPRS">https://github.com/getian107/rtPRS</a> </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.07.12.24310357 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Tubbs, J. D., Chen, Y., Duan, R., Huang, H. &amp; Ge, T. Real-time dynamic polygenic prediction for streaming data. bioRxiv 2024.07.12.24310357 (2024) doi:10.1101/2024.07.12.24310357. </li>
</ul>
<h2 id="standards">Standards</h2>
<h3 id="prs-rs">PRS-RS</h3>
<ul>
<li><strong>NAME</strong> : PRS-RS </li>
<li><strong>SHORT NAME</strong> : PRS-RS </li>
<li><strong>FULL NAME</strong> : Polygenic Risk Score Reporting Standards </li>
<li><strong>TITLE</strong> : Improving reporting standards for polygenic scores in risk prediction studies </li>
<li><strong>DOI</strong> : 10.1038/s41586-021-03243-6 </li>
<li><strong>ABSTRACT</strong> : Polygenic risk scores (PRSs), which often aggregate results from genome-wide association studies, can bridge the gap between initial discovery efforts and clinical applications for the estimation of disease risk using genetics. However, there is notable heterogeneity in the application and reporting of these risk scores, which hinders the translation of PRSs into clinical care. Here, in a collaboration between the Clinical Genome Resource (ClinGen) Complex Disease Working Group and the Polygenic Score (PGS) Catalog, we present the Polygenic Risk Score Reporting Standards (PRS-RS), in which we update the Genetic Risk Prediction Studies (GRIPS) Statement to reflect the present state of the field. Drawing on the input of experts in epidemiology, statistics, disease-specific applications, implementation and policy, this comprehensive reporting framework defines the minimal information that is needed to interpret and evaluate PRSs, especially with respect to downstream clinical applications. Items span detailed descriptions of study populations, statistical methods for the development and validation of PRSs and considerations for the potential limitations of these scores. In addition, we emphasize the need for data availability and transparency, and we encourage researchers to deposit and share PRSs through the PGS Catalog to facilitate reproducibility and comparative benchmarking. By providing these criteria in a structured format that builds on existing standards and ontologies, the use of this framework in publishing PRSs will facilitate translation into clinical care and progress towards defining best practice. </li>
<li><strong>CITATION</strong> : Wand H, Lambert SA, Tamburro C, Iacocca MA, ...&amp;, Wojcik GL. (2021) Improving reporting standards for polygenic scores in risk prediction studies Nature, 591 (7849) 211-219. doi:10.1038/s41586-021-03243-6. PMID 33692554 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2021 ; 591 ; 7849 ; 211-219 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33692554">33692554</a> </li>
</ul>
<h2 id="tutorial">Tutorial</h2>
<h3 id="tutorial-choi">Tutorial-Choi</h3>
<ul>
<li><strong>NAME</strong> : Tutorial-Choi </li>
<li><strong>SHORT NAME</strong> : PRS Tutorial </li>
<li><strong>FULL NAME</strong> : PRS Tutorial </li>
<li><strong>DESCRIPTION</strong> : This tutorial provides a step-by-step guide to performing basic polygenic risk score (PRS) analyses and accompanies our PRS Guide paper. The aim of this tutorial is to provide a simple introduction of PRS analyses to those new to PRS, while equipping existing users with a better understanding of the processes and implementation "underneath the hood" of popular PRS software. </li>
<li><strong>URL</strong> : <a href="https://choishingwan.github.io/PRS-Tutorial/">https://choishingwan.github.io/PRS-Tutorial/</a> </li>
<li><strong>TITLE</strong> : Tutorial: a guide to performing polygenic risk score analyses </li>
<li><strong>DOI</strong> : 10.1038/s41596-020-0353-1 </li>
<li><strong>ABSTRACT</strong> : A polygenic score (PGS) or polygenic risk score (PRS) is an estimate of an individual's genetic liability to a trait or disease, calculated according to their genotype profile and relevant genome-wide association study (GWAS) data. While present PRSs typically explain only a small fraction of trait variance, their correlation with the single largest contributor to phenotypic variation-genetic liability-has led to the routine application of PRSs across biomedical research. Among a range of applications, PRSs are exploited to assess shared etiology between phenotypes, to evaluate the clinical utility of genetic data for complex disease and as part of experimental studies in which, for example, experiments are performed that compare outcomes (e.g., gene expression and cellular response to treatment) between individuals with low and high PRS values. As GWAS sample sizes increase and PRSs become more powerful, PRSs are set to play a key role in research and stratified medicine. However, despite the importance and growing application of PRSs, there are limited guidelines for performing PRS analyses, which can lead to inconsistency between studies and misinterpretation of results. Here, we provide detailed guidelines for performing and interpreting PRS analyses. We outline standard quality control steps, discuss different methods for the calculation of PRSs, provide an introductory online tutorial, highlight common misconceptions relating to PRS results, offer recommendations for best practice and discuss future challenges. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Choi SW, Mak TS, O'Reilly PF. (2020) Tutorial: a guide to performing polygenic risk score analyses Nat. Protoc., 15 (9) 2759-2772. doi:10.1038/s41596-020-0353-1. PMID 32709988 </li>
<li><strong>JOURNAL_INFO</strong> : Nature protocols ; Nat. Protoc. ; 2020 ; 15 ; 9 ; 2759-2772 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32709988">32709988</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>