# Contents
- METAL
- MANTRA
- MR-MEGA
- GWAMA
- mvGWAMA
- MTAG
- Genomic-SEM

## METAL 
- SHORT NAME: METAL
- FULL NAME: METAL
- YEAR: 2012
- DESCRIPTION: METAL is a tool for meta-analysis genomewide association scans. METAL can combine either (a) test statistics and standard errors or (b) p-values across studies (taking sample size and direction of effect into account). METAL analysis is a convenient alternative to a direct analysis of merged data from multiple studies. It is especially appropriate when data from the individual studies cannot be analyzed together because of differences in ethnicity, phenotype distribution, gender or constraints in sharing of individual level data imposed. Meta-analysis results in little or no loss of efficiency compared to analysis of a combined dataset including data from all individual studies.
- URL : https://genome.sph.umich.edu/wiki/METAL_Documentation
- CITATION: Willer, C. J., Li, Y., & Abecasis, G. R. (2010). METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26(17), 2190-2191.

## MANTRA 
- SHORT NAME: MANTRA
- FULL NAME: Meta-ANalysis of Transethnic Association studies
- YEAR: 2011
- CITATION: Morris, A. P. (2011). Transethnic meta‐analysis of genomewide association studies. Genetic epidemiology, 35(8), 809-822.
- KEY WORDS: cross-population

## MR-MEGA 
- SHORT NAME: MR-MEGA
- FULL NAME: Meta-Regression of Multi-AncEstry Genetic Association
- YEAR: 2017
- DESCRIPTION: MR-MEGA (Meta-Regression of Multi-AncEstry Genetic Association) is a tool to detect and fine-map complex trait association signals via multi-ancestry meta-regression.  This approach uses genome-wide metrics of diversity between populations to derive axes of genetic variation via multi-dimensional scaling [Purcell 2007].  Allelic effects of a variant across GWAS, weighted by their corresponding standard errors, can then be modelled in a linear regression framework, including the axes of genetic variation as covariates.  The flexibility of this model enables partitioning of the heterogeneity into components due to ancestry and residual variation, which would be expected to improve fine-mapping resolution.
- URL : https://genomics.ut.ee/en/tools
- CITATION: Mägi, R., Horikoshi, M., Sofer, T., Mahajan, A., Kitajima, H., Franceschini, N., ... & Morris, A. P. (2017). Trans-ethnic meta-regression of genome-wide association studies accounting for ancestry increases power for discovery and improves fine-mapping resolution. Human molecular genetics, 26(18), 3639-3650.
- KEY WORDS: cross-population, Meta-Regression

## GWAMA 
- SHORT NAME:  GWAMA 
- FULL NAME:  Genome-Wide Association Meta-Analysis
- YEAR: 2010
- DESCRIPTION: Software tool for meta analysis of whole genome association data
- URL : https://genomics.ut.ee/en/tools
- CITATION: Mägi, R., & Morris, A. P. (2010). GWAMA: software for genome-wide association meta-analysis. BMC bioinformatics, 11(1), 1-6.


## mvGWAMA 
- SHORT NAME:  mvGWAMA 
- FULL NAME:  Multivariate Genome-Wide Association Meta-Analysis
- YEAR: 2019
- DESCRIPTION: mvGWAMA is a python script to perform a GWAS meta-analysis when there are sample overlap.
- URL : https://github.com/Kyoko-wtnb/mvGWAMA
- CITATION: Jansen, Iris E., et al. "Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer’s disease risk." Nature genetics 51.3 (2019): 404-413.

## MTAG
- SHORT NAME: MTAG
- FULL NAME: Multi-Trait Analysis of GWAS
- YEAR: 2018
- DESCRIPTION: mtag is a Python-based command line tool for jointly analyzing multiple sets of GWAS summary statistics as described by Turley et. al. (2018). It can also be used as a tool to meta-analyze GWAS results.
- URL : https://github.com/JonJala/mtag
- CITATION: Turley, P., Walters, R. K., Maghzian, O., Okbay, A., Lee, J. J., Fontana, M. A., ... & Benjamin, D. J. (2018). Multi-trait analysis of genome-wide association summary statistics using MTAG. Nature genetics, 50(2), 229-237.
- KEY WORDS: Multi-trait

## Genomic-SEM
- SHORT NAME: Genomic-SEM
- FULL NAME: genomic structural equation modelling 
- YEAR: 2019
- DESCRIPTION: R-package which allows the user to fit structural equation models based on the summary statistics obtained from genome wide association studies (GWAS). 
- URL : https://github.com/GenomicSEM/GenomicSEM
- CITATION: Grotzinger, A. D., Rhemtulla, M., de Vlaming, R., Ritchie, S. J., Mallard, T. T., Hill, W. D., ... & Tucker-Drob, E. M. (2019). Genomic structural equation modelling provides insights into the multivariate genetic architecture of complex traits. Nature human behaviour, 3(5), 513-525.
- KEY WORDS: SEM

##Template 
- SHORT NAME: 
- FULL NAME: 
- YEAR: 
- DESCRIPTION: 
- URL : 
- CITATION: 
- KEY WORDS:
