
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Gene_set_pathway_analysis_README/">
      
      
        <link rel="next" href="../Tools_HLA_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.2">
    
    
      
        <title>GxE_interactions - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              GxE_interactions
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gplemma" class="md-nav__link">
    <span class="md-ellipsis">
      GPLEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lemma" class="md-nav__link">
    <span class="md-ellipsis">
      LEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#structlmm" class="md-nav__link">
    <span class="md-ellipsis">
      StructLMM
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review_1" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_2" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_3" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_4" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_5" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
    <nav class="md-nav" aria-label="MISC">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gplemma" class="md-nav__link">
    <span class="md-ellipsis">
      GPLEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lemma" class="md-nav__link">
    <span class="md-ellipsis">
      LEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#structlmm" class="md-nav__link">
    <span class="md-ellipsis">
      StructLMM
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review_1" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_2" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_3" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_4" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#review_5" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>GxE_interactions</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: right;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#gplemma">GPLEMMA</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Kerin M, Marchini J. (2021) A non-linear regression method for estimation of gene-environment heritability. Bioinformatics, 36 (24) 5632-5639. doi:10.1093/bioinformatics/btaa1079. PMID 33367483</td>
<td style="text-align: right;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lemma">LEMMA</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Kerin M, Marchini J. (2020) Inferring Gene-by-Environment Interactions with a Bayesian Whole-Genome Regression Model. Am J Hum Genet, 107 (4) 698-713. doi:10.1016/j.ajhg.2020.08.009. PMID 32888427</td>
<td style="text-align: right;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#structlmm">StructLMM</a></td>
<td style="text-align: left;">MISC</td>
<td style="text-align: left;">Moore R, ...&amp;, Stegle O. (2019) A linear mixed-model approach to study multivariate gene-environment interactions. Nat Genet, 51 (1) 180-186. doi:10.1038/s41588-018-0271-0. PMID 30478441</td>
<td style="text-align: right;">2019</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review">Review</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Ottman, R. (1996). Gene–environment interaction: definitions and study design. Preventive medicine, 25(6), 764-770.<br><br></td>
<td style="text-align: right;">1996</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review">Review</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Hunter, D. J. (2005). Gene–environment interactions in human diseases. Nature reviews genetics, 6(4), 287-298.</td>
<td style="text-align: right;">2005</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review">Review</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Manuck SB, McCaffery JM. (2014) Gene-environment interaction. Annu Rev Psychol, 65  41-70. doi:10.1146/annurev-psych-010213-115100. PMID 24405358</td>
<td style="text-align: right;">2014</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review">Review</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Zhang X, Belsky J. (2022) Three phases of Gene × Environment interaction research: Theoretical assumptions underlying gene selection. Dev Psychopathol, 34 (1) 295-306. doi:10.1017/S0954579420000966. PMID 32880244</td>
<td style="text-align: right;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review">Review</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Boyce WT, ...&amp;, Robinson GE. (2020) Genes and environments, development and time. Proc Natl Acad Sci U S A, 117 (38) 23235-23241. doi:10.1073/pnas.2016710117. PMID 32967067</td>
<td style="text-align: right;">2020</td>
</tr>
</tbody>
</table>
<h2 id="misc">MISC</h2>
<h3 id="gplemma">GPLEMMA</h3>
<ul>
<li><strong>NAME</strong> : GPLEMMA </li>
<li><strong>SHORT NAME</strong> : GPLEMMA </li>
<li><strong>FULL NAME</strong> : Gaussian Prior Linear Environment Mixed Model Analysis </li>
<li><strong>DESCRIPTION</strong> : GPLEMMA (Gaussian Prior Linear Environment Mixed Model Analysis) non-linear randomized Haseman-Elston regression method for flexible modeling of gene-environment interactions in large datasets such as the UK Biobank. </li>
<li><strong>URL</strong> : <a href="https://github.com/mkerin/LEMMA">https://github.com/mkerin/LEMMA</a> </li>
<li><strong>TITLE</strong> : A non-linear regression method for estimation of gene-environment heritability. </li>
<li><strong>CITATION</strong> : Kerin M, Marchini J. (2021) A non-linear regression method for estimation of gene-environment heritability. Bioinformatics, 36 (24) 5632-5639. doi:10.1093/bioinformatics/btaa1079. PMID 33367483 </li>
<li><strong>ABSTRACT</strong> : Gene-environment (GxE) interactions are one of the least studied aspects of the genetic architecture of human traits and diseases. The environment of an individual is inherently high dimensional, evolves through time and can be expensive and time consuming to measure. The UK Biobank study, with all 500 000 participants having undergone an extensive baseline questionnaire, represents a unique opportunity to assess GxE heritability for many traits and diseases in a well powered setting.|We have developed a randomized Haseman-Elston non-linear regression method applicable when many environmental variables have been measured on each individual. The method (GPLEMMA) simultaneously estimates a linear environmental score (ES) and its GxE heritability. We compare the method via simulation to a whole-genome regression approach (LEMMA) for estimating GxE heritability. We show that GPLEMMA is more computationally efficient than LEMMA on large datasets, and produces results highly correlated with those from LEMMA when applied to simulated data and real data from the UK Biobank.|Software implementing the GPLEMMA method is available from https://jmarchini.org/gplemma/.|Supplementary data are available at Bioinformatics online. </li>
<li><strong>COPYRIGHT</strong> : © The Author(s) 2020. Published by Oxford University Press. </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btaa1079 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics (Oxford, England) ; Bioinformatics ; 2021 ; 36 ; 24 ; 5632-5639 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33367483">33367483</a> </li>
</ul>
<h3 id="lemma">LEMMA</h3>
<ul>
<li><strong>NAME</strong> : LEMMA </li>
<li><strong>SHORT NAME</strong> : LEMMA </li>
<li><strong>FULL NAME</strong> : Linear Environment Mixed Model Analysis </li>
<li><strong>DESCRIPTION</strong> : LEMMA (Linear Environment Mixed Model Analysis) is a whole genome wide regression method for flexible modeling of gene-environment interactions in large datasets such as the UK Biobank. </li>
<li><strong>URL</strong> : <a href="https://github.com/mkerin/LEMMA">https://github.com/mkerin/LEMMA</a> </li>
<li><strong>TITLE</strong> : Inferring Gene-by-Environment Interactions with a Bayesian Whole-Genome Regression Model. </li>
<li><strong>CITATION</strong> : Kerin M, Marchini J. (2020) Inferring Gene-by-Environment Interactions with a Bayesian Whole-Genome Regression Model. Am J Hum Genet, 107 (4) 698-713. doi:10.1016/j.ajhg.2020.08.009. PMID 32888427 </li>
<li><strong>MESH_MAJOR</strong> : Gene-Environment Interaction, Genome, Human, Models, Statistical, Quantitative Trait Loci, Quantitative Trait, Heritable </li>
<li><strong>MESH_MINOR</strong> : Bayes Theorem, Blood Pressure, Body Mass Index, Datasets as Topic, Genetic Markers, Humans, United Kingdom </li>
<li><strong>ABSTRACT</strong> : The contribution of gene-by-environment (GxE) interactions for many human traits and diseases is poorly characterized. We propose a Bayesian whole-genome regression model for joint modeling of main genetic effects and GxE interactions in large-scale datasets, such as the UK Biobank, where many environmental variables have been measured. The method is called LEMMA (Linear Environment Mixed Model Analysis) and estimates a linear combination of environmental variables, called an environmental score (ES), that interacts with genetic markers throughout the genome. The ES provides a readily interpretable way to examine the combined effect of many environmental variables. The ES can be used both to estimate the proportion of phenotypic variance attributable to GxE effects and to test for GxE effects at genetic variants across the genome. GxE effects can induce heteroskedasticity in quantitative traits, and LEMMA accounts for this by using robust standard error estimates when testing for GxE effects. When applied to body mass index, systolic blood pressure, diastolic blood pressure, and pulse pressure in the UK Biobank, we estimate that 9.3%, 3.9%, 1.6%, and 12.5%, respectively, of phenotypic variance is explained by GxE interactions and that low-frequency variants explain most of this variance. We also identify three loci that interact with the estimated environmental scores (-log<sub>10</sub>p&gt;7.3). </li>
<li><strong>COPYRIGHT</strong> : Copyright © 2020 The Authors. Published by Elsevier Inc. All rights reserved. </li>
<li><strong>DOI</strong> : 10.1016/j.ajhg.2020.08.009 </li>
<li><strong>JOURNAL_INFO</strong> : American journal of human genetics ; Am J Hum Genet ; 2020 ; 107 ; 4 ; 698-713 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32888427">32888427</a> </li>
</ul>
<h3 id="structlmm">StructLMM</h3>
<ul>
<li><strong>NAME</strong> : StructLMM </li>
<li><strong>SHORT NAME</strong> : StructLMM </li>
<li><strong>FULL NAME</strong> : Structured Linear Mixed Model </li>
<li><strong>DESCRIPTION</strong> : Structured Linear Mixed Model (StructLMM) is a computationally efficient method to test for and characterize loci that interact with multiple environments </li>
<li><strong>URL</strong> : <a href="https://github.com/limix/struct-lmm
https://github.com/limix/limix">https://github.com/limix/struct-lmm
https://github.com/limix/limix</a> </li>
<li><strong>TITLE</strong> : A linear mixed-model approach to study multivariate gene-environment interactions. </li>
<li><strong>CITATION</strong> : Moore R, ...&amp;, Stegle O. (2019) A linear mixed-model approach to study multivariate gene-environment interactions. Nat Genet, 51 (1) 180-186. doi:10.1038/s41588-018-0271-0. PMID 30478441 </li>
<li><strong>MESH_MAJOR</strong> : Gene-Environment Interaction </li>
<li><strong>MESH_MINOR</strong> : Algorithms, Computer Simulation, Environment, Genotype, Humans, Linear Models, Models, Genetic, Quantitative Trait Loci </li>
<li><strong>ABSTRACT</strong> : Different exposures, including diet, physical activity, or external conditions can contribute to genotype-environment interactions (G×E). Although high-dimensional environmental data are increasingly available and multiple exposures have been implicated with G×E at the same loci, multi-environment tests for G×E are not established. Here, we propose the structured linear mixed model (StructLMM), a computationally efficient method to identify and characterize loci that interact with one or more environments. After validating our model using simulations, we applied StructLMM to body mass index in the UK Biobank, where our model yields previously known and novel G×E signals. Finally, in an application to a large blood eQTL dataset, we demonstrate that StructLMM can be used to study interactions with hundreds of environmental variables. </li>
<li><strong>DOI</strong> : 10.1038/s41588-018-0271-0 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat Genet ; 2019 ; 51 ; 1 ; 180-186 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30478441">30478441</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review_1">Review</h3>
<ul>
<li><strong>NAME</strong> : Review </li>
<li><strong>TITLE</strong> : Gene-environment interaction: definitions and study designs. </li>
<li><strong>CITATION</strong> : Ottman, R. (1996). Gene–environment interaction: definitions and study design. Preventive medicine, 25(6), 764-770.<br><br> </li>
<li><strong>MESH_MAJOR</strong> : Causality, Environment, Epidemiologic Methods, Genetic Predisposition to Disease, Odds Ratio </li>
<li><strong>MESH_MINOR</strong> : Disease Susceptibility, Genetics, Medical, Humans, Models, Biological, Models, Genetic </li>
<li><strong>ABSTRACT</strong> : Study of gene-environment interaction is important for improving accuracy and precision in the assessment of both genetic and environmental influences. This overview presents a simple definition of gene-environment interaction and suggests study designs for detecting it. Gene-environment interaction is defined as "a different effect of an environmental exposure on disease risk in persons with different genotypes," or, alternatively, "a different effect of a genotype on disease risk in persons with different environmental exposures." Under this strictly statistical definition, the presence or absence of interaction depends upon the scale of measurement (additive or multiplicative). The decision of which scale is appropriate will be governed by many factors, including the main objective of an investigation (discovery of etiology, public health prediction, etc.) and the hypothesized pathophysiologic model. Five biologically plausible models are described for the relations between genotypes and environmental exposures, in terms of their effects on disease risk. Each of these models leads to a different set of predictions about disease risk in individuals classified by presence or absence of a high-risk genotype and environmental exposure. Classification according to the exposure is relatively easy, using conventional epidemiologic methods. Classification according to the high-risk genotype is more difficult, but several alternative strategies are suggested. </li>
<li><strong>JOURNAL_INFO</strong> : Preventive medicine ; Prev Med ; 1996 ; 25 ; 6 ; 764-70 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/8936580">8936580</a> </li>
</ul>
<h3 id="review_2">Review</h3>
<ul>
<li><strong>NAME</strong> : Review </li>
<li><strong>TITLE</strong> : Gene-environment interactions in human diseases. </li>
<li><strong>CITATION</strong> : Hunter, D. J. (2005). Gene–environment interactions in human diseases. Nature reviews genetics, 6(4), 287-298. </li>
<li><strong>MESH_MAJOR</strong> : Environment, Genetic Predisposition to Disease </li>
<li><strong>MESH_MINOR</strong> : Epidemiologic Studies, Genotype, Humans, Models, Genetic, Prospective Studies, Reproducibility of Results, Risk Factors, Sample Size </li>
<li><strong>ABSTRACT</strong> : Studies of gene-environment interactions aim to describe how genetic and environmental factors jointly influence the risk of developing a human disease. Gene-environment interactions can be described by using several models, which take into account the various ways in which genetic effects can be modified by environmental exposures, the number of levels of these exposures and the model on which the genetic effects are based. Choice of study design, sample size and genotyping technology influence the analysis and interpretation of observed gene-environment interactions. Current systems for reporting epidemiological studies make it difficult to assess whether the observed interactions are reproducible, so suggestions are made for improvements in this area. </li>
<li><strong>JOURNAL_INFO</strong> : Nature reviews. Genetics ; Nat Rev Genet ; 2005 ; 6 ; 4 ; 287-98 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/15803198">15803198</a> </li>
</ul>
<h3 id="review_3">Review</h3>
<ul>
<li><strong>NAME</strong> : Review </li>
<li><strong>TITLE</strong> : Gene-environment interaction. </li>
<li><strong>CITATION</strong> : Manuck SB, McCaffery JM. (2014) Gene-environment interaction. Annu Rev Psychol, 65  41-70. doi:10.1146/annurev-psych-010213-115100. PMID 24405358 </li>
<li><strong>MESH_MAJOR</strong> : Gene-Environment Interaction, Phenotype </li>
<li><strong>MESH_MINOR</strong> : Environment, Genetic Predisposition to Disease, Genetic Variation, Humans, Mental Disorders, Models, Psychological </li>
<li><strong>ABSTRACT</strong> : With the advent of increasingly accessible technologies for typing genetic variation, studies of gene-environment (G×E) interactions have proliferated in psychological research. Among the aims of such studies are testing developmental hypotheses and models of the etiology of behavioral disorders, defining boundaries of genetic and environmental influences, and identifying individuals most susceptible to risk exposures or most amenable to preventive and therapeutic interventions. This research also coincides with the emergence of unanticipated difficulties in detecting genetic variants of direct association with behavioral traits and disorders, which may be obscured if genetic effects are expressed only in predisposing environments. In this essay we consider these and other rationales for positing G×E interactions, review conceptual models meant to inform G×E interpretations from a psychological perspective, discuss points of common critique to which G×E research is vulnerable, and address the role of the environment in G×E interactions. </li>
<li><strong>DOI</strong> : 10.1146/annurev-psych-010213-115100 </li>
<li><strong>JOURNAL_INFO</strong> : Annual review of psychology ; Annu Rev Psychol ; 2014 ; 65 ; <NA> ; 41-70 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/24405358">24405358</a> </li>
</ul>
<h3 id="review_4">Review</h3>
<ul>
<li><strong>NAME</strong> : Review </li>
<li><strong>TITLE</strong> : Three phases of Gene × Environment interaction research: Theoretical assumptions underlying gene selection. </li>
<li><strong>CITATION</strong> : Zhang X, Belsky J. (2022) Three phases of Gene × Environment interaction research: Theoretical assumptions underlying gene selection. Dev Psychopathol, 34 (1) 295-306. doi:10.1017/S0954579420000966. PMID 32880244 </li>
<li><strong>MESH_MAJOR</strong> : Gene-Environment Interaction, Multifactorial Inheritance </li>
<li><strong>MESH_MINOR</strong> : Disease Susceptibility, Humans </li>
<li><strong>ABSTRACT</strong> : Some Gene × Environment interaction (G×E) research has focused upon single candidate genes, whereas other related work has targeted multiple genes (e.g., polygenic scores). Each approach has informed efforts to identify individuals who are either especially vulnerable to the negative effects of contextual adversity (diathesis stress) or especially susceptible to both positive and negative contextual conditions (differential susceptibility). A critical step in all such molecular G×E research is the selection of genetic variants thought to moderate environmental influences, a subject that has not received a great deal of attention in critiques of G×E research (beyond the observation of small effects of individual genes). Here we conceptually distinguish three phases of G×E work based on the selection of genes presumed to moderate environmental effects and the theoretical basis of such decisions: (a) single candidate genes, (b) composited (multiple) candidate genes, and (c) GWAS-derived polygenic scores. This illustrative, not exhaustive, review makes it clear that implicit or explicit theoretical assumptions inform gene selection in ways that have not been clearly articulated or fully appreciated. </li>
<li><strong>DOI</strong> : 10.1017/S0954579420000966 </li>
<li><strong>JOURNAL_INFO</strong> : Development and psychopathology ; Dev Psychopathol ; 2022 ; 34 ; 1 ; 295-306 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32880244">32880244</a> </li>
</ul>
<h3 id="review_5">Review</h3>
<ul>
<li><strong>NAME</strong> : Review </li>
<li><strong>TITLE</strong> : Genes and environments, development and time. </li>
<li><strong>CITATION</strong> : Boyce WT, ...&amp;, Robinson GE. (2020) Genes and environments, development and time. Proc Natl Acad Sci U S A, 117 (38) 23235-23241. doi:10.1073/pnas.2016710117. PMID 32967067 </li>
<li><strong>MESH_MAJOR</strong> : Gene-Environment Interaction </li>
<li><strong>MESH_MINOR</strong> : Animals, Biological Evolution, Epigenesis, Genetic, Gene Expression Regulation, Humans, Time </li>
<li><strong>ABSTRACT</strong> : A now substantial body of science implicates a dynamic interplay between genetic and environmental variation in the development of individual differences in behavior and health. Such outcomes are affected by molecular, often epigenetic, processes involving gene-environment (G-E) interplay that can influence gene expression. Early environments with exposures to poverty, chronic adversities, and acutely stressful events have been linked to maladaptive development and compromised health and behavior. Genetic differences can impart either enhanced or blunted susceptibility to the effects of such pathogenic environments. However, largely missing from present discourse regarding G-E interplay is the role of time, a "third factor" guiding the emergence of complex developmental endpoints across different scales of time. Trajectories of development increasingly appear best accounted for by a complex, dynamic interchange among the highly linked elements of genes, contexts, and time at multiple scales, including neurobiological (minutes to milliseconds), genomic (hours to minutes), developmental (years and months), and evolutionary (centuries and millennia) time. This special issue of PNAS thus explores time and timing among G-E transactions: The importance of timing and timescales in plasticity and critical periods of brain development; epigenetics and the molecular underpinnings of biologically embedded experience; the encoding of experience across time and biological levels of organization; and gene-regulatory networks in behavior and development and their linkages to neuronal networks. Taken together, the collection of papers offers perspectives on how G-E interplay operates contingently within and against a backdrop of time and timescales. </li>
<li><strong>DOI</strong> : 10.1073/pnas.2016710117 </li>
<li><strong>JOURNAL_INFO</strong> : Proceedings of the National Academy of Sciences of the United States of America ; Proc Natl Acad Sci U S A ; 2020 ; 117 ; 38 ; 23235-23241 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32967067">32967067</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.d7c377c4.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>