
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Proteomics_Proteomics_README/">
      
      
        <link rel="next" href="../Tools_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.13">
    
    
      
        <title>Imaging - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Imaging
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  
  Population_Genetics

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Phenotype_Phenotype_README/" class="md-tabs__link">
          
  
  
  Reference

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
      
        
        
      
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sumstats
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Imaging
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Imaging
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deepflowgomes-b-38082205" class="md-nav__link">
    <span class="md-ellipsis">
      DeepFlow/Gomes B-38082205
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elliott-lt-30305740" class="md-nav__link">
    <span class="md-ellipsis">
      Elliott LT-30305740
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fu-j-38811844" class="md-nav__link">
    <span class="md-ellipsis">
      Fu J-38811844
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#haas-me-34957434" class="md-nav__link">
    <span class="md-ellipsis">
      Haas ME-34957434
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ikram-m-28627999" class="md-nav__link">
    <span class="md-ellipsis">
      Ikram M.-28627999
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#karkar-s-33664500" class="md-nav__link">
    <span class="md-ellipsis">
      Karkar S-33664500
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#khurshid-s-36944631" class="md-nav__link">
    <span class="md-ellipsis">
      Khurshid S-36944631
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-f-23028347" class="md-nav__link">
    <span class="md-ellipsis">
      Liu F-23028347
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-m-38038215" class="md-nav__link">
    <span class="md-ellipsis">
      Liu M-38038215
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-y-34128465" class="md-nav__link">
    <span class="md-ellipsis">
      Liu Y-34128465
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ning-c-38036550" class="md-nav__link">
    <span class="md-ellipsis">
      Ning C-38036550
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parisinos-c-32247823" class="md-nav__link">
    <span class="md-ellipsis">
      Parisinos C-32247823
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#persyn-e-32358547" class="md-nav__link">
    <span class="md-ellipsis">
      Persyn E-32358547
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pirruccello-jp-32382064" class="md-nav__link">
    <span class="md-ellipsis">
      Pirruccello JP-32382064
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rakowski-a" class="md-nav__link">
    <span class="md-ellipsis">
      Rakowski A
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shah-m-37604819" class="md-nav__link">
    <span class="md-ellipsis">
      Shah M-37604819
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smith-sm-33875891" class="md-nav__link">
    <span class="md-ellipsis">
      Smith SM-33875891
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun-bb-36241887" class="md-nav__link">
    <span class="md-ellipsis">
      Sun BB-36241887
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wang-c-35606419" class="md-nav__link">
    <span class="md-ellipsis">
      Wang C-35606419
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#warrier-v-37592024" class="md-nav__link">
    <span class="md-ellipsis">
      Warrier V-37592024
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yu-s" class="md-nav__link">
    <span class="md-ellipsis">
      Yu S
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#method" class="md-nav__link">
    <span class="md-ellipsis">
      Method
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Method">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#kirchler-m-35640976" class="md-nav__link">
    <span class="md-ellipsis">
      Kirchler M-35640976
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu-z-28736311" class="md-nav__link">
    <span class="md-ellipsis">
      Xu Z-28736311
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prediction" class="md-nav__link">
    <span class="md-ellipsis">
      Prediction
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Prediction">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#huang-yj-38762475" class="md-nav__link">
    <span class="md-ellipsis">
      Huang YJ-38762475
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#littlejohns-tj-32457287" class="md-nav__link">
    <span class="md-ellipsis">
      Littlejohns TJ-32457287
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tools
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Annotation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" >
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Association_tests
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    RWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    TWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    eQTL
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    sQTL
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Colocalization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data_processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Dimension_reduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Drug_discovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Fine_mapping
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_prioritization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GxE_interactions
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    HLA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Imputation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mendelian_randomization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Power_analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proteomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simulation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Winners_curse
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Visualization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Chromosome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Forest_plot
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    GWAS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Heatmap
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    LD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variants_on_protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Population_Genetics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Admixture
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Genealogy_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genealogy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phylogenetic_tree
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Selection
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Reference
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Reference
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Phenotype_Phenotype_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Phenotype
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Genome_Genome_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Gene_Gene_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Gene
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Variant_Variant_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Variant
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Protein_Protein_README/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Protein
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deepflowgomes-b-38082205" class="md-nav__link">
    <span class="md-ellipsis">
      DeepFlow/Gomes B-38082205
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#elliott-lt-30305740" class="md-nav__link">
    <span class="md-ellipsis">
      Elliott LT-30305740
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fu-j-38811844" class="md-nav__link">
    <span class="md-ellipsis">
      Fu J-38811844
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#haas-me-34957434" class="md-nav__link">
    <span class="md-ellipsis">
      Haas ME-34957434
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ikram-m-28627999" class="md-nav__link">
    <span class="md-ellipsis">
      Ikram M.-28627999
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#karkar-s-33664500" class="md-nav__link">
    <span class="md-ellipsis">
      Karkar S-33664500
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#khurshid-s-36944631" class="md-nav__link">
    <span class="md-ellipsis">
      Khurshid S-36944631
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-f-23028347" class="md-nav__link">
    <span class="md-ellipsis">
      Liu F-23028347
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-m-38038215" class="md-nav__link">
    <span class="md-ellipsis">
      Liu M-38038215
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#liu-y-34128465" class="md-nav__link">
    <span class="md-ellipsis">
      Liu Y-34128465
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ning-c-38036550" class="md-nav__link">
    <span class="md-ellipsis">
      Ning C-38036550
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#parisinos-c-32247823" class="md-nav__link">
    <span class="md-ellipsis">
      Parisinos C-32247823
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#persyn-e-32358547" class="md-nav__link">
    <span class="md-ellipsis">
      Persyn E-32358547
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pirruccello-jp-32382064" class="md-nav__link">
    <span class="md-ellipsis">
      Pirruccello JP-32382064
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rakowski-a" class="md-nav__link">
    <span class="md-ellipsis">
      Rakowski A
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#shah-m-37604819" class="md-nav__link">
    <span class="md-ellipsis">
      Shah M-37604819
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#smith-sm-33875891" class="md-nav__link">
    <span class="md-ellipsis">
      Smith SM-33875891
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sun-bb-36241887" class="md-nav__link">
    <span class="md-ellipsis">
      Sun BB-36241887
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wang-c-35606419" class="md-nav__link">
    <span class="md-ellipsis">
      Wang C-35606419
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#warrier-v-37592024" class="md-nav__link">
    <span class="md-ellipsis">
      Warrier V-37592024
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#yu-s" class="md-nav__link">
    <span class="md-ellipsis">
      Yu S
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#method" class="md-nav__link">
    <span class="md-ellipsis">
      Method
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Method">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#kirchler-m-35640976" class="md-nav__link">
    <span class="md-ellipsis">
      Kirchler M-35640976
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#xu-z-28736311" class="md-nav__link">
    <span class="md-ellipsis">
      Xu Z-28736311
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prediction" class="md-nav__link">
    <span class="md-ellipsis">
      Prediction
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Prediction">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#huang-yj-38762475" class="md-nav__link">
    <span class="md-ellipsis">
      Huang YJ-38762475
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#littlejohns-tj-32457287" class="md-nav__link">
    <span class="md-ellipsis">
      Littlejohns TJ-32457287
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>Imaging</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#deepflowgomes-b-38082205">DeepFlow/Gomes B-38082205</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Gomes B, Singh A, O'Sullivan JW, Schnurr TM, ...&amp;, Ashley EA. (2024) Genetic architecture of cardiac dynamic flow volumes Nat. Genet., 56 (2) 245-257. doi:10.1038/s41588-023-01587-5. PMID 38082205</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#elliott-lt-30305740">Elliott LT-30305740</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Elliott LT, Sharp K, Alfaro-Almagro F, Shi S, ...&amp;, Smith SM. (2018) Genome-wide association studies of brain imaging phenotypes in UK Biobank Nature, 562 (7726) 210-216. doi:10.1038/s41586-018-0571-7. PMID 30305740</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fu-j-38811844">Fu J-38811844</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Fu J, Zhang Q, Wang J, Wang M, ...&amp;, CHIMGEN Consortium. (2024) Cross-ancestry genome-wide association studies of brain imaging phenotypes Nat. Genet.,  () . doi:10.1038/s41588-024-01766-y. PMID 38811844</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#haas-me-34957434">Haas ME-34957434</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Haas ME, Pirruccello JP, Friedman SN, Wang M, ...&amp;, Khera AV. (2021) Machine learning enables new insights into genetic contributions to liver fat accumulation Cell Genom., 1 (3) 100066. doi:10.1016/j.xgen.2021.100066. PMID 34957434</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ikram-m-28627999">Ikram M.-28627999</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Ikram MA, Zonneveld HI, Roshchupkin G, Smith AV, ...&amp;, Adams HH. (2018) Heritability and genome-wide associations studies of cerebral blood flow in the general population J. Cereb. Blood Flow Metab., 38 (9) 1598-1608. doi:10.1177/0271678X17715861. PMID 28627999</td>
<td style="text-align: left;">2018</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#karkar-s-33664500">Karkar S-33664500</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Karkar S, Dandine-Roulland C, Mangin JF, Le Guen Y, ...&amp;, Frouin V. (2021) Genome-wide haplotype association study in imaging genetics using whole-brain sulcal openings of 16,304 UK Biobank subjects Eur. J. Hum. Genet., 29 (9) 1424-1437. doi:10.1038/s41431-021-00827-8. PMID 33664500</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#khurshid-s-36944631">Khurshid S-36944631</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Khurshid S, Lazarte J, Pirruccello JP, Weng LC, ...&amp;, Lubitz SA. (2023) Clinical and genetic associations of deep learning-derived cardiac magnetic resonance-based left ventricular mass Nat. Commun., 14 (1) 1558. doi:10.1038/s41467-023-37173-w. PMID 36944631</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#liu-f-23028347">Liu F-23028347</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Liu F, van der Lijn F, Schurmann C, Zhu G, ...&amp;, Kayser M. (2012) A genome-wide association study identifies five loci influencing facial morphology in Europeans PLoS Genet., 8 (9) e1002932. doi:10.1371/journal.pgen.1002932. PMID 23028347</td>
<td style="text-align: left;">2012</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#liu-m-38038215">Liu M-38038215</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Liu M, Khasiyev F, Sariya S, Spagnolo-Allende A, ...&amp;, Gutierrez J. (2023) Chromosome 10q24.32 variants associate with brain arterial diameters in diverse populations: A genome-wide association study J. Am. Heart Assoc., 12 (23) e030935. doi:10.1161/JAHA.123.030935. PMID 38038215</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#liu-y-34128465">Liu Y-34128465</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Liu Y, Basty N, Whitcher B, Bell JD, ...&amp;, Cule M. (2021) Genetic architecture of 11 organ traits derived from abdominal MRI using deep learning Elife, 10 () . doi:10.7554/eLife.65554. PMID 34128465</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ning-c-38036550">Ning C-38036550</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Ning C, Fan L, Jin M, Wang W, ...&amp;, Miao X. (2023) Genome-wide association analysis of left ventricular imaging-derived phenotypes identifies 72 risk loci and yields genetic insights into hypertrophic cardiomyopathy Nat. Commun., 14 (1) 7900. doi:10.1038/s41467-023-43771-5. PMID 38036550</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#parisinos-c-32247823">Parisinos C-32247823</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Parisinos CA, Wilman HR, Thomas EL, Kelly M, ...&amp;, Yaghootkar H. (2020) Genome-wide and Mendelian randomisation studies of liver MRI yield insights into the pathogenesis of steatohepatitis J. Hepatol., 73 (2) 241-251. doi:10.1016/j.jhep.2020.03.032. PMID 32247823</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#persyn-e-32358547">Persyn E-32358547</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Persyn E, Hanscombe KB, Howson JMM, Lewis CM, ...&amp;, Markus HS. (2020) Genome-wide association study of MRI markers of cerebral small vessel disease in 42,310 participants Nat. Commun., 11 (1) 2175. doi:10.1038/s41467-020-15932-3. PMID 32358547</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pirruccello-jp-32382064">Pirruccello JP-32382064</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Pirruccello JP, Bick A, Wang M, Chaffin M, ...&amp;, Aragam KG. (2020) Analysis of cardiac magnetic resonance imaging in 36,000 individuals yields genetic insights into dilated cardiomyopathy Nat. Commun., 11 (1) 2254. doi:10.1038/s41467-020-15823-7. PMID 32382064</td>
<td style="text-align: left;">2020</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#rakowski-a">Rakowski A</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Rakowski, A., Monti, R. &amp; Lippert, C. TransferGWAS of T1-weighted brain MRI data from the UK Biobank. bioRxiv 2024.06.11.24308721 (2024) doi:10.1101/2024.06.11.24308721.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shah-m-37604819">Shah M-37604819</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Shah M, de A Inácio MH, Lu C, Schiratti PR, ...&amp;, O'Regan DP. (2023) Environmental and genetic predictors of human cardiovascular ageing Nat. Commun., 14 (1) 4941. doi:10.1038/s41467-023-40566-6. PMID 37604819</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#smith-sm-33875891">Smith SM-33875891</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Smith SM, Douaud G, Chen W, Hanayik T, ...&amp;, Elliott LT. (2021) An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank Nat. Neurosci., 24 (5) 737-745. doi:10.1038/s41593-021-00826-4. PMID 33875891</td>
<td style="text-align: left;">2021</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sun-bb-36241887">Sun BB-36241887</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Sun BB, Loomis SJ, Pizzagalli F, Shatokhina N, ...&amp;, Whelan CD. (2022) Genetic map of regional sulcal morphology in the human brain from UK biobank data Nat. Commun., 13 (1) 6071. doi:10.1038/s41467-022-33829-1. PMID 36241887</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wang-c-35606419">Wang C-35606419</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Wang C, Martins-Bach AB, Alfaro-Almagro F, Douaud G, ...&amp;, Miller KL. (2022) Phenotypic and genetic associations of quantitative magnetic susceptibility in UK Biobank brain imaging Nat. Neurosci., 25 (6) 818-831. doi:10.1038/s41593-022-01074-w. PMID 35606419</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#warrier-v-37592024">Warrier V-37592024</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Warrier V, Stauffer EM, Huang QQ, Wigdor EM, ...&amp;, Bethlehem RAI. (2023) Genetic insights into human cortical organization and development through genome-wide analyses of 2,347 neuroimaging phenotypes Nat. Genet., 55 (9) 1483-1493. doi:10.1038/s41588-023-01475-y. PMID 37592024</td>
<td style="text-align: left;">2023</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#yu-s">Yu S</a></td>
<td style="text-align: left;">GWAS</td>
<td style="text-align: left;">Yu, S. et al. A novel classification framework for genome-wide association study of whole brain MRI images using deep learning. bioRxiv 2024.01.11.575251 (2024) doi:10.1101/2024.01.11.575251.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#kirchler-m-35640976">Kirchler M-35640976</a></td>
<td style="text-align: left;">Method</td>
<td style="text-align: left;">Kirchler M, Konigorski S, Norden M, Meltendorf C, ...&amp;, Lippert C. (2022) transferGWAS: GWAS of images using deep transfer learning Bioinformatics, 38 (14) 3621-3628. doi:10.1093/bioinformatics/btac369. PMID 35640976</td>
<td style="text-align: left;">2022</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#xu-z-28736311">Xu Z-28736311</a></td>
<td style="text-align: left;">Method</td>
<td style="text-align: left;">Xu Z, Wu C, Pan W, Alzheimer's Disease Neuroimaging Initiative. (2017) Imaging-wide association study: Integrating imaging endophenotypes in GWAS Neuroimage, 159 () 159-169. doi:10.1016/j.neuroimage.2017.07.036. PMID 28736311</td>
<td style="text-align: left;">2017</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#huang-yj-38762475">Huang YJ-38762475</a></td>
<td style="text-align: left;">Prediction</td>
<td style="text-align: left;">Huang YJ, Chen CH, Yang HC. (2024) AI-enhanced integration of genetic and medical imaging data for risk assessment of Type 2 diabetes Nat. Commun., 15 (1) 4230. doi:10.1038/s41467-024-48618-1. PMID 38762475</td>
<td style="text-align: left;">2024</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#littlejohns-tj-32457287">Littlejohns TJ-32457287</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Littlejohns TJ, Holliday J, Gibson LM, Garratt S, ...&amp;, Allen NE. (2020) The UK Biobank imaging enhancement of 100,000 participants: rationale, data collection, management and future directions Nat. Commun., 11 (1) 2624. doi:10.1038/s41467-020-15948-9. PMID 32457287</td>
<td style="text-align: left;">2020</td>
</tr>
</tbody>
</table>
<h2 id="gwas">GWAS</h2>
<h3 id="deepflowgomes-b-38082205">DeepFlow/Gomes B-38082205</h3>
<ul>
<li><strong>NAME</strong> : DeepFlow/Gomes B-38082205 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genetic architecture of cardiac dynamic flow volumes </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01587-5 </li>
<li><strong>ABSTRACT</strong> : Cardiac blood flow is a critical determinant of human health. However, the definition of its genetic architecture is limited by the technical challenge of capturing dynamic flow volumes from cardiac imaging at scale. We present DeepFlow, a deep-learning system to extract cardiac flow and volumes from phase-contrast cardiac magnetic resonance imaging. A mixed-linear model applied to 37,653 individuals from the UK Biobank reveals genome-wide significant associations across cardiac dynamic flow volumes spanning from aortic forward velocity to aortic regurgitation fraction. Mendelian randomization reveals a causal role for aortic root size in aortic valve regurgitation. Among the most significant contributing variants, localizing genes (near ELN, PRDM6 and ADAMTS7) are implicated in connective tissue and blood pressure pathways. Here we show that DeepFlow cardiac flow phenotyping at scale, combined with genotyping data, reinforces the contribution of connective tissue genes, blood pressure and root size to aortic valve function. </li>
<li><strong>COPYRIGHT</strong> : https://www.springernature.com/gp/researchers/text-and-data-mining </li>
<li><strong>CITATION</strong> : Gomes B, Singh A, O'Sullivan JW, Schnurr TM, ...&amp;, Ashley EA. (2024) Genetic architecture of cardiac dynamic flow volumes Nat. Genet., 56 (2) 245-257. doi:10.1038/s41588-023-01587-5. PMID 38082205 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2024 ; 56 ; 2 ; 245-257 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38082205">38082205</a> </li>
</ul>
<h3 id="elliott-lt-30305740">Elliott LT-30305740</h3>
<ul>
<li><strong>NAME</strong> : Elliott LT-30305740 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genome-wide association studies of brain imaging phenotypes in UK Biobank </li>
<li><strong>DOI</strong> : 10.1038/s41586-018-0571-7 </li>
<li><strong>ABSTRACT</strong> : The genetic architecture of brain structure and function is largely unknown. To investigate this, we carried out genome-wide association studies of 3,144 functional and structural brain imaging phenotypes from UK Biobank (discovery dataset 8,428 subjects). Here we show that many of these phenotypes are heritable. We identify 148 clusters of associations between single nucleotide polymorphisms and imaging phenotypes that replicate at P &lt; 0.05, when we would expect 21 to replicate by chance. Notable significant, interpretable associations include: iron transport and storage genes, related to magnetic susceptibility of subcortical brain tissue; extracellular matrix and epidermal growth factor genes, associated with white matter micro-structure and lesions; genes that regulate mid-line axon development, associated with organization of the pontine crossing tract; and overall 17 genes involved in development, pathway signalling and plasticity. Our results provide insights into the genetic architecture of the brain that are relevant to neurological and psychiatric disorders, brain development and ageing. </li>
<li><strong>CITATION</strong> : Elliott LT, Sharp K, Alfaro-Almagro F, Shi S, ...&amp;, Smith SM. (2018) Genome-wide association studies of brain imaging phenotypes in UK Biobank Nature, 562 (7726) 210-216. doi:10.1038/s41586-018-0571-7. PMID 30305740 </li>
<li><strong>JOURNAL_INFO</strong> : Nature ; Nature ; 2018 ; 562 ; 7726 ; 210-216 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30305740">30305740</a> </li>
</ul>
<h3 id="fu-j-38811844">Fu J-38811844</h3>
<ul>
<li><strong>NAME</strong> : Fu J-38811844 </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS,EUR </li>
<li><strong>TITLE</strong> : Cross-ancestry genome-wide association studies of brain imaging phenotypes </li>
<li><strong>DOI</strong> : 10.1038/s41588-024-01766-y </li>
<li><strong>ABSTRACT</strong> : Genome-wide association studies of brain imaging phenotypes are mainly performed in European populations, but other populations are severely under-represented. Here, we conducted Chinese-alone and cross-ancestry genome-wide association studies of 3,414 brain imaging phenotypes in 7,058 Chinese Han and 33,224 white British participants. We identified 38 new associations in Chinese-alone analyses and 486 additional new associations in cross-ancestry meta-analyses at P &lt; 1.46 × 10-11 for discovery and P &lt; 0.05 for replication. We pooled significant autosomal associations identified by single- or cross-ancestry analyses into 6,443 independent associations, which showed uneven distribution in the genome and the phenotype subgroups. We further divided them into 44 associations with different effect sizes and 3,557 associations with similar effect sizes between ancestries. Loci of these associations were shared with 15 brain-related non-imaging traits including cognition and neuropsychiatric disorders. Our results provide a valuable catalog of genetic associations for brain imaging phenotypes in more diverse populations. </li>
<li><strong>CITATION</strong> : Fu J, Zhang Q, Wang J, Wang M, ...&amp;, CHIMGEN Consortium. (2024) Cross-ancestry genome-wide association studies of brain imaging phenotypes Nat. Genet.,  () . doi:10.1038/s41588-024-01766-y. PMID 38811844 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2024 ; <NA> ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38811844">38811844</a> </li>
</ul>
<h3 id="haas-me-34957434">Haas ME-34957434</h3>
<ul>
<li><strong>NAME</strong> : Haas ME-34957434 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Machine learning enables new insights into genetic contributions to liver fat accumulation </li>
<li><strong>DOI</strong> : 10.1016/j.xgen.2021.100066 </li>
<li><strong>ABSTRACT</strong> : Excess liver fat, called hepatic steatosis, is a leading risk factor for end-stage liver disease and cardiometabolic diseases but often remains undiagnosed in clinical practice because of the need for direct imaging assessments. We developed an abdominal MRI-based machine-learning algorithm to accurately estimate liver fat (correlation coefficients, 0.97-0.99) from a truth dataset of 4,511 middle-aged UK Biobank participants, enabling quantification in 32,192 additional individuals. 17% of participants had predicted liver fat levels indicative of steatosis, and liver fat could not have been reliably estimated based on clinical factors such as BMI. A genome-wide association study of common genetic variants and liver fat replicated three known associations and identified five newly associated variants in or near the MTARC1, ADH1B, TRIB1, GPAM, and MAST3 genes (p 1.32 per SD score, p &lt; 9 × 10-17). Rare inactivating variants in the APOB or MTTP genes were identified in 0.8% of individuals with steatosis and conferred more than 6-fold risk (p &lt; 2 × 10-5), highlighting a molecular subtype of hepatic steatosis characterized by defective secretion of apolipoprotein B-containing lipoproteins. We demonstrate that our imaging-based machine-learning model accurately estimates liver fat and may be useful in epidemiological and genetic studies of hepatic steatosis. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by-nc-nd/4.0/ </li>
<li><strong>CITATION</strong> : Haas ME, Pirruccello JP, Friedman SN, Wang M, ...&amp;, Khera AV. (2021) Machine learning enables new insights into genetic contributions to liver fat accumulation Cell Genom., 1 (3) 100066. doi:10.1016/j.xgen.2021.100066. PMID 34957434 </li>
<li><strong>JOURNAL_INFO</strong> : Cell genomics ; Cell Genom. ; 2021 ; 1 ; 3 ; 100066 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34957434">34957434</a> </li>
</ul>
<h3 id="ikram-m-28627999">Ikram M.-28627999</h3>
<ul>
<li><strong>NAME</strong> : Ikram M.-28627999 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : Heritability and genome-wide associations studies of cerebral blood flow in the general population </li>
<li><strong>DOI</strong> : 10.1177/0271678X17715861 </li>
<li><strong>ABSTRACT</strong> : Cerebral blood flow is an important process for brain functioning and its dysregulation is implicated in multiple neurological disorders. While environmental risk factors have been identified, it remains unclear to what extent the flow is regulated by genetics. Here we performed heritability and genome-wide association analyses of cerebral blood flow in a population-based cohort study. We included 4472 persons free of cortical infarcts who underwent genotyping and phase-contrast magnetic resonance flow imaging (mean age 64.8 ± 10.8 years). The flow rate, cross-sectional area of the vessel, and flow velocity through the vessel were measured in the basilar artery and bilateral carotids. We found that the flow rate of the basilar artery is most heritable (h2 (SE) = 24.1 (9.8), p-value = 0.0056), and this increased over age. The association studies revealed two significant loci for the right carotid artery area (rs12546630, p-value = 2.0 × 10-8) and velocity (rs2971609, p-value = 1.4 × 10-8), with the latter showing a concordant effect in an independent sample (N = 1350, p-value = 0.057, meta-analyzed p-value = 2.5 × 10-9). These loci were also associated with other cerebral blood flow parameters below genome-wide significance, and rs2971609 lies in a known migraine locus. These findings establish that cerebral blood flow is under genetic control with potential relevance for neurological diseases. </li>
<li><strong>CITATION</strong> : Ikram MA, Zonneveld HI, Roshchupkin G, Smith AV, ...&amp;, Adams HH. (2018) Heritability and genome-wide associations studies of cerebral blood flow in the general population J. Cereb. Blood Flow Metab., 38 (9) 1598-1608. doi:10.1177/0271678X17715861. PMID 28627999 </li>
<li><strong>JOURNAL_INFO</strong> : Journal of cerebral blood flow and metabolism: official journal of the International Society of Cerebral Blood Flow and Metabolism ; J. Cereb. Blood Flow Metab. ; 2018 ; 38 ; 9 ; 1598-1608 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28627999">28627999</a> </li>
</ul>
<h3 id="karkar-s-33664500">Karkar S-33664500</h3>
<ul>
<li><strong>NAME</strong> : Karkar S-33664500 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genome-wide haplotype association study in imaging genetics using whole-brain sulcal openings of 16,304 UK Biobank subjects </li>
<li><strong>DOI</strong> : 10.1038/s41431-021-00827-8 </li>
<li><strong>ABSTRACT</strong> : Neuroimaging-genetics cohorts gather two types of data: brain imaging and genetic data. They allow the discovery of associations between genetic variants and brain imaging features. They are invaluable resources to study the influence of genetics and environment in the brain features variance observed in normal and pathological populations. This study presents a genome-wide haplotype analysis for 123 brain sulcus opening value (a measure of sulcal width) across the whole brain that include 16,304 subjects from UK Biobank. Using genetic maps, we defined 119,548 blocks of low recombination rate distributed along the 22 autosomal chromosomes and analyzed 1,051,316 haplotypes. To test associations between haplotypes and complex traits, we designed three statistical approaches. Two of them use a model that includes all the haplotypes for a single block, while the last approach considers each haplotype independently. All the statistics produced were assessed as rigorously as possible. Thanks to the rich imaging dataset at hand, we used resampling techniques to assess False Positive Rate for each statistical approach in a genome-wide and brain-wide context. The results on real data show that genome-wide haplotype analyses are more sensitive than single-SNP approach and account for local complex Linkage Disequilibrium (LD) structure, which makes genome-wide haplotype analysis an interesting and statistically sound alternative to the single-SNP counterpart. </li>
<li><strong>CITATION</strong> : Karkar S, Dandine-Roulland C, Mangin JF, Le Guen Y, ...&amp;, Frouin V. (2021) Genome-wide haplotype association study in imaging genetics using whole-brain sulcal openings of 16,304 UK Biobank subjects Eur. J. Hum. Genet., 29 (9) 1424-1437. doi:10.1038/s41431-021-00827-8. PMID 33664500 </li>
<li><strong>JOURNAL_INFO</strong> : European journal of human genetics: EJHG ; Eur. J. Hum. Genet. ; 2021 ; 29 ; 9 ; 1424-1437 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33664500">33664500</a> </li>
</ul>
<h3 id="khurshid-s-36944631">Khurshid S-36944631</h3>
<ul>
<li><strong>NAME</strong> : Khurshid S-36944631 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Clinical and genetic associations of deep learning-derived cardiac magnetic resonance-based left ventricular mass </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-37173-w </li>
<li><strong>ABSTRACT</strong> : Left ventricular mass is a risk marker for cardiovascular events, and may indicate an underlying cardiomyopathy. Cardiac magnetic resonance is the gold-standard for left ventricular mass estimation, but is challenging to obtain at scale. Here, we use deep learning to enable genome-wide association study of cardiac magnetic resonance-derived left ventricular mass indexed to body surface area within 43,230 UK Biobank participants. We identify 12 genome-wide associations (1 known at TTN and 11 novel for left ventricular mass), implicating genes previously associated with cardiac contractility and cardiomyopathy. Cardiac magnetic resonance-derived indexed left ventricular mass is associated with incident dilated and hypertrophic cardiomyopathies, and implantable cardioverter-defibrillator implant. An indexed left ventricular mass polygenic risk score ≥90th percentile is also associated with incident implantable cardioverter-defibrillator implant in separate UK Biobank (hazard ratio 1.22, 95% CI 1.05-1.44) and Mass General Brigham (hazard ratio 1.75, 95% CI 1.12-2.74) samples. Here, we perform a genome-wide association study of cardiac magnetic resonance-derived indexed left ventricular mass to identify 11 novel variants and demonstrate that cardiac magnetic resonance-derived and genetically predicted indexed left ventricular mass are associated with incident cardiomyopathy. </li>
<li><strong>CITATION</strong> : Khurshid S, Lazarte J, Pirruccello JP, Weng LC, ...&amp;, Lubitz SA. (2023) Clinical and genetic associations of deep learning-derived cardiac magnetic resonance-based left ventricular mass Nat. Commun., 14 (1) 1558. doi:10.1038/s41467-023-37173-w. PMID 36944631 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 1558 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36944631">36944631</a> </li>
</ul>
<h3 id="liu-f-23028347">Liu F-23028347</h3>
<ul>
<li><strong>NAME</strong> : Liu F-23028347 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>TITLE</strong> : A genome-wide association study identifies five loci influencing facial morphology in Europeans </li>
<li><strong>DOI</strong> : 10.1371/journal.pgen.1002932 </li>
<li><strong>ABSTRACT</strong> : Inter-individual variation in facial shape is one of the most noticeable phenotypes in humans, and it is clearly under genetic regulation; however, almost nothing is known about the genetic basis of normal human facial morphology. We therefore conducted a genome-wide association study for facial shape phenotypes in multiple discovery and replication cohorts, considering almost ten thousand individuals of European descent from several countries. Phenotyping of facial shape features was based on landmark data obtained from three-dimensional head magnetic resonance images (MRIs) and two-dimensional portrait images. We identified five independent genetic loci associated with different facial phenotypes, suggesting the involvement of five candidate genes--PRDM16, PAX3, TP63, C5orf50, and COL17A1--in the determination of the human face. Three of them have been implicated previously in vertebrate craniofacial development and disease, and the remaining two genes potentially represent novel players in the molecular networks governing facial development. Our finding at PAX3 influencing the position of the nasion replicates a recent GWAS of facial features. In addition to the reported GWA findings, we established links between common DNA variants previously associated with NSCL/P at 2p21, 8q24, 13q31, and 17q22 and normal facial-shape variations based on a candidate gene approach. Overall our study implies that DNA variants in genes essential for craniofacial development contribute with relatively small effect size to the spectrum of normal variation in human facial morphology. This observation has important consequences for future studies aiming to identify more genes involved in the human facial morphology, as well as for potential applications of DNA prediction of facial shape such as in future forensic applications. </li>
<li><strong>CITATION</strong> : Liu F, van der Lijn F, Schurmann C, Zhu G, ...&amp;, Kayser M. (2012) A genome-wide association study identifies five loci influencing facial morphology in Europeans PLoS Genet., 8 (9) e1002932. doi:10.1371/journal.pgen.1002932. PMID 23028347 </li>
<li><strong>JOURNAL_INFO</strong> : PLoS genetics ; PLoS Genet. ; 2012 ; 8 ; 9 ; e1002932 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/23028347">23028347</a> </li>
</ul>
<h3 id="liu-m-38038215">Liu M-38038215</h3>
<ul>
<li><strong>NAME</strong> : Liu M-38038215 </li>
<li><strong>MAIN_ANCESTRY</strong> : Cross-ancestry </li>
<li><strong>TITLE</strong> : Chromosome 10q24.32 variants associate with brain arterial diameters in diverse populations: A genome-wide association study </li>
<li><strong>DOI</strong> : 10.1161/JAHA.123.030935 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND: Brain arterial diameters (BADs) are novel imaging biomarkers of cerebrovascular disease, cognitive decline, and dementia. Traditional vascular risk factors have been associated with BADs, but whether there may be genetic determinants of BADs is unknown. METHODS AND RESULTS: The authors studied 4150 participants from 6 geographically diverse population-based cohorts (40% European, 14% African, 22% Hispanic, 24% Asian ancestries). Brain arterial diameters for 13 segments were measured and averaged to obtain a global measure of BADs as well as the posterior and anterior circulations. A genome-wide association study revealed 14 variants at one locus associated with global BAD at genome-wide significance (P&lt;5×10-8) (top single-nucleotide polymorphism, rs7921574; β=0.06 [P=1.54×10-8]). This locus mapped to an intron of CNNM2. A trans-ancestry genome-wide association study meta-analysis identified 2 more loci at NT5C2 (rs10748839; P=2.54×10-8) and AS3MT (rs10786721; P=4.97×10-8), associated with global BAD. In addition, 2 single-nucleotide polymorphisms colocalized with expression of CNNM2 (rs7897654; β=0.12 [P=6.17×10-7]) and AL356608.1 (rs10786719; β=-0.17 [P=6.60×10-6]) in brain tissue. For the posterior BAD, 2 variants at one locus mapped to an intron of TCF25 were identified (top single-nucleotide polymorphism, rs35994878; β=0.11 [P=2.94×10-8]). For the anterior BAD, one locus at ADAP1 was identified in trans-ancestry genome-wide association analysis (rs34217249; P=3.11×10-8). CONCLUSIONS: The current study reveals 3 novel risk loci (CNNM2, NT5C2, and AS3MT) associated with BADs. These findings may help elucidate the mechanism by which BADs may influence cerebrovascular health. </li>
<li><strong>CITATION</strong> : Liu M, Khasiyev F, Sariya S, Spagnolo-Allende A, ...&amp;, Gutierrez J. (2023) Chromosome 10q24.32 variants associate with brain arterial diameters in diverse populations: A genome-wide association study J. Am. Heart Assoc., 12 (23) e030935. doi:10.1161/JAHA.123.030935. PMID 38038215 </li>
<li><strong>JOURNAL_INFO</strong> : Journal of the American Heart Association ; J. Am. Heart Assoc. ; 2023 ; 12 ; 23 ; e030935 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38038215">38038215</a> </li>
</ul>
<h3 id="liu-y-34128465">Liu Y-34128465</h3>
<ul>
<li><strong>NAME</strong> : Liu Y-34128465 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genetic architecture of 11 organ traits derived from abdominal MRI using deep learning </li>
<li><strong>DOI</strong> : 10.7554/eLife.65554 </li>
<li><strong>ABSTRACT</strong> : Cardiometabolic diseases are an increasing global health burden. While socioeconomic, environmental, behavioural, and genetic risk factors have been identified, a better understanding of the underlying mechanisms is required to develop more effective interventions. Magnetic resonance imaging (MRI) has been used to assess organ health, but biobank-scale studies are still in their infancy. Using over 38,000 abdominal MRI scans in the UK Biobank, we used deep learning to quantify volume, fat, and iron in seven organs and tissues, and demonstrate that imaging-derived phenotypes reflect health status. We show that these traits have a substantial heritable component (8-44%) and identify 93 independent genome-wide significant associations, including four associations with liver traits that have not previously been reported. Our work demonstrates the tractability of deep learning to systematically quantify health parameters from high-throughput MRI across a range of organs and tissues, and use the largest-ever study of its kind to generate new insights into the genetic architecture of these traits. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Liu Y, Basty N, Whitcher B, Bell JD, ...&amp;, Cule M. (2021) Genetic architecture of 11 organ traits derived from abdominal MRI using deep learning Elife, 10 () . doi:10.7554/eLife.65554. PMID 34128465 </li>
<li><strong>JOURNAL_INFO</strong> : eLife ; Elife ; 2021 ; 10 ; <NA> ; <NA> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34128465">34128465</a> </li>
</ul>
<h3 id="ning-c-38036550">Ning C-38036550</h3>
<ul>
<li><strong>NAME</strong> : Ning C-38036550 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genome-wide association analysis of left ventricular imaging-derived phenotypes identifies 72 risk loci and yields genetic insights into hypertrophic cardiomyopathy </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-43771-5 </li>
<li><strong>ABSTRACT</strong> : Left ventricular regional wall thickness (LVRWT) is an independent predictor of morbidity and mortality in cardiovascular diseases (CVDs). To identify specific genetic influences on individual LVRWT, we established a novel deep learning algorithm to calculate 12 LVRWTs accurately in 42,194 individuals from the UK Biobank with cardiac magnetic resonance (CMR) imaging. Genome-wide association studies of CMR-derived 12 LVRWTs identified 72 significant genetic loci associated with at least one LVRWT phenotype (P &lt; 5 × 10-8), which were revealed to actively participate in heart development and contraction pathways. Significant causal relationships were observed between the LVRWT traits and hypertrophic cardiomyopathy (HCM) using genetic correlation and Mendelian randomization analyses (P &lt; 0.01). The polygenic risk score of inferoseptal LVRWT at end systole exhibited a notable association with incident HCM, facilitating the identification of high-risk individuals. The findings yield insights into the genetic determinants of LVRWT phenotypes and shed light on the biological basis for HCM etiology. </li>
<li><strong>CITATION</strong> : Ning C, Fan L, Jin M, Wang W, ...&amp;, Miao X. (2023) Genome-wide association analysis of left ventricular imaging-derived phenotypes identifies 72 risk loci and yields genetic insights into hypertrophic cardiomyopathy Nat. Commun., 14 (1) 7900. doi:10.1038/s41467-023-43771-5. PMID 38036550 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 7900 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38036550">38036550</a> </li>
</ul>
<h3 id="parisinos-c-32247823">Parisinos C-32247823</h3>
<ul>
<li><strong>NAME</strong> : Parisinos C-32247823 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genome-wide and Mendelian randomisation studies of liver MRI yield insights into the pathogenesis of steatohepatitis </li>
<li><strong>DOI</strong> : 10.1016/j.jhep.2020.03.032 </li>
<li><strong>ABSTRACT</strong> : BACKGROUND &amp; AIMS: MRI-based corrected T1 (cT1) is a non-invasive method to grade the severity of steatohepatitis and liver fibrosis. We aimed to identify genetic variants influencing liver cT1 and use genetics to understand mechanisms underlying liver fibroinflammatory disease and its link with other metabolic traits and diseases. METHODS: First, we performed a genome-wide association study (GWAS) in 14,440 Europeans, with liver cT1 measures, from the UK Biobank. Second, we explored the effects of the cT1 variants on liver blood tests, and a range of metabolic traits and diseases. Third, we used Mendelian randomisation to test the causal effects of 24 predominantly metabolic traits on liver cT1 measures. RESULTS: We identified 6 independent genetic variants associated with liver cT1 that reached the GWAS significance threshold (p &lt;5×10-8). Four of the variants (rs759359281 in SLC30A10, rs13107325 in SLC39A8, rs58542926 in TM6SF2, rs738409 in PNPLA3) were also associated with elevated aminotransferases and had variable effects on liver fat and other metabolic traits. Insulin resistance, type 2 diabetes, non-alcoholic fatty liver and body mass index were causally associated with elevated cT1, whilst favourable adiposity (instrumented by variants associated with higher adiposity but lower risk of cardiometabolic disease and lower liver fat) was found to be protective. CONCLUSION: The association between 2 metal ion transporters and cT1 indicates an important new mechanism in steatohepatitis. Future studies are needed to determine whether interventions targeting the identified transporters might prevent liver disease in at-risk individuals. LAY SUMMARY: We estimated levels of liver inflammation and scarring based on magnetic resonance imaging of 14,440 UK Biobank participants. We performed a genetic study and identified variations in 6 genes associated with levels of liver inflammation and scarring. Participants with variations in 4 of these genes also had higher levels of markers of liver cell injury in blood samples, further validating their role in liver health. Two identified genes are involved in the transport of metal ions in our body. Further investigation of these variations may lead to better detection, assessment, and/or treatment of liver inflammation and scarring. </li>
<li><strong>COPYRIGHT</strong> : http://creativecommons.org/licenses/by/4.0/ </li>
<li><strong>CITATION</strong> : Parisinos CA, Wilman HR, Thomas EL, Kelly M, ...&amp;, Yaghootkar H. (2020) Genome-wide and Mendelian randomisation studies of liver MRI yield insights into the pathogenesis of steatohepatitis J. Hepatol., 73 (2) 241-251. doi:10.1016/j.jhep.2020.03.032. PMID 32247823 </li>
<li><strong>JOURNAL_INFO</strong> : Journal of hepatology ; J. Hepatol. ; 2020 ; 73 ; 2 ; 241-251 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32247823">32247823</a> </li>
</ul>
<h3 id="persyn-e-32358547">Persyn E-32358547</h3>
<ul>
<li><strong>NAME</strong> : Persyn E-32358547 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genome-wide association study of MRI markers of cerebral small vessel disease in 42,310 participants </li>
<li><strong>DOI</strong> : 10.1038/s41467-020-15932-3 </li>
<li><strong>ABSTRACT</strong> : Cerebral small vessel disease is a major cause of stroke and dementia, but its genetic basis is incompletely understood. We perform a genetic study of three MRI markers of the disease in UK Biobank imaging data and other sources: white matter hyperintensities (N = 42,310), fractional anisotropy (N = 17,663) and mean diffusivity (N = 17,467). Our aim is to better understand the disease pathophysiology. Across the three traits, we identify 31 loci, of which 21 were previously unreported. We perform a transcriptome-wide association study to identify associations with gene expression in relevant tissues, identifying 66 associated genes across the three traits. This genetic study provides insights into the understanding of the biological mechanisms underlying small vessel disease. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Persyn E, Hanscombe KB, Howson JMM, Lewis CM, ...&amp;, Markus HS. (2020) Genome-wide association study of MRI markers of cerebral small vessel disease in 42,310 participants Nat. Commun., 11 (1) 2175. doi:10.1038/s41467-020-15932-3. PMID 32358547 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2020 ; 11 ; 1 ; 2175 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32358547">32358547</a> </li>
</ul>
<h3 id="pirruccello-jp-32382064">Pirruccello JP-32382064</h3>
<ul>
<li><strong>NAME</strong> : Pirruccello JP-32382064 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Analysis of cardiac magnetic resonance imaging in 36,000 individuals yields genetic insights into dilated cardiomyopathy </li>
<li><strong>DOI</strong> : 10.1038/s41467-020-15823-7 </li>
<li><strong>ABSTRACT</strong> : Dilated cardiomyopathy (DCM) is an important cause of heart failure and the leading indication for heart transplantation. Many rare genetic variants have been associated with DCM, but common variant studies of the disease have yielded few associated loci. As structural changes in the heart are a defining feature of DCM, we report a genome-wide association study of cardiac magnetic resonance imaging (MRI)-derived left ventricular measurements in 36,041 UK Biobank participants, with replication in 2184 participants from the Multi-Ethnic Study of Atherosclerosis. We identify 45 previously unreported loci associated with cardiac structure and function, many near well-established genes for Mendelian cardiomyopathies. A polygenic score of MRI-derived left ventricular end systolic volume strongly associates with incident DCM in the general population. Even among carriers of TTN truncating mutations, this polygenic score influences the size and function of the human heart. These results further implicate common genetic polymorphisms in the pathogenesis of DCM. </li>
<li><strong>CITATION</strong> : Pirruccello JP, Bick A, Wang M, Chaffin M, ...&amp;, Aragam KG. (2020) Analysis of cardiac magnetic resonance imaging in 36,000 individuals yields genetic insights into dilated cardiomyopathy Nat. Commun., 11 (1) 2254. doi:10.1038/s41467-020-15823-7. PMID 32382064 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2020 ; 11 ; 1 ; 2254 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32382064">32382064</a> </li>
</ul>
<h3 id="rakowski-a">Rakowski A</h3>
<ul>
<li><strong>NAME</strong> : Rakowski A </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.06.11.24308721 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>CITATION</strong> : Rakowski, A., Monti, R. &amp; Lippert, C. TransferGWAS of T1-weighted brain MRI data from the UK Biobank. bioRxiv 2024.06.11.24308721 (2024) doi:10.1101/2024.06.11.24308721. </li>
</ul>
<h3 id="shah-m-37604819">Shah M-37604819</h3>
<ul>
<li><strong>NAME</strong> : Shah M-37604819 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Environmental and genetic predictors of human cardiovascular ageing </li>
<li><strong>DOI</strong> : 10.1038/s41467-023-40566-6 </li>
<li><strong>ABSTRACT</strong> : Cardiovascular ageing is a process that begins early in life and leads to a progressive change in structure and decline in function due to accumulated damage across diverse cell types, tissues and organs contributing to multi-morbidity. Damaging biophysical, metabolic and immunological factors exceed endogenous repair mechanisms resulting in a pro-fibrotic state, cellular senescence and end-organ damage, however the genetic architecture of cardiovascular ageing is not known. Here we use machine learning approaches to quantify cardiovascular age from image-derived traits of vascular function, cardiac motion and myocardial fibrosis, as well as conduction traits from electrocardiograms, in 39,559 participants of UK Biobank. Cardiovascular ageing is found to be significantly associated with common or rare variants in genes regulating sarcomere homeostasis, myocardial immunomodulation, and tissue responses to biophysical stress. Ageing is accelerated by cardiometabolic risk factors and we also identify prescribed medications that are potential modifiers of ageing. Through large-scale modelling of ageing across multiple traits our results reveal insights into the mechanisms driving premature cardiovascular ageing and reveal potential molecular targets to attenuate age-related processes. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Shah M, de A Inácio MH, Lu C, Schiratti PR, ...&amp;, O'Regan DP. (2023) Environmental and genetic predictors of human cardiovascular ageing Nat. Commun., 14 (1) 4941. doi:10.1038/s41467-023-40566-6. PMID 37604819 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2023 ; 14 ; 1 ; 4941 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37604819">37604819</a> </li>
</ul>
<h3 id="smith-sm-33875891">Smith SM-33875891</h3>
<ul>
<li><strong>NAME</strong> : Smith SM-33875891 </li>
<li><strong>DESCRIPTION</strong> : Oxford Brain Imaging Genetics (BIG40) </li>
<li><strong>URL</strong> : <a href="https://open.win.ox.ac.uk/ukbiobank/big40/pheweb33k/">https://open.win.ox.ac.uk/ukbiobank/big40/pheweb33k/</a> </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank </li>
<li><strong>DOI</strong> : 10.1038/s41593-021-00826-4 </li>
<li><strong>ABSTRACT</strong> : UK Biobank is a major prospective epidemiological study, including multimodal brain imaging, genetics and ongoing health outcomes. Previously, we published genome-wide associations of 3,144 brain imaging-derived phenotypes, with a discovery sample of 8,428 individuals. Here we present a new open resource of genome-wide association study summary statistics, using the 2020 data release, almost tripling the discovery sample size. We now include the X chromosome and new classes of imaging-derived phenotypes (subcortical volumes and tissue contrast). Previously, we found 148 replicated clusters of associations between genetic variants and imaging phenotypes; in this study, we found 692, including 12 on the X chromosome. We describe some of the newly found associations, focusing on the X chromosome and autosomal associations involving the new classes of imaging-derived phenotypes. Our novel associations implicate, for example, pathways involved in the rare X-linked STAR (syndactyly, telecanthus and anogenital and renal malformations) syndrome, Alzheimer's disease and mitochondrial disorders. </li>
<li><strong>CITATION</strong> : Smith SM, Douaud G, Chen W, Hanayik T, ...&amp;, Elliott LT. (2021) An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank Nat. Neurosci., 24 (5) 737-745. doi:10.1038/s41593-021-00826-4. PMID 33875891 </li>
<li><strong>JOURNAL_INFO</strong> : Nature neuroscience ; Nat. Neurosci. ; 2021 ; 24 ; 5 ; 737-745 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33875891">33875891</a> </li>
</ul>
<h3 id="sun-bb-36241887">Sun BB-36241887</h3>
<ul>
<li><strong>NAME</strong> : Sun BB-36241887 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genetic map of regional sulcal morphology in the human brain from UK biobank data </li>
<li><strong>DOI</strong> : 10.1038/s41467-022-33829-1 </li>
<li><strong>ABSTRACT</strong> : Genetic associations with macroscopic brain structure can provide insights into brain function and disease. However, specific associations with measures of local brain folding are largely under-explored. Here, we conducted large-scale genome- and exome-wide associations of regional cortical sulcal measures derived from magnetic resonance imaging scans of 40,169 individuals in UK Biobank. We discovered 388 regional brain folding associations across 77 genetic loci, with genes in associated loci enriched for expression in the cerebral cortex, neuronal development processes, and differential regulation during early brain development. We integrated brain eQTLs to refine genes for various loci, implicated several genes involved in neurodevelopmental disorders, and highlighted global genetic correlations with neuropsychiatric phenotypes. We provide an interactive 3D visualisation of our summary associations, emphasising added resolution of regional analyses. Our results offer new insights into the genetic architecture of brain folding and provide a resource for future studies of sulcal morphology in health and disease. </li>
<li><strong>CITATION</strong> : Sun BB, Loomis SJ, Pizzagalli F, Shatokhina N, ...&amp;, Whelan CD. (2022) Genetic map of regional sulcal morphology in the human brain from UK biobank data Nat. Commun., 13 (1) 6071. doi:10.1038/s41467-022-33829-1. PMID 36241887 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2022 ; 13 ; 1 ; 6071 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36241887">36241887</a> </li>
</ul>
<h3 id="wang-c-35606419">Wang C-35606419</h3>
<ul>
<li><strong>NAME</strong> : Wang C-35606419 </li>
<li><strong>DESCRIPTION</strong> : Quantitative susceptibility mapping </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Phenotypic and genetic associations of quantitative magnetic susceptibility in UK Biobank brain imaging </li>
<li><strong>DOI</strong> : 10.1038/s41593-022-01074-w </li>
<li><strong>ABSTRACT</strong> : A key aim in epidemiological neuroscience is identification of markers to assess brain health and monitor therapeutic interventions. Quantitative susceptibility mapping (QSM) is an emerging magnetic resonance imaging technique that measures tissue magnetic susceptibility and has been shown to detect pathological changes in tissue iron, myelin and calcification. We present an open resource of QSM-based imaging measures of multiple brain structures in 35,273 individuals from the UK Biobank prospective epidemiological study. We identify statistically significant associations of 251 phenotypes with magnetic susceptibility that include body iron, disease, diet and alcohol consumption. Genome-wide associations relate magnetic susceptibility to 76 replicating clusters of genetic variants with biological functions involving iron, calcium, myelin and extracellular matrix. These patterns of associations include relationships that are unique to QSM, in particular being complementary to T2* signal decay time measures. These new imaging phenotypes are being integrated into the core UK Biobank measures provided to researchers worldwide, creating the potential to discover new, non-invasive markers of brain health. </li>
<li><strong>CITATION</strong> : Wang C, Martins-Bach AB, Alfaro-Almagro F, Douaud G, ...&amp;, Miller KL. (2022) Phenotypic and genetic associations of quantitative magnetic susceptibility in UK Biobank brain imaging Nat. Neurosci., 25 (6) 818-831. doi:10.1038/s41593-022-01074-w. PMID 35606419 </li>
<li><strong>JOURNAL_INFO</strong> : Nature neuroscience ; Nat. Neurosci. ; 2022 ; 25 ; 6 ; 818-831 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35606419">35606419</a> </li>
</ul>
<h3 id="warrier-v-37592024">Warrier V-37592024</h3>
<ul>
<li><strong>NAME</strong> : Warrier V-37592024 </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : Genetic insights into human cortical organization and development through genome-wide analyses of 2,347 neuroimaging phenotypes </li>
<li><strong>DOI</strong> : 10.1038/s41588-023-01475-y </li>
<li><strong>ABSTRACT</strong> : Our understanding of the genetics of the human cerebral cortex is limited both in terms of the diversity and the anatomical granularity of brain structural phenotypes. Here we conducted a genome-wide association meta-analysis of 13 structural and diffusion magnetic resonance imaging-derived cortical phenotypes, measured globally and at 180 bilaterally averaged regions in 36,663 individuals and identified 4,349 experiment-wide significant loci. These phenotypes include cortical thickness, surface area, gray matter volume, measures of folding, neurite density and water diffusion. We identified four genetic latent structures and causal relationships between surface area and some measures of cortical folding. These latent structures partly relate to different underlying gene expression trajectories during development and are enriched for different cell types. We also identified differential enrichment for neurodevelopmental and constrained genes and demonstrate that common genetic variants associated with cortical expansion are associated with cephalic disorders. Finally, we identified complex interphenotype and inter-regional genetic relationships among the 13 phenotypes, reflecting the developmental differences among them. Together, these analyses identify distinct genetic organizational principles of the cortex and their correlates with neurodevelopment. </li>
<li><strong>CITATION</strong> : Warrier V, Stauffer EM, Huang QQ, Wigdor EM, ...&amp;, Bethlehem RAI. (2023) Genetic insights into human cortical organization and development through genome-wide analyses of 2,347 neuroimaging phenotypes Nat. Genet., 55 (9) 1483-1493. doi:10.1038/s41588-023-01475-y. PMID 37592024 </li>
<li><strong>JOURNAL_INFO</strong> : Nature genetics ; Nat. Genet. ; 2023 ; 55 ; 9 ; 1483-1493 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37592024">37592024</a> </li>
</ul>
<h3 id="yu-s">Yu S</h3>
<ul>
<li><strong>NAME</strong> : Yu S </li>
<li><strong>PREPRINT_DOI</strong> : 10.1101/2024.01.11.575251 </li>
<li><strong>SERVER</strong> : biorxiv </li>
<li><strong>CITATION</strong> : Yu, S. et al. A novel classification framework for genome-wide association study of whole brain MRI images using deep learning. bioRxiv 2024.01.11.575251 (2024) doi:10.1101/2024.01.11.575251. </li>
</ul>
<h2 id="method">Method</h2>
<h3 id="kirchler-m-35640976">Kirchler M-35640976</h3>
<ul>
<li><strong>NAME</strong> : Kirchler M-35640976 </li>
<li><strong>SHORT NAME</strong> : transferGWAS </li>
<li><strong>DESCRIPTION</strong> : transferGWAS is a method for performing genome-wide association studies on whole images. </li>
<li><strong>URL</strong> : <a href="https://github.com/mkirchler/transferGWAS/">https://github.com/mkirchler/transferGWAS/</a> </li>
<li><strong>TITLE</strong> : transferGWAS: GWAS of images using deep transfer learning </li>
<li><strong>DOI</strong> : 10.1093/bioinformatics/btac369 </li>
<li><strong>ABSTRACT</strong> : MOTIVATION: Medical images can provide rich information about diseases and their biology. However, investigating their association with genetic variation requires non-standard methods. We propose transferGWAS, a novel approach to perform genome-wide association studies directly on full medical images. First, we learn semantically meaningful representations of the images based on a transfer learning task, during which a deep neural network is trained on independent but similar data. Then, we perform genetic association tests with these representations. RESULTS: We validate the type I error rates and power of transferGWAS in simulation studies of synthetic images. Then we apply transferGWAS in a genome-wide association study of retinal fundus images from the UK Biobank. This first-of-a-kind GWAS of full imaging data yielded 60 genomic regions associated with retinal fundus images, of which 7 are novel candidate loci for eye-related traits and diseases. AVAILABILITY AND IMPLEMENTATION: Our method is implemented in Python and available at https://github.com/mkirchler/transferGWAS/. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online. </li>
<li><strong>CITATION</strong> : Kirchler M, Konigorski S, Norden M, Meltendorf C, ...&amp;, Lippert C. (2022) transferGWAS: GWAS of images using deep transfer learning Bioinformatics, 38 (14) 3621-3628. doi:10.1093/bioinformatics/btac369. PMID 35640976 </li>
<li><strong>JOURNAL_INFO</strong> : Bioinformatics  ; Bioinformatics ; 2022 ; 38 ; 14 ; 3621-3628 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35640976">35640976</a> </li>
</ul>
<h3 id="xu-z-28736311">Xu Z-28736311</h3>
<ul>
<li><strong>NAME</strong> : Xu Z-28736311 </li>
<li><strong>TITLE</strong> : Imaging-wide association study: Integrating imaging endophenotypes in GWAS </li>
<li><strong>DOI</strong> : 10.1016/j.neuroimage.2017.07.036 </li>
<li><strong>ABSTRACT</strong> : A new and powerful approach, called imaging-wide association study (IWAS), is proposed to integrate imaging endophenotypes with GWAS to boost statistical power and enhance biological interpretation for GWAS discoveries. IWAS extends the promising transcriptome-wide association study (TWAS) from using gene expression endophenotypes to using imaging and other endophenotypes with a much wider range of possible applications. As illustration, we use gray-matter volumes of several brain regions of interest (ROIs) drawn from the ADNI-1 structural MRI data as imaging endophenotypes, which are then applied to the individual-level GWAS data of ADNI-GO/2 and a large meta-analyzed GWAS summary statistics dataset (based on about 74,000 individuals), uncovering some novel genes significantly associated with Alzheimer's disease (AD). We also compare the performance of IWAS with TWAS, showing much larger numbers of significant AD-associated genes discovered by IWAS, presumably due to the stronger link between brain atrophy and AD than that between gene expression of normal individuals and the risk for AD. The proposed IWAS is general and can be applied to other imaging endophenotypes, and GWAS individual-level or summary association data. </li>
<li><strong>CITATION</strong> : Xu Z, Wu C, Pan W, Alzheimer's Disease Neuroimaging Initiative. (2017) Imaging-wide association study: Integrating imaging endophenotypes in GWAS Neuroimage, 159 () 159-169. doi:10.1016/j.neuroimage.2017.07.036. PMID 28736311 </li>
<li><strong>JOURNAL_INFO</strong> : NeuroImage ; Neuroimage ; 2017 ; 159 ; <NA> ; 159-169 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28736311">28736311</a> </li>
</ul>
<h2 id="prediction">Prediction</h2>
<h3 id="huang-yj-38762475">Huang YJ-38762475</h3>
<ul>
<li><strong>NAME</strong> : Huang YJ-38762475 </li>
<li><strong>DESCRIPTION</strong> : ABD,
carotid artery ultrasonography (CAU), BMD, ECG, and thyroid ultra- sonography (TU) : 28 ABD features, 29 CAU features, 85 BMD features, and 10 ECG features </li>
<li><strong>MAIN_ANCESTRY</strong> : EAS </li>
<li><strong>RELATED_BIOBANK</strong> : TWB </li>
<li><strong>TITLE</strong> : AI-enhanced integration of genetic and medical imaging data for risk assessment of Type 2 diabetes </li>
<li><strong>DOI</strong> : 10.1038/s41467-024-48618-1 </li>
<li><strong>ABSTRACT</strong> : Type 2 diabetes (T2D) presents a formidable global health challenge, highlighted by its escalating prevalence, underscoring the critical need for precision health strategies and early detection initiatives. Leveraging artificial intelligence, particularly eXtreme Gradient Boosting (XGBoost), we devise robust risk assessment models for T2D. Drawing upon comprehensive genetic and medical imaging datasets from 68,911 individuals in the Taiwan Biobank, our models integrate Polygenic Risk Scores (PRS), Multi-image Risk Scores (MRS), and demographic variables, such as age, sex, and T2D family history. Here, we show that our model achieves an Area Under the Receiver Operating Curve (AUC) of 0.94, effectively identifying high-risk T2D subgroups. A streamlined model featuring eight key variables also maintains a high AUC of 0.939. This high accuracy for T2D risk assessment promises to catalyze early detection and preventive strategies. Moreover, we introduce an accessible online risk assessment tool for T2D, facilitating broader applicability and dissemination of our findings. </li>
<li><strong>COPYRIGHT</strong> : https://creativecommons.org/licenses/by/4.0 </li>
<li><strong>CITATION</strong> : Huang YJ, Chen CH, Yang HC. (2024) AI-enhanced integration of genetic and medical imaging data for risk assessment of Type 2 diabetes Nat. Commun., 15 (1) 4230. doi:10.1038/s41467-024-48618-1. PMID 38762475 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2024 ; 15 ; 1 ; 4230 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/38762475">38762475</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="littlejohns-tj-32457287">Littlejohns TJ-32457287</h3>
<ul>
<li><strong>NAME</strong> : Littlejohns TJ-32457287 </li>
<li><strong>DESCRIPTION</strong> : brain, cardiac and abdominal magnetic resonance imaging, dual-energy X-ray absorptiometry and carotid ultrasound </li>
<li><strong>MAIN_ANCESTRY</strong> : EUR </li>
<li><strong>RELATED_BIOBANK</strong> : UKB </li>
<li><strong>TITLE</strong> : The UK Biobank imaging enhancement of 100,000 participants: rationale, data collection, management and future directions </li>
<li><strong>DOI</strong> : 10.1038/s41467-020-15948-9 </li>
<li><strong>ABSTRACT</strong> : UK Biobank is a population-based cohort of half a million participants aged 40-69 years recruited between 2006 and 2010. In 2014, UK Biobank started the world's largest multi-modal imaging study, with the aim of re-inviting 100,000 participants to undergo brain, cardiac and abdominal magnetic resonance imaging, dual-energy X-ray absorptiometry and carotid ultrasound. The combination of large-scale multi-modal imaging with extensive phenotypic and genetic data offers an unprecedented resource for scientists to conduct health-related research. This article provides an in-depth overview of the imaging enhancement, including the data collected, how it is managed and processed, and future directions. </li>
<li><strong>CITATION</strong> : Littlejohns TJ, Holliday J, Gibson LM, Garratt S, ...&amp;, Allen NE. (2020) The UK Biobank imaging enhancement of 100,000 participants: rationale, data collection, management and future directions Nat. Commun., 11 (1) 2624. doi:10.1038/s41467-020-15948-9. PMID 32457287 </li>
<li><strong>JOURNAL_INFO</strong> : Nature communications ; Nat. Commun. ; 2020 ; 11 ; 1 ; 2624 </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32457287">32457287</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.13a4f30d.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>