
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Annotation_README/">
      
      
        <link rel="next" href="../Tools_Association_tests_RWAS_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Association_tests - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Association_tests
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Metabolomics_Metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Proteomics_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Imaging_Imaging_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imaging
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_3" checked>
        
          
          <label class="md-nav__link" for="__nav_3_3" id="__nav_3_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_3_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#case-case-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      Case-case GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Case-case GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cc-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      CC-GWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas-of-longitudinal-trajectories" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS of longitudinal trajectories
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS of longitudinal trajectories">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#trajgwas" class="md-nav__link">
    <span class="md-ellipsis">
      TrajGWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas-using-family-history" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS using family history
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS using family history">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwax" class="md-nav__link">
    <span class="md-ellipsis">
      GWAX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lt-fh" class="md-nav__link">
    <span class="md-ellipsis">
      LT-FH
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#siblinggwas" class="md-nav__link">
    <span class="md-ellipsis">
      SiblingGWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snipar" class="md-nav__link">
    <span class="md-ellipsis">
      snipar
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gene-based-analysis-rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Gene-based analysis (rare variant)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Gene-based analysis (rare variant)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#regenie" class="md-nav__link">
    <span class="md-ellipsis">
      REGENIE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige-gene" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE-GENE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige-gene_1" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE-GENE+
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skat" class="md-nav__link">
    <span class="md-ellipsis">
      SKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skat-o" class="md-nav__link">
    <span class="md-ellipsis">
      SKAT-O
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#staar" class="md-nav__link">
    <span class="md-ellipsis">
      STAAR
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gene-based-analysis-sumstats" class="md-nav__link">
    <span class="md-ellipsis">
      Gene-based analysis (sumstats)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Gene-based analysis (sumstats)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ldak-gbat" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK-GBAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgs-adjusted-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="PGS-adjusted GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#pgs-adjusted-gwas_1" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgs-adjusted-rvats" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted RVATs
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-povysil" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Povysil
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#single-variant-association-tests" class="md-nav__link">
    <span class="md-ellipsis">
      Single variant association tests
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Single variant association tests">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bolt-lmm" class="md-nav__link">
    <span class="md-ellipsis">
      BOLT-lMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emmax" class="md-nav__link">
    <span class="md-ellipsis">
      EMMAX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gemma" class="md-nav__link">
    <span class="md-ellipsis">
      GEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak-kvik" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK-KVIK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#polmm" class="md-nav__link">
    <span class="md-ellipsis">
      POLMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qrgwas" class="md-nav__link">
    <span class="md-ellipsis">
      QRGWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#regenie_1" class="md-nav__link">
    <span class="md-ellipsis">
      REGENIE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastgwa" class="md-nav__link">
    <span class="md-ellipsis">
      fastGWA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastgwa-glmm" class="md-nav__link">
    <span class="md-ellipsis">
      fastGWA-GLMM
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Polygenic_risk_scores_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#case-case-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      Case-case GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Case-case GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#cc-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      CC-GWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas-of-longitudinal-trajectories" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS of longitudinal trajectories
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS of longitudinal trajectories">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#trajgwas" class="md-nav__link">
    <span class="md-ellipsis">
      TrajGWAS
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas-using-family-history" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS using family history
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS using family history">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwax" class="md-nav__link">
    <span class="md-ellipsis">
      GWAX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lt-fh" class="md-nav__link">
    <span class="md-ellipsis">
      LT-FH
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#siblinggwas" class="md-nav__link">
    <span class="md-ellipsis">
      SiblingGWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#snipar" class="md-nav__link">
    <span class="md-ellipsis">
      snipar
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gene-based-analysis-rare-variant" class="md-nav__link">
    <span class="md-ellipsis">
      Gene-based analysis (rare variant)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Gene-based analysis (rare variant)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#regenie" class="md-nav__link">
    <span class="md-ellipsis">
      REGENIE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige-gene" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE-GENE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige-gene_1" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE-GENE+
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skat" class="md-nav__link">
    <span class="md-ellipsis">
      SKAT
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#skat-o" class="md-nav__link">
    <span class="md-ellipsis">
      SKAT-O
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#staar" class="md-nav__link">
    <span class="md-ellipsis">
      STAAR
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gene-based-analysis-sumstats" class="md-nav__link">
    <span class="md-ellipsis">
      Gene-based analysis (sumstats)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Gene-based analysis (sumstats)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#ldak-gbat" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK-GBAT
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgs-adjusted-gwas" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted GWAS
    </span>
  </a>
  
    <nav class="md-nav" aria-label="PGS-adjusted GWAS">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#pgs-adjusted-gwas_1" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted GWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pgs-adjusted-rvats" class="md-nav__link">
    <span class="md-ellipsis">
      PGS-adjusted RVATs
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review" class="md-nav__link">
    <span class="md-ellipsis">
      Review
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Review">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#review-povysil" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Povysil
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#single-variant-association-tests" class="md-nav__link">
    <span class="md-ellipsis">
      Single variant association tests
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Single variant association tests">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#bolt-lmm" class="md-nav__link">
    <span class="md-ellipsis">
      BOLT-lMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#emmax" class="md-nav__link">
    <span class="md-ellipsis">
      EMMAX
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gemma" class="md-nav__link">
    <span class="md-ellipsis">
      GEMMA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ldak-kvik" class="md-nav__link">
    <span class="md-ellipsis">
      LDAK-KVIK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#polmm" class="md-nav__link">
    <span class="md-ellipsis">
      POLMM
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#qrgwas" class="md-nav__link">
    <span class="md-ellipsis">
      QRGWAS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#regenie_1" class="md-nav__link">
    <span class="md-ellipsis">
      REGENIE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#saige" class="md-nav__link">
    <span class="md-ellipsis">
      SAIGE
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastgwa" class="md-nav__link">
    <span class="md-ellipsis">
      fastGWA
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#fastgwa-glmm" class="md-nav__link">
    <span class="md-ellipsis">
      fastGWA-GLMM
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Association_tests</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CATEGORY</th>
<th style="text-align: left;">CITATION</th>
<th style="text-align: left;">YEAR</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#cc-gwas">CC-GWAS</a></td>
<td style="text-align: left;">Case-case GWAS</td>
<td style="text-align: left;">Peyrot, W. J., &amp; Price, A. L. (2021). Identifying loci with different allele frequencies among cases of eight psychiatric disorders using CC-GWAS. Nature genetics, 53(4), 445-454.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#trajgwas">TrajGWAS</a></td>
<td style="text-align: left;">GWAS of longitudinal trajectories</td>
<td style="text-align: left;">Ko, S., German, C. A., Jensen, A., Shen, J., Wang, A., Mehrotra, D. V., ... &amp; Zhou, J. J. (2022). GWAS of longitudinal trajectories at biobank scale. The American Journal of Human Genetics, 109(3), 433-445.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gwax">GWAX</a></td>
<td style="text-align: left;">GWAS using family history</td>
<td style="text-align: left;">Liu, J. Z., Erlich, Y., &amp; Pickrell, J. K. (2017). Case–control association mapping by proxy using family history of disease. Nature genetics, 49(3), 325-331.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lt-fh">LT-FH</a></td>
<td style="text-align: left;">GWAS using family history</td>
<td style="text-align: left;">Hujoel, M. L., Gazal, S., Loh, P. R., Patterson, N., &amp; Price, A. L. (2020). Liability threshold modeling of case–control status and family history of disease increases association power. Nature genetics, 52(5), 541-547.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#siblinggwas">SiblingGWAS</a></td>
<td style="text-align: left;">GWAS using family history</td>
<td style="text-align: left;">Howe, L. J., Nivard, M. G., Morris, T. T., Hansen, A. F., Rasheed, H., Cho, Y., ... &amp; Davies, N. M. (2022). Within-sibship genome-wide association analyses decrease bias in estimates of direct genetic effects. Nature genetics, 54(5), 581-592.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#snipar">snipar</a></td>
<td style="text-align: left;">GWAS using family history</td>
<td style="text-align: left;">Young, A. I., Nehzati, S. M., Benonisdottir, S., Okbay, A., Jayashankar, H., Lee, C., ... &amp; Kong, A. (2022). Mendelian imputation of parental genotypes improves estimates of direct genetic effects. Nature genetics, 54(6), 897-905.<br><br>Guan, J., Nehzati, S. M., Benjamin, D. J., &amp; Young, A. I. (2022). Novel estimators for family-based genome-wide association studies increase power and robustness. bioRxiv, 2022-10.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#regenie">REGENIE</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Mbatchou, Joelle, et al. "Computationally efficient whole-genome regression for quantitative and binary traits." Nature genetics 53.7 (2021): 1097-1103.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#saige-gene">SAIGE-GENE+</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Zhou, Wei, et al. "SAIGE-GENE+ improves the efficiency and accuracy of set-based rare variant association tests." Nature Genetics (2022): 1-4.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#saige-gene">SAIGE-GENE</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Zhou, Wei, et al. "Scalable generalized linear mixed model for region-based association tests in large biobanks and cohorts." Nature genetics 52.6 (2020): 634-639.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#skat-o">SKAT-O</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Lee, Seunggeun, Michael C. Wu, and Xihong Lin. "Optimal tests for rare variant effects in sequencing association studies." Biostatistics 13.4 (2012): 762-775.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#skat">SKAT</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Wu, Michael C., et al. "Rare-variant association testing for sequencing data with the sequence kernel association test." The American Journal of Human Genetics 89.1 (2011): 82-93.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#staar">STAAR</a></td>
<td style="text-align: left;">Gene-based analysis (rare variant)</td>
<td style="text-align: left;">Li, Xihao, et al. "Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale." Nature genetics 52.9 (2020): 969-983.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldak-gbat">LDAK-GBAT</a></td>
<td style="text-align: left;">Gene-based analysis (sumstats)</td>
<td style="text-align: left;"><br><br><br><br><br><br>Berrandou, T.-E., Balding, D. &amp; Speed, D. LDAK-GBAT: Fast and powerful gene-based association testing using summary statistics. Am. J. Hum. Genet. 110, 23–29 (2023).</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgs-adjusted-gwas">PGS-adjusted GWAS</a></td>
<td style="text-align: left;">PGS-adjusted GWAS</td>
<td style="text-align: left;">Campos, A. I., Namba, S., Lin, S. C., Nam, K., Sidorenko, J., Wang, H., ... &amp; Yengo, L. (2023). Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores. Nature Genetics, 1-8.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgs-adjusted-rvats">PGS-adjusted RVATs</a></td>
<td style="text-align: left;">PGS-adjusted GWAS</td>
<td style="text-align: left;">Jurgens, S. J., Pirruccello, J. P., Choi, S. H., Morrill, V. N., Chaffin, M., Lubitz, S. A., ... &amp; Ellinor, P. T. (2023). Adjusting for common variant polygenic scores improves yield in rare variant association analyses. Nature Genetics, 55(4), 544-548.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-povysil">Review-Povysil</a></td>
<td style="text-align: left;">Review</td>
<td style="text-align: left;">Povysil, Gundula, et al. "Rare-variant collapsing analyses for complex traits: guidelines and applications." Nature Reviews Genetics 20.12 (2019): 747-759.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bolt-lmm">BOLT-lMM</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Loh, Po-Ru, et al. "Efficient Bayesian mixed-model analysis increases association power in large cohorts." Nature genetics 47.3 (2015): 284-290.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#emmax">EMMAX</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">NA</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#gemma">GEMMA</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Zhou, Xiang, and Matthew Stephens. "Genome-wide efficient mixed-model analysis for association studies." Nature genetics 44.7 (2012): 821-824.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldak-kvik">LDAK-KVIK</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Hof, J. P. &amp; Speed, D. LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. bioRxiv 2024.07.25.24311005 (2024) doi:10.1101/2024.07.25.24311005.<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#plink2">PLINK2</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Chang, Christopher C., et al. "Second-generation PLINK: rising to the challenge of larger and richer datasets." Gigascience 4.1 (2015): s13742-015.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#plink">PLINK</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Purcell, Shaun, et al. "PLINK: a tool set for whole-genome association and population-based linkage analyses." The American journal of human genetics 81.3 (2007): 559-575.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#polmm">POLMM</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Bi, W., Zhou, W., Dey, R., Mukherjee, B., Sampson, J. N., &amp; Lee, S. (2021). Efficient mixed model approach for large-scale genome-wide association studies of ordinal categorical phenotypes. The American Journal of Human Genetics, 108(5), 825-839.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#qrgwas">QRGWAS</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;"><br><br>Wang, C. et al. Genome-wide discovery for biomarkers using quantile regression at biobank scale. Nat. Commun. 15, 6460 (2024).<br><br></td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#regenie">REGENIE</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Mbatchou, Joelle, et al. "Computationally efficient whole-genome regression for quantitative and binary traits." Nature genetics 53.7 (2021): 1097-1103.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#saige">SAIGE</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Zhou, Wei, et al. "Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies." Nature genetics 50.9 (2018): 1335-1341.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastgwa-glmm">fastGWA-GLMM</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Jiang, Longda, et al. "A generalized linear mixed model association tool for biobank-scale data." Nature genetics 53.11 (2021): 1616-1621.</td>
<td style="text-align: left;">NA</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#fastgwa">fastGWA</a></td>
<td style="text-align: left;">Single variant association tests</td>
<td style="text-align: left;">Jiang, Longda, et al. "A resource-efficient tool for mixed model association analysis of large-scale data." Nature genetics 51.12 (2019): 1749-1755.</td>
<td style="text-align: left;">NA</td>
</tr>
</tbody>
</table>
<h2 id="case-case-gwas">Case-case GWAS</h2>
<h3 id="cc-gwas">CC-GWAS</h3>
<ul>
<li><strong>NAME</strong> : CC-GWAS </li>
<li><strong>SHORT NAME</strong> : CC-GWAS </li>
<li><strong>FULL NAME</strong> : case–case genome-wide association study </li>
<li><strong>DESCRIPTION</strong> : The CCGWAS R package provides a tool for case-case association testing of two different disorders based on their respective case-control GWAS results </li>
<li><strong>URL</strong> : <a href="https://github.com/wouterpeyrot/CCGWAS">https://github.com/wouterpeyrot/CCGWAS</a> </li>
<li><strong>CITATION</strong> : Peyrot, W. J., &amp; Price, A. L. (2021). Identifying loci with different allele frequencies among cases of eight psychiatric disorders using CC-GWAS. Nature genetics, 53(4), 445-454. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33686288">33686288</a> </li>
</ul>
<h2 id="gwas-of-longitudinal-trajectories">GWAS of longitudinal trajectories</h2>
<h3 id="trajgwas">TrajGWAS</h3>
<ul>
<li><strong>NAME</strong> : TrajGWAS </li>
<li><strong>SHORT NAME</strong> : TrajGWAS </li>
<li><strong>FULL NAME</strong> : GWAS of longitudinal trajectories </li>
<li><strong>DESCRIPTION</strong> : TrajGWAS.jl is a Julia package for performing genome-wide association studies (GWAS) for continuous longitudinal phenotypes using a modified linear mixed effects model. It builds upon the within-subject variance estimation by robust regression (WiSER) method and can be used to identify variants associated with changes in the mean and within-subject variability of the longitduinal trait. </li>
<li><strong>URL</strong> : <a href="https://github.com/OpenMendel/TrajGWAS.jl">https://github.com/OpenMendel/TrajGWAS.jl</a> </li>
<li><strong>KEYWORDS</strong> : biomarker trajectories, mean, within-subject (WS) variability, linear mixed effect model, within-subject variance estimation by robust regression (WiSER) method </li>
<li><strong>CITATION</strong> : Ko, S., German, C. A., Jensen, A., Shen, J., Wang, A., Mehrotra, D. V., ... &amp; Zhou, J. J. (2022). GWAS of longitudinal trajectories at biobank scale. The American Journal of Human Genetics, 109(3), 433-445. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35196515">35196515</a> </li>
</ul>
<h2 id="gwas-using-family-history">GWAS using family history</h2>
<h3 id="gwax">GWAX</h3>
<ul>
<li><strong>NAME</strong> : GWAX </li>
<li><strong>SHORT NAME</strong> : GWAX </li>
<li><strong>FULL NAME</strong> : genome-wide association by proxy </li>
<li><strong>DESCRIPTION</strong> : In randomly ascertained cohorts, replacing cases with their first-degree relatives enables studies of diseases that are absent (or nearly absent) in the cohort. </li>
<li><strong>CITATION</strong> : Liu, J. Z., Erlich, Y., &amp; Pickrell, J. K. (2017). Case–control association mapping by proxy using family history of disease. Nature genetics, 49(3), 325-331. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/28092683">28092683</a> </li>
</ul>
<h3 id="lt-fh">LT-FH</h3>
<ul>
<li><strong>NAME</strong> : LT-FH </li>
<li><strong>SHORT NAME</strong> : LT-FH </li>
<li><strong>FULL NAME</strong> : liability threshold model, conditional on case–control status and family history </li>
<li><strong>DESCRIPTION</strong> : an association method based on posterior mean genetic liabilities under a liability threshold model, conditional on case-control status and family history (LT-FH) </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/UKBB/LTFH/">https://alkesgroup.broadinstitute.org/UKBB/LTFH/</a> </li>
<li><strong>CITATION</strong> : Hujoel, M. L., Gazal, S., Loh, P. R., Patterson, N., &amp; Price, A. L. (2020). Liability threshold modeling of case–control status and family history of disease increases association power. Nature genetics, 52(5), 541-547. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32313248">32313248</a> </li>
</ul>
<h3 id="siblinggwas">SiblingGWAS</h3>
<ul>
<li><strong>NAME</strong> : SiblingGWAS </li>
<li><strong>SHORT NAME</strong> : SiblingGWAS </li>
<li><strong>FULL NAME</strong> : Within-sibship genome-wide association analyses </li>
<li><strong>DESCRIPTION</strong> : Scripts for running GWAS using siblings to estimate Within-Family (WF) and Between-Family (BF) effects of genetic variants on continuous traits. Allows the inclusion of more than two siblings from one family. </li>
<li><strong>URL</strong> : <a href="https://github.com/LaurenceHowe/SiblingGWAS">https://github.com/LaurenceHowe/SiblingGWAS</a> </li>
<li><strong>CITATION</strong> : Howe, L. J., Nivard, M. G., Morris, T. T., Hansen, A. F., Rasheed, H., Cho, Y., ... &amp; Davies, N. M. (2022). Within-sibship genome-wide association analyses decrease bias in estimates of direct genetic effects. Nature genetics, 54(5), 581-592. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35534559">35534559</a> </li>
</ul>
<h3 id="snipar">snipar</h3>
<ul>
<li><strong>NAME</strong> : snipar </li>
<li><strong>SHORT NAME</strong> : snipar </li>
<li><strong>FULL NAME</strong> : single nucleotide imputation of parents </li>
<li><strong>DESCRIPTION</strong> : snipar (single nucleotide imputation of parents) is a Python package for inferring identity-by-descent (IBD) segments shared between siblings, imputing missing parental genotypes, and for performing family based genome-wide association and polygenic score analyses using observed and/or imputed parental genotypes. </li>
<li><strong>URL</strong> : <a href="https://github.com/AlexTISYoung/snipar">https://github.com/AlexTISYoung/snipar</a> </li>
<li><strong>CITATION</strong> : Young, A. I., Nehzati, S. M., Benonisdottir, S., Okbay, A., Jayashankar, H., Lee, C., ... &amp; Kong, A. (2022). Mendelian imputation of parental genotypes improves estimates of direct genetic effects. Nature genetics, 54(6), 897-905.<br><br>Guan, J., Nehzati, S. M., Benjamin, D. J., &amp; Young, A. I. (2022). Novel estimators for family-based genome-wide association studies increase power and robustness. bioRxiv, 2022-10. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/35681053">35681053</a> </li>
</ul>
<h2 id="gene-based-analysis-rare-variant">Gene-based analysis (rare variant)</h2>
<h3 id="regenie">REGENIE</h3>
<ul>
<li><strong>NAME</strong> : REGENIE </li>
<li><strong>SHORT NAME</strong> : REGENIE </li>
<li><strong>FULL NAME</strong> : REGENIE </li>
<li><strong>DESCRIPTION</strong> : regenie is a C++ program for whole genome regression modelling of large genome-wide association studies. It is developed and supported by a team of scientists at the Regeneron Genetics Center. </li>
<li><strong>URL</strong> : <a href="https://github.com/rgcgithub/regenie">https://github.com/rgcgithub/regenie</a> </li>
<li><strong>KEYWORDS</strong> : whole genome regression </li>
<li><strong>CITATION</strong> : Mbatchou, Joelle, et al. "Computationally efficient whole-genome regression for quantitative and binary traits." Nature genetics 53.7 (2021): 1097-1103. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34017140">34017140</a> </li>
</ul>
<h3 id="saige-gene">SAIGE-GENE</h3>
<ul>
<li><strong>NAME</strong> : SAIGE-GENE </li>
<li><strong>SHORT NAME</strong> : SAIGE-GENE </li>
<li><strong>FULL NAME</strong> : SAIGE-GENE </li>
<li><strong>URL</strong> : <a href="https://github.com/weizhouUMICH/SAIGE">https://github.com/weizhouUMICH/SAIGE</a> </li>
<li><strong>CITATION</strong> : Zhou, Wei, et al. "Scalable generalized linear mixed model for region-based association tests in large biobanks and cohorts." Nature genetics 52.6 (2020): 634-639. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32424355">32424355</a> </li>
</ul>
<h3 id="saige-gene_1">SAIGE-GENE+</h3>
<ul>
<li><strong>NAME</strong> : SAIGE-GENE+ </li>
<li><strong>SHORT NAME</strong> : SAIGE-GENE+ </li>
<li><strong>FULL NAME</strong> : SAIGE-GENE+ </li>
<li><strong>URL</strong> : <a href="https://github.com/weizhouUMICH/SAIGE">https://github.com/weizhouUMICH/SAIGE</a> </li>
<li><strong>CITATION</strong> : Zhou, Wei, et al. "SAIGE-GENE+ improves the efficiency and accuracy of set-based rare variant association tests." Nature Genetics (2022): 1-4. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36138231">36138231</a> </li>
</ul>
<h3 id="skat">SKAT</h3>
<ul>
<li><strong>NAME</strong> : SKAT </li>
<li><strong>SHORT NAME</strong> : SKAT </li>
<li><strong>FULL NAME</strong> : sequence kernel association test </li>
<li><strong>DESCRIPTION</strong> : SKAT is a SNP-set (e.g., a gene or a region) level test for association between a set of rare (or common) variants and dichotomous or quantitative phenotypes, SKAT aggregates individual score test statistics of SNPs in a SNP set  and efficiently computes SNP-set level p-values, e.g. a gene or a region level p-value, while adjusting for covariates, such as principal components to account for population stratification. SKAT also allows for power/sample size calculations for designing for sequence association studies. </li>
<li><strong>URL</strong> : <a href="https://www.hsph.harvard.edu/skat/">https://www.hsph.harvard.edu/skat/</a> </li>
<li><strong>CITATION</strong> : Wu, Michael C., et al. "Rare-variant association testing for sequencing data with the sequence kernel association test." The American Journal of Human Genetics 89.1 (2011): 82-93. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/21737059">21737059</a> </li>
</ul>
<h3 id="skat-o">SKAT-O</h3>
<ul>
<li><strong>NAME</strong> : SKAT-O </li>
<li><strong>SHORT NAME</strong> : SKAT-O </li>
<li><strong>FULL NAME</strong> : sequence kernel association test - optimal test </li>
<li><strong>DESCRIPTION</strong> : estimating the correlation parameter in the kernel matrix to maximize the power, which corresponds to the estimated weight in the linear combination of the burden test and SKAT test statistics that maximizes power. </li>
<li><strong>URL</strong> : <a href="https://www.hsph.harvard.edu/skat/">https://www.hsph.harvard.edu/skat/</a> </li>
<li><strong>CITATION</strong> : Lee, Seunggeun, Michael C. Wu, and Xihong Lin. "Optimal tests for rare variant effects in sequencing association studies." Biostatistics 13.4 (2012): 762-775. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22699862">22699862</a> </li>
</ul>
<h3 id="staar">STAAR</h3>
<ul>
<li><strong>NAME</strong> : STAAR </li>
<li><strong>SHORT NAME</strong> : STAAR </li>
<li><strong>FULL NAME</strong> : variant-set test for association using annotation information </li>
<li><strong>DESCRIPTION</strong> : STAAR is an R package for performing variant-Set Test for Association using Annotation infoRmation (STAAR) procedure in whole-genome sequencing (WGS) studies. STAAR is a general framework that incorporates both qualitative functional categories and quantitative complementary functional annotations using an omnibus multi-dimensional weighting scheme. STAAR accounts for population structure and relatedness, and is scalable for analyzing large WGS studies of continuous and dichotomous traits. </li>
<li><strong>URL</strong> : <a href="https://github.com/xihaoli/STAAR">https://github.com/xihaoli/STAAR</a> </li>
<li><strong>KEYWORDS</strong> : functional annotations </li>
<li><strong>CITATION</strong> : Li, Xihao, et al. "Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale." Nature genetics 52.9 (2020): 969-983. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/32839606">32839606</a> </li>
</ul>
<h2 id="gene-based-analysis-sumstats">Gene-based analysis (sumstats)</h2>
<h3 id="ldak-gbat">LDAK-GBAT</h3>
<ul>
<li><strong>NAME</strong> : LDAK-GBAT </li>
<li><strong>SHORT NAME</strong> : LDAK-GBAT </li>
<li><strong>FULL NAME</strong> : LDAK gene-based association testing </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>CITATION</strong> : <br><br><br><br><br><br>Berrandou, T.-E., Balding, D. &amp; Speed, D. LDAK-GBAT: Fast and powerful gene-based association testing using summary statistics. Am. J. Hum. Genet. 110, 23–29 (2023). </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36480927">36480927</a> </li>
</ul>
<h2 id="pgs-adjusted-gwas">PGS-adjusted GWAS</h2>
<h3 id="pgs-adjusted-gwas_1">PGS-adjusted GWAS</h3>
<ul>
<li><strong>NAME</strong> : PGS-adjusted GWAS </li>
<li><strong>SHORT NAME</strong> : PGS-adjusted GWAS </li>
<li><strong>FULL NAME</strong> : PGS-adjusted GWAS </li>
<li><strong>DESCRIPTION</strong> : adjustment of GWAS analyses for polygenic scores (PGSs) increases the statistical power for discovery across all ancestries </li>
<li><strong>KEYWORDS</strong> : LOCO-PGSs, two-stage meta-analysis strategy </li>
<li><strong>CITATION</strong> : Campos, A. I., Namba, S., Lin, S. C., Nam, K., Sidorenko, J., Wang, H., ... &amp; Yengo, L. (2023). Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores. Nature Genetics, 1-8. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/37723263">37723263</a> </li>
</ul>
<h3 id="pgs-adjusted-rvats">PGS-adjusted RVATs</h3>
<ul>
<li><strong>NAME</strong> : PGS-adjusted RVATs </li>
<li><strong>SHORT NAME</strong> : PGS-adjusted RVATs </li>
<li><strong>FULL NAME</strong> : PGS-adjusted rare variant association tests </li>
<li><strong>DESCRIPTION</strong> : adjusting for common variant polygenic scores improves yield in gene-based rare variant association tests </li>
<li><strong>KEYWORDS</strong> : PGS, Rare variants </li>
<li><strong>CITATION</strong> : Jurgens, S. J., Pirruccello, J. P., Choi, S. H., Morrill, V. N., Chaffin, M., Lubitz, S. A., ... &amp; Ellinor, P. T. (2023). Adjusting for common variant polygenic scores improves yield in rare variant association analyses. Nature Genetics, 55(4), 544-548. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/36959364">36959364</a> </li>
</ul>
<h2 id="review">Review</h2>
<h3 id="review-povysil">Review-Povysil</h3>
<ul>
<li><strong>NAME</strong> : Review-Povysil </li>
<li><strong>CITATION</strong> : Povysil, Gundula, et al. "Rare-variant collapsing analyses for complex traits: guidelines and applications." Nature Reviews Genetics 20.12 (2019): 747-759. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31605095">31605095</a> </li>
</ul>
<h2 id="single-variant-association-tests">Single variant association tests</h2>
<h3 id="bolt-lmm">BOLT-lMM</h3>
<ul>
<li><strong>NAME</strong> : BOLT-lMM </li>
<li><strong>SHORT NAME</strong> : BOLT-lMM </li>
<li><strong>FULL NAME</strong> : BOLT-lMM </li>
<li><strong>DESCRIPTION</strong> : The BOLT-LMM software package currently consists of two main algorithms, the BOLT-LMM algorithm for mixed model association testing, and the BOLT-REML algorithm for variance components analysis (i.e., partitioning of SNP-heritability and estimation of genetic correlations). </li>
<li><strong>URL</strong> : <a href="https://alkesgroup.broadinstitute.org/BOLT-LMM/BOLT-LMM_manual.html">https://alkesgroup.broadinstitute.org/BOLT-LMM/BOLT-LMM_manual.html</a> </li>
<li><strong>KEYWORDS</strong> : non-infinitesimal model,  mixture of two Gaussian distributions </li>
<li><strong>CITATION</strong> : Loh, Po-Ru, et al. "Efficient Bayesian mixed-model analysis increases association power in large cohorts." Nature genetics 47.3 (2015): 284-290. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25642633">25642633</a> </li>
</ul>
<h3 id="emmax">EMMAX</h3>
<ul>
<li><strong>NAME</strong> : EMMAX </li>
<li><strong>SHORT NAME</strong> : EMMAX </li>
<li><strong>FULL NAME</strong> : efficient mixed-model association eXpedited </li>
<li><strong>DESCRIPTION</strong> : EMMAX is a statistical test for large scale human or model organism association mapping accounting for the sample structure. In addition to the computational efficiency obtained by EMMA algorithm, EMMAX takes advantage of the fact that each loci explains only a small fraction of complex traits, which allows us to avoid repetitive variance component estimation procedure, resulting in a significant amount of increase in computational time of association mapping using mixed model. </li>
<li><strong>URL</strong> : <a href="https://genome.sph.umich.edu/wiki/EMMAX">https://genome.sph.umich.edu/wiki/EMMAX</a> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/20208533">20208533</a> </li>
</ul>
<h3 id="gemma">GEMMA</h3>
<ul>
<li><strong>NAME</strong> : GEMMA </li>
<li><strong>SHORT NAME</strong> : GEMMA </li>
<li><strong>FULL NAME</strong> : genome-wide efficient mixed-model association </li>
<li><strong>DESCRIPTION</strong> : GEMMA is the software implementing the Genome-wide Efficient Mixed Model Association algorithm for a standard linear mixed model and some of its close relatives for genome-wide association studies (GWAS). It fits a standard linear mixed model (LMM) to account for population stratification and sample structure for single marker association tests. It fits a Bayesian sparse linear mixed model (BSLMM) using Markov chain Monte Carlo (MCMC) for estimating the proportion of variance in phenotypes explained (PVE) by typed genotypes (i.e. chip heritability), predicting phenotypes, and identifying associated markers by jointly modeling all markers while controlling for population structure. It is computationally efficient for large scale GWAS and uses freely available open-source numerical libraries. </li>
<li><strong>URL</strong> : <a href="http://stephenslab.uchicago.edu/software.html#gemma">http://stephenslab.uchicago.edu/software.html#gemma</a> </li>
<li><strong>CITATION</strong> : Zhou, Xiang, and Matthew Stephens. "Genome-wide efficient mixed-model analysis for association studies." Nature genetics 44.7 (2012): 821-824. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/22706312">22706312</a> </li>
</ul>
<h3 id="ldak-kvik">LDAK-KVIK</h3>
<ul>
<li><strong>NAME</strong> : LDAK-KVIK </li>
<li><strong>SHORT NAME</strong> : LDAK-KVIK </li>
<li><strong>URL</strong> : <a href="http://www.ldak.org/">http://www.ldak.org/</a> </li>
<li><strong>CITATION</strong> : Hof, J. P. &amp; Speed, D. LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. bioRxiv 2024.07.25.24311005 (2024) doi:10.1101/2024.07.25.24311005.<br><br> </li>
</ul>
<h3 id="plink">PLINK</h3>
<ul>
<li><strong>NAME</strong> : PLINK </li>
<li><strong>SHORT NAME</strong> : PLINK </li>
<li><strong>FULL NAME</strong> : PLINK </li>
<li><strong>DESCRIPTION</strong> : A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses. PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner. The focus of PLINK is purely on analysis of genotype/phenotype data, so there is no support for steps prior to this (e.g. study design and planning, generating genotype or CNV calls from raw data). Through integration with gPLINK and Haploview, there is some support for the subsequent visualization, annotation and storage of results. </li>
<li><strong>URL</strong> : <a href="https://www.cog-genomics.org/plink/">https://www.cog-genomics.org/plink/</a> </li>
<li><strong>CITATION</strong> : Purcell, Shaun, et al. "PLINK: a tool set for whole-genome association and population-based linkage analyses." The American journal of human genetics 81.3 (2007): 559-575. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/17701901">17701901</a> </li>
</ul>
<h3 id="plink2">PLINK2</h3>
<ul>
<li><strong>NAME</strong> : PLINK2 </li>
<li><strong>SHORT NAME</strong> : PLINK2 </li>
<li><strong>FULL NAME</strong> : PLINK2 </li>
<li><strong>URL</strong> : <a href="https://www.cog-genomics.org/plink/2.0/">https://www.cog-genomics.org/plink/2.0/</a> </li>
<li><strong>CITATION</strong> : Chang, Christopher C., et al. "Second-generation PLINK: rising to the challenge of larger and richer datasets." Gigascience 4.1 (2015): s13742-015. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/25722852">25722852</a> </li>
</ul>
<h3 id="polmm">POLMM</h3>
<ul>
<li><strong>NAME</strong> : POLMM </li>
<li><strong>SHORT NAME</strong> : POLMM </li>
<li><strong>FULL NAME</strong> : proportional odds logistic mixed model (POLMM) </li>
<li><strong>DESCRIPTION</strong> : Proportional Odds Logistic Mixed Model (POLMM) for ordinal categorical data analysis </li>
<li><strong>URL</strong> : <a href="https://github.com/WenjianBI/POLMM">https://github.com/WenjianBI/POLMM</a> </li>
<li><strong>KEYWORDS</strong> : ordinal categorical phenotypes </li>
<li><strong>CITATION</strong> : Bi, W., Zhou, W., Dey, R., Mukherjee, B., Sampson, J. N., &amp; Lee, S. (2021). Efficient mixed model approach for large-scale genome-wide association studies of ordinal categorical phenotypes. The American Journal of Human Genetics, 108(5), 825-839. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/33836139">33836139</a> </li>
</ul>
<h3 id="qrgwas">QRGWAS</h3>
<ul>
<li><strong>NAME</strong> : QRGWAS </li>
<li><strong>SHORT NAME</strong> : QRGWAS </li>
<li><strong>FULL NAME</strong> : Quantile regression GWAS </li>
<li><strong>URL</strong> : <a href="https://github.com/Iuliana-Ionita-Laza/QRGWAS">https://github.com/Iuliana-Ionita-Laza/QRGWAS</a> </li>
<li><strong>CITATION</strong> : <br><br>Wang, C. et al. Genome-wide discovery for biomarkers using quantile regression at biobank scale. Nat. Commun. 15, 6460 (2024).<br><br> </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/39085219">39085219</a> </li>
</ul>
<h3 id="regenie_1">REGENIE</h3>
<ul>
<li><strong>NAME</strong> : REGENIE </li>
<li><strong>SHORT NAME</strong> : REGENIE </li>
<li><strong>FULL NAME</strong> : REGENIE </li>
<li><strong>DESCRIPTION</strong> : regenie is a C++ program for whole genome regression modelling of large genome-wide association studies. It is developed and supported by a team of scientists at the Regeneron Genetics Center. </li>
<li><strong>URL</strong> : <a href="https://github.com/rgcgithub/regenie">https://github.com/rgcgithub/regenie</a> </li>
<li><strong>KEYWORDS</strong> : whole genome regression </li>
<li><strong>CITATION</strong> : Mbatchou, Joelle, et al. "Computationally efficient whole-genome regression for quantitative and binary traits." Nature genetics 53.7 (2021): 1097-1103. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34017140">34017140</a> </li>
</ul>
<h3 id="saige">SAIGE</h3>
<ul>
<li><strong>NAME</strong> : SAIGE </li>
<li><strong>SHORT NAME</strong> : SAIGE </li>
<li><strong>FULL NAME</strong> : Scalable and Accurate Implementation of GEneralized mixed model </li>
<li><strong>DESCRIPTION</strong> : SAIGE is an R package with Scalable and Accurate Implementation of Generalized mixed model (Chen, H. et al. 2016). It accounts for sample relatedness and is feasible for genetic association tests in large cohorts and biobanks (N &gt; 400,000). SAIGE performs single-variant association tests for binary traits and quantitative taits. For binary traits, SAIGE uses the saddlepoint approximation (SPA)(mhof, J. P. , 1961; Kuonen, D. 1999; Dey, R. et.al 2017) to account for case-control imbalance. </li>
<li><strong>URL</strong> : <a href="https://github.com/weizhouUMICH/SAIGE">https://github.com/weizhouUMICH/SAIGE</a> </li>
<li><strong>KEYWORDS</strong> : case-control imbalance, saddlepoint approximation (SPA) </li>
<li><strong>CITATION</strong> : Zhou, Wei, et al. "Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies." Nature genetics 50.9 (2018): 1335-1341. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/30104761">30104761</a> </li>
</ul>
<h3 id="fastgwa">fastGWA</h3>
<ul>
<li><strong>NAME</strong> : fastGWA </li>
<li><strong>SHORT NAME</strong> : fastGWA </li>
<li><strong>FULL NAME</strong> : fastGWA </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#fastGWA">https://yanglab.westlake.edu.cn/software/gcta/#fastGWA</a> </li>
<li><strong>KEYWORDS</strong> : grid-search-based REML algorithm </li>
<li><strong>CITATION</strong> : Jiang, Longda, et al. "A resource-efficient tool for mixed model association analysis of large-scale data." Nature genetics 51.12 (2019): 1749-1755. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/31768069">31768069</a> </li>
</ul>
<h3 id="fastgwa-glmm">fastGWA-GLMM</h3>
<ul>
<li><strong>NAME</strong> : fastGWA-GLMM </li>
<li><strong>SHORT NAME</strong> : fastGWA-GLMM </li>
<li><strong>FULL NAME</strong> : fastGWA-GLMM </li>
<li><strong>URL</strong> : <a href="https://yanglab.westlake.edu.cn/software/gcta/#fastGWA">https://yanglab.westlake.edu.cn/software/gcta/#fastGWA</a> </li>
<li><strong>CITATION</strong> : Jiang, Longda, et al. "A generalized linear mixed model association tool for biobank-scale data." Nature genetics 53.11 (2021): 1616-1621. </li>
<li><strong>PUBMED_LINK</strong> : <a href="https://pubmed.ncbi.nlm.nih.gov/34737426">34737426</a> </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>