
## Protein database
### Uniprot 
- URL: https://www.uniprot.org/
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/40289485/193519879-bb1810ba-b2d1-418c-8588-67f9d026f161.png">
- CIATTION: "UniProt: the universal protein knowledgebase in 2021." Nucleic acids research 49, no. D1 (2021): D480-D489.
- CITATION: UniProt Consortium. (2019). UniProt: a worldwide hub of protein knowledge. Nucleic acids research, 47(D1), D506-D515.
- CITATION: UniProt Consortium. (2015). UniProt: a hub for protein information. Nucleic acids research, 43(D1), D204-D212.
- CITATION: "UniProt: the universal protein knowledgebase." Nucleic acids research 45, no. D1 (2017): D158-D169.
- CITATION: UniProt Consortium. (2007). The universal protein resource (UniProt). Nucleic acids research, 36(suppl_1), D190-D195.
- CITATION: UniProt Consortium. (2010). The universal protein resource (UniProt) in 2010. Nucleic acids research, 38(suppl_1), D142-D148.
- CITATION: Apweiler, R., Bairoch, A., Wu, C. H., Barker, W. C., Boeckmann, B., Ferro, S., ... & Yeh, L. S. L. (2004). UniProt: the universal protein knowledgebase. Nucleic acids research, 32(suppl_1), D115-D119.
- CITATION: Boutet, E., Lieberherr, D., Tognolli, M., Schneider, M., & Bairoch, A. (2007). Uniprotkb/swiss-prot. In Plant bioinformatics (pp. 89-112). Humana Press. 

### THE HUMAN PROTEIN ATLAS
- URL: https://www.proteinatlas.org/
- CITATION: Uhlén, M., Fagerberg, L., Hallström, B. M., Lindskog, C., Oksvold, P., Mardinoglu, A., ... & Pontén, F. (2015). Proteomics. Tissue-based map of the human proteome. Science (New York, NY), 347(6220), 1260419-1260419.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/40289485/193522785-af942794-dd75-4889-a6d1-e1f0d2e1bea0.png">

## Human Proteome Map (HPM)
- URL: http://www.humanproteomemap.org/
- DESCRIPTION:The Human Proteome Map (HPM) portal is an interactive resource to the scientific community by integrating the massive peptide sequencing result from the draft map of the human proteome project. The project was based on LC-MS/MS by utilizing of high resolution and high accuracy Fourier transform mass spectrometry. All mass spectrometry data including precursors and HCD-derived fragments were acquired on the Orbitrap mass analyzers in the high-high mode. Currently, the HPM contains direct evidence of translation of a number of protein products derived from over 17,000 human genes covering >84% of the annotated protein-coding genes in humans based on >290,000 non-redundant peptide identifications of multiple organs/tissues and cell types from individuals with clinically defined healthy tissues. This includes 17 adult tissues, 6 primary hematopoietic cells and 7 fetal tissues. The HPM portal provides an interactive web resource by reorganizing the label-free quantitative proteomic data set in a simple graphical view. In addition, the portal provides selected reaction monitoring (SRM) information for all peptides identified.
- CITATION:Kim, M. S., Pinto, S. M., Getnet, D., Nirujogi, R. S., Manda, S. S., Chaerkady, R., ... & Pandey, A. (2014). A draft map of the human proteome. Nature, 509(7502), 575-581.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/40289485/193718006-8549a09f-3011-4098-8dc6-ce6ba117c8a9.png">

## PeptideAtlas 
- URL: http://www.peptideatlas.org/
- DESCRIPTION:PeptideAtlas is a multi-organism, publicly accessible compendium of peptides identified in a large set of tandem mass spectrometry proteomics experiments.
- CITATION:Desiere, F., Deutsch, E. W., King, N. L., Nesvizhskii, A. I., Mallick, P., Eng, J., ... & Aebersold, R. (2006). The peptideatlas project. Nucleic acids research, 34(suppl_1), D655-D658.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/40289485/193718054-73e343b9-6950-4e43-b0dc-bbc1272f7eee.png">
## ProteomicsDB
- URL: https://www.proteomicsdb.org/
- DESCRIPTION:ProteomicsDB is a multi-omics and multi-organism resource for life science research. It covers e.g. proteomics, transcriptomics, and phenomics data for e.g. human, mouse, arabidopsis and rice. Different visualization are available allowing e.g. a protein- and drug-centric interrogation as well as combined analysis via our analytics section.
- CITATION:Samaras, P., Schmidt, T., Frejno, M., Gessulat, S., Reinecke, M., Jarzab, A., ... & Wilhelm, M. (2020). ProteomicsDB: a multi-omics and multi-organism resource for life science research. Nucleic acids research, 48(D1), D1153-D1163.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/40289485/193718101-5738ba87-3e83-44dc-ada5-6f5a99059f6a.png">
