
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="HE Yunye">
      
      
      
        <link rel="prev" href="../Tools_Meta_and_Multi_triat_README/">
      
      
        <link rel="next" href="../Tools_Power_analysis_README/">
      
      
      <link rel="icon" href="../assets/logo.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Polygenic_risk_scores - CTGCatalog</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="blue">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#summary-table" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CTGCatalog" class="md-header__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CTGCatalog
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Polygenic_risk_scores
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../Tools_README/" class="md-tabs__link">
          
  
  Tools

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Sumstats_README/" class="md-tabs__link">
          
  
  Sumstats

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Reference_data_README/" class="md-tabs__link">
          
  
  Reference_data

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Visualization_README/" class="md-tabs__link">
          
  
  Visualization

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../Population_Genetics_README/" class="md-tabs__link">
          
  
  Population_Genetics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CTGCatalog" class="md-nav__button md-logo" aria-label="CTGCatalog" data-md-component="logo">
      
  <img src="../assets/logo.png" alt="logo">

    </a>
    CTGCatalog
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/Cloufield/CTGCatalog/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Tools
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Tools
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Annotation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Annotation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          <label class="md-nav__link" for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Association_tests
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Association_tests
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_RWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_TWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    TWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_eQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eQTL
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Association_tests_sQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Colocalization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Colocalization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Data_processing_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data_processing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Dimension_reduction_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Dimension_reduction
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Drug_discovery_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug_discovery
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Fine_mapping_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fine_mapping
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_prioritization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_prioritization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Gene_set_pathway_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene_set_pathway_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_GxE_interactions_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GxE_interactions
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_HLA_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HLA
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Heritability_and_genetic_correlation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heritability_and_genetic_correlation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Imputation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Imputation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Mendelian_randomization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Mendelian_randomization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Meta_and_Multi_triat_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meta_and_Multi_triat
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Polygenic_risk_scores
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayesr" class="md-nav__link">
    <span class="md-ellipsis">
      BayesR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayess" class="md-nav__link">
    <span class="md-ellipsis">
      BayesS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#benchmark-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark-Wang
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bridgeprs" class="md-nav__link">
    <span class="md-ellipsis">
      BridgePRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cancer-prsweb" class="md-nav__link">
    <span class="md-ellipsis">
      Cancer PRSweb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exprsweb" class="md-nav__link">
    <span class="md-ellipsis">
      ExPRSweb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred-funct" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred-funct
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred2" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred2-auto" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2-auto
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mixer" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-pgs" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-PGS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgscatalog" class="md-nav__link">
    <span class="md-ellipsis">
      PGSCatalog
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prosper" class="md-nav__link">
    <span class="md-ellipsis">
      PROSPER
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      PRS atlas
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-csx" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CSx
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prset" class="md-nav__link">
    <span class="md-ellipsis">
      PRSet
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prsice-2" class="md-nav__link">
    <span class="md-ellipsis">
      PRSice-2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prstuning" class="md-nav__link">
    <span class="md-ellipsis">
      PRStuning
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#puma-cubs" class="md-nav__link">
    <span class="md-ellipsis">
      PUMA-CUBS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-kachuri" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Kachuri
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-peter" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Peter
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wang
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayesr" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayesrc" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesRC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayess" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sdprx" class="md-nav__link">
    <span class="md-ellipsis">
      SDPRX
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tl-prs" class="md-nav__link">
    <span class="md-ellipsis">
      TL-PRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorial-choi" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial-Choi
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#viprs" class="md-nav__link">
    <span class="md-ellipsis">
      VIPRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lassosum" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabolites-prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      metabolites PRS atlas
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgsc_calc" class="md-nav__link">
    <span class="md-ellipsis">
      pgsc_calc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shaprs" class="md-nav__link">
    <span class="md-ellipsis">
      shaPRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wmt-sblup" class="md-nav__link">
    <span class="md-ellipsis">
      wMT-SBLUP
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Power_analysis_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Power_analysis
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Proteomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Simulation_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Simulation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Tissue_and_single_cell_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tissue_and_single_cell
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Tools_Winners_curse_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Winners_curse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Sumstats
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sumstats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_Biobanks_Cohorts_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Biobanks_Cohorts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_metabolomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_metabolomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Sumstats_GWAS_with_proteomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS_with_proteomics
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Reference_data
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Reference_data
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Allele_Frequency_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Allele_Frequency
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Epigenomics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Epigenomics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Expression_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Expression
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Gene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            Gene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Gene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_Ensembl_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Ensembl
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_GENCODE_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GENCODE
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_HGNC_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    HGNC
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_NCBI_refSeq_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    NCBI_refSeq
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_5_6" >
        
          
          <label class="md-nav__link" for="__nav_4_5_6" id="__nav_4_5_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_4_5_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5_6">
            <span class="md-nav__icon md-icon"></span>
            UCSC_refGene
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UCSC_refGene
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ucsc_genes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Gene_knownGenes_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    knownGenes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_6" >
        
          
          <label class="md-nav__link" for="__nav_4_6" id="__nav_4_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Genome
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_6">
            <span class="md-nav__icon md-icon"></span>
            Genome
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Genome_hg38_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    hg38
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Phenotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phenotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_Variant_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variant
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Reference_data_xQTL_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    xQTL
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Visualization
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualization
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Chromosome_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chromosome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Forest_plot_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Forest_plot
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_GWAS_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GWAS
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Heatmap_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Heatmap
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Karyotype_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Karyotype
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_LD_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    LD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Variants_on_protein_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variants_on_protein
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Venn_diagram_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Venn_diagram
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Visualization_Workflow_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Workflow
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Population_Genetics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Population_Genetics
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Admixture_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Admixture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Phylogenetic_tree_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Phylogenetic_tree
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Population_Genetics_Selection_README/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Selection
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#summary-table" class="md-nav__link">
    <span class="md-ellipsis">
      Summary Table
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayesr" class="md-nav__link">
    <span class="md-ellipsis">
      BayesR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayess" class="md-nav__link">
    <span class="md-ellipsis">
      BayesS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#benchmark-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Benchmark-Wang
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bridgeprs" class="md-nav__link">
    <span class="md-ellipsis">
      BridgePRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cancer-prsweb" class="md-nav__link">
    <span class="md-ellipsis">
      Cancer PRSweb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exprsweb" class="md-nav__link">
    <span class="md-ellipsis">
      ExPRSweb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred-funct" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred-funct
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred2" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldpred2-auto" class="md-nav__link">
    <span class="md-ellipsis">
      LDpred2-auto
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mixer" class="md-nav__link">
    <span class="md-ellipsis">
      MiXeR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multi-pgs" class="md-nav__link">
    <span class="md-ellipsis">
      Multi-PGS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgscatalog" class="md-nav__link">
    <span class="md-ellipsis">
      PGSCatalog
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plink2" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prosper" class="md-nav__link">
    <span class="md-ellipsis">
      PROSPER
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      PRS atlas
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-cs" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prs-csx" class="md-nav__link">
    <span class="md-ellipsis">
      PRS-CSx
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prset" class="md-nav__link">
    <span class="md-ellipsis">
      PRSet
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prsice-2" class="md-nav__link">
    <span class="md-ellipsis">
      PRSice-2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prstuning" class="md-nav__link">
    <span class="md-ellipsis">
      PRStuning
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#puma-cubs" class="md-nav__link">
    <span class="md-ellipsis">
      PUMA-CUBS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-kachuri" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Kachuri
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-peter" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Peter
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#review-wang" class="md-nav__link">
    <span class="md-ellipsis">
      Review-Wang
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayesr" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesR
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayesrc" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesRC
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbayess" class="md-nav__link">
    <span class="md-ellipsis">
      SBayesS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sdprx" class="md-nav__link">
    <span class="md-ellipsis">
      SDPRX
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tl-prs" class="md-nav__link">
    <span class="md-ellipsis">
      TL-PRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tutorial-choi" class="md-nav__link">
    <span class="md-ellipsis">
      Tutorial-Choi
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#viprs" class="md-nav__link">
    <span class="md-ellipsis">
      VIPRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lassosum" class="md-nav__link">
    <span class="md-ellipsis">
      lassosum
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabolites-prs-atlas" class="md-nav__link">
    <span class="md-ellipsis">
      metabolites PRS atlas
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgsc_calc" class="md-nav__link">
    <span class="md-ellipsis">
      pgsc_calc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shaprs" class="md-nav__link">
    <span class="md-ellipsis">
      shaPRS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wmt-sblup" class="md-nav__link">
    <span class="md-ellipsis">
      wMT-SBLUP
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  <h1>Polygenic_risk_scores</h1>

<h2 id="summary-table">Summary Table</h2>
<table>
<thead>
<tr>
<th style="text-align: left;">NAME</th>
<th style="text-align: left;">CITATION</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><a href="#bayesr">BayesR</a></td>
<td style="text-align: left;">Moser, G., Lee, S. H., Hayes, B. J., Goddard, M. E., Wray, N. R., &amp; Visscher, P. M. (2015). Simultaneous discovery, estimation and prediction analysis of complex traits using a Bayesian mixture model. PLoS genetics, 11(4), e1004969.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bayess">BayesS</a></td>
<td style="text-align: left;">Zeng, J., De Vlaming, R., Wu, Y., Robinson, M. R., Lloyd-Jones, L. R., Yengo, L., ... &amp; Yang, J. (2018). Signatures of negative selection in the genetic architecture of human complex traits. Nature genetics, 50(5), 746-753.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#benchmark-wang">Benchmark-Wang</a></td>
<td style="text-align: left;">Wang, C., Zhang, J., Zhou, X., &amp; Zhang, L. (2022). A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants. bioRxiv.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#bridgeprs">BridgePRS</a></td>
<td style="text-align: left;">Hoggart, C. J., Choi, S. W., García-González, J., Souaiaia, T., Preuss, M., &amp; O’Reilly, P. F. (2023). BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability. Nature Genetics, 1-7.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#cancer prsweb">Cancer PRSweb</a></td>
<td style="text-align: left;">Fritsche, L. G., Patil, S., Beesley, L. J., VandeHaar, P., Salvatore, M., Ma, Y., ... &amp; Mukherjee, B. (2020). Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. The American Journal of Human Genetics, 107(5), 815-836.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#exprsweb">ExPRSweb</a></td>
<td style="text-align: left;">Ma, Y., Patil, S., Zhou, X., Mukherjee, B., &amp; Fritsche, L. G. (2022). ExPRSweb-An Online Repository with Polygenic Risk Scores for Common Health-related Exposures. medRxiv.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred-funct">LDpred-funct</a></td>
<td style="text-align: left;">Márquez-Luna, C., Gazal, S., Loh, P. R., Kim, S. S., Furlotte, N., Auton, A., &amp; Price, A. L. (2021). Incorporating functional priors improves polygenic prediction accuracy in UK Biobank and 23andMe data sets. Nature Communications, 12(1), 1-11.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred2-auto">LDpred2-auto</a></td>
<td style="text-align: left;">Inferring disease architecture and predictive ability with LDpred2-auto Florian Privé, Clara Albiñana, Bogdan Pasaniuc, Bjarni J. Vilhjálmsson bioRxiv 2022.10.10.511629; doi: https://doi.org/10.1101/2022.10.10.511629</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred2">LDpred2</a></td>
<td style="text-align: left;">Privé, F., Arbel, J., &amp; Vilhjálmsson, B. J. (2020). LDpred2: better, faster, stronger. Bioinformatics, 36(22-23), 5424-5431.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#ldpred">LDpred</a></td>
<td style="text-align: left;">Vilhjálmsson, B. J., Yang, J., Finucane, H. K., Gusev, A., Lindström, S., Ripke, S., ... &amp; Marsal, S. (2015). Modeling linkage disequilibrium increases accuracy of polygenic risk scores. The american journal of human genetics, 97(4), 576-592.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#mixer">MiXeR</a></td>
<td style="text-align: left;">（univariate） Holland, Dominic, et al. "Beyond SNP heritability: Polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model." PLoS Genetics 16.5 (2020): e1008612.<br><br>（cross-trait analysis）Frei, Oleksandr, et al. "Bivariate causal mixture model quantifies polygenic overlap between complex traits beyond genetic correlation." Nature communications 10.1 (2019): 1-11.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#multi-pgs">Multi-PGS</a></td>
<td style="text-align: left;">Albiñana, C., Zhu, Z., Schork, A. J., Ingason, A., Aschard, H., Brikell, I., ... &amp; Vilhjálmsson, B. J. (2023). Multi-PGS enhances polygenic prediction by combining 937 polygenic scores. Nature Communications, 14(1), 4702.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgscatalog">PGSCatalog</a></td>
<td style="text-align: left;">Lambert, S. A., Gil, L., Jupp, S., Ritchie, S. C., Xu, Y., Buniello, A., ... &amp; Inouye, M. (2021). The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nature Genetics, 53(4), 420-425.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#plink2">PLINK2</a></td>
<td style="text-align: left;">Chang, C.C., Chow, C.C., Tellier, L.C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaSci 4, 7 (2015). https://doi.org/10.1186/s13742-015-0047-8</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prosper">PROSPER</a></td>
<td style="text-align: left;">Zhang, J., Zhan, J., Jin, J., Ma, C., Zhao, R., O’Connell, J., ... &amp; 23andMe Research Team. (2023). An ensemble penalized regression method for multi-ancestry polygenic risk prediction. BioRxiv.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs atlas">PRS atlas</a></td>
<td style="text-align: left;">Richardson, T. G., Harrison, S., Hemani, G., &amp; Smith, G. D. (2019). An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome. Elife, 8, e43657.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-cs">PRS-CS</a></td>
<td style="text-align: left;">Ge, T., Chen, CY., Ni, Y. et al. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat Commun 10, 1776 (2019). https://doi.org/10.1038/s41467-019-09718-5</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prs-csx">PRS-CSx</a></td>
<td style="text-align: left;">Ruan, Y., Lin, YF., Feng, YC.A. et al. Improving polygenic prediction in ancestrally diverse populations. Nat Genet 54, 573–580 (2022). https://doi.org/10.1038/s41588-022-01054-7</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prset">PRSet</a></td>
<td style="text-align: left;">Choi SW, Garcia-Gonzalez J, Ruan Y, et al. The power of pathway-based polygenic risk scores. Research Square; 2021. DOI: 10.21203/rs.3.rs-643696/v1.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prsice-2">PRSice-2</a></td>
<td style="text-align: left;">Shing Wan Choi, Paul F O'Reilly, PRSice-2: Polygenic Risk Score software for biobank-scale data, GigaScience, Volume 8, Issue 7, July 2019, giz082, https://doi.org/10.1093/gigascience/giz082</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#prstuning">PRStuning</a></td>
<td style="text-align: left;">Jiang, W., Chen, L., Girgenti, M. J., &amp; Zhao, H. (2023). Tuning Parameters for Polygenic Risk Score Methods Using GWAS Summary Statistics from Training Data. Research Square.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#puma-cubs">PUMA-CUBS</a></td>
<td style="text-align: left;">Zhao, Zijie, et al. "Optimizing and benchmarking polygenic risk scores with GWAS summary statistics." bioRxiv (2022).</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-kachuri">Review-Kachuri</a></td>
<td style="text-align: left;">Kachuri, L., Chatterjee, N., Hirbo, J. et al. Principles and methods for transferring polygenic risk scores across global populations. Nat Rev Genet (2023).</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-peter">Review-Peter</a></td>
<td style="text-align: left;">Peter M. Visscher,Loic Yengo,Nancy J. Cox,Naomi R. Wray,Discovery and implications of polygenicity of common diseases, Science, 373, 6562, (1468-1473), (2021). /doi/10.1126/science.abi8206</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#review-wang">Review-Wang</a></td>
<td style="text-align: left;">Wang, Y., Tsuo, K., Kanai, M., Neale, B. M., &amp; Martin, A. R. (2022). Challenges and Opportunities for Developing More Generalizable Polygenic Risk Scores. Annual Review of Biomedical Data Science, 5.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayesrc">SBayesRC</a></td>
<td style="text-align: left;">Zheng, Z., Liu, S., Sidorenko, J., Yengo, L., Turley, P., Ani, A., ... &amp; Zeng, J. (2022). Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. bioRxiv, 2022-10.<br><br>Zheng, Z., Liu, S., Sidorenko, J. et al. Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. Nat Genet (2024). https://doi.org/10.1038/s41588-024-01704-y</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayesr">SBayesR</a></td>
<td style="text-align: left;">Lloyd-Jones, L. R., Zeng, J., Sidorenko, J., Yengo, L., Moser, G., Kemper, K. E., ... &amp; Visscher, P. M. (2019). Improved polygenic prediction by Bayesian multiple regression on summary statistics. Nature communications, 10(1), 1-11.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sbayess">SBayesS</a></td>
<td style="text-align: left;">Zeng, J., Xue, A., Jiang, L., Lloyd-Jones, L. R., Wu, Y., Wang, H., ... &amp; Yang, J. (2021). Widespread signatures of natural selection across human complex traits and functional genomic categories. Nature communications, 12(1), 1-12.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#sdprx">SDPRX</a></td>
<td style="text-align: left;">Zhou, Geyu, Tianqi Chen, and Hongyu Zhao. "SDPRX: A statistical method for cross-population prediction of complex traits." The American Journal of Human Genetics (2022).</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tl-prs">TL-PRS</a></td>
<td style="text-align: left;">Zhao, Z., Fritsche, L. G., Smith, J. A., Mukherjee, B., &amp; Lee, S. (2022). The Construction of Multi-ethnic Polygenic Risk Score using Transfer Learning. medRxiv.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#tutorial-choi">Tutorial-Choi</a></td>
<td style="text-align: left;">Choi, S. W., Mak, T. S. H., &amp; O’Reilly, P. F. (2020). Tutorial: a guide to performing polygenic risk score analyses. Nature protocols, 15(9), 2759-2772.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#viprs">VIPRS</a></td>
<td style="text-align: left;">Zabad, S., Gravel, S., &amp; Li, Y. (2022). Fast and Accurate Bayesian Polygenic Risk Modeling with Variational Inference. bioRxiv.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#lassosum">lassosum</a></td>
<td style="text-align: left;">Mak, T. S. H., Porsch, R. M., Choi, S. W., Zhou, X., &amp; Sham, P. C. (2017). Polygenic scores via penalized regression on summary statistics. Genetic epidemiology, 41(6), 469-480.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#metabolites prs atlas">metabolites PRS atlas</a></td>
<td style="text-align: left;">Fang, S., Holmes, M. V., Gaunt, T. R., Smith, G. D., &amp; Richardson, T. G. (2022). Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers. eLife.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#pgsc_calc">pgsc_calc</a></td>
<td style="text-align: left;">https://github.com/PGScatalog/pgsc_calc</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#shaprs">shaPRS</a></td>
<td style="text-align: left;">Kelemen, M., Vigorito, E., Fachal, L., Anderson, C. A., &amp; Wallace, C. (2021). ShaPRS: leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores. The American Journal of Human Genetics.</td>
</tr>
<tr>
<td style="text-align: left;"><a href="#wmt-sblup">wMT-SBLUP</a></td>
<td style="text-align: left;">Maier, R.M., Zhu, Z., Lee, S.H. et al. Improving genetic prediction by leveraging genetic correlations among human diseases and traits. Nat Commun 9, 989 (2018). https://doi.org/10.1038/s41467-017-02769-6</td>
</tr>
</tbody>
</table>
<h2 id="bayesr">BayesR</h2>
<ul>
<li>NAME : BayesR </li>
<li>SHORT NAME : BayesR </li>
<li>FULL NAME : BayesR </li>
<li>DESCRIPTION : Bayesian mixture model to dissect genetic variation for disease in human populations and to construct more powerful risk predictors </li>
<li>URL : https://cnsgenomics.com/software/gctb/#Overview </li>
<li>CITATION : Moser, G., Lee, S. H., Hayes, B. J., Goddard, M. E., Wray, N. R., &amp; Visscher, P. M. (2015). Simultaneous discovery, estimation and prediction analysis of complex traits using a Bayesian mixture model. PLoS genetics, 11(4), e1004969. </li>
<li>YEAR : 2015 </li>
</ul>
<h2 id="bayess">BayesS</h2>
<ul>
<li>NAME : BayesS </li>
<li>SHORT NAME : BayesS </li>
<li>FULL NAME : BayesS </li>
<li>URL : https://cnsgenomics.com/software/gctb/#Overview </li>
<li>CITATION : Zeng, J., De Vlaming, R., Wu, Y., Robinson, M. R., Lloyd-Jones, L. R., Yengo, L., ... &amp; Yang, J. (2018). Signatures of negative selection in the genetic architecture of human complex traits. Nature genetics, 50(5), 746-753. </li>
<li>YEAR : 2018 </li>
</ul>
<h2 id="benchmark-wang">Benchmark-Wang</h2>
<ul>
<li>NAME : Benchmark-Wang </li>
<li>CITATION : Wang, C., Zhang, J., Zhou, X., &amp; Zhang, L. (2022). A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants. bioRxiv. </li>
</ul>
<h2 id="bridgeprs">BridgePRS</h2>
<ul>
<li>NAME : BridgePRS </li>
<li>SHORT NAME : BridgePRS </li>
<li>FULL NAME : BridgePRS </li>
<li>DESCRIPTION : BridgePRS is a Bayesian-ridge (Bridge) approach, which "bridges" the PRS between two populations of different ancestry, developed to tackle the "PRS Portability Problem". The PRS Portability Problem causes lower accuracy PRS in underrepresented populations due to the biased sampling in GWAS data collection. </li>
<li>URL : https://www.bridgeprs.net/ </li>
<li>CITATION : Hoggart, C. J., Choi, S. W., García-González, J., Souaiaia, T., Preuss, M., &amp; O’Reilly, P. F. (2023). BridgePRS leverages shared genetic effects across ancestries to increase polygenic risk score portability. Nature Genetics, 1-7. </li>
<li>YEAR : 2023 </li>
</ul>
<h2 id="cancer-prsweb">Cancer PRSweb</h2>
<ul>
<li>NAME : Cancer PRSweb </li>
<li>SHORT NAME : Cancer PRSweb </li>
<li>FULL NAME : Cancer PRSweb </li>
<li>DESCRIPTION : Our framework condenses these summary statistics into PRS using linkage disequilibrium pruning and p-value thresholding (fixed or data-adaptively optimized thresholds) or penalized, genome-wide effect size weighting. We evaluate them in the cancer-enriched cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each PRS construct, measures on performance, calibration, and discrimination are provided. Beyond the cancer PRS evaluation in MGI and UKB, the PRSweb platform features construct downloads, risk evaluation in the top percentiles, and phenome-wide PRS association studies (PRS-PheWAS) for a subset of PRS that are predictive for the primary cancer. </li>
<li>URL : https://prsweb.sph.umich.edu:8443/ </li>
<li>CITATION : Fritsche, L. G., Patil, S., Beesley, L. J., VandeHaar, P., Salvatore, M., Ma, Y., ... &amp; Mukherjee, B. (2020). Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. The American Journal of Human Genetics, 107(5), 815-836. </li>
<li>KEYWORDS : Cancer PRS </li>
<li>YEAR : 2020 </li>
</ul>
<h2 id="exprsweb">ExPRSweb</h2>
<ul>
<li>NAME : ExPRSweb </li>
<li>SHORT NAME : ExPRSweb </li>
<li>FULL NAME : exposure polygenic risk scores (ExPRSs) </li>
<li>DESCRIPTION : Integrating published and freely available genome-wide association studies (GWAS) summary statistics from multiple sources (published GWAS, the NHGRI-EBI GWAS Catalog, FinnGen- or UKB-based GWAS), we created an online repository for exposure polygenic risk scores (ExPRS) for health-related exposure traits. Our framework condenses these summary statistics into ExPRS using linkage disequilibrium pruning and p-value thresholding (P&amp;T) or penalized, genome-wide effect size weighting. We evaluate them in the cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each ExPRS construct, measures on performance, accuracy, and discrimination are provided. Beyond the ExPRS evaluation in MGI and UKB, the ExPRSweb platform features construct downloads, evaluation in the top percentiles, and phenome-wide ExPRS association studies (ExPRS-PheWAS) for a subset of ExPRS that are predictive for the corresponding exposure. </li>
<li>URL : https://exprsweb.sph.umich.edu:8443/ </li>
<li>CITATION : Ma, Y., Patil, S., Zhou, X., Mukherjee, B., &amp; Fritsche, L. G. (2022). ExPRSweb-An Online Repository with Polygenic Risk Scores for Common Health-related Exposures. medRxiv. </li>
<li>KEYWORDS : exposure PRS </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="ldpred">LDpred</h2>
<ul>
<li>NAME : LDpred </li>
<li>SHORT NAME : LDpred </li>
<li>FULL NAME : LDpred </li>
<li>DESCRIPTION : LDpred is a Python based software package that adjusts GWAS summary statistics for the effects of linkage disequilibrium (LD). </li>
<li>URL : https://github.com/bvilhjal/ldpred </li>
<li>CITATION : Vilhjálmsson, B. J., Yang, J., Finucane, H. K., Gusev, A., Lindström, S., Ripke, S., ... &amp; Marsal, S. (2015). Modeling linkage disequilibrium increases accuracy of polygenic risk scores. The american journal of human genetics, 97(4), 576-592. </li>
<li>KEYWORDS : Bayesian, Gaussian infinitesimal prior, python </li>
</ul>
<h2 id="ldpred-funct">LDpred-funct</h2>
<ul>
<li>NAME : LDpred-funct </li>
<li>SHORT NAME : LDpred-funct </li>
<li>FULL NAME : LDpred-funct </li>
<li>DESCRIPTION : LDpred-funct is a method for polygenic prediction that leverages trait-specific functional priors to increase prediction accuracy. </li>
<li>URL : https://github.com/carlaml/LDpred-funct </li>
<li>CITATION : Márquez-Luna, C., Gazal, S., Loh, P. R., Kim, S. S., Furlotte, N., Auton, A., &amp; Price, A. L. (2021). Incorporating functional priors improves polygenic prediction accuracy in UK Biobank and 23andMe data sets. Nature Communications, 12(1), 1-11. </li>
<li>KEYWORDS : Bayesian, functional priors </li>
</ul>
<h2 id="ldpred2">LDpred2</h2>
<ul>
<li>NAME : LDpred2 </li>
<li>SHORT NAME : LDpred2 </li>
<li>FULL NAME : LDpred2 </li>
<li>DESCRIPTION : LDpred-2 is one of the dedicated PRS programs which is an R package that uses a Bayesian approach to polygenic risk scoring. </li>
<li>URL : https://privefl.github.io/bigsnpr/articles/LDpred2.html </li>
<li>CITATION : Privé, F., Arbel, J., &amp; Vilhjálmsson, B. J. (2020). LDpred2: better, faster, stronger. Bioinformatics, 36(22-23), 5424-5431. </li>
<li>KEYWORDS : Bayesian, R, LDpred2-grid (LDpred2), LDpred2-auto, LDpred2-sparse </li>
</ul>
<h2 id="ldpred2-auto">LDpred2-auto</h2>
<ul>
<li>NAME : LDpred2-auto </li>
<li>SHORT NAME : LDpred2-auto </li>
<li>FULL NAME : LDpred2-auto </li>
<li>DESCRIPTION : LDpred2 is a widely used Bayesian method for building polygenic scores (PGS). LDpred2-auto can infer the two parameters from the LDpred model, h^2 and p, so that it does not require an additional validation dataset to choose best-performing parameters. Here, we present a new version of LDpred2-auto, which adds a third parameter alpha to its model for modeling negative selection. Additional changes are also made to provide better sampling of these parameters. </li>
<li>URL : https://privefl.github.io/bigsnpr/articles/LDpred2.html </li>
<li>CITATION : Inferring disease architecture and predictive ability with LDpred2-auto Florian Privé, Clara Albiñana, Bogdan Pasaniuc, Bjarni J. Vilhjálmsson bioRxiv 2022.10.10.511629; doi: https://doi.org/10.1101/2022.10.10.511629 </li>
<li>KEYWORDS : Bayesian, new LDpred2-auto, α (relationship between MAF and beta) </li>
</ul>
<h2 id="mixer">MiXeR</h2>
<ul>
<li>NAME : MiXeR </li>
<li>SHORT NAME : MiXeR </li>
<li>FULL NAME : MiXeR </li>
<li>DESCRIPTION : Causal Mixture Model for GWAS summary statistics </li>
<li>URL : https://github.com/precimed/mixer </li>
<li>CITATION : （univariate） Holland, Dominic, et al. "Beyond SNP heritability: Polygenicity and discoverability of phenotypes estimated with a univariate Gaussian mixture model." PLoS Genetics 16.5 (2020): e1008612.
（cross-trait analysis）Frei, Oleksandr, et al. "Bivariate causal mixture model quantifies polygenic overlap between complex traits beyond genetic correlation." Nature communications 10.1 (2019): 1-11. </li>
</ul>
<h2 id="multi-pgs">Multi-PGS</h2>
<ul>
<li>NAME : Multi-PGS </li>
<li>SHORT NAME : Multi-PGS </li>
<li>FULL NAME : Multi-PGS </li>
<li>DESCRIPTION : a framework to generate enriched PGS from a wealth of publicly available genome-wide association studies, combining thousands of studies focused on many different phenotypes, into a multi-PGS </li>
<li>URL : https://github.com/ClaraAlbi/paper_multiPGS </li>
<li>CITATION : Albiñana, C., Zhu, Z., Schork, A. J., Ingason, A., Aschard, H., Brikell, I., ... &amp; Vilhjálmsson, B. J. (2023). Multi-PGS enhances polygenic prediction by combining 937 polygenic scores. Nature Communications, 14(1), 4702. </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="pgscatalog">PGSCatalog</h2>
<ul>
<li>NAME : PGSCatalog </li>
<li>SHORT NAME : PGS Catalog </li>
<li>FULL NAME : PGS Catalog </li>
<li>DESCRIPTION : The PGS Catalog is an open database of published polygenic scores (PGS). Each PGS in the Catalog is consistently annotated with relevant metadata; including scoring files (variants, effect alleles/weights), annotations of how the PGS was developed and applied, and evaluations of their predictive performance. </li>
<li>URL : https://www.pgscatalog.org/ </li>
<li>CITATION : Lambert, S. A., Gil, L., Jupp, S., Ritchie, S. C., Xu, Y., Buniello, A., ... &amp; Inouye, M. (2021). The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nature Genetics, 53(4), 420-425. </li>
<li>KEYWORDS : PGS database </li>
</ul>
<h2 id="plink2">PLINK2</h2>
<ul>
<li>NAME : PLINK2 </li>
<li>SHORT NAME : PLINK2 </li>
<li>FULL NAME : PLINK2 </li>
<li>DESCRIPTION : The second-generation versions of PLINK will offer dramatic improvements in performance and compatibility. </li>
<li>URL : https://www.cog-genomics.org/plink/2.0/ </li>
<li>CITATION : Chang, C.C., Chow, C.C., Tellier, L.C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaSci 4, 7 (2015). https://doi.org/10.1186/s13742-015-0047-8 </li>
<li>USE : calculate PRS using genotype data. </li>
</ul>
<h2 id="prosper">PROSPER</h2>
<ul>
<li>NAME : PROSPER </li>
<li>SHORT NAME : PROSPER </li>
<li>FULL NAME : Polygenic Risk scOres based on enSemble of PEnalized Regression models </li>
<li>DESCRIPTION : PROSPER is a new multi-ancestry PRS method with penalized regression followed by ensemble learning. This software is a command line tool based on R programming language. Large-scale benchmarking study shows that PROSPER could be the leading method to reduce the disparity of PRS performance across ancestry groups </li>
<li>URL : https://github.com/Jingning-Zhang/PROSPER </li>
<li>CITATION : Zhang, J., Zhan, J., Jin, J., Ma, C., Zhao, R., O’Connell, J., ... &amp; 23andMe Research Team. (2023). An ensemble penalized regression method for multi-ancestry polygenic risk prediction. BioRxiv. </li>
<li>YEAR : 2023 </li>
</ul>
<h2 id="prs-atlas">PRS atlas</h2>
<ul>
<li>NAME : PRS atlas </li>
<li>SHORT NAME : PRS atlas </li>
<li>FULL NAME : PRS atlas </li>
<li>DESCRIPTION : This web application can be used to query findings from an analysis of 162 polygenic risk scores and 551 complex traits using data from the UK Biobank study1. Traits were selected based on the heritability analysis conducted by the Neale Lab2 (P&lt;0.05). We encourage users of this resource to conduct follow-up analyses of associations to robustly identify causal relationships between complex traits. </li>
<li>URL : http://mrcieu.mrsoftware.org/PRS_atlas/ </li>
<li>CITATION : Richardson, T. G., Harrison, S., Hemani, G., &amp; Smith, G. D. (2019). An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome. Elife, 8, e43657. </li>
</ul>
<h2 id="prs-cs">PRS-CS</h2>
<ul>
<li>NAME : PRS-CS </li>
<li>SHORT NAME : PRS-CS </li>
<li>FULL NAME : PRS-CS </li>
<li>DESCRIPTION : PRS-CS is a Python based command line tool that infers posterior SNP effect sizes under continuous shrinkage (CS) priors using GWAS summary statistics and an external LD reference panel. </li>
<li>URL : https://github.com/getian107/PRScs </li>
<li>CITATION : Ge, T., Chen, CY., Ni, Y. et al. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat Commun 10, 1776 (2019). https://doi.org/10.1038/s41467-019-09718-5 </li>
<li>KEYWORDS : continuous shrinkage (CS) prior </li>
<li>YEAR : 2019 </li>
</ul>
<h2 id="prs-csx">PRS-CSx</h2>
<ul>
<li>NAME : PRS-CSx </li>
<li>SHORT NAME : PRS-CSx </li>
<li>FULL NAME : PRS-CSx </li>
<li>DESCRIPTION : PRS-CSx is a Python based command line tool that integrates GWAS summary statistics and external LD reference panels from multiple populations to improve cross-population polygenic prediction. Posterior SNP effect sizes are inferred under coupled continuous shrinkage (CS) priors across populations. </li>
<li>URL : https://github.com/getian107/PRScsx </li>
<li>CITATION : Ruan, Y., Lin, YF., Feng, YC.A. et al. Improving polygenic prediction in ancestrally diverse populations. Nat Genet 54, 573–580 (2022). https://doi.org/10.1038/s41588-022-01054-7 </li>
<li>KEYWORDS : continuous shrinkage (CS) prior,  cross-population </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="prset">PRSet</h2>
<ul>
<li>NAME : PRSet </li>
<li>SHORT NAME : PRSet </li>
<li>FULL NAME : PRSet </li>
<li>DESCRIPTION : A new feature of PRSice is the ability to perform set base/pathway based analysis. This new feature is called PRSet. </li>
<li>URL : https://www.prsice.info/quick_start_prset/ </li>
<li>CITATION : Choi SW, Garcia-Gonzalez J, Ruan Y, et al. The power of pathway-based polygenic risk scores. Research Square; 2021. DOI: 10.21203/rs.3.rs-643696/v1. </li>
<li>KEYWORDS : pathway-based </li>
</ul>
<h2 id="prsice-2">PRSice-2</h2>
<ul>
<li>NAME : PRSice-2 </li>
<li>SHORT NAME : PRSice-2 </li>
<li>FULL NAME : PRSice-2 </li>
<li>DESCRIPTION : PRSice (pronounced 'precise') is a Polygenic Risk Score software for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS) analyses. </li>
<li>URL : https://www.prsice.info/ </li>
<li>CITATION : Shing Wan Choi, Paul F O'Reilly, PRSice-2: Polygenic Risk Score software for biobank-scale data, GigaScience, Volume 8, Issue 7, July 2019, giz082, https://doi.org/10.1093/gigascience/giz082 </li>
</ul>
<h2 id="prstuning">PRStuning</h2>
<ul>
<li>NAME : PRStuning </li>
<li>SHORT NAME : PRStuning </li>
<li>FULL NAME : PRStuning </li>
<li>DESCRIPTION : Estimate Testing AUC for Binary Phenotype Using GWAS Summary Statistics from the Training Data </li>
<li>CITATION : Jiang, W., Chen, L., Girgenti, M. J., &amp; Zhao, H. (2023). Tuning Parameters for Polygenic Risk Score Methods Using GWAS Summary Statistics from Training Data. Research Square. </li>
</ul>
<h2 id="puma-cubs">PUMA-CUBS</h2>
<ul>
<li>NAME : PUMA-CUBS </li>
<li>SHORT NAME : PUMA-CUBS </li>
<li>FULL NAME : PUMA-CUBS </li>
<li>DESCRIPTION : an ensemble learning strategy named PUMACUBS to combine multiple PRS models into an ensemble score without requiring external data for model fitting. </li>
<li>URL : https://github.com/qlu-lab/PUMAS </li>
<li>CITATION : Zhao, Zijie, et al. "Optimizing and benchmarking polygenic risk scores with GWAS summary statistics." bioRxiv (2022). </li>
</ul>
<h2 id="review-kachuri">Review-Kachuri</h2>
<ul>
<li>NAME : Review-Kachuri </li>
<li>CITATION : Kachuri, L., Chatterjee, N., Hirbo, J. et al. Principles and methods for transferring polygenic risk scores across global populations. Nat Rev Genet (2023). </li>
</ul>
<h2 id="review-peter">Review-Peter</h2>
<ul>
<li>NAME : Review-Peter </li>
<li>CITATION : Peter M. Visscher,Loic Yengo,Nancy J. Cox,Naomi R. Wray,Discovery and implications of polygenicity of common diseases, Science, 373, 6562, (1468-1473), (2021). /doi/10.1126/science.abi8206 </li>
</ul>
<h2 id="review-wang">Review-Wang</h2>
<ul>
<li>NAME : Review-Wang </li>
<li>CITATION : Wang, Y., Tsuo, K., Kanai, M., Neale, B. M., &amp; Martin, A. R. (2022). Challenges and Opportunities for Developing More Generalizable Polygenic Risk Scores. Annual Review of Biomedical Data Science, 5. </li>
</ul>
<h2 id="sbayesr">SBayesR</h2>
<ul>
<li>NAME : SBayesR </li>
<li>SHORT NAME : SBayesR </li>
<li>FULL NAME : SBayesR </li>
<li>DESCRIPTION : extend a powerful individual-level data Bayesian multiple regression model (BayesR) to one that utilises summary statistics from genome-wide association studies, SBayesR. </li>
<li>URL : https://cnsgenomics.com/software/gctb/#Overview </li>
<li>CITATION : Lloyd-Jones, L. R., Zeng, J., Sidorenko, J., Yengo, L., Moser, G., Kemper, K. E., ... &amp; Visscher, P. M. (2019). Improved polygenic prediction by Bayesian multiple regression on summary statistics. Nature communications, 10(1), 1-11. </li>
<li>YEAR : 2019 </li>
</ul>
<h2 id="sbayesrc">SBayesRC</h2>
<ul>
<li>NAME : SBayesRC </li>
<li>SHORT NAME : SBayesRC </li>
<li>FULL NAME : SBayesRC </li>
<li>DESCRIPTION : SBayesRC integrates GWAS summary statistics with functional genomic annotations to improve polygenic prediction of complex traits. </li>
<li>URL : https://cnsgenomics.com/software/gctb/#Overview </li>
<li>CITATION : Zheng, Z., Liu, S., Sidorenko, J., Yengo, L., Turley, P., Ani, A., ... &amp; Zeng, J. (2022). Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. bioRxiv, 2022-10.
Zheng, Z., Liu, S., Sidorenko, J. et al. Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. Nat Genet (2024). https://doi.org/10.1038/s41588-024-01704-y </li>
<li>KEYWORDS : functional genomic annotation, whole-genome variants,  cross-ancestry </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="sbayess">SBayesS</h2>
<ul>
<li>NAME : SBayesS </li>
<li>SHORT NAME : SBayesS </li>
<li>FULL NAME : SBayesS </li>
<li>DESCRIPTION : estimate multiple genetic architecture parameters including selection signature using only GWAS summary statistics </li>
<li>URL : https://cnsgenomics.com/software/gctb/#Overview </li>
<li>CITATION : Zeng, J., Xue, A., Jiang, L., Lloyd-Jones, L. R., Wu, Y., Wang, H., ... &amp; Yang, J. (2021). Widespread signatures of natural selection across human complex traits and functional genomic categories. Nature communications, 12(1), 1-12. </li>
<li>YEAR : 2021 </li>
</ul>
<h2 id="sdprx">SDPRX</h2>
<ul>
<li>NAME : SDPRX </li>
<li>SHORT NAME : SDPRX </li>
<li>FULL NAME : SDPRX </li>
<li>DESCRIPTION : SDPRX is a statistical method for cross-population prediction of complex traits. It integrates GWAS summary statistics and LD matrices from two populations (EUR and non-EUR) to compuate polygenic risk scores. </li>
<li>URL : https://github.com/eldronzhou/SDPRX </li>
<li>CITATION : Zhou, Geyu, Tianqi Chen, and Hongyu Zhao. "SDPRX: A statistical method for cross-population prediction of complex traits." The American Journal of Human Genetics (2022). </li>
</ul>
<h2 id="tl-prs">TL-PRS</h2>
<ul>
<li>NAME : TL-PRS </li>
<li>SHORT NAME : TL-PRS </li>
<li>FULL NAME : transfer learning PRS </li>
<li>DESCRIPTION : This R package helps users to construct multi-ethnic polygenic risk score (PRS) using transfer learning. It can help predict PRS of minor ancestry using summary statistics from exsiting resources, such as UK Biobank. </li>
<li>URL : https://github.com/ZhangchenZhao/TLPRS </li>
<li>CITATION : Zhao, Z., Fritsche, L. G., Smith, J. A., Mukherjee, B., &amp; Lee, S. (2022). The Construction of Multi-ethnic Polygenic Risk Score using Transfer Learning. medRxiv. </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="tutorial-choi">Tutorial-Choi</h2>
<ul>
<li>NAME : Tutorial-Choi </li>
<li>SHORT NAME : PRS Tutorial </li>
<li>FULL NAME : PRS Tutorial </li>
<li>DESCRIPTION : This tutorial provides a step-by-step guide to performing basic polygenic risk score (PRS) analyses and accompanies our PRS Guide paper. The aim of this tutorial is to provide a simple introduction of PRS analyses to those new to PRS, while equipping existing users with a better understanding of the processes and implementation "underneath the hood" of popular PRS software. </li>
<li>URL : https://choishingwan.github.io/PRS-Tutorial/ </li>
<li>CITATION : Choi, S. W., Mak, T. S. H., &amp; O’Reilly, P. F. (2020). Tutorial: a guide to performing polygenic risk score analyses. Nature protocols, 15(9), 2759-2772. </li>
</ul>
<h2 id="viprs">VIPRS</h2>
<ul>
<li>NAME : VIPRS </li>
<li>SHORT NAME : VIPRS </li>
<li>FULL NAME : Variational inference of polygenic risk scores </li>
<li>DESCRIPTION : viprs is a python package that implements scripts and utilities for running variational inference algorithms on genome-wide association study (GWAS) data for the purposes polygenic risk estimation. </li>
<li>URL : https://github.com/shz9/viprs </li>
<li>CITATION : Zabad, S., Gravel, S., &amp; Li, Y. (2022). Fast and Accurate Bayesian Polygenic Risk Modeling with Variational Inference. bioRxiv. </li>
<li>KEYWORDS : Variational Inference (VI) </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="lassosum">lassosum</h2>
<ul>
<li>NAME : lassosum </li>
<li>SHORT NAME : lassosum </li>
<li>FULL NAME : lassosum </li>
<li>DESCRIPTION : lassosum is a method for computing LASSO/Elastic Net estimates of a linear regression problem given summary statistics from GWAS and Genome-wide meta-analyses, accounting for Linkage Disequilibrium (LD), via a reference panel. </li>
<li>URL : https://github.com/tshmak/lassosum </li>
<li>CITATION : Mak, T. S. H., Porsch, R. M., Choi, S. W., Zhou, X., &amp; Sham, P. C. (2017). Polygenic scores via penalized regression on summary statistics. Genetic epidemiology, 41(6), 469-480. </li>
<li>KEYWORDS : penalized regression </li>
<li>YEAR : 2017 </li>
</ul>
<h2 id="metabolites-prs-atlas">metabolites PRS atlas</h2>
<ul>
<li>NAME : metabolites PRS atlas </li>
<li>SHORT NAME : metabolites PRS atlas </li>
<li>FULL NAME : metabolites PRS atlas </li>
<li>DESCRIPTION : This web application can be used to query findings from a systematic analysis of 129 polygenic risk scores and 249 circulating metabolits using high-throughput nuclear magnetic resonance data from the UK Biobank study1,2. We encourage users of this resource to conduct follow-up analyses of associations to investigate potential causal and non-causal metabolic biomarkers. Age-stratified results can be used to investigate how potential sources of collider bias (e.g. statin therapy) may influence findings in the full sample </li>
<li>URL : http://mrcieu.mrsoftware.org/metabolites_PRS_atlas/ </li>
<li>CITATION : Fang, S., Holmes, M. V., Gaunt, T. R., Smith, G. D., &amp; Richardson, T. G. (2022). Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers. eLife. </li>
</ul>
<h2 id="pgsc_calc">pgsc_calc</h2>
<ul>
<li>NAME : pgsc_calc </li>
<li>SHORT NAME : pgsc_calc </li>
<li>FULL NAME : The Polygenic Score Catalog Calculator </li>
<li>DESCRIPTION : pgsc_calc is a bioinformatics best-practice analysis pipeline for calculating polygenic [risk] scores on samples with imputed genotypes using existing scoring files from the Polygenic Score (PGS) Catalog and/or user-defined PGS/PRS. </li>
<li>URL : https://github.com/PGScatalog/pgsc_calc </li>
<li>CITATION : https://github.com/PGScatalog/pgsc_calc </li>
<li>KEYWORDS : PRS calculation pipeline </li>
<li>YEAR : 2022 </li>
</ul>
<h2 id="shaprs">shaPRS</h2>
<ul>
<li>NAME : shaPRS </li>
<li>SHORT NAME : shaPRS </li>
<li>FULL NAME : shaPRS </li>
<li>DESCRIPTION : Leveraging shared genetic effects across traits and ancestries improves accuracy of polygenic scores </li>
<li>URL : https://github.com/mkelcb/shaprs </li>
<li>CITATION : Kelemen, M., Vigorito, E., Fachal, L., Anderson, C. A., &amp; Wallace, C. (2021). ShaPRS: leveraging shared genetic effects across traits or ancestries improves accuracy of polygenic scores. The American Journal of Human Genetics. </li>
<li>KEYWORDS : cross-ancestry, genetic correlation </li>
<li>YEAR : 2024 </li>
</ul>
<h2 id="wmt-sblup">wMT-SBLUP</h2>
<ul>
<li>NAME : wMT-SBLUP </li>
<li>SHORT NAME : wMT-SBLUP </li>
<li>FULL NAME : weighted approximate multi-trait summary statistic BLUP </li>
<li>URL : https://github.com/uqrmaie1/smtpred </li>
<li>CITATION : Maier, R.M., Zhu, Z., Lee, S.H. et al. Improving genetic prediction by leveraging genetic correlations among human diseases and traits. Nat Commun 9, 989 (2018). https://doi.org/10.1038/s41467-017-02769-6 </li>
<li>YEAR : 2018 </li>
</ul>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      CTGCatalog is licensed under the MIT license
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.top", "search.highlight", "search.suggest", "search.share"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>