<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <meta name="author" content="HE Yunye">
    
    <link rel="shortcut icon" href="../img/favicon.ico">

    
    <title>Polygenic_risk_scores - CTGCatalog</title>
    

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/v4-shims.css">
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/hack-font@3.3.0/build/web/hack.min.css">
    <link href='//rsms.me/inter/inter.css' rel='stylesheet' type='text/css'>
    <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin' rel='stylesheet' type='text/css'>
    <link href="../css/bootstrap-custom.min.css" rel="stylesheet">
    <link href="../css/base.min.css" rel="stylesheet">
    <link href="../css/cinder.min.css" rel="stylesheet">

    
        
        <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/styles/github.min.css">
        
    

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
            <script src="https://cdn.jsdelivr.net/npm/html5shiv@3.7.3/dist/html5shiv.min.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/respond.js@1.4.2/dest/respond.min.js"></script>
        <![endif]-->

    

     
</head>

<body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">

        <!-- Collapsed navigation -->
        <div class="navbar-header">
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            

            <!-- Main title -->

            
              <a class="navbar-brand" href="..">CTGCatalog</a>
            
        </div>

        <!-- Expanded navigation -->
        <div class="navbar-collapse collapse">
                <!-- Main navigation -->
                <ul class="nav navbar-nav">
                
                
                    <li >
                        <a href="..">Home</a>
                    </li>
                
                
                
                    <li class="dropdown active">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Tools <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                        
                            
<li >
    <a href="../Tools_README/">Tools</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Visualization</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Tools_Visualization_README/">Visualization</a>
</li>

        
            
<li >
    <a href="../Tools_Visualization_Variants_on_protein_README/">Variants_on_protein</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Tools_Drug_discovery_README/">Drug_discovery</a>
</li>

                        
                            
<li >
    <a href="../Tools_Gene_prioritization_README/">Gene_prioritization</a>
</li>

                        
                            
<li class="active">
    <a href="./">Polygenic_risk_scores</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Association_tests</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Tools_Association_tests_README/">Association_tests</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_TWAS_README/">TWAS</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_sQTL_README/">sQTL</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_eQTL_README/">eQTL</a>
</li>

        
            
<li >
    <a href="../Tools_Association_tests_RWAS_README/">RWAS</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Tools_Colocalization_README/">Colocalization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Simulation_README/">Simulation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Data_processing_README/">Data_processing</a>
</li>

                        
                            
<li >
    <a href="../Tools_Annotation_README/">Annotation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Fine_mapping_README/">Fine_mapping</a>
</li>

                        
                            
<li >
    <a href="../Tools_Gene_set_pathway_analysis_README/">Gene_set_pathway_analysis</a>
</li>

                        
                            
<li >
    <a href="../Tools_Tissue_and_single_cell_README/">Tissue_and_single_cell</a>
</li>

                        
                            
<li >
    <a href="../Tools_Imputation_README/">Imputation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Heritability_and_genetic_correlation_README/">Heritability_and_genetic_correlation</a>
</li>

                        
                            
<li >
    <a href="../Tools_Proteomics_README/">Proteomics</a>
</li>

                        
                            
<li >
    <a href="../Tools_GxE_interactions_README/">GxE_interactions</a>
</li>

                        
                            
<li >
    <a href="../Tools_Mendelian_randomization_README/">Mendelian_randomization</a>
</li>

                        
                            
<li >
    <a href="../Tools_Meta_and_Multi_triat_README/">Meta_and_Multi_triat</a>
</li>

                        
                        </ul>
                    </li>
                
                
                
                    <li >
                        <a href="../Sumstats_README/">Sumstats</a>
                    </li>
                
                
                
                    <li class="dropdown">
                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Reference_data <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                        
                            
<li >
    <a href="../Reference_data_README/">Reference_data</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Biobanks_Cohorts_README/">Biobanks_Cohorts</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Protein_README/">Protein</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Gene</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Gene_README/">Gene</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_NCBI_refSeq_README/">NCBI_refSeq</a>
</li>

        
            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">UCSC_refGene</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Gene_UCSC_refGene_README/">UCSC_refGene</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_UCSC_refGene_ucsc_genes_README/">ucsc_genes</a>
</li>

        
    </ul>
  </li>

        
            
<li >
    <a href="../Reference_data_Gene_HGNC_README/">HGNC</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_knownGenes_README/">knownGenes</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_Ensembl_README/">Ensembl</a>
</li>

        
            
<li >
    <a href="../Reference_data_Gene_GENCODE_README/">GENCODE</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Reference_data_Phenotype_README/">Phenotype</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Variant_README/">Variant</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Epigenomics_README/">Epigenomics</a>
</li>

                        
                            
<li >
    <a href="../Reference_data_Expression_README/">Expression</a>
</li>

                        
                            
  <li class="dropdown-submenu">
    <a tabindex="-1" href="">Genome</a>
    <ul class="dropdown-menu">
        
            
<li >
    <a href="../Reference_data_Genome_README/">Genome</a>
</li>

        
            
<li >
    <a href="../Reference_data_Genome_hg38_README/">hg38</a>
</li>

        
    </ul>
  </li>

                        
                            
<li >
    <a href="../Reference_data_xQTL_README/">xQTL</a>
</li>

                        
                        </ul>
                    </li>
                
                
                </ul>

            <ul class="nav navbar-nav navbar-right">
                    <li>
                        <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
                            <i class="fas fa-search"></i> Search
                        </a>
                    </li>
                    <li >
                        <a rel="prev" href="../Tools_Gene_prioritization_README/">
                            <i class="fas fa-arrow-left"></i> Previous
                        </a>
                    </li>
                    <li >
                        <a rel="next" href="../Tools_Association_tests_README/">
                            Next <i class="fas fa-arrow-right"></i>
                        </a>
                    </li>
                    <li>
                        <a href="https://github.com/Cloufield/CTGCatalog/"><i class="fab fa-github"></i> GitHub</a>
                    </li>
            </ul>
        </div>
    </div>
</div>

    <div class="container">
        
        
        <div class="col-md-3"><div class="bs-sidebar hidden-print affix well" role="complementary">
    <ul class="nav bs-sidenav">
        <li class="first-level active"><a href="#prs-methods-and-tools">PRS methods and tools:</a></li>
            <li class="second-level"><a href="#prs-tutorial">PRS Tutorial</a></li>
                
            <li class="second-level"><a href="#plink2">PLINK2</a></li>
                
            <li class="second-level"><a href="#prsice-2">PRSice-2</a></li>
                
            <li class="second-level"><a href="#prset">PRSet</a></li>
                
            <li class="second-level"><a href="#ldpred2">LDpred2</a></li>
                
            <li class="second-level"><a href="#ldpred2-auto">LDpred2-auto</a></li>
                
            <li class="second-level"><a href="#prs-cs">PRS-CS</a></li>
                
            <li class="second-level"><a href="#prs-csx">PRS-CSx</a></li>
                
            <li class="second-level"><a href="#lassosum">lassosum</a></li>
                
            <li class="second-level"><a href="#bayesr">BayesR</a></li>
                
            <li class="second-level"><a href="#bayess">BayesS</a></li>
                
            <li class="second-level"><a href="#sbayesr">SBayesR</a></li>
                
            <li class="second-level"><a href="#sbayess">SBayesS</a></li>
                
            <li class="second-level"><a href="#sbayesrc">SBayesRC</a></li>
                
            <li class="second-level"><a href="#tl-prs">TL-PRS</a></li>
                
            <li class="second-level"><a href="#multi-pgs">Multi-PGS</a></li>
                
            <li class="second-level"><a href="#wmt-sblup">wMT-SBLUP</a></li>
                
            <li class="second-level"><a href="#pgscatalog">PGSCatalog</a></li>
                
            <li class="second-level"><a href="#pgsc_calc">pgsc_calc</a></li>
                
            <li class="second-level"><a href="#viprs">VIPRS</a></li>
                
            <li class="second-level"><a href="#exprsweb">ExPRSweb</a></li>
                
            <li class="second-level"><a href="#cancer-prsweb">Cancer PRSweb</a></li>
                
            <li class="second-level"><a href="#prs-atlas">PRS atlas</a></li>
                
            <li class="second-level"><a href="#metabolites-prs-atlas">metabolites PRS atlas</a></li>
                
            <li class="second-level"><a href="#reviewstutorial">Reviews&amp;Tutorial</a></li>
                
    </ul>
</div></div>
        <div class="col-md-9" role="main">

<h1 id="prs-methods-and-tools">PRS methods and tools:</h1>
<ul>
<li>PLINK2</li>
<li>PRSice-2</li>
<li>PRSet</li>
<li>LDpred2</li>
<li>LDpred2-auto</li>
<li>PRS-CS</li>
<li>lassosum</li>
<li>PRS-CSx</li>
<li>Multi-PGS</li>
<li>PGSCatalog</li>
<li>wMT-SBLUP</li>
<li>pgsc_calc</li>
<li>VIPRS</li>
<li>MegaPRS</li>
<li>BayesS</li>
<li>BayesR</li>
<li>SBayesR</li>
<li>SBayesS</li>
<li>SBayesRC</li>
<li>TL-PRS</li>
<li>ExPRSweb</li>
<li>Cancer PRSweb</li>
<li>PRS atlas</li>
<li>metabolites PRS atlas</li>
</ul>
<h2 id="prs-tutorial">PRS Tutorial</h2>
<ul>
<li>SHORT NAME: PRS Tutorial</li>
<li>FULL NAME: PRS Tutorial</li>
<li>DESCRIPTION: This tutorial provides a step-by-step guide to performing basic polygenic risk score (PRS) analyses and accompanies our PRS Guide paper. The aim of this tutorial is to provide a simple introduction of PRS analyses to those new to PRS, while equipping existing users with a better understanding of the processes and implementation "underneath the hood" of popular PRS software.</li>
<li>URL : https://choishingwan.github.io/PRS-Tutorial/</li>
<li>CITATION: Choi, S. W., Mak, T. S. H., &amp; O’Reilly, P. F. (2020). Tutorial: a guide to performing polygenic risk score analyses. Nature protocols, 15(9), 2759-2772.</li>
</ul>
<h2 id="plink2">PLINK2</h2>
<ul>
<li>SHORT NAME: PLINK2 </li>
<li>FULL NAME: PLINK2</li>
<li>DESCRIPTION: The second-generation versions of PLINK will offer dramatic improvements in performance and compatibility.</li>
<li>URL :  https://www.cog-genomics.org/plink/2.0/</li>
<li>CITATION: Chang, C.C., Chow, C.C., Tellier, L.C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaSci 4, 7 (2015). https://doi.org/10.1186/s13742-015-0047-8</li>
<li>USE: calculate PRS using genotype data.</li>
</ul>
<h2 id="prsice-2">PRSice-2</h2>
<ul>
<li>SHORT NAME: PRSice-2</li>
<li>FULL NAME: PRSice-2</li>
<li>DESCRIPTION: PRSice (pronounced 'precise') is a Polygenic Risk Score software for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS) analyses.</li>
<li>URL : https://www.prsice.info/</li>
<li>CITATION: Shing Wan Choi, Paul F O'Reilly, PRSice-2: Polygenic Risk Score software for biobank-scale data, GigaScience, Volume 8, Issue 7, July 2019, giz082, https://doi.org/10.1093/gigascience/giz082</li>
</ul>
<h2 id="prset">PRSet</h2>
<ul>
<li>SHORT NAME: PRSet</li>
<li>FULL NAME: PRSet</li>
<li>DESCRIPTION: A new feature of PRSice is the ability to perform set base/pathway based analysis. This new feature is called PRSet.</li>
<li>URL : https://www.prsice.info/quick_start_prset/</li>
<li>CITATION: Choi SW, Garcia-Gonzalez J, Ruan Y, et al. The power of pathway-based polygenic risk scores. Research Square; 2021. DOI: 10.21203/rs.3.rs-643696/v1.</li>
<li>KEY WORDS: pathway-based</li>
</ul>
<h2 id="ldpred2">LDpred2</h2>
<ul>
<li>SHORT NAME: LDpred2</li>
<li>FULL NAME: LDpred2</li>
<li>DESCRIPTION: LDpred-2 is one of the dedicated PRS programs which is an R package that uses a Bayesian approach to polygenic risk scoring.</li>
<li>URL : https://privefl.github.io/bigsnpr/articles/LDpred2.html</li>
<li>CITATION: Privé, F., Arbel, J., &amp; Vilhjálmsson, B. J. (2020). LDpred2: better, faster, stronger. Bioinformatics, 36(22-23), 5424-5431.</li>
<li>KEY WORDS: Bayesian</li>
</ul>
<h2 id="ldpred2-auto">LDpred2-auto</h2>
<ul>
<li>SHORT NAME: LDpred2-auto</li>
<li>FULL NAME: LDpred2-auto</li>
<li>DESCRIPTION: LDpred2 is a widely used Bayesian method for building polygenic scores (PGS). LDpred2-auto can infer the two parameters from the LDpred model, h^2 and p, so that it does not require an additional validation dataset to choose best-performing parameters. Here, we present a new version of LDpred2-auto, which adds a third parameter alpha to its model for modeling negative selection. Additional changes are also made to provide better sampling of these parameters. </li>
<li>URL : https://privefl.github.io/bigsnpr/articles/LDpred2.html</li>
<li>CITATION: Inferring disease architecture and predictive ability with LDpred2-auto Florian Privé, Clara Albiñana, Bogdan Pasaniuc, Bjarni J. Vilhjálmsson bioRxiv 2022.10.10.511629; doi: https://doi.org/10.1101/2022.10.10.511629</li>
</ul>
<h2 id="prs-cs">PRS-CS</h2>
<ul>
<li>SHORT NAME: PRS-CS</li>
<li>FULL NAME: PRS-CS</li>
<li>YEAR: 2019</li>
<li>DESCRIPTION: PRS-CS is a Python based command line tool that infers posterior SNP effect sizes under continuous shrinkage (CS) priors using GWAS summary statistics and an external LD reference panel.</li>
<li>URL : https://github.com/getian107/PRScs</li>
<li>CITATION: Ge, T., Chen, CY., Ni, Y. et al. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat Commun 10, 1776 (2019). https://doi.org/10.1038/s41467-019-09718-5</li>
<li>KEY WORDS: continuous shrinkage (CS) prior</li>
</ul>
<h2 id="prs-csx">PRS-CSx</h2>
<ul>
<li>SHORT NAME: PRS-CSx</li>
<li>FULL NAME: PRS-CSx</li>
<li>YEAR: 2022</li>
<li>DESCRIPTION: PRS-CSx is a Python based command line tool that integrates GWAS summary statistics and external LD reference panels from multiple populations to improve cross-population polygenic prediction. Posterior SNP effect sizes are inferred under coupled continuous shrinkage (CS) priors across populations. </li>
<li>URL : https://github.com/getian107/PRScsx</li>
<li>CITATION: Ruan, Y., Lin, YF., Feng, YC.A. et al. Improving polygenic prediction in ancestrally diverse populations. Nat Genet 54, 573–580 (2022). https://doi.org/10.1038/s41588-022-01054-7</li>
<li>KEY WORDS: continuous shrinkage (CS) prior,  cross-population</li>
</ul>
<h2 id="lassosum">lassosum</h2>
<ul>
<li>SHORT NAME: lassosum</li>
<li>FULL NAME: lassosum</li>
<li>YEAR: 2017</li>
<li>DESCRIPTION: lassosum is a method for computing LASSO/Elastic Net estimates of a linear regression problem given summary statistics from GWAS and Genome-wide meta-analyses, accounting for Linkage Disequilibrium (LD), via a reference panel.</li>
<li>URL : https://github.com/tshmak/lassosum</li>
<li>CITATION: Mak, T. S. H., Porsch, R. M., Choi, S. W., Zhou, X., &amp; Sham, P. C. (2017). Polygenic scores via penalized regression on summary statistics. Genetic epidemiology, 41(6), 469-480.</li>
<li>KEY WORDS: penalized regression</li>
</ul>
<h2 id="bayesr">BayesR</h2>
<ul>
<li>SHORT NAME: BayesR</li>
<li>FULL NAME: BayesR</li>
<li>YEAR: 2015</li>
<li>DESCRIPTION: Bayesian mixture model to dissect genetic variation for disease in human populations and to construct more powerful risk predictors</li>
<li>URL:<a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a></li>
<li>CITATION:Moser, G., Lee, S. H., Hayes, B. J., Goddard, M. E., Wray, N. R., &amp; Visscher, P. M. (2015). Simultaneous discovery, estimation and prediction analysis of complex traits using a Bayesian mixture model. PLoS genetics, 11(4), e1004969.</li>
</ul>
<h2 id="bayess">BayesS</h2>
<ul>
<li>SHORT NAME: BayesS</li>
<li>FULL NAME: BayesS</li>
<li>YEAR: 2018</li>
<li>URL:<a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a></li>
<li>CITATION:Zeng, J., De Vlaming, R., Wu, Y., Robinson, M. R., Lloyd-Jones, L. R., Yengo, L., ... &amp; Yang, J. (2018). Signatures of negative selection in the genetic architecture of human complex traits. Nature genetics, 50(5), 746-753.</li>
</ul>
<h2 id="sbayesr">SBayesR</h2>
<ul>
<li>SHORT NAME: SBayesR</li>
<li>FULL NAME: SBayesR</li>
<li>YEAR: 2019</li>
<li>DESCRIPTION: extend a powerful individual-level data Bayesian multiple regression model (BayesR) to one that utilises summary statistics from genome-wide association studies, SBayesR.</li>
<li>URL:<a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a></li>
<li>CITATION:Lloyd-Jones, L. R., Zeng, J., Sidorenko, J., Yengo, L., Moser, G., Kemper, K. E., ... &amp; Visscher, P. M. (2019). Improved polygenic prediction by Bayesian multiple regression on summary statistics. Nature communications, 10(1), 1-11.</li>
</ul>
<h2 id="sbayess">SBayesS</h2>
<ul>
<li>SHORT NAME: SBayesS</li>
<li>FULL NAME: SBayesS</li>
<li>YEAR: 2021</li>
<li>DESCRIPTION: estimate multiple genetic architecture parameters including selection signature using only GWAS summary statistics</li>
<li>URL:<a href="https://cnsgenomics.com/software/gctb/#Overview">https://cnsgenomics.com/software/gctb/#Overview</a></li>
<li>CITATION:Zeng, J., Xue, A., Jiang, L., Lloyd-Jones, L. R., Wu, Y., Wang, H., ... &amp; Yang, J. (2021). Widespread signatures of natural selection across human complex traits and functional genomic categories. Nature communications, 12(1), 1-12.</li>
</ul>
<h2 id="sbayesrc">SBayesRC</h2>
<ul>
<li>SHORT NAME: SBayesRC</li>
<li>FULL NAME: SBayesRC</li>
<li>YEAR: 2022</li>
<li>DESCRIPTION: SBayesRC integrates GWAS summary statistics with functional genomic annotations to improve polygenic prediction of complex traits.</li>
<li>CITATION: Leveraging functional genomic annotations and genome coverage to improve polygenic prediction of complex traits within and between ancestries. Zhili Zheng, Shouye Liu, Julia Sidorenko, Loic Yengo, Patrick Turley, Alireza Ani, Rujia Wang, Ilja M Nolte, Harold Snieder, Lifelines Cohort Study, Jian Yang, Naomi R. Wray, Michael E Goddard, Peter M Visscher, Jian Zeng bioRxiv 2022.10.12.510418; doi: https://doi.org/10.1101/2022.10.12.510418</li>
</ul>
<h2 id="tl-prs">TL-PRS</h2>
<ul>
<li>SHORT NAME: TL-PRS</li>
<li>FULL NAME: transfer learning PRS</li>
<li>YEAR: 2022</li>
<li>URL: https://github.com/ZhangchenZhao/TLPRS</li>
<li>DESCRIPTION: This R package helps users to construct multi-ethnic polygenic risk score (PRS) using transfer learning. It can help predict PRS of minor ancestry using summary statistics from exsiting resources, such as UK Biobank.</li>
<li>CITATION: Zhao, Z., Fritsche, L. G., Smith, J. A., Mukherjee, B., &amp; Lee, S. (2022). The Construction of Multi-ethnic Polygenic Risk Score using Transfer Learning. medRxiv.</li>
</ul>
<h2 id="multi-pgs">Multi-PGS</h2>
<ul>
<li>SHORT NAME: Multi-PGS</li>
<li>YEAR: 2022</li>
<li>FULL NAME: Multi-PGS</li>
<li>DESCRIPTION: a framework to generate enriched PGS from a wealth of publicly available genome-wide association studies, combining thousands of studies focused on many different phenotypes, into a multi-PGS</li>
<li>URL : https://github.com/ClaraAlbi/paper_multiPGS</li>
<li>CITATION: https://www.medrxiv.org/content/10.1101/2022.09.14.22279940v1</li>
</ul>
<h2 id="wmt-sblup">wMT-SBLUP</h2>
<ul>
<li>SHORT NAME: wMT-SBLUP</li>
<li>YEAR: 2018</li>
<li>FULL NAME: weighted approximate multi-trait summary statistic BLUP</li>
<li>URL : https://github.com/uqrmaie1/smtpred</li>
<li>CITATION: Maier, R.M., Zhu, Z., Lee, S.H. et al. Improving genetic prediction by leveraging genetic correlations among human diseases and traits. Nat Commun 9, 989 (2018). https://doi.org/10.1038/s41467-017-02769-6</li>
</ul>
<h2 id="pgscatalog">PGSCatalog</h2>
<ul>
<li>SHORT NAME: PGS Catalog</li>
<li>FULL NAME: PGS Catalog</li>
<li>DESCRIPTION: The PGS Catalog is an open database of published polygenic scores (PGS). Each PGS in the Catalog is consistently annotated with relevant metadata; including scoring files (variants, effect alleles/weights), annotations of how the PGS was developed and applied, and evaluations of their predictive performance.</li>
<li>URL : https://www.pgscatalog.org/</li>
<li>CITATION: Lambert, S. A., Gil, L., Jupp, S., Ritchie, S. C., Xu, Y., Buniello, A., ... &amp; Inouye, M. (2021). The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nature Genetics, 53(4), 420-425.</li>
<li>KEY WORDS: PGS database</li>
</ul>
<h2 id="pgsc_calc">pgsc_calc</h2>
<ul>
<li>SHORT NAME: pgsc_calc</li>
<li>FULL NAME: The Polygenic Score Catalog Calculator</li>
<li>YEAR: 2022</li>
<li>DESCRIPTION: pgsc_calc is a bioinformatics best-practice analysis pipeline for calculating polygenic [risk] scores on samples with imputed genotypes using existing scoring files from the Polygenic Score (PGS) Catalog and/or user-defined PGS/PRS.</li>
<li>URL : https://github.com/PGScatalog/pgsc_calc</li>
<li>CITATION: https://github.com/PGScatalog/pgsc_calc</li>
<li>KEY WORDS: PRS calculation pipeline</li>
</ul>
<h2 id="viprs">VIPRS</h2>
<ul>
<li>SHORT NAME: VIPRS</li>
<li>FULL NAME: Variational inference of polygenic risk scores</li>
<li>YEAR: 2022</li>
<li>DESCRIPTION: viprs is a python package that implements scripts and utilities for running variational inference algorithms on genome-wide association study (GWAS) data for the purposes polygenic risk estimation.</li>
<li>URL : https://github.com/shz9/viprs</li>
<li>CITATION: Zabad, S., Gravel, S., &amp; Li, Y. (2022). Fast and Accurate Bayesian Polygenic Risk Modeling with Variational Inference. bioRxiv.</li>
<li>KEY WORDS:  Variational Inference (VI)</li>
</ul>
<h2 id="exprsweb">ExPRSweb</h2>
<ul>
<li>SHORT NAME: ExPRSweb</li>
<li>FULL NAME: exposure polygenic risk scores (ExPRSs)</li>
<li>YEAR: 2022</li>
<li>DESCRIPTION: Integrating published and freely available genome-wide association studies (GWAS) summary statistics from multiple sources (published GWAS, the NHGRI-EBI GWAS Catalog, FinnGen- or UKB-based GWAS), we created an online repository for exposure polygenic risk scores (ExPRS) for health-related exposure traits. Our framework condenses these summary statistics into ExPRS using linkage disequilibrium pruning and p-value thresholding (P&amp;T) or penalized, genome-wide effect size weighting. We evaluate them in the cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each ExPRS construct, measures on performance, accuracy, and discrimination are provided. Beyond the ExPRS evaluation in MGI and UKB, the ExPRSweb platform features construct downloads, evaluation in the top percentiles, and phenome-wide ExPRS association studies (ExPRS-PheWAS) for a subset of ExPRS that are predictive for the corresponding exposure.</li>
<li>URL : https://exprsweb.sph.umich.edu:8443/</li>
<li>CITATION: Ma, Y., Patil, S., Zhou, X., Mukherjee, B., &amp; Fritsche, L. G. (2022). ExPRSweb-An Online Repository with Polygenic Risk Scores for Common Health-related Exposures. medRxiv.</li>
<li>KEY WORDS: exposure PRS</li>
</ul>
<h2 id="cancer-prsweb">Cancer PRSweb</h2>
<ul>
<li>SHORT NAME: Cancer PRSweb</li>
<li>FULL NAME: Cancer PRSweb</li>
<li>YEAR: 2020</li>
<li>DESCRIPTION: Our framework condenses these summary statistics into PRS using linkage disequilibrium pruning and p-value thresholding (fixed or data-adaptively optimized thresholds) or penalized, genome-wide effect size weighting. We evaluate them in the cancer-enriched cohort of the Michigan Genomics Initiative (MGI), a longitudinal biorepository effort at Michigan Medicine, and in the population-based UK Biobank Study (UKB). For each PRS construct, measures on performance, calibration, and discrimination are provided. Beyond the cancer PRS evaluation in MGI and UKB, the PRSweb platform features construct downloads, risk evaluation in the top percentiles, and phenome-wide PRS association studies (PRS-PheWAS) for a subset of PRS that are predictive for the primary cancer.</li>
<li>URL : https://prsweb.sph.umich.edu:8443/</li>
<li>CITATION: Fritsche, L. G., Patil, S., Beesley, L. J., VandeHaar, P., Salvatore, M., Ma, Y., ... &amp; Mukherjee, B. (2020). Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. The American Journal of Human Genetics, 107(5), 815-836.</li>
<li>KEY WORDS: Cancer PRS</li>
</ul>
<h2 id="prs-atlas">PRS atlas</h2>
<ul>
<li>SHORT NAME: PRS atlas</li>
<li>FULL NAME: PRS atlas</li>
<li>DESCRIPTION: This web application can be used to query findings from an analysis of 162 polygenic risk scores and 551 complex traits using data from the UK Biobank study1. Traits were selected based on the heritability analysis conducted by the Neale Lab2 (P&lt;0.05). We encourage users of this resource to conduct follow-up analyses of associations to robustly identify causal relationships between complex traits.</li>
<li>URL : http://mrcieu.mrsoftware.org/PRS_atlas/</li>
<li>CITATION: Richardson, T. G., Harrison, S., Hemani, G., &amp; Smith, G. D. (2019). An atlas of polygenic risk score associations to highlight putative causal relationships across the human phenome. Elife, 8, e43657.</li>
</ul>
<h2 id="metabolites-prs-atlas">metabolites PRS atlas</h2>
<ul>
<li>SHORT NAME: metabolites PRS atlas</li>
<li>FULL NAME: metabolites PRS atlas</li>
<li>DESCRIPTION: This web application can be used to query findings from a systematic analysis of 129 polygenic risk scores and 249 circulating metabolits using high-throughput nuclear magnetic resonance data from the UK Biobank study1,2. We encourage users of this resource to conduct follow-up analyses of associations to investigate potential causal and non-causal metabolic biomarkers. Age-stratified results can be used to investigate how potential sources of collider bias (e.g. statin therapy) may influence findings in the full sample</li>
<li>URL : http://mrcieu.mrsoftware.org/metabolites_PRS_atlas/</li>
<li>CITATION:Fang, S., Holmes, M. V., Gaunt, T. R., Smith, G. D., &amp; Richardson, T. G. (2022). Constructing an atlas of associations between polygenic scores from across the human phenome and circulating metabolic biomarkers. eLife.</li>
</ul>
<h2 id="reviewstutorial">Reviews&amp;Tutorial</h2>
<ul>
<li>CITATION: Peter M. Visscher,Loic Yengo,Nancy J. Cox,Naomi R. Wray,Discovery and implications of polygenicity of common diseases, Science, 373, 6562, (1468-1473), (2021). /doi/10.1126/science.abi8206</li>
<li>CITATION: Wang, Y., Tsuo, K., Kanai, M., Neale, B. M., &amp; Martin, A. R. (2022). Challenges and Opportunities for Developing More Generalizable Polygenic Risk Scores. Annual Review of Biomedical Data Science, 5.</li>
</ul></div>
        
        
    </div>

    
      <footer class="col-md-12 text-center">
          
          
            <hr>
            <p>
            <small>CTGCatalog is licensed under the MIT license</small><br>
            
            <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</small>
            </p>
          

          
          
      </footer>
    
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="../js/bootstrap-3.0.3.min.js"></script>

    
    <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/highlight.min.js"></script>
        
    <script>hljs.initHighlightingOnLoad();</script>
    

    <script>var base_url = ".."</script>
    
    <script src="../js/base.js"></script>
    <script src="../search/main.js"></script>

    <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal">
                    <span aria-hidden="true">&times;</span>
                    <span class="sr-only">Close</span>
                </button>
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
            </div>
            <div class="modal-body">
                <p>
                    From here you can search these documents. Enter
                    your search terms below.
                </p>
                <form>
                    <div class="form-group">
                        <input type="text" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>
    </body>

</html>
