# Contents
- PLINK1.9
- PLINK2
- GCTA
- LDSC
- QCTOOLS
- bgenix
- GCTA
- mungesumstats
- GWASLab

## PLINK1.9
- SHORT NAME: PLINK1.9
- FULL NAME: PLINK1.9
- YEAR: 2007
- DESCRIPTION:  PLINK: a tool set for whole-genome association and population-based linkage analyses. This is a comprehensive update to Shaun Purcell's PLINK command-line program, developed by Christopher Chang with support from the NIH-NIDDK's Laboratory of Biological Modeling, the Purcell Lab, and others. 
- URL : https://www.cog-genomics.org/plink/1.9/
- CITATION: Purcell, Shaun, et al. "PLINK: a tool set for whole-genome association and population-based linkage analyses." The American journal of human genetics 81.3 (2007): 559-575. 

## PLINK2
- SHORT NAME: PLINK2
- FULL NAME: PLINK2
- YEAR: 2015
- DESCRIPTION: The second-generation versions of PLINK will offer dramatic improvements in performance and compatibility. For the first time, users without access to high-end computing resources can perform several essential analyses of the feature-rich and very large genetic datasets coming into use.
- URL : https://www.cog-genomics.org/plink/2.0/
- CITATION: Chang, Christopher C., et al. "Second-generation PLINK: rising to the challenge of larger and richer datasets." Gigascience 4.1 (2015): s13742-015.

## QCTOOL v2
- SHORT NAME: QCTOOL
- FULL NAME: QCTOOL
- DESCRIPTION: QCTOOL is a command-line utility program for manipulation and quality control of gwas datasets and other genome-wide data.
- URL : https://www.well.ox.ac.uk/~gav/qctool_v2/index.html

## Hail
- SHORT NAME: Hail
- FULL NAME: Hail
- DESCRIPTION: Hail is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data.
- URL : https://hail.is/
- CITATION: Hail Team. Hail 0.2.13-81ab564db2b4. https://github.com/hail-is/hail/releases/tag/0.2.13.

## LDSC - ld score calculation
- SHORT NAME:LDSC
- FULL NAME:LD Score Regression
- YEAR:2015
- DESCRIPTION:ldsc is a command line tool for estimating heritability and genetic correlation from GWAS summary statistics. ldsc also computes LD Scores.
- URL:https://github.com/bulik/ldsc
- CITATION :Bulik-Sullivan, Brendan K., et al. "LD Score regression distinguishes confounding from polygenicity in genome-wide association studies." Nature genetics 47.3 (2015): 291-295.

## GCTA - ld matrix
- SHORT NAME: GCTA
- FULL NAME:Genome-wide complex trait analysis (GCTA)
- YEAR:2010
- DESCRIPTION:GCTA-GREML analysis:GCTA can simulate a GWAS based on real genotype data.
- URL: https://yanglab.westlake.edu.cn/software/gcta/

## LDSTORE2
- SHORT NAME: LDSTORE2
- FULL NAME: LDSTORE2
- YEAR: 2017
- DESCRIPTION: LDstore is a computationally efficient program for estimating and storing Linkage Disequilibrium (SNP correlations). It combines some of the best features from RAREMETALWORKER and PLINK by implementing parallel processing using OPENMP and storing of the SNP correlations with information about the SNPs in the same binary file for fast lookups. LDstore is therefore the ideal tool for sharing SNP correlations in large-scale meta-analyses of genome-wide association studies and for on-the-fly computing/querying within web portals.
- URL : http://www.christianbenner.com/#
- CITATION: Benner, C. et al. Prospects of fine-papping trait-associated genomic regions by using summary statistics from genome-wide association studies. Am. J. Hum. Genet. (2017).


## MungeSumstats
- SHORT NAME: MungeSumstats
- FULL NAME: MungeSumstats
- YEAR: 2021
- DESCRIPTION: a Bioconductor package for the standardization and quality control of many GWAS summary statistics
- URL : https://github.com/neurogenomics/MungeSumstats
- CITATION: Murphy, Alan E., Brian M. Schilder, and Nathan G. Skene. "MungeSumstats: a Bioconductor package for the standardization and quality control of many GWAS summary statistics." Bioinformatics 37.23 (2021): 4593-4596.

## GWASLab
- SHORT NAME: GWASLab
- FULL NAME: GWASLab
- YEAR: 2022
- DESCRIPTION: a python package for handling GWAS sumstats.
- URL : https://github.com/Cloufield/gwaslab
- CITATION: GWASLab preprint: He, Y., Koido, M., Shimmori, Y., Kamatani, Y. (2023). GWASLab: a Python package for processing and visualizing GWAS summary statistics. Preprint at Jxiv, 2023-5. https://doi.org/10.51094/jxiv.370
